<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>Count features in TCGA PRAD small RNA sequencing data</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>

<!-- Styles for R syntax highlighter -->
<style type="text/css">
   pre .operator,
   pre .paren {
     color: rgb(104, 118, 135)
   }

   pre .literal {
     color: #990073
   }

   pre .number {
     color: #099;
   }

   pre .comment {
     color: #998;
     font-style: italic
   }

   pre .keyword {
     color: #900;
     font-weight: bold
   }

   pre .identifier {
     color: rgb(0, 0, 0);
   }

   pre .string {
     color: #d14;
   }
</style>

<!-- R syntax highlighter -->
<script type="text/javascript">
var hljs=new function(){function m(p){return p.replace(/&/gm,"&amp;").replace(/</gm,"&lt;")}function f(r,q,p){return RegExp(q,"m"+(r.cI?"i":"")+(p?"g":""))}function b(r){for(var p=0;p<r.childNodes.length;p++){var q=r.childNodes[p];if(q.nodeName=="CODE"){return q}if(!(q.nodeType==3&&q.nodeValue.match(/\s+/))){break}}}function h(t,s){var p="";for(var r=0;r<t.childNodes.length;r++){if(t.childNodes[r].nodeType==3){var q=t.childNodes[r].nodeValue;if(s){q=q.replace(/\n/g,"")}p+=q}else{if(t.childNodes[r].nodeName=="BR"){p+="\n"}else{p+=h(t.childNodes[r])}}}if(/MSIE [678]/.test(navigator.userAgent)){p=p.replace(/\r/g,"\n")}return p}function a(s){var r=s.className.split(/\s+/);r=r.concat(s.parentNode.className.split(/\s+/));for(var q=0;q<r.length;q++){var p=r[q].replace(/^language-/,"");if(e[p]){return p}}}function c(q){var p=[];(function(s,t){for(var r=0;r<s.childNodes.length;r++){if(s.childNodes[r].nodeType==3){t+=s.childNodes[r].nodeValue.length}else{if(s.childNodes[r].nodeName=="BR"){t+=1}else{if(s.childNodes[r].nodeType==1){p.push({event:"start",offset:t,node:s.childNodes[r]});t=arguments.callee(s.childNodes[r],t);p.push({event:"stop",offset:t,node:s.childNodes[r]})}}}}return t})(q,0);return p}function k(y,w,x){var q=0;var z="";var s=[];function u(){if(y.length&&w.length){if(y[0].offset!=w[0].offset){return(y[0].offset<w[0].offset)?y:w}else{return w[0].event=="start"?y:w}}else{return y.length?y:w}}function t(D){var A="<"+D.nodeName.toLowerCase();for(var B=0;B<D.attributes.length;B++){var C=D.attributes[B];A+=" "+C.nodeName.toLowerCase();if(C.value!==undefined&&C.value!==false&&C.value!==null){A+='="'+m(C.value)+'"'}}return A+">"}while(y.length||w.length){var v=u().splice(0,1)[0];z+=m(x.substr(q,v.offset-q));q=v.offset;if(v.event=="start"){z+=t(v.node);s.push(v.node)}else{if(v.event=="stop"){var p,r=s.length;do{r--;p=s[r];z+=("</"+p.nodeName.toLowerCase()+">")}while(p!=v.node);s.splice(r,1);while(r<s.length){z+=t(s[r]);r++}}}}return z+m(x.substr(q))}function j(){function q(x,y,v){if(x.compiled){return}var u;var s=[];if(x.k){x.lR=f(y,x.l||hljs.IR,true);for(var w in x.k){if(!x.k.hasOwnProperty(w)){continue}if(x.k[w] instanceof Object){u=x.k[w]}else{u=x.k;w="keyword"}for(var r in u){if(!u.hasOwnProperty(r)){continue}x.k[r]=[w,u[r]];s.push(r)}}}if(!v){if(x.bWK){x.b="\\b("+s.join("|")+")\\s"}x.bR=f(y,x.b?x.b:"\\B|\\b");if(!x.e&&!x.eW){x.e="\\B|\\b"}if(x.e){x.eR=f(y,x.e)}}if(x.i){x.iR=f(y,x.i)}if(x.r===undefined){x.r=1}if(!x.c){x.c=[]}x.compiled=true;for(var t=0;t<x.c.length;t++){if(x.c[t]=="self"){x.c[t]=x}q(x.c[t],y,false)}if(x.starts){q(x.starts,y,false)}}for(var p in e){if(!e.hasOwnProperty(p)){continue}q(e[p].dM,e[p],true)}}function d(B,C){if(!j.called){j();j.called=true}function q(r,M){for(var L=0;L<M.c.length;L++){if((M.c[L].bR.exec(r)||[null])[0]==r){return M.c[L]}}}function v(L,r){if(D[L].e&&D[L].eR.test(r)){return 1}if(D[L].eW){var M=v(L-1,r);return M?M+1:0}return 0}function w(r,L){return L.i&&L.iR.test(r)}function K(N,O){var M=[];for(var L=0;L<N.c.length;L++){M.push(N.c[L].b)}var r=D.length-1;do{if(D[r].e){M.push(D[r].e)}r--}while(D[r+1].eW);if(N.i){M.push(N.i)}return f(O,M.join("|"),true)}function p(M,L){var N=D[D.length-1];if(!N.t){N.t=K(N,E)}N.t.lastIndex=L;var r=N.t.exec(M);return r?[M.substr(L,r.index-L),r[0],false]:[M.substr(L),"",true]}function z(N,r){var L=E.cI?r[0].toLowerCase():r[0];var M=N.k[L];if(M&&M instanceof Array){return M}return false}function F(L,P){L=m(L);if(!P.k){return L}var r="";var O=0;P.lR.lastIndex=0;var M=P.lR.exec(L);while(M){r+=L.substr(O,M.index-O);var N=z(P,M);if(N){x+=N[1];r+='<span class="'+N[0]+'">'+M[0]+"</span>"}else{r+=M[0]}O=P.lR.lastIndex;M=P.lR.exec(L)}return r+L.substr(O,L.length-O)}function J(L,M){if(M.sL&&e[M.sL]){var r=d(M.sL,L);x+=r.keyword_count;return r.value}else{return F(L,M)}}function I(M,r){var L=M.cN?'<span class="'+M.cN+'">':"";if(M.rB){y+=L;M.buffer=""}else{if(M.eB){y+=m(r)+L;M.buffer=""}else{y+=L;M.buffer=r}}D.push(M);A+=M.r}function G(N,M,Q){var R=D[D.length-1];if(Q){y+=J(R.buffer+N,R);return false}var P=q(M,R);if(P){y+=J(R.buffer+N,R);I(P,M);return P.rB}var L=v(D.length-1,M);if(L){var O=R.cN?"</span>":"";if(R.rE){y+=J(R.buffer+N,R)+O}else{if(R.eE){y+=J(R.buffer+N,R)+O+m(M)}else{y+=J(R.buffer+N+M,R)+O}}while(L>1){O=D[D.length-2].cN?"</span>":"";y+=O;L--;D.length--}var r=D[D.length-1];D.length--;D[D.length-1].buffer="";if(r.starts){I(r.starts,"")}return R.rE}if(w(M,R)){throw"Illegal"}}var E=e[B];var D=[E.dM];var A=0;var x=0;var y="";try{var s,u=0;E.dM.buffer="";do{s=p(C,u);var t=G(s[0],s[1],s[2]);u+=s[0].length;if(!t){u+=s[1].length}}while(!s[2]);if(D.length>1){throw"Illegal"}return{r:A,keyword_count:x,value:y}}catch(H){if(H=="Illegal"){return{r:0,keyword_count:0,value:m(C)}}else{throw H}}}function g(t){var p={keyword_count:0,r:0,value:m(t)};var r=p;for(var q in e){if(!e.hasOwnProperty(q)){continue}var s=d(q,t);s.language=q;if(s.keyword_count+s.r>r.keyword_count+r.r){r=s}if(s.keyword_count+s.r>p.keyword_count+p.r){r=p;p=s}}if(r.language){p.second_best=r}return p}function i(r,q,p){if(q){r=r.replace(/^((<[^>]+>|\t)+)/gm,function(t,w,v,u){return w.replace(/\t/g,q)})}if(p){r=r.replace(/\n/g,"<br>")}return r}function n(t,w,r){var x=h(t,r);var v=a(t);var y,s;if(v){y=d(v,x)}else{return}var q=c(t);if(q.length){s=document.createElement("pre");s.innerHTML=y.value;y.value=k(q,c(s),x)}y.value=i(y.value,w,r);var u=t.className;if(!u.match("(\\s|^)(language-)?"+v+"(\\s|$)")){u=u?(u+" "+v):v}if(/MSIE [678]/.test(navigator.userAgent)&&t.tagName=="CODE"&&t.parentNode.tagName=="PRE"){s=t.parentNode;var p=document.createElement("div");p.innerHTML="<pre><code>"+y.value+"</code></pre>";t=p.firstChild.firstChild;p.firstChild.cN=s.cN;s.parentNode.replaceChild(p.firstChild,s)}else{t.innerHTML=y.value}t.className=u;t.result={language:v,kw:y.keyword_count,re:y.r};if(y.second_best){t.second_best={language:y.second_best.language,kw:y.second_best.keyword_count,re:y.second_best.r}}}function o(){if(o.called){return}o.called=true;var r=document.getElementsByTagName("pre");for(var p=0;p<r.length;p++){var q=b(r[p]);if(q){n(q,hljs.tabReplace)}}}function l(){if(window.addEventListener){window.addEventListener("DOMContentLoaded",o,false);window.addEventListener("load",o,false)}else{if(window.attachEvent){window.attachEvent("onload",o)}else{window.onload=o}}}var e={};this.LANGUAGES=e;this.highlight=d;this.highlightAuto=g;this.fixMarkup=i;this.highlightBlock=n;this.initHighlighting=o;this.initHighlightingOnLoad=l;this.IR="[a-zA-Z][a-zA-Z0-9_]*";this.UIR="[a-zA-Z_][a-zA-Z0-9_]*";this.NR="\\b\\d+(\\.\\d+)?";this.CNR="\\b(0[xX][a-fA-F0-9]+|(\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)";this.BNR="\\b(0b[01]+)";this.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|\\.|-|-=|/|/=|:|;|<|<<|<<=|<=|=|==|===|>|>=|>>|>>=|>>>|>>>=|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~";this.ER="(?![\\s\\S])";this.BE={b:"\\\\.",r:0};this.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[this.BE],r:0};this.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[this.BE],r:0};this.CLCM={cN:"comment",b:"//",e:"$"};this.CBLCLM={cN:"comment",b:"/\\*",e:"\\*/"};this.HCM={cN:"comment",b:"#",e:"$"};this.NM={cN:"number",b:this.NR,r:0};this.CNM={cN:"number",b:this.CNR,r:0};this.BNM={cN:"number",b:this.BNR,r:0};this.inherit=function(r,s){var p={};for(var q in r){p[q]=r[q]}if(s){for(var q in s){p[q]=s[q]}}return p}}();hljs.LANGUAGES.cpp=function(){var a={keyword:{"false":1,"int":1,"float":1,"while":1,"private":1,"char":1,"catch":1,"export":1,virtual:1,operator:2,sizeof:2,dynamic_cast:2,typedef:2,const_cast:2,"const":1,struct:1,"for":1,static_cast:2,union:1,namespace:1,unsigned:1,"long":1,"throw":1,"volatile":2,"static":1,"protected":1,bool:1,template:1,mutable:1,"if":1,"public":1,friend:2,"do":1,"return":1,"goto":1,auto:1,"void":2,"enum":1,"else":1,"break":1,"new":1,extern:1,using:1,"true":1,"class":1,asm:1,"case":1,typeid:1,"short":1,reinterpret_cast:2,"default":1,"double":1,register:1,explicit:1,signed:1,typename:1,"try":1,"this":1,"switch":1,"continue":1,wchar_t:1,inline:1,"delete":1,alignof:1,char16_t:1,char32_t:1,constexpr:1,decltype:1,noexcept:1,nullptr:1,static_assert:1,thread_local:1,restrict:1,_Bool:1,complex:1},built_in:{std:1,string:1,cin:1,cout:1,cerr:1,clog:1,stringstream:1,istringstream:1,ostringstream:1,auto_ptr:1,deque:1,list:1,queue:1,stack:1,vector:1,map:1,set:1,bitset:1,multiset:1,multimap:1,unordered_set:1,unordered_map:1,unordered_multiset:1,unordered_multimap:1,array:1,shared_ptr:1}};return{dM:{k:a,i:"</",c:[hljs.CLCM,hljs.CBLCLM,hljs.QSM,{cN:"string",b:"'\\\\?.",e:"'",i:"."},{cN:"number",b:"\\b(\\d+(\\.\\d*)?|\\.\\d+)(u|U|l|L|ul|UL|f|F)"},hljs.CNM,{cN:"preprocessor",b:"#",e:"$"},{cN:"stl_container",b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:a,r:10,c:["self"]}]}}}();hljs.LANGUAGES.r={dM:{c:[hljs.HCM,{cN:"number",b:"\\b0[xX][0-9a-fA-F]+[Li]?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+(?:[eE][+\\-]?\\d*)?L\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+\\.(?!\\d)(?:i\\b)?",e:hljs.IMMEDIATE_RE,r:1},{cN:"number",b:"\\b\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"keyword",b:"(?:tryCatch|library|setGeneric|setGroupGeneric)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\.",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\d+(?![\\w.])",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\b(?:function)",e:hljs.IMMEDIATE_RE,r:2},{cN:"keyword",b:"(?:if|in|break|next|repeat|else|for|return|switch|while|try|stop|warning|require|attach|detach|source|setMethod|setClass)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"literal",b:"(?:NA|NA_integer_|NA_real_|NA_character_|NA_complex_)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"literal",b:"(?:NULL|TRUE|FALSE|T|F|Inf|NaN)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"identifier",b:"[a-zA-Z.][a-zA-Z0-9._]*\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"<\\-(?!\\s*\\d)",e:hljs.IMMEDIATE_RE,r:2},{cN:"operator",b:"\\->|<\\-",e:hljs.IMMEDIATE_RE,r:1},{cN:"operator",b:"%%|~",e:hljs.IMMEDIATE_RE},{cN:"operator",b:">=|<=|==|!=|\\|\\||&&|=|\\+|\\-|\\*|/|\\^|>|<|!|&|\\||\\$|:",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"%",e:"%",i:"\\n",r:1},{cN:"identifier",b:"`",e:"`",r:0},{cN:"string",b:'"',e:'"',c:[hljs.BE],r:0},{cN:"string",b:"'",e:"'",c:[hljs.BE],r:0},{cN:"paren",b:"[[({\\])}]",e:hljs.IMMEDIATE_RE,r:0}]}};
hljs.initHighlightingOnLoad();
</script>



<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<h1>Count features in TCGA PRAD small RNA sequencing data</h1>

<p><em>Last modified 05:06 PM on Jun 19, 2015. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/prad/feature-counts/</code>.</em></p>

<pre><code class="r">library(GenomicRanges)
library(GenomicAlignments)
library(Biobase)
library(stringr)
library(DESeq2)
library(ggplot2)
source(&#39;/inside/grotto/blin/programs/convertChrNames.R&#39;)
load(&#39;/inside/grotto/blin/trna-markers/process-reads/prad-metadata.RData&#39;)
</code></pre>

<h2>Building a features list</h2>

<h3>tRFs</h3>

<p>Recently, <a href="http://genome.bioch.virginia.edu/trfdb/">tRFdb</a> was released, with 552 human tRNA fragments. These were downloaded into <code>/inside/grotto/blin/data/tRFdb-human.tab</code>, and processed into a better file format, <code>tRFdb-human.gtf</code>. I can read this in and convert to a <code>GRanges</code> object.</p>

<pre><code class="r">frags &lt;- read.delim(&#39;/inside/grotto/blin/data/tRFdb-human.gtf&#39;, comment.char = &quot;#&quot;, header = FALSE, col.names = c(&quot;seqid&quot;, &quot;source&quot;, &quot;type&quot;, &quot;start&quot;, &quot;end&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;phase&quot;, &quot;attributes&quot;))
frags$strand &lt;- ifelse(frags$start - frags$end &lt; 0, &quot;+&quot;, &quot;-&quot;)
frags[frags$strand == &quot;-&quot;, c(&quot;start&quot;, &quot;end&quot;)] &lt;- frags[frags$strand == &quot;-&quot;, c(&quot;end&quot;, &quot;start&quot;)]
frags &lt;- GRanges(seqnames = frags$seqid, ranges = IRanges(frags$start, frags$end), strand = ifelse(frags$start - frags$end &lt; 0, &quot;+&quot;, &quot;-&quot;), tx_name = paste0(str_extract(frags$attributes, &quot;(chr.+trna\\d+-\\w+)&quot;), &quot;-&quot;, str_extract(frags$attributes, &quot;\\d\\d\\d\\d\\w?&quot;)))
mcols(frags)$class &lt;- &quot;tRF1&quot;
mcols(frags[str_detect(mcols(frags)$tx_name, &quot;-3&quot;)])$class &lt;- &quot;tRF3&quot;
mcols(frags[str_detect(mcols(frags)$tx_name, &quot;-5&quot;)])$class &lt;- &quot;tRF5&quot;
frags$tx_name &lt;- str_match(unlist(str_split(frags$tx_name, &quot;\\-&quot;, 2))[seq(2, length(frags)*2, 2)], &quot;\\w+-\\d+&quot;)[, 1]
frags &lt;- frags[!duplicated(frags$tx_name)]
frags &lt;- convertChrNames(frags, &quot;Ensembl&quot;)
</code></pre>

<!-- 
TCGA miRNA-Seq protocol filters these out through size selection.
### Angiogenin dependent tRNA halves

Andrew has generated a feature list (available at `/inside/grotto/blin/trna-markers/prad/feature-counts/hg19-trnahalfs.gtf`), so we'll use that for now. It's stored as a `.gtf` file, so it follows a similar process as above. Currently, there is an issue with Andrew's covariance model that results in reversed 5' tRFs, with end = start - 1. For the time being, those features have been deleted from the list. This list 

 features-halves}
halves <- read.delim('hg19-trnahalves.gtf', header = FALSE, col.names = c("seqid", "source", "type", "start", "end", "score", "strand", "phase", "attributes"))
halves <- GRanges(seqnames = halves$seqid, ranges = IRanges(halves$start, halves$end), strand = halves$strand, tx_name = str_extract(halves$attributes, "(tRNA|chr\\w+\\.\\w+|nmt)(-\\w+\\?*)+_(threehalf|fivehalf|trailer)"))
mcols(halves)$class <- "trailer"
mcols(halves[str_detect(mcols(halves)$tx_name, "threehalf")])$class <- "3half"
mcols(halves[str_detect(mcols(halves)$tx_name, "fivehalf")])$class <- "5half"
halves <- convertChrNames(halves, "Ensembl")
 -->

<h3>microRNAs</h3>

<p>I had what Aaron called an &ldquo;A-list&rdquo; of miRNAs involved in prostate cancer. These are now documented in <a href="correlate-mirnas.html">correlate-mirnas</a>. I am now following suggestions to just throw the entire human miRNA pool in and extracting what we need later.</p>

<p>Combining the miRNAs with the tRFs is simple with <code>GRanges</code>. We just need to read them in first. The miRNAs are formatted as a bed file. In the interest of having a more complete set of data, I am using the entire miRNA primary transcript instead of miRNA levels.</p>

<pre><code class="r">mirnas &lt;- read.delim(&#39;/inside/grotto/blin/data/mir-hsa.gff3&#39;, header = FALSE, stringsAsFactors = FALSE, comment.char = &quot;#&quot;, col.names = c(&quot;seqid&quot;, &quot;source&quot;, &quot;type&quot;, &quot;start&quot;, &quot;end&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;phase&quot;, &quot;attributes&quot;))
mirnas &lt;- GRanges(seqnames = mirnas$seqid, ranges = IRanges(start = mirnas$start, end = mirnas$end), strand = mirnas$strand, tx_name = str_extract(mirnas$attributes, &quot;hsa(-\\w+)+&quot;))
mirnas &lt;- convertChrNames(mirnas, &quot;Ensembl&quot;)
mcols(mirnas)$class &lt;- &quot;miRNA&quot;
mirnas &lt;- mirnas[!duplicated(mirnas$tx_name)]
features &lt;- c(frags, mirnas) 
</code></pre>

<h2>Count features</h2>

<h3>Small RNA reads</h3>

<p>Reads for TCGA miRNA tend to be between 20-30 bp. If the overlap between a locus and a pair of reads is greater than or equal to 5 bp, we consider it one transcript spanning the feature. The following function is similar to the above, and returns a matrix with rows as features and columns as the sample. We don&#39;t need to divide the counts by feature length because miRNA sequencing is typically not fragmented prior to sequencing. </p>

<pre><code class="r">countFeaturesFromBam &lt;- function(bamfiles, features, dir = &quot;.&quot;) {
  bamfiles &lt;- paste0(dir, &quot;/&quot;, bamfiles, &quot;-mapped.bam&quot;)
  counts &lt;- vector()
  # use for loop because r will run out of memory
  for (bamfile in bamfiles) {
    sample &lt;- readGAlignments(bamfile)
    sample &lt;- convertChrNames(sample, &quot;Ensembl&quot;)
    counts &lt;- cbind(counts, countOverlaps(features, sample, minoverlap = 5))
  }
  counts
}
</code></pre>

<pre><code class="r">raw_counts &lt;- countFeaturesFromBam(prad_metadata$shortnames, features, &#39;/inside/grotto/blin/trna-markers/mirna/prad&#39;)
rownames(raw_counts) &lt;- features$tx_name
colnames(raw_counts) &lt;- prad_metadata$barcode
raw_counts &lt;- as.data.frame(raw_counts)
</code></pre>

<h3>Gene counts</h3>

<p>Top 10 genes recommended by <a href="http://link.springer.com/article/10.1007/s00109-005-0703-z">this paper</a> for prostate cancer housekeeping genes: HPRT1, ALAS1, K-ALPHA-1 (TUBA1B), SDHA, POLR2A, G6PD, GAPD (GAPDH), TBP, PPIA, YWHAZ</p>

<p>Actually, I changed my mind - just going to grab all gene counts here.</p>

<pre><code class="r">getGeneCounts &lt;- function(rnaseq_dir, rnaseq_metadata_filename, mirna_metadata) {
  rnaseq_files &lt;- list.files(rnaseq_dir, full.names = TRUE)
  rnaseq_metadata &lt;- read.delim(paste0(rnaseq_dir, rnaseq_metadata_filename), header = TRUE, stringsAsFactors = FALSE)
  # split at character index 15, e.g. TCGA-ZG-A9N3-01
  rnaseq_metadata &lt;- data.frame(barcode = substr(rnaseq_metadata$Comment..TCGA.Barcode., 1, 15), analysis_id = rnaseq_metadata$Extract.Name, stringsAsFactors = FALSE)
  counts &lt;- vector()
  for (barcode in mirna_metadata$barcode) {
    barcode4 &lt;- substr(barcode, 1, 15)
    analysis_id &lt;- rnaseq_metadata$analysis_id[match(barcode4, rnaseq_metadata$barcode)]
    if (is.na(str_detect(rnaseq_files, analysis_id))) {
        counts &lt;- cbind(counts, NA)
        next
    }
    gene_counts_file &lt;- rnaseq_files[which(str_detect(rnaseq_files, analysis_id))]
    gene_counts &lt;- read.table(gene_counts_file, row.names = 1, check.names = TRUE, stringsAsFactors = FALSE, header = TRUE)
    rownames(gene_counts) &lt;- make.unique(str_replace(rownames(gene_counts), &#39;\\|\\d+&#39;, &#39;&#39;))
    gene_counts &lt;- gene_counts[!grepl(&quot;\\?&quot;, rownames(gene_counts)), ]
    gene_ids &lt;- rownames(gene_counts)
    gene_counts &lt;- gene_counts$raw_count
    names(gene_counts) &lt;- gene_ids
    counts &lt;- cbind(counts, gene_counts)
  }
  colnames(counts) &lt;- mirna_metadata$barcode
  counts
}
</code></pre>

<pre><code class="r">gene_counts &lt;- getGeneCounts(&#39;/inside/home/blin/grotto/trna-markers/datasets/prad/rnaseq/&#39;, &#39;unc.edu_PRAD.IlluminaHiSeq_RNASeqV2.1.16.0.sdrf.txt&#39;, prad_metadata)
feature_counts &lt;- rbind(raw_counts, gene_counts)
</code></pre>

<h2>Cleaning and normalizing data</h2>

<pre><code class="r">feature_counts &lt;- raw_counts
feature_counts &lt;- feature_counts[apply(feature_counts, 1, function(row) length(which(row &gt; 0)) &gt; length(colnames(feature_counts)) * 0.05), ] # no. of samples with reads mapping to this feature &gt; no. samples * 0.05. at least 1/20 of the samples must have reads for this feature to be considered
feature_counts &lt;- feature_counts[apply(feature_counts, 1, function(row) sum(row, na.rm = TRUE) &gt; 20), ]
feature_counts &lt;- feature_counts[!apply(feature_counts, 1, function(row) any(is.na(row))), ]
</code></pre>

<pre><code class="r">colData &lt;- data.frame(row.names = colnames(feature_counts), patient = as.factor(prad_metadata$participant_id), sample_type = as.factor(prad_metadata$sample_type), type = &quot;single-read&quot;) # feature count column names correspond to metadata rows
deseq_prad_dds &lt;- DESeqDataSetFromMatrix(countData = feature_counts, colData = colData, design = ~ sample_type)
deseq_prad_dds &lt;- DESeq(deseq_prad_dds)
normalized_counts &lt;- counts(deseq_prad_dds, normalized = TRUE) 
deseq_prad &lt;- results(deseq_prad_dds)
</code></pre>

<p>The below section is now deprecated, since we are no longer using gene counts for normalization.</p>

<h2>Cleaning and normalizing data - DEPRECATED</h2>

<p>Now that we have all of the raw counts, we need to clean up the NAs and duplicated values, then normalize them.</p>

<p>First things first, let&#39;s remove NA values (which happens when there is no corresponding RNA-Seq count data for housekeeping genes) and duplicated tsRNA/miRNA rows. Duplicates arise from different loci transcribing the same gene. We want to keep only 1 locus. We will also eliminate features that have 0 counts for over 95% of the samples. This means that for a tRF or miRNA to be considered in the analysis, the feature must have at least 20 reads and be expressed in at least 5% of samples. These filters are stricter than usual because reads can be mapped to multiple loci, so a sample with 10 mapped locations may really just be the same read mapped 10 times.</p>

<p>DESeq2 estimates gene dispersions in a 3 step process. First, it considers individual genes, and looks at their dispersions across all samples. Second, it fits a curve to the individual gene dispersions. Third, it uses the curve as a prior to shrink the individual dispersions towards the curve. Thus, duplicated genes will affect the fitted curve, and subsequently the dispersion adjustments towards the prior.</p>

<pre><code class="r">feature_counts &lt;- feature_counts[apply(feature_counts, 1, function(row) length(which(row &gt; 0)) &gt; length(colnames(feature_counts)) * 0.05), ] # no. of samples with reads mapping to this feature &gt; no. samples * 0.05. at least 1/20 of the samples must have reads for this feature to be considered
feature_counts &lt;- feature_counts[apply(feature_counts, 1, function(row) sum(row, na.rm = TRUE) &gt; 20), ]
feature_counts &lt;- feature_counts[!apply(feature_counts, 1, function(row) any(is.na(row))), ]
</code></pre>

<h2>Differential expression and EDA</h2>

<h3>DESeq2</h3>

<pre><code class="r">colData &lt;- data.frame(row.names = colnames(raw_counts), patient = as.factor(prad_metadata$participant_id), sample_type = as.factor(prad_metadata$sample_type), type = &quot;single-read&quot;) # feature count column names correspond to metadata rows
deseq_prad_dds &lt;- DESeqDataSetFromMatrix(countData = raw_counts, colData = colData, design = ~ sample_type)
deseq_prad_dds &lt;- DESeq(deseq_prad_dds)
normalized_counts &lt;- counts(deseq_prad_dds, normalized = TRUE) 
deseq_prad &lt;- results(deseq_prad_dds)
</code></pre>

<h3>Diagnostic plots</h3>

<pre><code class="r">df &lt;- data.frame(fc = 2^(deseq_prad$log2FoldChange), pval = -log(deseq_prad$pvalue, 2))
ggplot(df) + geom_point(aes(x = fc, y = pval)) + xlab(&quot;Fold change&quot;) + ylab(&quot;P-values (-log2)&quot;) + xlim(-0.5, 2.5) + ggtitle(&quot;Volcano plot&quot;)
df &lt;- data.frame(m = deseq_prad$log2FoldChange, a = log(rowMeans(normalized_counts), 2), significant = ifelse(deseq_prad$pvalue &lt; 0.05, TRUE, FALSE))
ggplot(df) + geom_point(aes(x = a, y = m, color = significant)) + geom_abline(intercept = 0, slope = 0, color = 2) + scale_color_manual(values = c(1,2), guide = FALSE) + xlab(&quot;Average expression (log2)&quot;) + ylab(&quot;Fold change (log2)&quot;) + ylim(-3, 3) + ggtitle(&quot;MA plot&quot;)
ggplot(data.frame(padj = deseq_prad$padj)) + geom_histogram(aes(x = padj)) + xlab(&quot;Adjusted p-value&quot;) + ggtitle(&quot;Histogram of adjusted p-values&quot;)
ggplot(data.frame(padj = with(deseq_prad, padj[padj &lt; 0.05 &amp; log2FoldChange &gt;= 1]))) + geom_histogram(aes(x = padj)) + xlab(&quot;Adjusted p-value&quot;) + ggtitle(&quot;Histogram of adjusted p-values, filtered data&quot;)
plotDispEsts(deseq_prad_dds)
</code></pre>

<h3>Batch effects</h3>

<pre><code class="r">svobj &lt;- svd(normalized_counts - rowMeans(normalized_counts))
</code></pre>

<pre><code class="r">batch &lt;- as.factor(ifelse(!is.na(prad_clinical$batch), as.character(prad_clinical$batch), &quot;Batch ?&quot;))
df &lt;- data.frame(PC1 = svobj$v[, 1], PC2 = svobj$v[, 2], Batch = batch, sample_type = prad_metadata$sample_type, Gleason = as.factor(prad_clinical$gleason))
ggplot(df) + geom_point(aes(x = PC1, y = PC2, color = Batch, shape = Batch, size = sample_type), alpha = 0.5) + scale_shape_manual(values = 1:19) + scale_size_manual(labels = c(&quot;NT&quot;, &quot;TP&quot;), values = c(5, 2)) + guides(color = guide_legend(ncol = 2), shape = guide_legend(ncol = 2), size = guide_legend(ncol = 2)) + xlim(-0.17, 0.06) + ylim(-0.05, 0.17) + ggtitle(&quot;SVD and PCA&quot;)

ggplot(df) + geom_point(aes(x = PC1, y = PC2, color = Gleason, shape = sample_type), alpha = 0.5) + xlim(-0.17, 0.06) + ylim(-0.05, 0.17) + ggtitle(&quot;SVD and PCA&quot;)
</code></pre>

<pre><code class="r">df &lt;- data.frame(Score = t(cor(as.numeric(batch), svobj$v, method = &quot;spearman&quot;)), PC = 1:dim(svobj$v)[1])
ggplot(df) + geom_point(aes(x = PC, y = Score)) + ggtitle(&#39;Spearman correlation between batch and PCs&#39;)
</code></pre>

<h2>Test if gene counts are necessary for this analysis</h2>

<pre><code class="r">all_prad_dds &lt;- DESeqDataSetFromMatrix(countData = feature_counts, colData = data.frame(row.names = colnames(raw_counts), patient = as.factor(prad_metadata$participant_id), sample_type = as.factor(prad_metadata$sample_type), type = &quot;single-read&quot;), design = ~ sample_type)
all_prad_dds &lt;- DESeq(all_prad_dds)
plotDispEsts(all_prad_dds)
</code></pre>

</body>

</html>
