## correlate-mirnas
_Last modified `r format(Sys.time(), "%I:%M %p on %b %d, %Y")`. This document, R session image, knitr cache, figures, and other associated datasets are located in `cruncher:/inside/grotto/blin/trna-markers/correlate-mirnas/`._

```{r setup, echo=FALSE, warning=FALSE, results=FALSE, message=FALSE, errors=FALSE, cache=FALSE}
library(knitr)
opts_chunk$set(cache=TRUE, cache.path="/inside/grotto/blin/trna-markers/correlate-mirnas/cache/", eval=TRUE, echo=TRUE, warning=FALSE, results=FALSE, message=FALSE, autodep=TRUE, dev="png", dpi=300)
```

```{r libraries, cache=FALSE}
library(reshape2)
library(ggplot2)
library(stringr)
library(gplots)
library(GenomicRanges)
library(BiocParallel)
register(MulticoreParam(8)) # turn this off depending on your machine
attach('/inside/grotto/blin/trna-markers/batch-effects/counts.RData')
attach('/inside/grotto/blin/trna-markers/process-reads/prad-metadata.RData')
attach('/inside/grotto/blin/trna-markers/feature-counts/features.RData')
```

The correlation will be performed on a matrix of fold changes. I'm not sure whether I should take the log<sub>2</sub> fold change before then. I do in this case, to minimize heteroscedasticity - reduce the variability between sub-populations, which in this case is individual genes or groups of genes. Since these are `log2` values, and there are fragments with counts of 0, we will use Laplacian smoothing (add a pseudocount of 1) to avoid `-Inf` values.

```{r fold-change}
# NT normalized counts - TP normalized counts
getFoldChanges <- function(metadata, counts) {
  # first, get counts for matched samples
  nt_matched_samples <- subset(metadata, paired == TRUE & sample_type == "NT")$barcode
  tp_matched_samples <- subset(metadata, paired == TRUE & sample_type == "TP")$barcode
  fold_changes <- counts[, nt_matched_samples] - counts[, tp_matched_samples]
  # second, get counts for unmatched samples (no extra normal tissue samples, only extra primary tumor samples)
  nt_samples <- subset(metadata, sample_type == "NT")$barcode # get normal tumor sample barcodes
  nt_median <- apply(counts[, nt_samples], 1, median) # get median counts for each feature
  tp_unmatched_samples <- which(metadata$sample_type == "TP" & metadata$paired == FALSE)
  cbind(fold_changes, nt_median - counts[, tp_unmatched_samples]) # subtract tumor counts from median nt counts to get the fold change
}
# TODO: add column (sample) names to fold changes object
fold_changes <- getFoldChanges(prad_metadata, normalized_counts)
```

I also have to choose between Pearson's r and Spearman's $\rho$. Apparently, Pearson's r has more power and finds linear (as opposed to monotonic) relationships, so I'll start with that.

```{r pearson}
correlations <- cor(t(fold_changes), method = "pearson") # function operates on columns, so need to get features onto columns.
```

```{r mir-correlations}
mir_corr <- correlations[!str_detect(colnames(correlations), "hsa"), str_detect(colnames(correlations), "hsa")] # isolate for miRNAs
mir_corr <- melt(mir_corr)
colnames(mir_corr) <- c("Feature", "miRNA", "r")
mir_corr <- mir_corr[order(mir_corr$r, decreasing = TRUE), ]
mir_corr$Class <- mcols(features)$class[match(mir_corr$Feature, mcols(features)$tx_name)] # add classifications to correlation data frame
head(mir_corr, 10)
tail(mir_corr, 10)
```

For this analysis, these are the most strongest negative and positive correlations with mir-hsa-31.


### Selecting key miRNAs
#### Differentially expressed miRNAs identified by Schaefer et al. (2009)
Originally, I used the miRNAs derived from [this paper](http://dx.doi.org/10.1002/ijc.24827), which describes miRNAs dysregulated in prostate carcinoma. This list of miRNAs can be found at `/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-1.bed`. Four of these are upregulated, and the other five are downregulated.

The problem with this set of miRNAs is that only `hsa-mir-31` survives the purge.

#### Broad GDAC Firehose PRAD T stage miRNAs
Later analyses showed that most of these miRNAs weren't expressed at a high enough level to be correlated with anything at all. Yikes. I'm going to try using [Broad Firehose](http://gdac.broadinstitute.org/runs/analyses__latest/reports/cancer/PRAD/Correlate_Clinical_vs_miRseq/nozzle.html) instead. There are 16 miRNAs negatively correlated with T stage. I don't know what "negatively correlated with lower stage" means in this context - will need to ask someone from Josh's lab to see what's going on. 

Actually, there aren't 16 miRNAs. The very first one, [hsa-mir-3676](http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MI0016077), is actually a tRF! I'll look into this later. There are 32 sequences associated from the remaining 15 miRNAs - mostly due to 5' and 3' miRNA fragments from the precursor sequence, but also due to +/- strand entries. These sequences are located at `/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-2.gff`.

### Student's t-test for individual features

A two-tailed t-test with unequal variance is appropriate for determining if a feature is differentially expressed in normal vs. tumor, a two class feature. This is exactly what Broad Firehose uses. I'll do three t-tests:

1. a paired test with matched tumor-normal pairs - each pair is from the same individual
2. a paired test with matched tumor-normal pairs and unpaired samples paired with the median of the other class. In this case, the TP unpaired samples will be paired with the median of the NT sample expression.
3. an independent t-test with data simply separated into tumor and normal

The t-test assumes that both samples are normally distributed. In other words, for a particular feature, it assumes that the expression of that feature across all samples follows a normal distribution. I mean, yeah, it's not perfect - the negative binomial distribution is empirically a better fit - but it's close enough. In any case, if someone objects, I can throw in a Wilcoxon rank-sum test too.

```{r t-test-1}
nt_paired_counts <- normalized_counts[, subset(prad_metadata, paired == TRUE & sample_type == "NT")$barcode]
tp_paired_counts <- normalized_counts[, subset(prad_metadata, paired == TRUE & sample_type == "TP")$barcode]
test_1_results <- lapply(1:nrow(nt_paired_counts), function(index) t.test(nt_paired_counts[index, ], tp_paired_counts[index, ], paired = TRUE, var.equal = FALSE))
bonferroni_corrected_pvalue <- 0.05/nrow(nt_paired_counts)
test_1_features <- rownames(nt_paired_counts)[which(sapply(test_1_results, function(t_test) t_test$p.value < bonferroni_corrected_pvalue))]
```

```{r t-test-2}
generatePairedTTestMatrices <- function(metadata, counts) {
	tp_median <- apply(counts[, subset(metadata, sample_type == "TP")$barcode], 1, median) # get TP median counts for each feature
	nt_median <- apply(counts[, subset(metadata, sample_type == "NT")$barcode], 1, median) # get TP median counts for each feature
	nt_unmatched_paired <- counts[, subset(metadata, sample_type == "NT")$barcode]
	tp_unmatched_paired <- counts[, subset(metadata, sample_type == "TP")$barcode]
	num_nt_unmatched <- nrow(subset(metadata, paired == FALSE & sample_type == "NT"))
	num_tp_unmatched <- nrow(subset(metadata, paired == FALSE & sample_type == "TP"))
	if (num_tp_unmatched > 0) nt_unmatched_paired <- cbind(nt_unmatched_paired,
                                                           matrix(rep(nt_median, times = num_tp_unmatched), ncol = num_tp_unmatched))
	if (num_nt_unmatched > 0) tp_unmatched_paired <- cbind(tp_unmatched_paired,
                                                           matrix(rep(tp_median, times = num_nt_unmatched), ncol = num_nt_unmatched))
	return(list(nt_unmatched_paired, tp_unmatched_paired))
}
t_test_matrices <- generatePairedTTestMatrices(prad_metadata, normalized_counts)
test_2_results <- lapply(1:nrow(t_test_matrices[[1]]), function(index) t.test(t_test_matrices[[1]][index, ], t_test_matrices[[2]][index, ], paired = TRUE, var.equal = FALSE))
test_2_features <- rownames(normalized_counts)[which(sapply(test_2_results, function(t_test) t_test$p.value < bonferroni_corrected_pvalue))]
```

```{r t-test-3}
test_3_results <- lapply(1:nrow(normalized_counts), function(index) {
	t_test_input <- melt(normalized_counts[, subset(prad_metadata, sample_type == "NT")$barcode][index, ])
	t_test_input <- data.frame(t_test_input, cancer = FALSE)
	t_test_input <- rbind(t_test_input, data.frame(melt(normalized_counts[, subset(prad_metadata, sample_type == "TP")$barcode][index, ]), cancer = TRUE))
	t.test(t_test_input, paired = FALSE, var.equal = FALSE, formula = value ~ cancer)
})
test_3_features <- rownames(normalized_counts)[which(sapply(test_3_results, function(t_test) t_test$p.value < bonferroni_corrected_pvalue))]
```

```{r t-test-results}
intersect(intersect(test_1_features, test_2_features), test_3_features)
```


### Cluster

We can also cluster the features by their fold changes.

```{r clusters, run=FALSE}
distance_matrix <- as.matrix(dist(t(fold_changes))) # Euclidean
clusters <- hclust(distance_matrix)
heatmap.2(distance_matrix, Rowv = as.dendrogram(clusters), symm = TRUE, trace = "none")
```

```{r save-image, echo=FALSE, cache=FALSE}
save(file = 'fold-changes.RData', fold_changes)
save.image('correlate-mirnas-image.RData')
```


