<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>correlate-mirnas</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>

<!-- Styles for R syntax highlighter -->
<style type="text/css">
   pre .operator,
   pre .paren {
     color: rgb(104, 118, 135)
   }

   pre .literal {
     color: #990073
   }

   pre .number {
     color: #099;
   }

   pre .comment {
     color: #998;
     font-style: italic
   }

   pre .keyword {
     color: #900;
     font-weight: bold
   }

   pre .identifier {
     color: rgb(0, 0, 0);
   }

   pre .string {
     color: #d14;
   }
</style>

<!-- R syntax highlighter -->
<script type="text/javascript">
var hljs=new function(){function m(p){return p.replace(/&/gm,"&amp;").replace(/</gm,"&lt;")}function f(r,q,p){return RegExp(q,"m"+(r.cI?"i":"")+(p?"g":""))}function b(r){for(var p=0;p<r.childNodes.length;p++){var q=r.childNodes[p];if(q.nodeName=="CODE"){return q}if(!(q.nodeType==3&&q.nodeValue.match(/\s+/))){break}}}function h(t,s){var p="";for(var r=0;r<t.childNodes.length;r++){if(t.childNodes[r].nodeType==3){var q=t.childNodes[r].nodeValue;if(s){q=q.replace(/\n/g,"")}p+=q}else{if(t.childNodes[r].nodeName=="BR"){p+="\n"}else{p+=h(t.childNodes[r])}}}if(/MSIE [678]/.test(navigator.userAgent)){p=p.replace(/\r/g,"\n")}return p}function a(s){var r=s.className.split(/\s+/);r=r.concat(s.parentNode.className.split(/\s+/));for(var q=0;q<r.length;q++){var p=r[q].replace(/^language-/,"");if(e[p]){return p}}}function c(q){var p=[];(function(s,t){for(var r=0;r<s.childNodes.length;r++){if(s.childNodes[r].nodeType==3){t+=s.childNodes[r].nodeValue.length}else{if(s.childNodes[r].nodeName=="BR"){t+=1}else{if(s.childNodes[r].nodeType==1){p.push({event:"start",offset:t,node:s.childNodes[r]});t=arguments.callee(s.childNodes[r],t);p.push({event:"stop",offset:t,node:s.childNodes[r]})}}}}return t})(q,0);return p}function k(y,w,x){var q=0;var z="";var s=[];function u(){if(y.length&&w.length){if(y[0].offset!=w[0].offset){return(y[0].offset<w[0].offset)?y:w}else{return w[0].event=="start"?y:w}}else{return y.length?y:w}}function t(D){var A="<"+D.nodeName.toLowerCase();for(var B=0;B<D.attributes.length;B++){var C=D.attributes[B];A+=" "+C.nodeName.toLowerCase();if(C.value!==undefined&&C.value!==false&&C.value!==null){A+='="'+m(C.value)+'"'}}return A+">"}while(y.length||w.length){var v=u().splice(0,1)[0];z+=m(x.substr(q,v.offset-q));q=v.offset;if(v.event=="start"){z+=t(v.node);s.push(v.node)}else{if(v.event=="stop"){var p,r=s.length;do{r--;p=s[r];z+=("</"+p.nodeName.toLowerCase()+">")}while(p!=v.node);s.splice(r,1);while(r<s.length){z+=t(s[r]);r++}}}}return z+m(x.substr(q))}function j(){function q(x,y,v){if(x.compiled){return}var u;var s=[];if(x.k){x.lR=f(y,x.l||hljs.IR,true);for(var w in x.k){if(!x.k.hasOwnProperty(w)){continue}if(x.k[w] instanceof Object){u=x.k[w]}else{u=x.k;w="keyword"}for(var r in u){if(!u.hasOwnProperty(r)){continue}x.k[r]=[w,u[r]];s.push(r)}}}if(!v){if(x.bWK){x.b="\\b("+s.join("|")+")\\s"}x.bR=f(y,x.b?x.b:"\\B|\\b");if(!x.e&&!x.eW){x.e="\\B|\\b"}if(x.e){x.eR=f(y,x.e)}}if(x.i){x.iR=f(y,x.i)}if(x.r===undefined){x.r=1}if(!x.c){x.c=[]}x.compiled=true;for(var t=0;t<x.c.length;t++){if(x.c[t]=="self"){x.c[t]=x}q(x.c[t],y,false)}if(x.starts){q(x.starts,y,false)}}for(var p in e){if(!e.hasOwnProperty(p)){continue}q(e[p].dM,e[p],true)}}function d(B,C){if(!j.called){j();j.called=true}function q(r,M){for(var L=0;L<M.c.length;L++){if((M.c[L].bR.exec(r)||[null])[0]==r){return M.c[L]}}}function v(L,r){if(D[L].e&&D[L].eR.test(r)){return 1}if(D[L].eW){var M=v(L-1,r);return M?M+1:0}return 0}function w(r,L){return L.i&&L.iR.test(r)}function K(N,O){var M=[];for(var L=0;L<N.c.length;L++){M.push(N.c[L].b)}var r=D.length-1;do{if(D[r].e){M.push(D[r].e)}r--}while(D[r+1].eW);if(N.i){M.push(N.i)}return f(O,M.join("|"),true)}function p(M,L){var N=D[D.length-1];if(!N.t){N.t=K(N,E)}N.t.lastIndex=L;var r=N.t.exec(M);return r?[M.substr(L,r.index-L),r[0],false]:[M.substr(L),"",true]}function z(N,r){var L=E.cI?r[0].toLowerCase():r[0];var M=N.k[L];if(M&&M instanceof Array){return M}return false}function F(L,P){L=m(L);if(!P.k){return L}var r="";var O=0;P.lR.lastIndex=0;var M=P.lR.exec(L);while(M){r+=L.substr(O,M.index-O);var N=z(P,M);if(N){x+=N[1];r+='<span class="'+N[0]+'">'+M[0]+"</span>"}else{r+=M[0]}O=P.lR.lastIndex;M=P.lR.exec(L)}return r+L.substr(O,L.length-O)}function J(L,M){if(M.sL&&e[M.sL]){var r=d(M.sL,L);x+=r.keyword_count;return r.value}else{return F(L,M)}}function I(M,r){var L=M.cN?'<span class="'+M.cN+'">':"";if(M.rB){y+=L;M.buffer=""}else{if(M.eB){y+=m(r)+L;M.buffer=""}else{y+=L;M.buffer=r}}D.push(M);A+=M.r}function G(N,M,Q){var R=D[D.length-1];if(Q){y+=J(R.buffer+N,R);return false}var P=q(M,R);if(P){y+=J(R.buffer+N,R);I(P,M);return P.rB}var L=v(D.length-1,M);if(L){var O=R.cN?"</span>":"";if(R.rE){y+=J(R.buffer+N,R)+O}else{if(R.eE){y+=J(R.buffer+N,R)+O+m(M)}else{y+=J(R.buffer+N+M,R)+O}}while(L>1){O=D[D.length-2].cN?"</span>":"";y+=O;L--;D.length--}var r=D[D.length-1];D.length--;D[D.length-1].buffer="";if(r.starts){I(r.starts,"")}return R.rE}if(w(M,R)){throw"Illegal"}}var E=e[B];var D=[E.dM];var A=0;var x=0;var y="";try{var s,u=0;E.dM.buffer="";do{s=p(C,u);var t=G(s[0],s[1],s[2]);u+=s[0].length;if(!t){u+=s[1].length}}while(!s[2]);if(D.length>1){throw"Illegal"}return{r:A,keyword_count:x,value:y}}catch(H){if(H=="Illegal"){return{r:0,keyword_count:0,value:m(C)}}else{throw H}}}function g(t){var p={keyword_count:0,r:0,value:m(t)};var r=p;for(var q in e){if(!e.hasOwnProperty(q)){continue}var s=d(q,t);s.language=q;if(s.keyword_count+s.r>r.keyword_count+r.r){r=s}if(s.keyword_count+s.r>p.keyword_count+p.r){r=p;p=s}}if(r.language){p.second_best=r}return p}function i(r,q,p){if(q){r=r.replace(/^((<[^>]+>|\t)+)/gm,function(t,w,v,u){return w.replace(/\t/g,q)})}if(p){r=r.replace(/\n/g,"<br>")}return r}function n(t,w,r){var x=h(t,r);var v=a(t);var y,s;if(v){y=d(v,x)}else{return}var q=c(t);if(q.length){s=document.createElement("pre");s.innerHTML=y.value;y.value=k(q,c(s),x)}y.value=i(y.value,w,r);var u=t.className;if(!u.match("(\\s|^)(language-)?"+v+"(\\s|$)")){u=u?(u+" "+v):v}if(/MSIE [678]/.test(navigator.userAgent)&&t.tagName=="CODE"&&t.parentNode.tagName=="PRE"){s=t.parentNode;var p=document.createElement("div");p.innerHTML="<pre><code>"+y.value+"</code></pre>";t=p.firstChild.firstChild;p.firstChild.cN=s.cN;s.parentNode.replaceChild(p.firstChild,s)}else{t.innerHTML=y.value}t.className=u;t.result={language:v,kw:y.keyword_count,re:y.r};if(y.second_best){t.second_best={language:y.second_best.language,kw:y.second_best.keyword_count,re:y.second_best.r}}}function o(){if(o.called){return}o.called=true;var r=document.getElementsByTagName("pre");for(var p=0;p<r.length;p++){var q=b(r[p]);if(q){n(q,hljs.tabReplace)}}}function l(){if(window.addEventListener){window.addEventListener("DOMContentLoaded",o,false);window.addEventListener("load",o,false)}else{if(window.attachEvent){window.attachEvent("onload",o)}else{window.onload=o}}}var e={};this.LANGUAGES=e;this.highlight=d;this.highlightAuto=g;this.fixMarkup=i;this.highlightBlock=n;this.initHighlighting=o;this.initHighlightingOnLoad=l;this.IR="[a-zA-Z][a-zA-Z0-9_]*";this.UIR="[a-zA-Z_][a-zA-Z0-9_]*";this.NR="\\b\\d+(\\.\\d+)?";this.CNR="\\b(0[xX][a-fA-F0-9]+|(\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)";this.BNR="\\b(0b[01]+)";this.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|\\.|-|-=|/|/=|:|;|<|<<|<<=|<=|=|==|===|>|>=|>>|>>=|>>>|>>>=|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~";this.ER="(?![\\s\\S])";this.BE={b:"\\\\.",r:0};this.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[this.BE],r:0};this.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[this.BE],r:0};this.CLCM={cN:"comment",b:"//",e:"$"};this.CBLCLM={cN:"comment",b:"/\\*",e:"\\*/"};this.HCM={cN:"comment",b:"#",e:"$"};this.NM={cN:"number",b:this.NR,r:0};this.CNM={cN:"number",b:this.CNR,r:0};this.BNM={cN:"number",b:this.BNR,r:0};this.inherit=function(r,s){var p={};for(var q in r){p[q]=r[q]}if(s){for(var q in s){p[q]=s[q]}}return p}}();hljs.LANGUAGES.cpp=function(){var a={keyword:{"false":1,"int":1,"float":1,"while":1,"private":1,"char":1,"catch":1,"export":1,virtual:1,operator:2,sizeof:2,dynamic_cast:2,typedef:2,const_cast:2,"const":1,struct:1,"for":1,static_cast:2,union:1,namespace:1,unsigned:1,"long":1,"throw":1,"volatile":2,"static":1,"protected":1,bool:1,template:1,mutable:1,"if":1,"public":1,friend:2,"do":1,"return":1,"goto":1,auto:1,"void":2,"enum":1,"else":1,"break":1,"new":1,extern:1,using:1,"true":1,"class":1,asm:1,"case":1,typeid:1,"short":1,reinterpret_cast:2,"default":1,"double":1,register:1,explicit:1,signed:1,typename:1,"try":1,"this":1,"switch":1,"continue":1,wchar_t:1,inline:1,"delete":1,alignof:1,char16_t:1,char32_t:1,constexpr:1,decltype:1,noexcept:1,nullptr:1,static_assert:1,thread_local:1,restrict:1,_Bool:1,complex:1},built_in:{std:1,string:1,cin:1,cout:1,cerr:1,clog:1,stringstream:1,istringstream:1,ostringstream:1,auto_ptr:1,deque:1,list:1,queue:1,stack:1,vector:1,map:1,set:1,bitset:1,multiset:1,multimap:1,unordered_set:1,unordered_map:1,unordered_multiset:1,unordered_multimap:1,array:1,shared_ptr:1}};return{dM:{k:a,i:"</",c:[hljs.CLCM,hljs.CBLCLM,hljs.QSM,{cN:"string",b:"'\\\\?.",e:"'",i:"."},{cN:"number",b:"\\b(\\d+(\\.\\d*)?|\\.\\d+)(u|U|l|L|ul|UL|f|F)"},hljs.CNM,{cN:"preprocessor",b:"#",e:"$"},{cN:"stl_container",b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:a,r:10,c:["self"]}]}}}();hljs.LANGUAGES.r={dM:{c:[hljs.HCM,{cN:"number",b:"\\b0[xX][0-9a-fA-F]+[Li]?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+(?:[eE][+\\-]?\\d*)?L\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+\\.(?!\\d)(?:i\\b)?",e:hljs.IMMEDIATE_RE,r:1},{cN:"number",b:"\\b\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"keyword",b:"(?:tryCatch|library|setGeneric|setGroupGeneric)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\.",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\d+(?![\\w.])",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\b(?:function)",e:hljs.IMMEDIATE_RE,r:2},{cN:"keyword",b:"(?:if|in|break|next|repeat|else|for|return|switch|while|try|stop|warning|require|attach|detach|source|setMethod|setClass)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"literal",b:"(?:NA|NA_integer_|NA_real_|NA_character_|NA_complex_)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"literal",b:"(?:NULL|TRUE|FALSE|T|F|Inf|NaN)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"identifier",b:"[a-zA-Z.][a-zA-Z0-9._]*\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"<\\-(?!\\s*\\d)",e:hljs.IMMEDIATE_RE,r:2},{cN:"operator",b:"\\->|<\\-",e:hljs.IMMEDIATE_RE,r:1},{cN:"operator",b:"%%|~",e:hljs.IMMEDIATE_RE},{cN:"operator",b:">=|<=|==|!=|\\|\\||&&|=|\\+|\\-|\\*|/|\\^|>|<|!|&|\\||\\$|:",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"%",e:"%",i:"\\n",r:1},{cN:"identifier",b:"`",e:"`",r:0},{cN:"string",b:'"',e:'"',c:[hljs.BE],r:0},{cN:"string",b:"'",e:"'",c:[hljs.BE],r:0},{cN:"paren",b:"[[({\\])}]",e:hljs.IMMEDIATE_RE,r:0}]}};
hljs.initHighlightingOnLoad();
</script>

<!-- MathJax scripts -->
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>


<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<h2>correlate-mirnas</h2>

<p><em>Last modified 03:37 PM on Jan 14, 2015. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/correlate-mirnas/</code>.</em></p>

<pre><code class="r">library(reshape2)
library(ggplot2)
library(stringr)
library(gplots)
library(GenomicRanges)
library(BiocParallel)
register(MulticoreParam(8)) # turn this off depending on your machine
attach(&#39;/inside/grotto/blin/trna-markers/batch-effects/counts.RData&#39;)
attach(&#39;/inside/grotto/blin/trna-markers/process-reads/prad-metadata.RData&#39;)
attach(&#39;/inside/grotto/blin/trna-markers/feature-counts/features.RData&#39;)
</code></pre>

<p>The correlation will be performed on a matrix of fold changes. I&#39;m not sure whether I should take the log<sub>2</sub> fold change before then. I do in this case, to minimize heteroscedasticity - reduce the variability between sub-populations, which in this case is individual genes or groups of genes. Since these are <code>log2</code> values, and there are fragments with counts of 0, we will use Laplacian smoothing (add a pseudocount of 1) to avoid <code>-Inf</code> values.</p>

<pre><code class="r"># NT normalized counts - TP normalized counts
getFoldChanges &lt;- function(metadata, counts) {
  # first, get counts for matched samples
  nt_matched_samples &lt;- subset(metadata, paired == TRUE &amp; sample_type == &quot;NT&quot;)$barcode
  tp_matched_samples &lt;- subset(metadata, paired == TRUE &amp; sample_type == &quot;TP&quot;)$barcode
  fold_changes &lt;- counts[, nt_matched_samples] - counts[, tp_matched_samples]
  # second, get counts for unmatched samples (no extra normal tissue samples, only extra primary tumor samples)
  nt_samples &lt;- subset(metadata, sample_type == &quot;NT&quot;)$barcode # get normal tumor sample barcodes
  nt_median &lt;- apply(counts[, nt_samples], 1, median) # get median counts for each feature
  tp_unmatched_samples &lt;- which(metadata$sample_type == &quot;TP&quot; &amp; metadata$paired == FALSE)
  cbind(fold_changes, nt_median - counts[, tp_unmatched_samples]) # subtract tumor counts from median nt counts to get the fold change
}
# TODO: add column (sample) names to fold changes object
fold_changes &lt;- getFoldChanges(prad_metadata, normalized_counts)
</code></pre>

<p>I also have to choose between Pearson&#39;s r and Spearman&#39;s \(\rho\). Apparently, Pearson&#39;s r has more power and finds linear (as opposed to monotonic) relationships, so I&#39;ll start with that.</p>

<pre><code class="r">correlations &lt;- cor(t(fold_changes), method = &quot;pearson&quot;) # function operates on columns, so need to get features onto columns.
</code></pre>

<pre><code class="r">mir_corr &lt;- correlations[!str_detect(colnames(correlations), &quot;hsa&quot;), str_detect(colnames(correlations), &quot;hsa&quot;)] # isolate for miRNAs
mir_corr &lt;- melt(mir_corr)
colnames(mir_corr) &lt;- c(&quot;Feature&quot;, &quot;miRNA&quot;, &quot;r&quot;)
mir_corr &lt;- mir_corr[order(mir_corr$r, decreasing = TRUE), ]
mir_corr$Class &lt;- mcols(features)$class[match(mir_corr$Feature, mcols(features)$tx_name)] # add classifications to correlation data frame
head(mir_corr, 10)
</code></pre>

<pre><code>##                                 Feature         miRNA         r   Class
## 62791         tRNA-Arg-TCT-4-1_fivehalf  hsa-mir-6718 0.7472128   5half
## 17000        tRNA-Gln-TTG-3-1_threehalf  hsa-mir-4294 0.7129454   3half
## 18338  chr6.numttRNA14-GlnTTG_threehalf  hsa-mir-4294 0.7129147   3half
## 17005        tRNA-Gln-TTG-3-3_threehalf  hsa-mir-4294 0.7047468   3half
## 18048  chr6.numttRNA13-GlnTTG_threehalf  hsa-mir-4294 0.7047404   3half
## 153010        tRNA-His-GTG-1-8_fivehalf hsa-mir-101-2 0.6984400   5half
## 48327   chr5.numttRNA6-AspGTC_threehalf  hsa-mir-4505 0.6959277   3half
## 48222     chr8.numttRNA2-ThrTGT_trailer  hsa-mir-4505 0.6768582 trailer
## 17002        tRNA-Gln-TTG-3-2_threehalf  hsa-mir-4294 0.6767908   3half
## 64275    chr7.numttRNA14-CysGCA_trailer  hsa-mir-6718 0.6568405 trailer
</code></pre>

<pre><code class="r">tail(mir_corr, 10)
</code></pre>

<pre><code>##                              Feature        miRNA          r   Class
## 60352     tRNA-Ala-AGC-6-1_threehalf hsa-mir-5010 -0.3469308   3half
## 60610     tRNA-Asp-GTC-7-1_threehalf hsa-mir-5010 -0.3490210   3half
## 47747      tRNA-iMet-CAT-1-2_trailer hsa-mir-4505 -0.3496288 trailer
## 60781     tRNA-Glu-CTC-9-1_threehalf hsa-mir-5010 -0.3496598   3half
## 46536       tRNA-Ala-AGC-4-1_trailer hsa-mir-4505 -0.3528099 trailer
## 47173       tRNA-Ile-TAT-2-1_trailer hsa-mir-4505 -0.3534715 trailer
## 61772 chr17.numttRNA1-ThrTGT_trailer hsa-mir-5010 -0.3549946 trailer
## 63163     tRNA-Gly-GCC-2-2_threehalf hsa-mir-6718 -0.3688662   3half
## 61225      tRNA-Pro-AGG-2-1_fivehalf hsa-mir-5010 -0.4172380   5half
## 61068       tRNA-Leu-TAA-4-1_trailer hsa-mir-5010 -0.4229768 trailer
</code></pre>

<p>For this analysis, these are the most strongest negative and positive correlations with mir-hsa-31.</p>

<h3>Selecting key miRNAs</h3>

<h4>Differentially expressed miRNAs identified by Schaefer et al. (2009)</h4>

<p>Originally, I used the miRNAs derived from <a href="http://dx.doi.org/10.1002/ijc.24827">this paper</a>, which describes miRNAs dysregulated in prostate carcinoma. This list of miRNAs can be found at <code>/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-1.bed</code>. Four of these are upregulated, and the other five are downregulated.</p>

<p>The problem with this set of miRNAs is that only <code>hsa-mir-31</code> survives the purge.</p>

<h4>Broad GDAC Firehose PRAD T stage miRNAs</h4>

<p>Later analyses showed that most of these miRNAs weren&#39;t expressed at a high enough level to be correlated with anything at all. Yikes. I&#39;m going to try using <a href="http://gdac.broadinstitute.org/runs/analyses__latest/reports/cancer/PRAD/Correlate_Clinical_vs_miRseq/nozzle.html">Broad Firehose</a> instead. There are 16 miRNAs negatively correlated with T stage. I don&#39;t know what &ldquo;negatively correlated with lower stage&rdquo; means in this context - will need to ask someone from Josh&#39;s lab to see what&#39;s going on. </p>

<p>Actually, there aren&#39;t 16 miRNAs. The very first one, <a href="http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MI0016077">hsa-mir-3676</a>, is actually a tRF! I&#39;ll look into this later. There are 32 sequences associated from the remaining 15 miRNAs - mostly due to 5&#39; and 3&#39; miRNA fragments from the precursor sequence, but also due to +/- strand entries. These sequences are located at <code>/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-2.gff</code>.</p>

<h3>Student&#39;s t-test for individual features</h3>

<p>A two-tailed t-test with unequal variance is appropriate for determining if a feature is differentially expressed in normal vs. tumor, a two class feature. This is exactly what Broad Firehose uses. I&#39;ll do three t-tests:
1) a paired test with matched tumor-normal pairs - each pair is from the same individual
2) a paired test with matched tumor-normal pairs and unpaired samples paired with the median of the other class. In this case, the TP unpaired samples will be paired with the median of the NT sample expression.
3) an independent t-test with data simply separated into tumor and normal</p>

<pre><code class="r">nt_paired_counts &lt;- normalized_counts[, subset(prad_metadata, paired == TRUE &amp; sample_type == &quot;NT&quot;)$barcode]
tp_paired_counts &lt;- normalized_counts[, subset(prad_metadata, paired == TRUE &amp; sample_type == &quot;TP&quot;)$barcode]
test_1_results &lt;- lapply(1:nrow(nt_paired_counts), function(index) t.test(nt_paired_counts[index, ], tp_paired_counts[index, ], paired = TRUE, var.equal = FALSE))
bonferroni_corrected_pvalue &lt;- 0.05/nrow(nt_paired_counts)
test_1_features &lt;- rownames(nt_paired_counts)[which(sapply(test_1_results, function(t_test) t_test$p.value &lt; bonferroni_corrected_pvalue))]
</code></pre>

<pre><code class="r">generatePairedTTestMatrices &lt;- function(metadata, counts) {
    tp_median &lt;- apply(counts[, subset(metadata, sample_type == &quot;TP&quot;)$barcode], 1, median) # get TP median counts for each feature
    nt_median &lt;- apply(counts[, subset(metadata, sample_type == &quot;NT&quot;)$barcode], 1, median) # get TP median counts for each feature
    nt_unmatched_paired &lt;- counts[, subset(metadata, sample_type == &quot;NT&quot;)$barcode]
    tp_unmatched_paired &lt;- counts[, subset(metadata, sample_type == &quot;TP&quot;)$barcode]
    num_nt_unmatched &lt;- nrow(subset(metadata, paired == FALSE &amp; sample_type == &quot;NT&quot;))
    num_tp_unmatched &lt;- nrow(subset(metadata, paired == FALSE &amp; sample_type == &quot;TP&quot;))
    if (num_tp_unmatched &gt; 0) nt_unmatched_paired &lt;- cbind(nt_unmatched_paired,
                                                           matrix(rep(nt_median, times = num_tp_unmatched), ncol = num_tp_unmatched))
    if (num_nt_unmatched &gt; 0) tp_unmatched_paired &lt;- cbind(tp_unmatched_paired,
                                                           matrix(rep(tp_median, times = num_nt_unmatched), ncol = num_nt_unmatched))
    return(list(nt_unmatched_paired, tp_unmatched_paired))
}
t_test_matrices &lt;- generatePairedTTestMatrices(prad_metadata, normalized_counts)
test_2_results &lt;- lapply(1:nrow(t_test_matrices[[1]]), function(index) t.test(t_test_matrices[[1]][index, ], t_test_matrices[[2]][index, ], paired = TRUE, var.equal = FALSE))
test_2_features &lt;- rownames(normalized_counts)[which(sapply(test_2_results, function(t_test) t_test$p.value &lt; bonferroni_corrected_pvalue))]
</code></pre>

<pre><code class="r">test_3_results &lt;- lapply(1:nrow(normalized_counts), function(index) {
    t_test_input &lt;- melt(normalized_counts[, subset(prad_metadata, sample_type == &quot;NT&quot;)$barcode][index, ])
    t_test_input &lt;- data.frame(t_test_input, condition = &quot;NT&quot;)
    t_test_input &lt;- rbind(t_test_input, data.frame(melt(normalized_counts[, subset(prad_metadata, sample_type == &quot;TP&quot;)$barcode][index, ]), condition = &quot;TP&quot;))
    t.test(t_test_input, paired = FALSE, var.equal = FALSE, formula = value ~ condition)
})
</code></pre>

<pre><code>## Error in if (stderr &lt; 10 * .Machine$double.eps * abs(mx)) stop(&quot;data are essentially constant&quot;): missing value where TRUE/FALSE needed
</code></pre>

<pre><code class="r">test_3_features &lt;- rownames(normalized_counts)[which(sapply(test_3_results, function(t_test) t_test$p.value &lt; bonferroni_corrected_pvalue))]
</code></pre>

<pre><code class="r">intersect(intersect(test_1_features, test_2_features), test_3_features)
</code></pre>

<pre><code>##  [1] &quot;chr1.trna54-LysTTT-3006a&quot;         &quot;chr1.trna54-LysTTT-3006b&quot;        
##  [3] &quot;chr6.trna74-LeuCAA-3012a&quot;         &quot;chr6.trna74-LeuCAA-3012b&quot;        
##  [5] &quot;chr1.trna19-GlnCTG-5021a&quot;         &quot;chr1.trna15-GlnCTG-5021a&quot;        
##  [7] &quot;chr1.trna28-GlnCTG-5021a&quot;         &quot;chr1.trna19-GlnCTG-5021b&quot;        
##  [9] &quot;chr1.trna15-GlnCTG-5021b&quot;         &quot;chr1.trna28-GlnCTG-5021b&quot;        
## [11] &quot;tRNA-Asn-GTT-2-4_threehalf&quot;       &quot;tRNA-Asn-GTT-18-1_fivehalf&quot;      
## [13] &quot;tRNA-Asp-GTC-7-1_fivehalf&quot;        &quot;tRNA-Cys-ACA-1-1_fivehalf&quot;       
## [15] &quot;tRNA-Gln-CTG-3-1_fivehalf&quot;        &quot;tRNA-Gln-CTG-3-1_threehalf&quot;      
## [17] &quot;tRNA-Gln-CTG-3-2_fivehalf&quot;        &quot;tRNA-Gln-CTG-4-1_fivehalf&quot;       
## [19] &quot;tRNA-Gln-CTG-4-1_threehalf&quot;       &quot;tRNA-Gln-CTG-4-2_fivehalf&quot;       
## [21] &quot;tRNA-Gln-CTG-4-2_threehalf&quot;       &quot;tRNA-Gln-CTG-6-1_fivehalf&quot;       
## [23] &quot;tRNA-Gln-CTG-6-1_threehalf&quot;       &quot;tRNA-Gly-CCC-7-1_threehalf&quot;      
## [25] &quot;tRNA-Gly-GCC-1-5_threehalf&quot;       &quot;tRNA-Gly-GCC-2-5_threehalf&quot;      
## [27] &quot;tRNA-Gly-GCC-3-1_threehalf&quot;       &quot;tRNA-Gly-TCC-1-1_fivehalf&quot;       
## [29] &quot;tRNA-Leu-CAG-1-1_threehalf&quot;       &quot;tRNA-Leu-CAG-1-2_threehalf&quot;      
## [31] &quot;tRNA-Leu-CAG-1-3_threehalf&quot;       &quot;tRNA-Leu-CAG-1-4_threehalf&quot;      
## [33] &quot;tRNA-Leu-CAG-1-5_threehalf&quot;       &quot;tRNA-Leu-CAG-1-6_threehalf&quot;      
## [35] &quot;tRNA-Leu-CAG-1-7_threehalf&quot;       &quot;tRNA-Leu-CAG-2-1_threehalf&quot;      
## [37] &quot;tRNA-Lys-TTT-3-1_threehalf&quot;       &quot;tRNA-Met-CAT-6-1_fivehalf&quot;       
## [39] &quot;tRNA-Phe-GAA-1-3_trailer&quot;         &quot;tRNA-Ser-AGA-1-1_fivehalf&quot;       
## [41] &quot;tRNA-Ser-AGA-2-1_fivehalf&quot;        &quot;tRNA-Ser-AGA-2-6_fivehalf&quot;       
## [43] &quot;tRNA-Ser-AGA-4-1_fivehalf&quot;        &quot;tRNA-Ser-TGA-3-1_fivehalf&quot;       
## [45] &quot;tRNA-Ser-TGA-4-1_fivehalf&quot;        &quot;tRNA-Thr-CGT-3-1_fivehalf&quot;       
## [47] &quot;chr5.numttRNA2-CysGCA_fivehalf&quot;   &quot;chr9.numttRNA3-SerTGA_trailer&quot;   
## [49] &quot;chr1.numttRNA7-SerTGA_threehalf&quot;  &quot;chr6.numttRNA15-GlnCTG_fivehalf&quot; 
## [51] &quot;chr6.numttRNA15-GlnCTG_threehalf&quot; &quot;chr18.numttRNA4-GlnCTG_fivehalf&quot; 
## [53] &quot;chr18.numttRNA4-GlnCTG_threehalf&quot; &quot;hsa-mir-873&quot;
</code></pre>

<h3>Cluster</h3>

<p>We can also cluster the features by their fold changes.</p>

<pre><code class="r">distance_matrix &lt;- as.matrix(dist(t(fold_changes))) # Euclidean
clusters &lt;- hclust(distance_matrix)
</code></pre>

<pre><code>## Error in if (is.na(n) || n &gt; 65536L) stop(&quot;size cannot be NA nor exceed 65536&quot;): missing value where TRUE/FALSE needed
</code></pre>

<pre><code class="r">heatmap.2(distance_matrix, Rowv = as.dendrogram(clusters), symm = TRUE, trace = &quot;none&quot;)
</code></pre>

<pre><code>## Error in as.dendrogram(clusters): object &#39;clusters&#39; not found
</code></pre>

</body>

</html>
