## correlate-mirnas
_Last modified `r format(Sys.time(), "%I:%M %p on %b %d, %Y")`. This document, R session image, knitr cache, figures, and other associated datasets are located in `cruncher:/inside/grotto/blin/trna-markers/correlate-mirnas/`._

```{r setup, echo=FALSE, warning=FALSE, results=FALSE, message=FALSE, errors=FALSE, cache=FALSE}
library(knitr)
opts_chunk$set(cache=TRUE, cache.path="/inside/grotto/blin/trna-markers/correlate-mirnas/cache/", eval=TRUE, echo=TRUE, warning=FALSE, results=FALSE, message=FALSE, autodep=TRUE, dev="png", dpi=300)
```

```{r libraries, cache=FALSE}
library(reshape2)
library(ggplot2)
library(stringr)
library(gplots)
library(BiocParallel)
register(MulticoreParam(8)) # turn this off depending on your machine!
load('/inside/grotto/blin/trna-markers/batch-effects/counts.RData')
load('/inside/grotto/blin/trna-markers/process-reads/prad-metadata.RData')
load('/inside/grotto/blin/trna-markers/feature-counts/features.RData')
```

The correlation will be performed on a matrix of fold changes. I'm not sure whether I should take the log<sub>2</sub> fold change before then. I do in this case, to minimize heteroscedasticity - reduce the variability between sub-populations, which in this case is individual genes or groups of genes. Since these are `log2` values, and there are fragments with counts of 0, we will use Laplacian smoothing (add a pseudocount of 1) to avoid `-Inf` values.

```{r fold-change}
# NT normalized counts - TP normalized counts
getFoldChanges <- function(metadata, counts) {
  # first, get counts for matched samples
  nt_matched_samples <- subset(metadata, paired == TRUE & sample_type == "NT")$barcode
  tp_matched_samples <- subset(metadata, paired == TRUE & sample_type == "TP")$barcode
  fold_changes <- counts[, nt_matched_samples] - counts[, tp_matched_samples]
  # second, get counts for unmatched samples (no extra normal tissue samples, only extra primary tumor samples)
  nt_samples <- subset(metadata, sample_type == "NT")$barcode # get normal tumor sample barcodes
  nt_median <- apply(counts[, nt_samples], 1, median) # get median counts for each feature
  tp_unmatched_samples <- which(metadata$sample_type == "TP" & metadata$paired == FALSE)
  cbind(fold_changes, nt_median - counts[, tp_unmatched_samples]) # subtract tumor counts from median nt counts to get the fold change
}
# TODO: add column (sample) names to fold changes object
fold_changes <- getFoldChanges(prad_metadata, counts)
```

I also have to choose between Pearson's r and Spearman's $\rho$. Apparently, Pearson's r has more power and finds linear (as opposed to monotonic) relationships, so I'll start with that.

```{r pearson}
correlations <- cor(t(fold_changes), method = "pearson") # function operates on columns, so need to get features onto columns.
```

Let's 

```{r mir-correlations}
mir_corr <- correlations[!str_detect(colnames(correlations), "hsa"), str_detect(colnames(correlations), "hsa")] # isolate for miRNAs
mir_corr <- melt(mir_corr)
colnames(mir_corr) <- c("Feature", "miRNA", "r")
mir_corr <- mir_corr[order(mir_corr$r, decreasing = TRUE), ]
mir_corr$Class <- mcols(features)$class[match(mir_corr_df$Feature, mcols(features)$tx_name)] # add classifications to correlation df
head(mir_corr, 10)
tail(mir_corr, 10)
```

For this analysis, these are the most strongest negative and positive correlations with mir-hsa-31.


### Selecting key miRNAs
#### Differentially expressed miRNAs identified by Schaefer et al. (2009)
Originally, I used the miRNAs derived from [this paper](http://dx.doi.org/10.1002/ijc.24827), which describes miRNAs dysregulated in prostate carcinoma. This list of miRNAs can be found at `/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-1.bed`. Four of these are upregulated, and the other five are downregulated.

The problem with this set of miRNAs is that only `hsa-mir-31` survives the purge.

#### Broad GDAC Firehose PRAD T stage miRNAs
Later analyses showed that most of these miRNAs weren't expressed at a high enough level to be correlated with anything at all. Yikes. I'm going to try using [Broad Firehose](http://gdac.broadinstitute.org/runs/analyses__latest/reports/cancer/PRAD/Correlate_Clinical_vs_miRseq/nozzle.html) instead. There are 16 miRNAs negatively correlated with T stage. I don't know what "negatively correlated with lower stage" means in this context - will need to ask someone from Josh's lab to see what's going on. 

Actually, there aren't 16 miRNAs. The very first one, [hsa-mir-3676](http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MI0016077), is actually a tRF! I'll look into this later. There are 32 sequences associated from the remaining 15 miRNAs - mostly due to 5' and 3' miRNA fragments from the precursor sequence, but also due to +/- strand entries. These sequences are located at `/inside/grotto/blin/trna-markers/feature-counts/prad-mirnas-2.gff`.


### Cluster

We can also cluster the features by their fold changes.

```{r clusters}
distance_matrix <- as.matrix(dist(t(fold_changes))) # Euclidean
clusters <- hclust(distance_matrix)
heatmap.2(distance_matrix, Rowv = as.dendrogram(clusters), symm = TRUE, trace = "none")
```

```{r save-image, echo=FALSE, cache=FALSE}
save.image('correlate-mirnas-image.RData')
```


