<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>more-cancers</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>

<!-- Styles for R syntax highlighter -->
<style type="text/css">
   pre .operator,
   pre .paren {
     color: rgb(104, 118, 135)
   }

   pre .literal {
     color: #990073
   }

   pre .number {
     color: #099;
   }

   pre .comment {
     color: #998;
     font-style: italic
   }

   pre .keyword {
     color: #900;
     font-weight: bold
   }

   pre .identifier {
     color: rgb(0, 0, 0);
   }

   pre .string {
     color: #d14;
   }
</style>

<!-- R syntax highlighter -->
<script type="text/javascript">
var hljs=new function(){function m(p){return p.replace(/&/gm,"&amp;").replace(/</gm,"&lt;")}function f(r,q,p){return RegExp(q,"m"+(r.cI?"i":"")+(p?"g":""))}function b(r){for(var p=0;p<r.childNodes.length;p++){var q=r.childNodes[p];if(q.nodeName=="CODE"){return q}if(!(q.nodeType==3&&q.nodeValue.match(/\s+/))){break}}}function h(t,s){var p="";for(var r=0;r<t.childNodes.length;r++){if(t.childNodes[r].nodeType==3){var q=t.childNodes[r].nodeValue;if(s){q=q.replace(/\n/g,"")}p+=q}else{if(t.childNodes[r].nodeName=="BR"){p+="\n"}else{p+=h(t.childNodes[r])}}}if(/MSIE [678]/.test(navigator.userAgent)){p=p.replace(/\r/g,"\n")}return p}function a(s){var r=s.className.split(/\s+/);r=r.concat(s.parentNode.className.split(/\s+/));for(var q=0;q<r.length;q++){var p=r[q].replace(/^language-/,"");if(e[p]){return p}}}function c(q){var p=[];(function(s,t){for(var r=0;r<s.childNodes.length;r++){if(s.childNodes[r].nodeType==3){t+=s.childNodes[r].nodeValue.length}else{if(s.childNodes[r].nodeName=="BR"){t+=1}else{if(s.childNodes[r].nodeType==1){p.push({event:"start",offset:t,node:s.childNodes[r]});t=arguments.callee(s.childNodes[r],t);p.push({event:"stop",offset:t,node:s.childNodes[r]})}}}}return t})(q,0);return p}function k(y,w,x){var q=0;var z="";var s=[];function u(){if(y.length&&w.length){if(y[0].offset!=w[0].offset){return(y[0].offset<w[0].offset)?y:w}else{return w[0].event=="start"?y:w}}else{return y.length?y:w}}function t(D){var A="<"+D.nodeName.toLowerCase();for(var B=0;B<D.attributes.length;B++){var C=D.attributes[B];A+=" "+C.nodeName.toLowerCase();if(C.value!==undefined&&C.value!==false&&C.value!==null){A+='="'+m(C.value)+'"'}}return A+">"}while(y.length||w.length){var v=u().splice(0,1)[0];z+=m(x.substr(q,v.offset-q));q=v.offset;if(v.event=="start"){z+=t(v.node);s.push(v.node)}else{if(v.event=="stop"){var p,r=s.length;do{r--;p=s[r];z+=("</"+p.nodeName.toLowerCase()+">")}while(p!=v.node);s.splice(r,1);while(r<s.length){z+=t(s[r]);r++}}}}return z+m(x.substr(q))}function j(){function q(x,y,v){if(x.compiled){return}var u;var s=[];if(x.k){x.lR=f(y,x.l||hljs.IR,true);for(var w in x.k){if(!x.k.hasOwnProperty(w)){continue}if(x.k[w] instanceof Object){u=x.k[w]}else{u=x.k;w="keyword"}for(var r in u){if(!u.hasOwnProperty(r)){continue}x.k[r]=[w,u[r]];s.push(r)}}}if(!v){if(x.bWK){x.b="\\b("+s.join("|")+")\\s"}x.bR=f(y,x.b?x.b:"\\B|\\b");if(!x.e&&!x.eW){x.e="\\B|\\b"}if(x.e){x.eR=f(y,x.e)}}if(x.i){x.iR=f(y,x.i)}if(x.r===undefined){x.r=1}if(!x.c){x.c=[]}x.compiled=true;for(var t=0;t<x.c.length;t++){if(x.c[t]=="self"){x.c[t]=x}q(x.c[t],y,false)}if(x.starts){q(x.starts,y,false)}}for(var p in e){if(!e.hasOwnProperty(p)){continue}q(e[p].dM,e[p],true)}}function d(B,C){if(!j.called){j();j.called=true}function q(r,M){for(var L=0;L<M.c.length;L++){if((M.c[L].bR.exec(r)||[null])[0]==r){return M.c[L]}}}function v(L,r){if(D[L].e&&D[L].eR.test(r)){return 1}if(D[L].eW){var M=v(L-1,r);return M?M+1:0}return 0}function w(r,L){return L.i&&L.iR.test(r)}function K(N,O){var M=[];for(var L=0;L<N.c.length;L++){M.push(N.c[L].b)}var r=D.length-1;do{if(D[r].e){M.push(D[r].e)}r--}while(D[r+1].eW);if(N.i){M.push(N.i)}return f(O,M.join("|"),true)}function p(M,L){var N=D[D.length-1];if(!N.t){N.t=K(N,E)}N.t.lastIndex=L;var r=N.t.exec(M);return r?[M.substr(L,r.index-L),r[0],false]:[M.substr(L),"",true]}function z(N,r){var L=E.cI?r[0].toLowerCase():r[0];var M=N.k[L];if(M&&M instanceof Array){return M}return false}function F(L,P){L=m(L);if(!P.k){return L}var r="";var O=0;P.lR.lastIndex=0;var M=P.lR.exec(L);while(M){r+=L.substr(O,M.index-O);var N=z(P,M);if(N){x+=N[1];r+='<span class="'+N[0]+'">'+M[0]+"</span>"}else{r+=M[0]}O=P.lR.lastIndex;M=P.lR.exec(L)}return r+L.substr(O,L.length-O)}function J(L,M){if(M.sL&&e[M.sL]){var r=d(M.sL,L);x+=r.keyword_count;return r.value}else{return F(L,M)}}function I(M,r){var L=M.cN?'<span class="'+M.cN+'">':"";if(M.rB){y+=L;M.buffer=""}else{if(M.eB){y+=m(r)+L;M.buffer=""}else{y+=L;M.buffer=r}}D.push(M);A+=M.r}function G(N,M,Q){var R=D[D.length-1];if(Q){y+=J(R.buffer+N,R);return false}var P=q(M,R);if(P){y+=J(R.buffer+N,R);I(P,M);return P.rB}var L=v(D.length-1,M);if(L){var O=R.cN?"</span>":"";if(R.rE){y+=J(R.buffer+N,R)+O}else{if(R.eE){y+=J(R.buffer+N,R)+O+m(M)}else{y+=J(R.buffer+N+M,R)+O}}while(L>1){O=D[D.length-2].cN?"</span>":"";y+=O;L--;D.length--}var r=D[D.length-1];D.length--;D[D.length-1].buffer="";if(r.starts){I(r.starts,"")}return R.rE}if(w(M,R)){throw"Illegal"}}var E=e[B];var D=[E.dM];var A=0;var x=0;var y="";try{var s,u=0;E.dM.buffer="";do{s=p(C,u);var t=G(s[0],s[1],s[2]);u+=s[0].length;if(!t){u+=s[1].length}}while(!s[2]);if(D.length>1){throw"Illegal"}return{r:A,keyword_count:x,value:y}}catch(H){if(H=="Illegal"){return{r:0,keyword_count:0,value:m(C)}}else{throw H}}}function g(t){var p={keyword_count:0,r:0,value:m(t)};var r=p;for(var q in e){if(!e.hasOwnProperty(q)){continue}var s=d(q,t);s.language=q;if(s.keyword_count+s.r>r.keyword_count+r.r){r=s}if(s.keyword_count+s.r>p.keyword_count+p.r){r=p;p=s}}if(r.language){p.second_best=r}return p}function i(r,q,p){if(q){r=r.replace(/^((<[^>]+>|\t)+)/gm,function(t,w,v,u){return w.replace(/\t/g,q)})}if(p){r=r.replace(/\n/g,"<br>")}return r}function n(t,w,r){var x=h(t,r);var v=a(t);var y,s;if(v){y=d(v,x)}else{return}var q=c(t);if(q.length){s=document.createElement("pre");s.innerHTML=y.value;y.value=k(q,c(s),x)}y.value=i(y.value,w,r);var u=t.className;if(!u.match("(\\s|^)(language-)?"+v+"(\\s|$)")){u=u?(u+" "+v):v}if(/MSIE [678]/.test(navigator.userAgent)&&t.tagName=="CODE"&&t.parentNode.tagName=="PRE"){s=t.parentNode;var p=document.createElement("div");p.innerHTML="<pre><code>"+y.value+"</code></pre>";t=p.firstChild.firstChild;p.firstChild.cN=s.cN;s.parentNode.replaceChild(p.firstChild,s)}else{t.innerHTML=y.value}t.className=u;t.result={language:v,kw:y.keyword_count,re:y.r};if(y.second_best){t.second_best={language:y.second_best.language,kw:y.second_best.keyword_count,re:y.second_best.r}}}function o(){if(o.called){return}o.called=true;var r=document.getElementsByTagName("pre");for(var p=0;p<r.length;p++){var q=b(r[p]);if(q){n(q,hljs.tabReplace)}}}function l(){if(window.addEventListener){window.addEventListener("DOMContentLoaded",o,false);window.addEventListener("load",o,false)}else{if(window.attachEvent){window.attachEvent("onload",o)}else{window.onload=o}}}var e={};this.LANGUAGES=e;this.highlight=d;this.highlightAuto=g;this.fixMarkup=i;this.highlightBlock=n;this.initHighlighting=o;this.initHighlightingOnLoad=l;this.IR="[a-zA-Z][a-zA-Z0-9_]*";this.UIR="[a-zA-Z_][a-zA-Z0-9_]*";this.NR="\\b\\d+(\\.\\d+)?";this.CNR="\\b(0[xX][a-fA-F0-9]+|(\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)";this.BNR="\\b(0b[01]+)";this.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|\\.|-|-=|/|/=|:|;|<|<<|<<=|<=|=|==|===|>|>=|>>|>>=|>>>|>>>=|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~";this.ER="(?![\\s\\S])";this.BE={b:"\\\\.",r:0};this.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[this.BE],r:0};this.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[this.BE],r:0};this.CLCM={cN:"comment",b:"//",e:"$"};this.CBLCLM={cN:"comment",b:"/\\*",e:"\\*/"};this.HCM={cN:"comment",b:"#",e:"$"};this.NM={cN:"number",b:this.NR,r:0};this.CNM={cN:"number",b:this.CNR,r:0};this.BNM={cN:"number",b:this.BNR,r:0};this.inherit=function(r,s){var p={};for(var q in r){p[q]=r[q]}if(s){for(var q in s){p[q]=s[q]}}return p}}();hljs.LANGUAGES.cpp=function(){var a={keyword:{"false":1,"int":1,"float":1,"while":1,"private":1,"char":1,"catch":1,"export":1,virtual:1,operator:2,sizeof:2,dynamic_cast:2,typedef:2,const_cast:2,"const":1,struct:1,"for":1,static_cast:2,union:1,namespace:1,unsigned:1,"long":1,"throw":1,"volatile":2,"static":1,"protected":1,bool:1,template:1,mutable:1,"if":1,"public":1,friend:2,"do":1,"return":1,"goto":1,auto:1,"void":2,"enum":1,"else":1,"break":1,"new":1,extern:1,using:1,"true":1,"class":1,asm:1,"case":1,typeid:1,"short":1,reinterpret_cast:2,"default":1,"double":1,register:1,explicit:1,signed:1,typename:1,"try":1,"this":1,"switch":1,"continue":1,wchar_t:1,inline:1,"delete":1,alignof:1,char16_t:1,char32_t:1,constexpr:1,decltype:1,noexcept:1,nullptr:1,static_assert:1,thread_local:1,restrict:1,_Bool:1,complex:1},built_in:{std:1,string:1,cin:1,cout:1,cerr:1,clog:1,stringstream:1,istringstream:1,ostringstream:1,auto_ptr:1,deque:1,list:1,queue:1,stack:1,vector:1,map:1,set:1,bitset:1,multiset:1,multimap:1,unordered_set:1,unordered_map:1,unordered_multiset:1,unordered_multimap:1,array:1,shared_ptr:1}};return{dM:{k:a,i:"</",c:[hljs.CLCM,hljs.CBLCLM,hljs.QSM,{cN:"string",b:"'\\\\?.",e:"'",i:"."},{cN:"number",b:"\\b(\\d+(\\.\\d*)?|\\.\\d+)(u|U|l|L|ul|UL|f|F)"},hljs.CNM,{cN:"preprocessor",b:"#",e:"$"},{cN:"stl_container",b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:a,r:10,c:["self"]}]}}}();hljs.LANGUAGES.r={dM:{c:[hljs.HCM,{cN:"number",b:"\\b0[xX][0-9a-fA-F]+[Li]?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+(?:[eE][+\\-]?\\d*)?L\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+\\.(?!\\d)(?:i\\b)?",e:hljs.IMMEDIATE_RE,r:1},{cN:"number",b:"\\b\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"keyword",b:"(?:tryCatch|library|setGeneric|setGroupGeneric)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\.",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\d+(?![\\w.])",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\b(?:function)",e:hljs.IMMEDIATE_RE,r:2},{cN:"keyword",b:"(?:if|in|break|next|repeat|else|for|return|switch|while|try|stop|warning|require|attach|detach|source|setMethod|setClass)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"literal",b:"(?:NA|NA_integer_|NA_real_|NA_character_|NA_complex_)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"literal",b:"(?:NULL|TRUE|FALSE|T|F|Inf|NaN)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"identifier",b:"[a-zA-Z.][a-zA-Z0-9._]*\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"<\\-(?!\\s*\\d)",e:hljs.IMMEDIATE_RE,r:2},{cN:"operator",b:"\\->|<\\-",e:hljs.IMMEDIATE_RE,r:1},{cN:"operator",b:"%%|~",e:hljs.IMMEDIATE_RE},{cN:"operator",b:">=|<=|==|!=|\\|\\||&&|=|\\+|\\-|\\*|/|\\^|>|<|!|&|\\||\\$|:",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"%",e:"%",i:"\\n",r:1},{cN:"identifier",b:"`",e:"`",r:0},{cN:"string",b:'"',e:'"',c:[hljs.BE],r:0},{cN:"string",b:"'",e:"'",c:[hljs.BE],r:0},{cN:"paren",b:"[[({\\])}]",e:hljs.IMMEDIATE_RE,r:0}]}};
hljs.initHighlightingOnLoad();
</script>



<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<h2>more-cancers</h2>

<p><em>Last modified 01:34 PM on Apr 12, 2015. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/more-cancers/</code>.</em></p>

<pre><code class="r">library(stringr)
library(GenomicRanges)
library(GenomicAlignments)
library(Biobase)
library(DESeq2)
library(sva)
source(&#39;/inside/grotto/blin/programs/convertChrNames.R&#39;)
load(&#39;/inside/grotto/blin/trna-markers/feature-counts/features.RData&#39;)
</code></pre>

<p>We are expanding our analysis to lung adenocarcinoma (LUAD) and breast invasive carcinoma (BRCA), two of the most common cancers.</p>

<pre><code class="r"># metadata_file: sample metadata downloaded from CGHub in tab delimited form
# cancer_code: short abbreviation in lowercase describing the cancer type, e.g. &quot;prad&quot; or &quot;luad&quot;
parseMetadata &lt;- function(metadata_file, cancer_code) {
  metadata &lt;- read.table(metadata_file, header = TRUE, sep = &#39;\t&#39;, as.is = TRUE)
  metadata &lt;- metadata[, c(&#39;barcode&#39;, &#39;sample_type&#39;, &#39;filename&#39;, &#39;analysis_id&#39;, &#39;uploaded&#39;, &#39;participant_id&#39;)]
  # convert character &quot;uploaded&quot; column to date object for easier comparison
  metadata$uploaded &lt;- strptime(gsub(&quot;/&quot;, &quot;-&quot;, metadata$uploaded), &quot;%Y-%m-%d&quot;)
  # start by getting matched pairs
  # all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.
  paired &lt;- subset(metadata, participant_id %in% subset(metadata, sample_type == &quot;NT&quot;)$participant_id &amp; participant_id %in% subset(metadata, sample_type == &quot;TP&quot;)$participant_id)
  paired$paired &lt;- TRUE # remember which samples are matched tumor/normal
  parsed_metadata &lt;- data.frame()
  for (current_id in unique(paired$participant_id)) {
    nt &lt;- subset(paired, participant_id == current_id &amp; sample_type == &quot;NT&quot;)
    tp &lt;- subset(paired, participant_id == current_id &amp; sample_type == &quot;TP&quot;)
    parsed_metadata &lt;- rbind(parsed_metadata, tp[which(tp$uploaded == max(tp$uploaded)), ], nt[which(nt$uploaded == max(nt$uploaded)), ])
  }
  # now for the unpaired samples
  singles &lt;- metadata[-which(metadata$participant_id %in% parsed_metadata$participant_id), ]
  singles$paired &lt;- FALSE
  for (current_id in unique(singles$participant_id)) {
    ps &lt;- subset(singles, participant_id == current_id) # get all samples for this participant
    parsed_metadata &lt;- rbind(parsed_metadata, ps[which(ps$uploaded == max(ps$uploaded)), ])
  }
  # create participant ID numbers and new filenames
  numbering &lt;- match(parsed_metadata$participant_id, unique(parsed_metadata$participant_id))
  parsed_metadata$shortids &lt;- numbering
  numbering &lt;- formatC(numbering, flag = &quot;0&quot;, digits = 3)
  parsed_metadata$shortnames &lt;- paste0(cancer_code, &quot;-&quot;, numbering, &quot;-&quot;, parsed_metadata$sample_type)
  # reencode upload date as a string - it&#39;s encoded as a time object, screwing up data frame operations
  parsed_metadata$uploaded &lt;- as.character(parsed_metadata$uploaded)
  parsed_metadata
}
</code></pre>

<pre><code class="r">luad_metadata &lt;- parseMetadata(&#39;/inside/grotto/blin/trna-markers/more-cancers/luad-summary.tab&#39;, &#39;luad&#39;)
brca_metadata &lt;- parseMetadata(&#39;/inside/grotto/blin/trna-markers/more-cancers/brca-summary.tab&#39;, &#39;brca&#39;)
</code></pre>

<pre><code class="r">metadatas &lt;- list(luad = luad_metadata, brca = brca_metadata)
currentwd &lt;- getwd()
setwd(&#39;/inside/grotto/blin/trna-markers/datasets/raw&#39;)
for (cancer_code in names(metadatas)) { 
  metadata &lt;- metadatas[[cancer_code]]
  for (analysis_id in metadata$analysis_id) {
    if (file.exists(analysis_id)) next # if exists, assume file was downloaded correctly and is not corrupted
    system(paste0(&#39;gtdownload -c $CGKEY -d &#39;, analysis_id))
  }
  file.symlink(paste0(getwd(), &#39;/&#39;, metadata$analysis_id, &#39;/&#39;, metadata$barcode, &#39;_mirna.bam&#39;), paste0(&#39;/inside/grotto/blin/trna-markers/datasets/&#39;, cancer_code, &#39;/&#39;, metadata$shortnames, &#39;.bam&#39;))
}
setwd(currentwd)
</code></pre>

<p>We are missing <code>01810b1a-84e4-43d5-8a1e-42b132a1126f</code>, which is <code>brca-0939-TP.bam</code>. It&#39;s been removed for now.</p>

<pre><code class="bash">remap() {
  currentwd=`pwd`
  cd $1
  for prefix in `ls *-TP.bam *-NT.bam | cut -f 1 -d .`; do
    if [ ! -e $prefix-mapped.bam ]
    then
      if [ ! -e $prefix.fastq ]
      then
        bam2fastq --force -o $prefix# `readlink $prefix.bam`
        mv ${prefix}_M $prefix.fastq
        rm ${prefix}_1 ${prefix}_2
      fi
      bowtie2 -x ~/grotto/data/hg19 -k 100 -U ${prefix}.fastq -S $prefix-mapped-1.sam
      samtools view -S -F 4 -h -u $prefix-mapped-1.sam | samtools view -F 512 -h  - &gt; $prefix-mapped-2.sam
      bowtie2-best-mapped.py $prefix-mapped-2.sam | samtools view -S -u - | samtools sort - $prefix-mapped # auto appends .bam
    fi
    rm -f $prefix.fastq
    rm -f $prefix-mapped-1.sam
    rm -f $prefix-mapped-2.sam
  done
  cd $currentwd
}
remap /inside/grotto/blin/trna-markers/datasets/luad
remap /inside/grotto/blin/trna-markers/datasets/brca
</code></pre>

<pre><code class="r">countFeaturesFromBam &lt;- function(metadata, features, dir = &quot;.&quot;) {
  bamfiles &lt;- paste0(dir, &quot;/&quot;, metadata$shortnames, &quot;-mapped.bam&quot;)
  counts &lt;- vector()
  # use a for loop because r will run out of memory
  for (bamfile in bamfiles) {
    sample &lt;- readGAlignments(bamfile)
    sample &lt;- convertChrNames(sample, &quot;Ensembl&quot;)
    counts &lt;- cbind(counts, countOverlaps(features, sample, minoverlap = 5))
  }
  rownames(counts) &lt;- mcols(features)$tx_name
  colnames(counts) &lt;- metadata$barcode
  as.data.frame(counts)
}
</code></pre>

<pre><code class="r">luad_raw_counts &lt;- countFeaturesFromBam(luad_metadata, features, &#39;/inside/grotto/blin/trna-markers/datasets/luad/&#39;) 
save(file = &#39;luad-counts.RData&#39;, luad_raw_counts)
</code></pre>

<pre><code class="r">brca_raw_counts &lt;- countFeaturesFromBam(brca_metadata, features, &#39;/inside/grotto/blin/trna-markers/datasets/brca/&#39;)
</code></pre>

<pre><code>## Error in value[[3L]](cond): failed to open BamFile: SAM/BAM header missing or empty
##   file: &#39;/inside/grotto/blin/trna-markers/datasets/brca//brca-0939-TP-mapped.bam&#39;
</code></pre>

<pre><code class="r">save(file = &#39;brca-counts.RData&#39;, brca_raw_counts)
</code></pre>

<pre><code>## Error in save(file = &quot;brca-counts.RData&quot;, brca_raw_counts): object &#39;brca_raw_counts&#39; not found
</code></pre>

<pre><code class="r">getGeneCounts &lt;- function(rnaseq_dir, rnaseq_metadata_filename, mirna_metadata) {
  rnaseq_files &lt;- list.files(rnaseq_dir, full.names = TRUE)
  rnaseq_metadata &lt;- read.delim(paste0(rnaseq_dir, rnaseq_metadata_filename), header = TRUE, stringsAsFactors = FALSE)
  # split at character index 15, e.g. TCGA-ZG-A9N3-01
  rnaseq_metadata &lt;- data.frame(barcode = substr(rnaseq_metadata$Comment..TCGA.Barcode., 1, 15), analysis_id = rnaseq_metadata$Extract.Name, stringsAsFactors = FALSE)
  counts &lt;- NA
  for (barcode in mirna_metadata$barcode) {
    barcode4 &lt;- substr(barcode, 1, 15)
    analysis_id &lt;- rnaseq_metadata$analysis_id[match(barcode4, rnaseq_metadata$barcode)]
    if (is.na(str_detect(rnaseq_files, analysis_id))) {
      print(analysis_id)
      counts &lt;- cbind(counts, NA)
      next
    }
    gene_counts &lt;- read.table(gene_counts_file, row.names = 1, check.names = TRUE, stringsAsFactors = FALSE, header = TRUE)
    rownames(gene_counts) &lt;- make.unique(str_replace(rownames(gene_counts), &#39;\\|\\d+&#39;, &#39;&#39;))
    gene_counts &lt;- gene_counts[!grepl(&quot;\\?&quot;, rownames(gene_counts)), ]
    gene_ids &lt;- rownames(gene_counts)
    gene_counts &lt;- gene_counts$raw_count
    names(gene_counts) &lt;- gene_ids
    if (is.na(counts)) {
      counts &lt;- gene_counts
    }
    else {
      counts &lt;- cbind(counts, gene_counts)
    }
  }
  colnames(counts) &lt;- mirna_metadata$barcode
  counts
}
</code></pre>

<p>Lung housekeeping genes: ESD, POLR2A (<a href="http://www.sciencedirect.com/science/article/pii/S0169500208001931">Kuner et al. 2009</a>), GUSB (<a href="http://jco.ascopubs.org/content/27/27/4481.full">Giaccone et al. 2009</a>), ACTB, BAT1, B2M, TBP (<a href="http://www.pnas.org/content/106/8/2824.short">Boutros et al. 2009</a>), and GAPDH (<a href="http://onlinelibrary.wiley.com/doi/10.1002/ijc.25704/full">Sanchez-Palencia et al. 2011</a>)</p>

<pre><code class="r">brca_raw_counts &lt;- rbind(brca_raw_counts, getGeneCounts(&#39;/inside/home/blin/grotto/trna-markers/datasets/brca/rnaseq/&#39;, &#39;unc.edu_BRCA.IlluminaHiSeq_RNASeqV2.1.12.0.sdrf.txt&#39;, brca_metadata))
</code></pre>

<pre><code>## Error in eval(expr, envir, enclos): object &#39;brca_raw_counts&#39; not found
</code></pre>

<pre><code class="r">luad_raw_counts &lt;- rbind(luad_raw_counts, getGeneCounts(&#39;/inside/home/blin/grotto/trna-markers/datasets/luad/rnaseq/&#39;, &#39;unc.edu_LUAD.IlluminaHiSeq_RNASeqV2.1.15.0.sdrf.txt&#39;, luad_metadata))
</code></pre>

<pre><code>## Error in read.table(gene_counts_file, row.names = 1, check.names = TRUE, : object &#39;gene_counts_file&#39; not found
</code></pre>

<pre><code class="r">normalizeCounts &lt;- function(counts, metadata, control_indices) {
    counts &lt;- counts[apply(counts, 1, function(row) length(which(row &gt; 0)) &gt; length(colnames(features_counts)) * 0.05), ]
    counts &lt;- counts[apply(counts, 1, function(row) sum(row) &gt; 20), ]
    col_data &lt;- data.frame(row.names = colnames(counts), condition = metadata$sample_type, type = &#39;single-read&#39;)
    dds &lt;- DESeqDataSetFromMatrix(countData = counts, colData = col_data, design = ~ condition)
    dds &lt;- estimateSizeFactors(dds, controlGenes = control_indices)
    t(t(feature_counts) / sizeFactors(dds))
}
luad_normalized_counts &lt;- normalizeCounts(luad_raw_counts, luad_metadata, (nrow(luad_raw_counts) - 7):nrow(luad_raw_counts))
</code></pre>

<pre><code>## Error in colnames(features_counts): error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;colnames&#39;: Error: object &#39;features_counts&#39; not found
</code></pre>

<pre><code class="r">brca_normalized_counts &lt;- normalizeCounts(brca_raw_counts, brca_metadata)
</code></pre>

<pre><code>## Error in normalizeCounts(brca_raw_counts, brca_metadata): object &#39;brca_raw_counts&#39; not found
</code></pre>

<pre><code class="r">processClinicalDataFile &lt;- function(biotab_file, batch_file) {
  biotab &lt;- read.table(biotab_file, header = TRUE, sep = &quot;\t&quot;, stringsAsFactors = FALSE)
  batches &lt;- read.table(batch_file, header = TRUE, sep = &quot;\t&quot;, as.is = c(FALSE, TRUE))
  biotab$bcr_patient_uuid &lt;- tolower(biotab$bcr_patient_uuid) # all data file ids are in lowercase, not sure if matters
  biotab$days_survived &lt;- replace(biotab$death_days_to, biotab$death_days_to == &quot;[Not Applicable]&quot;, biotab$last_contact_days_to[which(biotab$death_days_to == &quot;[Not Applicable]&quot;)]) # either days from initial prognosis to death or initial prognosis to censor time
  biotab$batch &lt;- batches[match(biotab$bcr_patient_barcode, batches$barcode), &quot;batch&quot;]
  biotab
}
luad_clinical &lt;- processClinicalDataFile(&#39;luad-clinical-biotab-04032015.tab&#39;, &#39;luad-clinical-batches-04032015.tab&#39;)
luad_clinical &lt;- with(luad_clinical[match(luad_metadata$participant_id, luad_clinical$bcr_patient_uuid), ], 
                      data.frame(luad_metadata,
                                gender = gender, 
                                race = race,
                                smoker = ifelse(tobacco_smoking_history_indicator == &quot;Lifelong Non-smoker&quot;, 0, 1), # NA doesn&#39;t return true or false, just NA. this is useful.
                                t_stage = ajcc_tumor_pathologic_pt, 
                                m_stage = ajcc_metastasis_pathologic_pm, 
                                n_stage = ajcc_nodes_pathologic_pn,
                                stage = ajcc_pathologic_tumor_stage,
                                days_survived = as.integer(days_survived), 
                                vital_status = ifelse(vital_status == &quot;Alive&quot;, 0, 1),
                                age = age_at_initial_pathologic_diagnosis,
                                batch = batch))
brca_clinical &lt;- processClinicalDataFile(&#39;brca-clinical-biotab-04032015.tab&#39;, &#39;brca-clinical-batches-04032015.tab&#39;)
</code></pre>

<pre><code>## Error in scan(file, what, nmax, sep, dec, quote, skip, nlines, na.strings, : line 664 did not have 109 elements
</code></pre>

<pre><code class="r">brca_clinical &lt;- with(brca_clinical[match(brca_metadata$participant_id, brca_clinical$bcr_patient_uuid), ], 
                      data.frame(brca_metadata,
                                ethnicity = ethnicity,
                                race = race,
                                vital_status = ifelse(vital_status == &quot;Alive&quot;, 0, 1),
                                days_survived = as.integer(days_survived),
                                age = age_at_diagnosis,
                                t_stage = ajcc_tumor_pathologic_pt, 
                                m_stage = ajcc_metastasis_pathologic_pm, 
                                n_stage = ajcc_nodes_pathologic_pn,
                                stage = ajcc_pathologic_tumor_stage,
                                er_status = er_status_by_ihc,
                                pr_status = pr_status_by_ihc,
                                her2_status = her2_status_by_ihc,
                                batch = batch))
</code></pre>

<pre><code>## Error in with(brca_clinical[match(brca_metadata$participant_id, brca_clinical$bcr_patient_uuid), : error in evaluating the argument &#39;data&#39; in selecting a method for function &#39;with&#39;: Error: object &#39;brca_clinical&#39; not found
</code></pre>

<pre><code class="r">removeBatchEffects &lt;- function(metadata, counts) {
  pheno_data &lt;- data.frame(row.names = metadata$barcode, sample_type = metadata$sample_type)
  mod &lt;- model.matrix(~ 1, data = pheno_data)
  batch &lt;- as.factor(ifelse(!is.na(metadata$batch), as.character(metadata$batch), &quot;Batch Unknown&quot;))
  batch_removed_counts &lt;- ComBat(dat = counts, batch = batch, mod = mod)
}
luad_adjusted_counts &lt;- removeBatchEffects(luad_metadata, luad_normalized_counts)
</code></pre>

<pre><code>## Found 0 batches
## Found 0  categorical covariate(s)
</code></pre>

<pre><code>## Error in ComBat(dat = counts, batch = batch, mod = mod): object &#39;luad_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">brca_adjusted_counts &lt;- removeBatchEffects(brca_metadata, brca_normalized_counts)
</code></pre>

<pre><code>## Found 0 batches
## Found 0  categorical covariate(s)
</code></pre>

<pre><code>## Error in ComBat(dat = counts, batch = batch, mod = mod): object &#39;brca_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">plotPCA &lt;- function(counts) {
  pca &lt;- prcomp(counts, scale = TRUE)
  pca_scores &lt;- as.data.frame(pca$x)
  ggplot(pca_scores, aes(x = PC1, y = PC2)) + geom_point(shape = 1)
}
plotPCACancerType &lt;- function(normalized_counts, adjusted_counts, metadata, cancer_code) {
    plotPCA(t(normalized_counts)[, metadata$sample_type == &quot;TP&quot;]) + ggtitle(paste(cancer_code, &quot;TP Before&quot;))
    plotPCA(t(adjusted_counts)[, metadata$sample_type == &quot;TP&quot;]) + ggtitle(paste(cancer_code, &quot;TP After&quot;))
    plotPCA(t(normalized_counts)[, metadata$sample_type == &quot;NT&quot;]) + ggtitle(paste(cancer_code, &quot;NT Before&quot;))
    plotPCA(t(adjusted_counts)[, metadata$sample_type == &quot;NT&quot;]) + ggtitle(paste(cancer_code, &quot;NT After&quot;))
}
plotPCACancerType(luad_normalized_counts, luad_adjusted_counts, luad_metadata, &quot;LUAD&quot;)
</code></pre>

<pre><code>## Error in t(normalized_counts): error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;luad_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">plotPCACancerType(brca_normalized_counts, brca_adjusted_counts, brca_metadata, &quot;BRCA&quot;)
</code></pre>

<pre><code>## Error in t(normalized_counts): error in evaluating the argument &#39;x&#39; in selecting a method for function &#39;t&#39;: Error: object &#39;brca_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">save(file = &#39;luad-counts.RData&#39;, luad_raw_counts, luad_normalized_counts)
</code></pre>

<pre><code>## Error in save(file = &quot;luad-counts.RData&quot;, luad_raw_counts, luad_normalized_counts): object &#39;luad_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">save(file = &#39;brca-counts.RData&#39;, brca_raw_counts, brca_normalized_counts)
</code></pre>

<pre><code>## Error in save(file = &quot;brca-counts.RData&quot;, brca_raw_counts, brca_normalized_counts): objects &#39;brca_raw_counts&#39;, &#39;brca_normalized_counts&#39; not found
</code></pre>

<pre><code class="r">save(file = &#39;luad-metadata.RData&#39;, luad_metadata, luad_clinical)
save(file = &#39;brca-metadata.RData&#39;, brca_metadata, brca_clinical)
</code></pre>

<pre><code>## Error in save(file = &quot;brca-metadata.RData&quot;, brca_metadata, brca_clinical): object &#39;brca_clinical&#39; not found
</code></pre>

<pre><code class="r">save.image(&quot;more-cancers-image.RData&quot;)
</code></pre>

</body>

</html>
