<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>+/- </title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>





<style type="text/css">

</style>

<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>

<!-- highlight.js -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/styles/default.css" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/highlight.min.js"></script>
<script src="https://yandex.st/highlightjs/7.3/languages/r.min.js"></script>

<!-- bootstrap -->
<link href=https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css rel="stylesheet">
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>

<!-- Manific Popup -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/0.8.9/jquery.magnific-popup.min.js"></script>

<script defer="defer">
// Function to generate the dynamic table of contents
jQuery.fn.generate_TOC = function () {
  var base = $(this[0]);

  var selectors = ['h1', 'h2', 'h3', 'h4'];

  var last_ptr = [{}, {}, {}, {}];

  var anchors = {};

  generate_anchor = function (text) {
    var test = text.replace(/\W/g, '_');

    while(test in anchors){
      //if no suffix, add one
      if(test.match(/_\d+$/) === null){
        test = test + "_2";
      }
      //else generate unique id for duplicates by adding one to the suffix
      else {
        test = test.replace(/_(\d+)$/, function(match, number){ var num=+number+1; return("_" + num) });
      }
    }
    anchors[test]=1;
    return(test);
  }

  $(selectors.join(',')).each(function () {
    var heading = $(this);
    var idx = selectors.indexOf(heading.prop('tagName').toLowerCase());
    var itr = 0;

    while (itr <= idx) {
      if (jQuery.isEmptyObject(last_ptr[itr])) {
        last_ptr[itr] = $('<ul>').addClass('nav');
        if (itr === 0) {
          base.append(last_ptr[itr])
        } else {
          if(last_ptr[itr-1].children('li').length === 0){
            last_ptr[itr-1].append(last_ptr[itr]);
          }
          else {
            last_ptr[itr - 1].children('li').last().append(last_ptr[itr]);
          }
        }
      }
      itr++;
    }
    var anchor = generate_anchor(heading.text());
    heading.attr('id', anchor);
    var a = $('<a>')
    .text(heading.text())
    .attr('href', '#' + anchor);

  var li = $('<li>')
    .append(a);

  last_ptr[idx].append(li);
  for (i = idx + 1; i < last_ptr.length; i++) {
    last_ptr[i] = {};
  }
  });
}
/* run scripts when document is ready */
$(function() {
  "use strict";

  var $window = $(window);
  var $body = $(document.body);

  document.title = $('h1').first().text();

  /* size of thumbnails */
  var thumbsize = "col-md-3";

  var show_code =  false ;

  var show_output =  true ;

  var show_figure =  true ;

  /* included languages */
  var languages = [];

  /* Using render_html, so add in code block */
  $('pre.knitr').each(function(){
    $(this).removeClass('knitr');
    if($(this).find('code').length < 1){
      $(this).wrapInner('<code class=' + $(this).attr('class') + '></code>');
    }
  });

  /* style tables */
  $('table').addClass('table table-striped table-bordered table-hover table-condensed');

  /* add toggle panel to rcode blocks */
  $('div.source,div.output,div.message,div.warning,div.error').each(function() {
    var button = $('<h5 class="panel-title">+/- </h5>');

    if($(this).hasClass('source')){
      var code_block = $(this).find('code');
      var lang_type = code_block.attr('class');
      button.text(button.text() + lang_type + ' Code');
      button.addClass('source ' + lang_type);
      languages[lang_type]=0;
      code_block.each(function(i, e) {
        hljs.highlightBlock(e);
      });
      $(this).addClass('panel panel-primary ' + lang_type);
    }
    else if($(this).hasClass('output')){
      button.text(button.text() + 'Output');
      button.addClass('output');
      $(this).addClass('panel panel-success');
    }
    else if($(this).hasClass('message')){
      button.text(button.text() + 'Message');
      button.addClass('message');
      $(this).addClass('panel panel-info');
    }
    else if($(this).hasClass('warning')){
      button.text(button.text() + 'Warning');
      button.addClass('warning');
      $(this).addClass('panel panel-warning');
    }
    else if($(this).hasClass('error')){
      button.text(button.text() + 'Error');
      button.addClass('error');
      $(this).addClass('panel panel-danger');
    }
    else {
      //console.log($(this));
    }
    $(this).prepend($('<div class="panel-heading toggle" />').append(button));
  });

  /* give images a lightbox and thumbnail classes to allow lightbox and thumbnails TODO: make gallery if graphs are grouped */
  $('div.rimage img').each(function(){

    //remove rimage div
    $(this).unwrap();

    var a = $(this).
      wrap('<a href=# class="media-object pull-left mfp-image thumbnail ' + thumbsize + '"></a>').
      parent();

    var sibs = a.prevUntil('div.rimage,div.media');
    var div = $('<div class="media" />');
    if(sibs.length != 0){
      sibs.addClass('media-body');
      //need to reverse order as prevUntil puts objects in the order it found them
      $(sibs.get().reverse()).wrapAll(div).parent().prepend(a);
    }
    else {
      a.wrap(div);
    }
  });

  $('div.chunk').addClass('media');

  $('.rcode > .panel').addClass('media');
  /* Magnific Popup */
  $(".thumbnail").each(function(){
    $(this).magnificPopup({
      disableOn: 768,
      closeOnContentClick: true,

      type: 'image',
      items: {
        src: $(this).find('img').attr('src'),
      }
    });
  });

  /* add bootstrap classes */
  $('body').wrapInner('<div class="container"><div class="row"><div class="contents">');

  var create_language_links = function(){
    var text='';
    var language;
    for(language in languages){
      if(languages.hasOwnProperty(language)){
        text += '<li><a href=# class="toggle-global source ' + language + '" type="source.' + language + '">' + language + '</a></li>\n';
      }
    }
    return text;
  }

  var navbar =
  '<div class="navbar navbar-fixed-bottom navbar-inverse">\
    <div class="container">\
      <div class="navbar-header">\
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
        </button>\
      </div>\
      <div id="bottom-navbar" class="navbar-collapse collapse navbar-responsive-collapse">\
        <ul class="nav navbar-nav navbar-right">\
          <li class="nav"><p class="navbar-text">Toggle</p></li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Code <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Languages</li>'
              + create_language_links() +
              '<li><a href="#" type="all-source" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Output <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Type</li>\
                <li><a href="#" type="output" class="toggle-global">Output</a></li>\
                <li><a href="#" type="message" class="toggle-global">Message</a></li>\
                <li><a href="#" type="warning" class="toggle-global">Warning</a></li>\
                <li><a href="#" type="error" class="toggle-global">Error</a></li>\
                <li><a href="#" type="all-output" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li><a href="#" class="toggle-figure">Figures</a></li>\
        </ul>\
      </div><!-- /.nav-collapse -->\
    </div><!-- /.container -->\
  </div>';
  /* add navbar */
  $('.container').append(navbar);

  /* onclick toggle next code block */
  $('.toggle').click(function() {
    $(this).next('pre').slideToggle();
    return false;
  });

  // global toggles
  $('.toggle-global').click(function(){
    var type = $(this).attr('type');
    if(type == 'all-source'){
      for(var language in languages){
        $('li a[type="source.' + language + '"]').click();
      }
    }
    if(type == 'all-output'){
      $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error]').click();
    }
    else {
      if($(this).closest('li').hasClass('active')){
        $('div.' + type).children('pre').slideUp();
      }
      else {
        $('div.' + type).children('pre').slideDown();
      }
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });
  // global toggles figure, this is more complicated than just slideToggle because you have to hide the parent div as well
  $('.toggle-figure').click(function(){
    var imgs = $('.thumbnail img');
    if(imgs.is(":visible")){
      imgs.slideUp(400, function(){ $(this).parent().toggle(); });
    }
    else {
      imgs.parent().show();
      imgs.show();
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });

  /* add footer */
  $('body').wrapInner('<div id="wrap" />');
  $('#wrap').append('<div id="push" />');
  var p = $('p:contains("Author:")');
  var last_p = p.filter(':last');
  last_p.addClass('text-muted').attr('id','credit');
  last_p.append('<p>Styled with <a href="https://github.com/jimhester/knitrBootstrap">knitrBootstrap</a></p>');
  last_p = last_p.wrap('<div id="footer"><div class="container">').parent().parent();
  last_p.appendTo("body");

  $('.container > .row').prepend('<div class="col-md-3"><div id="toc" class="well sidebar sidenav affix hidden-print"/></div>');

  $('.contents').addClass('col-md-offset-3');

  /* table of contents */
  $('#toc').generate_TOC();

  if(show_code){
    /* toggle source buttons pressed */
    $('a.toggle-global.source').closest('li').addClass('active');
  }
  else {
    /* hide code blocks */
    $('div.source pre').hide();
  }

  if(show_output){
    /* toggle output buttons pressed */
    $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error], li a[type=all-output]').closest('li').addClass('active');
  }
  else {
    /* hide output blocks */
    $('div.output pre').hide();
  }

  if(show_figure){
    /* toggle figure button pressed */
    $('li a.toggle-figure').closest('li').addClass('active');
  }
  else {
    /* hide figures */
    $('.thumbnail').hide();
  }

  /* remove paragraphs with no content */
  $('p:empty').remove();

  $body.scrollspy({
    target: '.sidebar',
  });


  //TODO refresh on show/hide
  $window.on('load', function () {
    $body.scrollspy('refresh');
  })

});


</script>
<style>
/* Knitr_bootstrap styles */
#wrap .container-fluid {
  padding: 0;
  overflow: hidden;
}
.toggle{
  text-transform: capitalize;
}

.toggle-global{
  text-transform: capitalize;
}

/* Sticky footer styles */
* {
  margin:0;
}
html,
body {
    height: 100%;
    padding:0 !important;
    /* The html and body elements cannot have any padding or margin. */
    /*overflow-x: hidden;*/
}

/* Wrapper for page content to push down footer */
#wrap {
    min-height: 100%;
    height: auto !important;
    height: 100%;
    /* Negative indent footer by it's height */
    margin: 0 auto -120px;
}

/* Set the fixed height of the footer here */
#push,
#footer {
    height: 120px;
}

#footer {
  text-align: center;
}

/* Top level subheader elements.  These are the first nested items underneath a header element. */
.header li {
  font-size: 20px;
}

/* Makes the font smaller for all subheader elements. */
.sub-header li {
    font-size: 12px;
}

button.thumbnails {
  margin-left:0px;
}

/*
 * Side navigation
 *
 * Scrollspy and affixed enhanced navigation to highlight sections and secondary
 * sections of docs content.
 */

/* By default it's not affixed in mobile views, so undo that */
.sidebar.affix {
  position: static;
}

/* First level of nav */
.sidenav {
  margin-top: 30px;
  margin-bottom: 30px;
  padding-top:    10px;
  padding-bottom: 10px;
  border-radius: 5px;
}

/* All levels of nav */
.sidebar .nav > li > a {
  display: block;
  padding: 5px 20px;
}
.sidebar .nav > li > a:hover,
.sidebar .nav > li > a:focus {
  text-decoration: none;
  border-right: 1px solid;
}
.sidebar .nav > .active > a,
.sidebar .nav > .active:hover > a,
.sidebar .nav > .active:focus > a {
  font-weight: bold;
  background-color: transparent;
  border-right: 1px solid;
}

/* Nav: second level (shown on .active) */
.sidebar .nav .nav {
  display: none; /* Hide by default, but at >768px, show it */
  margin-bottom: 8px;
}
.sidebar .nav .nav > li > a {
  padding-top:    3px;
  padding-bottom: 3px;
  padding-left: 30px;
  font-size: 90%;
}

.sidebar .nav .nav .nav > li > a {
  padding-left: 40px;
}
.sidebar .nav .nav .nav .nav > li > a {
  padding-left: 50px;
}

/* Show and affix the side nav when space allows it */
@media screen and (min-width: 992px) {
  .sidebar .nav > .active > ul {
    display: block;
  }
  /* Widen the fixed sidebar */
  .sidebar.affix,
  .sidebar.affix-bottom {
    width: 213px;
  }
  .sidebar.affix-top,
  .sidebar.affix {
    position: fixed; /* Undo the static from mobile first approach */
    top: 30px;
  }
  .sidebar.affix-bottom {
    position: absolute; /* Undo the static from mobile first approach */
  }
  .sidebar.affix-bottom .sidenav,
  .sidebar.affix .sidenav {
    margin-top: 0;
    margin-bottom: 0;
  }
}
@media screen and (min-width: 1200px) {
  /* Widen the fixed sidebar again */
  .sidebar.affix-bottom,
  .sidebar.affix {
    width: 263px;
  }
}

#toc {
  padding: 10px 0px;
  margin:0;
  border:0;
}


.panel pre {
  border-radius: 0;
  border: 0;
  margin: 0;
  padding: 0;
}
.panel pre code {
  border-radius: 0;
}
/* Magnific Popup CSS */
.mfp-bg {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1042;
  overflow: hidden;
  position: fixed;
  background: #0b0b0b;
  opacity: 0.8;
  filter: alpha(opacity=80); }

.mfp-wrap {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1043;
  position: fixed;
  outline: none !important;
  -webkit-backface-visibility: hidden; }

.mfp-container {
  text-align: center;
  position: absolute;
  width: 100%;
  height: 100%;
  left: 0;
  top: 0;
  padding: 0 8px;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box; }

.mfp-container:before {
  content: '';
  display: inline-block;
  height: 100%;
  vertical-align: middle; }

.mfp-align-top .mfp-container:before {
  display: none; }

.mfp-content {
  position: relative;
  display: inline-block;
  vertical-align: middle;
  margin: 0 auto;
  text-align: left;
  z-index: 1045; }

.mfp-inline-holder .mfp-content,
.mfp-ajax-holder .mfp-content {
  width: 100%;
  cursor: auto; }

.mfp-ajax-cur {
  cursor: progress; }

.mfp-zoom-out-cur,
.mfp-zoom-out-cur .mfp-image-holder .mfp-close {
  cursor: -moz-zoom-out;
  cursor: -webkit-zoom-out;
  cursor: zoom-out; }

.mfp-zoom {
  cursor: pointer;
  cursor: -webkit-zoom-in;
  cursor: -moz-zoom-in;
  cursor: zoom-in; }

.mfp-auto-cursor .mfp-content {
  cursor: auto; }

.mfp-close,
.mfp-arrow,
.mfp-preloader,
.mfp-counter {
  -webkit-user-select: none;
  -moz-user-select: none;
  user-select: none; }

.mfp-loading.mfp-figure {
  display: none; }

.mfp-hide {
  display: none !important; }

.mfp-preloader {
  color: #cccccc;
  position: absolute;
  top: 50%;
  width: auto;
  text-align: center;
  margin-top: -0.8em;
  left: 8px;
  right: 8px;
  z-index: 1044; }

.mfp-preloader a {
  color: #cccccc; }

.mfp-preloader a:hover {
  color: white; }

.mfp-s-ready .mfp-preloader {
  display: none; }

.mfp-s-error .mfp-content {
  display: none; }

button.mfp-close,
button.mfp-arrow {
  overflow: visible;
  cursor: pointer;
  background: transparent;
  border: 0;
  -webkit-appearance: none;
  display: block;
  padding: 0;
  z-index: 1046;
  -webkit-box-shadow: none;
  box-shadow: none; }

button::-moz-focus-inner {
  padding: 0;
  border: 0; }

.mfp-close {
  width: 44px;
  height: 44px;
  line-height: 44px;
  position: absolute;
  right: 0;
  top: 0;
  text-decoration: none;
  text-align: center;
  opacity: 0.65;
  padding: 0 0 18px 10px;
  color: white;
  font-style: normal;
  font-size: 28px;
  font-family: Arial, Baskerville, monospace; }
  .mfp-close:hover, .mfp-close:focus {
    opacity: 1; }
  .mfp-close:active {
    top: 1px; }

.mfp-close-btn-in .mfp-close {
  color: #333333; }

.mfp-image-holder .mfp-close,
.mfp-iframe-holder .mfp-close {
  color: white;
  right: -6px;
  text-align: right;
  padding-right: 6px;
  width: 100%; }

.mfp-counter {
  position: absolute;
  top: 0;
  right: 0;
  color: #cccccc;
  font-size: 12px;
  line-height: 18px; }

.mfp-arrow {
  position: absolute;
  opacity: 0.65;
  margin: 0;
  top: 50%;
  margin-top: -55px;
  padding: 0;
  width: 90px;
  height: 110px;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0); }

.mfp-arrow:active {
  margin-top: -54px; }

.mfp-arrow:hover,
.mfp-arrow:focus {
  opacity: 1; }

.mfp-arrow:before, .mfp-arrow:after,
.mfp-arrow .mfp-b,
.mfp-arrow .mfp-a {
  content: '';
  display: block;
  width: 0;
  height: 0;
  position: absolute;
  left: 0;
  top: 0;
  margin-top: 35px;
  margin-left: 35px;
  border: medium inset transparent; }
.mfp-arrow:after,
.mfp-arrow .mfp-a {
  border-top-width: 13px;
  border-bottom-width: 13px;
  top: 8px; }
.mfp-arrow:before,
.mfp-arrow .mfp-b {
  border-top-width: 21px;
  border-bottom-width: 21px; }

.mfp-arrow-left {
  left: 0; }
  .mfp-arrow-left:after,
  .mfp-arrow-left .mfp-a {
    border-right: 17px solid white;
    margin-left: 31px; }
  .mfp-arrow-left:before,
  .mfp-arrow-left .mfp-b {
    margin-left: 25px;
    border-right: 27px solid #3f3f3f; }

.mfp-arrow-right {
  right: 0; }
  .mfp-arrow-right:after,
  .mfp-arrow-right .mfp-a {
    border-left: 17px solid white;
    margin-left: 39px; }
  .mfp-arrow-right:before,
  .mfp-arrow-right .mfp-b {
    border-left: 27px solid #3f3f3f; }

.mfp-iframe-holder {
  padding-top: 40px;
  padding-bottom: 40px; }

.mfp-iframe-holder .mfp-content {
  line-height: 0;
  width: 100%;
  max-width: 900px; }

.mfp-iframe-scaler {
  width: 100%;
  height: 0;
  overflow: hidden;
  padding-top: 56.25%; }

.mfp-iframe-scaler iframe {
  position: absolute;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: black; }

.mfp-iframe-holder .mfp-close {
  top: -40px; }

/* Main image in popup */
img.mfp-img {
  width: auto;
  max-width: 100%;
  height: auto;
  display: block;
  line-height: 0;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
  padding: 40px 0 40px;
  margin: 0 auto; }

/* The shadow behind the image */
.mfp-figure:after {
  content: '';
  position: absolute;
  left: 0;
  top: 40px;
  bottom: 40px;
  display: block;
  right: 0;
  width: auto;
  height: auto;
  z-index: -1;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: #444444; }

.mfp-figure {
  line-height: 0; }

.mfp-bottom-bar {
  margin-top: -36px;
  position: absolute;
  top: 100%;
  left: 0;
  width: 100%;
  cursor: auto; }

.mfp-title {
  text-align: left;
  line-height: 18px;
  color: #f3f3f3;
  word-wrap: break-word;
  padding-right: 36px; }

.mfp-figure small {
  color: #bdbdbd;
  display: block;
  font-size: 12px;
  line-height: 14px; }

.mfp-image-holder .mfp-content {
  max-width: 100%; }

.mfp-gallery .mfp-image-holder .mfp-figure {
  cursor: pointer; }

@media screen and (max-width: 800px) and (orientation: landscape), screen and (max-height: 300px) {
  /**
   * Remove all paddings around the image on small screen
   */
  .mfp-img-mobile .mfp-image-holder {
    padding-left: 0;
    padding-right: 0; }

  .mfp-img-mobile img.mfp-img {
    padding: 0; }

  /* The shadow behind the image */
  .mfp-img-mobile .mfp-figure:after {
    top: 0;
    bottom: 0; }

  .mfp-img-mobile .mfp-bottom-bar {
    background: rgba(0, 0, 0, 0.6);
    bottom: 0;
    margin: 0;
    top: auto;
    padding: 3px 5px;
    position: fixed;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box; }

  .mfp-img-mobile .mfp-bottom-bar:empty {
    padding: 0; }

  .mfp-img-mobile .mfp-counter {
    right: 5px;
    top: 3px; }

  .mfp-img-mobile .mfp-close {
    top: 0;
    right: 0;
    width: 35px;
    height: 35px;
    line-height: 35px;
    background: rgba(0, 0, 0, 0.6);
    position: fixed;
    text-align: center;
    padding: 0; }

  .mfp-img-mobile .mfp-figure small {
    display: inline;
    margin-left: 5px; } }
@media all and (max-width: 900px) {
  .mfp-arrow {
    -webkit-transform: scale(0.75);
    transform: scale(0.75); }

  .mfp-arrow-left {
    -webkit-transform-origin: 0;
    transform-origin: 0; }

  .mfp-arrow-right {
    -webkit-transform-origin: 100%;
    transform-origin: 100%; }

  .mfp-container {
    padding-left: 6px;
    padding-right: 6px; } }
.mfp-ie7 .mfp-img {
  padding: 0; }
.mfp-ie7 .mfp-bottom-bar {
  width: 600px;
  left: 50%;
  margin-left: -300px;
  margin-top: 5px;
  padding-bottom: 5px; }
.mfp-ie7 .mfp-container {
  padding: 0; }
.mfp-ie7 .mfp-content {
  padding-top: 44px; }
.mfp-ie7 .mfp-close {
  top: 0;
  right: 0;
  padding-top: 0; }

//Magnific overrides
.mfp-image img{
  background: white;
}
.mfp-figure:after {
  background: white;
}

</style>

 

</head>

<body>
<h1>Process data</h1>

<p><em>Last modified <code class="knitr inline">08:19 PM on May 13, 2016</code>. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/process-reads/</code>.</em></p>

<div class="chunk" id="libraries"><div class="rcode"><div class="source"><pre class="knitr r">library(stringr)
library(GenomicRanges)
library(GenomicAlignments)
library(BiocParallel)
source('/inside/grotto/blin/programs/convertChrNames.R')
load('/inside/grotto/blin/data/hg19-srnas.RData')
</pre></div>

<p></div></div></p>

<p>This script downloads and remaps reads. </p>

<ul>
<li>v1: Prostate adenocarcinoma (PRAD)</li>
<li>v2: Lung adenocarcinoma (LUAD) and breast invasive carcinoma (BRCA)</li>
<li>v3: Hepatocellular carcinoma (LIHC), Lymphoid neoplasm diffuse large b-cell lymphoma (DLBC), colorectal (COAD and READ), pancreatic (PAAD), testicular germ cell (TGCT), lower grade glioma (LGG)

<ul>
<li>acute myeloid leukemia (AML and LAML), glioblastoma (GBM), osteosarcoma (OS) do not have enough samples for miRNA-Seq data.</li>
</ul></li>
</ul>

<h2>Parse file and clinical metadata</h2>

<div class="chunk" id="parse-metadata-function"><div class="rcode"><div class="source"><pre class="knitr r"># metadata_file: sample metadata downloaded from CGHub in tab delimited form
# cancer_code: short abbreviation in lowercase describing the cancer type, e.g. "prad" or "luad"
parseMetadata <- function(metadata_file, cancer_code) {
  metadata <- read.table(metadata_file, header = TRUE, sep = '\t', as.is = TRUE)
  metadata <- metadata[, c('barcode', 'sample_type', 'filename', 'analysis_id', 'analyte_type', 'uploaded', 'participant_id')]
  # convert character "uploaded" column to date object for easier comparison
  metadata$uploaded <- strptime(gsub("/", "-", metadata$uploaded), "%Y-%m-%d")
  # start by getting matched pairs
  # first, check if there actually are normal tissue samples
  parsed_metadata <- data.frame()
  if (any(metadata$sample_type == "NT")) {
  # all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.
    paired <- subset(metadata, participant_id %in% subset(metadata, sample_type == "NT")$participant_id & participant_id %in% subset(metadata, sample_type == "TP")$participant_id)
    paired$paired <- TRUE # remember which samples are matched tumor/normal
    for (current_id in unique(paired$participant_id)) {
      nt <- subset(paired, participant_id == current_id & sample_type == "NT")
      tp <- subset(paired, participant_id == current_id & sample_type == "TP")
      parsed_metadata <- rbind(parsed_metadata, tp[which(tp$uploaded == max(tp$uploaded)), ], nt[which(nt$uploaded == max(nt$uploaded)), ])
    }
    singles <- metadata[-which(metadata$participant_id %in% parsed_metadata$participant_id), ]
  } else {
    singles <- metadata
  }
  # now for the unpaired samples
  singles$paired <- FALSE
  for (current_id in unique(singles$participant_id)) {
    ps <- subset(singles, participant_id == current_id) # get all samples for this participant
    parsed_metadata <- rbind(parsed_metadata, ps[which(ps$uploaded == max(ps$uploaded)), ])
  }
  # create sample ID numbers and new filenames
  numbering <- 1:nrow(parsed_metadata)
  parsed_metadata$shortids <- numbering
  numbering <- formatC(numbering, flag = "0", digits = 3)
  parsed_metadata$shortnames <- paste0(cancer_code, "-", numbering)
  # reencode upload date as a string - it's encoded as a time object, screwing up data frame operations
  parsed_metadata$uploaded <- as.character(parsed_metadata$uploaded)
  parsed_metadata
}
processClinicalDataFile = function(biotab_file, batch_file) {
  biotab = read.delim(biotab_file, header = TRUE, sep = "\t", stringsAsFactors = FALSE)
  batches = read.table(batch_file, header = TRUE, sep = "\t", as.is = c(FALSE, TRUE))
  biotab$bcr_patient_uuid = tolower(biotab$bcr_patient_uuid) # all data file ids are in lowercase, not sure if matters
  biotab$days_survived = replace(biotab$death_days_to, biotab$death_days_to == "[Not Applicable]", biotab$last_contact_days_to[which(biotab$death_days_to == "[Not Applicable]")]) # either days from initial prognosis to death or initial prognosis to censor time
  biotab$batch = batches[match(biotab$bcr_patient_barcode, substr(batches$barcode, 1, 12)), "batch"]
  biotab
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="parse-metadata"><div class="rcode"><div class="source"><pre class="knitr r">luad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/luad-summary.tab', 'luad')
brca_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/brca-summary.tab', 'brca')
prad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/prad-summary.tab', 'prad')
dlbc_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/dlbc-summary.tab', 'dlbc')
lihc_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lihc-summary.tab', 'lihc')
coad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/coad-summary.tab', 'coad')
paad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/paad-summary.tab', 'paad')
tgct_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/tgct-summary.tab', 'tgct')
lgg_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lgg-summary.tab', 'lgg')
</pre></div>

<p></div></div></p>

<h2>Clinical data</h2>

<div class="chunk" id="get-clinical-data"><div class="rcode"><div class="source"><pre class="knitr r">luad_clinical = processClinicalDataFile('luad-clinical-biotab-04032015.tab', 'luad-clinical-batches-04032015.tab')
luad_clinical = with(luad_clinical[match(luad_metadata$participant_id, luad_clinical$bcr_patient_uuid), ], 
                      data.frame(luad_metadata,
                                 gender = gender, 
                                 race = race,
                                 smoker = ifelse(tobacco_smoking_history_indicator == "Lifelong Non-smoker", 0, 1), # NA doesn't return true or false, just NA. this is useful.
                                 t_stage = ajcc_tumor_pathologic_pt, 
                                 m_stage = ajcc_metastasis_pathologic_pm, 
                                 n_stage = ajcc_nodes_pathologic_pn,
                                 stage = ajcc_pathologic_tumor_stage,
                                 days_survived = as.integer(days_survived), 
                                 vital_status = ifelse(vital_status == "Alive", 0, 1),
                                 age = age_at_initial_pathologic_diagnosis,
                                 batch = batch))
luad_metadata = luad_clinical
brca_clinical <- processClinicalDataFile('brca-clinical-biotab-04032015.tab', 'brca-clinical-batches-04032015.tab')
brca_clinical <- with(brca_clinical[match(brca_metadata$participant_id, brca_clinical$bcr_patient_uuid), ], 
                      data.frame(brca_metadata,
                                ethnicity = ethnicity,
                                race = race,
                                vital_status = ifelse(vital_status == "Alive", 0, 1),
                                days_survived = as.integer(days_survived),
                                age = age_at_diagnosis,
                                t_stage = ajcc_tumor_pathologic_pt, 
                                m_stage = ajcc_metastasis_pathologic_pm, 
                                n_stage = ajcc_nodes_pathologic_pn,
                                stage = ajcc_pathologic_tumor_stage,
                                er_status = er_status_by_ihc,
                                pr_status = pr_status_by_ihc,
                                her2_status = her2_status_by_ihc,
                                batch = batch))
brca_metadata = brca_clinical
prad_clinical <- processClinicalDataFile('prad-clinical-biotab-04012015.tab', 'prad-clinical-batches-04012015.tab')
prad_clinical <- with(prad_clinical[match(prad_metadata$participant_id, prad_clinical$bcr_patient_uuid), ],
                      data.frame(prad_metadata,
                                 days_survived = as.integer(days_survived),
                                 vital_status = ifelse(vital_status == "Alive", 0, 1),
                                 t_stage = pathologic_T,
                                 m_stage = pathologic_M,
                                 n_stage = pathologic_N,
                                 gleason_score = as.integer(gleason_score),
                                 age = abs(round(as.integer(birth_days_to)/365)),
                                 batch = batch)) # maybe change to age_at _initial_pathologic_diagnosis? 
prad_metadata = prad_clinical
</pre></div>

<p></div></div></p>

<div class="chunk" id="download-files"><div class="rcode"><div class="source"><pre class="knitr r">metadatas <- list(luad = luad_metadata, brca = brca_metadata, prad = prad_metadata, dlbc = dlbc_metadata, lihc = lihc_metadata, coad = coad_metadata, paad = paad_metadata, tgct = tgct_metadata, lgg = lgg_metadata)
currentwd <- getwd()
setwd('/inside/grotto/blin/trna-markers/datasets/raw')
for (cancer_code in names(metadatas)) { 
  metadata <- metadatas[[cancer_code]]
  for (analysis_id in metadata$analysis_id) {
    if (file.exists(analysis_id)) next # if exists, assume file was downloaded correctly and is not corrupted
    system(paste0('echo gtdownload -c $CGKEY -d ', analysis_id, cancer_code))
    system(paste0('gtdownload -c $CGKEY -d ', analysis_id))
  }
  file.symlink(paste0(getwd(), '/', metadata$analysis_id, '/', metadata$barcode, '_mirna.bam'), paste0('/inside/grotto/blin/trna-markers/datasets/', cancer_code, '/mirna/', metadata$shortnames, '-raw.bam'))
}
setwd(currentwd)
</pre></div>

<p></div></div></p>

<p>We are missing <code>01810b1a-84e4-43d5-8a1e-42b132a1126f</code>, which is <code>brca-0939-TP.bam</code>. It&#39;s been removed for now.</p>

<h2>Remap reads</h2>

<p>To count both tRFs processed from mature and immature tRNAs, we need to map to both hg19 and mature tRNAs.</p>

<h3>hg19</h3>

<p>Andrew has a few settings that I&#39;ve incorporated too, with his reasoning:</p>

<ul>
<li>--very-sensitive: necessary due to mismatch caused by modified base misreads </li>
<li>--np 5 --ignore-quals: quality scores are irrelevant, and N should be scored as only slightly better than a mismatch. ignoring quals forces the mismatches to have a single score, which is necessary for precise N weighting.</li>
</ul>

<div class="chunk" id="remap"><div class="rcode"><div class="source"><pre class="knitr bash">remap() {
  currentwd=`pwd`
  cd $1
  for prefix in `ls *-raw.bam | cut -f 1,2 -d -`; do
    if [ ! -e $prefix.bam ]
    then
      if [ ! -e $prefix.fastq ]
      then
        bam2fastq --force -o $prefix-raw# `readlink $prefix-raw.bam`
        mv ${prefix}-raw_M $prefix.fastq
        rm ${prefix}-raw_1 ${prefix}-raw_2
      fi
      bowtie2 -x ~/grotto/data/hg19-tRNAgenome -k 100 --very-sensitive --ignore-quals --np 5 -p 32 -U ${prefix}.fastq -S $prefix-temp-1.sam
      samtools view -S -F 4 -h -u $prefix-temp-1.sam | samtools view -F 512 -h  - > $prefix-temp-2.sam # filter out untemp and low quality reads
      bowtie2-best-mapped.py $prefix-temp-2.sam | samtools view -S -u - | samtools sort - $prefix # auto appends .bam
    fi
    rm -f $prefix.fastq
    rm -f $prefix-temp-1.sam
    rm -f $prefix-temp-2.sam
  done
  cd $currentwd
}
remap /inside/grotto/blin/trna-markers/datasets/luad/mirna
remap /inside/grotto/blin/trna-markers/datasets/brca/mirna
remap /inside/grotto/blin/trna-markers/datasets/prad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/dlbc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lihc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/coad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/paad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/tgct/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lgg/mirna
</pre></div>

<div class="output"><pre class="knitr bash">## 13869615 reads; of these:
##   13869615 (100.00%) were unpaired; of these:
##     2239115 (16.14%) aligned 0 times
##     7041442 (50.77%) aligned exactly 1 time
##     4589058 (33.09%) aligned &gt;1 times
## 83.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4589058/11630500
## 2 3507713/4589058
## 3 20116648/44228121
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0315-raw_1 and brca-0315-raw_2
## Single-end reads will be in brca-0315-raw_M
## 3003468 sequences in the BAM file
## 3003468 sequences exported
## 3003468 reads; of these:
##   3003468 (100.00%) were unpaired; of these:
##     606134 (20.18%) aligned 0 times
##     1608105 (53.54%) aligned exactly 1 time
##     789229 (26.28%) aligned &gt;1 times
## 79.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 789229/2397334
## 2 650261/789229
## 3 2610271/6280810
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0316-raw_1 and brca-0316-raw_2
## Single-end reads will be in brca-0316-raw_M
## 6299148 sequences in the BAM file
## 6299148 sequences exported
## 6299148 reads; of these:
##   6299148 (100.00%) were unpaired; of these:
##     1730120 (27.47%) aligned 0 times
##     2521776 (40.03%) aligned exactly 1 time
##     2047252 (32.50%) aligned &gt;1 times
## 72.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2047252/4569028
## 2 1506103/2047252
## 3 6609228/15538717
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0317-raw_1 and brca-0317-raw_2
## Single-end reads will be in brca-0317-raw_M
## 6288562 sequences in the BAM file
## 6288562 sequences exported
## 6288562 reads; of these:
##   6288562 (100.00%) were unpaired; of these:
##     552516 (8.79%) aligned 0 times
##     3165123 (50.33%) aligned exactly 1 time
##     2570923 (40.88%) aligned &gt;1 times
## 91.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2570923/5736046
## 2 2046210/2570923
## 3 13180644/25804839
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0318-raw_1 and brca-0318-raw_2
## Single-end reads will be in brca-0318-raw_M
## 5013636 sequences in the BAM file
## 5013636 sequences exported
## 5013636 reads; of these:
##   5013636 (100.00%) were unpaired; of these:
##     659362 (13.15%) aligned 0 times
##     2504121 (49.95%) aligned exactly 1 time
##     1850153 (36.90%) aligned &gt;1 times
## 86.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1850153/4354274
## 2 1565583/1850153
## 3 6558949/14013298
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0319-raw_1 and brca-0319-raw_2
## Single-end reads will be in brca-0319-raw_M
## 4330621 sequences in the BAM file
## 4330621 sequences exported
## 4330621 reads; of these:
##   4330621 (100.00%) were unpaired; of these:
##     1022978 (23.62%) aligned 0 times
##     1927057 (44.50%) aligned exactly 1 time
##     1380586 (31.88%) aligned &gt;1 times
## 76.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1380586/3307643
## 2 1204604/1380586
## 3 3025853/7595555
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0320-raw_1 and brca-0320-raw_2
## Single-end reads will be in brca-0320-raw_M
## 10827480 sequences in the BAM file
## 10827480 sequences exported
## 10827480 reads; of these:
##   10827480 (100.00%) were unpaired; of these:
##     2900516 (26.79%) aligned 0 times
##     5143996 (47.51%) aligned exactly 1 time
##     2782968 (25.70%) aligned &gt;1 times
## 73.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2782968/7926964
## 2 2220786/2782968
## 3 6914521/18713361
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0321-raw_1 and brca-0321-raw_2
## Single-end reads will be in brca-0321-raw_M
## 7971666 sequences in the BAM file
## 7971666 sequences exported
## 7971666 reads; of these:
##   7971666 (100.00%) were unpaired; of these:
##     1881401 (23.60%) aligned 0 times
##     4130439 (51.81%) aligned exactly 1 time
##     1959826 (24.58%) aligned &gt;1 times
## 76.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1959826/6090265
## 2 1572303/1959826
## 3 5167502/13702091
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0322-raw_1 and brca-0322-raw_2
## Single-end reads will be in brca-0322-raw_M
## 4157976 sequences in the BAM file
## 4157976 sequences exported
## 4157976 reads; of these:
##   4157976 (100.00%) were unpaired; of these:
##     986322 (23.72%) aligned 0 times
##     1800952 (43.31%) aligned exactly 1 time
##     1370702 (32.97%) aligned &gt;1 times
## 76.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1370702/3171654
## 2 1086157/1370702
## 3 4824579/10577665
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0323-raw_1 and brca-0323-raw_2
## Single-end reads will be in brca-0323-raw_M
## 9067861 sequences in the BAM file
## 9067861 sequences exported
## 9067861 reads; of these:
##   9067861 (100.00%) were unpaired; of these:
##     3145869 (34.69%) aligned 0 times
##     4227461 (46.62%) aligned exactly 1 time
##     1694531 (18.69%) aligned &gt;1 times
## 65.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1694531/5921992
## 2 1150409/1694531
## 3 5716161/15825839
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0324-raw_1 and brca-0324-raw_2
## Single-end reads will be in brca-0324-raw_M
## 6289448 sequences in the BAM file
## 6289448 sequences exported
## 6289448 reads; of these:
##   6289448 (100.00%) were unpaired; of these:
##     1565950 (24.90%) aligned 0 times
##     3243433 (51.57%) aligned exactly 1 time
##     1480065 (23.53%) aligned &gt;1 times
## 75.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1480065/4723498
## 2 1112238/1480065
## 3 4680499/12729378
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0325-raw_1 and brca-0325-raw_2
## Single-end reads will be in brca-0325-raw_M
## 2326469 sequences in the BAM file
## 2326469 sequences exported
## 2326469 reads; of these:
##   2326469 (100.00%) were unpaired; of these:
##     484733 (20.84%) aligned 0 times
##     1107958 (47.62%) aligned exactly 1 time
##     733778 (31.54%) aligned &gt;1 times
## 79.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 733778/1841736
## 2 525685/733778
## 3 4758617/9825082
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0326-raw_1 and brca-0326-raw_2
## Single-end reads will be in brca-0326-raw_M
## 2444164 sequences in the BAM file
## 2444164 sequences exported
## 2444164 reads; of these:
##   2444164 (100.00%) were unpaired; of these:
##     284438 (11.64%) aligned 0 times
##     1317795 (53.92%) aligned exactly 1 time
##     841931 (34.45%) aligned &gt;1 times
## 88.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 841931/2159726
## 2 641541/841931
## 3 4595816/9519437
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0327-raw_1 and brca-0327-raw_2
## Single-end reads will be in brca-0327-raw_M
## 7473585 sequences in the BAM file
## 7473585 sequences exported
## 7473585 reads; of these:
##   7473585 (100.00%) were unpaired; of these:
##     1787660 (23.92%) aligned 0 times
##     3665321 (49.04%) aligned exactly 1 time
##     2020604 (27.04%) aligned &gt;1 times
## 76.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2020604/5685925
## 2 1651600/2020604
## 3 8630439/20491916
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0328-raw_1 and brca-0328-raw_2
## Single-end reads will be in brca-0328-raw_M
## 8818748 sequences in the BAM file
## 8818748 sequences exported
## 8818748 reads; of these:
##   8818748 (100.00%) were unpaired; of these:
##     3909065 (44.33%) aligned 0 times
##     2468948 (28.00%) aligned exactly 1 time
##     2440735 (27.68%) aligned &gt;1 times
## 55.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2440735/4909683
## 2 2016763/2440735
## 3 9560771/18682692
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0329-raw_1 and brca-0329-raw_2
## Single-end reads will be in brca-0329-raw_M
## 3636062 sequences in the BAM file
## 3636062 sequences exported
## 3636062 reads; of these:
##   3636062 (100.00%) were unpaired; of these:
##     809283 (22.26%) aligned 0 times
##     1785233 (49.10%) aligned exactly 1 time
##     1041546 (28.64%) aligned &gt;1 times
## 77.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1041546/2826779
## 2 792108/1041546
## 3 4237407/9518390
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0330-raw_1 and brca-0330-raw_2
## Single-end reads will be in brca-0330-raw_M
## 10862372 sequences in the BAM file
## 10862372 sequences exported
## 10862372 reads; of these:
##   10862372 (100.00%) were unpaired; of these:
##     1893911 (17.44%) aligned 0 times
##     6214553 (57.21%) aligned exactly 1 time
##     2753908 (25.35%) aligned &gt;1 times
## 82.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2753908/8968461
## 2 1997575/2753908
## 3 10388241/25693598
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0331-raw_1 and brca-0331-raw_2
## Single-end reads will be in brca-0331-raw_M
## 8063168 sequences in the BAM file
## 8063168 sequences exported
## 8063168 reads; of these:
##   8063168 (100.00%) were unpaired; of these:
##     2394187 (29.69%) aligned 0 times
##     3394016 (42.09%) aligned exactly 1 time
##     2274965 (28.21%) aligned &gt;1 times
## 70.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2274965/5668981
## 2 1713413/2274965
## 3 20878578/36727942
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0332-raw_1 and brca-0332-raw_2
## Single-end reads will be in brca-0332-raw_M
## 8315675 sequences in the BAM file
## 8315675 sequences exported
## 8315675 reads; of these:
##   8315675 (100.00%) were unpaired; of these:
##     1253668 (15.08%) aligned 0 times
##     3652109 (43.92%) aligned exactly 1 time
##     3409898 (41.01%) aligned &gt;1 times
## 84.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3409898/7062007
## 2 2535425/3409898
## 3 17793984/38558112
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0333-raw_1 and brca-0333-raw_2
## Single-end reads will be in brca-0333-raw_M
## 9097357 sequences in the BAM file
## 9097357 sequences exported
## 9097357 reads; of these:
##   9097357 (100.00%) were unpaired; of these:
##     1857358 (20.42%) aligned 0 times
##     4437767 (48.78%) aligned exactly 1 time
##     2802232 (30.80%) aligned &gt;1 times
## 79.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2802232/7239999
## 2 2242203/2802232
## 3 8210486/19935397
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0334-raw_1 and brca-0334-raw_2
## Single-end reads will be in brca-0334-raw_M
## 8668691 sequences in the BAM file
## 8668691 sequences exported
## 8668691 reads; of these:
##   8668691 (100.00%) were unpaired; of these:
##     1518182 (17.51%) aligned 0 times
##     3966566 (45.76%) aligned exactly 1 time
##     3183943 (36.73%) aligned &gt;1 times
## 82.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3183943/7150509
## 2 2598316/3183943
## 3 6330404/16785579
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0335-raw_1 and brca-0335-raw_2
## Single-end reads will be in brca-0335-raw_M
## 3486659 sequences in the BAM file
## 3486659 sequences exported
## 3486659 reads; of these:
##   3486659 (100.00%) were unpaired; of these:
##     836847 (24.00%) aligned 0 times
##     1174359 (33.68%) aligned exactly 1 time
##     1475453 (42.32%) aligned &gt;1 times
## 76.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1475453/2649812
## 2 1164268/1475453
## 3 5401694/11144623
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0336-raw_1 and brca-0336-raw_2
## Single-end reads will be in brca-0336-raw_M
## 3776736 sequences in the BAM file
## 3776736 sequences exported
## 3776736 reads; of these:
##   3776736 (100.00%) were unpaired; of these:
##     1169723 (30.97%) aligned 0 times
##     1404843 (37.20%) aligned exactly 1 time
##     1202170 (31.83%) aligned &gt;1 times
## 69.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1202170/2607013
## 2 936166/1202170
## 3 3665138/8367242
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0337-raw_1 and brca-0337-raw_2
## Single-end reads will be in brca-0337-raw_M
## 13037426 sequences in the BAM file
## 13037426 sequences exported
## 13037426 reads; of these:
##   13037426 (100.00%) were unpaired; of these:
##     3575326 (27.42%) aligned 0 times
##     5433555 (41.68%) aligned exactly 1 time
##     4028545 (30.90%) aligned &gt;1 times
## 72.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4028545/9462100
## 2 2836506/4028545
## 3 8300495/22577818
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0338-raw_1 and brca-0338-raw_2
## Single-end reads will be in brca-0338-raw_M
## 2509202 sequences in the BAM file
## 2509202 sequences exported
## 2509202 reads; of these:
##   2509202 (100.00%) were unpaired; of these:
##     499365 (19.90%) aligned 0 times
##     1121011 (44.68%) aligned exactly 1 time
##     888826 (35.42%) aligned &gt;1 times
## 80.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 888826/2009837
## 2 732706/888826
## 3 2620287/6394721
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0339-raw_1 and brca-0339-raw_2
## Single-end reads will be in brca-0339-raw_M
## 5486086 sequences in the BAM file
## 5486086 sequences exported
## 5486086 reads; of these:
##   5486086 (100.00%) were unpaired; of these:
##     637135 (11.61%) aligned 0 times
##     2465514 (44.94%) aligned exactly 1 time
##     2383437 (43.45%) aligned &gt;1 times
## 88.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2383437/4848951
## 2 1923413/2383437
## 3 8459183/18216267
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0340-raw_1 and brca-0340-raw_2
## Single-end reads will be in brca-0340-raw_M
## 5657955 sequences in the BAM file
## 5657955 sequences exported
## 5657955 reads; of these:
##   5657955 (100.00%) were unpaired; of these:
##     1216773 (21.51%) aligned 0 times
##     3104360 (54.87%) aligned exactly 1 time
##     1336822 (23.63%) aligned &gt;1 times
## 78.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1336822/4441182
## 2 1012670/1336822
## 3 4972905/12477074
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0341-raw_1 and brca-0341-raw_2
## Single-end reads will be in brca-0341-raw_M
## 6835529 sequences in the BAM file
## 6835529 sequences exported
## 6835529 reads; of these:
##   6835529 (100.00%) were unpaired; of these:
##     589324 (8.62%) aligned 0 times
##     4243514 (62.08%) aligned exactly 1 time
##     2002691 (29.30%) aligned &gt;1 times
## 91.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2002691/6246205
## 2 1617435/2002691
## 3 6583013/16344769
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0342-raw_1 and brca-0342-raw_2
## Single-end reads will be in brca-0342-raw_M
## 6281611 sequences in the BAM file
## 6281611 sequences exported
## 6281611 reads; of these:
##   6281611 (100.00%) were unpaired; of these:
##     1551363 (24.70%) aligned 0 times
##     2734033 (43.52%) aligned exactly 1 time
##     1996215 (31.78%) aligned &gt;1 times
## 75.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1996215/4730248
## 2 1349790/1996215
## 3 4939820/13524081
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0343-raw_1 and brca-0343-raw_2
## Single-end reads will be in brca-0343-raw_M
## 2915946 sequences in the BAM file
## 2915946 sequences exported
## 2915946 reads; of these:
##   2915946 (100.00%) were unpaired; of these:
##     240803 (8.26%) aligned 0 times
##     1640316 (56.25%) aligned exactly 1 time
##     1034827 (35.49%) aligned &gt;1 times
## 91.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1034827/2675143
## 2 859744/1034827
## 3 5166755/10958039
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0344-raw_1 and brca-0344-raw_2
## Single-end reads will be in brca-0344-raw_M
## 11429153 sequences in the BAM file
## 11429153 sequences exported
## 11429153 reads; of these:
##   11429153 (100.00%) were unpaired; of these:
##     2285435 (20.00%) aligned 0 times
##     4822367 (42.19%) aligned exactly 1 time
##     4321351 (37.81%) aligned &gt;1 times
## 80.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4321351/9143718
## 2 3496398/4321351
## 3 7836802/20740808
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0345-raw_1 and brca-0345-raw_2
## Single-end reads will be in brca-0345-raw_M
## 4398216 sequences in the BAM file
## 4398216 sequences exported
## 4398216 reads; of these:
##   4398216 (100.00%) were unpaired; of these:
##     343788 (7.82%) aligned 0 times
##     2539985 (57.75%) aligned exactly 1 time
##     1514443 (34.43%) aligned &gt;1 times
## 92.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1514443/4054428
## 2 1172358/1514443
## 3 9995283/20114977
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0346-raw_1 and brca-0346-raw_2
## Single-end reads will be in brca-0346-raw_M
## 8455479 sequences in the BAM file
## 8455479 sequences exported
## 8455479 reads; of these:
##   8455479 (100.00%) were unpaired; of these:
##     1654491 (19.57%) aligned 0 times
##     4146061 (49.03%) aligned exactly 1 time
##     2654927 (31.40%) aligned &gt;1 times
## 80.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2654927/6800988
## 2 2001842/2654927
## 3 14928395/33388098
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0347-raw_1 and brca-0347-raw_2
## Single-end reads will be in brca-0347-raw_M
## 4151410 sequences in the BAM file
## 4151410 sequences exported
## 4151410 reads; of these:
##   4151410 (100.00%) were unpaired; of these:
##     621313 (14.97%) aligned 0 times
##     2148475 (51.75%) aligned exactly 1 time
##     1381622 (33.28%) aligned &gt;1 times
## 85.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1381622/3530097
## 2 1093161/1381622
## 3 7002230/14822459
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0348-raw_1 and brca-0348-raw_2
## Single-end reads will be in brca-0348-raw_M
## 12402285 sequences in the BAM file
## 12402285 sequences exported
## 12402285 reads; of these:
##   12402285 (100.00%) were unpaired; of these:
##     2926444 (23.60%) aligned 0 times
##     6988674 (56.35%) aligned exactly 1 time
##     2487167 (20.05%) aligned &gt;1 times
## 76.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2487167/9475841
## 2 1897733/2487167
## 3 20084151/43266821
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in brca-0349-raw_1 and brca-0349-raw_2
## Single-end reads will be in brca-0349-raw_M
## 5131842 sequences in the BAM file
## 5131842 sequences exported
## 5131842 reads; of these:
##   5131842 (100.00%) were unpaired; of these:
##     567779 (11.06%) aligned 0 times
##     2925775 (57.01%) aligned exactly 1 time
##     1638288 (31.92%) aligned &gt;1 times
## 88.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1638288/4564063
## 2 1309498/1638288
## 3 8366935/18035023
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0350-raw_1 and brca-0350-raw_2
## Single-end reads will be in brca-0350-raw_M
## 8601409 sequences in the BAM file
## 8601409 sequences exported
## 8601409 reads; of these:
##   8601409 (100.00%) were unpaired; of these:
##     1902975 (22.12%) aligned 0 times
##     3850981 (44.77%) aligned exactly 1 time
##     2847453 (33.10%) aligned &gt;1 times
## 77.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2847453/6698434
## 2 2278195/2847453
## 3 7631511/17961389
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0351-raw_1 and brca-0351-raw_2
## Single-end reads will be in brca-0351-raw_M
## 4916229 sequences in the BAM file
## 4916229 sequences exported
## 4916229 reads; of these:
##   4916229 (100.00%) were unpaired; of these:
##     1091138 (22.19%) aligned 0 times
##     2181621 (44.38%) aligned exactly 1 time
##     1643470 (33.43%) aligned &gt;1 times
## 77.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1643470/3825091
## 2 1432714/1643470
## 3 3554537/8605559
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0352-raw_1 and brca-0352-raw_2
## Single-end reads will be in brca-0352-raw_M
## 15184547 sequences in the BAM file
## 15184547 sequences exported
## 15184547 reads; of these:
##   15184547 (100.00%) were unpaired; of these:
##     3434614 (22.62%) aligned 0 times
##     6622258 (43.61%) aligned exactly 1 time
##     5127675 (33.77%) aligned &gt;1 times
## 77.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5127675/11749933
## 2 4406465/5127675
## 3 13201044/31629677
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0353-raw_1 and brca-0353-raw_2
## Single-end reads will be in brca-0353-raw_M
## 3426634 sequences in the BAM file
## 3426634 sequences exported
## 3426634 reads; of these:
##   3426634 (100.00%) were unpaired; of these:
##     515242 (15.04%) aligned 0 times
##     1497295 (43.70%) aligned exactly 1 time
##     1414097 (41.27%) aligned &gt;1 times
## 84.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1414097/2911392
## 2 1164348/1414097
## 3 6199752/12523426
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0354-raw_1 and brca-0354-raw_2
## Single-end reads will be in brca-0354-raw_M
## 10637698 sequences in the BAM file
## 10637698 sequences exported
## 10637698 reads; of these:
##   10637698 (100.00%) were unpaired; of these:
##     2831169 (26.61%) aligned 0 times
##     5172569 (48.62%) aligned exactly 1 time
##     2633960 (24.76%) aligned &gt;1 times
## 73.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2633960/7806529
## 2 2017508/2633960
## 3 8330947/21007903
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0355-raw_1 and brca-0355-raw_2
## Single-end reads will be in brca-0355-raw_M
## 8551569 sequences in the BAM file
## 8551569 sequences exported
## 8551569 reads; of these:
##   8551569 (100.00%) were unpaired; of these:
##     1856024 (21.70%) aligned 0 times
##     3699979 (43.27%) aligned exactly 1 time
##     2995566 (35.03%) aligned &gt;1 times
## 78.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2995566/6695545
## 2 2524880/2995566
## 3 7141806/16956100
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0356-raw_1 and brca-0356-raw_2
## Single-end reads will be in brca-0356-raw_M
## 3715095 sequences in the BAM file
## 3715095 sequences exported
## 3715095 reads; of these:
##   3715095 (100.00%) were unpaired; of these:
##     357041 (9.61%) aligned 0 times
##     2123402 (57.16%) aligned exactly 1 time
##     1234652 (33.23%) aligned &gt;1 times
## 90.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1234652/3358054
## 2 984709/1234652
## 3 4499671/9903932
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0357-raw_1 and brca-0357-raw_2
## Single-end reads will be in brca-0357-raw_M
## 3649693 sequences in the BAM file
## 3649693 sequences exported
## 3649693 reads; of these:
##   3649693 (100.00%) were unpaired; of these:
##     431403 (11.82%) aligned 0 times
##     1791266 (49.08%) aligned exactly 1 time
##     1427024 (39.10%) aligned &gt;1 times
## 88.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1427024/3218290
## 2 1183414/1427024
## 3 5406000/11013161
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0358-raw_1 and brca-0358-raw_2
## Single-end reads will be in brca-0358-raw_M
## 6201210 sequences in the BAM file
## 6201210 sequences exported
## 6201210 reads; of these:
##   6201210 (100.00%) were unpaired; of these:
##     1787098 (28.82%) aligned 0 times
##     2694390 (43.45%) aligned exactly 1 time
##     1719722 (27.73%) aligned &gt;1 times
## 71.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1719722/4414112
## 2 1094853/1719722
## 3 10217274/21327051
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0359-raw_1 and brca-0359-raw_2
## Single-end reads will be in brca-0359-raw_M
## 2094285 sequences in the BAM file
## 2094285 sequences exported
## 2094285 reads; of these:
##   2094285 (100.00%) were unpaired; of these:
##     641100 (30.61%) aligned 0 times
##     918270 (43.85%) aligned exactly 1 time
##     534915 (25.54%) aligned &gt;1 times
## 69.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 534915/1453185
## 2 402264/534915
## 3 3729991/7733253
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0360-raw_1 and brca-0360-raw_2
## Single-end reads will be in brca-0360-raw_M
## 5641358 sequences in the BAM file
## 5641358 sequences exported
## 5641358 reads; of these:
##   5641358 (100.00%) were unpaired; of these:
##     977430 (17.33%) aligned 0 times
##     2843781 (50.41%) aligned exactly 1 time
##     1820147 (32.26%) aligned &gt;1 times
## 82.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1820147/4663928
## 2 1464346/1820147
## 3 8186168/17765432
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0361-raw_1 and brca-0361-raw_2
## Single-end reads will be in brca-0361-raw_M
## 3473775 sequences in the BAM file
## 3473775 sequences exported
## 3473775 reads; of these:
##   3473775 (100.00%) were unpaired; of these:
##     859095 (24.73%) aligned 0 times
##     1678421 (48.32%) aligned exactly 1 time
##     936259 (26.95%) aligned &gt;1 times
## 75.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 936259/2614680
## 2 811125/936259
## 3 1977373/5620598
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0362-raw_1 and brca-0362-raw_2
## Single-end reads will be in brca-0362-raw_M
## 5185346 sequences in the BAM file
## 5185346 sequences exported
## 5185346 reads; of these:
##   5185346 (100.00%) were unpaired; of these:
##     650004 (12.54%) aligned 0 times
##     2298929 (44.34%) aligned exactly 1 time
##     2236413 (43.13%) aligned &gt;1 times
## 87.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2236413/4535342
## 2 1835770/2236413
## 3 8882752/18311396
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0363-raw_1 and brca-0363-raw_2
## Single-end reads will be in brca-0363-raw_M
## 4352907 sequences in the BAM file
## 4352907 sequences exported
## 4352907 reads; of these:
##   4352907 (100.00%) were unpaired; of these:
##     1073344 (24.66%) aligned 0 times
##     2300303 (52.85%) aligned exactly 1 time
##     979260 (22.50%) aligned &gt;1 times
## 75.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 979260/3279563
## 2 766953/979260
## 3 2524908/7412039
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0364-raw_1 and brca-0364-raw_2
## Single-end reads will be in brca-0364-raw_M
## 13600011 sequences in the BAM file
## 13600011 sequences exported
## 13600011 reads; of these:
##   13600011 (100.00%) were unpaired; of these:
##     2159486 (15.88%) aligned 0 times
##     8414199 (61.87%) aligned exactly 1 time
##     3026326 (22.25%) aligned &gt;1 times
## 84.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3026326/11440525
## 2 2349867/3026326
## 3 8505300/25289706
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0365-raw_1 and brca-0365-raw_2
## Single-end reads will be in brca-0365-raw_M
## 3597059 sequences in the BAM file
## 3597059 sequences exported
## 3597059 reads; of these:
##   3597059 (100.00%) were unpaired; of these:
##     849154 (23.61%) aligned 0 times
##     1621460 (45.08%) aligned exactly 1 time
##     1126445 (31.32%) aligned &gt;1 times
## 76.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1126445/2747905
## 2 844639/1126445
## 3 4950332/11369427
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0366-raw_1 and brca-0366-raw_2
## Single-end reads will be in brca-0366-raw_M
## 5001517 sequences in the BAM file
## 5001517 sequences exported
## 5001517 reads; of these:
##   5001517 (100.00%) were unpaired; of these:
##     1212877 (24.25%) aligned 0 times
##     2140429 (42.80%) aligned exactly 1 time
##     1648211 (32.95%) aligned &gt;1 times
## 75.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1648211/3788640
## 2 1346716/1648211
## 3 5855929/12876764
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0367-raw_1 and brca-0367-raw_2
## Single-end reads will be in brca-0367-raw_M
## 7905821 sequences in the BAM file
## 7905821 sequences exported
## 7905821 reads; of these:
##   7905821 (100.00%) were unpaired; of these:
##     1887583 (23.88%) aligned 0 times
##     4242385 (53.66%) aligned exactly 1 time
##     1775853 (22.46%) aligned &gt;1 times
## 76.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1775853/6018238
## 2 1346733/1775853
## 3 8565329/19967355
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0368-raw_1 and brca-0368-raw_2
## Single-end reads will be in brca-0368-raw_M
## 4972470 sequences in the BAM file
## 4972470 sequences exported
## 4972470 reads; of these:
##   4972470 (100.00%) were unpaired; of these:
##     575509 (11.57%) aligned 0 times
##     2104487 (42.32%) aligned exactly 1 time
##     2292474 (46.10%) aligned &gt;1 times
## 88.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2292474/4396961
## 2 2011432/2292474
## 3 5984462/12884208
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0369-raw_1 and brca-0369-raw_2
## Single-end reads will be in brca-0369-raw_M
## 9488378 sequences in the BAM file
## 9488378 sequences exported
## 9488378 reads; of these:
##   9488378 (100.00%) were unpaired; of these:
##     3236888 (34.11%) aligned 0 times
##     3526655 (37.17%) aligned exactly 1 time
##     2724835 (28.72%) aligned &gt;1 times
## 65.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2724835/6251490
## 2 2116889/2724835
## 3 10431345/21147571
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0370-raw_1 and brca-0370-raw_2
## Single-end reads will be in brca-0370-raw_M
## 5232487 sequences in the BAM file
## 5232487 sequences exported
## 5232487 reads; of these:
##   5232487 (100.00%) were unpaired; of these:
##     1281319 (24.49%) aligned 0 times
##     2213302 (42.30%) aligned exactly 1 time
##     1737866 (33.21%) aligned &gt;1 times
## 75.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1737866/3951168
## 2 1461574/1737866
## 3 5678780/12659154
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0371-raw_1 and brca-0371-raw_2
## Single-end reads will be in brca-0371-raw_M
## 5230501 sequences in the BAM file
## 5230501 sequences exported
## 5230501 reads; of these:
##   5230501 (100.00%) were unpaired; of these:
##     720101 (13.77%) aligned 0 times
##     2770637 (52.97%) aligned exactly 1 time
##     1739763 (33.26%) aligned &gt;1 times
## 86.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1739763/4510400
## 2 1317526/1739763
## 3 9376118/18520303
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0372-raw_1 and brca-0372-raw_2
## Single-end reads will be in brca-0372-raw_M
## 3567402 sequences in the BAM file
## 3567402 sequences exported
## 3567402 reads; of these:
##   3567402 (100.00%) were unpaired; of these:
##     879439 (24.65%) aligned 0 times
##     1451099 (40.68%) aligned exactly 1 time
##     1236864 (34.67%) aligned &gt;1 times
## 75.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1236864/2687963
## 2 960711/1236864
## 3 6875279/13581384
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0373-raw_1 and brca-0373-raw_2
## Single-end reads will be in brca-0373-raw_M
## 8501930 sequences in the BAM file
## 8501930 sequences exported
## 8501930 reads; of these:
##   8501930 (100.00%) were unpaired; of these:
##     486356 (5.72%) aligned 0 times
##     4918085 (57.85%) aligned exactly 1 time
##     3097489 (36.43%) aligned &gt;1 times
## 94.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3097489/8015574
## 2 2600910/3097489
## 3 10212136/22711727
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0374-raw_1 and brca-0374-raw_2
## Single-end reads will be in brca-0374-raw_M
## 10426808 sequences in the BAM file
## 10426808 sequences exported
## 10426808 reads; of these:
##   10426808 (100.00%) were unpaired; of these:
##     2697331 (25.87%) aligned 0 times
##     5308443 (50.91%) aligned exactly 1 time
##     2421034 (23.22%) aligned &gt;1 times
## 74.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2421034/7729477
## 2 1516191/2421034
## 3 13232239/30734842
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0375-raw_1 and brca-0375-raw_2
## Single-end reads will be in brca-0375-raw_M
## 11163973 sequences in the BAM file
## 11163973 sequences exported
## 11163973 reads; of these:
##   11163973 (100.00%) were unpaired; of these:
##     3875514 (34.71%) aligned 0 times
##     4651421 (41.66%) aligned exactly 1 time
##     2637038 (23.62%) aligned &gt;1 times
## 65.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2637038/7288459
## 2 2035544/2637038
## 3 6382840/17644009
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0376-raw_1 and brca-0376-raw_2
## Single-end reads will be in brca-0376-raw_M
## 13181064 sequences in the BAM file
## 13181064 sequences exported
## 13181064 reads; of these:
##   13181064 (100.00%) were unpaired; of these:
##     807543 (6.13%) aligned 0 times
##     6042779 (45.84%) aligned exactly 1 time
##     6330742 (48.03%) aligned &gt;1 times
## 93.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6330742/12373521
## 2 5657140/6330742
## 3 16323628/34127620
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0377-raw_1 and brca-0377-raw_2
## Single-end reads will be in brca-0377-raw_M
## 25233779 sequences in the BAM file
## 25233779 sequences exported
## 25233779 reads; of these:
##   25233779 (100.00%) were unpaired; of these:
##     3795232 (15.04%) aligned 0 times
##     11040414 (43.75%) aligned exactly 1 time
##     10398133 (41.21%) aligned &gt;1 times
## 84.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 10398133/21438547
## 2 8832342/10398133
## 3 28647206/64144931
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 4.
## Output will be in brca-0378-raw_1 and brca-0378-raw_2
## Single-end reads will be in brca-0378-raw_M
## 3565216 sequences in the BAM file
## 3565216 sequences exported
## 3565216 reads; of these:
##   3565216 (100.00%) were unpaired; of these:
##     484245 (13.58%) aligned 0 times
##     1774327 (49.77%) aligned exactly 1 time
##     1306644 (36.65%) aligned &gt;1 times
## 86.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1306644/3080971
## 2 1098527/1306644
## 3 5556297/11923468
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0379-raw_1 and brca-0379-raw_2
## Single-end reads will be in brca-0379-raw_M
## 1360664 sequences in the BAM file
## 1360664 sequences exported
## 1360664 reads; of these:
##   1360664 (100.00%) were unpaired; of these:
##     331327 (24.35%) aligned 0 times
##     556513 (40.90%) aligned exactly 1 time
##     472824 (34.75%) aligned &gt;1 times
## 75.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 472824/1029337
## 2 377700/472824
## 3 2548620/4775529
## This looks like paired data from lane 6.
## Output will be in brca-0380-raw_1 and brca-0380-raw_2
## Single-end reads will be in brca-0380-raw_M
## 3379568 sequences in the BAM file
## 3379568 sequences exported
## 3379568 reads; of these:
##   3379568 (100.00%) were unpaired; of these:
##     806918 (23.88%) aligned 0 times
##     1150474 (34.04%) aligned exactly 1 time
##     1422176 (42.08%) aligned &gt;1 times
## 76.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1422176/2572650
## 2 1223214/1422176
## 3 3193461/7220810
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0381-raw_1 and brca-0381-raw_2
## Single-end reads will be in brca-0381-raw_M
## 4926947 sequences in the BAM file
## 4926947 sequences exported
## 4926947 reads; of these:
##   4926947 (100.00%) were unpaired; of these:
##     977908 (19.85%) aligned 0 times
##     1773922 (36.00%) aligned exactly 1 time
##     2175117 (44.15%) aligned &gt;1 times
## 80.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2175117/3949039
## 2 2017669/2175117
## 3 4051551/8985442
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0382-raw_1 and brca-0382-raw_2
## Single-end reads will be in brca-0382-raw_M
## 3647701 sequences in the BAM file
## 3647701 sequences exported
## 3647701 reads; of these:
##   3647701 (100.00%) were unpaired; of these:
##     317952 (8.72%) aligned 0 times
##     2276773 (62.42%) aligned exactly 1 time
##     1052976 (28.87%) aligned &gt;1 times
## 91.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1052976/3329749
## 2 820070/1052976
## 3 4674995/10414898
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0383-raw_1 and brca-0383-raw_2
## Single-end reads will be in brca-0383-raw_M
## 2875154 sequences in the BAM file
## 2875154 sequences exported
## 2875154 reads; of these:
##   2875154 (100.00%) were unpaired; of these:
##     787844 (27.40%) aligned 0 times
##     1335927 (46.46%) aligned exactly 1 time
##     751383 (26.13%) aligned &gt;1 times
## 72.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 751383/2087310
## 2 644905/751383
## 3 1670171/4552621
## This looks like paired data from lane 3.
## Output will be in brca-0384-raw_1 and brca-0384-raw_2
## Single-end reads will be in brca-0384-raw_M
## 2975508 sequences in the BAM file
## 2975508 sequences exported
## 2975508 reads; of these:
##   2975508 (100.00%) were unpaired; of these:
##     638703 (21.47%) aligned 0 times
##     1394095 (46.85%) aligned exactly 1 time
##     942710 (31.68%) aligned &gt;1 times
## 78.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 942710/2336805
## 2 705400/942710
## 3 5269618/10293591
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0385-raw_1 and brca-0385-raw_2
## Single-end reads will be in brca-0385-raw_M
## 4537991 sequences in the BAM file
## 4537991 sequences exported
## 4537991 reads; of these:
##   4537991 (100.00%) were unpaired; of these:
##     361010 (7.96%) aligned 0 times
##     2138640 (47.13%) aligned exactly 1 time
##     2038341 (44.92%) aligned &gt;1 times
## 92.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2038341/4176981
## 2 1285889/2038341
## 3 11632136/23444195
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0386-raw_1 and brca-0386-raw_2
## Single-end reads will be in brca-0386-raw_M
## 7484598 sequences in the BAM file
## 7484598 sequences exported
## 7484598 reads; of these:
##   7484598 (100.00%) were unpaired; of these:
##     1429173 (19.09%) aligned 0 times
##     3033518 (40.53%) aligned exactly 1 time
##     3021907 (40.38%) aligned &gt;1 times
## 80.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3021907/6055425
## 2 1668554/3021907
## 3 14386987/31011777
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0387-raw_1 and brca-0387-raw_2
## Single-end reads will be in brca-0387-raw_M
## 1739894 sequences in the BAM file
## 1739894 sequences exported
## 1739894 reads; of these:
##   1739894 (100.00%) were unpaired; of these:
##     374937 (21.55%) aligned 0 times
##     890994 (51.21%) aligned exactly 1 time
##     473963 (27.24%) aligned &gt;1 times
## 78.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 473963/1364957
## 2 366260/473963
## 3 1664660/4222955
## This looks like paired data from lane 2.
## Output will be in brca-0388-raw_1 and brca-0388-raw_2
## Single-end reads will be in brca-0388-raw_M
## 2365406 sequences in the BAM file
## 2365406 sequences exported
## 2365406 reads; of these:
##   2365406 (100.00%) were unpaired; of these:
##     256197 (10.83%) aligned 0 times
##     1278093 (54.03%) aligned exactly 1 time
##     831116 (35.14%) aligned &gt;1 times
## 89.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 831116/2109209
## 2 632241/831116
## 3 6304342/13527793
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0389-raw_1 and brca-0389-raw_2
## Single-end reads will be in brca-0389-raw_M
## 3210687 sequences in the BAM file
## 3210687 sequences exported
## 3210687 reads; of these:
##   3210687 (100.00%) were unpaired; of these:
##     862008 (26.85%) aligned 0 times
##     1215600 (37.86%) aligned exactly 1 time
##     1133079 (35.29%) aligned &gt;1 times
## 73.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1133079/2348679
## 2 903894/1133079
## 3 7605992/15641274
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0390-raw_1 and brca-0390-raw_2
## Single-end reads will be in brca-0390-raw_M
## 2833928 sequences in the BAM file
## 2833928 sequences exported
## 2833928 reads; of these:
##   2833928 (100.00%) were unpaired; of these:
##     785377 (27.71%) aligned 0 times
##     1035565 (36.54%) aligned exactly 1 time
##     1012986 (35.74%) aligned &gt;1 times
## 72.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1012986/2048551
## 2 769814/1012986
## 3 3382232/7600843
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0391-raw_1 and brca-0391-raw_2
## Single-end reads will be in brca-0391-raw_M
## 3719700 sequences in the BAM file
## 3719700 sequences exported
## 3719700 reads; of these:
##   3719700 (100.00%) were unpaired; of these:
##     792092 (21.29%) aligned 0 times
##     1784382 (47.97%) aligned exactly 1 time
##     1143226 (30.73%) aligned &gt;1 times
## 78.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1143226/2927608
## 2 932446/1143226
## 3 2954522/7518380
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0392-raw_1 and brca-0392-raw_2
## Single-end reads will be in brca-0392-raw_M
## 13733662 sequences in the BAM file
## 13733662 sequences exported
## 13733662 reads; of these:
##   13733662 (100.00%) were unpaired; of these:
##     2980511 (21.70%) aligned 0 times
##     6294353 (45.83%) aligned exactly 1 time
##     4458798 (32.47%) aligned &gt;1 times
## 78.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4458798/10753151
## 2 3656716/4458798
## 3 15656012/35840586
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-0393-raw_1 and brca-0393-raw_2
## Single-end reads will be in brca-0393-raw_M
## 11833849 sequences in the BAM file
## 11833849 sequences exported
## 11833849 reads; of these:
##   11833849 (100.00%) were unpaired; of these:
##     3770958 (31.87%) aligned 0 times
##     5267775 (44.51%) aligned exactly 1 time
##     2795116 (23.62%) aligned &gt;1 times
## 68.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2795116/8062891
## 2 1900720/2795116
## 3 8999781/22077649
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0394-raw_1 and brca-0394-raw_2
## Single-end reads will be in brca-0394-raw_M
## 3597903 sequences in the BAM file
## 3597903 sequences exported
## 3597903 reads; of these:
##   3597903 (100.00%) were unpaired; of these:
##     718456 (19.97%) aligned 0 times
##     1736848 (48.27%) aligned exactly 1 time
##     1142599 (31.76%) aligned &gt;1 times
## 80.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1142599/2879447
## 2 977481/1142599
## 3 4525803/10320897
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0395-raw_1 and brca-0395-raw_2
## Single-end reads will be in brca-0395-raw_M
## 1732290 sequences in the BAM file
## 1732290 sequences exported
## 1732290 reads; of these:
##   1732290 (100.00%) were unpaired; of these:
##     162228 (9.36%) aligned 0 times
##     773644 (44.66%) aligned exactly 1 time
##     796418 (45.97%) aligned &gt;1 times
## 90.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 796418/1570062
## 2 679322/796418
## 3 2626599/5435105
## This looks like paired data from lane 1.
## Output will be in brca-0396-raw_1 and brca-0396-raw_2
## Single-end reads will be in brca-0396-raw_M
## 3394729 sequences in the BAM file
## 3394729 sequences exported
## 3394729 reads; of these:
##   3394729 (100.00%) were unpaired; of these:
##     754530 (22.23%) aligned 0 times
##     1660517 (48.91%) aligned exactly 1 time
##     979682 (28.86%) aligned &gt;1 times
## 77.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 979682/2640199
## 2 770199/979682
## 3 3413709/7803794
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0397-raw_1 and brca-0397-raw_2
## Single-end reads will be in brca-0397-raw_M
## 5922423 sequences in the BAM file
## 5922423 sequences exported
## 5922423 reads; of these:
##   5922423 (100.00%) were unpaired; of these:
##     589121 (9.95%) aligned 0 times
##     2484703 (41.95%) aligned exactly 1 time
##     2848599 (48.10%) aligned &gt;1 times
## 90.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2848599/5333302
## 2 2210721/2848599
## 3 24563945/44177519
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0398-raw_1 and brca-0398-raw_2
## Single-end reads will be in brca-0398-raw_M
## 2136449 sequences in the BAM file
## 2136449 sequences exported
## 2136449 reads; of these:
##   2136449 (100.00%) were unpaired; of these:
##     232205 (10.87%) aligned 0 times
##     1134786 (53.12%) aligned exactly 1 time
##     769458 (36.02%) aligned &gt;1 times
## 89.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 769458/1904244
## 2 571896/769458
## 3 4665303/9367272
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0399-raw_1 and brca-0399-raw_2
## Single-end reads will be in brca-0399-raw_M
## 3866651 sequences in the BAM file
## 3866651 sequences exported
## 3866651 reads; of these:
##   3866651 (100.00%) were unpaired; of these:
##     922129 (23.85%) aligned 0 times
##     1996190 (51.63%) aligned exactly 1 time
##     948332 (24.53%) aligned &gt;1 times
## 76.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 948332/2944522
## 2 730789/948332
## 3 5032636/10804269
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0400-raw_1 and brca-0400-raw_2
## Single-end reads will be in brca-0400-raw_M
## 3894532 sequences in the BAM file
## 3894532 sequences exported
## 3894532 reads; of these:
##   3894532 (100.00%) were unpaired; of these:
##     338660 (8.70%) aligned 0 times
##     2503098 (64.27%) aligned exactly 1 time
##     1052774 (27.03%) aligned &gt;1 times
## 91.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1052774/3555872
## 2 826739/1052774
## 3 4331246/10172369
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0401-raw_1 and brca-0401-raw_2
## Single-end reads will be in brca-0401-raw_M
## 9317992 sequences in the BAM file
## 9317992 sequences exported
## 9317992 reads; of these:
##   9317992 (100.00%) were unpaired; of these:
##     2705569 (29.04%) aligned 0 times
##     3577833 (38.40%) aligned exactly 1 time
##     3034590 (32.57%) aligned &gt;1 times
## 70.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3034590/6612423
## 2 2572410/3034590
## 3 6681562/16210444
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0402-raw_1 and brca-0402-raw_2
## Single-end reads will be in brca-0402-raw_M
## 3228258 sequences in the BAM file
## 3228258 sequences exported
## 3228258 reads; of these:
##   3228258 (100.00%) were unpaired; of these:
##     830634 (25.73%) aligned 0 times
##     1301386 (40.31%) aligned exactly 1 time
##     1096238 (33.96%) aligned &gt;1 times
## 74.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1096238/2397624
## 2 956032/1096238
## 3 2171111/5536832
## This looks like paired data from lane 7.
## Output will be in brca-0403-raw_1 and brca-0403-raw_2
## Single-end reads will be in brca-0403-raw_M
## 3371714 sequences in the BAM file
## 3371714 sequences exported
## 3371714 reads; of these:
##   3371714 (100.00%) were unpaired; of these:
##     1021680 (30.30%) aligned 0 times
##     1474803 (43.74%) aligned exactly 1 time
##     875231 (25.96%) aligned &gt;1 times
## 69.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 875231/2350034
## 2 624334/875231
## 3 5812109/11699686
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0404-raw_1 and brca-0404-raw_2
## Single-end reads will be in brca-0404-raw_M
## 3494090 sequences in the BAM file
## 3494090 sequences exported
## 3494090 reads; of these:
##   3494090 (100.00%) were unpaired; of these:
##     886165 (25.36%) aligned 0 times
##     1558802 (44.61%) aligned exactly 1 time
##     1049123 (30.03%) aligned &gt;1 times
## 74.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1049123/2607925
## 2 747800/1049123
## 3 6759803/12963450
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0405-raw_1 and brca-0405-raw_2
## Single-end reads will be in brca-0405-raw_M
## 5131076 sequences in the BAM file
## 5131076 sequences exported
## 5131076 reads; of these:
##   5131076 (100.00%) were unpaired; of these:
##     1357310 (26.45%) aligned 0 times
##     1907803 (37.18%) aligned exactly 1 time
##     1865963 (36.37%) aligned &gt;1 times
## 73.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1865963/3773766
## 2 1634994/1865963
## 3 8616864/16778637
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0406-raw_1 and brca-0406-raw_2
## Single-end reads will be in brca-0406-raw_M
## 9020881 sequences in the BAM file
## 9020881 sequences exported
## 9020881 reads; of these:
##   9020881 (100.00%) were unpaired; of these:
##     1539124 (17.06%) aligned 0 times
##     4782475 (53.02%) aligned exactly 1 time
##     2699282 (29.92%) aligned &gt;1 times
## 82.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2699282/7481757
## 2 2003327/2699282
## 3 8704904/22640935
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0407-raw_1 and brca-0407-raw_2
## Single-end reads will be in brca-0407-raw_M
## 9140317 sequences in the BAM file
## 9140317 sequences exported
## 9140317 reads; of these:
##   9140317 (100.00%) were unpaired; of these:
##     1980535 (21.67%) aligned 0 times
##     4685260 (51.26%) aligned exactly 1 time
##     2474522 (27.07%) aligned &gt;1 times
## 78.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2474522/7159782
## 2 1748781/2474522
## 3 13387774/30093958
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0408-raw_1 and brca-0408-raw_2
## Single-end reads will be in brca-0408-raw_M
## 6664068 sequences in the BAM file
## 6664068 sequences exported
## 6664068 reads; of these:
##   6664068 (100.00%) were unpaired; of these:
##     492657 (7.39%) aligned 0 times
##     3242872 (48.66%) aligned exactly 1 time
##     2928539 (43.95%) aligned &gt;1 times
## 92.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2928539/6171411
## 2 2608947/2928539
## 3 6964841/15799959
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0409-raw_1 and brca-0409-raw_2
## Single-end reads will be in brca-0409-raw_M
## 4853348 sequences in the BAM file
## 4853348 sequences exported
## 4853348 reads; of these:
##   4853348 (100.00%) were unpaired; of these:
##     378552 (7.80%) aligned 0 times
##     2526270 (52.05%) aligned exactly 1 time
##     1948526 (40.15%) aligned &gt;1 times
## 92.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1948526/4474796
## 2 1403878/1948526
## 3 6284730/14388664
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0410-raw_1 and brca-0410-raw_2
## Single-end reads will be in brca-0410-raw_M
## 9730571 sequences in the BAM file
## 9730571 sequences exported
## 9730571 reads; of these:
##   9730571 (100.00%) were unpaired; of these:
##     2672089 (27.46%) aligned 0 times
##     4364535 (44.85%) aligned exactly 1 time
##     2693947 (27.69%) aligned &gt;1 times
## 72.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2693947/7058482
## 2 2121659/2693947
## 3 9902108/23444261
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0411-raw_1 and brca-0411-raw_2
## Single-end reads will be in brca-0411-raw_M
## 9739270 sequences in the BAM file
## 9739270 sequences exported
## 9739270 reads; of these:
##   9739270 (100.00%) were unpaired; of these:
##     2789691 (28.64%) aligned 0 times
##     4517490 (46.38%) aligned exactly 1 time
##     2432089 (24.97%) aligned &gt;1 times
## 71.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2432089/6949579
## 2 1772066/2432089
## 3 7665271/18716848
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0412-raw_1 and brca-0412-raw_2
## Single-end reads will be in brca-0412-raw_M
## 4173236 sequences in the BAM file
## 4173236 sequences exported
## 4173236 reads; of these:
##   4173236 (100.00%) were unpaired; of these:
##     320568 (7.68%) aligned 0 times
##     2048223 (49.08%) aligned exactly 1 time
##     1804445 (43.24%) aligned &gt;1 times
## 92.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1804445/3852668
## 2 1499091/1804445
## 3 4835181/10729779
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0413-raw_1 and brca-0413-raw_2
## Single-end reads will be in brca-0413-raw_M
## 14882940 sequences in the BAM file
## 14882940 sequences exported
## 14882940 reads; of these:
##   14882940 (100.00%) were unpaired; of these:
##     3245215 (21.80%) aligned 0 times
##     7232116 (48.59%) aligned exactly 1 time
##     4405609 (29.60%) aligned &gt;1 times
## 78.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4405609/11637725
## 2 3615971/4405609
## 3 15397604/38521191
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in brca-0414-raw_1 and brca-0414-raw_2
## Single-end reads will be in brca-0414-raw_M
## 5010758 sequences in the BAM file
## 5010758 sequences exported
## 5010758 reads; of these:
##   5010758 (100.00%) were unpaired; of these:
##     1373328 (27.41%) aligned 0 times
##     2147398 (42.86%) aligned exactly 1 time
##     1490032 (29.74%) aligned &gt;1 times
## 72.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1490032/3637430
## 2 1045077/1490032
## 3 5677145/12826973
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0415-raw_1 and brca-0415-raw_2
## Single-end reads will be in brca-0415-raw_M
## 3164562 sequences in the BAM file
## 3164562 sequences exported
## 3164562 reads; of these:
##   3164562 (100.00%) were unpaired; of these:
##     649056 (20.51%) aligned 0 times
##     1352725 (42.75%) aligned exactly 1 time
##     1162781 (36.74%) aligned &gt;1 times
## 79.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1162781/2515506
## 2 1001929/1162781
## 3 4994940/10347427
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0416-raw_1 and brca-0416-raw_2
## Single-end reads will be in brca-0416-raw_M
## 4908464 sequences in the BAM file
## 4908464 sequences exported
## 4908464 reads; of these:
##   4908464 (100.00%) were unpaired; of these:
##     1336572 (27.23%) aligned 0 times
##     2039660 (41.55%) aligned exactly 1 time
##     1532232 (31.22%) aligned &gt;1 times
## 72.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1532232/3571892
## 2 1222205/1532232
## 3 5303734/12396335
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0417-raw_1 and brca-0417-raw_2
## Single-end reads will be in brca-0417-raw_M
## 16935794 sequences in the BAM file
## 16935794 sequences exported
## 16935794 reads; of these:
##   16935794 (100.00%) were unpaired; of these:
##     4409463 (26.04%) aligned 0 times
##     8477887 (50.06%) aligned exactly 1 time
##     4048444 (23.90%) aligned &gt;1 times
## 73.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4048444/12526331
## 2 2705265/4048444
## 3 9064895/27485578
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0418-raw_1 and brca-0418-raw_2
## Single-end reads will be in brca-0418-raw_M
## 6280815 sequences in the BAM file
## 6280815 sequences exported
## 6280815 reads; of these:
##   6280815 (100.00%) were unpaired; of these:
##     1437723 (22.89%) aligned 0 times
##     3486352 (55.51%) aligned exactly 1 time
##     1356740 (21.60%) aligned &gt;1 times
## 77.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1356740/4843092
## 2 1016151/1356740
## 3 4606626/11735566
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0419-raw_1 and brca-0419-raw_2
## Single-end reads will be in brca-0419-raw_M
## 1185085 sequences in the BAM file
## 1185085 sequences exported
## 1185085 reads; of these:
##   1185085 (100.00%) were unpaired; of these:
##     256933 (21.68%) aligned 0 times
##     470072 (39.67%) aligned exactly 1 time
##     458080 (38.65%) aligned &gt;1 times
## 78.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 458080/928152
## 2 387714/458080
## 3 2800205/4947793
## This looks like paired data from lane 6.
## Output will be in brca-0420-raw_1 and brca-0420-raw_2
## Single-end reads will be in brca-0420-raw_M
## 1882866 sequences in the BAM file
## 1882866 sequences exported
## 1882866 reads; of these:
##   1882866 (100.00%) were unpaired; of these:
##     255665 (13.58%) aligned 0 times
##     1071070 (56.89%) aligned exactly 1 time
##     556131 (29.54%) aligned &gt;1 times
## 86.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 556131/1627201
## 2 452386/556131
## 3 2751712/5810799
## This looks like paired data from lane 1.
## Output will be in brca-0421-raw_1 and brca-0421-raw_2
## Single-end reads will be in brca-0421-raw_M
## 4417859 sequences in the BAM file
## 4417859 sequences exported
## 4417859 reads; of these:
##   4417859 (100.00%) were unpaired; of these:
##     1007251 (22.80%) aligned 0 times
##     2055362 (46.52%) aligned exactly 1 time
##     1355246 (30.68%) aligned &gt;1 times
## 77.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1355246/3410608
## 2 1113985/1355246
## 3 4099650/9179627
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0422-raw_1 and brca-0422-raw_2
## Single-end reads will be in brca-0422-raw_M
## 9756594 sequences in the BAM file
## 9756594 sequences exported
## 9756594 reads; of these:
##   9756594 (100.00%) were unpaired; of these:
##     1945930 (19.94%) aligned 0 times
##     5235555 (53.66%) aligned exactly 1 time
##     2575109 (26.39%) aligned &gt;1 times
## 80.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2575109/7810664
## 2 2062441/2575109
## 3 9098339/21935258
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0423-raw_1 and brca-0423-raw_2
## Single-end reads will be in brca-0423-raw_M
## 3785674 sequences in the BAM file
## 3785674 sequences exported
## 3785674 reads; of these:
##   3785674 (100.00%) were unpaired; of these:
##     732716 (19.35%) aligned 0 times
##     2014798 (53.22%) aligned exactly 1 time
##     1038160 (27.42%) aligned &gt;1 times
## 80.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1038160/3052958
## 2 875347/1038160
## 3 2364016/6815774
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0424-raw_1 and brca-0424-raw_2
## Single-end reads will be in brca-0424-raw_M
## 4922668 sequences in the BAM file
## 4922668 sequences exported
## 4922668 reads; of these:
##   4922668 (100.00%) were unpaired; of these:
##     657154 (13.35%) aligned 0 times
##     2457518 (49.92%) aligned exactly 1 time
##     1807996 (36.73%) aligned &gt;1 times
## 86.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1807996/4265514
## 2 1551506/1807996
## 3 6978991/14712405
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0425-raw_1 and brca-0425-raw_2
## Single-end reads will be in brca-0425-raw_M
## 6016108 sequences in the BAM file
## 6016108 sequences exported
## 6016108 reads; of these:
##   6016108 (100.00%) were unpaired; of these:
##     1552444 (25.80%) aligned 0 times
##     2577496 (42.84%) aligned exactly 1 time
##     1886168 (31.35%) aligned &gt;1 times
## 74.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1886168/4463664
## 2 1338450/1886168
## 3 12885001/27211129
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0426-raw_1 and brca-0426-raw_2
## Single-end reads will be in brca-0426-raw_M
## 9592797 sequences in the BAM file
## 9592797 sequences exported
## 9592797 reads; of these:
##   9592797 (100.00%) were unpaired; of these:
##     2248502 (23.44%) aligned 0 times
##     3447858 (35.94%) aligned exactly 1 time
##     3896437 (40.62%) aligned &gt;1 times
## 76.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3896437/7344295
## 2 3398503/3896437
## 3 10722913/22852353
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0427-raw_1 and brca-0427-raw_2
## Single-end reads will be in brca-0427-raw_M
## 2104554 sequences in the BAM file
## 2104554 sequences exported
## 2104554 reads; of these:
##   2104554 (100.00%) were unpaired; of these:
##     618476 (29.39%) aligned 0 times
##     865291 (41.12%) aligned exactly 1 time
##     620787 (29.50%) aligned &gt;1 times
## 70.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 620787/1486078
## 2 493475/620787
## 3 2829429/5859216
## This looks like paired data from lane 5.
## Output will be in brca-0428-raw_1 and brca-0428-raw_2
## Single-end reads will be in brca-0428-raw_M
## 3683135 sequences in the BAM file
## 3683135 sequences exported
## 3683135 reads; of these:
##   3683135 (100.00%) were unpaired; of these:
##     364225 (9.89%) aligned 0 times
##     1970912 (53.51%) aligned exactly 1 time
##     1347998 (36.60%) aligned &gt;1 times
## 90.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1347998/3318910
## 2 1081850/1347998
## 3 6610901/13486973
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0429-raw_1 and brca-0429-raw_2
## Single-end reads will be in brca-0429-raw_M
## 3856632 sequences in the BAM file
## 3856632 sequences exported
## 3856632 reads; of these:
##   3856632 (100.00%) were unpaired; of these:
##     509808 (13.22%) aligned 0 times
##     2046886 (53.07%) aligned exactly 1 time
##     1299938 (33.71%) aligned &gt;1 times
## 86.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1299938/3346824
## 2 1094638/1299938
## 3 3750028/9160247
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0430-raw_1 and brca-0430-raw_2
## Single-end reads will be in brca-0430-raw_M
## 4440694 sequences in the BAM file
## 4440694 sequences exported
## 4440694 reads; of these:
##   4440694 (100.00%) were unpaired; of these:
##     251263 (5.66%) aligned 0 times
##     2552620 (57.48%) aligned exactly 1 time
##     1636811 (36.86%) aligned &gt;1 times
## 94.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1636811/4189431
## 2 1369235/1636811
## 3 7288110/14601386
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0431-raw_1 and brca-0431-raw_2
## Single-end reads will be in brca-0431-raw_M
## 16007843 sequences in the BAM file
## 16007843 sequences exported
## 16007843 reads; of these:
##   16007843 (100.00%) were unpaired; of these:
##     2980844 (18.62%) aligned 0 times
##     8991886 (56.17%) aligned exactly 1 time
##     4035113 (25.21%) aligned &gt;1 times
## 81.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4035113/13026999
## 2 3060736/4035113
## 3 13154787/34652713
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in brca-0432-raw_1 and brca-0432-raw_2
## Single-end reads will be in brca-0432-raw_M
## 2379094 sequences in the BAM file
## 2379094 sequences exported
## 2379094 reads; of these:
##   2379094 (100.00%) were unpaired; of these:
##     629222 (26.45%) aligned 0 times
##     1335807 (56.15%) aligned exactly 1 time
##     414065 (17.40%) aligned &gt;1 times
## 73.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 414065/1749872
## 2 329821/414065
## 3 1162307/3801528
## This looks like paired data from lane 5.
## Output will be in brca-0433-raw_1 and brca-0433-raw_2
## Single-end reads will be in brca-0433-raw_M
## 19261367 sequences in the BAM file
## 19261367 sequences exported
## 19261367 reads; of these:
##   19261367 (100.00%) were unpaired; of these:
##     5402914 (28.05%) aligned 0 times
##     9135950 (47.43%) aligned exactly 1 time
##     4722503 (24.52%) aligned &gt;1 times
## 71.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4722503/13858453
## 2 3230631/4722503
## 3 15498255/39061272
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in brca-0434-raw_1 and brca-0434-raw_2
## Single-end reads will be in brca-0434-raw_M
## 3485787 sequences in the BAM file
## 3485787 sequences exported
## 3485787 reads; of these:
##   3485787 (100.00%) were unpaired; of these:
##     303977 (8.72%) aligned 0 times
##     2053100 (58.90%) aligned exactly 1 time
##     1128710 (32.38%) aligned &gt;1 times
## 91.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1128710/3181810
## 2 828891/1128710
## 3 4774793/10510989
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0435-raw_1 and brca-0435-raw_2
## Single-end reads will be in brca-0435-raw_M
## 5560558 sequences in the BAM file
## 5560558 sequences exported
## 5560558 reads; of these:
##   5560558 (100.00%) were unpaired; of these:
##     1429422 (25.71%) aligned 0 times
##     2082657 (37.45%) aligned exactly 1 time
##     2048479 (36.84%) aligned &gt;1 times
## 74.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2048479/4131136
## 2 1592338/2048479
## 3 17651552/34247279
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0436-raw_1 and brca-0436-raw_2
## Single-end reads will be in brca-0436-raw_M
## 3944567 sequences in the BAM file
## 3944567 sequences exported
## 3944567 reads; of these:
##   3944567 (100.00%) were unpaired; of these:
##     425115 (10.78%) aligned 0 times
##     2228667 (56.50%) aligned exactly 1 time
##     1290785 (32.72%) aligned &gt;1 times
## 89.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1290785/3519452
## 2 1077786/1290785
## 3 7569782/15221859
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0437-raw_1 and brca-0437-raw_2
## Single-end reads will be in brca-0437-raw_M
## 18229969 sequences in the BAM file
## 18229969 sequences exported
## 18229969 reads; of these:
##   18229969 (100.00%) were unpaired; of these:
##     2412940 (13.24%) aligned 0 times
##     10555903 (57.90%) aligned exactly 1 time
##     5261126 (28.86%) aligned &gt;1 times
## 86.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5261126/15817029
## 2 3845977/5261126
## 3 16877031/43081649
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in brca-0438-raw_1 and brca-0438-raw_2
## Single-end reads will be in brca-0438-raw_M
## 8102698 sequences in the BAM file
## 8102698 sequences exported
## 8102698 reads; of these:
##   8102698 (100.00%) were unpaired; of these:
##     1948468 (24.05%) aligned 0 times
##     3688971 (45.53%) aligned exactly 1 time
##     2465259 (30.43%) aligned &gt;1 times
## 75.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2465259/6154230
## 2 2051251/2465259
## 3 17094598/38323709
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0439-raw_1 and brca-0439-raw_2
## Single-end reads will be in brca-0439-raw_M
## 15013102 sequences in the BAM file
## 15013102 sequences exported
## 15013102 reads; of these:
##   15013102 (100.00%) were unpaired; of these:
##     4010862 (26.72%) aligned 0 times
##     6975508 (46.46%) aligned exactly 1 time
##     4026732 (26.82%) aligned &gt;1 times
## 73.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4026732/11002240
## 2 3120917/4026732
## 3 8467759/24213370
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0440-raw_1 and brca-0440-raw_2
## Single-end reads will be in brca-0440-raw_M
## 21434324 sequences in the BAM file
## 21434324 sequences exported
## 21434324 reads; of these:
##   21434324 (100.00%) were unpaired; of these:
##     2744788 (12.81%) aligned 0 times
##     10434027 (48.68%) aligned exactly 1 time
##     8255509 (38.52%) aligned &gt;1 times
## 87.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 8255509/18689536
## 2 5883570/8255509
## 3 30516465/69128217
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in brca-0441-raw_1 and brca-0441-raw_2
## Single-end reads will be in brca-0441-raw_M
## 13120662 sequences in the BAM file
## 13120662 sequences exported
## 13120662 reads; of these:
##   13120662 (100.00%) were unpaired; of these:
##     2786360 (21.24%) aligned 0 times
##     4551916 (34.69%) aligned exactly 1 time
##     5782386 (44.07%) aligned &gt;1 times
## 78.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5782386/10334302
## 2 3762191/5782386
## 3 56495666/111725699
## [bam_sort_core] merging from 16 files...
## This looks like paired data from lane 1.
## Output will be in brca-0442-raw_1 and brca-0442-raw_2
## Single-end reads will be in brca-0442-raw_M
## 7574995 sequences in the BAM file
## 7574995 sequences exported
## 7574995 reads; of these:
##   7574995 (100.00%) were unpaired; of these:
##     1266734 (16.72%) aligned 0 times
##     2456055 (32.42%) aligned exactly 1 time
##     3852206 (50.85%) aligned &gt;1 times
## 83.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3852206/6308261
## 2 2424774/3852206
## 3 44562912/87080970
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 6.
## Output will be in brca-0443-raw_1 and brca-0443-raw_2
## Single-end reads will be in brca-0443-raw_M
## 10634193 sequences in the BAM file
## 10634193 sequences exported
## 10634193 reads; of these:
##   10634193 (100.00%) were unpaired; of these:
##     3123110 (29.37%) aligned 0 times
##     4546771 (42.76%) aligned exactly 1 time
##     2964312 (27.88%) aligned &gt;1 times
## 70.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2964312/7511083
## 2 2293506/2964312
## 3 7881540/19779454
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0444-raw_1 and brca-0444-raw_2
## Single-end reads will be in brca-0444-raw_M
## 5212563 sequences in the BAM file
## 5212563 sequences exported
## 5212563 reads; of these:
##   5212563 (100.00%) were unpaired; of these:
##     1145396 (21.97%) aligned 0 times
##     2457374 (47.14%) aligned exactly 1 time
##     1609793 (30.88%) aligned &gt;1 times
## 78.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1609793/4067167
## 2 1309662/1609793
## 3 11141018/23560423
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0445-raw_1 and brca-0445-raw_2
## Single-end reads will be in brca-0445-raw_M
## 4360441 sequences in the BAM file
## 4360441 sequences exported
## 4360441 reads; of these:
##   4360441 (100.00%) were unpaired; of these:
##     347604 (7.97%) aligned 0 times
##     2823047 (64.74%) aligned exactly 1 time
##     1189790 (27.29%) aligned &gt;1 times
## 92.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1189790/4012837
## 2 861164/1189790
## 3 6629776/14775152
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0446-raw_1 and brca-0446-raw_2
## Single-end reads will be in brca-0446-raw_M
## 3074428 sequences in the BAM file
## 3074428 sequences exported
## 3074428 reads; of these:
##   3074428 (100.00%) were unpaired; of these:
##     317219 (10.32%) aligned 0 times
##     1880204 (61.16%) aligned exactly 1 time
##     877005 (28.53%) aligned &gt;1 times
## 89.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 877005/2757209
## 2 658958/877005
## 3 5589370/12525333
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0447-raw_1 and brca-0447-raw_2
## Single-end reads will be in brca-0447-raw_M
## 8740839 sequences in the BAM file
## 8740839 sequences exported
## 8740839 reads; of these:
##   8740839 (100.00%) were unpaired; of these:
##     1570755 (17.97%) aligned 0 times
##     3920222 (44.85%) aligned exactly 1 time
##     3249862 (37.18%) aligned &gt;1 times
## 82.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3249862/7170084
## 2 2384808/3249862
## 3 14931651/33395052
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in brca-0448-raw_1 and brca-0448-raw_2
## Single-end reads will be in brca-0448-raw_M
## 5187585 sequences in the BAM file
## 5187585 sequences exported
## 5187585 reads; of these:
##   5187585 (100.00%) were unpaired; of these:
##     1259273 (24.27%) aligned 0 times
##     2414871 (46.55%) aligned exactly 1 time
##     1513441 (29.17%) aligned &gt;1 times
## 75.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1513441/3928312
## 2 1200009/1513441
## 3 4489594/10663430
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0449-raw_1 and brca-0449-raw_2
## Single-end reads will be in brca-0449-raw_M
## 11858702 sequences in the BAM file
## 11858702 sequences exported
## 11858702 reads; of these:
##   11858702 (100.00%) were unpaired; of these:
##     2907899 (24.52%) aligned 0 times
##     5470531 (46.13%) aligned exactly 1 time
##     3480272 (29.35%) aligned &gt;1 times
## 75.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3480272/8950803
## 2 2782993/3480272
## 3 8322097/21775910
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0450-raw_1 and brca-0450-raw_2
## Single-end reads will be in brca-0450-raw_M
## 15982687 sequences in the BAM file
## 15982687 sequences exported
## 15982687 reads; of these:
##   15982687 (100.00%) were unpaired; of these:
##     4655660 (29.13%) aligned 0 times
##     5988410 (37.47%) aligned exactly 1 time
##     5338617 (33.40%) aligned &gt;1 times
## 70.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5338617/11327027
## 2 4585108/5338617
## 3 19125315/42106747
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in brca-0451-raw_1 and brca-0451-raw_2
## Single-end reads will be in brca-0451-raw_M
## 10386201 sequences in the BAM file
## 10386201 sequences exported
## 10386201 reads; of these:
##   10386201 (100.00%) were unpaired; of these:
##     1878035 (18.08%) aligned 0 times
##     6236603 (60.05%) aligned exactly 1 time
##     2271563 (21.87%) aligned &gt;1 times
## 81.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2271563/8508166
## 2 1798886/2271563
## 3 5317887/17116959
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0452-raw_1 and brca-0452-raw_2
## Single-end reads will be in brca-0452-raw_M
## 17204084 sequences in the BAM file
## 17204084 sequences exported
## 17204084 reads; of these:
##   17204084 (100.00%) were unpaired; of these:
##     4456242 (25.90%) aligned 0 times
##     7257518 (42.18%) aligned exactly 1 time
##     5490324 (31.91%) aligned &gt;1 times
## 74.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5490324/12747842
## 2 3698727/5490324
## 3 20469337/43301295
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in brca-0453-raw_1 and brca-0453-raw_2
## Single-end reads will be in brca-0453-raw_M
## 11931870 sequences in the BAM file
## 11931870 sequences exported
## 11931870 reads; of these:
##   11931870 (100.00%) were unpaired; of these:
##     2163758 (18.13%) aligned 0 times
##     4680320 (39.23%) aligned exactly 1 time
##     5087792 (42.64%) aligned &gt;1 times
## 81.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5087792/9768112
## 2 4294683/5087792
## 3 26894017/56009201
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 2.
## Output will be in brca-0454-raw_1 and brca-0454-raw_2
## Single-end reads will be in brca-0454-raw_M
## 15326950 sequences in the BAM file
## 15326950 sequences exported
## 15326950 reads; of these:
##   15326950 (100.00%) were unpaired; of these:
##     4430771 (28.91%) aligned 0 times
##     7523374 (49.09%) aligned exactly 1 time
##     3372805 (22.01%) aligned &gt;1 times
## 71.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3372805/10896179
## 2 2651711/3372805
## 3 7291122/22240131
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0455-raw_1 and brca-0455-raw_2
## Single-end reads will be in brca-0455-raw_M
## 3115740 sequences in the BAM file
## 3115740 sequences exported
## 3115740 reads; of these:
##   3115740 (100.00%) were unpaired; of these:
##     281619 (9.04%) aligned 0 times
##     1639767 (52.63%) aligned exactly 1 time
##     1194354 (38.33%) aligned &gt;1 times
## 90.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1194354/2834121
## 2 951027/1194354
## 3 6622786/12734270
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0456-raw_1 and brca-0456-raw_2
## Single-end reads will be in brca-0456-raw_M
## 6112234 sequences in the BAM file
## 6112234 sequences exported
## 6112234 reads; of these:
##   6112234 (100.00%) were unpaired; of these:
##     759258 (12.42%) aligned 0 times
##     3618052 (59.19%) aligned exactly 1 time
##     1734924 (28.38%) aligned &gt;1 times
## 87.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1734924/5352976
## 2 1550451/1734924
## 3 4938707/12460687
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0457-raw_1 and brca-0457-raw_2
## Single-end reads will be in brca-0457-raw_M
## 11876144 sequences in the BAM file
## 11876144 sequences exported
## 11876144 reads; of these:
##   11876144 (100.00%) were unpaired; of these:
##     2418521 (20.36%) aligned 0 times
##     4677903 (39.39%) aligned exactly 1 time
##     4779720 (40.25%) aligned &gt;1 times
## 79.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4779720/9457623
## 2 4150203/4779720
## 3 11236248/26123772
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0458-raw_1 and brca-0458-raw_2
## Single-end reads will be in brca-0458-raw_M
## 3605549 sequences in the BAM file
## 3605549 sequences exported
## 3605549 reads; of these:
##   3605549 (100.00%) were unpaired; of these:
##     991240 (27.49%) aligned 0 times
##     1723355 (47.80%) aligned exactly 1 time
##     890954 (24.71%) aligned &gt;1 times
## 72.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 890954/2614309
## 2 684189/890954
## 3 4203834/9480963
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0459-raw_1 and brca-0459-raw_2
## Single-end reads will be in brca-0459-raw_M
## 13917422 sequences in the BAM file
## 13917422 sequences exported
## 13917422 reads; of these:
##   13917422 (100.00%) were unpaired; of these:
##     3582584 (25.74%) aligned 0 times
##     7425330 (53.35%) aligned exactly 1 time
##     2909508 (20.91%) aligned &gt;1 times
## 74.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2909508/10334838
## 2 1988918/2909508
## 3 19835347/43849324
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in brca-0460-raw_1 and brca-0460-raw_2
## Single-end reads will be in brca-0460-raw_M
## 5147414 sequences in the BAM file
## 5147414 sequences exported
## 5147414 reads; of these:
##   5147414 (100.00%) were unpaired; of these:
##     991722 (19.27%) aligned 0 times
##     1925254 (37.40%) aligned exactly 1 time
##     2230438 (43.33%) aligned &gt;1 times
## 80.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2230438/4155692
## 2 2009008/2230438
## 3 4036661/9656286
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0461-raw_1 and brca-0461-raw_2
## Single-end reads will be in brca-0461-raw_M
## 2095790 sequences in the BAM file
## 2095790 sequences exported
## 2095790 reads; of these:
##   2095790 (100.00%) were unpaired; of these:
##     306941 (14.65%) aligned 0 times
##     1017134 (48.53%) aligned exactly 1 time
##     771715 (36.82%) aligned &gt;1 times
## 85.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 771715/1788849
## 2 599984/771715
## 3 3857500/8029294
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0462-raw_1 and brca-0462-raw_2
## Single-end reads will be in brca-0462-raw_M
## 6379498 sequences in the BAM file
## 6379498 sequences exported
## 6379498 reads; of these:
##   6379498 (100.00%) were unpaired; of these:
##     1459035 (22.87%) aligned 0 times
##     2639283 (41.37%) aligned exactly 1 time
##     2281180 (35.76%) aligned &gt;1 times
## 77.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2281180/4920463
## 2 1874505/2281180
## 3 8906917/18795836
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0463-raw_1 and brca-0463-raw_2
## Single-end reads will be in brca-0463-raw_M
## 3181373 sequences in the BAM file
## 3181373 sequences exported
## 3181373 reads; of these:
##   3181373 (100.00%) were unpaired; of these:
##     659899 (20.74%) aligned 0 times
##     1375458 (43.23%) aligned exactly 1 time
##     1146016 (36.02%) aligned &gt;1 times
## 79.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1146016/2521474
## 2 1010587/1146016
## 3 3584375/8197162
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0464-raw_1 and brca-0464-raw_2
## Single-end reads will be in brca-0464-raw_M
## 2231458 sequences in the BAM file
## 2231458 sequences exported
## 2231458 reads; of these:
##   2231458 (100.00%) were unpaired; of these:
##     646232 (28.96%) aligned 0 times
##     814622 (36.51%) aligned exactly 1 time
##     770604 (34.53%) aligned &gt;1 times
## 71.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 770604/1585226
## 2 583998/770604
## 3 5276628/11272242
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0465-raw_1 and brca-0465-raw_2
## Single-end reads will be in brca-0465-raw_M
## 4041832 sequences in the BAM file
## 4041832 sequences exported
## 4041832 reads; of these:
##   4041832 (100.00%) were unpaired; of these:
##     474647 (11.74%) aligned 0 times
##     1940065 (48.00%) aligned exactly 1 time
##     1627120 (40.26%) aligned &gt;1 times
## 88.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1627120/3567185
## 2 1306621/1627120
## 3 6264155/13193460
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0466-raw_1 and brca-0466-raw_2
## Single-end reads will be in brca-0466-raw_M
## 3545557 sequences in the BAM file
## 3545557 sequences exported
## 3545557 reads; of these:
##   3545557 (100.00%) were unpaired; of these:
##     831262 (23.45%) aligned 0 times
##     1586433 (44.74%) aligned exactly 1 time
##     1127862 (31.81%) aligned &gt;1 times
## 76.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1127862/2714295
## 2 847515/1127862
## 3 6630999/13608508
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0467-raw_1 and brca-0467-raw_2
## Single-end reads will be in brca-0467-raw_M
## 3820826 sequences in the BAM file
## 3820826 sequences exported
## 3820826 reads; of these:
##   3820826 (100.00%) were unpaired; of these:
##     905800 (23.71%) aligned 0 times
##     1815016 (47.50%) aligned exactly 1 time
##     1100010 (28.79%) aligned &gt;1 times
## 76.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1100010/2915026
## 2 901548/1100010
## 3 3296069/7850471
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0468-raw_1 and brca-0468-raw_2
## Single-end reads will be in brca-0468-raw_M
## 5430765 sequences in the BAM file
## 5430765 sequences exported
## 5430765 reads; of these:
##   5430765 (100.00%) were unpaired; of these:
##     365784 (6.74%) aligned 0 times
##     2869872 (52.84%) aligned exactly 1 time
##     2195109 (40.42%) aligned &gt;1 times
## 93.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2195109/5064981
## 2 1789383/2195109
## 3 9892538/20255604
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0469-raw_1 and brca-0469-raw_2
## Single-end reads will be in brca-0469-raw_M
## 3578438 sequences in the BAM file
## 3578438 sequences exported
## 3578438 reads; of these:
##   3578438 (100.00%) were unpaired; of these:
##     850934 (23.78%) aligned 0 times
##     1445947 (40.41%) aligned exactly 1 time
##     1281557 (35.81%) aligned &gt;1 times
## 76.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1281557/2727504
## 2 1078133/1281557
## 3 2856767/7383981
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0470-raw_1 and brca-0470-raw_2
## Single-end reads will be in brca-0470-raw_M
## 3927902 sequences in the BAM file
## 3927902 sequences exported
## 3927902 reads; of these:
##   3927902 (100.00%) were unpaired; of these:
##     1042627 (26.54%) aligned 0 times
##     1970698 (50.17%) aligned exactly 1 time
##     914577 (23.28%) aligned &gt;1 times
## 73.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 914577/2885275
## 2 686144/914577
## 3 5573152/12026744
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0471-raw_1 and brca-0471-raw_2
## Single-end reads will be in brca-0471-raw_M
## 12103919 sequences in the BAM file
## 12103919 sequences exported
## 12103919 reads; of these:
##   12103919 (100.00%) were unpaired; of these:
##     3993815 (33.00%) aligned 0 times
##     5750872 (47.51%) aligned exactly 1 time
##     2359232 (19.49%) aligned &gt;1 times
## 67.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2359232/8110104
## 2 1520359/2359232
## 3 5896921/17968189
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0472-raw_1 and brca-0472-raw_2
## Single-end reads will be in brca-0472-raw_M
## 4010541 sequences in the BAM file
## 4010541 sequences exported
## 4010541 reads; of these:
##   4010541 (100.00%) were unpaired; of these:
##     275531 (6.87%) aligned 0 times
##     2466289 (61.50%) aligned exactly 1 time
##     1268721 (31.63%) aligned &gt;1 times
## 93.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1268721/3735010
## 2 1031033/1268721
## 3 6138589/13684063
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0473-raw_1 and brca-0473-raw_2
## Single-end reads will be in brca-0473-raw_M
## 15936531 sequences in the BAM file
## 15936531 sequences exported
## 15936531 reads; of these:
##   15936531 (100.00%) were unpaired; of these:
##     3952481 (24.80%) aligned 0 times
##     6887637 (43.22%) aligned exactly 1 time
##     5096413 (31.98%) aligned &gt;1 times
## 75.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5096413/11984050
## 2 3645502/5096413
## 3 12736153/32969506
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0474-raw_1 and brca-0474-raw_2
## Single-end reads will be in brca-0474-raw_M
## 4059572 sequences in the BAM file
## 4059572 sequences exported
## 4059572 reads; of these:
##   4059572 (100.00%) were unpaired; of these:
##     342244 (8.43%) aligned 0 times
##     2190748 (53.96%) aligned exactly 1 time
##     1526580 (37.60%) aligned &gt;1 times
## 91.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1526580/3717328
## 2 1237926/1526580
## 3 6101607/12910303
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0475-raw_1 and brca-0475-raw_2
## Single-end reads will be in brca-0475-raw_M
## 14886684 sequences in the BAM file
## 14886684 sequences exported
## 14886684 reads; of these:
##   14886684 (100.00%) were unpaired; of these:
##     4602949 (30.92%) aligned 0 times
##     7051820 (47.37%) aligned exactly 1 time
##     3231915 (21.71%) aligned &gt;1 times
## 69.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3231915/10283735
## 2 2350211/3231915
## 3 6735850/21207782
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0476-raw_1 and brca-0476-raw_2
## Single-end reads will be in brca-0476-raw_M
## 29716622 sequences in the BAM file
## 29716622 sequences exported
## 29716622 reads; of these:
##   29716622 (100.00%) were unpaired; of these:
##     8149072 (27.42%) aligned 0 times
##     13475115 (45.35%) aligned exactly 1 time
##     8092435 (27.23%) aligned &gt;1 times
## 72.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 8092435/21567550
## 2 6239764/8092435
## 3 27159864/67271864
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 1.
## Output will be in brca-0477-raw_1 and brca-0477-raw_2
## Single-end reads will be in brca-0477-raw_M
## 11428044 sequences in the BAM file
## 11428044 sequences exported
## 11428044 reads; of these:
##   11428044 (100.00%) were unpaired; of these:
##     2396504 (20.97%) aligned 0 times
##     5737150 (50.20%) aligned exactly 1 time
##     3294390 (28.83%) aligned &gt;1 times
## 79.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3294390/9031540
## 2 2297308/3294390
## 3 17647668/37392701
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0478-raw_1 and brca-0478-raw_2
## Single-end reads will be in brca-0478-raw_M
## 6028065 sequences in the BAM file
## 6028065 sequences exported
## 6028065 reads; of these:
##   6028065 (100.00%) were unpaired; of these:
##     1021298 (16.94%) aligned 0 times
##     2841254 (47.13%) aligned exactly 1 time
##     2165513 (35.92%) aligned &gt;1 times
## 83.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2165513/5006767
## 2 1437824/2165513
## 3 20743703/42037097
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0479-raw_1 and brca-0479-raw_2
## Single-end reads will be in brca-0479-raw_M
## 2381244 sequences in the BAM file
## 2381244 sequences exported
## 2381244 reads; of these:
##   2381244 (100.00%) were unpaired; of these:
##     572210 (24.03%) aligned 0 times
##     1067260 (44.82%) aligned exactly 1 time
##     741774 (31.15%) aligned &gt;1 times
## 75.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 741774/1809034
## 2 621472/741774
## 3 1823662/4467624
## This looks like paired data from lane 7.
## Output will be in brca-0480-raw_1 and brca-0480-raw_2
## Single-end reads will be in brca-0480-raw_M
## 16946144 sequences in the BAM file
## 16946144 sequences exported
## 16946144 reads; of these:
##   16946144 (100.00%) were unpaired; of these:
##     2690706 (15.88%) aligned 0 times
##     8794419 (51.90%) aligned exactly 1 time
##     5461019 (32.23%) aligned &gt;1 times
## 84.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5461019/14255438
## 2 4276110/5461019
## 3 17800957/41168388
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in brca-0481-raw_1 and brca-0481-raw_2
## Single-end reads will be in brca-0481-raw_M
## 4626016 sequences in the BAM file
## 4626016 sequences exported
## 4626016 reads; of these:
##   4626016 (100.00%) were unpaired; of these:
##     991415 (21.43%) aligned 0 times
##     1699078 (36.73%) aligned exactly 1 time
##     1935523 (41.84%) aligned &gt;1 times
## 78.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1935523/3634601
## 2 1672403/1935523
## 3 6693184/13297042
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0482-raw_1 and brca-0482-raw_2
## Single-end reads will be in brca-0482-raw_M
## 12009590 sequences in the BAM file
## 12009590 sequences exported
## 12009590 reads; of these:
##   12009590 (100.00%) were unpaired; of these:
##     2861729 (23.83%) aligned 0 times
##     5739195 (47.79%) aligned exactly 1 time
##     3408666 (28.38%) aligned &gt;1 times
## 76.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3408666/9147861
## 2 2731449/3408666
## 3 8523987/22400133
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0483-raw_1 and brca-0483-raw_2
## Single-end reads will be in brca-0483-raw_M
## 5484237 sequences in the BAM file
## 5484237 sequences exported
## 5484237 reads; of these:
##   5484237 (100.00%) were unpaired; of these:
##     936694 (17.08%) aligned 0 times
##     2743229 (50.02%) aligned exactly 1 time
##     1804314 (32.90%) aligned &gt;1 times
## 82.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1804314/4547543
## 2 1333338/1804314
## 3 9781608/19620090
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0484-raw_1 and brca-0484-raw_2
## Single-end reads will be in brca-0484-raw_M
## 2949857 sequences in the BAM file
## 2949857 sequences exported
## 2949857 reads; of these:
##   2949857 (100.00%) were unpaired; of these:
##     590410 (20.01%) aligned 0 times
##     1580725 (53.59%) aligned exactly 1 time
##     778722 (26.40%) aligned &gt;1 times
## 79.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 778722/2359447
## 2 607261/778722
## 3 3837273/8422362
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0485-raw_1 and brca-0485-raw_2
## Single-end reads will be in brca-0485-raw_M
## 5335211 sequences in the BAM file
## 5335211 sequences exported
## 5335211 reads; of these:
##   5335211 (100.00%) were unpaired; of these:
##     1983181 (37.17%) aligned 0 times
##     2241196 (42.01%) aligned exactly 1 time
##     1110834 (20.82%) aligned &gt;1 times
## 62.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1110834/3352030
## 2 788368/1110834
## 3 6437063/12281991
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0486-raw_1 and brca-0486-raw_2
## Single-end reads will be in brca-0486-raw_M
## 2374924 sequences in the BAM file
## 2374924 sequences exported
## 2374924 reads; of these:
##   2374924 (100.00%) were unpaired; of these:
##     271598 (11.44%) aligned 0 times
##     1223343 (51.51%) aligned exactly 1 time
##     879983 (37.05%) aligned &gt;1 times
## 88.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 879983/2103326
## 2 633420/879983
## 3 4877763/9970837
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0487-raw_1 and brca-0487-raw_2
## Single-end reads will be in brca-0487-raw_M
## 4499676 sequences in the BAM file
## 4499676 sequences exported
## 4499676 reads; of these:
##   4499676 (100.00%) were unpaired; of these:
##     370491 (8.23%) aligned 0 times
##     1873300 (41.63%) aligned exactly 1 time
##     2255885 (50.13%) aligned &gt;1 times
## 91.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2255885/4129185
## 2 1842971/2255885
## 3 14236216/29299758
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0488-raw_1 and brca-0488-raw_2
## Single-end reads will be in brca-0488-raw_M
## 30832278 sequences in the BAM file
## 30832278 sequences exported
## 30832278 reads; of these:
##   30832278 (100.00%) were unpaired; of these:
##     7523212 (24.40%) aligned 0 times
##     13835948 (44.87%) aligned exactly 1 time
##     9473118 (30.72%) aligned &gt;1 times
## 75.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 9473118/23309066
## 2 8037920/9473118
## 3 17088498/48029068
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 1.
## Output will be in brca-0489-raw_1 and brca-0489-raw_2
## Single-end reads will be in brca-0489-raw_M
## 1991447 sequences in the BAM file
## 1991447 sequences exported
## 1991447 reads; of these:
##   1991447 (100.00%) were unpaired; of these:
##     585977 (29.42%) aligned 0 times
##     1002359 (50.33%) aligned exactly 1 time
##     403111 (20.24%) aligned &gt;1 times
## 70.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 403111/1405470
## 2 293060/403111
## 3 1421382/4032032
## This looks like paired data from lane 5.
## Output will be in brca-0490-raw_1 and brca-0490-raw_2
## Single-end reads will be in brca-0490-raw_M
## 7262907 sequences in the BAM file
## 7262907 sequences exported
## 7262907 reads; of these:
##   7262907 (100.00%) were unpaired; of these:
##     582942 (8.03%) aligned 0 times
##     3708325 (51.06%) aligned exactly 1 time
##     2971640 (40.92%) aligned &gt;1 times
## 91.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2971640/6679965
## 2 2338591/2971640
## 3 15132553/29980596
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-0491-raw_1 and brca-0491-raw_2
## Single-end reads will be in brca-0491-raw_M
## 3478756 sequences in the BAM file
## 3478756 sequences exported
## 3478756 reads; of these:
##   3478756 (100.00%) were unpaired; of these:
##     772306 (22.20%) aligned 0 times
##     1353055 (38.89%) aligned exactly 1 time
##     1353395 (38.90%) aligned &gt;1 times
## 77.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1353395/2706450
## 2 1151007/1353395
## 3 5074813/10196601
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0492-raw_1 and brca-0492-raw_2
## Single-end reads will be in brca-0492-raw_M
## 10667676 sequences in the BAM file
## 10667676 sequences exported
## 10667676 reads; of these:
##   10667676 (100.00%) were unpaired; of these:
##     2298148 (21.54%) aligned 0 times
##     4460037 (41.81%) aligned exactly 1 time
##     3909491 (36.65%) aligned &gt;1 times
## 78.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3909491/8369528
## 2 3099707/3909491
## 3 8941487/21385049
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0493-raw_1 and brca-0493-raw_2
## Single-end reads will be in brca-0493-raw_M
## 4934541 sequences in the BAM file
## 4934541 sequences exported
## 4934541 reads; of these:
##   4934541 (100.00%) were unpaired; of these:
##     555906 (11.27%) aligned 0 times
##     2478077 (50.22%) aligned exactly 1 time
##     1900558 (38.52%) aligned &gt;1 times
## 88.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1900558/4378635
## 2 1645064/1900558
## 3 7058982/14799342
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0494-raw_1 and brca-0494-raw_2
## Single-end reads will be in brca-0494-raw_M
## 3103807 sequences in the BAM file
## 3103807 sequences exported
## 3103807 reads; of these:
##   3103807 (100.00%) were unpaired; of these:
##     720624 (23.22%) aligned 0 times
##     1374616 (44.29%) aligned exactly 1 time
##     1008567 (32.49%) aligned &gt;1 times
## 76.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1008567/2383183
## 2 839593/1008567
## 3 1937095/5303417
## This looks like paired data from lane 5.
## Output will be in brca-0495-raw_1 and brca-0495-raw_2
## Single-end reads will be in brca-0495-raw_M
## 4432994 sequences in the BAM file
## 4432994 sequences exported
## 4432994 reads; of these:
##   4432994 (100.00%) were unpaired; of these:
##     543776 (12.27%) aligned 0 times
##     1825536 (41.18%) aligned exactly 1 time
##     2063682 (46.55%) aligned &gt;1 times
## 87.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2063682/3889218
## 2 1784909/2063682
## 3 6255555/12943434
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0496-raw_1 and brca-0496-raw_2
## Single-end reads will be in brca-0496-raw_M
## 4384562 sequences in the BAM file
## 4384562 sequences exported
## 4384562 reads; of these:
##   4384562 (100.00%) were unpaired; of these:
##     366323 (8.35%) aligned 0 times
##     1385565 (31.60%) aligned exactly 1 time
##     2632674 (60.04%) aligned &gt;1 times
## 91.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2632674/4018239
## 2 1958304/2632674
## 3 29265839/55756867
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in brca-0497-raw_1 and brca-0497-raw_2
## Single-end reads will be in brca-0497-raw_M
## 3299507 sequences in the BAM file
## 3299507 sequences exported
## 3299507 reads; of these:
##   3299507 (100.00%) were unpaired; of these:
##     663934 (20.12%) aligned 0 times
##     1567413 (47.50%) aligned exactly 1 time
##     1068160 (32.37%) aligned &gt;1 times
## 79.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1068160/2635573
## 2 847523/1068160
## 3 7723782/15532218
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0498-raw_1 and brca-0498-raw_2
## Single-end reads will be in brca-0498-raw_M
## 5779811 sequences in the BAM file
## 5779811 sequences exported
## 5779811 reads; of these:
##   5779811 (100.00%) were unpaired; of these:
##     1186884 (20.53%) aligned 0 times
##     2354781 (40.74%) aligned exactly 1 time
##     2238146 (38.72%) aligned &gt;1 times
## 79.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2238146/4592927
## 2 1717446/2238146
## 3 19678716/39215938
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0499-raw_1 and brca-0499-raw_2
## Single-end reads will be in brca-0499-raw_M
## 11549633 sequences in the BAM file
## 11549633 sequences exported
## 11549633 reads; of these:
##   11549633 (100.00%) were unpaired; of these:
##     3368048 (29.16%) aligned 0 times
##     5315098 (46.02%) aligned exactly 1 time
##     2866487 (24.82%) aligned &gt;1 times
## 70.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2866487/8181585
## 2 2326861/2866487
## 3 7287816/19415281
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0500-raw_1 and brca-0500-raw_2
## Single-end reads will be in brca-0500-raw_M
## 16797014 sequences in the BAM file
## 16797014 sequences exported
## 16797014 reads; of these:
##   16797014 (100.00%) were unpaired; of these:
##     5043629 (30.03%) aligned 0 times
##     8292515 (49.37%) aligned exactly 1 time
##     3460870 (20.60%) aligned &gt;1 times
## 69.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3460870/11753385
## 2 2704009/3460870
## 3 11395779/30427680
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0501-raw_1 and brca-0501-raw_2
## Single-end reads will be in brca-0501-raw_M
## 10591927 sequences in the BAM file
## 10591927 sequences exported
## 10591927 reads; of these:
##   10591927 (100.00%) were unpaired; of these:
##     2401430 (22.67%) aligned 0 times
##     4812585 (45.44%) aligned exactly 1 time
##     3377912 (31.89%) aligned &gt;1 times
## 77.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3377912/8190497
## 2 2622161/3377912
## 3 6995851/18924784
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0502-raw_1 and brca-0502-raw_2
## Single-end reads will be in brca-0502-raw_M
## 4892952 sequences in the BAM file
## 4892952 sequences exported
## 4892952 reads; of these:
##   4892952 (100.00%) were unpaired; of these:
##     1017399 (20.79%) aligned 0 times
##     2175114 (44.45%) aligned exactly 1 time
##     1700439 (34.75%) aligned &gt;1 times
## 79.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1700439/3875553
## 2 1323450/1700439
## 3 5007550/12102256
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0503-raw_1 and brca-0503-raw_2
## Single-end reads will be in brca-0503-raw_M
## 13725028 sequences in the BAM file
## 13725028 sequences exported
## 13725028 reads; of these:
##   13725028 (100.00%) were unpaired; of these:
##     2576922 (18.78%) aligned 0 times
##     4594430 (33.47%) aligned exactly 1 time
##     6553676 (47.75%) aligned &gt;1 times
## 81.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6553676/11148106
## 2 4475698/6553676
## 3 56063355/111321609
## [bam_sort_core] merging from 16 files...
## This looks like paired data from lane 4.
## Output will be in brca-0504-raw_1 and brca-0504-raw_2
## Single-end reads will be in brca-0504-raw_M
## 7691244 sequences in the BAM file
## 7691244 sequences exported
## 7691244 reads; of these:
##   7691244 (100.00%) were unpaired; of these:
##     1588545 (20.65%) aligned 0 times
##     4402097 (57.24%) aligned exactly 1 time
##     1700602 (22.11%) aligned &gt;1 times
## 79.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1700602/6102699
## 2 1271835/1700602
## 3 6969372/17653104
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0505-raw_1 and brca-0505-raw_2
## Single-end reads will be in brca-0505-raw_M
## 1901704 sequences in the BAM file
## 1901704 sequences exported
## 1901704 reads; of these:
##   1901704 (100.00%) were unpaired; of these:
##     473241 (24.89%) aligned 0 times
##     791889 (41.64%) aligned exactly 1 time
##     636574 (33.47%) aligned &gt;1 times
## 75.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 636574/1428463
## 2 529060/636574
## 3 3405964/6385069
## This looks like paired data from lane 5.
## Output will be in brca-0506-raw_1 and brca-0506-raw_2
## Single-end reads will be in brca-0506-raw_M
## 3411458 sequences in the BAM file
## 3411458 sequences exported
## 3411458 reads; of these:
##   3411458 (100.00%) were unpaired; of these:
##     441655 (12.95%) aligned 0 times
##     1903394 (55.79%) aligned exactly 1 time
##     1066409 (31.26%) aligned &gt;1 times
## 87.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1066409/2969803
## 2 853497/1066409
## 3 4804875/10189671
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0507-raw_1 and brca-0507-raw_2
## Single-end reads will be in brca-0507-raw_M
## 2581939 sequences in the BAM file
## 2581939 sequences exported
## 2581939 reads; of these:
##   2581939 (100.00%) were unpaired; of these:
##     575239 (22.28%) aligned 0 times
##     1330032 (51.51%) aligned exactly 1 time
##     676668 (26.21%) aligned &gt;1 times
## 77.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 676668/2006700
## 2 490445/676668
## 3 1817500/5157274
## This looks like paired data from lane 3.
## Output will be in brca-0508-raw_1 and brca-0508-raw_2
## Single-end reads will be in brca-0508-raw_M
## 3206013 sequences in the BAM file
## 3206013 sequences exported
## 3206013 reads; of these:
##   3206013 (100.00%) were unpaired; of these:
##     813082 (25.36%) aligned 0 times
##     998290 (31.14%) aligned exactly 1 time
##     1394641 (43.50%) aligned &gt;1 times
## 74.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1394641/2392931
## 2 1049791/1394641
## 3 10867448/20517319
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0509-raw_1 and brca-0509-raw_2
## Single-end reads will be in brca-0509-raw_M
## 5537113 sequences in the BAM file
## 5537113 sequences exported
## 5537113 reads; of these:
##   5537113 (100.00%) were unpaired; of these:
##     1380128 (24.93%) aligned 0 times
##     2088257 (37.71%) aligned exactly 1 time
##     2068728 (37.36%) aligned &gt;1 times
## 75.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2068728/4156985
## 2 1145601/2068728
## 3 5649424/13574712
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0510-raw_1 and brca-0510-raw_2
## Single-end reads will be in brca-0510-raw_M
## 3117859 sequences in the BAM file
## 3117859 sequences exported
## 3117859 reads; of these:
##   3117859 (100.00%) were unpaired; of these:
##     789600 (25.33%) aligned 0 times
##     1320895 (42.37%) aligned exactly 1 time
##     1007364 (32.31%) aligned &gt;1 times
## 74.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1007364/2328259
## 2 769691/1007364
## 3 4136083/8809054
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0511-raw_1 and brca-0511-raw_2
## Single-end reads will be in brca-0511-raw_M
## 4715779 sequences in the BAM file
## 4715779 sequences exported
## 4715779 reads; of these:
##   4715779 (100.00%) were unpaired; of these:
##     993188 (21.06%) aligned 0 times
##     2352321 (49.88%) aligned exactly 1 time
##     1370270 (29.06%) aligned &gt;1 times
## 78.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1370270/3722591
## 2 1146657/1370270
## 3 6379583/13583669
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0512-raw_1 and brca-0512-raw_2
## Single-end reads will be in brca-0512-raw_M
## 2790923 sequences in the BAM file
## 2790923 sequences exported
## 2790923 reads; of these:
##   2790923 (100.00%) were unpaired; of these:
##     349025 (12.51%) aligned 0 times
##     1454870 (52.13%) aligned exactly 1 time
##     987028 (35.37%) aligned &gt;1 times
## 87.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 987028/2441898
## 2 796931/987028
## 3 4581230/9367953
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0513-raw_1 and brca-0513-raw_2
## Single-end reads will be in brca-0513-raw_M
## 5131425 sequences in the BAM file
## 5131425 sequences exported
## 5131425 reads; of these:
##   5131425 (100.00%) were unpaired; of these:
##     400598 (7.81%) aligned 0 times
##     3324731 (64.79%) aligned exactly 1 time
##     1406096 (27.40%) aligned &gt;1 times
## 92.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1406096/4730827
## 2 1055081/1406096
## 3 9337303/19612124
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0514-raw_1 and brca-0514-raw_2
## Single-end reads will be in brca-0514-raw_M
## 3995415 sequences in the BAM file
## 3995415 sequences exported
## 3995415 reads; of these:
##   3995415 (100.00%) were unpaired; of these:
##     946468 (23.69%) aligned 0 times
##     1558367 (39.00%) aligned exactly 1 time
##     1490580 (37.31%) aligned &gt;1 times
## 76.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1490580/3048947
## 2 1122857/1490580
## 3 5816161/12471797
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0515-raw_1 and brca-0515-raw_2
## Single-end reads will be in brca-0515-raw_M
## 11432173 sequences in the BAM file
## 11432173 sequences exported
## 11432173 reads; of these:
##   11432173 (100.00%) were unpaired; of these:
##     3100423 (27.12%) aligned 0 times
##     5300340 (46.36%) aligned exactly 1 time
##     3031410 (26.52%) aligned &gt;1 times
## 72.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3031410/8331750
## 2 2273465/3031410
## 3 6968539/19256928
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0516-raw_1 and brca-0516-raw_2
## Single-end reads will be in brca-0516-raw_M
## 6312203 sequences in the BAM file
## 6312203 sequences exported
## 6312203 reads; of these:
##   6312203 (100.00%) were unpaired; of these:
##     1362908 (21.59%) aligned 0 times
##     2699132 (42.76%) aligned exactly 1 time
##     2250163 (35.65%) aligned &gt;1 times
## 78.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2250163/4949295
## 2 1917967/2250163
## 3 8669828/18013456
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0517-raw_1 and brca-0517-raw_2
## Single-end reads will be in brca-0517-raw_M
## 3602449 sequences in the BAM file
## 3602449 sequences exported
## 3602449 reads; of these:
##   3602449 (100.00%) were unpaired; of these:
##     577968 (16.04%) aligned 0 times
##     1884333 (52.31%) aligned exactly 1 time
##     1140148 (31.65%) aligned &gt;1 times
## 83.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1140148/3024481
## 2 913134/1140148
## 3 3711668/8804270
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0518-raw_1 and brca-0518-raw_2
## Single-end reads will be in brca-0518-raw_M
## 20726458 sequences in the BAM file
## 20726458 sequences exported
## 20726458 reads; of these:
##   20726458 (100.00%) were unpaired; of these:
##     3493165 (16.85%) aligned 0 times
##     8073136 (38.95%) aligned exactly 1 time
##     9160157 (44.20%) aligned &gt;1 times
## 83.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 9160157/17233293
## 2 7913734/9160157
## 3 21272614/47163658
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 4.
## Output will be in brca-0519-raw_1 and brca-0519-raw_2
## Single-end reads will be in brca-0519-raw_M
## 3708782 sequences in the BAM file
## 3708782 sequences exported
## 3708782 reads; of these:
##   3708782 (100.00%) were unpaired; of these:
##     1150652 (31.03%) aligned 0 times
##     1575954 (42.49%) aligned exactly 1 time
##     982176 (26.48%) aligned &gt;1 times
## 68.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 982176/2558130
## 2 813337/982176
## 3 4310951/8926387
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0520-raw_1 and brca-0520-raw_2
## Single-end reads will be in brca-0520-raw_M
## 3287907 sequences in the BAM file
## 3287907 sequences exported
## 3287907 reads; of these:
##   3287907 (100.00%) were unpaired; of these:
##     666050 (20.26%) aligned 0 times
##     1494458 (45.45%) aligned exactly 1 time
##     1127399 (34.29%) aligned &gt;1 times
## 79.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1127399/2621857
## 2 872151/1127399
## 3 4011920/9019736
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0521-raw_1 and brca-0521-raw_2
## Single-end reads will be in brca-0521-raw_M
## 4826130 sequences in the BAM file
## 4826130 sequences exported
## 4826130 reads; of these:
##   4826130 (100.00%) were unpaired; of these:
##     439229 (9.10%) aligned 0 times
##     3074451 (63.70%) aligned exactly 1 time
##     1312450 (27.19%) aligned &gt;1 times
## 90.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1312450/4386901
## 2 954819/1312450
## 3 6305017/14736882
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0522-raw_1 and brca-0522-raw_2
## Single-end reads will be in brca-0522-raw_M
## 2210197 sequences in the BAM file
## 2210197 sequences exported
## 2210197 reads; of these:
##   2210197 (100.00%) were unpaired; of these:
##     278718 (12.61%) aligned 0 times
##     1297447 (58.70%) aligned exactly 1 time
##     634032 (28.69%) aligned &gt;1 times
## 87.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 634032/1931479
## 2 483160/634032
## 3 4781895/9648191
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0523-raw_1 and brca-0523-raw_2
## Single-end reads will be in brca-0523-raw_M
## 3233832 sequences in the BAM file
## 3233832 sequences exported
## 3233832 reads; of these:
##   3233832 (100.00%) were unpaired; of these:
##     680353 (21.04%) aligned 0 times
##     1059287 (32.76%) aligned exactly 1 time
##     1494192 (46.20%) aligned &gt;1 times
## 78.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1494192/2553479
## 2 1164790/1494192
## 3 10530973/20830605
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0524-raw_1 and brca-0524-raw_2
## Single-end reads will be in brca-0524-raw_M
## 3404989 sequences in the BAM file
## 3404989 sequences exported
## 3404989 reads; of these:
##   3404989 (100.00%) were unpaired; of these:
##     623338 (18.31%) aligned 0 times
##     1279276 (37.57%) aligned exactly 1 time
##     1502375 (44.12%) aligned &gt;1 times
## 81.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1502375/2781651
## 2 1255674/1502375
## 3 5182907/10398438
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0525-raw_1 and brca-0525-raw_2
## Single-end reads will be in brca-0525-raw_M
## 10531491 sequences in the BAM file
## 10531491 sequences exported
## 10531491 reads; of these:
##   10531491 (100.00%) were unpaired; of these:
##     2080637 (19.76%) aligned 0 times
##     5373228 (51.02%) aligned exactly 1 time
##     3077626 (29.22%) aligned &gt;1 times
## 80.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3077626/8450854
## 2 2392127/3077626
## 3 11895991/26949117
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0526-raw_1 and brca-0526-raw_2
## Single-end reads will be in brca-0526-raw_M
## 10057740 sequences in the BAM file
## 10057740 sequences exported
## 10057740 reads; of these:
##   10057740 (100.00%) were unpaired; of these:
##     1775809 (17.66%) aligned 0 times
##     4879753 (48.52%) aligned exactly 1 time
##     3402178 (33.83%) aligned &gt;1 times
## 82.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3402178/8281931
## 2 2836497/3402178
## 3 10376825/28015651
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0527-raw_1 and brca-0527-raw_2
## Single-end reads will be in brca-0527-raw_M
## 11744714 sequences in the BAM file
## 11744714 sequences exported
## 11744714 reads; of these:
##   11744714 (100.00%) were unpaired; of these:
##     2326500 (19.81%) aligned 0 times
##     5208342 (44.35%) aligned exactly 1 time
##     4209872 (35.84%) aligned &gt;1 times
## 80.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4209872/9418214
## 2 3315736/4209872
## 3 8606878/22536396
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0528-raw_1 and brca-0528-raw_2
## Single-end reads will be in brca-0528-raw_M
## 2226322 sequences in the BAM file
## 2226322 sequences exported
## 2226322 reads; of these:
##   2226322 (100.00%) were unpaired; of these:
##     620636 (27.88%) aligned 0 times
##     958255 (43.04%) aligned exactly 1 time
##     647431 (29.08%) aligned &gt;1 times
## 72.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 647431/1605686
## 2 468368/647431
## 3 2259368/5538510
## This looks like paired data from lane 6.
## Output will be in brca-0529-raw_1 and brca-0529-raw_2
## Single-end reads will be in brca-0529-raw_M
## 2251311 sequences in the BAM file
## 2251311 sequences exported
## 2251311 reads; of these:
##   2251311 (100.00%) were unpaired; of these:
##     195586 (8.69%) aligned 0 times
##     1369125 (60.81%) aligned exactly 1 time
##     686600 (30.50%) aligned &gt;1 times
## 91.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 686600/2055725
## 2 505297/686600
## 3 3818867/8259229
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0530-raw_1 and brca-0530-raw_2
## Single-end reads will be in brca-0530-raw_M
## 12595031 sequences in the BAM file
## 12595031 sequences exported
## 12595031 reads; of these:
##   12595031 (100.00%) were unpaired; of these:
##     2025642 (16.08%) aligned 0 times
##     6293476 (49.97%) aligned exactly 1 time
##     4275913 (33.95%) aligned &gt;1 times
## 83.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4275913/10569389
## 2 2813922/4275913
## 3 41062203/86011791
## [bam_sort_core] merging from 13 files...
## This looks like paired data from lane 7.
## Output will be in brca-0531-raw_1 and brca-0531-raw_2
## Single-end reads will be in brca-0531-raw_M
## 2896237 sequences in the BAM file
## 2896237 sequences exported
## 2896237 reads; of these:
##   2896237 (100.00%) were unpaired; of these:
##     664561 (22.95%) aligned 0 times
##     1371171 (47.34%) aligned exactly 1 time
##     860505 (29.71%) aligned &gt;1 times
## 77.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 860505/2231676
## 2 721314/860505
## 3 2275744/5978251
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0532-raw_1 and brca-0532-raw_2
## Single-end reads will be in brca-0532-raw_M
## 6445641 sequences in the BAM file
## 6445641 sequences exported
## 6445641 reads; of these:
##   6445641 (100.00%) were unpaired; of these:
##     1127585 (17.49%) aligned 0 times
##     3502586 (54.34%) aligned exactly 1 time
##     1815470 (28.17%) aligned &gt;1 times
## 82.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1815470/5318056
## 2 1113004/1815470
## 3 6352990/16711989
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0533-raw_1 and brca-0533-raw_2
## Single-end reads will be in brca-0533-raw_M
## 6669010 sequences in the BAM file
## 6669010 sequences exported
## 6669010 reads; of these:
##   6669010 (100.00%) were unpaired; of these:
##     2026633 (30.39%) aligned 0 times
##     3363292 (50.43%) aligned exactly 1 time
##     1279085 (19.18%) aligned &gt;1 times
## 69.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1279085/4642377
## 2 966187/1279085
## 3 4242616/11854616
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0534-raw_1 and brca-0534-raw_2
## Single-end reads will be in brca-0534-raw_M
## 6516925 sequences in the BAM file
## 6516925 sequences exported
## 6516925 reads; of these:
##   6516925 (100.00%) were unpaired; of these:
##     1235814 (18.96%) aligned 0 times
##     2946848 (45.22%) aligned exactly 1 time
##     2334263 (35.82%) aligned &gt;1 times
## 81.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2334263/5281111
## 2 1794172/2334263
## 3 9362508/20341952
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0535-raw_1 and brca-0535-raw_2
## Single-end reads will be in brca-0535-raw_M
## 4025252 sequences in the BAM file
## 4025252 sequences exported
## 4025252 reads; of these:
##   4025252 (100.00%) were unpaired; of these:
##     303200 (7.53%) aligned 0 times
##     2339332 (58.12%) aligned exactly 1 time
##     1382720 (34.35%) aligned &gt;1 times
## 92.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1382720/3722052
## 2 1086020/1382720
## 3 9625171/18945962
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0536-raw_1 and brca-0536-raw_2
## Single-end reads will be in brca-0536-raw_M
## 2975953 sequences in the BAM file
## 2975953 sequences exported
## 2975953 reads; of these:
##   2975953 (100.00%) were unpaired; of these:
##     921536 (30.97%) aligned 0 times
##     1292216 (43.42%) aligned exactly 1 time
##     762201 (25.61%) aligned &gt;1 times
## 69.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 762201/2054417
## 2 600944/762201
## 3 3632685/7835884
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0537-raw_1 and brca-0537-raw_2
## Single-end reads will be in brca-0537-raw_M
## 3730793 sequences in the BAM file
## 3730793 sequences exported
## 3730793 reads; of these:
##   3730793 (100.00%) were unpaired; of these:
##     567807 (15.22%) aligned 0 times
##     1723789 (46.20%) aligned exactly 1 time
##     1439197 (38.58%) aligned &gt;1 times
## 84.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1439197/3162986
## 2 1238953/1439197
## 3 3344547/8009259
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0538-raw_1 and brca-0538-raw_2
## Single-end reads will be in brca-0538-raw_M
## 4967218 sequences in the BAM file
## 4967218 sequences exported
## 4967218 reads; of these:
##   4967218 (100.00%) were unpaired; of these:
##     893783 (17.99%) aligned 0 times
##     2812785 (56.63%) aligned exactly 1 time
##     1260650 (25.38%) aligned &gt;1 times
## 82.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1260650/4073435
## 2 956231/1260650
## 3 3658283/10082301
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0539-raw_1 and brca-0539-raw_2
## Single-end reads will be in brca-0539-raw_M
## 12502068 sequences in the BAM file
## 12502068 sequences exported
## 12502068 reads; of these:
##   12502068 (100.00%) were unpaired; of these:
##     3452225 (27.61%) aligned 0 times
##     5472716 (43.77%) aligned exactly 1 time
##     3577127 (28.61%) aligned &gt;1 times
## 72.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3577127/9049843
## 2 2794814/3577127
## 3 16395643/35699873
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0540-raw_1 and brca-0540-raw_2
## Single-end reads will be in brca-0540-raw_M
## 2394534 sequences in the BAM file
## 2394534 sequences exported
## 2394534 reads; of these:
##   2394534 (100.00%) were unpaired; of these:
##     373607 (15.60%) aligned 0 times
##     1069285 (44.66%) aligned exactly 1 time
##     951642 (39.74%) aligned &gt;1 times
## 84.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 951642/2020927
## 2 763139/951642
## 3 3779217/7689007
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0541-raw_1 and brca-0541-raw_2
## Single-end reads will be in brca-0541-raw_M
## 5167079 sequences in the BAM file
## 5167079 sequences exported
## 5167079 reads; of these:
##   5167079 (100.00%) were unpaired; of these:
##     1096393 (21.22%) aligned 0 times
##     2429556 (47.02%) aligned exactly 1 time
##     1641130 (31.76%) aligned &gt;1 times
## 78.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1641130/4070686
## 2 1385605/1641130
## 3 5010150/11456151
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0542-raw_1 and brca-0542-raw_2
## Single-end reads will be in brca-0542-raw_M
## 6612045 sequences in the BAM file
## 6612045 sequences exported
## 6612045 reads; of these:
##   6612045 (100.00%) were unpaired; of these:
##     1239922 (18.75%) aligned 0 times
##     2985861 (45.16%) aligned exactly 1 time
##     2386262 (36.09%) aligned &gt;1 times
## 81.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2386262/5372123
## 2 2097539/2386262
## 3 5348943/12721314
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0543-raw_1 and brca-0543-raw_2
## Single-end reads will be in brca-0543-raw_M
## 3174382 sequences in the BAM file
## 3174382 sequences exported
## 3174382 reads; of these:
##   3174382 (100.00%) were unpaired; of these:
##     713097 (22.46%) aligned 0 times
##     1598324 (50.35%) aligned exactly 1 time
##     862961 (27.19%) aligned &gt;1 times
## 77.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 862961/2461285
## 2 667724/862961
## 3 3055974/7647056
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0544-raw_1 and brca-0544-raw_2
## Single-end reads will be in brca-0544-raw_M
## 10986371 sequences in the BAM file
## 10986371 sequences exported
## 10986371 reads; of these:
##   10986371 (100.00%) were unpaired; of these:
##     2766320 (25.18%) aligned 0 times
##     4902512 (44.62%) aligned exactly 1 time
##     3317539 (30.20%) aligned &gt;1 times
## 74.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3317539/8220051
## 2 2689703/3317539
## 3 11502089/26382370
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0545-raw_1 and brca-0545-raw_2
## Single-end reads will be in brca-0545-raw_M
## 4210563 sequences in the BAM file
## 4210563 sequences exported
## 4210563 reads; of these:
##   4210563 (100.00%) were unpaired; of these:
##     949512 (22.55%) aligned 0 times
##     1675884 (39.80%) aligned exactly 1 time
##     1585167 (37.65%) aligned &gt;1 times
## 77.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1585167/3261051
## 2 1298169/1585167
## 3 6429201/13058365
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0546-raw_1 and brca-0546-raw_2
## Single-end reads will be in brca-0546-raw_M
## 8713474 sequences in the BAM file
## 8713474 sequences exported
## 8713474 reads; of these:
##   8713474 (100.00%) were unpaired; of these:
##     484912 (5.57%) aligned 0 times
##     4420005 (50.73%) aligned exactly 1 time
##     3808557 (43.71%) aligned &gt;1 times
## 94.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3808557/8228562
## 2 3243901/3808557
## 3 11604169/24468261
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0547-raw_1 and brca-0547-raw_2
## Single-end reads will be in brca-0547-raw_M
## 7805488 sequences in the BAM file
## 7805488 sequences exported
## 7805488 reads; of these:
##   7805488 (100.00%) were unpaired; of these:
##     2037799 (26.11%) aligned 0 times
##     2598066 (33.29%) aligned exactly 1 time
##     3169623 (40.61%) aligned &gt;1 times
## 73.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3169623/5767689
## 2 2069460/3169623
## 3 31407224/64567424
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 5.
## Output will be in brca-0548-raw_1 and brca-0548-raw_2
## Single-end reads will be in brca-0548-raw_M
## 1884074 sequences in the BAM file
## 1884074 sequences exported
## 1884074 reads; of these:
##   1884074 (100.00%) were unpaired; of these:
##     501898 (26.64%) aligned 0 times
##     974578 (51.73%) aligned exactly 1 time
##     407598 (21.63%) aligned &gt;1 times
## 73.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 407598/1382176
## 2 314000/407598
## 3 1207560/3447640
## This looks like paired data from lane 2.
## Output will be in brca-0549-raw_1 and brca-0549-raw_2
## Single-end reads will be in brca-0549-raw_M
## 3076126 sequences in the BAM file
## 3076126 sequences exported
## 3076126 reads; of these:
##   3076126 (100.00%) were unpaired; of these:
##     761812 (24.77%) aligned 0 times
##     1389914 (45.18%) aligned exactly 1 time
##     924400 (30.05%) aligned &gt;1 times
## 75.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 924400/2314314
## 2 712316/924400
## 3 2911176/6867048
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0550-raw_1 and brca-0550-raw_2
## Single-end reads will be in brca-0550-raw_M
## 5804407 sequences in the BAM file
## 5804407 sequences exported
## 5804407 reads; of these:
##   5804407 (100.00%) were unpaired; of these:
##     453954 (7.82%) aligned 0 times
##     3466752 (59.73%) aligned exactly 1 time
##     1883701 (32.45%) aligned &gt;1 times
## 92.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1883701/5350453
## 2 1497273/1883701
## 3 7174367/15936106
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0551-raw_1 and brca-0551-raw_2
## Single-end reads will be in brca-0551-raw_M
## 4489839 sequences in the BAM file
## 4489839 sequences exported
## 4489839 reads; of these:
##   4489839 (100.00%) were unpaired; of these:
##     1207024 (26.88%) aligned 0 times
##     2272089 (50.61%) aligned exactly 1 time
##     1010726 (22.51%) aligned &gt;1 times
## 73.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1010726/3282815
## 2 732839/1010726
## 3 4794483/10794770
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0552-raw_1 and brca-0552-raw_2
## Single-end reads will be in brca-0552-raw_M
## 3638361 sequences in the BAM file
## 3638361 sequences exported
## 3638361 reads; of these:
##   3638361 (100.00%) were unpaired; of these:
##     791673 (21.76%) aligned 0 times
##     1661511 (45.67%) aligned exactly 1 time
##     1185177 (32.57%) aligned &gt;1 times
## 78.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1185177/2846688
## 2 940703/1185177
## 3 4506791/10251143
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0553-raw_1 and brca-0553-raw_2
## Single-end reads will be in brca-0553-raw_M
## 9623991 sequences in the BAM file
## 9623991 sequences exported
## 9623991 reads; of these:
##   9623991 (100.00%) were unpaired; of these:
##     1669382 (17.35%) aligned 0 times
##     5140645 (53.41%) aligned exactly 1 time
##     2813964 (29.24%) aligned &gt;1 times
## 82.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2813964/7954609
## 2 2129281/2813964
## 3 6455137/18274202
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0554-raw_1 and brca-0554-raw_2
## Single-end reads will be in brca-0554-raw_M
## 11702419 sequences in the BAM file
## 11702419 sequences exported
## 11702419 reads; of these:
##   11702419 (100.00%) were unpaired; of these:
##     2775286 (23.72%) aligned 0 times
##     6441265 (55.04%) aligned exactly 1 time
##     2485868 (21.24%) aligned &gt;1 times
## 76.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2485868/8927133
## 2 2014506/2485868
## 3 5676895/18018357
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0555-raw_1 and brca-0555-raw_2
## Single-end reads will be in brca-0555-raw_M
## 2679233 sequences in the BAM file
## 2679233 sequences exported
## 2679233 reads; of these:
##   2679233 (100.00%) were unpaired; of these:
##     876820 (32.73%) aligned 0 times
##     997284 (37.22%) aligned exactly 1 time
##     805129 (30.05%) aligned &gt;1 times
## 67.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 805129/1802413
## 2 469869/805129
## 3 3848197/8666772
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0556-raw_1 and brca-0556-raw_2
## Single-end reads will be in brca-0556-raw_M
## 4178623 sequences in the BAM file
## 4178623 sequences exported
## 4178623 reads; of these:
##   4178623 (100.00%) were unpaired; of these:
##     376716 (9.02%) aligned 0 times
##     2257079 (54.01%) aligned exactly 1 time
##     1544828 (36.97%) aligned &gt;1 times
## 90.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1544828/3801907
## 2 1244665/1544828
## 3 5973556/12532228
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0557-raw_1 and brca-0557-raw_2
## Single-end reads will be in brca-0557-raw_M
## 17255235 sequences in the BAM file
## 17255235 sequences exported
## 17255235 reads; of these:
##   17255235 (100.00%) were unpaired; of these:
##     3519370 (20.40%) aligned 0 times
##     8489457 (49.20%) aligned exactly 1 time
##     5246408 (30.40%) aligned &gt;1 times
## 79.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5246408/13735865
## 2 4118265/5246408
## 3 11123998/30500017
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0558-raw_1 and brca-0558-raw_2
## Single-end reads will be in brca-0558-raw_M
## 2017682 sequences in the BAM file
## 2017682 sequences exported
## 2017682 reads; of these:
##   2017682 (100.00%) were unpaired; of these:
##     436038 (21.61%) aligned 0 times
##     947325 (46.95%) aligned exactly 1 time
##     634319 (31.44%) aligned &gt;1 times
## 78.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 634319/1581644
## 2 505004/634319
## 3 3169059/6509768
## This looks like paired data from lane 1.
## Output will be in brca-0559-raw_1 and brca-0559-raw_2
## Single-end reads will be in brca-0559-raw_M
## 3927661 sequences in the BAM file
## 3927661 sequences exported
## 3927661 reads; of these:
##   3927661 (100.00%) were unpaired; of these:
##     425293 (10.83%) aligned 0 times
##     2262362 (57.60%) aligned exactly 1 time
##     1240006 (31.57%) aligned &gt;1 times
## 89.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1240006/3502368
## 2 942516/1240006
## 3 7106178/14861293
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0560-raw_1 and brca-0560-raw_2
## Single-end reads will be in brca-0560-raw_M
## 3894567 sequences in the BAM file
## 3894567 sequences exported
## 3894567 reads; of these:
##   3894567 (100.00%) were unpaired; of these:
##     294768 (7.57%) aligned 0 times
##     1813261 (46.56%) aligned exactly 1 time
##     1786538 (45.87%) aligned &gt;1 times
## 92.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1786538/3599799
## 2 1480751/1786538
## 3 4651600/10167593
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0561-raw_1 and brca-0561-raw_2
## Single-end reads will be in brca-0561-raw_M
## 9429092 sequences in the BAM file
## 9429092 sequences exported
## 9429092 reads; of these:
##   9429092 (100.00%) were unpaired; of these:
##     728387 (7.72%) aligned 0 times
##     6250799 (66.29%) aligned exactly 1 time
##     2449906 (25.98%) aligned &gt;1 times
## 92.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2449906/8700705
## 2 2106078/2449906
## 3 7936336/20396259
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0562-raw_1 and brca-0562-raw_2
## Single-end reads will be in brca-0562-raw_M
## 6151933 sequences in the BAM file
## 6151933 sequences exported
## 6151933 reads; of these:
##   6151933 (100.00%) were unpaired; of these:
##     603406 (9.81%) aligned 0 times
##     3294286 (53.55%) aligned exactly 1 time
##     2254241 (36.64%) aligned &gt;1 times
## 90.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2254241/5548527
## 2 1867599/2254241
## 3 5476757/13538579
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0563-raw_1 and brca-0563-raw_2
## Single-end reads will be in brca-0563-raw_M
## 3369570 sequences in the BAM file
## 3369570 sequences exported
## 3369570 reads; of these:
##   3369570 (100.00%) were unpaired; of these:
##     762198 (22.62%) aligned 0 times
##     1436415 (42.63%) aligned exactly 1 time
##     1170957 (34.75%) aligned &gt;1 times
## 77.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1170957/2607372
## 2 958020/1170957
## 3 3613742/7718248
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0564-raw_1 and brca-0564-raw_2
## Single-end reads will be in brca-0564-raw_M
## 5846272 sequences in the BAM file
## 5846272 sequences exported
## 5846272 reads; of these:
##   5846272 (100.00%) were unpaired; of these:
##     1253419 (21.44%) aligned 0 times
##     2969254 (50.79%) aligned exactly 1 time
##     1623599 (27.77%) aligned &gt;1 times
## 78.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1623599/4592853
## 2 1193656/1623599
## 3 5104171/13285660
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0565-raw_1 and brca-0565-raw_2
## Single-end reads will be in brca-0565-raw_M
## 7882554 sequences in the BAM file
## 7882554 sequences exported
## 7882554 reads; of these:
##   7882554 (100.00%) were unpaired; of these:
##     1546741 (19.62%) aligned 0 times
##     4025529 (51.07%) aligned exactly 1 time
##     2310284 (29.31%) aligned &gt;1 times
## 80.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2310284/6335813
## 2 1852675/2310284
## 3 12161649/26019150
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0566-raw_1 and brca-0566-raw_2
## Single-end reads will be in brca-0566-raw_M
## 13800669 sequences in the BAM file
## 13800669 sequences exported
## 13800669 reads; of these:
##   13800669 (100.00%) were unpaired; of these:
##     4073785 (29.52%) aligned 0 times
##     6635106 (48.08%) aligned exactly 1 time
##     3091778 (22.40%) aligned &gt;1 times
## 70.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3091778/9726884
## 2 2071624/3091778
## 3 11396827/28175251
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0567-raw_1 and brca-0567-raw_2
## Single-end reads will be in brca-0567-raw_M
## 5693463 sequences in the BAM file
## 5693463 sequences exported
## 5693463 reads; of these:
##   5693463 (100.00%) were unpaired; of these:
##     1509424 (26.51%) aligned 0 times
##     2236138 (39.28%) aligned exactly 1 time
##     1947901 (34.21%) aligned &gt;1 times
## 73.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1947901/4184039
## 2 1525188/1947901
## 3 5056394/12025779
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0568-raw_1 and brca-0568-raw_2
## Single-end reads will be in brca-0568-raw_M
## 13175568 sequences in the BAM file
## 13175568 sequences exported
## 13175568 reads; of these:
##   13175568 (100.00%) were unpaired; of these:
##     3055726 (23.19%) aligned 0 times
##     5261317 (39.93%) aligned exactly 1 time
##     4858525 (36.88%) aligned &gt;1 times
## 76.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4858525/10119842
## 2 3532771/4858525
## 3 10435893/26765446
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-0569-raw_1 and brca-0569-raw_2
## Single-end reads will be in brca-0569-raw_M
## 3178467 sequences in the BAM file
## 3178467 sequences exported
## 3178467 reads; of these:
##   3178467 (100.00%) were unpaired; of these:
##     775333 (24.39%) aligned 0 times
##     1438107 (45.25%) aligned exactly 1 time
##     965027 (30.36%) aligned &gt;1 times
## 75.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 965027/2403134
## 2 720531/965027
## 3 7328460/14109248
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0570-raw_1 and brca-0570-raw_2
## Single-end reads will be in brca-0570-raw_M
## 2373287 sequences in the BAM file
## 2373287 sequences exported
## 2373287 reads; of these:
##   2373287 (100.00%) were unpaired; of these:
##     599216 (25.25%) aligned 0 times
##     1137615 (47.93%) aligned exactly 1 time
##     636456 (26.82%) aligned &gt;1 times
## 74.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 636456/1774071
## 2 450089/636456
## 3 3431633/8510154
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0571-raw_1 and brca-0571-raw_2
## Single-end reads will be in brca-0571-raw_M
## 2040224 sequences in the BAM file
## 2040224 sequences exported
## 2040224 reads; of these:
##   2040224 (100.00%) were unpaired; of these:
##     182202 (8.93%) aligned 0 times
##     1024079 (50.19%) aligned exactly 1 time
##     833943 (40.88%) aligned &gt;1 times
## 91.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 833943/1858022
## 2 666344/833943
## 3 3363720/7020434
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0572-raw_1 and brca-0572-raw_2
## Single-end reads will be in brca-0572-raw_M
## 4227770 sequences in the BAM file
## 4227770 sequences exported
## 4227770 reads; of these:
##   4227770 (100.00%) were unpaired; of these:
##     1024978 (24.24%) aligned 0 times
##     1609116 (38.06%) aligned exactly 1 time
##     1593676 (37.70%) aligned &gt;1 times
## 75.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1593676/3202792
## 2 1389090/1593676
## 3 5899177/11998100
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0573-raw_1 and brca-0573-raw_2
## Single-end reads will be in brca-0573-raw_M
## 7882642 sequences in the BAM file
## 7882642 sequences exported
## 7882642 reads; of these:
##   7882642 (100.00%) were unpaired; of these:
##     1979591 (25.11%) aligned 0 times
##     4171689 (52.92%) aligned exactly 1 time
##     1731362 (21.96%) aligned &gt;1 times
## 74.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1731362/5903051
## 2 1213070/1731362
## 3 6542974/16438368
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0574-raw_1 and brca-0574-raw_2
## Single-end reads will be in brca-0574-raw_M
## 6867976 sequences in the BAM file
## 6867976 sequences exported
## 6867976 reads; of these:
##   6867976 (100.00%) were unpaired; of these:
##     1450566 (21.12%) aligned 0 times
##     3980085 (57.95%) aligned exactly 1 time
##     1437325 (20.93%) aligned &gt;1 times
## 78.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1437325/5417410
## 2 1067535/1437325
## 3 5031468/13847823
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0575-raw_1 and brca-0575-raw_2
## Single-end reads will be in brca-0575-raw_M
## 6263422 sequences in the BAM file
## 6263422 sequences exported
## 6263422 reads; of these:
##   6263422 (100.00%) were unpaired; of these:
##     384226 (6.13%) aligned 0 times
##     3496701 (55.83%) aligned exactly 1 time
##     2382495 (38.04%) aligned &gt;1 times
## 93.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2382495/5879196
## 2 2023466/2382495
## 3 7682428/16760056
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0576-raw_1 and brca-0576-raw_2
## Single-end reads will be in brca-0576-raw_M
## 3404543 sequences in the BAM file
## 3404543 sequences exported
## 3404543 reads; of these:
##   3404543 (100.00%) were unpaired; of these:
##     875093 (25.70%) aligned 0 times
##     1421837 (41.76%) aligned exactly 1 time
##     1107613 (32.53%) aligned &gt;1 times
## 74.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1107613/2529450
## 2 888168/1107613
## 3 6769308/12980467
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0577-raw_1 and brca-0577-raw_2
## Single-end reads will be in brca-0577-raw_M
## 14467413 sequences in the BAM file
## 14467413 sequences exported
## 14467413 reads; of these:
##   14467413 (100.00%) were unpaired; of these:
##     2674825 (18.49%) aligned 0 times
##     6700791 (46.32%) aligned exactly 1 time
##     5091797 (35.19%) aligned &gt;1 times
## 81.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5091797/11792588
## 2 4453714/5091797
## 3 13720471/32764153
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in brca-0578-raw_1 and brca-0578-raw_2
## Single-end reads will be in brca-0578-raw_M
## 10319795 sequences in the BAM file
## 10319795 sequences exported
## 10319795 reads; of these:
##   10319795 (100.00%) were unpaired; of these:
##     3201572 (31.02%) aligned 0 times
##     4501524 (43.62%) aligned exactly 1 time
##     2616699 (25.36%) aligned &gt;1 times
## 68.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2616699/7118223
## 2 2193911/2616699
## 3 5208633/14916258
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0579-raw_1 and brca-0579-raw_2
## Single-end reads will be in brca-0579-raw_M
## 7031478 sequences in the BAM file
## 7031478 sequences exported
## 7031478 reads; of these:
##   7031478 (100.00%) were unpaired; of these:
##     2071865 (29.47%) aligned 0 times
##     3674817 (52.26%) aligned exactly 1 time
##     1284796 (18.27%) aligned &gt;1 times
## 70.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1284796/4959613
## 2 935157/1284796
## 3 3979523/11575413
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0580-raw_1 and brca-0580-raw_2
## Single-end reads will be in brca-0580-raw_M
## 7532521 sequences in the BAM file
## 7532521 sequences exported
## 7532521 reads; of these:
##   7532521 (100.00%) were unpaired; of these:
##     1539619 (20.44%) aligned 0 times
##     3766440 (50.00%) aligned exactly 1 time
##     2226462 (29.56%) aligned &gt;1 times
## 79.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2226462/5992902
## 2 1674404/2226462
## 3 5750040/14800120
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0581-raw_1 and brca-0581-raw_2
## Single-end reads will be in brca-0581-raw_M
## 3797356 sequences in the BAM file
## 3797356 sequences exported
## 3797356 reads; of these:
##   3797356 (100.00%) were unpaired; of these:
##     676940 (17.83%) aligned 0 times
##     1514396 (39.88%) aligned exactly 1 time
##     1606020 (42.29%) aligned &gt;1 times
## 82.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1606020/3120416
## 2 1263395/1606020
## 3 14470016/27338118
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0582-raw_1 and brca-0582-raw_2
## Single-end reads will be in brca-0582-raw_M
## 9127406 sequences in the BAM file
## 9127406 sequences exported
## 9127406 reads; of these:
##   9127406 (100.00%) were unpaired; of these:
##     2978909 (32.64%) aligned 0 times
##     3490116 (38.24%) aligned exactly 1 time
##     2658381 (29.13%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2658381/6148497
## 2 2150951/2658381
## 3 5698516/14607565
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0583-raw_1 and brca-0583-raw_2
## Single-end reads will be in brca-0583-raw_M
## 2985657 sequences in the BAM file
## 2985657 sequences exported
## 2985657 reads; of these:
##   2985657 (100.00%) were unpaired; of these:
##     619828 (20.76%) aligned 0 times
##     1512168 (50.65%) aligned exactly 1 time
##     853661 (28.59%) aligned &gt;1 times
## 79.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 853661/2365829
## 2 682809/853661
## 3 3534587/8441267
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0584-raw_1 and brca-0584-raw_2
## Single-end reads will be in brca-0584-raw_M
## 3865056 sequences in the BAM file
## 3865056 sequences exported
## 3865056 reads; of these:
##   3865056 (100.00%) were unpaired; of these:
##     1120908 (29.00%) aligned 0 times
##     1383641 (35.80%) aligned exactly 1 time
##     1360507 (35.20%) aligned &gt;1 times
## 71.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1360507/2744148
## 2 968935/1360507
## 3 6387075/14281131
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0585-raw_1 and brca-0585-raw_2
## Single-end reads will be in brca-0585-raw_M
## 4143617 sequences in the BAM file
## 4143617 sequences exported
## 4143617 reads; of these:
##   4143617 (100.00%) were unpaired; of these:
##     995315 (24.02%) aligned 0 times
##     2006167 (48.42%) aligned exactly 1 time
##     1142135 (27.56%) aligned &gt;1 times
## 75.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1142135/3148302
## 2 917556/1142135
## 3 4487791/10046321
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0586-raw_1 and brca-0586-raw_2
## Single-end reads will be in brca-0586-raw_M
## 9500687 sequences in the BAM file
## 9500687 sequences exported
## 9500687 reads; of these:
##   9500687 (100.00%) were unpaired; of these:
##     2035585 (21.43%) aligned 0 times
##     4953262 (52.14%) aligned exactly 1 time
##     2511840 (26.44%) aligned &gt;1 times
## 78.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2511840/7465102
## 2 1749824/2511840
## 3 11310959/28661235
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0587-raw_1 and brca-0587-raw_2
## Single-end reads will be in brca-0587-raw_M
## 12798053 sequences in the BAM file
## 12798053 sequences exported
## 12798053 reads; of these:
##   12798053 (100.00%) were unpaired; of these:
##     2540572 (19.85%) aligned 0 times
##     6286583 (49.12%) aligned exactly 1 time
##     3970898 (31.03%) aligned &gt;1 times
## 80.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3970898/10257481
## 2 3123848/3970898
## 3 11646464/28227065
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0588-raw_1 and brca-0588-raw_2
## Single-end reads will be in brca-0588-raw_M
## 9224385 sequences in the BAM file
## 9224385 sequences exported
## 9224385 reads; of these:
##   9224385 (100.00%) were unpaired; of these:
##     1920185 (20.82%) aligned 0 times
##     4263459 (46.22%) aligned exactly 1 time
##     3040741 (32.96%) aligned &gt;1 times
## 79.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3040741/7304200
## 2 2183420/3040741
## 3 18744351/36917834
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0589-raw_1 and brca-0589-raw_2
## Single-end reads will be in brca-0589-raw_M
## 6399117 sequences in the BAM file
## 6399117 sequences exported
## 6399117 reads; of these:
##   6399117 (100.00%) were unpaired; of these:
##     680261 (10.63%) aligned 0 times
##     3057748 (47.78%) aligned exactly 1 time
##     2661108 (41.59%) aligned &gt;1 times
## 89.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2661108/5718856
## 2 2208287/2661108
## 3 13220571/26804952
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0590-raw_1 and brca-0590-raw_2
## Single-end reads will be in brca-0590-raw_M
## 4532592 sequences in the BAM file
## 4532592 sequences exported
## 4532592 reads; of these:
##   4532592 (100.00%) were unpaired; of these:
##     1160140 (25.60%) aligned 0 times
##     1989732 (43.90%) aligned exactly 1 time
##     1382720 (30.51%) aligned &gt;1 times
## 74.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1382720/3372452
## 2 1144895/1382720
## 3 4009729/9508591
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0591-raw_1 and brca-0591-raw_2
## Single-end reads will be in brca-0591-raw_M
## 2605812 sequences in the BAM file
## 2605812 sequences exported
## 2605812 reads; of these:
##   2605812 (100.00%) were unpaired; of these:
##     553269 (21.23%) aligned 0 times
##     1206121 (46.29%) aligned exactly 1 time
##     846422 (32.48%) aligned &gt;1 times
## 78.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 846422/2052543
## 2 641073/846422
## 3 5263253/10914934
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0592-raw_1 and brca-0592-raw_2
## Single-end reads will be in brca-0592-raw_M
## 2102873 sequences in the BAM file
## 2102873 sequences exported
## 2102873 reads; of these:
##   2102873 (100.00%) were unpaired; of these:
##     522955 (24.87%) aligned 0 times
##     1007164 (47.89%) aligned exactly 1 time
##     572754 (27.24%) aligned &gt;1 times
## 75.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 572754/1579918
## 2 454495/572754
## 3 1962114/5008712
## This looks like paired data from lane 6.
## Output will be in brca-0593-raw_1 and brca-0593-raw_2
## Single-end reads will be in brca-0593-raw_M
## 2934940 sequences in the BAM file
## 2934940 sequences exported
## 2934940 reads; of these:
##   2934940 (100.00%) were unpaired; of these:
##     565269 (19.26%) aligned 0 times
##     1444131 (49.20%) aligned exactly 1 time
##     925540 (31.54%) aligned &gt;1 times
## 80.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 925540/2369671
## 2 751923/925540
## 3 2970209/6807667
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0594-raw_1 and brca-0594-raw_2
## Single-end reads will be in brca-0594-raw_M
## 12102499 sequences in the BAM file
## 12102499 sequences exported
## 12102499 reads; of these:
##   12102499 (100.00%) were unpaired; of these:
##     3202822 (26.46%) aligned 0 times
##     6888360 (56.92%) aligned exactly 1 time
##     2011317 (16.62%) aligned &gt;1 times
## 73.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2011317/8899677
## 2 1566929/2011317
## 3 4447564/15953102
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0595-raw_1 and brca-0595-raw_2
## Single-end reads will be in brca-0595-raw_M
## 5620268 sequences in the BAM file
## 5620268 sequences exported
## 5620268 reads; of these:
##   5620268 (100.00%) were unpaired; of these:
##     513094 (9.13%) aligned 0 times
##     3160314 (56.23%) aligned exactly 1 time
##     1946860 (34.64%) aligned &gt;1 times
## 90.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1946860/5107174
## 2 1520032/1946860
## 3 8049420/17251072
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0596-raw_1 and brca-0596-raw_2
## Single-end reads will be in brca-0596-raw_M
## 3257041 sequences in the BAM file
## 3257041 sequences exported
## 3257041 reads; of these:
##   3257041 (100.00%) were unpaired; of these:
##     942990 (28.95%) aligned 0 times
##     1420998 (43.63%) aligned exactly 1 time
##     893053 (27.42%) aligned &gt;1 times
## 71.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 893053/2314051
## 2 659437/893053
## 3 6009658/11678377
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0597-raw_1 and brca-0597-raw_2
## Single-end reads will be in brca-0597-raw_M
## 3570186 sequences in the BAM file
## 3570186 sequences exported
## 3570186 reads; of these:
##   3570186 (100.00%) were unpaired; of these:
##     365577 (10.24%) aligned 0 times
##     2093278 (58.63%) aligned exactly 1 time
##     1111331 (31.13%) aligned &gt;1 times
## 89.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1111331/3204609
## 2 836981/1111331
## 3 7371034/14685585
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0598-raw_1 and brca-0598-raw_2
## Single-end reads will be in brca-0598-raw_M
## 2033350 sequences in the BAM file
## 2033350 sequences exported
## 2033350 reads; of these:
##   2033350 (100.00%) were unpaired; of these:
##     483146 (23.76%) aligned 0 times
##     1133457 (55.74%) aligned exactly 1 time
##     416747 (20.50%) aligned &gt;1 times
## 76.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 416747/1550204
## 2 314822/416747
## 3 1663810/4510274
## This looks like paired data from lane 3.
## Output will be in brca-0599-raw_1 and brca-0599-raw_2
## Single-end reads will be in brca-0599-raw_M
## 2605092 sequences in the BAM file
## 2605092 sequences exported
## 2605092 reads; of these:
##   2605092 (100.00%) were unpaired; of these:
##     549139 (21.08%) aligned 0 times
##     1056713 (40.56%) aligned exactly 1 time
##     999240 (38.36%) aligned &gt;1 times
## 78.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 999240/2055953
## 2 891570/999240
## 3 2759963/6412322
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0600-raw_1 and brca-0600-raw_2
## Single-end reads will be in brca-0600-raw_M
## 13146609 sequences in the BAM file
## 13146609 sequences exported
## 13146609 reads; of these:
##   13146609 (100.00%) were unpaired; of these:
##     2460781 (18.72%) aligned 0 times
##     6552108 (49.84%) aligned exactly 1 time
##     4133720 (31.44%) aligned &gt;1 times
## 81.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4133720/10685828
## 2 3342230/4133720
## 3 9463944/25538958
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0601-raw_1 and brca-0601-raw_2
## Single-end reads will be in brca-0601-raw_M
## 8886128 sequences in the BAM file
## 8886128 sequences exported
## 8886128 reads; of these:
##   8886128 (100.00%) were unpaired; of these:
##     2128906 (23.96%) aligned 0 times
##     3761949 (42.34%) aligned exactly 1 time
##     2995273 (33.71%) aligned &gt;1 times
## 76.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2995273/6757222
## 2 2461914/2995273
## 3 9164248/23049377
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0602-raw_1 and brca-0602-raw_2
## Single-end reads will be in brca-0602-raw_M
## 7918862 sequences in the BAM file
## 7918862 sequences exported
## 7918862 reads; of these:
##   7918862 (100.00%) were unpaired; of these:
##     2391714 (30.20%) aligned 0 times
##     3030644 (38.27%) aligned exactly 1 time
##     2496504 (31.53%) aligned &gt;1 times
## 69.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2496504/5527148
## 2 2041849/2496504
## 3 5432199/13791638
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0603-raw_1 and brca-0603-raw_2
## Single-end reads will be in brca-0603-raw_M
## 4066421 sequences in the BAM file
## 4066421 sequences exported
## 4066421 reads; of these:
##   4066421 (100.00%) were unpaired; of these:
##     1230396 (30.26%) aligned 0 times
##     1808491 (44.47%) aligned exactly 1 time
##     1027534 (25.27%) aligned &gt;1 times
## 69.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1027534/2836025
## 2 828915/1027534
## 3 4181744/8898806
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0604-raw_1 and brca-0604-raw_2
## Single-end reads will be in brca-0604-raw_M
## 15099415 sequences in the BAM file
## 15099415 sequences exported
## 15099415 reads; of these:
##   15099415 (100.00%) were unpaired; of these:
##     2514203 (16.65%) aligned 0 times
##     6783098 (44.92%) aligned exactly 1 time
##     5802114 (38.43%) aligned &gt;1 times
## 83.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5802114/12585212
## 2 4788901/5802114
## 3 10318743/27853345
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0605-raw_1 and brca-0605-raw_2
## Single-end reads will be in brca-0605-raw_M
## 2450259 sequences in the BAM file
## 2450259 sequences exported
## 2450259 reads; of these:
##   2450259 (100.00%) were unpaired; of these:
##     571032 (23.30%) aligned 0 times
##     1038388 (42.38%) aligned exactly 1 time
##     840839 (34.32%) aligned &gt;1 times
## 76.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 840839/1879227
## 2 651699/840839
## 3 3769398/8644762
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0606-raw_1 and brca-0606-raw_2
## Single-end reads will be in brca-0606-raw_M
## 6942591 sequences in the BAM file
## 6942591 sequences exported
## 6942591 reads; of these:
##   6942591 (100.00%) were unpaired; of these:
##     1263765 (18.20%) aligned 0 times
##     2228805 (32.10%) aligned exactly 1 time
##     3450021 (49.69%) aligned &gt;1 times
## 81.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3450021/5678826
## 2 3163630/3450021
## 3 8031047/16380728
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0607-raw_1 and brca-0607-raw_2
## Single-end reads will be in brca-0607-raw_M
## 27805151 sequences in the BAM file
## 27805151 sequences exported
## 27805151 reads; of these:
##   27805151 (100.00%) were unpaired; of these:
##     7310743 (26.29%) aligned 0 times
##     13336170 (47.96%) aligned exactly 1 time
##     7158238 (25.74%) aligned &gt;1 times
## 73.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7158238/20494408
## 2 5638009/7158238
## 3 13606449/41030317
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in brca-0608-raw_1 and brca-0608-raw_2
## Single-end reads will be in brca-0608-raw_M
## 8918311 sequences in the BAM file
## 8918311 sequences exported
## 8918311 reads; of these:
##   8918311 (100.00%) were unpaired; of these:
##     3123136 (35.02%) aligned 0 times
##     3517188 (39.44%) aligned exactly 1 time
##     2277987 (25.54%) aligned &gt;1 times
## 64.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2277987/5795175
## 2 1882767/2277987
## 3 6917682/16312167
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0609-raw_1 and brca-0609-raw_2
## Single-end reads will be in brca-0609-raw_M
## 6014650 sequences in the BAM file
## 6014650 sequences exported
## 6014650 reads; of these:
##   6014650 (100.00%) were unpaired; of these:
##     836980 (13.92%) aligned 0 times
##     3035098 (50.46%) aligned exactly 1 time
##     2142572 (35.62%) aligned &gt;1 times
## 86.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2142572/5177670
## 2 1831238/2142572
## 3 7606459/16397682
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0610-raw_1 and brca-0610-raw_2
## Single-end reads will be in brca-0610-raw_M
## 2045753 sequences in the BAM file
## 2045753 sequences exported
## 2045753 reads; of these:
##   2045753 (100.00%) were unpaired; of these:
##     535578 (26.18%) aligned 0 times
##     750246 (36.67%) aligned exactly 1 time
##     759929 (37.15%) aligned &gt;1 times
## 73.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 759929/1510175
## 2 573573/759929
## 3 3849430/8209727
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0611-raw_1 and brca-0611-raw_2
## Single-end reads will be in brca-0611-raw_M
## 5231944 sequences in the BAM file
## 5231944 sequences exported
## 5231944 reads; of these:
##   5231944 (100.00%) were unpaired; of these:
##     1217427 (23.27%) aligned 0 times
##     2017346 (38.56%) aligned exactly 1 time
##     1997171 (38.17%) aligned &gt;1 times
## 76.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1997171/4014517
## 2 1643337/1997171
## 3 8662602/18020962
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0612-raw_1 and brca-0612-raw_2
## Single-end reads will be in brca-0612-raw_M
## 3178756 sequences in the BAM file
## 3178756 sequences exported
## 3178756 reads; of these:
##   3178756 (100.00%) were unpaired; of these:
##     649898 (20.45%) aligned 0 times
##     1695706 (53.34%) aligned exactly 1 time
##     833152 (26.21%) aligned &gt;1 times
## 79.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 833152/2528858
## 2 589345/833152
## 3 2000598/5935954
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0613-raw_1 and brca-0613-raw_2
## Single-end reads will be in brca-0613-raw_M
## 4046879 sequences in the BAM file
## 4046879 sequences exported
## 4046879 reads; of these:
##   4046879 (100.00%) were unpaired; of these:
##     591348 (14.61%) aligned 0 times
##     2193163 (54.19%) aligned exactly 1 time
##     1262368 (31.19%) aligned &gt;1 times
## 85.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1262368/3455531
## 2 1044010/1262368
## 3 5556686/12189884
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0614-raw_1 and brca-0614-raw_2
## Single-end reads will be in brca-0614-raw_M
## 11515940 sequences in the BAM file
## 11515940 sequences exported
## 11515940 reads; of these:
##   11515940 (100.00%) were unpaired; of these:
##     3496531 (30.36%) aligned 0 times
##     6280815 (54.54%) aligned exactly 1 time
##     1738594 (15.10%) aligned &gt;1 times
## 69.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1738594/8019409
## 2 1321393/1738594
## 3 4115952/14793738
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0615-raw_1 and brca-0615-raw_2
## Single-end reads will be in brca-0615-raw_M
## 2010447 sequences in the BAM file
## 2010447 sequences exported
## 2010447 reads; of these:
##   2010447 (100.00%) were unpaired; of these:
##     481776 (23.96%) aligned 0 times
##     984508 (48.97%) aligned exactly 1 time
##     544163 (27.07%) aligned &gt;1 times
## 76.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 544163/1528671
## 2 448473/544163
## 3 1446187/3914316
## This looks like paired data from lane 7.
## Output will be in brca-0616-raw_1 and brca-0616-raw_2
## Single-end reads will be in brca-0616-raw_M
## 4567053 sequences in the BAM file
## 4567053 sequences exported
## 4567053 reads; of these:
##   4567053 (100.00%) were unpaired; of these:
##     452296 (9.90%) aligned 0 times
##     2076900 (45.48%) aligned exactly 1 time
##     2037857 (44.62%) aligned &gt;1 times
## 90.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2037857/4114757
## 2 1776873/2037857
## 3 4903680/11149330
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0617-raw_1 and brca-0617-raw_2
## Single-end reads will be in brca-0617-raw_M
## 1632157 sequences in the BAM file
## 1632157 sequences exported
## 1632157 reads; of these:
##   1632157 (100.00%) were unpaired; of these:
##     216650 (13.27%) aligned 0 times
##     806984 (49.44%) aligned exactly 1 time
##     608523 (37.28%) aligned &gt;1 times
## 86.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 608523/1415507
## 2 459323/608523
## 3 3217566/6473408
## This looks like paired data from lane 1.
## Output will be in brca-0618-raw_1 and brca-0618-raw_2
## Single-end reads will be in brca-0618-raw_M
## 3641035 sequences in the BAM file
## 3641035 sequences exported
## 3641035 reads; of these:
##   3641035 (100.00%) were unpaired; of these:
##     772615 (21.22%) aligned 0 times
##     1819214 (49.96%) aligned exactly 1 time
##     1049206 (28.82%) aligned &gt;1 times
## 78.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1049206/2868420
## 2 748634/1049206
## 3 5400743/12484639
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0619-raw_1 and brca-0619-raw_2
## Single-end reads will be in brca-0619-raw_M
## 8139936 sequences in the BAM file
## 8139936 sequences exported
## 8139936 reads; of these:
##   8139936 (100.00%) were unpaired; of these:
##     812158 (9.98%) aligned 0 times
##     4940736 (60.70%) aligned exactly 1 time
##     2387042 (29.33%) aligned &gt;1 times
## 90.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2387042/7327778
## 2 1893989/2387042
## 3 10066288/22476224
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0620-raw_1 and brca-0620-raw_2
## Single-end reads will be in brca-0620-raw_M
## 2118092 sequences in the BAM file
## 2118092 sequences exported
## 2118092 reads; of these:
##   2118092 (100.00%) were unpaired; of these:
##     381941 (18.03%) aligned 0 times
##     895344 (42.27%) aligned exactly 1 time
##     840807 (39.70%) aligned &gt;1 times
## 81.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 840807/1736151
## 2 547393/840807
## 3 5363677/10660250
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0621-raw_1 and brca-0621-raw_2
## Single-end reads will be in brca-0621-raw_M
## 3457139 sequences in the BAM file
## 3457139 sequences exported
## 3457139 reads; of these:
##   3457139 (100.00%) were unpaired; of these:
##     1073713 (31.06%) aligned 0 times
##     1344360 (38.89%) aligned exactly 1 time
##     1039066 (30.06%) aligned &gt;1 times
## 68.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1039066/2383426
## 2 761336/1039066
## 3 8534385/14538387
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0622-raw_1 and brca-0622-raw_2
## Single-end reads will be in brca-0622-raw_M
## 4379724 sequences in the BAM file
## 4379724 sequences exported
## 4379724 reads; of these:
##   4379724 (100.00%) were unpaired; of these:
##     1098622 (25.08%) aligned 0 times
##     1947780 (44.47%) aligned exactly 1 time
##     1333322 (30.44%) aligned &gt;1 times
## 74.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1333322/3281102
## 2 940457/1333322
## 3 9067525/19000425
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0623-raw_1 and brca-0623-raw_2
## Single-end reads will be in brca-0623-raw_M
## 3534260 sequences in the BAM file
## 3534260 sequences exported
## 3534260 reads; of these:
##   3534260 (100.00%) were unpaired; of these:
##     335987 (9.51%) aligned 0 times
##     2186500 (61.87%) aligned exactly 1 time
##     1011773 (28.63%) aligned &gt;1 times
## 90.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1011773/3198273
## 2 702109/1011773
## 3 6886627/14227808
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0624-raw_1 and brca-0624-raw_2
## Single-end reads will be in brca-0624-raw_M
## 2539295 sequences in the BAM file
## 2539295 sequences exported
## 2539295 reads; of these:
##   2539295 (100.00%) were unpaired; of these:
##     278427 (10.96%) aligned 0 times
##     942403 (37.11%) aligned exactly 1 time
##     1318465 (51.92%) aligned &gt;1 times
## 89.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1318465/2260868
## 2 1056465/1318465
## 3 14601386/27970639
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0625-raw_1 and brca-0625-raw_2
## Single-end reads will be in brca-0625-raw_M
## 10184908 sequences in the BAM file
## 10184908 sequences exported
## 10184908 reads; of these:
##   10184908 (100.00%) were unpaired; of these:
##     1856715 (18.23%) aligned 0 times
##     6184408 (60.72%) aligned exactly 1 time
##     2143785 (21.05%) aligned &gt;1 times
## 81.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2143785/8328193
## 2 1159848/2143785
## 3 7766370/21486921
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0626-raw_1 and brca-0626-raw_2
## Single-end reads will be in brca-0626-raw_M
## 4583382 sequences in the BAM file
## 4583382 sequences exported
## 4583382 reads; of these:
##   4583382 (100.00%) were unpaired; of these:
##     1277035 (27.86%) aligned 0 times
##     1950225 (42.55%) aligned exactly 1 time
##     1356122 (29.59%) aligned &gt;1 times
## 72.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1356122/3306347
## 2 998524/1356122
## 3 7755475/16638780
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0627-raw_1 and brca-0627-raw_2
## Single-end reads will be in brca-0627-raw_M
## 8837818 sequences in the BAM file
## 8837818 sequences exported
## 8837818 reads; of these:
##   8837818 (100.00%) were unpaired; of these:
##     2580592 (29.20%) aligned 0 times
##     3141101 (35.54%) aligned exactly 1 time
##     3116125 (35.26%) aligned &gt;1 times
## 70.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3116125/6257226
## 2 2486163/3116125
## 3 36649990/71290511
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 7.
## Output will be in brca-0628-raw_1 and brca-0628-raw_2
## Single-end reads will be in brca-0628-raw_M
## 13508864 sequences in the BAM file
## 13508864 sequences exported
## 13508864 reads; of these:
##   13508864 (100.00%) were unpaired; of these:
##     2403986 (17.80%) aligned 0 times
##     5289407 (39.16%) aligned exactly 1 time
##     5815471 (43.05%) aligned &gt;1 times
## 82.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5815471/11104878
## 2 4367789/5815471
## 3 33338476/72476771
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 4.
## Output will be in brca-0629-raw_1 and brca-0629-raw_2
## Single-end reads will be in brca-0629-raw_M
## 10075113 sequences in the BAM file
## 10075113 sequences exported
## 10075113 reads; of these:
##   10075113 (100.00%) were unpaired; of these:
##     1925904 (19.12%) aligned 0 times
##     4617833 (45.83%) aligned exactly 1 time
##     3531376 (35.05%) aligned &gt;1 times
## 80.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3531376/8149209
## 2 2934718/3531376
## 3 10014195/23489736
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0630-raw_1 and brca-0630-raw_2
## Single-end reads will be in brca-0630-raw_M
## 18480197 sequences in the BAM file
## 18480197 sequences exported
## 18480197 reads; of these:
##   18480197 (100.00%) were unpaired; of these:
##     4921050 (26.63%) aligned 0 times
##     8975123 (48.57%) aligned exactly 1 time
##     4584024 (24.81%) aligned &gt;1 times
## 73.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4584024/13559147
## 2 3362530/4584024
## 3 12873665/34026368
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0631-raw_1 and brca-0631-raw_2
## Single-end reads will be in brca-0631-raw_M
## 3026153 sequences in the BAM file
## 3026153 sequences exported
## 3026153 reads; of these:
##   3026153 (100.00%) were unpaired; of these:
##     809310 (26.74%) aligned 0 times
##     1181271 (39.04%) aligned exactly 1 time
##     1035572 (34.22%) aligned &gt;1 times
## 73.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1035572/2216843
## 2 889075/1035572
## 3 4337520/9556896
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0632-raw_1 and brca-0632-raw_2
## Single-end reads will be in brca-0632-raw_M
## 2044055 sequences in the BAM file
## 2044055 sequences exported
## 2044055 reads; of these:
##   2044055 (100.00%) were unpaired; of these:
##     147034 (7.19%) aligned 0 times
##     1219316 (59.65%) aligned exactly 1 time
##     677705 (33.15%) aligned &gt;1 times
## 92.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 677705/1897021
## 2 527121/677705
## 3 3579938/7358636
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0633-raw_1 and brca-0633-raw_2
## Single-end reads will be in brca-0633-raw_M
## 7068047 sequences in the BAM file
## 7068047 sequences exported
## 7068047 reads; of these:
##   7068047 (100.00%) were unpaired; of these:
##     628275 (8.89%) aligned 0 times
##     4146075 (58.66%) aligned exactly 1 time
##     2293697 (32.45%) aligned &gt;1 times
## 91.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2293697/6439772
## 2 1831043/2293697
## 3 6245834/15711015
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0634-raw_1 and brca-0634-raw_2
## Single-end reads will be in brca-0634-raw_M
## 3994937 sequences in the BAM file
## 3994937 sequences exported
## 3994937 reads; of these:
##   3994937 (100.00%) were unpaired; of these:
##     405004 (10.14%) aligned 0 times
##     2277395 (57.01%) aligned exactly 1 time
##     1312538 (32.86%) aligned &gt;1 times
## 89.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1312538/3589933
## 2 1014414/1312538
## 3 6431166/13183813
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0635-raw_1 and brca-0635-raw_2
## Single-end reads will be in brca-0635-raw_M
## 13401481 sequences in the BAM file
## 13401481 sequences exported
## 13401481 reads; of these:
##   13401481 (100.00%) were unpaired; of these:
##     2317300 (17.29%) aligned 0 times
##     6615297 (49.36%) aligned exactly 1 time
##     4468884 (33.35%) aligned &gt;1 times
## 82.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4468884/11084181
## 2 3567669/4468884
## 3 10649927/27761517
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0636-raw_1 and brca-0636-raw_2
## Single-end reads will be in brca-0636-raw_M
## 20644587 sequences in the BAM file
## 20644587 sequences exported
## 20644587 reads; of these:
##   20644587 (100.00%) were unpaired; of these:
##     4672268 (22.63%) aligned 0 times
##     10286595 (49.83%) aligned exactly 1 time
##     5685724 (27.54%) aligned &gt;1 times
## 77.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5685724/15972319
## 2 4651578/5685724
## 3 19862644/48645836
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in brca-0637-raw_1 and brca-0637-raw_2
## Single-end reads will be in brca-0637-raw_M
## 2425118 sequences in the BAM file
## 2425118 sequences exported
## 2425118 reads; of these:
##   2425118 (100.00%) were unpaired; of these:
##     321636 (13.26%) aligned 0 times
##     1225421 (50.53%) aligned exactly 1 time
##     878061 (36.21%) aligned &gt;1 times
## 86.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 878061/2103482
## 2 693117/878061
## 3 4170082/8557245
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0638-raw_1 and brca-0638-raw_2
## Single-end reads will be in brca-0638-raw_M
## 3915046 sequences in the BAM file
## 3915046 sequences exported
## 3915046 reads; of these:
##   3915046 (100.00%) were unpaired; of these:
##     1011194 (25.83%) aligned 0 times
##     1743894 (44.54%) aligned exactly 1 time
##     1159958 (29.63%) aligned &gt;1 times
## 74.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1159958/2903852
## 2 931284/1159958
## 3 5274906/10818806
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0639-raw_1 and brca-0639-raw_2
## Single-end reads will be in brca-0639-raw_M
## 9537430 sequences in the BAM file
## 9537430 sequences exported
## 9537430 reads; of these:
##   9537430 (100.00%) were unpaired; of these:
##     2623163 (27.50%) aligned 0 times
##     4533890 (47.54%) aligned exactly 1 time
##     2380377 (24.96%) aligned &gt;1 times
## 72.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2380377/6914267
## 2 1912304/2380377
## 3 8708563/22645893
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0640-raw_1 and brca-0640-raw_2
## Single-end reads will be in brca-0640-raw_M
## 5604054 sequences in the BAM file
## 5604054 sequences exported
## 5604054 reads; of these:
##   5604054 (100.00%) were unpaired; of these:
##     1387868 (24.77%) aligned 0 times
##     2560646 (45.69%) aligned exactly 1 time
##     1655540 (29.54%) aligned &gt;1 times
## 75.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1655540/4216186
## 2 1289738/1655540
## 3 8447830/17314122
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0641-raw_1 and brca-0641-raw_2
## Single-end reads will be in brca-0641-raw_M
## 2985208 sequences in the BAM file
## 2985208 sequences exported
## 2985208 reads; of these:
##   2985208 (100.00%) were unpaired; of these:
##     786784 (26.36%) aligned 0 times
##     1345049 (45.06%) aligned exactly 1 time
##     853375 (28.59%) aligned &gt;1 times
## 73.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 853375/2198424
## 2 669357/853375
## 3 3583868/7778274
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0642-raw_1 and brca-0642-raw_2
## Single-end reads will be in brca-0642-raw_M
## 4156681 sequences in the BAM file
## 4156681 sequences exported
## 4156681 reads; of these:
##   4156681 (100.00%) were unpaired; of these:
##     313108 (7.53%) aligned 0 times
##     2503189 (60.22%) aligned exactly 1 time
##     1340384 (32.25%) aligned &gt;1 times
## 92.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1340384/3843573
## 2 1107536/1340384
## 3 3429236/8797229
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0643-raw_1 and brca-0643-raw_2
## Single-end reads will be in brca-0643-raw_M
## 3250674 sequences in the BAM file
## 3250674 sequences exported
## 3250674 reads; of these:
##   3250674 (100.00%) were unpaired; of these:
##     664422 (20.44%) aligned 0 times
##     1526258 (46.95%) aligned exactly 1 time
##     1059994 (32.61%) aligned &gt;1 times
## 79.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1059994/2586252
## 2 880242/1059994
## 3 4349893/9623049
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0644-raw_1 and brca-0644-raw_2
## Single-end reads will be in brca-0644-raw_M
## 8616150 sequences in the BAM file
## 8616150 sequences exported
## 8616150 reads; of these:
##   8616150 (100.00%) were unpaired; of these:
##     681804 (7.91%) aligned 0 times
##     4791685 (55.61%) aligned exactly 1 time
##     3142661 (36.47%) aligned &gt;1 times
## 92.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3142661/7934346
## 2 2526791/3142661
## 3 16039215/33927287
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-0645-raw_1 and brca-0645-raw_2
## Single-end reads will be in brca-0645-raw_M
## 5253169 sequences in the BAM file
## 5253169 sequences exported
## 5253169 reads; of these:
##   5253169 (100.00%) were unpaired; of these:
##     965361 (18.38%) aligned 0 times
##     1737969 (33.08%) aligned exactly 1 time
##     2549839 (48.54%) aligned &gt;1 times
## 81.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2549839/4287808
## 2 1876264/2549839
## 3 16066616/39468080
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in brca-0646-raw_1 and brca-0646-raw_2
## Single-end reads will be in brca-0646-raw_M
## 12489339 sequences in the BAM file
## 12489339 sequences exported
## 12489339 reads; of these:
##   12489339 (100.00%) were unpaired; of these:
##     2591947 (20.75%) aligned 0 times
##     6251753 (50.06%) aligned exactly 1 time
##     3645639 (29.19%) aligned &gt;1 times
## 79.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3645639/9897392
## 2 2394303/3645639
## 3 7093988/21577084
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0647-raw_1 and brca-0647-raw_2
## Single-end reads will be in brca-0647-raw_M
## 2425880 sequences in the BAM file
## 2425880 sequences exported
## 2425880 reads; of these:
##   2425880 (100.00%) were unpaired; of these:
##     595429 (24.54%) aligned 0 times
##     979110 (40.36%) aligned exactly 1 time
##     851341 (35.09%) aligned &gt;1 times
## 75.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 851341/1830451
## 2 714828/851341
## 3 2431608/5766407
## This looks like paired data from lane 2.
## Output will be in brca-0648-raw_1 and brca-0648-raw_2
## Single-end reads will be in brca-0648-raw_M
## 2398526 sequences in the BAM file
## 2398526 sequences exported
## 2398526 reads; of these:
##   2398526 (100.00%) were unpaired; of these:
##     744561 (31.04%) aligned 0 times
##     741978 (30.93%) aligned exactly 1 time
##     911987 (38.02%) aligned &gt;1 times
## 68.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 911987/1653965
## 2 759378/911987
## 3 3997995/8109546
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0649-raw_1 and brca-0649-raw_2
## Single-end reads will be in brca-0649-raw_M
## 12862020 sequences in the BAM file
## 12862020 sequences exported
## 12862020 reads; of these:
##   12862020 (100.00%) were unpaired; of these:
##     2842639 (22.10%) aligned 0 times
##     4616528 (35.89%) aligned exactly 1 time
##     5402853 (42.01%) aligned &gt;1 times
## 77.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5402853/10019381
## 2 4019911/5402853
## 3 31607651/66854273
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 2.
## Output will be in brca-0650-raw_1 and brca-0650-raw_2
## Single-end reads will be in brca-0650-raw_M
## 6714613 sequences in the BAM file
## 6714613 sequences exported
## 6714613 reads; of these:
##   6714613 (100.00%) were unpaired; of these:
##     2279420 (33.95%) aligned 0 times
##     2816584 (41.95%) aligned exactly 1 time
##     1618609 (24.11%) aligned &gt;1 times
## 66.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1618609/4435193
## 2 1265197/1618609
## 3 3685197/10092066
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0651-raw_1 and brca-0651-raw_2
## Single-end reads will be in brca-0651-raw_M
## 13891729 sequences in the BAM file
## 13891729 sequences exported
## 13891729 reads; of these:
##   13891729 (100.00%) were unpaired; of these:
##     1899744 (13.68%) aligned 0 times
##     7881915 (56.74%) aligned exactly 1 time
##     4110070 (29.59%) aligned &gt;1 times
## 86.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4110070/11991985
## 2 2827872/4110070
## 3 18339798/42835356
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in brca-0652-raw_1 and brca-0652-raw_2
## Single-end reads will be in brca-0652-raw_M
## 7603579 sequences in the BAM file
## 7603579 sequences exported
## 7603579 reads; of these:
##   7603579 (100.00%) were unpaired; of these:
##     1865072 (24.53%) aligned 0 times
##     4441489 (58.41%) aligned exactly 1 time
##     1297018 (17.06%) aligned &gt;1 times
## 75.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1297018/5738507
## 2 999063/1297018
## 3 3453499/11422262
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0653-raw_1 and brca-0653-raw_2
## Single-end reads will be in brca-0653-raw_M
## 15731922 sequences in the BAM file
## 15731922 sequences exported
## 15731922 reads; of these:
##   15731922 (100.00%) were unpaired; of these:
##     4160954 (26.45%) aligned 0 times
##     5931893 (37.71%) aligned exactly 1 time
##     5639075 (35.84%) aligned &gt;1 times
## 73.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5639075/11570968
## 2 4783462/5639075
## 3 13440167/30963101
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-0654-raw_1 and brca-0654-raw_2
## Single-end reads will be in brca-0654-raw_M
## 2707700 sequences in the BAM file
## 2707700 sequences exported
## 2707700 reads; of these:
##   2707700 (100.00%) were unpaired; of these:
##     654826 (24.18%) aligned 0 times
##     1018569 (37.62%) aligned exactly 1 time
##     1034305 (38.20%) aligned &gt;1 times
## 75.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1034305/2052874
## 2 801632/1034305
## 3 4631184/9402227
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0655-raw_1 and brca-0655-raw_2
## Single-end reads will be in brca-0655-raw_M
## 7257837 sequences in the BAM file
## 7257837 sequences exported
## 7257837 reads; of these:
##   7257837 (100.00%) were unpaired; of these:
##     2230073 (30.73%) aligned 0 times
##     3559178 (49.04%) aligned exactly 1 time
##     1468586 (20.23%) aligned &gt;1 times
## 69.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1468586/5027764
## 2 1118084/1468586
## 3 4268252/12016870
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0656-raw_1 and brca-0656-raw_2
## Single-end reads will be in brca-0656-raw_M
## 10201345 sequences in the BAM file
## 10201345 sequences exported
## 10201345 reads; of these:
##   10201345 (100.00%) were unpaired; of these:
##     2476480 (24.28%) aligned 0 times
##     5551324 (54.42%) aligned exactly 1 time
##     2173541 (21.31%) aligned &gt;1 times
## 75.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2173541/7724865
## 2 1486547/2173541
## 3 4886078/16421701
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0657-raw_1 and brca-0657-raw_2
## Single-end reads will be in brca-0657-raw_M
## 11761042 sequences in the BAM file
## 11761042 sequences exported
## 11761042 reads; of these:
##   11761042 (100.00%) were unpaired; of these:
##     2879630 (24.48%) aligned 0 times
##     5557803 (47.26%) aligned exactly 1 time
##     3323609 (28.26%) aligned &gt;1 times
## 75.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3323609/8881412
## 2 2373645/3323609
## 3 11073892/26310803
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0658-raw_1 and brca-0658-raw_2
## Single-end reads will be in brca-0658-raw_M
## 4939163 sequences in the BAM file
## 4939163 sequences exported
## 4939163 reads; of these:
##   4939163 (100.00%) were unpaired; of these:
##     1905882 (38.59%) aligned 0 times
##     1782718 (36.09%) aligned exactly 1 time
##     1250563 (25.32%) aligned &gt;1 times
## 61.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1250563/3033281
## 2 773721/1250563
## 3 4267157/10328366
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0659-raw_1 and brca-0659-raw_2
## Single-end reads will be in brca-0659-raw_M
## 7570889 sequences in the BAM file
## 7570889 sequences exported
## 7570889 reads; of these:
##   7570889 (100.00%) were unpaired; of these:
##     346985 (4.58%) aligned 0 times
##     4203856 (55.53%) aligned exactly 1 time
##     3020048 (39.89%) aligned &gt;1 times
## 95.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3020048/7223904
## 2 2550955/3020048
## 3 12091457/24517830
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0660-raw_1 and brca-0660-raw_2
## Single-end reads will be in brca-0660-raw_M
## 6354850 sequences in the BAM file
## 6354850 sequences exported
## 6354850 reads; of these:
##   6354850 (100.00%) were unpaired; of these:
##     1839398 (28.94%) aligned 0 times
##     3398337 (53.48%) aligned exactly 1 time
##     1117115 (17.58%) aligned &gt;1 times
## 71.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1117115/4515452
## 2 736805/1117115
## 3 3855529/11124640
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0661-raw_1 and brca-0661-raw_2
## Single-end reads will be in brca-0661-raw_M
## 10699680 sequences in the BAM file
## 10699680 sequences exported
## 10699680 reads; of these:
##   10699680 (100.00%) were unpaired; of these:
##     2358592 (22.04%) aligned 0 times
##     5559998 (51.96%) aligned exactly 1 time
##     2781090 (25.99%) aligned &gt;1 times
## 77.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2781090/8341088
## 2 1976709/2781090
## 3 7523013/20915278
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0662-raw_1 and brca-0662-raw_2
## Single-end reads will be in brca-0662-raw_M
## 4938301 sequences in the BAM file
## 4938301 sequences exported
## 4938301 reads; of these:
##   4938301 (100.00%) were unpaired; of these:
##     1521889 (30.82%) aligned 0 times
##     1986945 (40.24%) aligned exactly 1 time
##     1429467 (28.95%) aligned &gt;1 times
## 69.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1429467/3416412
## 2 1097244/1429467
## 3 8179340/15930474
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0663-raw_1 and brca-0663-raw_2
## Single-end reads will be in brca-0663-raw_M
## 3288873 sequences in the BAM file
## 3288873 sequences exported
## 3288873 reads; of these:
##   3288873 (100.00%) were unpaired; of these:
##     343703 (10.45%) aligned 0 times
##     1845002 (56.10%) aligned exactly 1 time
##     1100168 (33.45%) aligned &gt;1 times
## 89.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1100168/2945170
## 2 846396/1100168
## 3 6001739/12184260
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0664-raw_1 and brca-0664-raw_2
## Single-end reads will be in brca-0664-raw_M
## 2953489 sequences in the BAM file
## 2953489 sequences exported
## 2953489 reads; of these:
##   2953489 (100.00%) were unpaired; of these:
##     724882 (24.54%) aligned 0 times
##     1388625 (47.02%) aligned exactly 1 time
##     839982 (28.44%) aligned &gt;1 times
## 75.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 839982/2228607
## 2 624908/839982
## 3 4531404/9482164
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0665-raw_1 and brca-0665-raw_2
## Single-end reads will be in brca-0665-raw_M
## 9522699 sequences in the BAM file
## 9522699 sequences exported
## 9522699 reads; of these:
##   9522699 (100.00%) were unpaired; of these:
##     2901213 (30.47%) aligned 0 times
##     3545238 (37.23%) aligned exactly 1 time
##     3076248 (32.30%) aligned &gt;1 times
## 69.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3076248/6621486
## 2 2372556/3076248
## 3 6104841/16039153
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0666-raw_1 and brca-0666-raw_2
## Single-end reads will be in brca-0666-raw_M
## 2744795 sequences in the BAM file
## 2744795 sequences exported
## 2744795 reads; of these:
##   2744795 (100.00%) were unpaired; of these:
##     717381 (26.14%) aligned 0 times
##     1346542 (49.06%) aligned exactly 1 time
##     680872 (24.81%) aligned &gt;1 times
## 73.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 680872/2027414
## 2 507760/680872
## 3 4307726/8881732
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0667-raw_1 and brca-0667-raw_2
## Single-end reads will be in brca-0667-raw_M
## 13901921 sequences in the BAM file
## 13901921 sequences exported
## 13901921 reads; of these:
##   13901921 (100.00%) were unpaired; of these:
##     3750122 (26.98%) aligned 0 times
##     6889707 (49.56%) aligned exactly 1 time
##     3262092 (23.47%) aligned &gt;1 times
## 73.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3262092/10151799
## 2 2429641/3262092
## 3 10427959/26800290
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0668-raw_1 and brca-0668-raw_2
## Single-end reads will be in brca-0668-raw_M
## 3555413 sequences in the BAM file
## 3555413 sequences exported
## 3555413 reads; of these:
##   3555413 (100.00%) were unpaired; of these:
##     665166 (18.71%) aligned 0 times
##     2011314 (56.57%) aligned exactly 1 time
##     878933 (24.72%) aligned &gt;1 times
## 81.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 878933/2890247
## 2 689588/878933
## 3 7069299/14480034
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0669-raw_1 and brca-0669-raw_2
## Single-end reads will be in brca-0669-raw_M
## 5342823 sequences in the BAM file
## 5342823 sequences exported
## 5342823 reads; of these:
##   5342823 (100.00%) were unpaired; of these:
##     1216166 (22.76%) aligned 0 times
##     2293491 (42.93%) aligned exactly 1 time
##     1833166 (34.31%) aligned &gt;1 times
## 77.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1833166/4126657
## 2 1496702/1833166
## 3 10974348/22575891
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0670-raw_1 and brca-0670-raw_2
## Single-end reads will be in brca-0670-raw_M
## 2225706 sequences in the BAM file
## 2225706 sequences exported
## 2225706 reads; of these:
##   2225706 (100.00%) were unpaired; of these:
##     549572 (24.69%) aligned 0 times
##     932676 (41.90%) aligned exactly 1 time
##     743458 (33.40%) aligned &gt;1 times
## 75.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 743458/1676134
## 2 622189/743458
## 3 3566908/7890823
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0671-raw_1 and brca-0671-raw_2
## Single-end reads will be in brca-0671-raw_M
## 9573116 sequences in the BAM file
## 9573116 sequences exported
## 9573116 reads; of these:
##   9573116 (100.00%) were unpaired; of these:
##     2892075 (30.21%) aligned 0 times
##     4712881 (49.23%) aligned exactly 1 time
##     1968160 (20.56%) aligned &gt;1 times
## 69.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1968160/6681041
## 2 1559252/1968160
## 3 6027526/16125972
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0672-raw_1 and brca-0672-raw_2
## Single-end reads will be in brca-0672-raw_M
## 7799070 sequences in the BAM file
## 7799070 sequences exported
## 7799070 reads; of these:
##   7799070 (100.00%) were unpaired; of these:
##     2432774 (31.19%) aligned 0 times
##     3065983 (39.31%) aligned exactly 1 time
##     2300313 (29.49%) aligned &gt;1 times
## 68.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2300313/5366296
## 2 1549528/2300313
## 3 10294594/21662632
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0673-raw_1 and brca-0673-raw_2
## Single-end reads will be in brca-0673-raw_M
## 3372073 sequences in the BAM file
## 3372073 sequences exported
## 3372073 reads; of these:
##   3372073 (100.00%) were unpaired; of these:
##     285854 (8.48%) aligned 0 times
##     1659862 (49.22%) aligned exactly 1 time
##     1426357 (42.30%) aligned &gt;1 times
## 91.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1426357/3086219
## 2 1166299/1426357
## 3 5724913/11694935
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0674-raw_1 and brca-0674-raw_2
## Single-end reads will be in brca-0674-raw_M
## 3722546 sequences in the BAM file
## 3722546 sequences exported
## 3722546 reads; of these:
##   3722546 (100.00%) were unpaired; of these:
##     319719 (8.59%) aligned 0 times
##     1891653 (50.82%) aligned exactly 1 time
##     1511174 (40.60%) aligned &gt;1 times
## 91.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1511174/3402827
## 2 1265387/1511174
## 3 6191415/12080914
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0675-raw_1 and brca-0675-raw_2
## Single-end reads will be in brca-0675-raw_M
## 4264082 sequences in the BAM file
## 4264082 sequences exported
## 4264082 reads; of these:
##   4264082 (100.00%) were unpaired; of these:
##     352271 (8.26%) aligned 0 times
##     2455954 (57.60%) aligned exactly 1 time
##     1455857 (34.14%) aligned &gt;1 times
## 91.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1455857/3911811
## 2 1115230/1455857
## 3 13092715/24053973
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0676-raw_1 and brca-0676-raw_2
## Single-end reads will be in brca-0676-raw_M
## 1853393 sequences in the BAM file
## 1853393 sequences exported
## 1853393 reads; of these:
##   1853393 (100.00%) were unpaired; of these:
##     250641 (13.52%) aligned 0 times
##     1057106 (57.04%) aligned exactly 1 time
##     545646 (29.44%) aligned &gt;1 times
## 86.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 545646/1602752
## 2 425971/545646
## 3 2621876/5735958
## This looks like paired data from lane 7.
## Output will be in brca-0677-raw_1 and brca-0677-raw_2
## Single-end reads will be in brca-0677-raw_M
## 4017695 sequences in the BAM file
## 4017695 sequences exported
## 4017695 reads; of these:
##   4017695 (100.00%) were unpaired; of these:
##     580790 (14.46%) aligned 0 times
##     1501220 (37.37%) aligned exactly 1 time
##     1935685 (48.18%) aligned &gt;1 times
## 85.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1935685/3436905
## 2 1632934/1935685
## 3 8954803/17729847
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0678-raw_1 and brca-0678-raw_2
## Single-end reads will be in brca-0678-raw_M
## 38565262 sequences in the BAM file
## 38565262 sequences exported
## 38565262 reads; of these:
##   38565262 (100.00%) were unpaired; of these:
##     6923173 (17.95%) aligned 0 times
##     19177787 (49.73%) aligned exactly 1 time
##     12464302 (32.32%) aligned &gt;1 times
## 82.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 12464302/31642089
## 2 9175406/12464302
## 3 47795976/111243320
## [bam_sort_core] merging from 18 files...
## This looks like paired data from lane 6.
## Output will be in brca-0679-raw_1 and brca-0679-raw_2
## Single-end reads will be in brca-0679-raw_M
## 9256932 sequences in the BAM file
## 9256932 sequences exported
## 9256932 reads; of these:
##   9256932 (100.00%) were unpaired; of these:
##     2150919 (23.24%) aligned 0 times
##     4199777 (45.37%) aligned exactly 1 time
##     2906236 (31.40%) aligned &gt;1 times
## 76.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2906236/7106013
## 2 2239215/2906236
## 3 7450805/19414921
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0680-raw_1 and brca-0680-raw_2
## Single-end reads will be in brca-0680-raw_M
## 3734347 sequences in the BAM file
## 3734347 sequences exported
## 3734347 reads; of these:
##   3734347 (100.00%) were unpaired; of these:
##     924593 (24.76%) aligned 0 times
##     1580579 (42.33%) aligned exactly 1 time
##     1229175 (32.92%) aligned &gt;1 times
## 75.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1229175/2809754
## 2 1052499/1229175
## 3 3132526/7629756
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0681-raw_1 and brca-0681-raw_2
## Single-end reads will be in brca-0681-raw_M
## 12121642 sequences in the BAM file
## 12121642 sequences exported
## 12121642 reads; of these:
##   12121642 (100.00%) were unpaired; of these:
##     3233485 (26.68%) aligned 0 times
##     5106589 (42.13%) aligned exactly 1 time
##     3781568 (31.20%) aligned &gt;1 times
## 73.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3781568/8888157
## 2 3138447/3781568
## 3 14884833/30338136
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0682-raw_1 and brca-0682-raw_2
## Single-end reads will be in brca-0682-raw_M
## 13924536 sequences in the BAM file
## 13924536 sequences exported
## 13924536 reads; of these:
##   13924536 (100.00%) were unpaired; of these:
##     4037662 (29.00%) aligned 0 times
##     6744435 (48.44%) aligned exactly 1 time
##     3142439 (22.57%) aligned &gt;1 times
## 71.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3142439/9886874
## 2 2369839/3142439
## 3 6339107/19795777
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0683-raw_1 and brca-0683-raw_2
## Single-end reads will be in brca-0683-raw_M
## 11049848 sequences in the BAM file
## 11049848 sequences exported
## 11049848 reads; of these:
##   11049848 (100.00%) were unpaired; of these:
##     2403476 (21.75%) aligned 0 times
##     5225475 (47.29%) aligned exactly 1 time
##     3420897 (30.96%) aligned &gt;1 times
## 78.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3420897/8646372
## 2 2825785/3420897
## 3 9564940/22992182
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0684-raw_1 and brca-0684-raw_2
## Single-end reads will be in brca-0684-raw_M
## 4812175 sequences in the BAM file
## 4812175 sequences exported
## 4812175 reads; of these:
##   4812175 (100.00%) were unpaired; of these:
##     1193257 (24.80%) aligned 0 times
##     1782883 (37.05%) aligned exactly 1 time
##     1836035 (38.15%) aligned &gt;1 times
## 75.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1836035/3618918
## 2 1583654/1836035
## 3 7471600/15101144
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0685-raw_1 and brca-0685-raw_2
## Single-end reads will be in brca-0685-raw_M
## 7482109 sequences in the BAM file
## 7482109 sequences exported
## 7482109 reads; of these:
##   7482109 (100.00%) were unpaired; of these:
##     1574092 (21.04%) aligned 0 times
##     3160932 (42.25%) aligned exactly 1 time
##     2747085 (36.72%) aligned &gt;1 times
## 78.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2747085/5908017
## 2 2255661/2747085
## 3 6015459/14859844
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0686-raw_1 and brca-0686-raw_2
## Single-end reads will be in brca-0686-raw_M
## 9213959 sequences in the BAM file
## 9213959 sequences exported
## 9213959 reads; of these:
##   9213959 (100.00%) were unpaired; of these:
##     1904666 (20.67%) aligned 0 times
##     3561621 (38.65%) aligned exactly 1 time
##     3747672 (40.67%) aligned &gt;1 times
## 79.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3747672/7309293
## 2 2568980/3747672
## 3 10744643/24851702
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0687-raw_1 and brca-0687-raw_2
## Single-end reads will be in brca-0687-raw_M
## 4379439 sequences in the BAM file
## 4379439 sequences exported
## 4379439 reads; of these:
##   4379439 (100.00%) were unpaired; of these:
##     1138750 (26.00%) aligned 0 times
##     1748111 (39.92%) aligned exactly 1 time
##     1492578 (34.08%) aligned &gt;1 times
## 74.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1492578/3240689
## 2 1340666/1492578
## 3 2662497/6916423
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0688-raw_1 and brca-0688-raw_2
## Single-end reads will be in brca-0688-raw_M
## 4030677 sequences in the BAM file
## 4030677 sequences exported
## 4030677 reads; of these:
##   4030677 (100.00%) were unpaired; of these:
##     483819 (12.00%) aligned 0 times
##     1582628 (39.26%) aligned exactly 1 time
##     1964230 (48.73%) aligned &gt;1 times
## 88.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1964230/3546858
## 2 1604233/1964230
## 3 15408917/29955158
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0689-raw_1 and brca-0689-raw_2
## Single-end reads will be in brca-0689-raw_M
## 3006429 sequences in the BAM file
## 3006429 sequences exported
## 3006429 reads; of these:
##   3006429 (100.00%) were unpaired; of these:
##     702036 (23.35%) aligned 0 times
##     1020625 (33.95%) aligned exactly 1 time
##     1283768 (42.70%) aligned &gt;1 times
## 76.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1283768/2304393
## 2 1013110/1283768
## 3 4424139/9972222
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0690-raw_1 and brca-0690-raw_2
## Single-end reads will be in brca-0690-raw_M
## 9770354 sequences in the BAM file
## 9770354 sequences exported
## 9770354 reads; of these:
##   9770354 (100.00%) were unpaired; of these:
##     2743836 (28.08%) aligned 0 times
##     3428241 (35.09%) aligned exactly 1 time
##     3598277 (36.83%) aligned &gt;1 times
## 71.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3598277/7026518
## 2 2803019/3598277
## 3 9730032/22025995
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0691-raw_1 and brca-0691-raw_2
## Single-end reads will be in brca-0691-raw_M
## 3076731 sequences in the BAM file
## 3076731 sequences exported
## 3076731 reads; of these:
##   3076731 (100.00%) were unpaired; of these:
##     529374 (17.21%) aligned 0 times
##     1433874 (46.60%) aligned exactly 1 time
##     1113483 (36.19%) aligned &gt;1 times
## 82.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1113483/2547357
## 2 901883/1113483
## 3 7066099/14316642
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0692-raw_1 and brca-0692-raw_2
## Single-end reads will be in brca-0692-raw_M
## 5477671 sequences in the BAM file
## 5477671 sequences exported
## 5477671 reads; of these:
##   5477671 (100.00%) were unpaired; of these:
##     453028 (8.27%) aligned 0 times
##     3194790 (58.32%) aligned exactly 1 time
##     1829853 (33.41%) aligned &gt;1 times
## 91.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1829853/5024643
## 2 1361095/1829853
## 3 6938153/15850085
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0693-raw_1 and brca-0693-raw_2
## Single-end reads will be in brca-0693-raw_M
## 5428976 sequences in the BAM file
## 5428976 sequences exported
## 5428976 reads; of these:
##   5428976 (100.00%) were unpaired; of these:
##     462771 (8.52%) aligned 0 times
##     2920541 (53.80%) aligned exactly 1 time
##     2045664 (37.68%) aligned &gt;1 times
## 91.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2045664/4966205
## 2 1677816/2045664
## 3 8043941/17202840
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0694-raw_1 and brca-0694-raw_2
## Single-end reads will be in brca-0694-raw_M
## 4760450 sequences in the BAM file
## 4760450 sequences exported
## 4760450 reads; of these:
##   4760450 (100.00%) were unpaired; of these:
##     719668 (15.12%) aligned 0 times
##     2942912 (61.82%) aligned exactly 1 time
##     1097870 (23.06%) aligned &gt;1 times
## 84.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1097870/4040782
## 2 833057/1097870
## 3 6341999/14158430
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0695-raw_1 and brca-0695-raw_2
## Single-end reads will be in brca-0695-raw_M
## 20597058 sequences in the BAM file
## 20597058 sequences exported
## 20597058 reads; of these:
##   20597058 (100.00%) were unpaired; of these:
##     4116655 (19.99%) aligned 0 times
##     10618197 (51.55%) aligned exactly 1 time
##     5862206 (28.46%) aligned &gt;1 times
## 80.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5862206/16480403
## 2 4632364/5862206
## 3 27720924/62626487
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 5.
## Output will be in brca-0696-raw_1 and brca-0696-raw_2
## Single-end reads will be in brca-0696-raw_M
## 3377225 sequences in the BAM file
## 3377225 sequences exported
## 3377225 reads; of these:
##   3377225 (100.00%) were unpaired; of these:
##     1001568 (29.66%) aligned 0 times
##     1464796 (43.37%) aligned exactly 1 time
##     910861 (26.97%) aligned &gt;1 times
## 70.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 910861/2375657
## 2 742356/910861
## 3 3673879/8195118
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0697-raw_1 and brca-0697-raw_2
## Single-end reads will be in brca-0697-raw_M
## 12202159 sequences in the BAM file
## 12202159 sequences exported
## 12202159 reads; of these:
##   12202159 (100.00%) were unpaired; of these:
##     1704308 (13.97%) aligned 0 times
##     6822844 (55.92%) aligned exactly 1 time
##     3675007 (30.12%) aligned &gt;1 times
## 86.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3675007/10497851
## 2 2593449/3675007
## 3 18314134/41685916
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in brca-0698-raw_1 and brca-0698-raw_2
## Single-end reads will be in brca-0698-raw_M
## 2661712 sequences in the BAM file
## 2661712 sequences exported
## 2661712 reads; of these:
##   2661712 (100.00%) were unpaired; of these:
##     315024 (11.84%) aligned 0 times
##     1019127 (38.29%) aligned exactly 1 time
##     1327561 (49.88%) aligned &gt;1 times
## 88.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1327561/2346688
## 2 988420/1327561
## 3 13445367/24620071
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0699-raw_1 and brca-0699-raw_2
## Single-end reads will be in brca-0699-raw_M
## 2735732 sequences in the BAM file
## 2735732 sequences exported
## 2735732 reads; of these:
##   2735732 (100.00%) were unpaired; of these:
##     731605 (26.74%) aligned 0 times
##     1038697 (37.97%) aligned exactly 1 time
##     965430 (35.29%) aligned &gt;1 times
## 73.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 965430/2004127
## 2 768251/965430
## 3 4519206/9383413
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0700-raw_1 and brca-0700-raw_2
## Single-end reads will be in brca-0700-raw_M
## 22746962 sequences in the BAM file
## 22746962 sequences exported
## 22746962 reads; of these:
##   22746962 (100.00%) were unpaired; of these:
##     4914489 (21.61%) aligned 0 times
##     11459853 (50.38%) aligned exactly 1 time
##     6372620 (28.02%) aligned &gt;1 times
## 78.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6372620/17832473
## 2 4941740/6372620
## 3 32888825/69582762
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 7.
## Output will be in brca-0701-raw_1 and brca-0701-raw_2
## Single-end reads will be in brca-0701-raw_M
## 5256105 sequences in the BAM file
## 5256105 sequences exported
## 5256105 reads; of these:
##   5256105 (100.00%) were unpaired; of these:
##     440578 (8.38%) aligned 0 times
##     2879502 (54.78%) aligned exactly 1 time
##     1936025 (36.83%) aligned &gt;1 times
## 91.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1936025/4815527
## 2 1666572/1936025
## 3 4881567/11489075
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0702-raw_1 and brca-0702-raw_2
## Single-end reads will be in brca-0702-raw_M
## 6103492 sequences in the BAM file
## 6103492 sequences exported
## 6103492 reads; of these:
##   6103492 (100.00%) were unpaired; of these:
##     403172 (6.61%) aligned 0 times
##     3008774 (49.30%) aligned exactly 1 time
##     2691546 (44.10%) aligned &gt;1 times
## 93.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2691546/5700320
## 2 2295311/2691546
## 3 10262789/20246011
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0703-raw_1 and brca-0703-raw_2
## Single-end reads will be in brca-0703-raw_M
## 18375566 sequences in the BAM file
## 18375566 sequences exported
## 18375566 reads; of these:
##   18375566 (100.00%) were unpaired; of these:
##     4723234 (25.70%) aligned 0 times
##     8285827 (45.09%) aligned exactly 1 time
##     5366505 (29.20%) aligned &gt;1 times
## 74.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5366505/13652332
## 2 4096612/5366505
## 3 10598307/30205153
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0704-raw_1 and brca-0704-raw_2
## Single-end reads will be in brca-0704-raw_M
## 5748297 sequences in the BAM file
## 5748297 sequences exported
## 5748297 reads; of these:
##   5748297 (100.00%) were unpaired; of these:
##     1086068 (18.89%) aligned 0 times
##     2550947 (44.38%) aligned exactly 1 time
##     2111282 (36.73%) aligned &gt;1 times
## 81.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2111282/4662229
## 2 1690992/2111282
## 3 21737739/40149630
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-0705-raw_1 and brca-0705-raw_2
## Single-end reads will be in brca-0705-raw_M
## 4417098 sequences in the BAM file
## 4417098 sequences exported
## 4417098 reads; of these:
##   4417098 (100.00%) were unpaired; of these:
##     473006 (10.71%) aligned 0 times
##     1990516 (45.06%) aligned exactly 1 time
##     1953576 (44.23%) aligned &gt;1 times
## 89.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1953576/3944092
## 2 1559564/1953576
## 3 6486493/13347716
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0706-raw_1 and brca-0706-raw_2
## Single-end reads will be in brca-0706-raw_M
## 17880873 sequences in the BAM file
## 17880873 sequences exported
## 17880873 reads; of these:
##   17880873 (100.00%) were unpaired; of these:
##     4328375 (24.21%) aligned 0 times
##     9130601 (51.06%) aligned exactly 1 time
##     4421897 (24.73%) aligned &gt;1 times
## 75.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4421897/13552498
## 2 3497320/4421897
## 3 10580303/29225424
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-0707-raw_1 and brca-0707-raw_2
## Single-end reads will be in brca-0707-raw_M
## 3525742 sequences in the BAM file
## 3525742 sequences exported
## 3525742 reads; of these:
##   3525742 (100.00%) were unpaired; of these:
##     729056 (20.68%) aligned 0 times
##     1774760 (50.34%) aligned exactly 1 time
##     1021926 (28.98%) aligned &gt;1 times
## 79.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1021926/2796686
## 2 807034/1021926
## 3 4255955/9450104
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0708-raw_1 and brca-0708-raw_2
## Single-end reads will be in brca-0708-raw_M
## 3925048 sequences in the BAM file
## 3925048 sequences exported
## 3925048 reads; of these:
##   3925048 (100.00%) were unpaired; of these:
##     1019166 (25.97%) aligned 0 times
##     2347804 (59.82%) aligned exactly 1 time
##     558078 (14.22%) aligned &gt;1 times
## 74.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 558078/2905882
## 2 408132/558078
## 3 3239673/7879523
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0709-raw_1 and brca-0709-raw_2
## Single-end reads will be in brca-0709-raw_M
## 3250269 sequences in the BAM file
## 3250269 sequences exported
## 3250269 reads; of these:
##   3250269 (100.00%) were unpaired; of these:
##     467010 (14.37%) aligned 0 times
##     1695366 (52.16%) aligned exactly 1 time
##     1087893 (33.47%) aligned &gt;1 times
## 85.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1087893/2783259
## 2 866022/1087893
## 3 6435323/12972602
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0710-raw_1 and brca-0710-raw_2
## Single-end reads will be in brca-0710-raw_M
## 4381801 sequences in the BAM file
## 4381801 sequences exported
## 4381801 reads; of these:
##   4381801 (100.00%) were unpaired; of these:
##     277726 (6.34%) aligned 0 times
##     2456636 (56.06%) aligned exactly 1 time
##     1647439 (37.60%) aligned &gt;1 times
## 93.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1647439/4104075
## 2 1356845/1647439
## 3 8974779/18766444
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0711-raw_1 and brca-0711-raw_2
## Single-end reads will be in brca-0711-raw_M
## 6495046 sequences in the BAM file
## 6495046 sequences exported
## 6495046 reads; of these:
##   6495046 (100.00%) were unpaired; of these:
##     501360 (7.72%) aligned 0 times
##     3237228 (49.84%) aligned exactly 1 time
##     2756458 (42.44%) aligned &gt;1 times
## 92.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2756458/5993686
## 2 1673577/2756458
## 3 13780251/28189294
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0712-raw_1 and brca-0712-raw_2
## Single-end reads will be in brca-0712-raw_M
## 7702599 sequences in the BAM file
## 7702599 sequences exported
## 7702599 reads; of these:
##   7702599 (100.00%) were unpaired; of these:
##     1429461 (18.56%) aligned 0 times
##     4124744 (53.55%) aligned exactly 1 time
##     2148394 (27.89%) aligned &gt;1 times
## 81.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2148394/6273138
## 2 1723537/2148394
## 3 5784535/15069840
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0713-raw_1 and brca-0713-raw_2
## Single-end reads will be in brca-0713-raw_M
## 2172815 sequences in the BAM file
## 2172815 sequences exported
## 2172815 reads; of these:
##   2172815 (100.00%) were unpaired; of these:
##     434474 (20.00%) aligned 0 times
##     1092283 (50.27%) aligned exactly 1 time
##     646058 (29.73%) aligned &gt;1 times
## 80.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 646058/1738341
## 2 489944/646058
## 3 4992074/10520378
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0714-raw_1 and brca-0714-raw_2
## Single-end reads will be in brca-0714-raw_M
## 17872867 sequences in the BAM file
## 17872867 sequences exported
## 17872867 reads; of these:
##   17872867 (100.00%) were unpaired; of these:
##     4231399 (23.67%) aligned 0 times
##     8274701 (46.30%) aligned exactly 1 time
##     5366767 (30.03%) aligned &gt;1 times
## 76.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5366767/13641468
## 2 3701283/5366767
## 3 24111901/53706316
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 2.
## Output will be in brca-0715-raw_1 and brca-0715-raw_2
## Single-end reads will be in brca-0715-raw_M
## 7244669 sequences in the BAM file
## 7244669 sequences exported
## 7244669 reads; of these:
##   7244669 (100.00%) were unpaired; of these:
##     1482112 (20.46%) aligned 0 times
##     3082095 (42.54%) aligned exactly 1 time
##     2680462 (37.00%) aligned &gt;1 times
## 79.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2680462/5762557
## 2 2062883/2680462
## 3 10757774/23223321
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0716-raw_1 and brca-0716-raw_2
## Single-end reads will be in brca-0716-raw_M
## 2202749 sequences in the BAM file
## 2202749 sequences exported
## 2202749 reads; of these:
##   2202749 (100.00%) were unpaired; of these:
##     580256 (26.34%) aligned 0 times
##     1087238 (49.36%) aligned exactly 1 time
##     535255 (24.30%) aligned &gt;1 times
## 73.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 535255/1622493
## 2 401653/535255
## 3 2403534/5581327
## This looks like paired data from lane 4.
## Output will be in brca-0717-raw_1 and brca-0717-raw_2
## Single-end reads will be in brca-0717-raw_M
## 2773530 sequences in the BAM file
## 2773530 sequences exported
## 2773530 reads; of these:
##   2773530 (100.00%) were unpaired; of these:
##     229565 (8.28%) aligned 0 times
##     1435712 (51.76%) aligned exactly 1 time
##     1108253 (39.96%) aligned &gt;1 times
## 91.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1108253/2543965
## 2 895110/1108253
## 3 4367754/9212834
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0718-raw_1 and brca-0718-raw_2
## Single-end reads will be in brca-0718-raw_M
## 2890719 sequences in the BAM file
## 2890719 sequences exported
## 2890719 reads; of these:
##   2890719 (100.00%) were unpaired; of these:
##     736041 (25.46%) aligned 0 times
##     1346189 (46.57%) aligned exactly 1 time
##     808489 (27.97%) aligned &gt;1 times
## 74.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 808489/2154678
## 2 651541/808489
## 3 2598310/6278071
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0719-raw_1 and brca-0719-raw_2
## Single-end reads will be in brca-0719-raw_M
## 4054893 sequences in the BAM file
## 4054893 sequences exported
## 4054893 reads; of these:
##   4054893 (100.00%) were unpaired; of these:
##     272455 (6.72%) aligned 0 times
##     2387050 (58.87%) aligned exactly 1 time
##     1395388 (34.41%) aligned &gt;1 times
## 93.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1395388/3782438
## 2 1151461/1395388
## 3 4760732/10593139
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0720-raw_1 and brca-0720-raw_2
## Single-end reads will be in brca-0720-raw_M
## 10166027 sequences in the BAM file
## 10166027 sequences exported
## 10166027 reads; of these:
##   10166027 (100.00%) were unpaired; of these:
##     2784159 (27.39%) aligned 0 times
##     5253853 (51.68%) aligned exactly 1 time
##     2128015 (20.93%) aligned &gt;1 times
## 72.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2128015/7381868
## 2 1578995/2128015
## 3 7442726/19231816
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0721-raw_1 and brca-0721-raw_2
## Single-end reads will be in brca-0721-raw_M
## 21278872 sequences in the BAM file
## 21278872 sequences exported
## 21278872 reads; of these:
##   21278872 (100.00%) were unpaired; of these:
##     6311612 (29.66%) aligned 0 times
##     9227294 (43.36%) aligned exactly 1 time
##     5739966 (26.97%) aligned &gt;1 times
## 70.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5739966/14967260
## 2 4418973/5739966
## 3 16016528/41457883
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in brca-0722-raw_1 and brca-0722-raw_2
## Single-end reads will be in brca-0722-raw_M
## 4241723 sequences in the BAM file
## 4241723 sequences exported
## 4241723 reads; of these:
##   4241723 (100.00%) were unpaired; of these:
##     1125016 (26.52%) aligned 0 times
##     1657323 (39.07%) aligned exactly 1 time
##     1459384 (34.41%) aligned &gt;1 times
## 73.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1459384/3116707
## 2 1234385/1459384
## 3 6124288/12554116
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0723-raw_1 and brca-0723-raw_2
## Single-end reads will be in brca-0723-raw_M
## 3156018 sequences in the BAM file
## 3156018 sequences exported
## 3156018 reads; of these:
##   3156018 (100.00%) were unpaired; of these:
##     270914 (8.58%) aligned 0 times
##     1951354 (61.83%) aligned exactly 1 time
##     933750 (29.59%) aligned &gt;1 times
## 91.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 933750/2885104
## 2 741308/933750
## 3 3289786/7530278
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0724-raw_1 and brca-0724-raw_2
## Single-end reads will be in brca-0724-raw_M
## 2733815 sequences in the BAM file
## 2733815 sequences exported
## 2733815 reads; of these:
##   2733815 (100.00%) were unpaired; of these:
##     338466 (12.38%) aligned 0 times
##     1488822 (54.46%) aligned exactly 1 time
##     906527 (33.16%) aligned &gt;1 times
## 87.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 906527/2395349
## 2 694856/906527
## 3 5203859/10587490
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0725-raw_1 and brca-0725-raw_2
## Single-end reads will be in brca-0725-raw_M
## 3539886 sequences in the BAM file
## 3539886 sequences exported
## 3539886 reads; of these:
##   3539886 (100.00%) were unpaired; of these:
##     833036 (23.53%) aligned 0 times
##     1510616 (42.67%) aligned exactly 1 time
##     1196234 (33.79%) aligned &gt;1 times
## 76.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1196234/2706850
## 2 1020360/1196234
## 3 3638255/8241264
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0726-raw_1 and brca-0726-raw_2
## Single-end reads will be in brca-0726-raw_M
## 10184557 sequences in the BAM file
## 10184557 sequences exported
## 10184557 reads; of these:
##   10184557 (100.00%) were unpaired; of these:
##     1920400 (18.86%) aligned 0 times
##     5862532 (57.56%) aligned exactly 1 time
##     2401625 (23.58%) aligned &gt;1 times
## 81.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2401625/8264157
## 2 1792602/2401625
## 3 5942789/18169146
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0727-raw_1 and brca-0727-raw_2
## Single-end reads will be in brca-0727-raw_M
## 9297925 sequences in the BAM file
## 9297925 sequences exported
## 9297925 reads; of these:
##   9297925 (100.00%) were unpaired; of these:
##     2259480 (24.30%) aligned 0 times
##     4555085 (48.99%) aligned exactly 1 time
##     2483360 (26.71%) aligned &gt;1 times
## 75.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2483360/7038445
## 2 1873806/2483360
## 3 9242510/22241281
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0728-raw_1 and brca-0728-raw_2
## Single-end reads will be in brca-0728-raw_M
## 8889557 sequences in the BAM file
## 8889557 sequences exported
## 8889557 reads; of these:
##   8889557 (100.00%) were unpaired; of these:
##     675132 (7.59%) aligned 0 times
##     5400376 (60.75%) aligned exactly 1 time
##     2814049 (31.66%) aligned &gt;1 times
## 92.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2814049/8214425
## 2 2370614/2814049
## 3 10272370/22837185
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0729-raw_1 and brca-0729-raw_2
## Single-end reads will be in brca-0729-raw_M
## 12394566 sequences in the BAM file
## 12394566 sequences exported
## 12394566 reads; of these:
##   12394566 (100.00%) were unpaired; of these:
##     3783988 (30.53%) aligned 0 times
##     6152666 (49.64%) aligned exactly 1 time
##     2457912 (19.83%) aligned &gt;1 times
## 69.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2457912/8610578
## 2 1885186/2457912
## 3 7840202/21366647
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0730-raw_1 and brca-0730-raw_2
## Single-end reads will be in brca-0730-raw_M
## 23750298 sequences in the BAM file
## 23750298 sequences exported
## 23750298 reads; of these:
##   23750298 (100.00%) were unpaired; of these:
##     4533876 (19.09%) aligned 0 times
##     13135585 (55.31%) aligned exactly 1 time
##     6080837 (25.60%) aligned &gt;1 times
## 80.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6080837/19216422
## 2 4385999/6080837
## 3 28288380/66189214
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 2.
## Output will be in brca-0731-raw_1 and brca-0731-raw_2
## Single-end reads will be in brca-0731-raw_M
## 13474934 sequences in the BAM file
## 13474934 sequences exported
## 13474934 reads; of these:
##   13474934 (100.00%) were unpaired; of these:
##     2628078 (19.50%) aligned 0 times
##     6304972 (46.79%) aligned exactly 1 time
##     4541884 (33.71%) aligned &gt;1 times
## 80.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4541884/10846856
## 2 3737134/4541884
## 3 15387971/34878479
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0732-raw_1 and brca-0732-raw_2
## Single-end reads will be in brca-0732-raw_M
## 7599495 sequences in the BAM file
## 7599495 sequences exported
## 7599495 reads; of these:
##   7599495 (100.00%) were unpaired; of these:
##     556241 (7.32%) aligned 0 times
##     4006136 (52.72%) aligned exactly 1 time
##     3037118 (39.96%) aligned &gt;1 times
## 92.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3037118/7043254
## 2 2566922/3037118
## 3 11923220/24500516
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0733-raw_1 and brca-0733-raw_2
## Single-end reads will be in brca-0733-raw_M
## 2453149 sequences in the BAM file
## 2453149 sequences exported
## 2453149 reads; of these:
##   2453149 (100.00%) were unpaired; of these:
##     602517 (24.56%) aligned 0 times
##     951486 (38.79%) aligned exactly 1 time
##     899146 (36.65%) aligned &gt;1 times
## 75.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 899146/1850632
## 2 693508/899146
## 3 4293693/9011268
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0734-raw_1 and brca-0734-raw_2
## Single-end reads will be in brca-0734-raw_M
## 7662714 sequences in the BAM file
## 7662714 sequences exported
## 7662714 reads; of these:
##   7662714 (100.00%) were unpaired; of these:
##     1674075 (21.85%) aligned 0 times
##     3129777 (40.84%) aligned exactly 1 time
##     2858862 (37.31%) aligned &gt;1 times
## 78.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2858862/5988639
## 2 2150377/2858862
## 3 26032916/46342676
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in brca-0735-raw_1 and brca-0735-raw_2
## Single-end reads will be in brca-0735-raw_M
## 4363409 sequences in the BAM file
## 4363409 sequences exported
## 4363409 reads; of these:
##   4363409 (100.00%) were unpaired; of these:
##     331032 (7.59%) aligned 0 times
##     2538129 (58.17%) aligned exactly 1 time
##     1494248 (34.24%) aligned &gt;1 times
## 92.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1494248/4032377
## 2 1187240/1494248
## 3 9336820/18594376
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0736-raw_1 and brca-0736-raw_2
## Single-end reads will be in brca-0736-raw_M
## 3741542 sequences in the BAM file
## 3741542 sequences exported
## 3741542 reads; of these:
##   3741542 (100.00%) were unpaired; of these:
##     920235 (24.60%) aligned 0 times
##     1917675 (51.25%) aligned exactly 1 time
##     903632 (24.15%) aligned &gt;1 times
## 75.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 903632/2821307
## 2 642533/903632
## 3 2894165/7930759
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0737-raw_1 and brca-0737-raw_2
## Single-end reads will be in brca-0737-raw_M
## 17575655 sequences in the BAM file
## 17575655 sequences exported
## 17575655 reads; of these:
##   17575655 (100.00%) were unpaired; of these:
##     3314782 (18.86%) aligned 0 times
##     6624141 (37.69%) aligned exactly 1 time
##     7636732 (43.45%) aligned &gt;1 times
## 81.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7636732/14260873
## 2 6152646/7636732
## 3 38874735/79394065
## [bam_sort_core] merging from 13 files...
## This looks like paired data from lane 1.
## Output will be in brca-0738-raw_1 and brca-0738-raw_2
## Single-end reads will be in brca-0738-raw_M
## 2140873 sequences in the BAM file
## 2140873 sequences exported
## 2140873 reads; of these:
##   2140873 (100.00%) were unpaired; of these:
##     266315 (12.44%) aligned 0 times
##     1148958 (53.67%) aligned exactly 1 time
##     725600 (33.89%) aligned &gt;1 times
## 87.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 725600/1874558
## 2 577472/725600
## 3 3298217/6952852
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0739-raw_1 and brca-0739-raw_2
## Single-end reads will be in brca-0739-raw_M
## 8850048 sequences in the BAM file
## 8850048 sequences exported
## 8850048 reads; of these:
##   8850048 (100.00%) were unpaired; of these:
##     2213228 (25.01%) aligned 0 times
##     4797039 (54.20%) aligned exactly 1 time
##     1839781 (20.79%) aligned &gt;1 times
## 74.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1839781/6636820
## 2 1467715/1839781
## 3 8199472/20564839
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0740-raw_1 and brca-0740-raw_2
## Single-end reads will be in brca-0740-raw_M
## 4392286 sequences in the BAM file
## 4392286 sequences exported
## 4392286 reads; of these:
##   4392286 (100.00%) were unpaired; of these:
##     653063 (14.87%) aligned 0 times
##     2263920 (51.54%) aligned exactly 1 time
##     1475303 (33.59%) aligned &gt;1 times
## 85.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1475303/3739223
## 2 1259110/1475303
## 3 5119195/11273642
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0741-raw_1 and brca-0741-raw_2
## Single-end reads will be in brca-0741-raw_M
## 5183232 sequences in the BAM file
## 5183232 sequences exported
## 5183232 reads; of these:
##   5183232 (100.00%) were unpaired; of these:
##     1033417 (19.94%) aligned 0 times
##     1897370 (36.61%) aligned exactly 1 time
##     2252445 (43.46%) aligned &gt;1 times
## 80.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2252445/4149815
## 2 1997625/2252445
## 3 5368315/11638914
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0742-raw_1 and brca-0742-raw_2
## Single-end reads will be in brca-0742-raw_M
## 3985286 sequences in the BAM file
## 3985286 sequences exported
## 3985286 reads; of these:
##   3985286 (100.00%) were unpaired; of these:
##     351795 (8.83%) aligned 0 times
##     1946752 (48.85%) aligned exactly 1 time
##     1686739 (42.32%) aligned &gt;1 times
## 91.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1686739/3633491
## 2 1288267/1686739
## 3 7288747/14469360
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0743-raw_1 and brca-0743-raw_2
## Single-end reads will be in brca-0743-raw_M
## 5962004 sequences in the BAM file
## 5962004 sequences exported
## 5962004 reads; of these:
##   5962004 (100.00%) were unpaired; of these:
##     377658 (6.33%) aligned 0 times
##     3381568 (56.72%) aligned exactly 1 time
##     2202778 (36.95%) aligned &gt;1 times
## 93.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2202778/5584346
## 2 1810485/2202778
## 3 8114675/18367444
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0744-raw_1 and brca-0744-raw_2
## Single-end reads will be in brca-0744-raw_M
## 6710294 sequences in the BAM file
## 6710294 sequences exported
## 6710294 reads; of these:
##   6710294 (100.00%) were unpaired; of these:
##     1313167 (19.57%) aligned 0 times
##     3483647 (51.91%) aligned exactly 1 time
##     1913480 (28.52%) aligned &gt;1 times
## 80.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1913480/5397127
## 2 1348379/1913480
## 3 11145611/24671177
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0745-raw_1 and brca-0745-raw_2
## Single-end reads will be in brca-0745-raw_M
## 3935363 sequences in the BAM file
## 3935363 sequences exported
## 3935363 reads; of these:
##   3935363 (100.00%) were unpaired; of these:
##     526621 (13.38%) aligned 0 times
##     2061981 (52.40%) aligned exactly 1 time
##     1346761 (34.22%) aligned &gt;1 times
## 86.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1346761/3408742
## 2 1125403/1346761
## 3 7309792/15072217
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0746-raw_1 and brca-0746-raw_2
## Single-end reads will be in brca-0746-raw_M
## 4297000 sequences in the BAM file
## 4297000 sequences exported
## 4297000 reads; of these:
##   4297000 (100.00%) were unpaired; of these:
##     854790 (19.89%) aligned 0 times
##     1778856 (41.40%) aligned exactly 1 time
##     1663354 (38.71%) aligned &gt;1 times
## 80.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1663354/3442210
## 2 1265869/1663354
## 3 10505821/21284763
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0747-raw_1 and brca-0747-raw_2
## Single-end reads will be in brca-0747-raw_M
## 1918168 sequences in the BAM file
## 1918168 sequences exported
## 1918168 reads; of these:
##   1918168 (100.00%) were unpaired; of these:
##     583163 (30.40%) aligned 0 times
##     807244 (42.08%) aligned exactly 1 time
##     527761 (27.51%) aligned &gt;1 times
## 69.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 527761/1335005
## 2 386404/527761
## 3 3169843/6454405
## This looks like paired data from lane 6.
## Output will be in brca-0748-raw_1 and brca-0748-raw_2
## Single-end reads will be in brca-0748-raw_M
## 4778958 sequences in the BAM file
## 4778958 sequences exported
## 4778958 reads; of these:
##   4778958 (100.00%) were unpaired; of these:
##     311476 (6.52%) aligned 0 times
##     2149708 (44.98%) aligned exactly 1 time
##     2317774 (48.50%) aligned &gt;1 times
## 93.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2317774/4467482
## 2 1643150/2317774
## 3 14873766/28945888
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0749-raw_1 and brca-0749-raw_2
## Single-end reads will be in brca-0749-raw_M
## 16466838 sequences in the BAM file
## 16466838 sequences exported
## 16466838 reads; of these:
##   16466838 (100.00%) were unpaired; of these:
##     2908846 (17.66%) aligned 0 times
##     9586899 (58.22%) aligned exactly 1 time
##     3971093 (24.12%) aligned &gt;1 times
## 82.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3971093/13557992
## 2 3086898/3971093
## 3 10402625/29766349
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0750-raw_1 and brca-0750-raw_2
## Single-end reads will be in brca-0750-raw_M
## 6965039 sequences in the BAM file
## 6965039 sequences exported
## 6965039 reads; of these:
##   6965039 (100.00%) were unpaired; of these:
##     465485 (6.68%) aligned 0 times
##     3728860 (53.54%) aligned exactly 1 time
##     2770694 (39.78%) aligned &gt;1 times
## 93.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2770694/6499554
## 2 2317722/2770694
## 3 8351932/18651653
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0751-raw_1 and brca-0751-raw_2
## Single-end reads will be in brca-0751-raw_M
## 3690257 sequences in the BAM file
## 3690257 sequences exported
## 3690257 reads; of these:
##   3690257 (100.00%) were unpaired; of these:
##     262985 (7.13%) aligned 0 times
##     2105248 (57.05%) aligned exactly 1 time
##     1322024 (35.82%) aligned &gt;1 times
## 92.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1322024/3427272
## 2 1047538/1322024
## 3 8318494/16354774
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0752-raw_1 and brca-0752-raw_2
## Single-end reads will be in brca-0752-raw_M
## 5159507 sequences in the BAM file
## 5159507 sequences exported
## 5159507 reads; of these:
##   5159507 (100.00%) were unpaired; of these:
##     1048760 (20.33%) aligned 0 times
##     3067089 (59.45%) aligned exactly 1 time
##     1043658 (20.23%) aligned &gt;1 times
## 79.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1043658/4110747
## 2 772497/1043658
## 3 4216529/10984225
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0753-raw_1 and brca-0753-raw_2
## Single-end reads will be in brca-0753-raw_M
## 2780975 sequences in the BAM file
## 2780975 sequences exported
## 2780975 reads; of these:
##   2780975 (100.00%) were unpaired; of these:
##     708770 (25.49%) aligned 0 times
##     1554615 (55.90%) aligned exactly 1 time
##     517590 (18.61%) aligned &gt;1 times
## 74.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 517590/2072205
## 2 368105/517590
## 3 2567618/6393464
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0754-raw_1 and brca-0754-raw_2
## Single-end reads will be in brca-0754-raw_M
## 5249857 sequences in the BAM file
## 5249857 sequences exported
## 5249857 reads; of these:
##   5249857 (100.00%) were unpaired; of these:
##     404657 (7.71%) aligned 0 times
##     3089271 (58.84%) aligned exactly 1 time
##     1755929 (33.45%) aligned &gt;1 times
## 92.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1755929/4845200
## 2 1405539/1755929
## 3 7348410/15906247
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0755-raw_1 and brca-0755-raw_2
## Single-end reads will be in brca-0755-raw_M
## 2933356 sequences in the BAM file
## 2933356 sequences exported
## 2933356 reads; of these:
##   2933356 (100.00%) were unpaired; of these:
##     615546 (20.98%) aligned 0 times
##     1704692 (58.11%) aligned exactly 1 time
##     613118 (20.90%) aligned &gt;1 times
## 79.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 613118/2317810
## 2 490364/613118
## 3 2136536/5946714
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0756-raw_1 and brca-0756-raw_2
## Single-end reads will be in brca-0756-raw_M
## 2382442 sequences in the BAM file
## 2382442 sequences exported
## 2382442 reads; of these:
##   2382442 (100.00%) were unpaired; of these:
##     290221 (12.18%) aligned 0 times
##     1521889 (63.88%) aligned exactly 1 time
##     570332 (23.94%) aligned &gt;1 times
## 87.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 570332/2092221
## 2 443864/570332
## 3 2217580/5251129
## This looks like paired data from lane 6.
## Output will be in brca-0757-raw_1 and brca-0757-raw_2
## Single-end reads will be in brca-0757-raw_M
## 3622362 sequences in the BAM file
## 3622362 sequences exported
## 3622362 reads; of these:
##   3622362 (100.00%) were unpaired; of these:
##     440021 (12.15%) aligned 0 times
##     2013497 (55.59%) aligned exactly 1 time
##     1168844 (32.27%) aligned &gt;1 times
## 87.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1168844/3182341
## 2 900772/1168844
## 3 4467643/10115372
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0758-raw_1 and brca-0758-raw_2
## Single-end reads will be in brca-0758-raw_M
## 3793451 sequences in the BAM file
## 3793451 sequences exported
## 3793451 reads; of these:
##   3793451 (100.00%) were unpaired; of these:
##     817721 (21.56%) aligned 0 times
##     1563797 (41.22%) aligned exactly 1 time
##     1411933 (37.22%) aligned &gt;1 times
## 78.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1411933/2975730
## 2 1062271/1411933
## 3 6774408/13781687
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0759-raw_1 and brca-0759-raw_2
## Single-end reads will be in brca-0759-raw_M
## 16621073 sequences in the BAM file
## 16621073 sequences exported
## 16621073 reads; of these:
##   16621073 (100.00%) were unpaired; of these:
##     3539063 (21.29%) aligned 0 times
##     7916091 (47.63%) aligned exactly 1 time
##     5165919 (31.08%) aligned &gt;1 times
## 78.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5165919/13082010
## 2 3833033/5165919
## 3 16147565/39346124
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0760-raw_1 and brca-0760-raw_2
## Single-end reads will be in brca-0760-raw_M
## 4973505 sequences in the BAM file
## 4973505 sequences exported
## 4973505 reads; of these:
##   4973505 (100.00%) were unpaired; of these:
##     266033 (5.35%) aligned 0 times
##     3310539 (66.56%) aligned exactly 1 time
##     1396933 (28.09%) aligned &gt;1 times
## 94.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1396933/4707472
## 2 1091307/1396933
## 3 7921502/16601808
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0761-raw_1 and brca-0761-raw_2
## Single-end reads will be in brca-0761-raw_M
## 7550371 sequences in the BAM file
## 7550371 sequences exported
## 7550371 reads; of these:
##   7550371 (100.00%) were unpaired; of these:
##     2170604 (28.75%) aligned 0 times
##     3469553 (45.95%) aligned exactly 1 time
##     1910214 (25.30%) aligned &gt;1 times
## 71.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1910214/5379767
## 2 1471208/1910214
## 3 6029816/15317467
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0762-raw_1 and brca-0762-raw_2
## Single-end reads will be in brca-0762-raw_M
## 5437021 sequences in the BAM file
## 5437021 sequences exported
## 5437021 reads; of these:
##   5437021 (100.00%) were unpaired; of these:
##     310033 (5.70%) aligned 0 times
##     2314112 (42.56%) aligned exactly 1 time
##     2812876 (51.74%) aligned &gt;1 times
## 94.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2812876/5126988
## 2 2139224/2812876
## 3 28625556/53496367
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in brca-0763-raw_1 and brca-0763-raw_2
## Single-end reads will be in brca-0763-raw_M
## 18045377 sequences in the BAM file
## 18045377 sequences exported
## 18045377 reads; of these:
##   18045377 (100.00%) were unpaired; of these:
##     3562490 (19.74%) aligned 0 times
##     8583709 (47.57%) aligned exactly 1 time
##     5899178 (32.69%) aligned &gt;1 times
## 80.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5899178/14482887
## 2 4378432/5899178
## 3 9588947/29407318
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0764-raw_1 and brca-0764-raw_2
## Single-end reads will be in brca-0764-raw_M
## 14687489 sequences in the BAM file
## 14687489 sequences exported
## 14687489 reads; of these:
##   14687489 (100.00%) were unpaired; of these:
##     3325694 (22.64%) aligned 0 times
##     6230667 (42.42%) aligned exactly 1 time
##     5131128 (34.94%) aligned &gt;1 times
## 77.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5131128/11361795
## 2 4276151/5131128
## 3 11775891/29745414
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0765-raw_1 and brca-0765-raw_2
## Single-end reads will be in brca-0765-raw_M
## 12887878 sequences in the BAM file
## 12887878 sequences exported
## 12887878 reads; of these:
##   12887878 (100.00%) were unpaired; of these:
##     2491471 (19.33%) aligned 0 times
##     6295518 (48.85%) aligned exactly 1 time
##     4100889 (31.82%) aligned &gt;1 times
## 80.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4100889/10396407
## 2 2990676/4100889
## 3 9729769/25610724
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0766-raw_1 and brca-0766-raw_2
## Single-end reads will be in brca-0766-raw_M
## 3652643 sequences in the BAM file
## 3652643 sequences exported
## 3652643 reads; of these:
##   3652643 (100.00%) were unpaired; of these:
##     309802 (8.48%) aligned 0 times
##     1774171 (48.57%) aligned exactly 1 time
##     1568670 (42.95%) aligned &gt;1 times
## 91.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1568670/3342841
## 2 1194490/1568670
## 3 9542365/19148971
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0767-raw_1 and brca-0767-raw_2
## Single-end reads will be in brca-0767-raw_M
## 2309763 sequences in the BAM file
## 2309763 sequences exported
## 2309763 reads; of these:
##   2309763 (100.00%) were unpaired; of these:
##     685272 (29.67%) aligned 0 times
##     867025 (37.54%) aligned exactly 1 time
##     757466 (32.79%) aligned &gt;1 times
## 70.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 757466/1624491
## 2 543624/757466
## 3 3286365/7106317
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0768-raw_1 and brca-0768-raw_2
## Single-end reads will be in brca-0768-raw_M
## 1566705 sequences in the BAM file
## 1566705 sequences exported
## 1566705 reads; of these:
##   1566705 (100.00%) were unpaired; of these:
##     365802 (23.35%) aligned 0 times
##     756486 (48.29%) aligned exactly 1 time
##     444417 (28.37%) aligned &gt;1 times
## 76.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 444417/1200903
## 2 347458/444417
## 3 2697608/5243201
## This looks like paired data from lane 4.
## Output will be in brca-0769-raw_1 and brca-0769-raw_2
## Single-end reads will be in brca-0769-raw_M
## 4836153 sequences in the BAM file
## 4836153 sequences exported
## 4836153 reads; of these:
##   4836153 (100.00%) were unpaired; of these:
##     382612 (7.91%) aligned 0 times
##     2663898 (55.08%) aligned exactly 1 time
##     1789643 (37.01%) aligned &gt;1 times
## 92.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1789643/4453541
## 2 1470540/1789643
## 3 9821033/19143079
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0770-raw_1 and brca-0770-raw_2
## Single-end reads will be in brca-0770-raw_M
## 16009376 sequences in the BAM file
## 16009376 sequences exported
## 16009376 reads; of these:
##   16009376 (100.00%) were unpaired; of these:
##     3288736 (20.54%) aligned 0 times
##     8347564 (52.14%) aligned exactly 1 time
##     4373076 (27.32%) aligned &gt;1 times
## 79.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4373076/12720640
## 2 3029611/4373076
## 3 16934029/43659649
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in brca-0771-raw_1 and brca-0771-raw_2
## Single-end reads will be in brca-0771-raw_M
## 4297203 sequences in the BAM file
## 4297203 sequences exported
## 4297203 reads; of these:
##   4297203 (100.00%) were unpaired; of these:
##     500373 (11.64%) aligned 0 times
##     2229105 (51.87%) aligned exactly 1 time
##     1567725 (36.48%) aligned &gt;1 times
## 88.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1567725/3796830
## 2 1252204/1567725
## 3 14504167/27395953
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0772-raw_1 and brca-0772-raw_2
## Single-end reads will be in brca-0772-raw_M
## 17429208 sequences in the BAM file
## 17429208 sequences exported
## 17429208 reads; of these:
##   17429208 (100.00%) were unpaired; of these:
##     3565635 (20.46%) aligned 0 times
##     7471112 (42.87%) aligned exactly 1 time
##     6392461 (36.68%) aligned &gt;1 times
## 79.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6392461/13863573
## 2 5298209/6392461
## 3 29405888/61638947
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 2.
## Output will be in brca-0773-raw_1 and brca-0773-raw_2
## Single-end reads will be in brca-0773-raw_M
## 3888085 sequences in the BAM file
## 3888085 sequences exported
## 3888085 reads; of these:
##   3888085 (100.00%) were unpaired; of these:
##     883546 (22.72%) aligned 0 times
##     1711895 (44.03%) aligned exactly 1 time
##     1292644 (33.25%) aligned &gt;1 times
## 77.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1292644/3004539
## 2 932406/1292644
## 3 6560764/13571324
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0774-raw_1 and brca-0774-raw_2
## Single-end reads will be in brca-0774-raw_M
## 5347591 sequences in the BAM file
## 5347591 sequences exported
## 5347591 reads; of these:
##   5347591 (100.00%) were unpaired; of these:
##     349503 (6.54%) aligned 0 times
##     2910597 (54.43%) aligned exactly 1 time
##     2087491 (39.04%) aligned &gt;1 times
## 93.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2087491/4998088
## 2 1804451/2087491
## 3 4634494/11134299
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0775-raw_1 and brca-0775-raw_2
## Single-end reads will be in brca-0775-raw_M
## 1927278 sequences in the BAM file
## 1927278 sequences exported
## 1927278 reads; of these:
##   1927278 (100.00%) were unpaired; of these:
##     507491 (26.33%) aligned 0 times
##     988496 (51.29%) aligned exactly 1 time
##     431291 (22.38%) aligned &gt;1 times
## 73.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 431291/1419787
## 2 331399/431291
## 3 1001773/3130580
## This looks like paired data from lane 4.
## Output will be in brca-0776-raw_1 and brca-0776-raw_2
## Single-end reads will be in brca-0776-raw_M
## 6874679 sequences in the BAM file
## 6874679 sequences exported
## 6874679 reads; of these:
##   6874679 (100.00%) were unpaired; of these:
##     587348 (8.54%) aligned 0 times
##     3095786 (45.03%) aligned exactly 1 time
##     3191545 (46.42%) aligned &gt;1 times
## 91.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3191545/6287331
## 2 2827674/3191545
## 3 10040343/20272450
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0777-raw_1 and brca-0777-raw_2
## Single-end reads will be in brca-0777-raw_M
## 12965436 sequences in the BAM file
## 12965436 sequences exported
## 12965436 reads; of these:
##   12965436 (100.00%) were unpaired; of these:
##     3101028 (23.92%) aligned 0 times
##     6901466 (53.23%) aligned exactly 1 time
##     2962942 (22.85%) aligned &gt;1 times
## 76.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2962942/9864408
## 2 2246419/2962942
## 3 11952256/30160215
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0778-raw_1 and brca-0778-raw_2
## Single-end reads will be in brca-0778-raw_M
## 2978394 sequences in the BAM file
## 2978394 sequences exported
## 2978394 reads; of these:
##   2978394 (100.00%) were unpaired; of these:
##     829088 (27.84%) aligned 0 times
##     1173001 (39.38%) aligned exactly 1 time
##     976305 (32.78%) aligned &gt;1 times
## 72.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 976305/2149306
## 2 707395/976305
## 3 6323793/13312749
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0779-raw_1 and brca-0779-raw_2
## Single-end reads will be in brca-0779-raw_M
## 6415738 sequences in the BAM file
## 6415738 sequences exported
## 6415738 reads; of these:
##   6415738 (100.00%) were unpaired; of these:
##     1579598 (24.62%) aligned 0 times
##     2877376 (44.85%) aligned exactly 1 time
##     1958764 (30.53%) aligned &gt;1 times
## 75.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1958764/4836140
## 2 1411754/1958764
## 3 9847282/21193820
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0780-raw_1 and brca-0780-raw_2
## Single-end reads will be in brca-0780-raw_M
## 6116058 sequences in the BAM file
## 6116058 sequences exported
## 6116058 reads; of these:
##   6116058 (100.00%) were unpaired; of these:
##     576781 (9.43%) aligned 0 times
##     2105880 (34.43%) aligned exactly 1 time
##     3433397 (56.14%) aligned &gt;1 times
## 90.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3433397/5539277
## 2 2528625/3433397
## 3 23302451/46653643
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0781-raw_1 and brca-0781-raw_2
## Single-end reads will be in brca-0781-raw_M
## 18847024 sequences in the BAM file
## 18847024 sequences exported
## 18847024 reads; of these:
##   18847024 (100.00%) were unpaired; of these:
##     5285958 (28.05%) aligned 0 times
##     11498224 (61.01%) aligned exactly 1 time
##     2062842 (10.95%) aligned &gt;1 times
## 71.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2062842/13561066
## 2 1607881/2062842
## 3 6814845/25114289
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0782-raw_1 and brca-0782-raw_2
## Single-end reads will be in brca-0782-raw_M
## 5743153 sequences in the BAM file
## 5743153 sequences exported
## 5743153 reads; of these:
##   5743153 (100.00%) were unpaired; of these:
##     794931 (13.84%) aligned 0 times
##     3359063 (58.49%) aligned exactly 1 time
##     1589159 (27.67%) aligned &gt;1 times
## 86.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1589159/4948222
## 2 1302530/1589159
## 3 7179346/15315823
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0783-raw_1 and brca-0783-raw_2
## Single-end reads will be in brca-0783-raw_M
## 2815040 sequences in the BAM file
## 2815040 sequences exported
## 2815040 reads; of these:
##   2815040 (100.00%) were unpaired; of these:
##     195549 (6.95%) aligned 0 times
##     1718172 (61.04%) aligned exactly 1 time
##     901319 (32.02%) aligned &gt;1 times
## 93.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 901319/2619491
## 2 731498/901319
## 3 3451085/7886834
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0784-raw_1 and brca-0784-raw_2
## Single-end reads will be in brca-0784-raw_M
## 8070199 sequences in the BAM file
## 8070199 sequences exported
## 8070199 reads; of these:
##   8070199 (100.00%) were unpaired; of these:
##     2382853 (29.53%) aligned 0 times
##     3096430 (38.37%) aligned exactly 1 time
##     2590916 (32.10%) aligned &gt;1 times
## 70.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2590916/5687346
## 2 1916581/2590916
## 3 25004745/50277680
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in brca-0785-raw_1 and brca-0785-raw_2
## Single-end reads will be in brca-0785-raw_M
## 4857885 sequences in the BAM file
## 4857885 sequences exported
## 4857885 reads; of these:
##   4857885 (100.00%) were unpaired; of these:
##     899859 (18.52%) aligned 0 times
##     2064930 (42.51%) aligned exactly 1 time
##     1893096 (38.97%) aligned &gt;1 times
## 81.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1893096/3958026
## 2 1553510/1893096
## 3 9943356/19169102
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0786-raw_1 and brca-0786-raw_2
## Single-end reads will be in brca-0786-raw_M
## 9353053 sequences in the BAM file
## 9353053 sequences exported
## 9353053 reads; of these:
##   9353053 (100.00%) were unpaired; of these:
##     554693 (5.93%) aligned 0 times
##     5092137 (54.44%) aligned exactly 1 time
##     3706223 (39.63%) aligned &gt;1 times
## 94.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3706223/8798360
## 2 3234031/3706223
## 3 10869209/24892557
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0787-raw_1 and brca-0787-raw_2
## Single-end reads will be in brca-0787-raw_M
## 7661538 sequences in the BAM file
## 7661538 sequences exported
## 7661538 reads; of these:
##   7661538 (100.00%) were unpaired; of these:
##     3241918 (42.31%) aligned 0 times
##     2668300 (34.83%) aligned exactly 1 time
##     1751320 (22.86%) aligned &gt;1 times
## 57.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1751320/4419620
## 2 1361178/1751320
## 3 11636249/20995896
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0788-raw_1 and brca-0788-raw_2
## Single-end reads will be in brca-0788-raw_M
## 4734495 sequences in the BAM file
## 4734495 sequences exported
## 4734495 reads; of these:
##   4734495 (100.00%) were unpaired; of these:
##     1263517 (26.69%) aligned 0 times
##     1810132 (38.23%) aligned exactly 1 time
##     1660846 (35.08%) aligned &gt;1 times
## 73.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1660846/3470978
## 2 1397799/1660846
## 3 5313118/11672853
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0789-raw_1 and brca-0789-raw_2
## Single-end reads will be in brca-0789-raw_M
## 5577424 sequences in the BAM file
## 5577424 sequences exported
## 5577424 reads; of these:
##   5577424 (100.00%) were unpaired; of these:
##     814874 (14.61%) aligned 0 times
##     2363941 (42.38%) aligned exactly 1 time
##     2398609 (43.01%) aligned &gt;1 times
## 85.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2398609/4762550
## 2 2027627/2398609
## 3 6516466/13909663
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0790-raw_1 and brca-0790-raw_2
## Single-end reads will be in brca-0790-raw_M
## 3294109 sequences in the BAM file
## 3294109 sequences exported
## 3294109 reads; of these:
##   3294109 (100.00%) were unpaired; of these:
##     748542 (22.72%) aligned 0 times
##     1524870 (46.29%) aligned exactly 1 time
##     1020697 (30.99%) aligned &gt;1 times
## 77.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1020697/2545567
## 2 750466/1020697
## 3 5617907/11858515
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0791-raw_1 and brca-0791-raw_2
## Single-end reads will be in brca-0791-raw_M
## 7369906 sequences in the BAM file
## 7369906 sequences exported
## 7369906 reads; of these:
##   7369906 (100.00%) were unpaired; of these:
##     1196678 (16.24%) aligned 0 times
##     3169813 (43.01%) aligned exactly 1 time
##     3003415 (40.75%) aligned &gt;1 times
## 83.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3003415/6173228
## 2 2193367/3003415
## 3 10627865/23800131
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0792-raw_1 and brca-0792-raw_2
## Single-end reads will be in brca-0792-raw_M
## 14182606 sequences in the BAM file
## 14182606 sequences exported
## 14182606 reads; of these:
##   14182606 (100.00%) were unpaired; of these:
##     3557195 (25.08%) aligned 0 times
##     7385209 (52.07%) aligned exactly 1 time
##     3240202 (22.85%) aligned &gt;1 times
## 74.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3240202/10625411
## 2 2150385/3240202
## 3 12299846/30840496
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-0793-raw_1 and brca-0793-raw_2
## Single-end reads will be in brca-0793-raw_M
## 2576000 sequences in the BAM file
## 2576000 sequences exported
## 2576000 reads; of these:
##   2576000 (100.00%) were unpaired; of these:
##     845208 (32.81%) aligned 0 times
##     1045533 (40.59%) aligned exactly 1 time
##     685259 (26.60%) aligned &gt;1 times
## 67.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 685259/1730792
## 2 552032/685259
## 3 2758020/5965158
## This looks like paired data from lane 5.
## Output will be in brca-0794-raw_1 and brca-0794-raw_2
## Single-end reads will be in brca-0794-raw_M
## 8883986 sequences in the BAM file
## 8883986 sequences exported
## 8883986 reads; of these:
##   8883986 (100.00%) were unpaired; of these:
##     1741562 (19.60%) aligned 0 times
##     3898940 (43.89%) aligned exactly 1 time
##     3243484 (36.51%) aligned &gt;1 times
## 80.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3243484/7142424
## 2 2515434/3243484
## 3 12340914/27048860
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0795-raw_1 and brca-0795-raw_2
## Single-end reads will be in brca-0795-raw_M
## 2358985 sequences in the BAM file
## 2358985 sequences exported
## 2358985 reads; of these:
##   2358985 (100.00%) were unpaired; of these:
##     300458 (12.74%) aligned 0 times
##     995859 (42.22%) aligned exactly 1 time
##     1062668 (45.05%) aligned &gt;1 times
## 87.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1062668/2058527
## 2 861659/1062668
## 3 3412981/6984560
## This looks like paired data from lane 1.
## Output will be in brca-0796-raw_1 and brca-0796-raw_2
## Single-end reads will be in brca-0796-raw_M
## 9509948 sequences in the BAM file
## 9509948 sequences exported
## 9509948 reads; of these:
##   9509948 (100.00%) were unpaired; of these:
##     2734910 (28.76%) aligned 0 times
##     2999034 (31.54%) aligned exactly 1 time
##     3776004 (39.71%) aligned &gt;1 times
## 71.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3776004/6775038
## 2 3198985/3776004
## 3 20174380/39886383
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0797-raw_1 and brca-0797-raw_2
## Single-end reads will be in brca-0797-raw_M
## 2870823 sequences in the BAM file
## 2870823 sequences exported
## 2870823 reads; of these:
##   2870823 (100.00%) were unpaired; of these:
##     326489 (11.37%) aligned 0 times
##     1729883 (60.26%) aligned exactly 1 time
##     814451 (28.37%) aligned &gt;1 times
## 88.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 814451/2544334
## 2 610966/814451
## 3 5640252/12100918
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0798-raw_1 and brca-0798-raw_2
## Single-end reads will be in brca-0798-raw_M
## 6525347 sequences in the BAM file
## 6525347 sequences exported
## 6525347 reads; of these:
##   6525347 (100.00%) were unpaired; of these:
##     1341917 (20.56%) aligned 0 times
##     3227441 (49.46%) aligned exactly 1 time
##     1955989 (29.98%) aligned &gt;1 times
## 79.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1955989/5183430
## 2 1598413/1955989
## 3 6703231/15795687
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0799-raw_1 and brca-0799-raw_2
## Single-end reads will be in brca-0799-raw_M
## 8675459 sequences in the BAM file
## 8675459 sequences exported
## 8675459 reads; of these:
##   8675459 (100.00%) were unpaired; of these:
##     1916103 (22.09%) aligned 0 times
##     3887937 (44.82%) aligned exactly 1 time
##     2871419 (33.10%) aligned &gt;1 times
## 77.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2871419/6759356
## 2 2168784/2871419
## 3 12239808/25360935
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0800-raw_1 and brca-0800-raw_2
## Single-end reads will be in brca-0800-raw_M
## 4868454 sequences in the BAM file
## 4868454 sequences exported
## 4868454 reads; of these:
##   4868454 (100.00%) were unpaired; of these:
##     1294459 (26.59%) aligned 0 times
##     2038861 (41.88%) aligned exactly 1 time
##     1535134 (31.53%) aligned &gt;1 times
## 73.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1535134/3573995
## 2 1288831/1535134
## 3 7447152/14348417
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0801-raw_1 and brca-0801-raw_2
## Single-end reads will be in brca-0801-raw_M
## 14790154 sequences in the BAM file
## 14790154 sequences exported
## 14790154 reads; of these:
##   14790154 (100.00%) were unpaired; of these:
##     4080638 (27.59%) aligned 0 times
##     6948021 (46.98%) aligned exactly 1 time
##     3761495 (25.43%) aligned &gt;1 times
## 72.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3761495/10709516
## 2 3142175/3761495
## 3 8956073/24438625
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0802-raw_1 and brca-0802-raw_2
## Single-end reads will be in brca-0802-raw_M
## 16475091 sequences in the BAM file
## 16475091 sequences exported
## 16475091 reads; of these:
##   16475091 (100.00%) were unpaired; of these:
##     3270614 (19.85%) aligned 0 times
##     7552543 (45.84%) aligned exactly 1 time
##     5651934 (34.31%) aligned &gt;1 times
## 80.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5651934/13204477
## 2 4610677/5651934
## 3 17339603/41193828
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0803-raw_1 and brca-0803-raw_2
## Single-end reads will be in brca-0803-raw_M
## 2549854 sequences in the BAM file
## 2549854 sequences exported
## 2549854 reads; of these:
##   2549854 (100.00%) were unpaired; of these:
##     249028 (9.77%) aligned 0 times
##     1374453 (53.90%) aligned exactly 1 time
##     926373 (36.33%) aligned &gt;1 times
## 90.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 926373/2300826
## 2 803303/926373
## 3 2940877/6670502
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0804-raw_1 and brca-0804-raw_2
## Single-end reads will be in brca-0804-raw_M
## 14875689 sequences in the BAM file
## 14875689 sequences exported
## 14875689 reads; of these:
##   14875689 (100.00%) were unpaired; of these:
##     3820324 (25.68%) aligned 0 times
##     7596195 (51.06%) aligned exactly 1 time
##     3459170 (23.25%) aligned &gt;1 times
## 74.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3459170/11055365
## 2 2654672/3459170
## 3 9831177/27061560
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0805-raw_1 and brca-0805-raw_2
## Single-end reads will be in brca-0805-raw_M
## 2678251 sequences in the BAM file
## 2678251 sequences exported
## 2678251 reads; of these:
##   2678251 (100.00%) were unpaired; of these:
##     592741 (22.13%) aligned 0 times
##     1197146 (44.70%) aligned exactly 1 time
##     888364 (33.17%) aligned &gt;1 times
## 77.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 888364/2085510
## 2 588117/888364
## 3 4347548/9454699
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0806-raw_1 and brca-0806-raw_2
## Single-end reads will be in brca-0806-raw_M
## 3648963 sequences in the BAM file
## 3648963 sequences exported
## 3648963 reads; of these:
##   3648963 (100.00%) were unpaired; of these:
##     426168 (11.68%) aligned 0 times
##     1632324 (44.73%) aligned exactly 1 time
##     1590471 (43.59%) aligned &gt;1 times
## 88.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1590471/3222795
## 2 1333732/1590471
## 3 8143416/14856907
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0807-raw_1 and brca-0807-raw_2
## Single-end reads will be in brca-0807-raw_M
## 6180560 sequences in the BAM file
## 6180560 sequences exported
## 6180560 reads; of these:
##   6180560 (100.00%) were unpaired; of these:
##     1352971 (21.89%) aligned 0 times
##     2898405 (46.90%) aligned exactly 1 time
##     1929184 (31.21%) aligned &gt;1 times
## 78.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1929184/4827589
## 2 1563380/1929184
## 3 12124724/23476681
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0808-raw_1 and brca-0808-raw_2
## Single-end reads will be in brca-0808-raw_M
## 5780984 sequences in the BAM file
## 5780984 sequences exported
## 5780984 reads; of these:
##   5780984 (100.00%) were unpaired; of these:
##     465559 (8.05%) aligned 0 times
##     2488570 (43.05%) aligned exactly 1 time
##     2826855 (48.90%) aligned &gt;1 times
## 91.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2826855/5315425
## 2 2462704/2826855
## 3 10264945/19023653
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0809-raw_1 and brca-0809-raw_2
## Single-end reads will be in brca-0809-raw_M
## 7064252 sequences in the BAM file
## 7064252 sequences exported
## 7064252 reads; of these:
##   7064252 (100.00%) were unpaired; of these:
##     347986 (4.93%) aligned 0 times
##     3413718 (48.32%) aligned exactly 1 time
##     3302548 (46.75%) aligned &gt;1 times
## 95.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3302548/6716266
## 2 2922573/3302548
## 3 15262994/29572447
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0810-raw_1 and brca-0810-raw_2
## Single-end reads will be in brca-0810-raw_M
## 17591884 sequences in the BAM file
## 17591884 sequences exported
## 17591884 reads; of these:
##   17591884 (100.00%) were unpaired; of these:
##     3561185 (20.24%) aligned 0 times
##     9150851 (52.02%) aligned exactly 1 time
##     4879848 (27.74%) aligned &gt;1 times
## 79.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4879848/14030699
## 2 4043745/4879848
## 3 15819220/40661306
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in brca-0811-raw_1 and brca-0811-raw_2
## Single-end reads will be in brca-0811-raw_M
## 4719368 sequences in the BAM file
## 4719368 sequences exported
## 4719368 reads; of these:
##   4719368 (100.00%) were unpaired; of these:
##     419647 (8.89%) aligned 0 times
##     2590261 (54.89%) aligned exactly 1 time
##     1709460 (36.22%) aligned &gt;1 times
## 91.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1709460/4299721
## 2 1458298/1709460
## 3 7097651/14249139
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0812-raw_1 and brca-0812-raw_2
## Single-end reads will be in brca-0812-raw_M
## 14847533 sequences in the BAM file
## 14847533 sequences exported
## 14847533 reads; of these:
##   14847533 (100.00%) were unpaired; of these:
##     3658261 (24.64%) aligned 0 times
##     6232285 (41.98%) aligned exactly 1 time
##     4956987 (33.39%) aligned &gt;1 times
## 75.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4956987/11189272
## 2 3757081/4956987
## 3 16370601/39131407
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0813-raw_1 and brca-0813-raw_2
## Single-end reads will be in brca-0813-raw_M
## 3662517 sequences in the BAM file
## 3662517 sequences exported
## 3662517 reads; of these:
##   3662517 (100.00%) were unpaired; of these:
##     1043565 (28.49%) aligned 0 times
##     1521324 (41.54%) aligned exactly 1 time
##     1097628 (29.97%) aligned &gt;1 times
## 71.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1097628/2618952
## 2 861208/1097628
## 3 4100802/8809435
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0814-raw_1 and brca-0814-raw_2
## Single-end reads will be in brca-0814-raw_M
## 2041811 sequences in the BAM file
## 2041811 sequences exported
## 2041811 reads; of these:
##   2041811 (100.00%) were unpaired; of these:
##     385492 (18.88%) aligned 0 times
##     948173 (46.44%) aligned exactly 1 time
##     708146 (34.68%) aligned &gt;1 times
## 81.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 708146/1656319
## 2 548323/708146
## 3 4479998/9088727
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0815-raw_1 and brca-0815-raw_2
## Single-end reads will be in brca-0815-raw_M
## 13671304 sequences in the BAM file
## 13671304 sequences exported
## 13671304 reads; of these:
##   13671304 (100.00%) were unpaired; of these:
##     2630315 (19.24%) aligned 0 times
##     5881818 (43.02%) aligned exactly 1 time
##     5159171 (37.74%) aligned &gt;1 times
## 80.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5159171/11040989
## 2 4092426/5159171
## 3 13276922/32201248
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0816-raw_1 and brca-0816-raw_2
## Single-end reads will be in brca-0816-raw_M
## 11209394 sequences in the BAM file
## 11209394 sequences exported
## 11209394 reads; of these:
##   11209394 (100.00%) were unpaired; of these:
##     2267390 (20.23%) aligned 0 times
##     5854344 (52.23%) aligned exactly 1 time
##     3087660 (27.55%) aligned &gt;1 times
## 79.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3087660/8942004
## 2 2365927/3087660
## 3 6724110/19187241
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0817-raw_1 and brca-0817-raw_2
## Single-end reads will be in brca-0817-raw_M
## 3616440 sequences in the BAM file
## 3616440 sequences exported
## 3616440 reads; of these:
##   3616440 (100.00%) were unpaired; of these:
##     709141 (19.61%) aligned 0 times
##     1552585 (42.93%) aligned exactly 1 time
##     1354714 (37.46%) aligned &gt;1 times
## 80.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1354714/2907299
## 2 1125992/1354714
## 3 12840792/24323816
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0818-raw_1 and brca-0818-raw_2
## Single-end reads will be in brca-0818-raw_M
## 2792327 sequences in the BAM file
## 2792327 sequences exported
## 2792327 reads; of these:
##   2792327 (100.00%) were unpaired; of these:
##     403326 (14.44%) aligned 0 times
##     1580393 (56.60%) aligned exactly 1 time
##     808608 (28.96%) aligned &gt;1 times
## 85.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 808608/2389001
## 2 679731/808608
## 3 2385821/5897380
## This looks like paired data from lane 2.
## Output will be in brca-0819-raw_1 and brca-0819-raw_2
## Single-end reads will be in brca-0819-raw_M
## 4761784 sequences in the BAM file
## 4761784 sequences exported
## 4761784 reads; of these:
##   4761784 (100.00%) were unpaired; of these:
##     676768 (14.21%) aligned 0 times
##     3046474 (63.98%) aligned exactly 1 time
##     1038542 (21.81%) aligned &gt;1 times
## 85.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1038542/4085016
## 2 882256/1038542
## 3 3855087/9761401
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0820-raw_1 and brca-0820-raw_2
## Single-end reads will be in brca-0820-raw_M
## 2126972 sequences in the BAM file
## 2126972 sequences exported
## 2126972 reads; of these:
##   2126972 (100.00%) were unpaired; of these:
##     563728 (26.50%) aligned 0 times
##     885900 (41.65%) aligned exactly 1 time
##     677344 (31.85%) aligned &gt;1 times
## 73.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 677344/1563244
## 2 503307/677344
## 3 2557281/5999743
## This looks like paired data from lane 6.
## Output will be in brca-0821-raw_1 and brca-0821-raw_2
## Single-end reads will be in brca-0821-raw_M
## 4520156 sequences in the BAM file
## 4520156 sequences exported
## 4520156 reads; of these:
##   4520156 (100.00%) were unpaired; of these:
##     375127 (8.30%) aligned 0 times
##     2453912 (54.29%) aligned exactly 1 time
##     1691117 (37.41%) aligned &gt;1 times
## 91.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1691117/4145029
## 2 1130065/1691117
## 3 9881693/20476018
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0822-raw_1 and brca-0822-raw_2
## Single-end reads will be in brca-0822-raw_M
## 2371302 sequences in the BAM file
## 2371302 sequences exported
## 2371302 reads; of these:
##   2371302 (100.00%) were unpaired; of these:
##     557717 (23.52%) aligned 0 times
##     1154436 (48.68%) aligned exactly 1 time
##     659149 (27.80%) aligned &gt;1 times
## 76.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 659149/1813585
## 2 474207/659149
## 3 1542022/4505088
## This looks like paired data from lane 2.
## Output will be in brca-0823-raw_1 and brca-0823-raw_2
## Single-end reads will be in brca-0823-raw_M
## 3619681 sequences in the BAM file
## 3619681 sequences exported
## 3619681 reads; of these:
##   3619681 (100.00%) were unpaired; of these:
##     1036955 (28.65%) aligned 0 times
##     1460268 (40.34%) aligned exactly 1 time
##     1122458 (31.01%) aligned &gt;1 times
## 71.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1122458/2582726
## 2 805146/1122458
## 3 6577812/13071692
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0824-raw_1 and brca-0824-raw_2
## Single-end reads will be in brca-0824-raw_M
## 5854825 sequences in the BAM file
## 5854825 sequences exported
## 5854825 reads; of these:
##   5854825 (100.00%) were unpaired; of these:
##     1252496 (21.39%) aligned 0 times
##     3196455 (54.60%) aligned exactly 1 time
##     1405874 (24.01%) aligned &gt;1 times
## 78.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1405874/4602329
## 2 1057231/1405874
## 3 7951647/17579541
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0825-raw_1 and brca-0825-raw_2
## Single-end reads will be in brca-0825-raw_M
## 9737352 sequences in the BAM file
## 9737352 sequences exported
## 9737352 reads; of these:
##   9737352 (100.00%) were unpaired; of these:
##     1589418 (16.32%) aligned 0 times
##     5600890 (57.52%) aligned exactly 1 time
##     2547044 (26.16%) aligned &gt;1 times
## 83.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2547044/8147934
## 2 2041995/2547044
## 3 6353055/18153064
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0826-raw_1 and brca-0826-raw_2
## Single-end reads will be in brca-0826-raw_M
## 3095177 sequences in the BAM file
## 3095177 sequences exported
## 3095177 reads; of these:
##   3095177 (100.00%) were unpaired; of these:
##     644757 (20.83%) aligned 0 times
##     1550843 (50.11%) aligned exactly 1 time
##     899577 (29.06%) aligned &gt;1 times
## 79.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 899577/2450420
## 2 705344/899577
## 3 3936669/8890913
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0827-raw_1 and brca-0827-raw_2
## Single-end reads will be in brca-0827-raw_M
## 1889878 sequences in the BAM file
## 1889878 sequences exported
## 1889878 reads; of these:
##   1889878 (100.00%) were unpaired; of these:
##     577641 (30.56%) aligned 0 times
##     789216 (41.76%) aligned exactly 1 time
##     523021 (27.67%) aligned &gt;1 times
## 69.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 523021/1312237
## 2 401832/523021
## 3 4024653/7551578
## This looks like paired data from lane 3.
## Output will be in brca-0828-raw_1 and brca-0828-raw_2
## Single-end reads will be in brca-0828-raw_M
## 9408241 sequences in the BAM file
## 9408241 sequences exported
## 9408241 reads; of these:
##   9408241 (100.00%) were unpaired; of these:
##     2434368 (25.87%) aligned 0 times
##     4325350 (45.97%) aligned exactly 1 time
##     2648523 (28.15%) aligned &gt;1 times
## 74.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2648523/6973873
## 2 2197164/2648523
## 3 6566622/17125712
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0829-raw_1 and brca-0829-raw_2
## Single-end reads will be in brca-0829-raw_M
## 1872175 sequences in the BAM file
## 1872175 sequences exported
## 1872175 reads; of these:
##   1872175 (100.00%) were unpaired; of these:
##     482649 (25.78%) aligned 0 times
##     880931 (47.05%) aligned exactly 1 time
##     508595 (27.17%) aligned &gt;1 times
## 74.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 508595/1389526
## 2 416131/508595
## 3 1429444/3773697
## This looks like paired data from lane 6.
## Output will be in brca-0830-raw_1 and brca-0830-raw_2
## Single-end reads will be in brca-0830-raw_M
## 2371017 sequences in the BAM file
## 2371017 sequences exported
## 2371017 reads; of these:
##   2371017 (100.00%) were unpaired; of these:
##     161383 (6.81%) aligned 0 times
##     1299728 (54.82%) aligned exactly 1 time
##     909906 (38.38%) aligned &gt;1 times
## 93.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 909906/2209634
## 2 724550/909906
## 3 5138860/9634484
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0831-raw_1 and brca-0831-raw_2
## Single-end reads will be in brca-0831-raw_M
## 3749307 sequences in the BAM file
## 3749307 sequences exported
## 3749307 reads; of these:
##   3749307 (100.00%) were unpaired; of these:
##     497322 (13.26%) aligned 0 times
##     1905884 (50.83%) aligned exactly 1 time
##     1346101 (35.90%) aligned &gt;1 times
## 86.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1346101/3251985
## 2 1162932/1346101
## 3 3303331/8043722
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0832-raw_1 and brca-0832-raw_2
## Single-end reads will be in brca-0832-raw_M
## 9478795 sequences in the BAM file
## 9478795 sequences exported
## 9478795 reads; of these:
##   9478795 (100.00%) were unpaired; of these:
##     591623 (6.24%) aligned 0 times
##     5105089 (53.86%) aligned exactly 1 time
##     3782083 (39.90%) aligned &gt;1 times
## 93.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3782083/8887172
## 2 3197663/3782083
## 3 11482753/25315462
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-0833-raw_1 and brca-0833-raw_2
## Single-end reads will be in brca-0833-raw_M
## 12959611 sequences in the BAM file
## 12959611 sequences exported
## 12959611 reads; of these:
##   12959611 (100.00%) were unpaired; of these:
##     2199522 (16.97%) aligned 0 times
##     6500646 (50.16%) aligned exactly 1 time
##     4259443 (32.87%) aligned &gt;1 times
## 83.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4259443/10760089
## 2 3159571/4259443
## 3 19679694/41359900
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0834-raw_1 and brca-0834-raw_2
## Single-end reads will be in brca-0834-raw_M
## 5405340 sequences in the BAM file
## 5405340 sequences exported
## 5405340 reads; of these:
##   5405340 (100.00%) were unpaired; of these:
##     1026050 (18.98%) aligned 0 times
##     2679095 (49.56%) aligned exactly 1 time
##     1700195 (31.45%) aligned &gt;1 times
## 81.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1700195/4379290
## 2 1395025/1700195
## 3 7750343/17065734
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0835-raw_1 and brca-0835-raw_2
## Single-end reads will be in brca-0835-raw_M
## 7091473 sequences in the BAM file
## 7091473 sequences exported
## 7091473 reads; of these:
##   7091473 (100.00%) were unpaired; of these:
##     467298 (6.59%) aligned 0 times
##     3992992 (56.31%) aligned exactly 1 time
##     2631183 (37.10%) aligned &gt;1 times
## 93.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2631183/6624175
## 2 2235499/2631183
## 3 8714572/19570573
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0836-raw_1 and brca-0836-raw_2
## Single-end reads will be in brca-0836-raw_M
## 12206455 sequences in the BAM file
## 12206455 sequences exported
## 12206455 reads; of these:
##   12206455 (100.00%) were unpaired; of these:
##     2133001 (17.47%) aligned 0 times
##     3742717 (30.66%) aligned exactly 1 time
##     6330737 (51.86%) aligned &gt;1 times
## 82.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6330737/10073454
## 2 5030691/6330737
## 3 74131640/139951890
## [bam_sort_core] merging from 20 files...
## This looks like paired data from lane 6.
## Output will be in brca-0837-raw_1 and brca-0837-raw_2
## Single-end reads will be in brca-0837-raw_M
## 3789246 sequences in the BAM file
## 3789246 sequences exported
## 3789246 reads; of these:
##   3789246 (100.00%) were unpaired; of these:
##     800205 (21.12%) aligned 0 times
##     1519528 (40.10%) aligned exactly 1 time
##     1469513 (38.78%) aligned &gt;1 times
## 78.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1469513/2989041
## 2 1163719/1469513
## 3 2678208/6869207
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0838-raw_1 and brca-0838-raw_2
## Single-end reads will be in brca-0838-raw_M
## 4588432 sequences in the BAM file
## 4588432 sequences exported
## 4588432 reads; of these:
##   4588432 (100.00%) were unpaired; of these:
##     1070260 (23.33%) aligned 0 times
##     1909989 (41.63%) aligned exactly 1 time
##     1608183 (35.05%) aligned &gt;1 times
## 76.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1608183/3518172
## 2 1407201/1608183
## 3 6836302/14610760
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0839-raw_1 and brca-0839-raw_2
## Single-end reads will be in brca-0839-raw_M
## 3576189 sequences in the BAM file
## 3576189 sequences exported
## 3576189 reads; of these:
##   3576189 (100.00%) were unpaired; of these:
##     311995 (8.72%) aligned 0 times
##     2212823 (61.88%) aligned exactly 1 time
##     1051371 (29.40%) aligned &gt;1 times
## 91.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1051371/3264194
## 2 824772/1051371
## 3 3869679/9259390
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0840-raw_1 and brca-0840-raw_2
## Single-end reads will be in brca-0840-raw_M
## 17208109 sequences in the BAM file
## 17208109 sequences exported
## 17208109 reads; of these:
##   17208109 (100.00%) were unpaired; of these:
##     4086341 (23.75%) aligned 0 times
##     8555683 (49.72%) aligned exactly 1 time
##     4566085 (26.53%) aligned &gt;1 times
## 76.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4566085/13121768
## 2 3915092/4566085
## 3 8107563/25068513
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0841-raw_1 and brca-0841-raw_2
## Single-end reads will be in brca-0841-raw_M
## 16741966 sequences in the BAM file
## 16741966 sequences exported
## 16741966 reads; of these:
##   16741966 (100.00%) were unpaired; of these:
##     3853444 (23.02%) aligned 0 times
##     6246947 (37.31%) aligned exactly 1 time
##     6641575 (39.67%) aligned &gt;1 times
## 76.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6641575/12888522
## 2 5876542/6641575
## 3 14569200/35613730
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0842-raw_1 and brca-0842-raw_2
## Single-end reads will be in brca-0842-raw_M
## 4802768 sequences in the BAM file
## 4802768 sequences exported
## 4802768 reads; of these:
##   4802768 (100.00%) were unpaired; of these:
##     742174 (15.45%) aligned 0 times
##     2607362 (54.29%) aligned exactly 1 time
##     1453232 (30.26%) aligned &gt;1 times
## 84.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1453232/4060594
## 2 1118673/1453232
## 3 7113995/15110625
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0843-raw_1 and brca-0843-raw_2
## Single-end reads will be in brca-0843-raw_M
## 9436038 sequences in the BAM file
## 9436038 sequences exported
## 9436038 reads; of these:
##   9436038 (100.00%) were unpaired; of these:
##     3086230 (32.71%) aligned 0 times
##     2153628 (22.82%) aligned exactly 1 time
##     4196180 (44.47%) aligned &gt;1 times
## 67.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4196180/6349808
## 2 3351237/4196180
## 3 44951927/84127381
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 1.
## Output will be in brca-0844-raw_1 and brca-0844-raw_2
## Single-end reads will be in brca-0844-raw_M
## 3481748 sequences in the BAM file
## 3481748 sequences exported
## 3481748 reads; of these:
##   3481748 (100.00%) were unpaired; of these:
##     801215 (23.01%) aligned 0 times
##     1576031 (45.27%) aligned exactly 1 time
##     1104502 (31.72%) aligned &gt;1 times
## 76.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1104502/2680533
## 2 888366/1104502
## 3 3952176/8538341
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0845-raw_1 and brca-0845-raw_2
## Single-end reads will be in brca-0845-raw_M
## 9301443 sequences in the BAM file
## 9301443 sequences exported
## 9301443 reads; of these:
##   9301443 (100.00%) were unpaired; of these:
##     3027062 (32.54%) aligned 0 times
##     3391380 (36.46%) aligned exactly 1 time
##     2883001 (31.00%) aligned &gt;1 times
## 67.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2883001/6274381
## 2 2294797/2883001
## 3 6303828/15893656
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0846-raw_1 and brca-0846-raw_2
## Single-end reads will be in brca-0846-raw_M
## 5473765 sequences in the BAM file
## 5473765 sequences exported
## 5473765 reads; of these:
##   5473765 (100.00%) were unpaired; of these:
##     596536 (10.90%) aligned 0 times
##     2865176 (52.34%) aligned exactly 1 time
##     2012053 (36.76%) aligned &gt;1 times
## 89.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2012053/4877229
## 2 1558397/2012053
## 3 14700388/30638652
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-0847-raw_1 and brca-0847-raw_2
## Single-end reads will be in brca-0847-raw_M
## 3855808 sequences in the BAM file
## 3855808 sequences exported
## 3855808 reads; of these:
##   3855808 (100.00%) were unpaired; of these:
##     517511 (13.42%) aligned 0 times
##     1753778 (45.48%) aligned exactly 1 time
##     1584519 (41.09%) aligned &gt;1 times
## 86.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1584519/3338297
## 2 1313559/1584519
## 3 7109845/13968607
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0848-raw_1 and brca-0848-raw_2
## Single-end reads will be in brca-0848-raw_M
## 7399886 sequences in the BAM file
## 7399886 sequences exported
## 7399886 reads; of these:
##   7399886 (100.00%) were unpaired; of these:
##     2233735 (30.19%) aligned 0 times
##     3639061 (49.18%) aligned exactly 1 time
##     1527090 (20.64%) aligned &gt;1 times
## 69.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1527090/5166151
## 2 1106840/1527090
## 3 5956382/14962135
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0849-raw_1 and brca-0849-raw_2
## Single-end reads will be in brca-0849-raw_M
## 13757088 sequences in the BAM file
## 13757088 sequences exported
## 13757088 reads; of these:
##   13757088 (100.00%) were unpaired; of these:
##     2495418 (18.14%) aligned 0 times
##     6254647 (45.46%) aligned exactly 1 time
##     5007023 (36.40%) aligned &gt;1 times
## 81.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5007023/11261670
## 2 3807329/5007023
## 3 18068333/39811059
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0850-raw_1 and brca-0850-raw_2
## Single-end reads will be in brca-0850-raw_M
## 3853950 sequences in the BAM file
## 3853950 sequences exported
## 3853950 reads; of these:
##   3853950 (100.00%) were unpaired; of these:
##     1417621 (36.78%) aligned 0 times
##     1336709 (34.68%) aligned exactly 1 time
##     1099620 (28.53%) aligned &gt;1 times
## 63.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1099620/2436329
## 2 767366/1099620
## 3 5366199/12012337
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0851-raw_1 and brca-0851-raw_2
## Single-end reads will be in brca-0851-raw_M
## 3650711 sequences in the BAM file
## 3650711 sequences exported
## 3650711 reads; of these:
##   3650711 (100.00%) were unpaired; of these:
##     926361 (25.37%) aligned 0 times
##     1672923 (45.82%) aligned exactly 1 time
##     1051427 (28.80%) aligned &gt;1 times
## 74.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1051427/2724350
## 2 887018/1051427
## 3 2592142/6748460
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0852-raw_1 and brca-0852-raw_2
## Single-end reads will be in brca-0852-raw_M
## 4233783 sequences in the BAM file
## 4233783 sequences exported
## 4233783 reads; of these:
##   4233783 (100.00%) were unpaired; of these:
##     819755 (19.36%) aligned 0 times
##     2272400 (53.67%) aligned exactly 1 time
##     1141628 (26.96%) aligned &gt;1 times
## 80.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1141628/3414028
## 2 821809/1141628
## 3 4306731/9865839
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0853-raw_1 and brca-0853-raw_2
## Single-end reads will be in brca-0853-raw_M
## 16161331 sequences in the BAM file
## 16161331 sequences exported
## 16161331 reads; of these:
##   16161331 (100.00%) were unpaired; of these:
##     5228750 (32.35%) aligned 0 times
##     7136958 (44.16%) aligned exactly 1 time
##     3795623 (23.49%) aligned &gt;1 times
## 67.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3795623/10932581
## 2 3112332/3795623
## 3 9244812/25665260
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-0854-raw_1 and brca-0854-raw_2
## Single-end reads will be in brca-0854-raw_M
## 6347459 sequences in the BAM file
## 6347459 sequences exported
## 6347459 reads; of these:
##   6347459 (100.00%) were unpaired; of these:
##     1352178 (21.30%) aligned 0 times
##     2579952 (40.65%) aligned exactly 1 time
##     2415329 (38.05%) aligned &gt;1 times
## 78.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2415329/4995281
## 2 2116732/2415329
## 3 8345623/17708248
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0855-raw_1 and brca-0855-raw_2
## Single-end reads will be in brca-0855-raw_M
## 4106968 sequences in the BAM file
## 4106968 sequences exported
## 4106968 reads; of these:
##   4106968 (100.00%) were unpaired; of these:
##     344157 (8.38%) aligned 0 times
##     2064829 (50.28%) aligned exactly 1 time
##     1697982 (41.34%) aligned &gt;1 times
## 91.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1697982/3762811
## 2 1416772/1697982
## 3 4919746/10772852
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0856-raw_1 and brca-0856-raw_2
## Single-end reads will be in brca-0856-raw_M
## 10519703 sequences in the BAM file
## 10519703 sequences exported
## 10519703 reads; of these:
##   10519703 (100.00%) were unpaired; of these:
##     752656 (7.15%) aligned 0 times
##     5014748 (47.67%) aligned exactly 1 time
##     4752299 (45.18%) aligned &gt;1 times
## 92.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4752299/9767047
## 2 4304404/4752299
## 3 10945357/25022637
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0857-raw_1 and brca-0857-raw_2
## Single-end reads will be in brca-0857-raw_M
## 3050390 sequences in the BAM file
## 3050390 sequences exported
## 3050390 reads; of these:
##   3050390 (100.00%) were unpaired; of these:
##     902385 (29.58%) aligned 0 times
##     1218728 (39.95%) aligned exactly 1 time
##     929277 (30.46%) aligned &gt;1 times
## 70.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 929277/2148005
## 2 757493/929277
## 3 4182841/9395625
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0858-raw_1 and brca-0858-raw_2
## Single-end reads will be in brca-0858-raw_M
## 10293122 sequences in the BAM file
## 10293122 sequences exported
## 10293122 reads; of these:
##   10293122 (100.00%) were unpaired; of these:
##     3258276 (31.65%) aligned 0 times
##     4502997 (43.75%) aligned exactly 1 time
##     2531849 (24.60%) aligned &gt;1 times
## 68.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2531849/7034846
## 2 2021962/2531849
## 3 5626708/15597121
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0859-raw_1 and brca-0859-raw_2
## Single-end reads will be in brca-0859-raw_M
## 13674084 sequences in the BAM file
## 13674084 sequences exported
## 13674084 reads; of these:
##   13674084 (100.00%) were unpaired; of these:
##     3443509 (25.18%) aligned 0 times
##     6469653 (47.31%) aligned exactly 1 time
##     3760922 (27.50%) aligned &gt;1 times
## 74.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3760922/10230575
## 2 2650120/3760922
## 3 12406904/30789421
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-0860-raw_1 and brca-0860-raw_2
## Single-end reads will be in brca-0860-raw_M
## 5623370 sequences in the BAM file
## 5623370 sequences exported
## 5623370 reads; of these:
##   5623370 (100.00%) were unpaired; of these:
##     1213147 (21.57%) aligned 0 times
##     2230318 (39.66%) aligned exactly 1 time
##     2179905 (38.77%) aligned &gt;1 times
## 78.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2179905/4410223
## 2 1836765/2179905
## 3 6061534/13521092
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0861-raw_1 and brca-0861-raw_2
## Single-end reads will be in brca-0861-raw_M
## 9405662 sequences in the BAM file
## 9405662 sequences exported
## 9405662 reads; of these:
##   9405662 (100.00%) were unpaired; of these:
##     2697052 (28.67%) aligned 0 times
##     4423659 (47.03%) aligned exactly 1 time
##     2284951 (24.29%) aligned &gt;1 times
## 71.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2284951/6708610
## 2 1680302/2284951
## 3 9070670/20665599
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0862-raw_1 and brca-0862-raw_2
## Single-end reads will be in brca-0862-raw_M
## 7811101 sequences in the BAM file
## 7811101 sequences exported
## 7811101 reads; of these:
##   7811101 (100.00%) were unpaired; of these:
##     729556 (9.34%) aligned 0 times
##     4248470 (54.39%) aligned exactly 1 time
##     2833075 (36.27%) aligned &gt;1 times
## 90.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2833075/7081545
## 2 2191964/2833075
## 3 9245540/21689903
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0863-raw_1 and brca-0863-raw_2
## Single-end reads will be in brca-0863-raw_M
## 12765773 sequences in the BAM file
## 12765773 sequences exported
## 12765773 reads; of these:
##   12765773 (100.00%) were unpaired; of these:
##     3575924 (28.01%) aligned 0 times
##     6530535 (51.16%) aligned exactly 1 time
##     2659314 (20.83%) aligned &gt;1 times
## 71.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2659314/9189849
## 2 1961392/2659314
## 3 7821244/22471171
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0864-raw_1 and brca-0864-raw_2
## Single-end reads will be in brca-0864-raw_M
## 7128917 sequences in the BAM file
## 7128917 sequences exported
## 7128917 reads; of these:
##   7128917 (100.00%) were unpaired; of these:
##     1801551 (25.27%) aligned 0 times
##     3719052 (52.17%) aligned exactly 1 time
##     1608314 (22.56%) aligned &gt;1 times
## 74.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1608314/5327366
## 2 1193695/1608314
## 3 7142372/17443966
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0865-raw_1 and brca-0865-raw_2
## Single-end reads will be in brca-0865-raw_M
## 2296864 sequences in the BAM file
## 2296864 sequences exported
## 2296864 reads; of these:
##   2296864 (100.00%) were unpaired; of these:
##     618791 (26.94%) aligned 0 times
##     789281 (34.36%) aligned exactly 1 time
##     888792 (38.70%) aligned &gt;1 times
## 73.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 888792/1678073
## 2 728276/888792
## 3 3986329/7682157
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0866-raw_1 and brca-0866-raw_2
## Single-end reads will be in brca-0866-raw_M
## 7999409 sequences in the BAM file
## 7999409 sequences exported
## 7999409 reads; of these:
##   7999409 (100.00%) were unpaired; of these:
##     2688475 (33.61%) aligned 0 times
##     3751824 (46.90%) aligned exactly 1 time
##     1559110 (19.49%) aligned &gt;1 times
## 66.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1559110/5310934
## 2 1259632/1559110
## 3 3570924/10817982
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0867-raw_1 and brca-0867-raw_2
## Single-end reads will be in brca-0867-raw_M
## 2236125 sequences in the BAM file
## 2236125 sequences exported
## 2236125 reads; of these:
##   2236125 (100.00%) were unpaired; of these:
##     543495 (24.31%) aligned 0 times
##     1099653 (49.18%) aligned exactly 1 time
##     592977 (26.52%) aligned &gt;1 times
## 75.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 592977/1692630
## 2 453825/592977
## 3 2045950/5188912
## This looks like paired data from lane 7.
## Output will be in brca-0868-raw_1 and brca-0868-raw_2
## Single-end reads will be in brca-0868-raw_M
## 4104612 sequences in the BAM file
## 4104612 sequences exported
## 4104612 reads; of these:
##   4104612 (100.00%) were unpaired; of these:
##     379690 (9.25%) aligned 0 times
##     2357972 (57.45%) aligned exactly 1 time
##     1366950 (33.30%) aligned &gt;1 times
## 90.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1366950/3724922
## 2 1133445/1366950
## 3 8734373/18256040
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0869-raw_1 and brca-0869-raw_2
## Single-end reads will be in brca-0869-raw_M
## 2737244 sequences in the BAM file
## 2737244 sequences exported
## 2737244 reads; of these:
##   2737244 (100.00%) were unpaired; of these:
##     717551 (26.21%) aligned 0 times
##     1010317 (36.91%) aligned exactly 1 time
##     1009376 (36.88%) aligned &gt;1 times
## 73.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1009376/2019693
## 2 741498/1009376
## 3 7155907/14139744
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0870-raw_1 and brca-0870-raw_2
## Single-end reads will be in brca-0870-raw_M
## 3169407 sequences in the BAM file
## 3169407 sequences exported
## 3169407 reads; of these:
##   3169407 (100.00%) were unpaired; of these:
##     810826 (25.58%) aligned 0 times
##     1178658 (37.19%) aligned exactly 1 time
##     1179923 (37.23%) aligned &gt;1 times
## 74.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1179923/2358581
## 2 952185/1179923
## 3 4973396/9855007
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0871-raw_1 and brca-0871-raw_2
## Single-end reads will be in brca-0871-raw_M
## 10196040 sequences in the BAM file
## 10196040 sequences exported
## 10196040 reads; of these:
##   10196040 (100.00%) were unpaired; of these:
##     2420072 (23.74%) aligned 0 times
##     4479858 (43.94%) aligned exactly 1 time
##     3296110 (32.33%) aligned &gt;1 times
## 76.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3296110/7775968
## 2 2659252/3296110
## 3 8998806/22737099
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0872-raw_1 and brca-0872-raw_2
## Single-end reads will be in brca-0872-raw_M
## 2900737 sequences in the BAM file
## 2900737 sequences exported
## 2900737 reads; of these:
##   2900737 (100.00%) were unpaired; of these:
##     797374 (27.49%) aligned 0 times
##     1159282 (39.97%) aligned exactly 1 time
##     944081 (32.55%) aligned &gt;1 times
## 72.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 944081/2103363
## 2 771047/944081
## 3 5504022/10995109
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0873-raw_1 and brca-0873-raw_2
## Single-end reads will be in brca-0873-raw_M
## 6419492 sequences in the BAM file
## 6419492 sequences exported
## 6419492 reads; of these:
##   6419492 (100.00%) were unpaired; of these:
##     1324158 (20.63%) aligned 0 times
##     2673740 (41.65%) aligned exactly 1 time
##     2421594 (37.72%) aligned &gt;1 times
## 79.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2421594/5095334
## 2 2067344/2421594
## 3 5602421/13006319
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0874-raw_1 and brca-0874-raw_2
## Single-end reads will be in brca-0874-raw_M
## 5065814 sequences in the BAM file
## 5065814 sequences exported
## 5065814 reads; of these:
##   5065814 (100.00%) were unpaired; of these:
##     734839 (14.51%) aligned 0 times
##     1677255 (33.11%) aligned exactly 1 time
##     2653720 (52.38%) aligned &gt;1 times
## 85.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2653720/4330975
## 2 2029249/2653720
## 3 25053132/48642104
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in brca-0875-raw_1 and brca-0875-raw_2
## Single-end reads will be in brca-0875-raw_M
## 5386206 sequences in the BAM file
## 5386206 sequences exported
## 5386206 reads; of these:
##   5386206 (100.00%) were unpaired; of these:
##     1387005 (25.75%) aligned 0 times
##     2579525 (47.89%) aligned exactly 1 time
##     1419676 (26.36%) aligned &gt;1 times
## 74.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1419676/3999201
## 2 1099983/1419676
## 3 3779533/9793333
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0876-raw_1 and brca-0876-raw_2
## Single-end reads will be in brca-0876-raw_M
## 5145297 sequences in the BAM file
## 5145297 sequences exported
## 5145297 reads; of these:
##   5145297 (100.00%) were unpaired; of these:
##     459772 (8.94%) aligned 0 times
##     2843640 (55.27%) aligned exactly 1 time
##     1841885 (35.80%) aligned &gt;1 times
## 91.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1841885/4685525
## 2 1497252/1841885
## 3 5544467/12629988
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0877-raw_1 and brca-0877-raw_2
## Single-end reads will be in brca-0877-raw_M
## 2388772 sequences in the BAM file
## 2388772 sequences exported
## 2388772 reads; of these:
##   2388772 (100.00%) were unpaired; of these:
##     572807 (23.98%) aligned 0 times
##     1079086 (45.17%) aligned exactly 1 time
##     736879 (30.85%) aligned &gt;1 times
## 76.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 736879/1815965
## 2 544141/736879
## 3 4216557/9006966
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0878-raw_1 and brca-0878-raw_2
## Single-end reads will be in brca-0878-raw_M
## 12180336 sequences in the BAM file
## 12180336 sequences exported
## 12180336 reads; of these:
##   12180336 (100.00%) were unpaired; of these:
##     3458498 (28.39%) aligned 0 times
##     3596557 (29.53%) aligned exactly 1 time
##     5125281 (42.08%) aligned &gt;1 times
## 71.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5125281/8721838
## 2 4495814/5125281
## 3 26325772/51226039
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in brca-0879-raw_1 and brca-0879-raw_2
## Single-end reads will be in brca-0879-raw_M
## 4713452 sequences in the BAM file
## 4713452 sequences exported
## 4713452 reads; of these:
##   4713452 (100.00%) were unpaired; of these:
##     1040533 (22.08%) aligned 0 times
##     2393676 (50.78%) aligned exactly 1 time
##     1279243 (27.14%) aligned &gt;1 times
## 77.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1279243/3672919
## 2 826282/1279243
## 3 4397940/10471121
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0880-raw_1 and brca-0880-raw_2
## Single-end reads will be in brca-0880-raw_M
## 6547742 sequences in the BAM file
## 6547742 sequences exported
## 6547742 reads; of these:
##   6547742 (100.00%) were unpaired; of these:
##     721342 (11.02%) aligned 0 times
##     3715053 (56.74%) aligned exactly 1 time
##     2111347 (32.25%) aligned &gt;1 times
## 88.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2111347/5826400
## 2 1711677/2111347
## 3 6516072/15270508
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0881-raw_1 and brca-0881-raw_2
## Single-end reads will be in brca-0881-raw_M
## 3624209 sequences in the BAM file
## 3624209 sequences exported
## 3624209 reads; of these:
##   3624209 (100.00%) were unpaired; of these:
##     992912 (27.40%) aligned 0 times
##     1491919 (41.17%) aligned exactly 1 time
##     1139378 (31.44%) aligned &gt;1 times
## 72.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1139378/2631297
## 2 977361/1139378
## 3 4362520/9561650
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0882-raw_1 and brca-0882-raw_2
## Single-end reads will be in brca-0882-raw_M
## 3020610 sequences in the BAM file
## 3020610 sequences exported
## 3020610 reads; of these:
##   3020610 (100.00%) were unpaired; of these:
##     528764 (17.51%) aligned 0 times
##     1572019 (52.04%) aligned exactly 1 time
##     919827 (30.45%) aligned &gt;1 times
## 82.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 919827/2491846
## 2 722706/919827
## 3 4814428/10299743
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0883-raw_1 and brca-0883-raw_2
## Single-end reads will be in brca-0883-raw_M
## 6483694 sequences in the BAM file
## 6483694 sequences exported
## 6483694 reads; of these:
##   6483694 (100.00%) were unpaired; of these:
##     1709609 (26.37%) aligned 0 times
##     2955361 (45.58%) aligned exactly 1 time
##     1818724 (28.05%) aligned &gt;1 times
## 73.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1818724/4774085
## 2 1262698/1818724
## 3 9927348/19035959
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0884-raw_1 and brca-0884-raw_2
## Single-end reads will be in brca-0884-raw_M
## 2546737 sequences in the BAM file
## 2546737 sequences exported
## 2546737 reads; of these:
##   2546737 (100.00%) were unpaired; of these:
##     523731 (20.56%) aligned 0 times
##     862959 (33.88%) aligned exactly 1 time
##     1160047 (45.55%) aligned &gt;1 times
## 79.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1160047/2023006
## 2 958418/1160047
## 3 5193714/9881964
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0885-raw_1 and brca-0885-raw_2
## Single-end reads will be in brca-0885-raw_M
## 6726660 sequences in the BAM file
## 6726660 sequences exported
## 6726660 reads; of these:
##   6726660 (100.00%) were unpaired; of these:
##     474261 (7.05%) aligned 0 times
##     3728058 (55.42%) aligned exactly 1 time
##     2524341 (37.53%) aligned &gt;1 times
## 92.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2524341/6252399
## 2 1942947/2524341
## 3 10046947/20875214
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0886-raw_1 and brca-0886-raw_2
## Single-end reads will be in brca-0886-raw_M
## 1970670 sequences in the BAM file
## 1970670 sequences exported
## 1970670 reads; of these:
##   1970670 (100.00%) were unpaired; of these:
##     522045 (26.49%) aligned 0 times
##     1016557 (51.58%) aligned exactly 1 time
##     432068 (21.92%) aligned &gt;1 times
## 73.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 432068/1448625
## 2 339969/432068
## 3 2434999/5203886
## This looks like paired data from lane 7.
## Output will be in brca-0887-raw_1 and brca-0887-raw_2
## Single-end reads will be in brca-0887-raw_M
## 3621512 sequences in the BAM file
## 3621512 sequences exported
## 3621512 reads; of these:
##   3621512 (100.00%) were unpaired; of these:
##     958412 (26.46%) aligned 0 times
##     1538940 (42.49%) aligned exactly 1 time
##     1124160 (31.04%) aligned &gt;1 times
## 73.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1124160/2663100
## 2 962380/1124160
## 3 4895486/10373536
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0888-raw_1 and brca-0888-raw_2
## Single-end reads will be in brca-0888-raw_M
## 4116524 sequences in the BAM file
## 4116524 sequences exported
## 4116524 reads; of these:
##   4116524 (100.00%) were unpaired; of these:
##     897473 (21.80%) aligned 0 times
##     1582068 (38.43%) aligned exactly 1 time
##     1636983 (39.77%) aligned &gt;1 times
## 78.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1636983/3219051
## 2 1334559/1636983
## 3 3574837/8498890
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0889-raw_1 and brca-0889-raw_2
## Single-end reads will be in brca-0889-raw_M
## 13213287 sequences in the BAM file
## 13213287 sequences exported
## 13213287 reads; of these:
##   13213287 (100.00%) were unpaired; of these:
##     3290089 (24.90%) aligned 0 times
##     5512985 (41.72%) aligned exactly 1 time
##     4410213 (33.38%) aligned &gt;1 times
## 75.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4410213/9923198
## 2 3733443/4410213
## 3 14840887/33862731
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0890-raw_1 and brca-0890-raw_2
## Single-end reads will be in brca-0890-raw_M
## 8048241 sequences in the BAM file
## 8048241 sequences exported
## 8048241 reads; of these:
##   8048241 (100.00%) were unpaired; of these:
##     1623434 (20.17%) aligned 0 times
##     3713357 (46.14%) aligned exactly 1 time
##     2711450 (33.69%) aligned &gt;1 times
## 79.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2711450/6424807
## 2 2184934/2711450
## 3 10687797/23774052
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0891-raw_1 and brca-0891-raw_2
## Single-end reads will be in brca-0891-raw_M
## 21094404 sequences in the BAM file
## 21094404 sequences exported
## 21094404 reads; of these:
##   21094404 (100.00%) were unpaired; of these:
##     4898615 (23.22%) aligned 0 times
##     9160876 (43.43%) aligned exactly 1 time
##     7034913 (33.35%) aligned &gt;1 times
## 76.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7034913/16195789
## 2 5575631/7034913
## 3 22117752/55265760
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in brca-0892-raw_1 and brca-0892-raw_2
## Single-end reads will be in brca-0892-raw_M
## 3613695 sequences in the BAM file
## 3613695 sequences exported
## 3613695 reads; of these:
##   3613695 (100.00%) were unpaired; of these:
##     468783 (12.97%) aligned 0 times
##     1629379 (45.09%) aligned exactly 1 time
##     1515533 (41.94%) aligned &gt;1 times
## 87.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1515533/3144912
## 2 1210520/1515533
## 3 9712131/18360518
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0893-raw_1 and brca-0893-raw_2
## Single-end reads will be in brca-0893-raw_M
## 5074612 sequences in the BAM file
## 5074612 sequences exported
## 5074612 reads; of these:
##   5074612 (100.00%) were unpaired; of these:
##     1721773 (33.93%) aligned 0 times
##     1988943 (39.19%) aligned exactly 1 time
##     1363896 (26.88%) aligned &gt;1 times
## 66.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1363896/3352839
## 2 963404/1363896
## 3 11233146/20255177
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0894-raw_1 and brca-0894-raw_2
## Single-end reads will be in brca-0894-raw_M
## 2285110 sequences in the BAM file
## 2285110 sequences exported
## 2285110 reads; of these:
##   2285110 (100.00%) were unpaired; of these:
##     236259 (10.34%) aligned 0 times
##     1346750 (58.94%) aligned exactly 1 time
##     702101 (30.73%) aligned &gt;1 times
## 89.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 702101/2048851
## 2 531298/702101
## 3 3025274/7097165
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0895-raw_1 and brca-0895-raw_2
## Single-end reads will be in brca-0895-raw_M
## 6036665 sequences in the BAM file
## 6036665 sequences exported
## 6036665 reads; of these:
##   6036665 (100.00%) were unpaired; of these:
##     666936 (11.05%) aligned 0 times
##     3724651 (61.70%) aligned exactly 1 time
##     1645078 (27.25%) aligned &gt;1 times
## 88.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1645078/5369729
## 2 1273580/1645078
## 3 6715350/16031727
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0896-raw_1 and brca-0896-raw_2
## Single-end reads will be in brca-0896-raw_M
## 17200473 sequences in the BAM file
## 17200473 sequences exported
## 17200473 reads; of these:
##   17200473 (100.00%) were unpaired; of these:
##     3715374 (21.60%) aligned 0 times
##     7935068 (46.13%) aligned exactly 1 time
##     5550031 (32.27%) aligned &gt;1 times
## 78.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5550031/13485099
## 2 4647091/5550031
## 3 13793718/33607717
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-0897-raw_1 and brca-0897-raw_2
## Single-end reads will be in brca-0897-raw_M
## 5213226 sequences in the BAM file
## 5213226 sequences exported
## 5213226 reads; of these:
##   5213226 (100.00%) were unpaired; of these:
##     377822 (7.25%) aligned 0 times
##     3130038 (60.04%) aligned exactly 1 time
##     1705366 (32.71%) aligned &gt;1 times
## 92.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1705366/4835404
## 2 1366024/1705366
## 3 6363085/14475065
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0898-raw_1 and brca-0898-raw_2
## Single-end reads will be in brca-0898-raw_M
## 2957907 sequences in the BAM file
## 2957907 sequences exported
## 2957907 reads; of these:
##   2957907 (100.00%) were unpaired; of these:
##     589225 (19.92%) aligned 0 times
##     1146833 (38.77%) aligned exactly 1 time
##     1221849 (41.31%) aligned &gt;1 times
## 80.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1221849/2368682
## 2 1041497/1221849
## 3 3829892/7729121
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0899-raw_1 and brca-0899-raw_2
## Single-end reads will be in brca-0899-raw_M
## 8130385 sequences in the BAM file
## 8130385 sequences exported
## 8130385 reads; of these:
##   8130385 (100.00%) were unpaired; of these:
##     938362 (11.54%) aligned 0 times
##     4254793 (52.33%) aligned exactly 1 time
##     2937230 (36.13%) aligned &gt;1 times
## 88.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2937230/7192023
## 2 2480449/2937230
## 3 10466910/23001283
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-0900-raw_1 and brca-0900-raw_2
## Single-end reads will be in brca-0900-raw_M
## 4011565 sequences in the BAM file
## 4011565 sequences exported
## 4011565 reads; of these:
##   4011565 (100.00%) were unpaired; of these:
##     560285 (13.97%) aligned 0 times
##     2058717 (51.32%) aligned exactly 1 time
##     1392563 (34.71%) aligned &gt;1 times
## 86.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1392563/3451280
## 2 1204519/1392563
## 3 4997068/10908970
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0901-raw_1 and brca-0901-raw_2
## Single-end reads will be in brca-0901-raw_M
## 15129200 sequences in the BAM file
## 15129200 sequences exported
## 15129200 reads; of these:
##   15129200 (100.00%) were unpaired; of these:
##     3491249 (23.08%) aligned 0 times
##     4679066 (30.93%) aligned exactly 1 time
##     6958885 (46.00%) aligned &gt;1 times
## 76.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6958885/11637951
## 2 5746125/6958885
## 3 57761272/105495754
## [bam_sort_core] merging from 15 files...
## This looks like paired data from lane 1.
## Output will be in brca-0902-raw_1 and brca-0902-raw_2
## Single-end reads will be in brca-0902-raw_M
## 12435270 sequences in the BAM file
## 12435270 sequences exported
## 12435270 reads; of these:
##   12435270 (100.00%) were unpaired; of these:
##     2798850 (22.51%) aligned 0 times
##     5463601 (43.94%) aligned exactly 1 time
##     4172819 (33.56%) aligned &gt;1 times
## 77.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4172819/9636420
## 2 3346873/4172819
## 3 10993742/26498324
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0903-raw_1 and brca-0903-raw_2
## Single-end reads will be in brca-0903-raw_M
## 3874425 sequences in the BAM file
## 3874425 sequences exported
## 3874425 reads; of these:
##   3874425 (100.00%) were unpaired; of these:
##     747409 (19.29%) aligned 0 times
##     1305957 (33.71%) aligned exactly 1 time
##     1821059 (47.00%) aligned &gt;1 times
## 80.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1821059/3127016
## 2 1362055/1821059
## 3 13849618/26257120
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0904-raw_1 and brca-0904-raw_2
## Single-end reads will be in brca-0904-raw_M
## 5727251 sequences in the BAM file
## 5727251 sequences exported
## 5727251 reads; of these:
##   5727251 (100.00%) were unpaired; of these:
##     1396486 (24.38%) aligned 0 times
##     2940603 (51.34%) aligned exactly 1 time
##     1390162 (24.27%) aligned &gt;1 times
## 75.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1390162/4330765
## 2 1235933/1390162
## 3 2968148/8973582
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0905-raw_1 and brca-0905-raw_2
## Single-end reads will be in brca-0905-raw_M
## 2575451 sequences in the BAM file
## 2575451 sequences exported
## 2575451 reads; of these:
##   2575451 (100.00%) were unpaired; of these:
##     247795 (9.62%) aligned 0 times
##     1539485 (59.78%) aligned exactly 1 time
##     788171 (30.60%) aligned &gt;1 times
## 90.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 788171/2327656
## 2 561366/788171
## 3 4196405/9119394
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0906-raw_1 and brca-0906-raw_2
## Single-end reads will be in brca-0906-raw_M
## 2738470 sequences in the BAM file
## 2738470 sequences exported
## 2738470 reads; of these:
##   2738470 (100.00%) were unpaired; of these:
##     744820 (27.20%) aligned 0 times
##     1192766 (43.56%) aligned exactly 1 time
##     800884 (29.25%) aligned &gt;1 times
## 72.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 800884/1993650
## 2 676330/800884
## 3 2506258/5754459
## This looks like paired data from lane 2.
## Output will be in brca-0907-raw_1 and brca-0907-raw_2
## Single-end reads will be in brca-0907-raw_M
## 16069623 sequences in the BAM file
## 16069623 sequences exported
## 16069623 reads; of these:
##   16069623 (100.00%) were unpaired; of these:
##     4866544 (30.28%) aligned 0 times
##     7023850 (43.71%) aligned exactly 1 time
##     4179229 (26.01%) aligned &gt;1 times
## 69.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4179229/11203079
## 2 3266812/4179229
## 3 7867436/23580058
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0908-raw_1 and brca-0908-raw_2
## Single-end reads will be in brca-0908-raw_M
## 11349938 sequences in the BAM file
## 11349938 sequences exported
## 11349938 reads; of these:
##   11349938 (100.00%) were unpaired; of these:
##     2874257 (25.32%) aligned 0 times
##     5513017 (48.57%) aligned exactly 1 time
##     2962664 (26.10%) aligned &gt;1 times
## 74.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2962664/8475681
## 2 2195825/2962664
## 3 10642554/25696494
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0909-raw_1 and brca-0909-raw_2
## Single-end reads will be in brca-0909-raw_M
## 5203950 sequences in the BAM file
## 5203950 sequences exported
## 5203950 reads; of these:
##   5203950 (100.00%) were unpaired; of these:
##     439344 (8.44%) aligned 0 times
##     2855137 (54.86%) aligned exactly 1 time
##     1909469 (36.69%) aligned &gt;1 times
## 91.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1909469/4764606
## 2 1580354/1909469
## 3 7701446/15702372
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0910-raw_1 and brca-0910-raw_2
## Single-end reads will be in brca-0910-raw_M
## 3695771 sequences in the BAM file
## 3695771 sequences exported
## 3695771 reads; of these:
##   3695771 (100.00%) were unpaired; of these:
##     461173 (12.48%) aligned 0 times
##     2077881 (56.22%) aligned exactly 1 time
##     1156717 (31.30%) aligned &gt;1 times
## 87.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1156717/3234598
## 2 903244/1156717
## 3 5923671/13289627
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0911-raw_1 and brca-0911-raw_2
## Single-end reads will be in brca-0911-raw_M
## 2338895 sequences in the BAM file
## 2338895 sequences exported
## 2338895 reads; of these:
##   2338895 (100.00%) were unpaired; of these:
##     544062 (23.26%) aligned 0 times
##     939756 (40.18%) aligned exactly 1 time
##     855077 (36.56%) aligned &gt;1 times
## 76.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 855077/1794833
## 2 730012/855077
## 3 2149741/5035198
## This looks like paired data from lane 5.
## Output will be in brca-0912-raw_1 and brca-0912-raw_2
## Single-end reads will be in brca-0912-raw_M
## 5822284 sequences in the BAM file
## 5822284 sequences exported
## 5822284 reads; of these:
##   5822284 (100.00%) were unpaired; of these:
##     661368 (11.36%) aligned 0 times
##     2818689 (48.41%) aligned exactly 1 time
##     2342227 (40.23%) aligned &gt;1 times
## 88.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2342227/5160916
## 2 1930676/2342227
## 3 9788091/20301508
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0913-raw_1 and brca-0913-raw_2
## Single-end reads will be in brca-0913-raw_M
## 8565741 sequences in the BAM file
## 8565741 sequences exported
## 8565741 reads; of these:
##   8565741 (100.00%) were unpaired; of these:
##     2502823 (29.22%) aligned 0 times
##     2544390 (29.70%) aligned exactly 1 time
##     3518528 (41.08%) aligned &gt;1 times
## 70.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3518528/6062918
## 2 2432461/3518528
## 3 35574830/65391662
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 5.
## Output will be in brca-0914-raw_1 and brca-0914-raw_2
## Single-end reads will be in brca-0914-raw_M
## 4953537 sequences in the BAM file
## 4953537 sequences exported
## 4953537 reads; of these:
##   4953537 (100.00%) were unpaired; of these:
##     701263 (14.16%) aligned 0 times
##     2037420 (41.13%) aligned exactly 1 time
##     2214854 (44.71%) aligned &gt;1 times
## 85.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2214854/4252274
## 2 1819455/2214854
## 3 5657123/12537493
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0915-raw_1 and brca-0915-raw_2
## Single-end reads will be in brca-0915-raw_M
## 9736793 sequences in the BAM file
## 9736793 sequences exported
## 9736793 reads; of these:
##   9736793 (100.00%) were unpaired; of these:
##     2596331 (26.67%) aligned 0 times
##     5049044 (51.86%) aligned exactly 1 time
##     2091418 (21.48%) aligned &gt;1 times
## 73.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2091418/7140462
## 2 1460748/2091418
## 3 8042836/19978316
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0916-raw_1 and brca-0916-raw_2
## Single-end reads will be in brca-0916-raw_M
## 5032726 sequences in the BAM file
## 5032726 sequences exported
## 5032726 reads; of these:
##   5032726 (100.00%) were unpaired; of these:
##     1160361 (23.06%) aligned 0 times
##     2635194 (52.36%) aligned exactly 1 time
##     1237171 (24.58%) aligned &gt;1 times
## 76.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1237171/3872365
## 2 917612/1237171
## 3 4506731/10782337
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0917-raw_1 and brca-0917-raw_2
## Single-end reads will be in brca-0917-raw_M
## 2828376 sequences in the BAM file
## 2828376 sequences exported
## 2828376 reads; of these:
##   2828376 (100.00%) were unpaired; of these:
##     338063 (11.95%) aligned 0 times
##     1417384 (50.11%) aligned exactly 1 time
##     1072929 (37.93%) aligned &gt;1 times
## 88.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1072929/2490313
## 2 904691/1072929
## 3 4681870/9231826
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0918-raw_1 and brca-0918-raw_2
## Single-end reads will be in brca-0918-raw_M
## 5313252 sequences in the BAM file
## 5313252 sequences exported
## 5313252 reads; of these:
##   5313252 (100.00%) were unpaired; of these:
##     727728 (13.70%) aligned 0 times
##     2774279 (52.21%) aligned exactly 1 time
##     1811245 (34.09%) aligned &gt;1 times
## 86.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1811245/4585524
## 2 1486742/1811245
## 3 6212084/13854273
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0919-raw_1 and brca-0919-raw_2
## Single-end reads will be in brca-0919-raw_M
## 2351537 sequences in the BAM file
## 2351537 sequences exported
## 2351537 reads; of these:
##   2351537 (100.00%) were unpaired; of these:
##     595685 (25.33%) aligned 0 times
##     1224435 (52.07%) aligned exactly 1 time
##     531417 (22.60%) aligned &gt;1 times
## 74.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 531417/1755852
## 2 416625/531417
## 3 1693253/4568139
## This looks like paired data from lane 7.
## Output will be in brca-0920-raw_1 and brca-0920-raw_2
## Single-end reads will be in brca-0920-raw_M
## 2143179 sequences in the BAM file
## 2143179 sequences exported
## 2143179 reads; of these:
##   2143179 (100.00%) were unpaired; of these:
##     243318 (11.35%) aligned 0 times
##     1348070 (62.90%) aligned exactly 1 time
##     551791 (25.75%) aligned &gt;1 times
## 88.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 551791/1899861
## 2 429667/551791
## 3 2315856/5310499
## This looks like paired data from lane 3.
## Output will be in brca-0921-raw_1 and brca-0921-raw_2
## Single-end reads will be in brca-0921-raw_M
## 3169396 sequences in the BAM file
## 3169396 sequences exported
## 3169396 reads; of these:
##   3169396 (100.00%) were unpaired; of these:
##     455102 (14.36%) aligned 0 times
##     1433653 (45.23%) aligned exactly 1 time
##     1280641 (40.41%) aligned &gt;1 times
## 85.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1280641/2714294
## 2 971693/1280641
## 3 8029818/15713436
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0922-raw_1 and brca-0922-raw_2
## Single-end reads will be in brca-0922-raw_M
## 12669880 sequences in the BAM file
## 12669880 sequences exported
## 12669880 reads; of these:
##   12669880 (100.00%) were unpaired; of these:
##     3798723 (29.98%) aligned 0 times
##     4003954 (31.60%) aligned exactly 1 time
##     4867203 (38.42%) aligned &gt;1 times
## 70.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4867203/8871157
## 2 3898349/4867203
## 3 38075883/72902091
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in brca-0923-raw_1 and brca-0923-raw_2
## Single-end reads will be in brca-0923-raw_M
## 1570861 sequences in the BAM file
## 1570861 sequences exported
## 1570861 reads; of these:
##   1570861 (100.00%) were unpaired; of these:
##     221344 (14.09%) aligned 0 times
##     895574 (57.01%) aligned exactly 1 time
##     453943 (28.90%) aligned &gt;1 times
## 85.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 453943/1349517
## 2 345059/453943
## 3 1896882/4337626
## This looks like paired data from lane 7.
## Output will be in brca-0924-raw_1 and brca-0924-raw_2
## Single-end reads will be in brca-0924-raw_M
## 8607202 sequences in the BAM file
## 8607202 sequences exported
## 8607202 reads; of these:
##   8607202 (100.00%) were unpaired; of these:
##     1559922 (18.12%) aligned 0 times
##     4747337 (55.16%) aligned exactly 1 time
##     2299943 (26.72%) aligned &gt;1 times
## 81.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2299943/7047280
## 2 1613148/2299943
## 3 6418435/18049857
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-0925-raw_1 and brca-0925-raw_2
## Single-end reads will be in brca-0925-raw_M
## 4486810 sequences in the BAM file
## 4486810 sequences exported
## 4486810 reads; of these:
##   4486810 (100.00%) were unpaired; of these:
##     883561 (19.69%) aligned 0 times
##     2103452 (46.88%) aligned exactly 1 time
##     1499797 (33.43%) aligned &gt;1 times
## 80.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1499797/3603249
## 2 1212668/1499797
## 3 4598556/10378339
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0926-raw_1 and brca-0926-raw_2
## Single-end reads will be in brca-0926-raw_M
## 4197370 sequences in the BAM file
## 4197370 sequences exported
## 4197370 reads; of these:
##   4197370 (100.00%) were unpaired; of these:
##     537355 (12.80%) aligned 0 times
##     2086168 (49.70%) aligned exactly 1 time
##     1573847 (37.50%) aligned &gt;1 times
## 87.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1573847/3660015
## 2 1266725/1573847
## 3 8248300/16971822
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0927-raw_1 and brca-0927-raw_2
## Single-end reads will be in brca-0927-raw_M
## 5070652 sequences in the BAM file
## 5070652 sequences exported
## 5070652 reads; of these:
##   5070652 (100.00%) were unpaired; of these:
##     587029 (11.58%) aligned 0 times
##     2186804 (43.13%) aligned exactly 1 time
##     2296819 (45.30%) aligned &gt;1 times
## 88.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2296819/4483623
## 2 2000930/2296819
## 3 7538115/15717667
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0928-raw_1 and brca-0928-raw_2
## Single-end reads will be in brca-0928-raw_M
## 4467379 sequences in the BAM file
## 4467379 sequences exported
## 4467379 reads; of these:
##   4467379 (100.00%) were unpaired; of these:
##     945480 (21.16%) aligned 0 times
##     1912631 (42.81%) aligned exactly 1 time
##     1609268 (36.02%) aligned &gt;1 times
## 78.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1609268/3521899
## 2 1354013/1609268
## 3 4194826/9553348
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0929-raw_1 and brca-0929-raw_2
## Single-end reads will be in brca-0929-raw_M
## 2080164 sequences in the BAM file
## 2080164 sequences exported
## 2080164 reads; of these:
##   2080164 (100.00%) were unpaired; of these:
##     319811 (15.37%) aligned 0 times
##     1075985 (51.73%) aligned exactly 1 time
##     684368 (32.90%) aligned &gt;1 times
## 84.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 684368/1760353
## 2 519596/684368
## 3 4291964/9083451
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0930-raw_1 and brca-0930-raw_2
## Single-end reads will be in brca-0930-raw_M
## 29548363 sequences in the BAM file
## 29548363 sequences exported
## 29548363 reads; of these:
##   29548363 (100.00%) were unpaired; of these:
##     7037949 (23.82%) aligned 0 times
##     10104698 (34.20%) aligned exactly 1 time
##     12405716 (41.98%) aligned &gt;1 times
## 76.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 12405716/22510414
## 2 10934638/12405716
## 3 21447431/51713930
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in brca-0931-raw_1 and brca-0931-raw_2
## Single-end reads will be in brca-0931-raw_M
## 2851466 sequences in the BAM file
## 2851466 sequences exported
## 2851466 reads; of these:
##   2851466 (100.00%) were unpaired; of these:
##     228773 (8.02%) aligned 0 times
##     1328950 (46.61%) aligned exactly 1 time
##     1293743 (45.37%) aligned &gt;1 times
## 91.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1293743/2622693
## 2 931719/1293743
## 3 10341256/20653851
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0932-raw_1 and brca-0932-raw_2
## Single-end reads will be in brca-0932-raw_M
## 11519716 sequences in the BAM file
## 11519716 sequences exported
## 11519716 reads; of these:
##   11519716 (100.00%) were unpaired; of these:
##     2446637 (21.24%) aligned 0 times
##     5109509 (44.35%) aligned exactly 1 time
##     3963570 (34.41%) aligned &gt;1 times
## 78.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3963570/9073079
## 2 3009384/3963570
## 3 10531246/25865980
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-0933-raw_1 and brca-0933-raw_2
## Single-end reads will be in brca-0933-raw_M
## 22920093 sequences in the BAM file
## 22920093 sequences exported
## 22920093 reads; of these:
##   22920093 (100.00%) were unpaired; of these:
##     6264219 (27.33%) aligned 0 times
##     11603620 (50.63%) aligned exactly 1 time
##     5052254 (22.04%) aligned &gt;1 times
## 72.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5052254/16655874
## 2 3918897/5052254
## 3 12992028/35995392
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in brca-0934-raw_1 and brca-0934-raw_2
## Single-end reads will be in brca-0934-raw_M
## 4807227 sequences in the BAM file
## 4807227 sequences exported
## 4807227 reads; of these:
##   4807227 (100.00%) were unpaired; of these:
##     326437 (6.79%) aligned 0 times
##     3129142 (65.09%) aligned exactly 1 time
##     1351648 (28.12%) aligned &gt;1 times
## 93.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1351648/4480790
## 2 1151924/1351648
## 3 4331356/10941188
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0935-raw_1 and brca-0935-raw_2
## Single-end reads will be in brca-0935-raw_M
## 2398746 sequences in the BAM file
## 2398746 sequences exported
## 2398746 reads; of these:
##   2398746 (100.00%) were unpaired; of these:
##     528630 (22.04%) aligned 0 times
##     1124309 (46.87%) aligned exactly 1 time
##     745807 (31.09%) aligned &gt;1 times
## 77.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 745807/1870116
## 2 645339/745807
## 3 2500570/5942837
## This looks like paired data from lane 3.
## Output will be in brca-0936-raw_1 and brca-0936-raw_2
## Single-end reads will be in brca-0936-raw_M
## 1947268 sequences in the BAM file
## 1947268 sequences exported
## 1947268 reads; of these:
##   1947268 (100.00%) were unpaired; of these:
##     447959 (23.00%) aligned 0 times
##     923590 (47.43%) aligned exactly 1 time
##     575719 (29.57%) aligned &gt;1 times
## 77.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 575719/1499309
## 2 383432/575719
## 3 2253480/5353475
## This looks like paired data from lane 2.
## Output will be in brca-0937-raw_1 and brca-0937-raw_2
## Single-end reads will be in brca-0937-raw_M
## 4387277 sequences in the BAM file
## 4387277 sequences exported
## 4387277 reads; of these:
##   4387277 (100.00%) were unpaired; of these:
##     836224 (19.06%) aligned 0 times
##     1810831 (41.27%) aligned exactly 1 time
##     1740222 (39.67%) aligned &gt;1 times
## 80.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1740222/3551053
## 2 1404080/1740222
## 3 14494256/27983500
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-0938-raw_1 and brca-0938-raw_2
## Single-end reads will be in brca-0938-raw_M
## 5270084 sequences in the BAM file
## 5270084 sequences exported
## 5270084 reads; of these:
##   5270084 (100.00%) were unpaired; of these:
##     1482745 (28.14%) aligned 0 times
##     2427131 (46.05%) aligned exactly 1 time
##     1360208 (25.81%) aligned &gt;1 times
## 71.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1360208/3787339
## 2 1029619/1360208
## 3 5373508/12308529
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0939-raw_1 and brca-0939-raw_2
## Single-end reads will be in brca-0939-raw_M
## 6019852 sequences in the BAM file
## 6019852 sequences exported
## 6019852 reads; of these:
##   6019852 (100.00%) were unpaired; of these:
##     1538526 (25.56%) aligned 0 times
##     2519639 (41.86%) aligned exactly 1 time
##     1961687 (32.59%) aligned &gt;1 times
## 74.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1961687/4481326
## 2 1669235/1961687
## 3 5872323/13042465
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0940-raw_1 and brca-0940-raw_2
## Single-end reads will be in brca-0940-raw_M
## 2623404 sequences in the BAM file
## 2623404 sequences exported
## 2623404 reads; of these:
##   2623404 (100.00%) were unpaired; of these:
##     370025 (14.10%) aligned 0 times
##     1431855 (54.58%) aligned exactly 1 time
##     821524 (31.32%) aligned &gt;1 times
## 85.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 821524/2253379
## 2 689576/821524
## 3 2248327/5651924
## This looks like paired data from lane 2.
## Output will be in brca-0941-raw_1 and brca-0941-raw_2
## Single-end reads will be in brca-0941-raw_M
## 12371791 sequences in the BAM file
## 12371791 sequences exported
## 12371791 reads; of these:
##   12371791 (100.00%) were unpaired; of these:
##     3633297 (29.37%) aligned 0 times
##     4919811 (39.77%) aligned exactly 1 time
##     3818683 (30.87%) aligned &gt;1 times
## 70.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3818683/8738494
## 2 3289388/3818683
## 3 9184534/22735198
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0942-raw_1 and brca-0942-raw_2
## Single-end reads will be in brca-0942-raw_M
## 12025838 sequences in the BAM file
## 12025838 sequences exported
## 12025838 reads; of these:
##   12025838 (100.00%) were unpaired; of these:
##     1003428 (8.34%) aligned 0 times
##     6600818 (54.89%) aligned exactly 1 time
##     4421592 (36.77%) aligned &gt;1 times
## 91.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4421592/11022410
## 2 3539276/4421592
## 3 16604297/37321923
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0943-raw_1 and brca-0943-raw_2
## Single-end reads will be in brca-0943-raw_M
## 3488998 sequences in the BAM file
## 3488998 sequences exported
## 3488998 reads; of these:
##   3488998 (100.00%) were unpaired; of these:
##     883050 (25.31%) aligned 0 times
##     1471459 (42.17%) aligned exactly 1 time
##     1134489 (32.52%) aligned &gt;1 times
## 74.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1134489/2605948
## 2 943934/1134489
## 3 4839145/9902941
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0944-raw_1 and brca-0944-raw_2
## Single-end reads will be in brca-0944-raw_M
## 5261806 sequences in the BAM file
## 5261806 sequences exported
## 5261806 reads; of these:
##   5261806 (100.00%) were unpaired; of these:
##     645408 (12.27%) aligned 0 times
##     2438870 (46.35%) aligned exactly 1 time
##     2177528 (41.38%) aligned &gt;1 times
## 87.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2177528/4616398
## 2 1872499/2177528
## 3 9062425/18574837
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0945-raw_1 and brca-0945-raw_2
## Single-end reads will be in brca-0945-raw_M
## 3225047 sequences in the BAM file
## 3225047 sequences exported
## 3225047 reads; of these:
##   3225047 (100.00%) were unpaired; of these:
##     885461 (27.46%) aligned 0 times
##     1425552 (44.20%) aligned exactly 1 time
##     914034 (28.34%) aligned &gt;1 times
## 72.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 914034/2339586
## 2 754539/914034
## 3 3682811/7661855
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0946-raw_1 and brca-0946-raw_2
## Single-end reads will be in brca-0946-raw_M
## 8956935 sequences in the BAM file
## 8956935 sequences exported
## 8956935 reads; of these:
##   8956935 (100.00%) were unpaired; of these:
##     2036344 (22.73%) aligned 0 times
##     3752290 (41.89%) aligned exactly 1 time
##     3168301 (35.37%) aligned &gt;1 times
## 77.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3168301/6920591
## 2 2754088/3168301
## 3 11531463/25876478
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-0947-raw_1 and brca-0947-raw_2
## Single-end reads will be in brca-0947-raw_M
## 6311941 sequences in the BAM file
## 6311941 sequences exported
## 6311941 reads; of these:
##   6311941 (100.00%) were unpaired; of these:
##     305930 (4.85%) aligned 0 times
##     3314986 (52.52%) aligned exactly 1 time
##     2691025 (42.63%) aligned &gt;1 times
## 95.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2691025/6006011
## 2 2241982/2691025
## 3 10677342/20741407
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0948-raw_1 and brca-0948-raw_2
## Single-end reads will be in brca-0948-raw_M
## 16467303 sequences in the BAM file
## 16467303 sequences exported
## 16467303 reads; of these:
##   16467303 (100.00%) were unpaired; of these:
##     832958 (5.06%) aligned 0 times
##     8392270 (50.96%) aligned exactly 1 time
##     7242075 (43.98%) aligned &gt;1 times
## 94.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7242075/15634345
## 2 5518217/7242075
## 3 53418861/96447614
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 4.
## Output will be in brca-0949-raw_1 and brca-0949-raw_2
## Single-end reads will be in brca-0949-raw_M
## 8509608 sequences in the BAM file
## 8509608 sequences exported
## 8509608 reads; of these:
##   8509608 (100.00%) were unpaired; of these:
##     2263489 (26.60%) aligned 0 times
##     3163626 (37.18%) aligned exactly 1 time
##     3082493 (36.22%) aligned &gt;1 times
## 73.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3082493/6246119
## 2 2537090/3082493
## 3 14187069/27891004
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0950-raw_1 and brca-0950-raw_2
## Single-end reads will be in brca-0950-raw_M
## 9270911 sequences in the BAM file
## 9270911 sequences exported
## 9270911 reads; of these:
##   9270911 (100.00%) were unpaired; of these:
##     2688726 (29.00%) aligned 0 times
##     3889427 (41.95%) aligned exactly 1 time
##     2692758 (29.05%) aligned &gt;1 times
## 71.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2692758/6582185
## 2 2053175/2692758
## 3 11316447/25852824
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-0951-raw_1 and brca-0951-raw_2
## Single-end reads will be in brca-0951-raw_M
## 7947185 sequences in the BAM file
## 7947185 sequences exported
## 7947185 reads; of these:
##   7947185 (100.00%) were unpaired; of these:
##     834285 (10.50%) aligned 0 times
##     3870402 (48.70%) aligned exactly 1 time
##     3242498 (40.80%) aligned &gt;1 times
## 89.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3242498/7112900
## 2 2931165/3242498
## 3 10333987/21495693
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-0952-raw_1 and brca-0952-raw_2
## Single-end reads will be in brca-0952-raw_M
## 12844300 sequences in the BAM file
## 12844300 sequences exported
## 12844300 reads; of these:
##   12844300 (100.00%) were unpaired; of these:
##     3095068 (24.10%) aligned 0 times
##     5643014 (43.93%) aligned exactly 1 time
##     4106218 (31.97%) aligned &gt;1 times
## 75.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4106218/9749232
## 2 3392349/4106218
## 3 7115658/20165833
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-0953-raw_1 and brca-0953-raw_2
## Single-end reads will be in brca-0953-raw_M
## 4362395 sequences in the BAM file
## 4362395 sequences exported
## 4362395 reads; of these:
##   4362395 (100.00%) were unpaired; of these:
##     323068 (7.41%) aligned 0 times
##     2348096 (53.83%) aligned exactly 1 time
##     1691231 (38.77%) aligned &gt;1 times
## 92.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1691231/4039327
## 2 1449236/1691231
## 3 5667704/12335823
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0954-raw_1 and brca-0954-raw_2
## Single-end reads will be in brca-0954-raw_M
## 3345518 sequences in the BAM file
## 3345518 sequences exported
## 3345518 reads; of these:
##   3345518 (100.00%) were unpaired; of these:
##     533282 (15.94%) aligned 0 times
##     1944209 (58.11%) aligned exactly 1 time
##     868027 (25.95%) aligned &gt;1 times
## 84.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 868027/2812236
## 2 713687/868027
## 3 4009876/8939058
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0955-raw_1 and brca-0955-raw_2
## Single-end reads will be in brca-0955-raw_M
## 11475715 sequences in the BAM file
## 11475715 sequences exported
## 11475715 reads; of these:
##   11475715 (100.00%) were unpaired; of these:
##     3304302 (28.79%) aligned 0 times
##     5044731 (43.96%) aligned exactly 1 time
##     3126682 (27.25%) aligned &gt;1 times
## 71.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3126682/8171413
## 2 2418791/3126682
## 3 15777959/33271723
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0956-raw_1 and brca-0956-raw_2
## Single-end reads will be in brca-0956-raw_M
## 3960842 sequences in the BAM file
## 3960842 sequences exported
## 3960842 reads; of these:
##   3960842 (100.00%) were unpaired; of these:
##     1047627 (26.45%) aligned 0 times
##     1820795 (45.97%) aligned exactly 1 time
##     1092420 (27.58%) aligned &gt;1 times
## 73.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1092420/2913215
## 2 658374/1092420
## 3 3954179/9835937
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0957-raw_1 and brca-0957-raw_2
## Single-end reads will be in brca-0957-raw_M
## 6471171 sequences in the BAM file
## 6471171 sequences exported
## 6471171 reads; of these:
##   6471171 (100.00%) were unpaired; of these:
##     1416407 (21.89%) aligned 0 times
##     2914242 (45.03%) aligned exactly 1 time
##     2140522 (33.08%) aligned &gt;1 times
## 78.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2140522/5054764
## 2 1837954/2140522
## 3 5046401/12236795
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0958-raw_1 and brca-0958-raw_2
## Single-end reads will be in brca-0958-raw_M
## 7363583 sequences in the BAM file
## 7363583 sequences exported
## 7363583 reads; of these:
##   7363583 (100.00%) were unpaired; of these:
##     2539816 (34.49%) aligned 0 times
##     2463710 (33.46%) aligned exactly 1 time
##     2360057 (32.05%) aligned &gt;1 times
## 65.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2360057/4823767
## 2 1762632/2360057
## 3 8386093/16605858
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-0959-raw_1 and brca-0959-raw_2
## Single-end reads will be in brca-0959-raw_M
## 3177343 sequences in the BAM file
## 3177343 sequences exported
## 3177343 reads; of these:
##   3177343 (100.00%) were unpaired; of these:
##     807214 (25.41%) aligned 0 times
##     1559278 (49.07%) aligned exactly 1 time
##     810851 (25.52%) aligned &gt;1 times
## 74.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 810851/2370129
## 2 635203/810851
## 3 3716153/8706453
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0960-raw_1 and brca-0960-raw_2
## Single-end reads will be in brca-0960-raw_M
## 4689225 sequences in the BAM file
## 4689225 sequences exported
## 4689225 reads; of these:
##   4689225 (100.00%) were unpaired; of these:
##     1592649 (33.96%) aligned 0 times
##     1614623 (34.43%) aligned exactly 1 time
##     1481953 (31.60%) aligned &gt;1 times
## 66.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1481953/3096576
## 2 1166981/1481953
## 3 5307244/11938963
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-0961-raw_1 and brca-0961-raw_2
## Single-end reads will be in brca-0961-raw_M
## 5637930 sequences in the BAM file
## 5637930 sequences exported
## 5637930 reads; of these:
##   5637930 (100.00%) were unpaired; of these:
##     1256444 (22.29%) aligned 0 times
##     2382344 (42.26%) aligned exactly 1 time
##     1999142 (35.46%) aligned &gt;1 times
## 77.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1999142/4381486
## 2 1380758/1999142
## 3 14146832/28949443
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-0962-raw_1 and brca-0962-raw_2
## Single-end reads will be in brca-0962-raw_M
## 14323732 sequences in the BAM file
## 14323732 sequences exported
## 14323732 reads; of these:
##   14323732 (100.00%) were unpaired; of these:
##     4288776 (29.94%) aligned 0 times
##     4845146 (33.83%) aligned exactly 1 time
##     5189810 (36.23%) aligned &gt;1 times
## 70.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5189810/10034956
## 2 4393801/5189810
## 3 30574231/59326315
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in brca-0963-raw_1 and brca-0963-raw_2
## Single-end reads will be in brca-0963-raw_M
## 11181472 sequences in the BAM file
## 11181472 sequences exported
## 11181472 reads; of these:
##   11181472 (100.00%) were unpaired; of these:
##     2422040 (21.66%) aligned 0 times
##     4296424 (38.42%) aligned exactly 1 time
##     4463008 (39.91%) aligned &gt;1 times
## 78.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4463008/8759432
## 2 3618902/4463008
## 3 10257145/24636531
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0964-raw_1 and brca-0964-raw_2
## Single-end reads will be in brca-0964-raw_M
## 4232279 sequences in the BAM file
## 4232279 sequences exported
## 4232279 reads; of these:
##   4232279 (100.00%) were unpaired; of these:
##     832955 (19.68%) aligned 0 times
##     2011813 (47.53%) aligned exactly 1 time
##     1387511 (32.78%) aligned &gt;1 times
## 80.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1387511/3399324
## 2 954247/1387511
## 3 11578996/22492737
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0965-raw_1 and brca-0965-raw_2
## Single-end reads will be in brca-0965-raw_M
## 10723298 sequences in the BAM file
## 10723298 sequences exported
## 10723298 reads; of these:
##   10723298 (100.00%) were unpaired; of these:
##     1898941 (17.71%) aligned 0 times
##     5961490 (55.59%) aligned exactly 1 time
##     2862867 (26.70%) aligned &gt;1 times
## 82.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2862867/8824357
## 2 2136452/2862867
## 3 13760536/30682298
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0966-raw_1 and brca-0966-raw_2
## Single-end reads will be in brca-0966-raw_M
## 3558003 sequences in the BAM file
## 3558003 sequences exported
## 3558003 reads; of these:
##   3558003 (100.00%) were unpaired; of these:
##     804002 (22.60%) aligned 0 times
##     1561827 (43.90%) aligned exactly 1 time
##     1192174 (33.51%) aligned &gt;1 times
## 77.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1192174/2754001
## 2 938861/1192174
## 3 6288430/12898253
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0967-raw_1 and brca-0967-raw_2
## Single-end reads will be in brca-0967-raw_M
## 11406600 sequences in the BAM file
## 11406600 sequences exported
## 11406600 reads; of these:
##   11406600 (100.00%) were unpaired; of these:
##     3192313 (27.99%) aligned 0 times
##     4605997 (40.38%) aligned exactly 1 time
##     3608290 (31.63%) aligned &gt;1 times
## 72.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3608290/8214287
## 2 2992773/3608290
## 3 13714871/30466694
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-0968-raw_1 and brca-0968-raw_2
## Single-end reads will be in brca-0968-raw_M
## 2402405 sequences in the BAM file
## 2402405 sequences exported
## 2402405 reads; of these:
##   2402405 (100.00%) were unpaired; of these:
##     638228 (26.57%) aligned 0 times
##     983131 (40.92%) aligned exactly 1 time
##     781046 (32.51%) aligned &gt;1 times
## 73.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 781046/1764177
## 2 667642/781046
## 3 2300018/5438555
## This looks like paired data from lane 3.
## Output will be in brca-0969-raw_1 and brca-0969-raw_2
## Single-end reads will be in brca-0969-raw_M
## 3117676 sequences in the BAM file
## 3117676 sequences exported
## 3117676 reads; of these:
##   3117676 (100.00%) were unpaired; of these:
##     560226 (17.97%) aligned 0 times
##     1648149 (52.86%) aligned exactly 1 time
##     909301 (29.17%) aligned &gt;1 times
## 82.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 909301/2557450
## 2 734940/909301
## 3 3223627/7804939
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0970-raw_1 and brca-0970-raw_2
## Single-end reads will be in brca-0970-raw_M
## 4514426 sequences in the BAM file
## 4514426 sequences exported
## 4514426 reads; of these:
##   4514426 (100.00%) were unpaired; of these:
##     337378 (7.47%) aligned 0 times
##     2714840 (60.14%) aligned exactly 1 time
##     1462208 (32.39%) aligned &gt;1 times
## 92.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1462208/4177048
## 2 1201441/1462208
## 3 4948833/10958462
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0971-raw_1 and brca-0971-raw_2
## Single-end reads will be in brca-0971-raw_M
## 3672682 sequences in the BAM file
## 3672682 sequences exported
## 3672682 reads; of these:
##   3672682 (100.00%) were unpaired; of these:
##     828936 (22.57%) aligned 0 times
##     1666201 (45.37%) aligned exactly 1 time
##     1177545 (32.06%) aligned &gt;1 times
## 77.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1177545/2843746
## 2 979263/1177545
## 3 3046483/7663213
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0972-raw_1 and brca-0972-raw_2
## Single-end reads will be in brca-0972-raw_M
## 3604106 sequences in the BAM file
## 3604106 sequences exported
## 3604106 reads; of these:
##   3604106 (100.00%) were unpaired; of these:
##     293832 (8.15%) aligned 0 times
##     1821201 (50.53%) aligned exactly 1 time
##     1489073 (41.32%) aligned &gt;1 times
## 91.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1489073/3310274
## 2 1212884/1489073
## 3 4395886/9699896
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-0973-raw_1 and brca-0973-raw_2
## Single-end reads will be in brca-0973-raw_M
## 10608010 sequences in the BAM file
## 10608010 sequences exported
## 10608010 reads; of these:
##   10608010 (100.00%) were unpaired; of these:
##     1888965 (17.81%) aligned 0 times
##     5190833 (48.93%) aligned exactly 1 time
##     3528212 (33.26%) aligned &gt;1 times
## 82.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3528212/8719045
## 2 2600322/3528212
## 3 13546005/30904424
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-0974-raw_1 and brca-0974-raw_2
## Single-end reads will be in brca-0974-raw_M
## 16882284 sequences in the BAM file
## 16882284 sequences exported
## 16882284 reads; of these:
##   16882284 (100.00%) were unpaired; of these:
##     4135178 (24.49%) aligned 0 times
##     8912692 (52.79%) aligned exactly 1 time
##     3834414 (22.71%) aligned &gt;1 times
## 75.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3834414/12747106
## 2 2459089/3834414
## 3 14636992/37945985
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in brca-0975-raw_1 and brca-0975-raw_2
## Single-end reads will be in brca-0975-raw_M
## 8895747 sequences in the BAM file
## 8895747 sequences exported
## 8895747 reads; of these:
##   8895747 (100.00%) were unpaired; of these:
##     2390261 (26.87%) aligned 0 times
##     4272711 (48.03%) aligned exactly 1 time
##     2232775 (25.10%) aligned &gt;1 times
## 73.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2232775/6505486
## 2 1815426/2232775
## 3 7707900/18576653
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-0976-raw_1 and brca-0976-raw_2
## Single-end reads will be in brca-0976-raw_M
## 4410955 sequences in the BAM file
## 4410955 sequences exported
## 4410955 reads; of these:
##   4410955 (100.00%) were unpaired; of these:
##     354306 (8.03%) aligned 0 times
##     2515971 (57.04%) aligned exactly 1 time
##     1540678 (34.93%) aligned &gt;1 times
## 91.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1540678/4056649
## 2 1213127/1540678
## 3 7698335/15539755
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0977-raw_1 and brca-0977-raw_2
## Single-end reads will be in brca-0977-raw_M
## 16673196 sequences in the BAM file
## 16673196 sequences exported
## 16673196 reads; of these:
##   16673196 (100.00%) were unpaired; of these:
##     4252560 (25.51%) aligned 0 times
##     6475585 (38.84%) aligned exactly 1 time
##     5945051 (35.66%) aligned &gt;1 times
## 74.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5945051/12420636
## 2 4560248/5945051
## 3 30931738/59704229
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in brca-0978-raw_1 and brca-0978-raw_2
## Single-end reads will be in brca-0978-raw_M
## 6974986 sequences in the BAM file
## 6974986 sequences exported
## 6974986 reads; of these:
##   6974986 (100.00%) were unpaired; of these:
##     2051216 (29.41%) aligned 0 times
##     3049354 (43.72%) aligned exactly 1 time
##     1874416 (26.87%) aligned &gt;1 times
## 70.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1874416/4923770
## 2 1481157/1874416
## 3 6505122/15254775
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-0979-raw_1 and brca-0979-raw_2
## Single-end reads will be in brca-0979-raw_M
## 9781857 sequences in the BAM file
## 9781857 sequences exported
## 9781857 reads; of these:
##   9781857 (100.00%) were unpaired; of these:
##     412208 (4.21%) aligned 0 times
##     5074407 (51.88%) aligned exactly 1 time
##     4295242 (43.91%) aligned &gt;1 times
## 95.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4295242/9369649
## 2 3627773/4295242
## 3 23463709/43885726
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0980-raw_1 and brca-0980-raw_2
## Single-end reads will be in brca-0980-raw_M
## 13310751 sequences in the BAM file
## 13310751 sequences exported
## 13310751 reads; of these:
##   13310751 (100.00%) were unpaired; of these:
##     3135817 (23.56%) aligned 0 times
##     7250306 (54.47%) aligned exactly 1 time
##     2924628 (21.97%) aligned &gt;1 times
## 76.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2924628/10174934
## 2 1931699/2924628
## 3 19990464/39502765
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in brca-0981-raw_1 and brca-0981-raw_2
## Single-end reads will be in brca-0981-raw_M
## 3632893 sequences in the BAM file
## 3632893 sequences exported
## 3632893 reads; of these:
##   3632893 (100.00%) were unpaired; of these:
##     793378 (21.84%) aligned 0 times
##     1885343 (51.90%) aligned exactly 1 time
##     954172 (26.26%) aligned &gt;1 times
## 78.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 954172/2839515
## 2 711907/954172
## 3 5032862/11171407
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-0982-raw_1 and brca-0982-raw_2
## Single-end reads will be in brca-0982-raw_M
## 2836671 sequences in the BAM file
## 2836671 sequences exported
## 2836671 reads; of these:
##   2836671 (100.00%) were unpaired; of these:
##     324217 (11.43%) aligned 0 times
##     1148750 (40.50%) aligned exactly 1 time
##     1363704 (48.07%) aligned &gt;1 times
## 88.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1363704/2512454
## 2 1248767/1363704
## 3 3153357/7145219
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0983-raw_1 and brca-0983-raw_2
## Single-end reads will be in brca-0983-raw_M
## 10603750 sequences in the BAM file
## 10603750 sequences exported
## 10603750 reads; of these:
##   10603750 (100.00%) were unpaired; of these:
##     3109466 (29.32%) aligned 0 times
##     5059521 (47.71%) aligned exactly 1 time
##     2434763 (22.96%) aligned &gt;1 times
## 70.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2434763/7494284
## 2 1961024/2434763
## 3 5598095/16035481
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in brca-0984-raw_1 and brca-0984-raw_2
## Single-end reads will be in brca-0984-raw_M
## 19673224 sequences in the BAM file
## 19673224 sequences exported
## 19673224 reads; of these:
##   19673224 (100.00%) were unpaired; of these:
##     5393686 (27.42%) aligned 0 times
##     8407664 (42.74%) aligned exactly 1 time
##     5871874 (29.85%) aligned &gt;1 times
## 72.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5871874/14279538
## 2 4927239/5871874
## 3 12048999/31975341
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-0985-raw_1 and brca-0985-raw_2
## Single-end reads will be in brca-0985-raw_M
## 19360518 sequences in the BAM file
## 19360518 sequences exported
## 19360518 reads; of these:
##   19360518 (100.00%) were unpaired; of these:
##     5035796 (26.01%) aligned 0 times
##     8992752 (46.45%) aligned exactly 1 time
##     5331970 (27.54%) aligned &gt;1 times
## 73.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5331970/14324722
## 2 4002591/5331970
## 3 24246990/50053435
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in brca-0986-raw_1 and brca-0986-raw_2
## Single-end reads will be in brca-0986-raw_M
## 4378138 sequences in the BAM file
## 4378138 sequences exported
## 4378138 reads; of these:
##   4378138 (100.00%) were unpaired; of these:
##     1209805 (27.63%) aligned 0 times
##     1861808 (42.53%) aligned exactly 1 time
##     1306525 (29.84%) aligned &gt;1 times
## 72.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1306525/3168333
## 2 1080545/1306525
## 3 4539601/9797634
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0987-raw_1 and brca-0987-raw_2
## Single-end reads will be in brca-0987-raw_M
## 5655829 sequences in the BAM file
## 5655829 sequences exported
## 5655829 reads; of these:
##   5655829 (100.00%) were unpaired; of these:
##     1347319 (23.82%) aligned 0 times
##     2278808 (40.29%) aligned exactly 1 time
##     2029702 (35.89%) aligned &gt;1 times
## 76.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2029702/4308510
## 2 1729510/2029702
## 3 10730277/21779021
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-0988-raw_1 and brca-0988-raw_2
## Single-end reads will be in brca-0988-raw_M
## 4329666 sequences in the BAM file
## 4329666 sequences exported
## 4329666 reads; of these:
##   4329666 (100.00%) were unpaired; of these:
##     477591 (11.03%) aligned 0 times
##     2137743 (49.37%) aligned exactly 1 time
##     1714332 (39.60%) aligned &gt;1 times
## 88.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1714332/3852075
## 2 1348757/1714332
## 3 9711867/18805753
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-0989-raw_1 and brca-0989-raw_2
## Single-end reads will be in brca-0989-raw_M
## 5771531 sequences in the BAM file
## 5771531 sequences exported
## 5771531 reads; of these:
##   5771531 (100.00%) were unpaired; of these:
##     1562607 (27.07%) aligned 0 times
##     2313740 (40.09%) aligned exactly 1 time
##     1895184 (32.84%) aligned &gt;1 times
## 72.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1895184/4208924
## 2 1572527/1895184
## 3 6653951/13963367
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-0990-raw_1 and brca-0990-raw_2
## Single-end reads will be in brca-0990-raw_M
## 2922223 sequences in the BAM file
## 2922223 sequences exported
## 2922223 reads; of these:
##   2922223 (100.00%) were unpaired; of these:
##     889024 (30.42%) aligned 0 times
##     1055872 (36.13%) aligned exactly 1 time
##     977327 (33.44%) aligned &gt;1 times
## 69.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 977327/2033199
## 2 766661/977327
## 3 6541591/12754411
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0991-raw_1 and brca-0991-raw_2
## Single-end reads will be in brca-0991-raw_M
## 8512862 sequences in the BAM file
## 8512862 sequences exported
## 8512862 reads; of these:
##   8512862 (100.00%) were unpaired; of these:
##     2075176 (24.38%) aligned 0 times
##     4342281 (51.01%) aligned exactly 1 time
##     2095405 (24.61%) aligned &gt;1 times
## 75.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2095405/6437686
## 2 1680319/2095405
## 3 5793873/14954745
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-0992-raw_1 and brca-0992-raw_2
## Single-end reads will be in brca-0992-raw_M
## 2302076 sequences in the BAM file
## 2302076 sequences exported
## 2302076 reads; of these:
##   2302076 (100.00%) were unpaired; of these:
##     263972 (11.47%) aligned 0 times
##     1426841 (61.98%) aligned exactly 1 time
##     611263 (26.55%) aligned &gt;1 times
## 88.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 611263/2038104
## 2 455230/611263
## 3 3488435/7731997
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0993-raw_1 and brca-0993-raw_2
## Single-end reads will be in brca-0993-raw_M
## 4661439 sequences in the BAM file
## 4661439 sequences exported
## 4661439 reads; of these:
##   4661439 (100.00%) were unpaired; of these:
##     290454 (6.23%) aligned 0 times
##     2819469 (60.48%) aligned exactly 1 time
##     1551516 (33.28%) aligned &gt;1 times
## 93.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1551516/4370985
## 2 1251779/1551516
## 3 5571488/12692151
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0994-raw_1 and brca-0994-raw_2
## Single-end reads will be in brca-0994-raw_M
## 2527302 sequences in the BAM file
## 2527302 sequences exported
## 2527302 reads; of these:
##   2527302 (100.00%) were unpaired; of these:
##     645978 (25.56%) aligned 0 times
##     839707 (33.23%) aligned exactly 1 time
##     1041617 (41.21%) aligned &gt;1 times
## 74.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1041617/1881324
## 2 743284/1041617
## 3 6020594/11464701
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-0995-raw_1 and brca-0995-raw_2
## Single-end reads will be in brca-0995-raw_M
## 2501970 sequences in the BAM file
## 2501970 sequences exported
## 2501970 reads; of these:
##   2501970 (100.00%) were unpaired; of these:
##     356433 (14.25%) aligned 0 times
##     1310302 (52.37%) aligned exactly 1 time
##     835235 (33.38%) aligned &gt;1 times
## 85.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 835235/2145537
## 2 670436/835235
## 3 3465642/7869774
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0996-raw_1 and brca-0996-raw_2
## Single-end reads will be in brca-0996-raw_M
## 5289801 sequences in the BAM file
## 5289801 sequences exported
## 5289801 reads; of these:
##   5289801 (100.00%) were unpaired; of these:
##     1516713 (28.67%) aligned 0 times
##     2786430 (52.68%) aligned exactly 1 time
##     986658 (18.65%) aligned &gt;1 times
## 71.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 986658/3773088
## 2 668264/986658
## 3 2855645/8794326
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-0997-raw_1 and brca-0997-raw_2
## Single-end reads will be in brca-0997-raw_M
## 3933972 sequences in the BAM file
## 3933972 sequences exported
## 3933972 reads; of these:
##   3933972 (100.00%) were unpaired; of these:
##     540516 (13.74%) aligned 0 times
##     1899682 (48.29%) aligned exactly 1 time
##     1493774 (37.97%) aligned &gt;1 times
## 86.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1493774/3393456
## 2 1276432/1493774
## 3 5059992/10665145
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-0998-raw_1 and brca-0998-raw_2
## Single-end reads will be in brca-0998-raw_M
## 3965132 sequences in the BAM file
## 3965132 sequences exported
## 3965132 reads; of these:
##   3965132 (100.00%) were unpaired; of these:
##     560562 (14.14%) aligned 0 times
##     2529615 (63.80%) aligned exactly 1 time
##     874955 (22.07%) aligned &gt;1 times
## 85.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 874955/3404570
## 2 681465/874955
## 3 5064854/11536140
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-0999-raw_1 and brca-0999-raw_2
## Single-end reads will be in brca-0999-raw_M
## 4932733 sequences in the BAM file
## 4932733 sequences exported
## 4932733 reads; of these:
##   4932733 (100.00%) were unpaired; of these:
##     1192342 (24.17%) aligned 0 times
##     2208157 (44.77%) aligned exactly 1 time
##     1532234 (31.06%) aligned &gt;1 times
## 75.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1532234/3740391
## 2 1331381/1532234
## 3 3322762/8672971
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1000-raw_1 and brca-1000-raw_2
## Single-end reads will be in brca-1000-raw_M
## 3762546 sequences in the BAM file
## 3762546 sequences exported
## 3762546 reads; of these:
##   3762546 (100.00%) were unpaired; of these:
##     302354 (8.04%) aligned 0 times
##     2323557 (61.75%) aligned exactly 1 time
##     1136635 (30.21%) aligned &gt;1 times
## 91.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1136635/3460192
## 2 927057/1136635
## 3 3476596/8581733
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1001-raw_1 and brca-1001-raw_2
## Single-end reads will be in brca-1001-raw_M
## 8998504 sequences in the BAM file
## 8998504 sequences exported
## 8998504 reads; of these:
##   8998504 (100.00%) were unpaired; of these:
##     3294366 (36.61%) aligned 0 times
##     2707329 (30.09%) aligned exactly 1 time
##     2996809 (33.30%) aligned &gt;1 times
## 63.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2996809/5704138
## 2 2527252/2996809
## 3 11215255/21350671
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1002-raw_1 and brca-1002-raw_2
## Single-end reads will be in brca-1002-raw_M
## 14649907 sequences in the BAM file
## 14649907 sequences exported
## 14649907 reads; of these:
##   14649907 (100.00%) were unpaired; of these:
##     2654959 (18.12%) aligned 0 times
##     7220089 (49.28%) aligned exactly 1 time
##     4774859 (32.59%) aligned &gt;1 times
## 81.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4774859/11994948
## 2 3948503/4774859
## 3 11126876/29384455
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-1003-raw_1 and brca-1003-raw_2
## Single-end reads will be in brca-1003-raw_M
## 4357463 sequences in the BAM file
## 4357463 sequences exported
## 4357463 reads; of these:
##   4357463 (100.00%) were unpaired; of these:
##     1016109 (23.32%) aligned 0 times
##     2446922 (56.15%) aligned exactly 1 time
##     894432 (20.53%) aligned &gt;1 times
## 76.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 894432/3341354
## 2 738808/894432
## 3 2008304/6398426
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1004-raw_1 and brca-1004-raw_2
## Single-end reads will be in brca-1004-raw_M
## 12038227 sequences in the BAM file
## 12038227 sequences exported
## 12038227 reads; of these:
##   12038227 (100.00%) were unpaired; of these:
##     2180548 (18.11%) aligned 0 times
##     6257343 (51.98%) aligned exactly 1 time
##     3600336 (29.91%) aligned &gt;1 times
## 81.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3600336/9857679
## 2 2816443/3600336
## 3 8609600/23211593
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-1005-raw_1 and brca-1005-raw_2
## Single-end reads will be in brca-1005-raw_M
## 17127626 sequences in the BAM file
## 17127626 sequences exported
## 17127626 reads; of these:
##   17127626 (100.00%) were unpaired; of these:
##     3347155 (19.54%) aligned 0 times
##     8441015 (49.28%) aligned exactly 1 time
##     5339456 (31.17%) aligned &gt;1 times
## 80.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5339456/13780471
## 2 4293075/5339456
## 3 17995672/40572151
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in brca-1006-raw_1 and brca-1006-raw_2
## Single-end reads will be in brca-1006-raw_M
## 2477665 sequences in the BAM file
## 2477665 sequences exported
## 2477665 reads; of these:
##   2477665 (100.00%) were unpaired; of these:
##     680322 (27.46%) aligned 0 times
##     1147664 (46.32%) aligned exactly 1 time
##     649679 (26.22%) aligned &gt;1 times
## 72.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 649679/1797343
## 2 519759/649679
## 3 2468007/5860608
## This looks like paired data from lane 6.
## Output will be in brca-1007-raw_1 and brca-1007-raw_2
## Single-end reads will be in brca-1007-raw_M
## 2722770 sequences in the BAM file
## 2722770 sequences exported
## 2722770 reads; of these:
##   2722770 (100.00%) were unpaired; of these:
##     596998 (21.93%) aligned 0 times
##     1444934 (53.07%) aligned exactly 1 time
##     680838 (25.01%) aligned &gt;1 times
## 78.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 680838/2125772
## 2 545360/680838
## 3 2609283/6079864
## This looks like paired data from lane 7.
## Output will be in brca-1008-raw_1 and brca-1008-raw_2
## Single-end reads will be in brca-1008-raw_M
## 10818239 sequences in the BAM file
## 10818239 sequences exported
## 10818239 reads; of these:
##   10818239 (100.00%) were unpaired; of these:
##     2852859 (26.37%) aligned 0 times
##     5321327 (49.19%) aligned exactly 1 time
##     2644053 (24.44%) aligned &gt;1 times
## 73.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2644053/7965380
## 2 1813964/2644053
## 3 20114245/38469040
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-1009-raw_1 and brca-1009-raw_2
## Single-end reads will be in brca-1009-raw_M
## 8631109 sequences in the BAM file
## 8631109 sequences exported
## 8631109 reads; of these:
##   8631109 (100.00%) were unpaired; of these:
##     1029170 (11.92%) aligned 0 times
##     4430233 (51.33%) aligned exactly 1 time
##     3171706 (36.75%) aligned &gt;1 times
## 88.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3171706/7601939
## 2 2737047/3171706
## 3 9627995/21028130
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-1010-raw_1 and brca-1010-raw_2
## Single-end reads will be in brca-1010-raw_M
## 4013482 sequences in the BAM file
## 4013482 sequences exported
## 4013482 reads; of these:
##   4013482 (100.00%) were unpaired; of these:
##     212003 (5.28%) aligned 0 times
##     2223338 (55.40%) aligned exactly 1 time
##     1578141 (39.32%) aligned &gt;1 times
## 94.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1578141/3801479
## 2 1153002/1578141
## 3 7446434/15106009
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1011-raw_1 and brca-1011-raw_2
## Single-end reads will be in brca-1011-raw_M
## 5209974 sequences in the BAM file
## 5209974 sequences exported
## 5209974 reads; of these:
##   5209974 (100.00%) were unpaired; of these:
##     1432974 (27.50%) aligned 0 times
##     2151311 (41.29%) aligned exactly 1 time
##     1625689 (31.20%) aligned &gt;1 times
## 72.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1625689/3777000
## 2 1132170/1625689
## 3 10731713/22045690
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-1012-raw_1 and brca-1012-raw_2
## Single-end reads will be in brca-1012-raw_M
## 4044019 sequences in the BAM file
## 4044019 sequences exported
## 4044019 reads; of these:
##   4044019 (100.00%) were unpaired; of these:
##     504405 (12.47%) aligned 0 times
##     1837316 (45.43%) aligned exactly 1 time
##     1702298 (42.09%) aligned &gt;1 times
## 87.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1702298/3539614
## 2 1458763/1702298
## 3 8169303/16237444
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1013-raw_1 and brca-1013-raw_2
## Single-end reads will be in brca-1013-raw_M
## 2444378 sequences in the BAM file
## 2444378 sequences exported
## 2444378 reads; of these:
##   2444378 (100.00%) were unpaired; of these:
##     712022 (29.13%) aligned 0 times
##     931100 (38.09%) aligned exactly 1 time
##     801256 (32.78%) aligned &gt;1 times
## 70.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 801256/1732356
## 2 591439/801256
## 3 3612743/7266042
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1014-raw_1 and brca-1014-raw_2
## Single-end reads will be in brca-1014-raw_M
## 6666973 sequences in the BAM file
## 6666973 sequences exported
## 6666973 reads; of these:
##   6666973 (100.00%) were unpaired; of these:
##     454540 (6.82%) aligned 0 times
##     3606272 (54.09%) aligned exactly 1 time
##     2606161 (39.09%) aligned &gt;1 times
## 93.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2606161/6212433
## 2 2138770/2606161
## 3 14592176/29869416
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-1015-raw_1 and brca-1015-raw_2
## Single-end reads will be in brca-1015-raw_M
## 2559619 sequences in the BAM file
## 2559619 sequences exported
## 2559619 reads; of these:
##   2559619 (100.00%) were unpaired; of these:
##     526752 (20.58%) aligned 0 times
##     1109137 (43.33%) aligned exactly 1 time
##     923730 (36.09%) aligned &gt;1 times
## 79.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 923730/2032867
## 2 721072/923730
## 3 6593260/12787560
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1016-raw_1 and brca-1016-raw_2
## Single-end reads will be in brca-1016-raw_M
## 2727855 sequences in the BAM file
## 2727855 sequences exported
## 2727855 reads; of these:
##   2727855 (100.00%) were unpaired; of these:
##     807862 (29.62%) aligned 0 times
##     998745 (36.61%) aligned exactly 1 time
##     921248 (33.77%) aligned &gt;1 times
## 70.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 921248/1919993
## 2 649452/921248
## 3 4031893/8466596
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1017-raw_1 and brca-1017-raw_2
## Single-end reads will be in brca-1017-raw_M
## 8300921 sequences in the BAM file
## 8300921 sequences exported
## 8300921 reads; of these:
##   8300921 (100.00%) were unpaired; of these:
##     3197571 (38.52%) aligned 0 times
##     3187547 (38.40%) aligned exactly 1 time
##     1915803 (23.08%) aligned &gt;1 times
## 61.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1915803/5103350
## 2 1535894/1915803
## 3 8396395/16778402
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1018-raw_1 and brca-1018-raw_2
## Single-end reads will be in brca-1018-raw_M
## 1990666 sequences in the BAM file
## 1990666 sequences exported
## 1990666 reads; of these:
##   1990666 (100.00%) were unpaired; of these:
##     242575 (12.19%) aligned 0 times
##     1262045 (63.40%) aligned exactly 1 time
##     486046 (24.42%) aligned &gt;1 times
## 87.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 486046/1748091
## 2 361964/486046
## 3 2158506/5139509
## This looks like paired data from lane 6.
## Output will be in brca-1019-raw_1 and brca-1019-raw_2
## Single-end reads will be in brca-1019-raw_M
## 11892297 sequences in the BAM file
## 11892297 sequences exported
## 11892297 reads; of these:
##   11892297 (100.00%) were unpaired; of these:
##     2756689 (23.18%) aligned 0 times
##     6305667 (53.02%) aligned exactly 1 time
##     2829941 (23.80%) aligned &gt;1 times
## 76.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2829941/9135608
## 2 2101466/2829941
## 3 15735217/35216376
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-1020-raw_1 and brca-1020-raw_2
## Single-end reads will be in brca-1020-raw_M
## 16740400 sequences in the BAM file
## 16740400 sequences exported
## 16740400 reads; of these:
##   16740400 (100.00%) were unpaired; of these:
##     4458861 (26.64%) aligned 0 times
##     7940483 (47.43%) aligned exactly 1 time
##     4341056 (25.93%) aligned &gt;1 times
## 73.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4341056/12281539
## 2 2930118/4341056
## 3 20895138/49992309
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 3.
## Output will be in brca-1021-raw_1 and brca-1021-raw_2
## Single-end reads will be in brca-1021-raw_M
## 5345277 sequences in the BAM file
## 5345277 sequences exported
## 5345277 reads; of these:
##   5345277 (100.00%) were unpaired; of these:
##     1029335 (19.26%) aligned 0 times
##     3124010 (58.44%) aligned exactly 1 time
##     1191932 (22.30%) aligned &gt;1 times
## 80.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1191932/4315942
## 2 931487/1191932
## 3 6952875/15965181
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1022-raw_1 and brca-1022-raw_2
## Single-end reads will be in brca-1022-raw_M
## 5588946 sequences in the BAM file
## 5588946 sequences exported
## 5588946 reads; of these:
##   5588946 (100.00%) were unpaired; of these:
##     413974 (7.41%) aligned 0 times
##     2929898 (52.42%) aligned exactly 1 time
##     2245074 (40.17%) aligned &gt;1 times
## 92.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2245074/5174972
## 2 1794825/2245074
## 3 13098396/26917184
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-1023-raw_1 and brca-1023-raw_2
## Single-end reads will be in brca-1023-raw_M
## 23101119 sequences in the BAM file
## 23101119 sequences exported
## 23101119 reads; of these:
##   23101119 (100.00%) were unpaired; of these:
##     4936926 (21.37%) aligned 0 times
##     10457041 (45.27%) aligned exactly 1 time
##     7707152 (33.36%) aligned &gt;1 times
## 78.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7707152/18164193
## 2 5987154/7707152
## 3 29275372/67360532
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 6.
## Output will be in brca-1024-raw_1 and brca-1024-raw_2
## Single-end reads will be in brca-1024-raw_M
## 4538415 sequences in the BAM file
## 4538415 sequences exported
## 4538415 reads; of these:
##   4538415 (100.00%) were unpaired; of these:
##     527972 (11.63%) aligned 0 times
##     2629336 (57.94%) aligned exactly 1 time
##     1381107 (30.43%) aligned &gt;1 times
## 88.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1381107/4010443
## 2 1090052/1381107
## 3 5263298/11942714
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1025-raw_1 and brca-1025-raw_2
## Single-end reads will be in brca-1025-raw_M
## 4137822 sequences in the BAM file
## 4137822 sequences exported
## 4137822 reads; of these:
##   4137822 (100.00%) were unpaired; of these:
##     901194 (21.78%) aligned 0 times
##     1574535 (38.05%) aligned exactly 1 time
##     1662093 (40.17%) aligned &gt;1 times
## 78.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1662093/3236628
## 2 1439151/1662093
## 3 4721273/10006871
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1026-raw_1 and brca-1026-raw_2
## Single-end reads will be in brca-1026-raw_M
## 4992412 sequences in the BAM file
## 4992412 sequences exported
## 4992412 reads; of these:
##   4992412 (100.00%) were unpaired; of these:
##     364700 (7.31%) aligned 0 times
##     2726884 (54.62%) aligned exactly 1 time
##     1900828 (38.07%) aligned &gt;1 times
## 92.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1900828/4627712
## 2 1564239/1900828
## 3 6106764/13825414
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1027-raw_1 and brca-1027-raw_2
## Single-end reads will be in brca-1027-raw_M
## 13270439 sequences in the BAM file
## 13270439 sequences exported
## 13270439 reads; of these:
##   13270439 (100.00%) were unpaired; of these:
##     4065988 (30.64%) aligned 0 times
##     6652771 (50.13%) aligned exactly 1 time
##     2551680 (19.23%) aligned &gt;1 times
## 69.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2551680/9204451
## 2 1958530/2551680
## 3 6390710/19625932
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-1028-raw_1 and brca-1028-raw_2
## Single-end reads will be in brca-1028-raw_M
## 4152529 sequences in the BAM file
## 4152529 sequences exported
## 4152529 reads; of these:
##   4152529 (100.00%) were unpaired; of these:
##     1019881 (24.56%) aligned 0 times
##     1763231 (42.46%) aligned exactly 1 time
##     1369417 (32.98%) aligned &gt;1 times
## 75.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1369417/3132648
## 2 1223074/1369417
## 3 2640877/6645576
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1029-raw_1 and brca-1029-raw_2
## Single-end reads will be in brca-1029-raw_M
## 8185541 sequences in the BAM file
## 8185541 sequences exported
## 8185541 reads; of these:
##   8185541 (100.00%) were unpaired; of these:
##     1736150 (21.21%) aligned 0 times
##     4186862 (51.15%) aligned exactly 1 time
##     2262529 (27.64%) aligned &gt;1 times
## 78.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2262529/6449391
## 2 1751664/2262529
## 3 5590884/15541666
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1030-raw_1 and brca-1030-raw_2
## Single-end reads will be in brca-1030-raw_M
## 4007249 sequences in the BAM file
## 4007249 sequences exported
## 4007249 reads; of these:
##   4007249 (100.00%) were unpaired; of these:
##     990825 (24.73%) aligned 0 times
##     1667822 (41.62%) aligned exactly 1 time
##     1348602 (33.65%) aligned &gt;1 times
## 75.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1348602/3016424
## 2 1064071/1348602
## 3 8654490/17961622
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1031-raw_1 and brca-1031-raw_2
## Single-end reads will be in brca-1031-raw_M
## 4733632 sequences in the BAM file
## 4733632 sequences exported
## 4733632 reads; of these:
##   4733632 (100.00%) were unpaired; of these:
##     1350621 (28.53%) aligned 0 times
##     2350402 (49.65%) aligned exactly 1 time
##     1032609 (21.81%) aligned &gt;1 times
## 71.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1032609/3383011
## 2 739073/1032609
## 3 5482138/11642754
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1032-raw_1 and brca-1032-raw_2
## Single-end reads will be in brca-1032-raw_M
## 11929865 sequences in the BAM file
## 11929865 sequences exported
## 11929865 reads; of these:
##   11929865 (100.00%) were unpaired; of these:
##     2158655 (18.09%) aligned 0 times
##     5235717 (43.89%) aligned exactly 1 time
##     4535493 (38.02%) aligned &gt;1 times
## 81.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4535493/9771210
## 2 3014318/4535493
## 3 14863564/34760862
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-1033-raw_1 and brca-1033-raw_2
## Single-end reads will be in brca-1033-raw_M
## 4083351 sequences in the BAM file
## 4083351 sequences exported
## 4083351 reads; of these:
##   4083351 (100.00%) were unpaired; of these:
##     1207366 (29.57%) aligned 0 times
##     1652417 (40.47%) aligned exactly 1 time
##     1223568 (29.96%) aligned &gt;1 times
## 70.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1223568/2875985
## 2 1069497/1223568
## 3 3434881/7849400
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1034-raw_1 and brca-1034-raw_2
## Single-end reads will be in brca-1034-raw_M
## 2361086 sequences in the BAM file
## 2361086 sequences exported
## 2361086 reads; of these:
##   2361086 (100.00%) were unpaired; of these:
##     538619 (22.81%) aligned 0 times
##     1124652 (47.63%) aligned exactly 1 time
##     697815 (29.55%) aligned &gt;1 times
## 77.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 697815/1822467
## 2 577635/697815
## 3 3732216/7994271
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1035-raw_1 and brca-1035-raw_2
## Single-end reads will be in brca-1035-raw_M
## 10583384 sequences in the BAM file
## 10583384 sequences exported
## 10583384 reads; of these:
##   10583384 (100.00%) were unpaired; of these:
##     2464576 (23.29%) aligned 0 times
##     5423587 (51.25%) aligned exactly 1 time
##     2695221 (25.47%) aligned &gt;1 times
## 76.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2695221/8118808
## 2 2210269/2695221
## 3 6768197/18252230
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-1036-raw_1 and brca-1036-raw_2
## Single-end reads will be in brca-1036-raw_M
## 12863631 sequences in the BAM file
## 12863631 sequences exported
## 12863631 reads; of these:
##   12863631 (100.00%) were unpaired; of these:
##     2337228 (18.17%) aligned 0 times
##     3014912 (23.44%) aligned exactly 1 time
##     7511491 (58.39%) aligned &gt;1 times
## 81.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7511491/10526403
## 2 5154566/7511491
## 3 78864375/148507751
## [bam_sort_core] merging from 20 files...
## This looks like paired data from lane 5.
## Output will be in brca-1037-raw_1 and brca-1037-raw_2
## Single-end reads will be in brca-1037-raw_M
## 10979496 sequences in the BAM file
## 10979496 sequences exported
## 10979496 reads; of these:
##   10979496 (100.00%) were unpaired; of these:
##     1781083 (16.22%) aligned 0 times
##     6381678 (58.12%) aligned exactly 1 time
##     2816735 (25.65%) aligned &gt;1 times
## 83.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2816735/9198413
## 2 2140113/2816735
## 3 8606718/23602666
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-1038-raw_1 and brca-1038-raw_2
## Single-end reads will be in brca-1038-raw_M
## 14226807 sequences in the BAM file
## 14226807 sequences exported
## 14226807 reads; of these:
##   14226807 (100.00%) were unpaired; of these:
##     3569634 (25.09%) aligned 0 times
##     6390936 (44.92%) aligned exactly 1 time
##     4266237 (29.99%) aligned &gt;1 times
## 74.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4266237/10657173
## 2 3043359/4266237
## 3 16143443/36699332
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in brca-1039-raw_1 and brca-1039-raw_2
## Single-end reads will be in brca-1039-raw_M
## 3272128 sequences in the BAM file
## 3272128 sequences exported
## 3272128 reads; of these:
##   3272128 (100.00%) were unpaired; of these:
##     246108 (7.52%) aligned 0 times
##     1832448 (56.00%) aligned exactly 1 time
##     1193572 (36.48%) aligned &gt;1 times
## 92.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1193572/3026020
## 2 1023902/1193572
## 3 3730217/8663817
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1040-raw_1 and brca-1040-raw_2
## Single-end reads will be in brca-1040-raw_M
## 5794809 sequences in the BAM file
## 5794809 sequences exported
## 5794809 reads; of these:
##   5794809 (100.00%) were unpaired; of these:
##     1050162 (18.12%) aligned 0 times
##     3271096 (56.45%) aligned exactly 1 time
##     1473551 (25.43%) aligned &gt;1 times
## 81.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1473551/4744647
## 2 1142084/1473551
## 3 8417606/17809002
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-1041-raw_1 and brca-1041-raw_2
## Single-end reads will be in brca-1041-raw_M
## 3446366 sequences in the BAM file
## 3446366 sequences exported
## 3446366 reads; of these:
##   3446366 (100.00%) were unpaired; of these:
##     262433 (7.61%) aligned 0 times
##     2010466 (58.34%) aligned exactly 1 time
##     1173467 (34.05%) aligned &gt;1 times
## 92.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1173467/3183933
## 2 877178/1173467
## 3 7078755/14671435
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1042-raw_1 and brca-1042-raw_2
## Single-end reads will be in brca-1042-raw_M
## 4340439 sequences in the BAM file
## 4340439 sequences exported
## 4340439 reads; of these:
##   4340439 (100.00%) were unpaired; of these:
##     840051 (19.35%) aligned 0 times
##     2425661 (55.89%) aligned exactly 1 time
##     1074727 (24.76%) aligned &gt;1 times
## 80.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1074727/3500388
## 2 818865/1074727
## 3 4533419/10682227
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1043-raw_1 and brca-1043-raw_2
## Single-end reads will be in brca-1043-raw_M
## 10680249 sequences in the BAM file
## 10680249 sequences exported
## 10680249 reads; of these:
##   10680249 (100.00%) were unpaired; of these:
##     1963401 (18.38%) aligned 0 times
##     3335703 (31.23%) aligned exactly 1 time
##     5381145 (50.38%) aligned &gt;1 times
## 81.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5381145/8716848
## 2 4255491/5381145
## 3 53805941/103477492
## [bam_sort_core] merging from 16 files...
## This looks like paired data from lane 7.
## Output will be in brca-1044-raw_1 and brca-1044-raw_2
## Single-end reads will be in brca-1044-raw_M
## 2251913 sequences in the BAM file
## 2251913 sequences exported
## 2251913 reads; of these:
##   2251913 (100.00%) were unpaired; of these:
##     512136 (22.74%) aligned 0 times
##     1051866 (46.71%) aligned exactly 1 time
##     687911 (30.55%) aligned &gt;1 times
## 77.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 687911/1739777
## 2 494544/687911
## 3 2914630/6998100
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1045-raw_1 and brca-1045-raw_2
## Single-end reads will be in brca-1045-raw_M
## 4839894 sequences in the BAM file
## 4839894 sequences exported
## 4839894 reads; of these:
##   4839894 (100.00%) were unpaired; of these:
##     1078519 (22.28%) aligned 0 times
##     2344713 (48.45%) aligned exactly 1 time
##     1416662 (29.27%) aligned &gt;1 times
## 77.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1416662/3761375
## 2 1133742/1416662
## 3 5808177/12748433
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1046-raw_1 and brca-1046-raw_2
## Single-end reads will be in brca-1046-raw_M
## 2580370 sequences in the BAM file
## 2580370 sequences exported
## 2580370 reads; of these:
##   2580370 (100.00%) were unpaired; of these:
##     447104 (17.33%) aligned 0 times
##     1189576 (46.10%) aligned exactly 1 time
##     943690 (36.57%) aligned &gt;1 times
## 82.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 943690/2133266
## 2 765813/943690
## 3 4941113/9576206
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1047-raw_1 and brca-1047-raw_2
## Single-end reads will be in brca-1047-raw_M
## 11923587 sequences in the BAM file
## 11923587 sequences exported
## 11923587 reads; of these:
##   11923587 (100.00%) were unpaired; of these:
##     2223238 (18.65%) aligned 0 times
##     6253218 (52.44%) aligned exactly 1 time
##     3447131 (28.91%) aligned &gt;1 times
## 81.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3447131/9700349
## 2 2647012/3447131
## 3 17552260/40356381
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in brca-1048-raw_1 and brca-1048-raw_2
## Single-end reads will be in brca-1048-raw_M
## 8045296 sequences in the BAM file
## 8045296 sequences exported
## 8045296 reads; of these:
##   8045296 (100.00%) were unpaired; of these:
##     2191094 (27.23%) aligned 0 times
##     3679871 (45.74%) aligned exactly 1 time
##     2174331 (27.03%) aligned &gt;1 times
## 72.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2174331/5854202
## 2 1623532/2174331
## 3 8873061/19935959
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-1049-raw_1 and brca-1049-raw_2
## Single-end reads will be in brca-1049-raw_M
## 3361391 sequences in the BAM file
## 3361391 sequences exported
## 3361391 reads; of these:
##   3361391 (100.00%) were unpaired; of these:
##     603885 (17.97%) aligned 0 times
##     1551135 (46.15%) aligned exactly 1 time
##     1206371 (35.89%) aligned &gt;1 times
## 82.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1206371/2757506
## 2 976431/1206371
## 3 5609311/11922929
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1050-raw_1 and brca-1050-raw_2
## Single-end reads will be in brca-1050-raw_M
## 7097176 sequences in the BAM file
## 7097176 sequences exported
## 7097176 reads; of these:
##   7097176 (100.00%) were unpaired; of these:
##     630285 (8.88%) aligned 0 times
##     3510623 (49.47%) aligned exactly 1 time
##     2956268 (41.65%) aligned &gt;1 times
## 91.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2956268/6466891
## 2 2385670/2956268
## 3 11698805/25639003
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-1051-raw_1 and brca-1051-raw_2
## Single-end reads will be in brca-1051-raw_M
## 2468302 sequences in the BAM file
## 2468302 sequences exported
## 2468302 reads; of these:
##   2468302 (100.00%) were unpaired; of these:
##     775927 (31.44%) aligned 0 times
##     1039120 (42.10%) aligned exactly 1 time
##     653255 (26.47%) aligned &gt;1 times
## 68.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 653255/1692375
## 2 438373/653255
## 3 3963644/8394069
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1052-raw_1 and brca-1052-raw_2
## Single-end reads will be in brca-1052-raw_M
## 4350449 sequences in the BAM file
## 4350449 sequences exported
## 4350449 reads; of these:
##   4350449 (100.00%) were unpaired; of these:
##     964819 (22.18%) aligned 0 times
##     2001474 (46.01%) aligned exactly 1 time
##     1384156 (31.82%) aligned &gt;1 times
## 77.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1384156/3385630
## 2 1197686/1384156
## 3 5510947/11492944
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1053-raw_1 and brca-1053-raw_2
## Single-end reads will be in brca-1053-raw_M
## 13611943 sequences in the BAM file
## 13611943 sequences exported
## 13611943 reads; of these:
##   13611943 (100.00%) were unpaired; of these:
##     3755179 (27.59%) aligned 0 times
##     5946197 (43.68%) aligned exactly 1 time
##     3910567 (28.73%) aligned &gt;1 times
## 72.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3910567/9856764
## 2 3047815/3910567
## 3 29438495/58813286
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in brca-1054-raw_1 and brca-1054-raw_2
## Single-end reads will be in brca-1054-raw_M
## 11392266 sequences in the BAM file
## 11392266 sequences exported
## 11392266 reads; of these:
##   11392266 (100.00%) were unpaired; of these:
##     2883307 (25.31%) aligned 0 times
##     4425278 (38.84%) aligned exactly 1 time
##     4083681 (35.85%) aligned &gt;1 times
## 74.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4083681/8508959
## 2 3452462/4083681
## 3 25720523/52024396
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in brca-1055-raw_1 and brca-1055-raw_2
## Single-end reads will be in brca-1055-raw_M
## 6621213 sequences in the BAM file
## 6621213 sequences exported
## 6621213 reads; of these:
##   6621213 (100.00%) were unpaired; of these:
##     542641 (8.20%) aligned 0 times
##     3681650 (55.60%) aligned exactly 1 time
##     2396922 (36.20%) aligned &gt;1 times
## 91.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2396922/6078572
## 2 1873203/2396922
## 3 16569463/31866621
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-1056-raw_1 and brca-1056-raw_2
## Single-end reads will be in brca-1056-raw_M
## 10655375 sequences in the BAM file
## 10655375 sequences exported
## 10655375 reads; of these:
##   10655375 (100.00%) were unpaired; of these:
##     1908077 (17.91%) aligned 0 times
##     5217900 (48.97%) aligned exactly 1 time
##     3529398 (33.12%) aligned &gt;1 times
## 82.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3529398/8747298
## 2 2451477/3529398
## 3 12689842/29655496
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-1057-raw_1 and brca-1057-raw_2
## Single-end reads will be in brca-1057-raw_M
## 2821196 sequences in the BAM file
## 2821196 sequences exported
## 2821196 reads; of these:
##   2821196 (100.00%) were unpaired; of these:
##     652012 (23.11%) aligned 0 times
##     1206456 (42.76%) aligned exactly 1 time
##     962728 (34.12%) aligned &gt;1 times
## 76.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 962728/2169184
## 2 743655/962728
## 3 5595379/11231558
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1058-raw_1 and brca-1058-raw_2
## Single-end reads will be in brca-1058-raw_M
## 2257455 sequences in the BAM file
## 2257455 sequences exported
## 2257455 reads; of these:
##   2257455 (100.00%) were unpaired; of these:
##     658885 (29.19%) aligned 0 times
##     903808 (40.04%) aligned exactly 1 time
##     694762 (30.78%) aligned &gt;1 times
## 70.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 694762/1598570
## 2 589023/694762
## 3 1725617/4274503
## This looks like paired data from lane 6.
## Output will be in brca-1059-raw_1 and brca-1059-raw_2
## Single-end reads will be in brca-1059-raw_M
## 11465072 sequences in the BAM file
## 11465072 sequences exported
## 11465072 reads; of these:
##   11465072 (100.00%) were unpaired; of these:
##     2581907 (22.52%) aligned 0 times
##     5998401 (52.32%) aligned exactly 1 time
##     2884764 (25.16%) aligned &gt;1 times
## 77.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2884764/8883165
## 2 2183175/2884764
## 3 6336796/19166838
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in brca-1060-raw_1 and brca-1060-raw_2
## Single-end reads will be in brca-1060-raw_M
## 11762313 sequences in the BAM file
## 11762313 sequences exported
## 11762313 reads; of these:
##   11762313 (100.00%) were unpaired; of these:
##     3126729 (26.58%) aligned 0 times
##     7360230 (62.57%) aligned exactly 1 time
##     1275354 (10.84%) aligned &gt;1 times
## 73.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1275354/8635584
## 2 938414/1275354
## 3 3375687/14291723
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-1061-raw_1 and brca-1061-raw_2
## Single-end reads will be in brca-1061-raw_M
## 1708989 sequences in the BAM file
## 1708989 sequences exported
## 1708989 reads; of these:
##   1708989 (100.00%) were unpaired; of these:
##     468241 (27.40%) aligned 0 times
##     856036 (50.09%) aligned exactly 1 time
##     384712 (22.51%) aligned &gt;1 times
## 72.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 384712/1240748
## 2 299894/384712
## 3 1280285/3445066
## This looks like paired data from lane 6.
## Output will be in brca-1062-raw_1 and brca-1062-raw_2
## Single-end reads will be in brca-1062-raw_M
## 4870818 sequences in the BAM file
## 4870818 sequences exported
## 4870818 reads; of these:
##   4870818 (100.00%) were unpaired; of these:
##     1281540 (26.31%) aligned 0 times
##     2335925 (47.96%) aligned exactly 1 time
##     1253353 (25.73%) aligned &gt;1 times
## 73.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1253353/3589278
## 2 878116/1253353
## 3 6593329/14906117
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1063-raw_1 and brca-1063-raw_2
## Single-end reads will be in brca-1063-raw_M
## 3583900 sequences in the BAM file
## 3583900 sequences exported
## 3583900 reads; of these:
##   3583900 (100.00%) were unpaired; of these:
##     1025682 (28.62%) aligned 0 times
##     1630886 (45.51%) aligned exactly 1 time
##     927332 (25.87%) aligned &gt;1 times
## 71.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 927332/2558218
## 2 726996/927332
## 3 4096440/8786412
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1064-raw_1 and brca-1064-raw_2
## Single-end reads will be in brca-1064-raw_M
## 15078412 sequences in the BAM file
## 15078412 sequences exported
## 15078412 reads; of these:
##   15078412 (100.00%) were unpaired; of these:
##     5165060 (34.25%) aligned 0 times
##     5532953 (36.69%) aligned exactly 1 time
##     4380399 (29.05%) aligned &gt;1 times
## 65.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4380399/9913352
## 2 3565644/4380399
## 3 7703516/21679923
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-1065-raw_1 and brca-1065-raw_2
## Single-end reads will be in brca-1065-raw_M
## 4640414 sequences in the BAM file
## 4640414 sequences exported
## 4640414 reads; of these:
##   4640414 (100.00%) were unpaired; of these:
##     1025276 (22.09%) aligned 0 times
##     1966947 (42.39%) aligned exactly 1 time
##     1648191 (35.52%) aligned &gt;1 times
## 77.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1648191/3615138
## 2 1451675/1648191
## 3 5207939/10908887
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1066-raw_1 and brca-1066-raw_2
## Single-end reads will be in brca-1066-raw_M
## 5384024 sequences in the BAM file
## 5384024 sequences exported
## 5384024 reads; of these:
##   5384024 (100.00%) were unpaired; of these:
##     1486914 (27.62%) aligned 0 times
##     2775993 (51.56%) aligned exactly 1 time
##     1121117 (20.82%) aligned &gt;1 times
## 72.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1121117/3897110
## 2 831392/1121117
## 3 5524126/13104449
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1067-raw_1 and brca-1067-raw_2
## Single-end reads will be in brca-1067-raw_M
## 6679010 sequences in the BAM file
## 6679010 sequences exported
## 6679010 reads; of these:
##   6679010 (100.00%) were unpaired; of these:
##     2118224 (31.71%) aligned 0 times
##     3319605 (49.70%) aligned exactly 1 time
##     1241181 (18.58%) aligned &gt;1 times
## 68.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1241181/4560786
## 2 899658/1241181
## 3 4333378/12159991
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1068-raw_1 and brca-1068-raw_2
## Single-end reads will be in brca-1068-raw_M
## 14328562 sequences in the BAM file
## 14328562 sequences exported
## 14328562 reads; of these:
##   14328562 (100.00%) were unpaired; of these:
##     4161639 (29.04%) aligned 0 times
##     6431825 (44.89%) aligned exactly 1 time
##     3735098 (26.07%) aligned &gt;1 times
## 70.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3735098/10166923
## 2 2874717/3735098
## 3 8829247/23605735
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-1069-raw_1 and brca-1069-raw_2
## Single-end reads will be in brca-1069-raw_M
## 3667675 sequences in the BAM file
## 3667675 sequences exported
## 3667675 reads; of these:
##   3667675 (100.00%) were unpaired; of these:
##     914162 (24.92%) aligned 0 times
##     1483439 (40.45%) aligned exactly 1 time
##     1270074 (34.63%) aligned &gt;1 times
## 75.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1270074/2753513
## 2 1105826/1270074
## 3 3122715/7386908
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1070-raw_1 and brca-1070-raw_2
## Single-end reads will be in brca-1070-raw_M
## 2782376 sequences in the BAM file
## 2782376 sequences exported
## 2782376 reads; of these:
##   2782376 (100.00%) were unpaired; of these:
##     759268 (27.29%) aligned 0 times
##     1074893 (38.63%) aligned exactly 1 time
##     948215 (34.08%) aligned &gt;1 times
## 72.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 948215/2023108
## 2 721682/948215
## 3 5874325/12127642
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1071-raw_1 and brca-1071-raw_2
## Single-end reads will be in brca-1071-raw_M
## 3971316 sequences in the BAM file
## 3971316 sequences exported
## 3971316 reads; of these:
##   3971316 (100.00%) were unpaired; of these:
##     980745 (24.70%) aligned 0 times
##     1808186 (45.53%) aligned exactly 1 time
##     1182385 (29.77%) aligned &gt;1 times
## 75.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1182385/2990571
## 2 872397/1182385
## 3 3903009/9371229
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1072-raw_1 and brca-1072-raw_2
## Single-end reads will be in brca-1072-raw_M
## 6548554 sequences in the BAM file
## 6548554 sequences exported
## 6548554 reads; of these:
##   6548554 (100.00%) were unpaired; of these:
##     2058953 (31.44%) aligned 0 times
##     2716062 (41.48%) aligned exactly 1 time
##     1773539 (27.08%) aligned &gt;1 times
## 68.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1773539/4489601
## 2 1408265/1773539
## 3 11646673/24218473
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-1073-raw_1 and brca-1073-raw_2
## Single-end reads will be in brca-1073-raw_M
## 2706867 sequences in the BAM file
## 2706867 sequences exported
## 2706867 reads; of these:
##   2706867 (100.00%) were unpaired; of these:
##     538177 (19.88%) aligned 0 times
##     1279742 (47.28%) aligned exactly 1 time
##     888948 (32.84%) aligned &gt;1 times
## 80.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 888948/2168690
## 2 661484/888948
## 3 2232877/5729815
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1074-raw_1 and brca-1074-raw_2
## Single-end reads will be in brca-1074-raw_M
## 3959500 sequences in the BAM file
## 3959500 sequences exported
## 3959500 reads; of these:
##   3959500 (100.00%) were unpaired; of these:
##     950683 (24.01%) aligned 0 times
##     1550030 (39.15%) aligned exactly 1 time
##     1458787 (36.84%) aligned &gt;1 times
## 75.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1458787/3008817
## 2 1244029/1458787
## 3 3464740/8237419
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1075-raw_1 and brca-1075-raw_2
## Single-end reads will be in brca-1075-raw_M
## 3359106 sequences in the BAM file
## 3359106 sequences exported
## 3359106 reads; of these:
##   3359106 (100.00%) were unpaired; of these:
##     739739 (22.02%) aligned 0 times
##     1669777 (49.71%) aligned exactly 1 time
##     949590 (28.27%) aligned &gt;1 times
## 77.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 949590/2619367
## 2 699928/949590
## 3 3986507/8784639
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1076-raw_1 and brca-1076-raw_2
## Single-end reads will be in brca-1076-raw_M
## 2327679 sequences in the BAM file
## 2327679 sequences exported
## 2327679 reads; of these:
##   2327679 (100.00%) were unpaired; of these:
##     661946 (28.44%) aligned 0 times
##     751545 (32.29%) aligned exactly 1 time
##     914188 (39.27%) aligned &gt;1 times
## 71.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 914188/1665733
## 2 764488/914188
## 3 5792831/10511432
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1077-raw_1 and brca-1077-raw_2
## Single-end reads will be in brca-1077-raw_M
## 6580884 sequences in the BAM file
## 6580884 sequences exported
## 6580884 reads; of these:
##   6580884 (100.00%) were unpaired; of these:
##     1800461 (27.36%) aligned 0 times
##     2897132 (44.02%) aligned exactly 1 time
##     1883291 (28.62%) aligned &gt;1 times
## 72.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1883291/4780423
## 2 1175760/1883291
## 3 6700621/15927636
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-1078-raw_1 and brca-1078-raw_2
## Single-end reads will be in brca-1078-raw_M
## 4492363 sequences in the BAM file
## 4492363 sequences exported
## 4492363 reads; of these:
##   4492363 (100.00%) were unpaired; of these:
##     1165012 (25.93%) aligned 0 times
##     1927118 (42.90%) aligned exactly 1 time
##     1400233 (31.17%) aligned &gt;1 times
## 74.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1400233/3327351
## 2 1045608/1400233
## 3 7009743/16155503
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-1079-raw_1 and brca-1079-raw_2
## Single-end reads will be in brca-1079-raw_M
## 2313615 sequences in the BAM file
## 2313615 sequences exported
## 2313615 reads; of these:
##   2313615 (100.00%) were unpaired; of these:
##     484536 (20.94%) aligned 0 times
##     1310253 (56.63%) aligned exactly 1 time
##     518826 (22.42%) aligned &gt;1 times
## 79.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 518826/1829079
## 2 436686/518826
## 3 1608162/4610697
## This looks like paired data from lane 5.
## Output will be in brca-1080-raw_1 and brca-1080-raw_2
## Single-end reads will be in brca-1080-raw_M
## 13600423 sequences in the BAM file
## 13600423 sequences exported
## 13600423 reads; of these:
##   13600423 (100.00%) were unpaired; of these:
##     2322962 (17.08%) aligned 0 times
##     7192145 (52.88%) aligned exactly 1 time
##     4085316 (30.04%) aligned &gt;1 times
## 82.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4085316/11277461
## 2 3277629/4085316
## 3 8050920/23911827
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-1081-raw_1 and brca-1081-raw_2
## Single-end reads will be in brca-1081-raw_M
## 7984493 sequences in the BAM file
## 7984493 sequences exported
## 7984493 reads; of these:
##   7984493 (100.00%) were unpaired; of these:
##     1972157 (24.70%) aligned 0 times
##     3964726 (49.66%) aligned exactly 1 time
##     2047610 (25.64%) aligned &gt;1 times
## 75.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2047610/6012336
## 2 1551431/2047610
## 3 12303248/27081037
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-1082-raw_1 and brca-1082-raw_2
## Single-end reads will be in brca-1082-raw_M
## 5809592 sequences in the BAM file
## 5809592 sequences exported
## 5809592 reads; of these:
##   5809592 (100.00%) were unpaired; of these:
##     1649654 (28.40%) aligned 0 times
##     2775536 (47.78%) aligned exactly 1 time
##     1384402 (23.83%) aligned &gt;1 times
## 71.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1384402/4159938
## 2 1082920/1384402
## 3 5945032/12902454
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1083-raw_1 and brca-1083-raw_2
## Single-end reads will be in brca-1083-raw_M
## 13786570 sequences in the BAM file
## 13786570 sequences exported
## 13786570 reads; of these:
##   13786570 (100.00%) were unpaired; of these:
##     3885647 (28.18%) aligned 0 times
##     6895539 (50.02%) aligned exactly 1 time
##     3005384 (21.80%) aligned &gt;1 times
## 71.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3005384/9900923
## 2 2328221/3005384
## 3 8142843/23410614
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in brca-1084-raw_1 and brca-1084-raw_2
## Single-end reads will be in brca-1084-raw_M
## 3671662 sequences in the BAM file
## 3671662 sequences exported
## 3671662 reads; of these:
##   3671662 (100.00%) were unpaired; of these:
##     491387 (13.38%) aligned 0 times
##     2094215 (57.04%) aligned exactly 1 time
##     1086060 (29.58%) aligned &gt;1 times
## 86.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1086060/3180275
## 2 852395/1086060
## 3 5579650/11875326
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1085-raw_1 and brca-1085-raw_2
## Single-end reads will be in brca-1085-raw_M
## 2986152 sequences in the BAM file
## 2986152 sequences exported
## 2986152 reads; of these:
##   2986152 (100.00%) were unpaired; of these:
##     791852 (26.52%) aligned 0 times
##     1558887 (52.20%) aligned exactly 1 time
##     635413 (21.28%) aligned &gt;1 times
## 73.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 635413/2194300
## 2 456620/635413
## 3 3219512/7854687
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1086-raw_1 and brca-1086-raw_2
## Single-end reads will be in brca-1086-raw_M
## 1898892 sequences in the BAM file
## 1898892 sequences exported
## 1898892 reads; of these:
##   1898892 (100.00%) were unpaired; of these:
##     440338 (23.19%) aligned 0 times
##     848138 (44.66%) aligned exactly 1 time
##     610416 (32.15%) aligned &gt;1 times
## 76.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 610416/1458554
## 2 437533/610416
## 3 1529765/3879938
## This looks like paired data from lane 6.
## Output will be in brca-1087-raw_1 and brca-1087-raw_2
## Single-end reads will be in brca-1087-raw_M
## 4145948 sequences in the BAM file
## 4145948 sequences exported
## 4145948 reads; of these:
##   4145948 (100.00%) were unpaired; of these:
##     456486 (11.01%) aligned 0 times
##     2085897 (50.31%) aligned exactly 1 time
##     1603565 (38.68%) aligned &gt;1 times
## 88.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1603565/3689462
## 2 1255407/1603565
## 3 8738069/18073086
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1088-raw_1 and brca-1088-raw_2
## Single-end reads will be in brca-1088-raw_M
## 12915897 sequences in the BAM file
## 12915897 sequences exported
## 12915897 reads; of these:
##   12915897 (100.00%) were unpaired; of these:
##     2222493 (17.21%) aligned 0 times
##     5731115 (44.37%) aligned exactly 1 time
##     4962289 (38.42%) aligned &gt;1 times
## 82.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4962289/10693404
## 2 4043833/4962289
## 3 20953209/47627655
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in brca-1089-raw_1 and brca-1089-raw_2
## Single-end reads will be in brca-1089-raw_M
## 2501419 sequences in the BAM file
## 2501419 sequences exported
## 2501419 reads; of these:
##   2501419 (100.00%) were unpaired; of these:
##     507302 (20.28%) aligned 0 times
##     1239796 (49.56%) aligned exactly 1 time
##     754321 (30.16%) aligned &gt;1 times
## 79.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 754321/1994117
## 2 499623/754321
## 3 3501250/8030143
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1090-raw_1 and brca-1090-raw_2
## Single-end reads will be in brca-1090-raw_M
## 2396695 sequences in the BAM file
## 2396695 sequences exported
## 2396695 reads; of these:
##   2396695 (100.00%) were unpaired; of these:
##     559535 (23.35%) aligned 0 times
##     954838 (39.84%) aligned exactly 1 time
##     882322 (36.81%) aligned &gt;1 times
## 76.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 882322/1837160
## 2 693113/882322
## 3 6078744/11818263
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1091-raw_1 and brca-1091-raw_2
## Single-end reads will be in brca-1091-raw_M
## 7441551 sequences in the BAM file
## 7441551 sequences exported
## 7441551 reads; of these:
##   7441551 (100.00%) were unpaired; of these:
##     1317292 (17.70%) aligned 0 times
##     3586394 (48.19%) aligned exactly 1 time
##     2537865 (34.10%) aligned &gt;1 times
## 82.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2537865/6124259
## 2 2125053/2537865
## 3 5431435/14034570
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1092-raw_1 and brca-1092-raw_2
## Single-end reads will be in brca-1092-raw_M
## 4639059 sequences in the BAM file
## 4639059 sequences exported
## 4639059 reads; of these:
##   4639059 (100.00%) were unpaired; of these:
##     844509 (18.20%) aligned 0 times
##     2632843 (56.75%) aligned exactly 1 time
##     1161707 (25.04%) aligned &gt;1 times
## 81.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1161707/3794550
## 2 913659/1161707
## 3 3525019/8977980
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1093-raw_1 and brca-1093-raw_2
## Single-end reads will be in brca-1093-raw_M
## 2292889 sequences in the BAM file
## 2292889 sequences exported
## 2292889 reads; of these:
##   2292889 (100.00%) were unpaired; of these:
##     617976 (26.95%) aligned 0 times
##     1093588 (47.69%) aligned exactly 1 time
##     581325 (25.35%) aligned &gt;1 times
## 73.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 581325/1674913
## 2 430665/581325
## 3 1580610/4401761
## This looks like paired data from lane 1.
## Output will be in brca-1094-raw_1 and brca-1094-raw_2
## Single-end reads will be in brca-1094-raw_M
## 2894561 sequences in the BAM file
## 2894561 sequences exported
## 2894561 reads; of these:
##   2894561 (100.00%) were unpaired; of these:
##     269181 (9.30%) aligned 0 times
##     1363566 (47.11%) aligned exactly 1 time
##     1261814 (43.59%) aligned &gt;1 times
## 90.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1261814/2625380
## 2 1040437/1261814
## 3 5706876/11383795
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1095-raw_1 and brca-1095-raw_2
## Single-end reads will be in brca-1095-raw_M
## 3249641 sequences in the BAM file
## 3249641 sequences exported
## 3249641 reads; of these:
##   3249641 (100.00%) were unpaired; of these:
##     968744 (29.81%) aligned 0 times
##     1481728 (45.60%) aligned exactly 1 time
##     799169 (24.59%) aligned &gt;1 times
## 70.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 799169/2280897
## 2 624287/799169
## 3 4098405/8970724
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1096-raw_1 and brca-1096-raw_2
## Single-end reads will be in brca-1096-raw_M
## 3721367 sequences in the BAM file
## 3721367 sequences exported
## 3721367 reads; of these:
##   3721367 (100.00%) were unpaired; of these:
##     792546 (21.30%) aligned 0 times
##     1894966 (50.92%) aligned exactly 1 time
##     1033855 (27.78%) aligned &gt;1 times
## 78.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1033855/2928821
## 2 857535/1033855
## 3 2485210/6926953
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1097-raw_1 and brca-1097-raw_2
## Single-end reads will be in brca-1097-raw_M
## 1581075 sequences in the BAM file
## 1581075 sequences exported
## 1581075 reads; of these:
##   1581075 (100.00%) were unpaired; of these:
##     130582 (8.26%) aligned 0 times
##     1002879 (63.43%) aligned exactly 1 time
##     447614 (28.31%) aligned &gt;1 times
## 91.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 447614/1450493
## 2 315280/447614
## 3 2176728/4875692
## This looks like paired data from lane 4.
## Output will be in brca-1098-raw_1 and brca-1098-raw_2
## Single-end reads will be in brca-1098-raw_M
## 5738590 sequences in the BAM file
## 5738590 sequences exported
## 5738590 reads; of these:
##   5738590 (100.00%) were unpaired; of these:
##     1380713 (24.06%) aligned 0 times
##     2236659 (38.98%) aligned exactly 1 time
##     2121218 (36.96%) aligned &gt;1 times
## 75.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2121218/4357877
## 2 1781703/2121218
## 3 5549794/12169183
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1099-raw_1 and brca-1099-raw_2
## Single-end reads will be in brca-1099-raw_M
## 10791426 sequences in the BAM file
## 10791426 sequences exported
## 10791426 reads; of these:
##   10791426 (100.00%) were unpaired; of these:
##     2581427 (23.92%) aligned 0 times
##     4816272 (44.63%) aligned exactly 1 time
##     3393727 (31.45%) aligned &gt;1 times
## 76.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3393727/8209999
## 2 2806189/3393727
## 3 8707207/21865968
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-1100-raw_1 and brca-1100-raw_2
## Single-end reads will be in brca-1100-raw_M
## 15111197 sequences in the BAM file
## 15111197 sequences exported
## 15111197 reads; of these:
##   15111197 (100.00%) were unpaired; of these:
##     2866972 (18.97%) aligned 0 times
##     5859204 (38.77%) aligned exactly 1 time
##     6385021 (42.25%) aligned &gt;1 times
## 81.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6385021/12244225
## 2 5122117/6385021
## 3 51495911/99408369
## [bam_sort_core] merging from 14 files...
## This looks like paired data from lane 2.
## Output will be in brca-1101-raw_1 and brca-1101-raw_2
## Single-end reads will be in brca-1101-raw_M
## 4368366 sequences in the BAM file
## 4368366 sequences exported
## 4368366 reads; of these:
##   4368366 (100.00%) were unpaired; of these:
##     1286269 (29.45%) aligned 0 times
##     2020529 (46.25%) aligned exactly 1 time
##     1061568 (24.30%) aligned &gt;1 times
## 70.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1061568/3082097
## 2 760845/1061568
## 3 6548200/13715816
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1102-raw_1 and brca-1102-raw_2
## Single-end reads will be in brca-1102-raw_M
## 10939392 sequences in the BAM file
## 10939392 sequences exported
## 10939392 reads; of these:
##   10939392 (100.00%) were unpaired; of these:
##     2287383 (20.91%) aligned 0 times
##     4771607 (43.62%) aligned exactly 1 time
##     3880402 (35.47%) aligned &gt;1 times
## 79.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3880402/8652009
## 2 3277462/3880402
## 3 9566150/22478386
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in brca-1103-raw_1 and brca-1103-raw_2
## Single-end reads will be in brca-1103-raw_M
## 5483822 sequences in the BAM file
## 5483822 sequences exported
## 5483822 reads; of these:
##   5483822 (100.00%) were unpaired; of these:
##     572185 (10.43%) aligned 0 times
##     2822397 (51.47%) aligned exactly 1 time
##     2089240 (38.10%) aligned &gt;1 times
## 89.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2089240/4911637
## 2 1733536/2089240
## 3 16227041/31644471
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-1104-raw_1 and brca-1104-raw_2
## Single-end reads will be in brca-1104-raw_M
## 2715583 sequences in the BAM file
## 2715583 sequences exported
## 2715583 reads; of these:
##   2715583 (100.00%) were unpaired; of these:
##     862763 (31.77%) aligned 0 times
##     1072753 (39.50%) aligned exactly 1 time
##     780067 (28.73%) aligned &gt;1 times
## 68.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 780067/1852820
## 2 617790/780067
## 3 2759299/6072650
## This looks like paired data from lane 5.
## Output will be in brca-1105-raw_1 and brca-1105-raw_2
## Single-end reads will be in brca-1105-raw_M
## 5447495 sequences in the BAM file
## 5447495 sequences exported
## 5447495 reads; of these:
##   5447495 (100.00%) were unpaired; of these:
##     703777 (12.92%) aligned 0 times
##     2880047 (52.87%) aligned exactly 1 time
##     1863671 (34.21%) aligned &gt;1 times
## 87.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1863671/4743718
## 2 1346514/1863671
## 3 7871778/17048008
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1106-raw_1 and brca-1106-raw_2
## Single-end reads will be in brca-1106-raw_M
## 15042074 sequences in the BAM file
## 15042074 sequences exported
## 15042074 reads; of these:
##   15042074 (100.00%) were unpaired; of these:
##     3250446 (21.61%) aligned 0 times
##     6711210 (44.62%) aligned exactly 1 time
##     5080418 (33.77%) aligned &gt;1 times
## 78.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5080418/11791628
## 2 4085502/5080418
## 3 11189215/28530293
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in brca-1107-raw_1 and brca-1107-raw_2
## Single-end reads will be in brca-1107-raw_M
## 4400223 sequences in the BAM file
## 4400223 sequences exported
## 4400223 reads; of these:
##   4400223 (100.00%) were unpaired; of these:
##     1163264 (26.44%) aligned 0 times
##     1887686 (42.90%) aligned exactly 1 time
##     1349273 (30.66%) aligned &gt;1 times
## 73.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1349273/3236959
## 2 1085780/1349273
## 3 4347243/9816243
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1108-raw_1 and brca-1108-raw_2
## Single-end reads will be in brca-1108-raw_M
## 3394345 sequences in the BAM file
## 3394345 sequences exported
## 3394345 reads; of these:
##   3394345 (100.00%) were unpaired; of these:
##     696273 (20.51%) aligned 0 times
##     1909578 (56.26%) aligned exactly 1 time
##     788494 (23.23%) aligned &gt;1 times
## 79.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 788494/2698072
## 2 592547/788494
## 3 3726616/8435714
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1109-raw_1 and brca-1109-raw_2
## Single-end reads will be in brca-1109-raw_M
## 11811898 sequences in the BAM file
## 11811898 sequences exported
## 11811898 reads; of these:
##   11811898 (100.00%) were unpaired; of these:
##     2696720 (22.83%) aligned 0 times
##     5069260 (42.92%) aligned exactly 1 time
##     4045918 (34.25%) aligned &gt;1 times
## 77.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4045918/9115178
## 2 3262029/4045918
## 3 15883155/34426660
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in brca-1110-raw_1 and brca-1110-raw_2
## Single-end reads will be in brca-1110-raw_M
## 10538010 sequences in the BAM file
## 10538010 sequences exported
## 10538010 reads; of these:
##   10538010 (100.00%) were unpaired; of these:
##     2997302 (28.44%) aligned 0 times
##     2332802 (22.14%) aligned exactly 1 time
##     5207906 (49.42%) aligned &gt;1 times
## 71.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5207906/7540708
## 2 4273962/5207906
## 3 77113332/138088186
## [bam_sort_core] merging from 19 files...
## This looks like paired data from lane 2.
## Output will be in brca-1111-raw_1 and brca-1111-raw_2
## Single-end reads will be in brca-1111-raw_M
## 3015575 sequences in the BAM file
## 3015575 sequences exported
## 3015575 reads; of these:
##   3015575 (100.00%) were unpaired; of these:
##     582085 (19.30%) aligned 0 times
##     951322 (31.55%) aligned exactly 1 time
##     1482168 (49.15%) aligned &gt;1 times
## 80.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1482168/2433490
## 2 876652/1482168
## 3 4970756/10464525
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1112-raw_1 and brca-1112-raw_2
## Single-end reads will be in brca-1112-raw_M
## 2303997 sequences in the BAM file
## 2303997 sequences exported
## 2303997 reads; of these:
##   2303997 (100.00%) were unpaired; of these:
##     327232 (14.20%) aligned 0 times
##     1029128 (44.67%) aligned exactly 1 time
##     947637 (41.13%) aligned &gt;1 times
## 85.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 947637/1976765
## 2 709325/947637
## 3 7108951/13735947
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1113-raw_1 and brca-1113-raw_2
## Single-end reads will be in brca-1113-raw_M
## 7383392 sequences in the BAM file
## 7383392 sequences exported
## 7383392 reads; of these:
##   7383392 (100.00%) were unpaired; of these:
##     1508080 (20.43%) aligned 0 times
##     3288320 (44.54%) aligned exactly 1 time
##     2586992 (35.04%) aligned &gt;1 times
## 79.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2586992/5875312
## 2 1850133/2586992
## 3 17163000/35364388
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in brca-1114-raw_1 and brca-1114-raw_2
## Single-end reads will be in brca-1114-raw_M
## 9277572 sequences in the BAM file
## 9277572 sequences exported
## 9277572 reads; of these:
##   9277572 (100.00%) were unpaired; of these:
##     3459113 (37.28%) aligned 0 times
##     3101427 (33.43%) aligned exactly 1 time
##     2717032 (29.29%) aligned &gt;1 times
## 62.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2717032/5818459
## 2 2333044/2717032
## 3 11165408/20599336
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1115-raw_1 and brca-1115-raw_2
## Single-end reads will be in brca-1115-raw_M
## 5574370 sequences in the BAM file
## 5574370 sequences exported
## 5574370 reads; of these:
##   5574370 (100.00%) were unpaired; of these:
##     724976 (13.01%) aligned 0 times
##     2756465 (49.45%) aligned exactly 1 time
##     2092929 (37.55%) aligned &gt;1 times
## 86.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2092929/4849394
## 2 1724638/2092929
## 3 6124192/13591236
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-1116-raw_1 and brca-1116-raw_2
## Single-end reads will be in brca-1116-raw_M
## 5652870 sequences in the BAM file
## 5652870 sequences exported
## 5652870 reads; of these:
##   5652870 (100.00%) were unpaired; of these:
##     1487382 (26.31%) aligned 0 times
##     2678910 (47.39%) aligned exactly 1 time
##     1486578 (26.30%) aligned &gt;1 times
## 73.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1486578/4165488
## 2 1196999/1486578
## 3 4949207/11204433
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1117-raw_1 and brca-1117-raw_2
## Single-end reads will be in brca-1117-raw_M
## 5076066 sequences in the BAM file
## 5076066 sequences exported
## 5076066 reads; of these:
##   5076066 (100.00%) were unpaired; of these:
##     1161219 (22.88%) aligned 0 times
##     1968820 (38.79%) aligned exactly 1 time
##     1946027 (38.34%) aligned &gt;1 times
## 77.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1946027/3914847
## 2 1079884/1946027
## 3 6460673/15435153
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1118-raw_1 and brca-1118-raw_2
## Single-end reads will be in brca-1118-raw_M
## 1830188 sequences in the BAM file
## 1830188 sequences exported
## 1830188 reads; of these:
##   1830188 (100.00%) were unpaired; of these:
##     293993 (16.06%) aligned 0 times
##     910922 (49.77%) aligned exactly 1 time
##     625273 (34.16%) aligned &gt;1 times
## 83.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 625273/1536195
## 2 521512/625273
## 3 2357714/4963580
## This looks like paired data from lane 5.
## Output will be in brca-1119-raw_1 and brca-1119-raw_2
## Single-end reads will be in brca-1119-raw_M
## 13621526 sequences in the BAM file
## 13621526 sequences exported
## 13621526 reads; of these:
##   13621526 (100.00%) were unpaired; of these:
##     2919437 (21.43%) aligned 0 times
##     6479993 (47.57%) aligned exactly 1 time
##     4222096 (31.00%) aligned &gt;1 times
## 78.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4222096/10702089
## 2 2820735/4222096
## 3 26691061/59594595
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in brca-1120-raw_1 and brca-1120-raw_2
## Single-end reads will be in brca-1120-raw_M
## 3536038 sequences in the BAM file
## 3536038 sequences exported
## 3536038 reads; of these:
##   3536038 (100.00%) were unpaired; of these:
##     284615 (8.05%) aligned 0 times
##     2026654 (57.31%) aligned exactly 1 time
##     1224769 (34.64%) aligned &gt;1 times
## 91.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1224769/3251423
## 2 1008876/1224769
## 3 4145683/9577704
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1121-raw_1 and brca-1121-raw_2
## Single-end reads will be in brca-1121-raw_M
## 2885443 sequences in the BAM file
## 2885443 sequences exported
## 2885443 reads; of these:
##   2885443 (100.00%) were unpaired; of these:
##     202531 (7.02%) aligned 0 times
##     1280622 (44.38%) aligned exactly 1 time
##     1402290 (48.60%) aligned &gt;1 times
## 92.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1402290/2682912
## 2 1216931/1402290
## 3 5353340/10779991
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1122-raw_1 and brca-1122-raw_2
## Single-end reads will be in brca-1122-raw_M
## 3212312 sequences in the BAM file
## 3212312 sequences exported
## 3212312 reads; of these:
##   3212312 (100.00%) were unpaired; of these:
##     316857 (9.86%) aligned 0 times
##     1714786 (53.38%) aligned exactly 1 time
##     1180669 (36.75%) aligned &gt;1 times
## 90.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1180669/2895455
## 2 973962/1180669
## 3 3922879/8896685
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1123-raw_1 and brca-1123-raw_2
## Single-end reads will be in brca-1123-raw_M
## 2310739 sequences in the BAM file
## 2310739 sequences exported
## 2310739 reads; of these:
##   2310739 (100.00%) were unpaired; of these:
##     525367 (22.74%) aligned 0 times
##     1109875 (48.03%) aligned exactly 1 time
##     675497 (29.23%) aligned &gt;1 times
## 77.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 675497/1785372
## 2 493599/675497
## 3 3939867/8286103
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1124-raw_1 and brca-1124-raw_2
## Single-end reads will be in brca-1124-raw_M
## 11026793 sequences in the BAM file
## 11026793 sequences exported
## 11026793 reads; of these:
##   11026793 (100.00%) were unpaired; of these:
##     2511757 (22.78%) aligned 0 times
##     4124490 (37.40%) aligned exactly 1 time
##     4390546 (39.82%) aligned &gt;1 times
## 77.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4390546/8515036
## 2 4006076/4390546
## 3 8514926/20478358
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-1125-raw_1 and brca-1125-raw_2
## Single-end reads will be in brca-1125-raw_M
## 2988115 sequences in the BAM file
## 2988115 sequences exported
## 2988115 reads; of these:
##   2988115 (100.00%) were unpaired; of these:
##     662409 (22.17%) aligned 0 times
##     1636163 (54.76%) aligned exactly 1 time
##     689543 (23.08%) aligned &gt;1 times
## 77.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 689543/2325706
## 2 513697/689543
## 3 3110222/7089108
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1126-raw_1 and brca-1126-raw_2
## Single-end reads will be in brca-1126-raw_M
## 2450984 sequences in the BAM file
## 2450984 sequences exported
## 2450984 reads; of these:
##   2450984 (100.00%) were unpaired; of these:
##     734505 (29.97%) aligned 0 times
##     1262646 (51.52%) aligned exactly 1 time
##     453833 (18.52%) aligned &gt;1 times
## 70.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 453833/1716479
## 2 293694/453833
## 3 1563262/4474855
## This looks like paired data from lane 6.
## Output will be in brca-1127-raw_1 and brca-1127-raw_2
## Single-end reads will be in brca-1127-raw_M
## 3804988 sequences in the BAM file
## 3804988 sequences exported
## 3804988 reads; of these:
##   3804988 (100.00%) were unpaired; of these:
##     1042542 (27.40%) aligned 0 times
##     1743255 (45.81%) aligned exactly 1 time
##     1019191 (26.79%) aligned &gt;1 times
## 72.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1019191/2762446
## 2 791211/1019191
## 3 4708434/10270787
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1128-raw_1 and brca-1128-raw_2
## Single-end reads will be in brca-1128-raw_M
## 9165237 sequences in the BAM file
## 9165237 sequences exported
## 9165237 reads; of these:
##   9165237 (100.00%) were unpaired; of these:
##     1861417 (20.31%) aligned 0 times
##     4185416 (45.67%) aligned exactly 1 time
##     3118404 (34.02%) aligned &gt;1 times
## 79.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3118404/7303820
## 2 2635697/3118404
## 3 9801254/23516738
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in brca-1129-raw_1 and brca-1129-raw_2
## Single-end reads will be in brca-1129-raw_M
## 1871819 sequences in the BAM file
## 1871819 sequences exported
## 1871819 reads; of these:
##   1871819 (100.00%) were unpaired; of these:
##     469041 (25.06%) aligned 0 times
##     855702 (45.71%) aligned exactly 1 time
##     547076 (29.23%) aligned &gt;1 times
## 74.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 547076/1402778
## 2 379999/547076
## 3 2583495/5952755
## This looks like paired data from lane 5.
## Output will be in brca-1130-raw_1 and brca-1130-raw_2
## Single-end reads will be in brca-1130-raw_M
## 4626432 sequences in the BAM file
## 4626432 sequences exported
## 4626432 reads; of these:
##   4626432 (100.00%) were unpaired; of these:
##     689375 (14.90%) aligned 0 times
##     2456921 (53.11%) aligned exactly 1 time
##     1480136 (31.99%) aligned &gt;1 times
## 85.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1480136/3937057
## 2 1234373/1480136
## 3 6688358/14527130
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1131-raw_1 and brca-1131-raw_2
## Single-end reads will be in brca-1131-raw_M
## 21121868 sequences in the BAM file
## 21121868 sequences exported
## 21121868 reads; of these:
##   21121868 (100.00%) were unpaired; of these:
##     5629916 (26.65%) aligned 0 times
##     9802422 (46.41%) aligned exactly 1 time
##     5689530 (26.94%) aligned &gt;1 times
## 73.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5689530/15491952
## 2 4537823/5689530
## 3 14320203/37176208
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in brca-1132-raw_1 and brca-1132-raw_2
## Single-end reads will be in brca-1132-raw_M
## 2270390 sequences in the BAM file
## 2270390 sequences exported
## 2270390 reads; of these:
##   2270390 (100.00%) were unpaired; of these:
##     364520 (16.06%) aligned 0 times
##     1206973 (53.16%) aligned exactly 1 time
##     698897 (30.78%) aligned &gt;1 times
## 83.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 698897/1905870
## 2 551416/698897
## 3 3775273/8217479
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1133-raw_1 and brca-1133-raw_2
## Single-end reads will be in brca-1133-raw_M
## 4595909 sequences in the BAM file
## 4595909 sequences exported
## 4595909 reads; of these:
##   4595909 (100.00%) were unpaired; of these:
##     378889 (8.24%) aligned 0 times
##     2690356 (58.54%) aligned exactly 1 time
##     1526664 (33.22%) aligned &gt;1 times
## 91.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1526664/4217020
## 2 1167188/1526664
## 3 8294204/17768504
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1134-raw_1 and brca-1134-raw_2
## Single-end reads will be in brca-1134-raw_M
## 1789021 sequences in the BAM file
## 1789021 sequences exported
## 1789021 reads; of these:
##   1789021 (100.00%) were unpaired; of these:
##     170540 (9.53%) aligned 0 times
##     921907 (51.53%) aligned exactly 1 time
##     696574 (38.94%) aligned &gt;1 times
## 90.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 696574/1618481
## 2 540029/696574
## 3 5537411/10783275
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1135-raw_1 and brca-1135-raw_2
## Single-end reads will be in brca-1135-raw_M
## 5354807 sequences in the BAM file
## 5354807 sequences exported
## 5354807 reads; of these:
##   5354807 (100.00%) were unpaired; of these:
##     397097 (7.42%) aligned 0 times
##     3649815 (68.16%) aligned exactly 1 time
##     1307895 (24.42%) aligned &gt;1 times
## 92.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1307895/4957710
## 2 1067909/1307895
## 3 4344060/11557633
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1136-raw_1 and brca-1136-raw_2
## Single-end reads will be in brca-1136-raw_M
## 5099024 sequences in the BAM file
## 5099024 sequences exported
## 5099024 reads; of these:
##   5099024 (100.00%) were unpaired; of these:
##     539376 (10.58%) aligned 0 times
##     2515630 (49.34%) aligned exactly 1 time
##     2044018 (40.09%) aligned &gt;1 times
## 89.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2044018/4559648
## 2 1545216/2044018
## 3 5991632/13588372
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1137-raw_1 and brca-1137-raw_2
## Single-end reads will be in brca-1137-raw_M
## 5847967 sequences in the BAM file
## 5847967 sequences exported
## 5847967 reads; of these:
##   5847967 (100.00%) were unpaired; of these:
##     430928 (7.37%) aligned 0 times
##     2980329 (50.96%) aligned exactly 1 time
##     2436710 (41.67%) aligned &gt;1 times
## 92.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2436710/5417039
## 2 1951979/2436710
## 3 8383251/18645570
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1138-raw_1 and brca-1138-raw_2
## Single-end reads will be in brca-1138-raw_M
## 11411225 sequences in the BAM file
## 11411225 sequences exported
## 11411225 reads; of these:
##   11411225 (100.00%) were unpaired; of these:
##     2446927 (21.44%) aligned 0 times
##     4808949 (42.14%) aligned exactly 1 time
##     4155349 (36.41%) aligned &gt;1 times
## 78.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4155349/8964298
## 2 3535720/4155349
## 3 10088029/23875144
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-1139-raw_1 and brca-1139-raw_2
## Single-end reads will be in brca-1139-raw_M
## 7888806 sequences in the BAM file
## 7888806 sequences exported
## 7888806 reads; of these:
##   7888806 (100.00%) were unpaired; of these:
##     666136 (8.44%) aligned 0 times
##     4968394 (62.98%) aligned exactly 1 time
##     2254276 (28.58%) aligned &gt;1 times
## 91.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2254276/7222670
## 2 1673036/2254276
## 3 9974924/23488735
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in brca-1140-raw_1 and brca-1140-raw_2
## Single-end reads will be in brca-1140-raw_M
## 4904016 sequences in the BAM file
## 4904016 sequences exported
## 4904016 reads; of these:
##   4904016 (100.00%) were unpaired; of these:
##     1074096 (21.90%) aligned 0 times
##     2398790 (48.91%) aligned exactly 1 time
##     1431130 (29.18%) aligned &gt;1 times
## 78.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1431130/3829920
## 2 1212191/1431130
## 3 4118069/9885165
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1141-raw_1 and brca-1141-raw_2
## Single-end reads will be in brca-1141-raw_M
## 2928181 sequences in the BAM file
## 2928181 sequences exported
## 2928181 reads; of these:
##   2928181 (100.00%) were unpaired; of these:
##     443292 (15.14%) aligned 0 times
##     1468115 (50.14%) aligned exactly 1 time
##     1016774 (34.72%) aligned &gt;1 times
## 84.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1016774/2484889
## 2 819713/1016774
## 3 4034636/8495167
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1142-raw_1 and brca-1142-raw_2
## Single-end reads will be in brca-1142-raw_M
## 4729810 sequences in the BAM file
## 4729810 sequences exported
## 4729810 reads; of these:
##   4729810 (100.00%) were unpaired; of these:
##     590349 (12.48%) aligned 0 times
##     2754676 (58.24%) aligned exactly 1 time
##     1384785 (29.28%) aligned &gt;1 times
## 87.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1384785/4139461
## 2 1118277/1384785
## 3 9081333/18915645
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1143-raw_1 and brca-1143-raw_2
## Single-end reads will be in brca-1143-raw_M
## 6521018 sequences in the BAM file
## 6521018 sequences exported
## 6521018 reads; of these:
##   6521018 (100.00%) were unpaired; of these:
##     1350221 (20.71%) aligned 0 times
##     2052095 (31.47%) aligned exactly 1 time
##     3118702 (47.83%) aligned &gt;1 times
## 79.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3118702/5170797
## 2 2692804/3118702
## 3 11135928/21161671
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1144-raw_1 and brca-1144-raw_2
## Single-end reads will be in brca-1144-raw_M
## 3113997 sequences in the BAM file
## 3113997 sequences exported
## 3113997 reads; of these:
##   3113997 (100.00%) were unpaired; of these:
##     277288 (8.90%) aligned 0 times
##     1816789 (58.34%) aligned exactly 1 time
##     1019920 (32.75%) aligned &gt;1 times
## 91.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1019920/2836709
## 2 816988/1019920
## 3 5425534/11124996
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1145-raw_1 and brca-1145-raw_2
## Single-end reads will be in brca-1145-raw_M
## 2248990 sequences in the BAM file
## 2248990 sequences exported
## 2248990 reads; of these:
##   2248990 (100.00%) were unpaired; of these:
##     415632 (18.48%) aligned 0 times
##     1183026 (52.60%) aligned exactly 1 time
##     650332 (28.92%) aligned &gt;1 times
## 81.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 650332/1833358
## 2 493595/650332
## 3 2840991/6527011
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in brca-1146-raw_1 and brca-1146-raw_2
## Single-end reads will be in brca-1146-raw_M
## 5118700 sequences in the BAM file
## 5118700 sequences exported
## 5118700 reads; of these:
##   5118700 (100.00%) were unpaired; of these:
##     559101 (10.92%) aligned 0 times
##     3143613 (61.41%) aligned exactly 1 time
##     1415986 (27.66%) aligned &gt;1 times
## 89.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1415986/4559599
## 2 1072874/1415986
## 3 4883387/12104128
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1147-raw_1 and brca-1147-raw_2
## Single-end reads will be in brca-1147-raw_M
## 7398047 sequences in the BAM file
## 7398047 sequences exported
## 7398047 reads; of these:
##   7398047 (100.00%) were unpaired; of these:
##     2507211 (33.89%) aligned 0 times
##     2672487 (36.12%) aligned exactly 1 time
##     2218349 (29.99%) aligned &gt;1 times
## 66.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2218349/4890836
## 2 1591506/2218349
## 3 7849777/15452531
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1148-raw_1 and brca-1148-raw_2
## Single-end reads will be in brca-1148-raw_M
## 7094820 sequences in the BAM file
## 7094820 sequences exported
## 7094820 reads; of these:
##   7094820 (100.00%) were unpaired; of these:
##     654284 (9.22%) aligned 0 times
##     3865826 (54.49%) aligned exactly 1 time
##     2574710 (36.29%) aligned &gt;1 times
## 90.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2574710/6440536
## 2 2004790/2574710
## 3 15563816/31769146
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in brca-1149-raw_1 and brca-1149-raw_2
## Single-end reads will be in brca-1149-raw_M
## 4901804 sequences in the BAM file
## 4901804 sequences exported
## 4901804 reads; of these:
##   4901804 (100.00%) were unpaired; of these:
##     997498 (20.35%) aligned 0 times
##     1634811 (33.35%) aligned exactly 1 time
##     2269495 (46.30%) aligned &gt;1 times
## 79.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2269495/3904306
## 2 2027071/2269495
## 3 7255972/14833415
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1150-raw_1 and brca-1150-raw_2
## Single-end reads will be in brca-1150-raw_M
## 4149058 sequences in the BAM file
## 4149058 sequences exported
## 4149058 reads; of these:
##   4149058 (100.00%) were unpaired; of these:
##     941070 (22.68%) aligned 0 times
##     2027336 (48.86%) aligned exactly 1 time
##     1180652 (28.46%) aligned &gt;1 times
## 77.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1180652/3207988
## 2 927688/1180652
## 3 5937268/12491878
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1151-raw_1 and brca-1151-raw_2
## Single-end reads will be in brca-1151-raw_M
## 4731827 sequences in the BAM file
## 4731827 sequences exported
## 4731827 reads; of these:
##   4731827 (100.00%) were unpaired; of these:
##     1105137 (23.36%) aligned 0 times
##     2103190 (44.45%) aligned exactly 1 time
##     1523500 (32.20%) aligned &gt;1 times
## 76.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1523500/3626690
## 2 1259399/1523500
## 3 8803805/17663115
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-1152-raw_1 and brca-1152-raw_2
## Single-end reads will be in brca-1152-raw_M
## 13753344 sequences in the BAM file
## 13753344 sequences exported
## 13753344 reads; of these:
##   13753344 (100.00%) were unpaired; of these:
##     3947016 (28.70%) aligned 0 times
##     5016687 (36.48%) aligned exactly 1 time
##     4789641 (34.83%) aligned &gt;1 times
## 71.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4789641/9806328
## 2 4083755/4789641
## 3 17632730/37321822
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in brca-1153-raw_1 and brca-1153-raw_2
## Single-end reads will be in brca-1153-raw_M
## 7987245 sequences in the BAM file
## 7987245 sequences exported
## 7987245 reads; of these:
##   7987245 (100.00%) were unpaired; of these:
##     2246305 (28.12%) aligned 0 times
##     3289481 (41.18%) aligned exactly 1 time
##     2451459 (30.69%) aligned &gt;1 times
## 71.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2451459/5740940
## 2 2044505/2451459
## 3 6415809/15617836
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1154-raw_1 and brca-1154-raw_2
## Single-end reads will be in brca-1154-raw_M
## 3419744 sequences in the BAM file
## 3419744 sequences exported
## 3419744 reads; of these:
##   3419744 (100.00%) were unpaired; of these:
##     704560 (20.60%) aligned 0 times
##     1607228 (47.00%) aligned exactly 1 time
##     1107956 (32.40%) aligned &gt;1 times
## 79.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1107956/2715184
## 2 870567/1107956
## 3 5431247/12639506
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in brca-1155-raw_1 and brca-1155-raw_2
## Single-end reads will be in brca-1155-raw_M
## 9940675 sequences in the BAM file
## 9940675 sequences exported
## 9940675 reads; of these:
##   9940675 (100.00%) were unpaired; of these:
##     2735934 (27.52%) aligned 0 times
##     5510881 (55.44%) aligned exactly 1 time
##     1693860 (17.04%) aligned &gt;1 times
## 72.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1693860/7204741
## 2 1255844/1693860
## 3 5960732/16512589
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-1156-raw_1 and brca-1156-raw_2
## Single-end reads will be in brca-1156-raw_M
## 3136693 sequences in the BAM file
## 3136693 sequences exported
## 3136693 reads; of these:
##   3136693 (100.00%) were unpaired; of these:
##     284276 (9.06%) aligned 0 times
##     1660418 (52.94%) aligned exactly 1 time
##     1191999 (38.00%) aligned &gt;1 times
## 90.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1191999/2852417
## 2 947425/1191999
## 3 7199531/14374684
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1157-raw_1 and brca-1157-raw_2
## Single-end reads will be in brca-1157-raw_M
## 3746135 sequences in the BAM file
## 3746135 sequences exported
## 3746135 reads; of these:
##   3746135 (100.00%) were unpaired; of these:
##     820692 (21.91%) aligned 0 times
##     1544515 (41.23%) aligned exactly 1 time
##     1380928 (36.86%) aligned &gt;1 times
## 78.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1380928/2925443
## 2 1136390/1380928
## 3 4279907/9094931
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1158-raw_1 and brca-1158-raw_2
## Single-end reads will be in brca-1158-raw_M
## 2409875 sequences in the BAM file
## 2409875 sequences exported
## 2409875 reads; of these:
##   2409875 (100.00%) were unpaired; of these:
##     496536 (20.60%) aligned 0 times
##     1124669 (46.67%) aligned exactly 1 time
##     788670 (32.73%) aligned &gt;1 times
## 79.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 788670/1913339
## 2 602996/788670
## 3 4873945/10079307
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1159-raw_1 and brca-1159-raw_2
## Single-end reads will be in brca-1159-raw_M
## 11689091 sequences in the BAM file
## 11689091 sequences exported
## 11689091 reads; of these:
##   11689091 (100.00%) were unpaired; of these:
##     3048674 (26.08%) aligned 0 times
##     5822702 (49.81%) aligned exactly 1 time
##     2817715 (24.11%) aligned &gt;1 times
## 73.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2817715/8640417
## 2 2254535/2817715
## 3 7650137/20855140
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in brca-1160-raw_1 and brca-1160-raw_2
## Single-end reads will be in brca-1160-raw_M
## 9477076 sequences in the BAM file
## 9477076 sequences exported
## 9477076 reads; of these:
##   9477076 (100.00%) were unpaired; of these:
##     1702417 (17.96%) aligned 0 times
##     5650202 (59.62%) aligned exactly 1 time
##     2124457 (22.42%) aligned &gt;1 times
## 82.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2124457/7774659
## 2 1776499/2124457
## 3 4687849/14990509
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in brca-1161-raw_1 and brca-1161-raw_2
## Single-end reads will be in brca-1161-raw_M
## 13329019 sequences in the BAM file
## 13329019 sequences exported
## 13329019 reads; of these:
##   13329019 (100.00%) were unpaired; of these:
##     3499709 (26.26%) aligned 0 times
##     5976441 (44.84%) aligned exactly 1 time
##     3852869 (28.91%) aligned &gt;1 times
## 73.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3852869/9829310
## 2 2898244/3852869
## 3 17629183/35294134
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-1162-raw_1 and brca-1162-raw_2
## Single-end reads will be in brca-1162-raw_M
## 5220705 sequences in the BAM file
## 5220705 sequences exported
## 5220705 reads; of these:
##   5220705 (100.00%) were unpaired; of these:
##     410171 (7.86%) aligned 0 times
##     3215683 (61.59%) aligned exactly 1 time
##     1594851 (30.55%) aligned &gt;1 times
## 92.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1594851/4810534
## 2 1248322/1594851
## 3 9217714/19047047
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in brca-1163-raw_1 and brca-1163-raw_2
## Single-end reads will be in brca-1163-raw_M
## 10870495 sequences in the BAM file
## 10870495 sequences exported
## 10870495 reads; of these:
##   10870495 (100.00%) were unpaired; of these:
##     2478385 (22.80%) aligned 0 times
##     4653682 (42.81%) aligned exactly 1 time
##     3738428 (34.39%) aligned &gt;1 times
## 77.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3738428/8392110
## 2 3065219/3738428
## 3 11242869/25655944
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in brca-1164-raw_1 and brca-1164-raw_2
## Single-end reads will be in brca-1164-raw_M
## 2156442 sequences in the BAM file
## 2156442 sequences exported
## 2156442 reads; of these:
##   2156442 (100.00%) were unpaired; of these:
##     557334 (25.85%) aligned 0 times
##     992205 (46.01%) aligned exactly 1 time
##     606903 (28.14%) aligned &gt;1 times
## 74.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 606903/1599108
## 2 492540/606903
## 3 2500740/5923859
## This looks like paired data from lane 4.
## Output will be in brca-1165-raw_1 and brca-1165-raw_2
## Single-end reads will be in brca-1165-raw_M
## 6819642 sequences in the BAM file
## 6819642 sequences exported
## 6819642 reads; of these:
##   6819642 (100.00%) were unpaired; of these:
##     2058734 (30.19%) aligned 0 times
##     2850748 (41.80%) aligned exactly 1 time
##     1910160 (28.01%) aligned &gt;1 times
## 69.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1910160/4760908
## 2 1329458/1910160
## 3 8366819/19041429
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in brca-1166-raw_1 and brca-1166-raw_2
## Single-end reads will be in brca-1166-raw_M
## 3333338 sequences in the BAM file
## 3333338 sequences exported
## 3333338 reads; of these:
##   3333338 (100.00%) were unpaired; of these:
##     594902 (17.85%) aligned 0 times
##     1688207 (50.65%) aligned exactly 1 time
##     1050229 (31.51%) aligned &gt;1 times
## 82.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1050229/2738436
## 2 805615/1050229
## 3 6788142/14335242
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in brca-1167-raw_1 and brca-1167-raw_2
## Single-end reads will be in brca-1167-raw_M
## 4077364 sequences in the BAM file
## 4077364 sequences exported
## 4077364 reads; of these:
##   4077364 (100.00%) were unpaired; of these:
##     1058037 (25.95%) aligned 0 times
##     1866042 (45.77%) aligned exactly 1 time
##     1153285 (28.29%) aligned &gt;1 times
## 74.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1153285/3019327
## 2 827336/1153285
## 3 6009488/13073937
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in brca-1168-raw_1 and brca-1168-raw_2
## Single-end reads will be in brca-1168-raw_M
## 10251237 sequences in the BAM file
## 10251237 sequences exported
## 10251237 reads; of these:
##   10251237 (100.00%) were unpaired; of these:
##     2874405 (28.04%) aligned 0 times
##     4389355 (42.82%) aligned exactly 1 time
##     2987477 (29.14%) aligned &gt;1 times
## 71.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2987477/7376832
## 2 2302070/2987477
## 3 6877186/17573126
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-1169-raw_1 and brca-1169-raw_2
## Single-end reads will be in brca-1169-raw_M
## 13053200 sequences in the BAM file
## 13053200 sequences exported
## 13053200 reads; of these:
##   13053200 (100.00%) were unpaired; of these:
##     3520885 (26.97%) aligned 0 times
##     6170990 (47.28%) aligned exactly 1 time
##     3361325 (25.75%) aligned &gt;1 times
## 73.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3361325/9532315
## 2 2557895/3361325
## 3 9099207/23659110
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in brca-1170-raw_1 and brca-1170-raw_2
## Single-end reads will be in brca-1170-raw_M
## 3160611 sequences in the BAM file
## 3160611 sequences exported
## 3160611 reads; of these:
##   3160611 (100.00%) were unpaired; of these:
##     683209 (21.62%) aligned 0 times
##     1549400 (49.02%) aligned exactly 1 time
##     928002 (29.36%) aligned &gt;1 times
## 78.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 928002/2477402
## 2 746022/928002
## 3 2785461/7066241
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1171-raw_1 and brca-1171-raw_2
## Single-end reads will be in brca-1171-raw_M
## 2420357 sequences in the BAM file
## 2420357 sequences exported
## 2420357 reads; of these:
##   2420357 (100.00%) were unpaired; of these:
##     595285 (24.59%) aligned 0 times
##     1138271 (47.03%) aligned exactly 1 time
##     686801 (28.38%) aligned &gt;1 times
## 75.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 686801/1825072
## 2 584061/686801
## 3 1918510/5073658
## This looks like paired data from lane 1.
## Output will be in brca-1172-raw_1 and brca-1172-raw_2
## Single-end reads will be in brca-1172-raw_M
## 5298885 sequences in the BAM file
## 5298885 sequences exported
## 5298885 reads; of these:
##   5298885 (100.00%) were unpaired; of these:
##     431464 (8.14%) aligned 0 times
##     3275370 (61.81%) aligned exactly 1 time
##     1592051 (30.05%) aligned &gt;1 times
## 91.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1592051/4867421
## 2 1329525/1592051
## 3 4444209/11372398
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1173-raw_1 and brca-1173-raw_2
## Single-end reads will be in brca-1173-raw_M
## 5149773 sequences in the BAM file
## 5149773 sequences exported
## 5149773 reads; of these:
##   5149773 (100.00%) were unpaired; of these:
##     1099699 (21.35%) aligned 0 times
##     2876778 (55.86%) aligned exactly 1 time
##     1173296 (22.78%) aligned &gt;1 times
## 78.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1173296/4050074
## 2 857553/1173296
## 3 3853829/10623033
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1174-raw_1 and brca-1174-raw_2
## Single-end reads will be in brca-1174-raw_M
## 24060486 sequences in the BAM file
## 24060486 sequences exported
## 24060486 reads; of these:
##   24060486 (100.00%) were unpaired; of these:
##     5699423 (23.69%) aligned 0 times
##     10135846 (42.13%) aligned exactly 1 time
##     8225217 (34.19%) aligned &gt;1 times
## 76.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 8225217/18361063
## 2 6377113/8225217
## 3 30919169/63392863
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in brca-1175-raw_1 and brca-1175-raw_2
## Single-end reads will be in brca-1175-raw_M
## 2976555 sequences in the BAM file
## 2976555 sequences exported
## 2976555 reads; of these:
##   2976555 (100.00%) were unpaired; of these:
##     871597 (29.28%) aligned 0 times
##     1185516 (39.83%) aligned exactly 1 time
##     919442 (30.89%) aligned &gt;1 times
## 70.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 919442/2104958
## 2 444818/919442
## 3 3809046/8972894
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in brca-1176-raw_1 and brca-1176-raw_2
## Single-end reads will be in brca-1176-raw_M
## 4075398 sequences in the BAM file
## 4075398 sequences exported
## 4075398 reads; of these:
##   4075398 (100.00%) were unpaired; of these:
##     1579103 (38.75%) aligned 0 times
##     1325341 (32.52%) aligned exactly 1 time
##     1170954 (28.73%) aligned &gt;1 times
## 61.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1170954/2496295
## 2 898481/1170954
## 3 5707989/11702769
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in brca-1177-raw_1 and brca-1177-raw_2
## Single-end reads will be in brca-1177-raw_M
## 2500252 sequences in the BAM file
## 2500252 sequences exported
## 2500252 reads; of these:
##   2500252 (100.00%) were unpaired; of these:
##     520809 (20.83%) aligned 0 times
##     1172496 (46.90%) aligned exactly 1 time
##     806947 (32.27%) aligned &gt;1 times
## 79.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 806947/1979443
## 2 592780/806947
## 3 3993138/8390155
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in brca-1178-raw_1 and brca-1178-raw_2
## Single-end reads will be in brca-1178-raw_M
## 4349660 sequences in the BAM file
## 4349660 sequences exported
## 4349660 reads; of these:
##   4349660 (100.00%) were unpaired; of these:
##     1002110 (23.04%) aligned 0 times
##     2319918 (53.34%) aligned exactly 1 time
##     1027632 (23.63%) aligned &gt;1 times
## 76.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1027632/3347550
## 2 769778/1027632
## 3 4952736/10862295
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1179-raw_1 and brca-1179-raw_2
## Single-end reads will be in brca-1179-raw_M
## 4840665 sequences in the BAM file
## 4840665 sequences exported
## 4840665 reads; of these:
##   4840665 (100.00%) were unpaired; of these:
##     1193031 (24.65%) aligned 0 times
##     2117893 (43.75%) aligned exactly 1 time
##     1529741 (31.60%) aligned &gt;1 times
## 75.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1529741/3647634
## 2 1270012/1529741
## 3 5258680/11613780
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in brca-1180-raw_1 and brca-1180-raw_2
## Single-end reads will be in brca-1180-raw_M
## 20414577 sequences in the BAM file
## 20414577 sequences exported
## 20414577 reads; of these:
##   20414577 (100.00%) were unpaired; of these:
##     6507933 (31.88%) aligned 0 times
##     8383182 (41.06%) aligned exactly 1 time
##     5523462 (27.06%) aligned &gt;1 times
## 68.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5523462/13906644
## 2 3998144/5523462
## 3 12230089/33467855
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in brca-1181-raw_1 and brca-1181-raw_2
## Single-end reads will be in brca-1181-raw_M
## 1497024 sequences in the BAM file
## 1497024 sequences exported
## 1497024 reads; of these:
##   1497024 (100.00%) were unpaired; of these:
##     218687 (14.61%) aligned 0 times
##     660228 (44.10%) aligned exactly 1 time
##     618109 (41.29%) aligned &gt;1 times
## 85.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 618109/1278337
## 2 529150/618109
## 3 2000702/4095007
## This looks like paired data from lane 2.
## Output will be in brca-1182-raw_1 and brca-1182-raw_2
## Single-end reads will be in brca-1182-raw_M
## 17759906 sequences in the BAM file
## 17759906 sequences exported
## 17759906 reads; of these:
##   17759906 (100.00%) were unpaired; of these:
##     5214731 (29.36%) aligned 0 times
##     9106312 (51.27%) aligned exactly 1 time
##     3438863 (19.36%) aligned &gt;1 times
## 70.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3438863/12545175
## 2 2716856/3438863
## 3 6808000/23368273
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in brca-1183-raw_1 and brca-1183-raw_2
## Single-end reads will be in brca-1183-raw_M
## 7858392 sequences in the BAM file
## 7858392 sequences exported
## 7858392 reads; of these:
##   7858392 (100.00%) were unpaired; of these:
##     1546196 (19.68%) aligned 0 times
##     3819474 (48.60%) aligned exactly 1 time
##     2492722 (31.72%) aligned &gt;1 times
## 80.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2492722/6312196
## 2 2051874/2492722
## 3 6267924/16032191
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in brca-1184-raw_1 and brca-1184-raw_2
## Single-end reads will be in brca-1184-raw_M
## 3084635 sequences in the BAM file
## 3084635 sequences exported
## 3084635 reads; of these:
##   3084635 (100.00%) were unpaired; of these:
##     491248 (15.93%) aligned 0 times
##     1696929 (55.01%) aligned exactly 1 time
##     896458 (29.06%) aligned &gt;1 times
## 84.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 896458/2593387
## 2 707290/896458
## 3 5835888/11989890
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1185-raw_1 and brca-1185-raw_2
## Single-end reads will be in brca-1185-raw_M
## 9825859 sequences in the BAM file
## 9825859 sequences exported
## 9825859 reads; of these:
##   9825859 (100.00%) were unpaired; of these:
##     2655384 (27.02%) aligned 0 times
##     3199325 (32.56%) aligned exactly 1 time
##     3971150 (40.42%) aligned &gt;1 times
## 72.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3971150/7170475
## 2 2906169/3971150
## 3 12226672/26933817
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in brca-1186-raw_1 and brca-1186-raw_2
## Single-end reads will be in brca-1186-raw_M
## 3141993 sequences in the BAM file
## 3141993 sequences exported
## 3141993 reads; of these:
##   3141993 (100.00%) were unpaired; of these:
##     990598 (31.53%) aligned 0 times
##     1198150 (38.13%) aligned exactly 1 time
##     953245 (30.34%) aligned &gt;1 times
## 68.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 953245/2151395
## 2 735136/953245
## 3 6740408/12885963
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in brca-1187-raw_1 and brca-1187-raw_2
## Single-end reads will be in brca-1187-raw_M
## 5212904 sequences in the BAM file
## 5212904 sequences exported
## 5212904 reads; of these:
##   5212904 (100.00%) were unpaired; of these:
##     1069894 (20.52%) aligned 0 times
##     2899345 (55.62%) aligned exactly 1 time
##     1243665 (23.86%) aligned &gt;1 times
## 79.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1243665/4143010
## 2 981248/1243665
## 3 5030331/12113991
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in brca-1188-raw_1 and brca-1188-raw_2
## Single-end reads will be in brca-1188-raw_M
## 6219964 sequences in the BAM file
## 6219964 sequences exported
## 6219964 reads; of these:
##   6219964 (100.00%) were unpaired; of these:
##     1908309 (30.68%) aligned 0 times
##     2612440 (42.00%) aligned exactly 1 time
##     1699215 (27.32%) aligned &gt;1 times
## 69.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1699215/4311655
## 2 1296033/1699215
## 3 9762882/22565983
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in brca-1189-raw_1 and brca-1189-raw_2
## Single-end reads will be in brca-1189-raw_M
## 2008250 sequences in the BAM file
## 2008250 sequences exported
## 2008250 reads; of these:
##   2008250 (100.00%) were unpaired; of these:
##     281351 (14.01%) aligned 0 times
##     1163873 (57.95%) aligned exactly 1 time
##     563026 (28.04%) aligned &gt;1 times
## 85.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 563026/1726899
## 2 419347/563026
## 3 2882086/6263620
## This looks like paired data from lane 2.
## Output will be in prad-0001-raw_1 and prad-0001-raw_2
## Single-end reads will be in prad-0001-raw_M
## 19266516 sequences in the BAM file
## 19266516 sequences exported
## 19266516 reads; of these:
##   19266516 (100.00%) were unpaired; of these:
##     5749431 (29.84%) aligned 0 times
##     10356042 (53.75%) aligned exactly 1 time
##     3161043 (16.41%) aligned &gt;1 times
## 70.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3161043/13517085
## 2 2066028/3161043
## 3 14995475/36500808
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0002-raw_1 and prad-0002-raw_2
## Single-end reads will be in prad-0002-raw_M
## 12455410 sequences in the BAM file
## 12455410 sequences exported
## 12455410 reads; of these:
##   12455410 (100.00%) were unpaired; of these:
##     3998306 (32.10%) aligned 0 times
##     6889599 (55.31%) aligned exactly 1 time
##     1567505 (12.58%) aligned &gt;1 times
## 67.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1567505/8457104
## 2 1109322/1567505
## 3 5242910/16035508
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0003-raw_1 and prad-0003-raw_2
## Single-end reads will be in prad-0003-raw_M
## 6275078 sequences in the BAM file
## 6275078 sequences exported
## 6275078 reads; of these:
##   6275078 (100.00%) were unpaired; of these:
##     1950510 (31.08%) aligned 0 times
##     3031093 (48.30%) aligned exactly 1 time
##     1293475 (20.61%) aligned &gt;1 times
## 68.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1293475/4324568
## 2 949047/1293475
## 3 7971368/17060541
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0004-raw_1 and prad-0004-raw_2
## Single-end reads will be in prad-0004-raw_M
## 11774330 sequences in the BAM file
## 11774330 sequences exported
## 11774330 reads; of these:
##   11774330 (100.00%) were unpaired; of these:
##     4178046 (35.48%) aligned 0 times
##     5824807 (49.47%) aligned exactly 1 time
##     1771477 (15.05%) aligned &gt;1 times
## 64.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1771477/7596284
## 2 1168303/1771477
## 3 12629472/26240955
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0005-raw_1 and prad-0005-raw_2
## Single-end reads will be in prad-0005-raw_M
## 7380085 sequences in the BAM file
## 7380085 sequences exported
## 7380085 reads; of these:
##   7380085 (100.00%) were unpaired; of these:
##     2454345 (33.26%) aligned 0 times
##     3691325 (50.02%) aligned exactly 1 time
##     1234415 (16.73%) aligned &gt;1 times
## 66.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1234415/4925740
## 2 865816/1234415
## 3 7053654/15520737
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0006-raw_1 and prad-0006-raw_2
## Single-end reads will be in prad-0006-raw_M
## 18149543 sequences in the BAM file
## 18149543 sequences exported
## 18149543 reads; of these:
##   18149543 (100.00%) were unpaired; of these:
##     5175699 (28.52%) aligned 0 times
##     10948933 (60.33%) aligned exactly 1 time
##     2024911 (11.16%) aligned &gt;1 times
## 71.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2024911/12973844
## 2 1310502/2024911
## 3 12365116/29388868
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0007-raw_1 and prad-0007-raw_2
## Single-end reads will be in prad-0007-raw_M
## 14338486 sequences in the BAM file
## 14338486 sequences exported
## 14338486 reads; of these:
##   14338486 (100.00%) were unpaired; of these:
##     4565015 (31.84%) aligned 0 times
##     7932951 (55.33%) aligned exactly 1 time
##     1840520 (12.84%) aligned &gt;1 times
## 68.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1840520/9773471
## 2 1263818/1840520
## 3 9831449/23803968
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0008-raw_1 and prad-0008-raw_2
## Single-end reads will be in prad-0008-raw_M
## 9342424 sequences in the BAM file
## 9342424 sequences exported
## 9342424 reads; of these:
##   9342424 (100.00%) were unpaired; of these:
##     2769577 (29.65%) aligned 0 times
##     5173465 (55.38%) aligned exactly 1 time
##     1399382 (14.98%) aligned &gt;1 times
## 70.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1399382/6572847
## 2 934538/1399382
## 3 6610584/15579257
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0009-raw_1 and prad-0009-raw_2
## Single-end reads will be in prad-0009-raw_M
## 9134191 sequences in the BAM file
## 9134191 sequences exported
## 9134191 reads; of these:
##   9134191 (100.00%) were unpaired; of these:
##     2880250 (31.53%) aligned 0 times
##     4868963 (53.30%) aligned exactly 1 time
##     1384978 (15.16%) aligned &gt;1 times
## 68.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1384978/6253941
## 2 994312/1384978
## 3 4886422/13670098
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0010-raw_1 and prad-0010-raw_2
## Single-end reads will be in prad-0010-raw_M
## 11694473 sequences in the BAM file
## 11694473 sequences exported
## 11694473 reads; of these:
##   11694473 (100.00%) were unpaired; of these:
##     4950353 (42.33%) aligned 0 times
##     5522418 (47.22%) aligned exactly 1 time
##     1221702 (10.45%) aligned &gt;1 times
## 57.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1221702/6744120
## 2 860951/1221702
## 3 5312030/14295991
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0011-raw_1 and prad-0011-raw_2
## Single-end reads will be in prad-0011-raw_M
## 11604369 sequences in the BAM file
## 11604369 sequences exported
## 11604369 reads; of these:
##   11604369 (100.00%) were unpaired; of these:
##     4067225 (35.05%) aligned 0 times
##     5942314 (51.21%) aligned exactly 1 time
##     1594830 (13.74%) aligned &gt;1 times
## 64.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1594830/7537144
## 2 1130616/1594830
## 3 7796761/19675036
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0012-raw_1 and prad-0012-raw_2
## Single-end reads will be in prad-0012-raw_M
## 10108064 sequences in the BAM file
## 10108064 sequences exported
## 10108064 reads; of these:
##   10108064 (100.00%) were unpaired; of these:
##     3728590 (36.89%) aligned 0 times
##     5085803 (50.31%) aligned exactly 1 time
##     1293671 (12.80%) aligned &gt;1 times
## 63.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1293671/6379474
## 2 911423/1293671
## 3 5413599/14054120
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0013-raw_1 and prad-0013-raw_2
## Single-end reads will be in prad-0013-raw_M
## 13113519 sequences in the BAM file
## 13113519 sequences exported
## 13113519 reads; of these:
##   13113519 (100.00%) were unpaired; of these:
##     3502015 (26.71%) aligned 0 times
##     7187868 (54.81%) aligned exactly 1 time
##     2423636 (18.48%) aligned &gt;1 times
## 73.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2423636/9611504
## 2 1664710/2423636
## 3 11487950/27828340
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0014-raw_1 and prad-0014-raw_2
## Single-end reads will be in prad-0014-raw_M
## 19966569 sequences in the BAM file
## 19966569 sequences exported
## 19966569 reads; of these:
##   19966569 (100.00%) were unpaired; of these:
##     3409250 (17.07%) aligned 0 times
##     14268222 (71.46%) aligned exactly 1 time
##     2289097 (11.46%) aligned &gt;1 times
## 82.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2289097/16557319
## 2 1657142/2289097
## 3 10802123/31111456
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0015-raw_1 and prad-0015-raw_2
## Single-end reads will be in prad-0015-raw_M
## 16952975 sequences in the BAM file
## 16952975 sequences exported
## 16952975 reads; of these:
##   16952975 (100.00%) were unpaired; of these:
##     4550936 (26.84%) aligned 0 times
##     9630276 (56.81%) aligned exactly 1 time
##     2771763 (16.35%) aligned &gt;1 times
## 73.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2771763/12402039
## 2 1817559/2771763
## 3 14353359/35234802
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0016-raw_1 and prad-0016-raw_2
## Single-end reads will be in prad-0016-raw_M
## 7636848 sequences in the BAM file
## 7636848 sequences exported
## 7636848 reads; of these:
##   7636848 (100.00%) were unpaired; of these:
##     2652900 (34.74%) aligned 0 times
##     3565041 (46.68%) aligned exactly 1 time
##     1418907 (18.58%) aligned &gt;1 times
## 65.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1418907/4983948
## 2 1064747/1418907
## 3 5399398/12893457
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0017-raw_1 and prad-0017-raw_2
## Single-end reads will be in prad-0017-raw_M
## 16516220 sequences in the BAM file
## 16516220 sequences exported
## 16516220 reads; of these:
##   16516220 (100.00%) were unpaired; of these:
##     5899134 (35.72%) aligned 0 times
##     8697024 (52.66%) aligned exactly 1 time
##     1920062 (11.63%) aligned &gt;1 times
## 64.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1920062/10617086
## 2 1181020/1920062
## 3 9679507/26814940
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0018-raw_1 and prad-0018-raw_2
## Single-end reads will be in prad-0018-raw_M
## 15960909 sequences in the BAM file
## 15960909 sequences exported
## 15960909 reads; of these:
##   15960909 (100.00%) were unpaired; of these:
##     4917266 (30.81%) aligned 0 times
##     8984573 (56.29%) aligned exactly 1 time
##     2059070 (12.90%) aligned &gt;1 times
## 69.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2059070/11043643
## 2 1336832/2059070
## 3 12979887/30520896
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0019-raw_1 and prad-0019-raw_2
## Single-end reads will be in prad-0019-raw_M
## 11113660 sequences in the BAM file
## 11113660 sequences exported
## 11113660 reads; of these:
##   11113660 (100.00%) were unpaired; of these:
##     3540765 (31.86%) aligned 0 times
##     5716123 (51.43%) aligned exactly 1 time
##     1856772 (16.71%) aligned &gt;1 times
## 68.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1856772/7572895
## 2 1307408/1856772
## 3 9818526/22431387
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0020-raw_1 and prad-0020-raw_2
## Single-end reads will be in prad-0020-raw_M
## 10228665 sequences in the BAM file
## 10228665 sequences exported
## 10228665 reads; of these:
##   10228665 (100.00%) were unpaired; of these:
##     2900406 (28.36%) aligned 0 times
##     5673212 (55.46%) aligned exactly 1 time
##     1655047 (16.18%) aligned &gt;1 times
## 71.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1655047/7328259
## 2 1117096/1655047
## 3 8833224/20625300
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0021-raw_1 and prad-0021-raw_2
## Single-end reads will be in prad-0021-raw_M
## 9733992 sequences in the BAM file
## 9733992 sequences exported
## 9733992 reads; of these:
##   9733992 (100.00%) were unpaired; of these:
##     3328504 (34.19%) aligned 0 times
##     4539198 (46.63%) aligned exactly 1 time
##     1866290 (19.17%) aligned &gt;1 times
## 65.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1866290/6405488
## 2 1378611/1866290
## 3 10590629/21562122
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0022-raw_1 and prad-0022-raw_2
## Single-end reads will be in prad-0022-raw_M
## 16162916 sequences in the BAM file
## 16162916 sequences exported
## 16162916 reads; of these:
##   16162916 (100.00%) were unpaired; of these:
##     5068652 (31.36%) aligned 0 times
##     8746377 (54.11%) aligned exactly 1 time
##     2347887 (14.53%) aligned &gt;1 times
## 68.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2347887/11094264
## 2 1653073/2347887
## 3 14331068/30472545
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0023-raw_1 and prad-0023-raw_2
## Single-end reads will be in prad-0023-raw_M
## 5850026 sequences in the BAM file
## 5850026 sequences exported
## 5850026 reads; of these:
##   5850026 (100.00%) were unpaired; of these:
##     1350089 (23.08%) aligned 0 times
##     3007772 (51.41%) aligned exactly 1 time
##     1492165 (25.51%) aligned &gt;1 times
## 76.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1492165/4499937
## 2 1156277/1492165
## 3 6207603/14330580
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0024-raw_1 and prad-0024-raw_2
## Single-end reads will be in prad-0024-raw_M
## 7478283 sequences in the BAM file
## 7478283 sequences exported
## 7478283 reads; of these:
##   7478283 (100.00%) were unpaired; of these:
##     3418664 (45.71%) aligned 0 times
##     2474465 (33.09%) aligned exactly 1 time
##     1585154 (21.20%) aligned &gt;1 times
## 54.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1585154/4059619
## 2 1102073/1585154
## 3 12650065/23340272
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0025-raw_1 and prad-0025-raw_2
## Single-end reads will be in prad-0025-raw_M
## 11777198 sequences in the BAM file
## 11777198 sequences exported
## 11777198 reads; of these:
##   11777198 (100.00%) were unpaired; of these:
##     3847857 (32.67%) aligned 0 times
##     5978528 (50.76%) aligned exactly 1 time
##     1950813 (16.56%) aligned &gt;1 times
## 67.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1950813/7929341
## 2 1358289/1950813
## 3 12430097/27006348
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0026-raw_1 and prad-0026-raw_2
## Single-end reads will be in prad-0026-raw_M
## 12932606 sequences in the BAM file
## 12932606 sequences exported
## 12932606 reads; of these:
##   12932606 (100.00%) were unpaired; of these:
##     4189532 (32.40%) aligned 0 times
##     7149991 (55.29%) aligned exactly 1 time
##     1593083 (12.32%) aligned &gt;1 times
## 67.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1593083/8743074
## 2 1034802/1593083
## 3 8069775/20705641
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0027-raw_1 and prad-0027-raw_2
## Single-end reads will be in prad-0027-raw_M
## 14423221 sequences in the BAM file
## 14423221 sequences exported
## 14423221 reads; of these:
##   14423221 (100.00%) were unpaired; of these:
##     4417882 (30.63%) aligned 0 times
##     7414705 (51.41%) aligned exactly 1 time
##     2590634 (17.96%) aligned &gt;1 times
## 69.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2590634/10005339
## 2 1808660/2590634
## 3 16302960/37833910
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0028-raw_1 and prad-0028-raw_2
## Single-end reads will be in prad-0028-raw_M
## 12462049 sequences in the BAM file
## 12462049 sequences exported
## 12462049 reads; of these:
##   12462049 (100.00%) were unpaired; of these:
##     4067193 (32.64%) aligned 0 times
##     6821003 (54.73%) aligned exactly 1 time
##     1573853 (12.63%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1573853/8394856
## 2 1073864/1573853
## 3 6301658/17438669
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0029-raw_1 and prad-0029-raw_2
## Single-end reads will be in prad-0029-raw_M
## 7174150 sequences in the BAM file
## 7174150 sequences exported
## 7174150 reads; of these:
##   7174150 (100.00%) were unpaired; of these:
##     2221373 (30.96%) aligned 0 times
##     3528144 (49.18%) aligned exactly 1 time
##     1424633 (19.86%) aligned &gt;1 times
## 69.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1424633/4952777
## 2 979809/1424633
## 3 7911715/17546645
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0030-raw_1 and prad-0030-raw_2
## Single-end reads will be in prad-0030-raw_M
## 11357443 sequences in the BAM file
## 11357443 sequences exported
## 11357443 reads; of these:
##   11357443 (100.00%) were unpaired; of these:
##     4189738 (36.89%) aligned 0 times
##     5695507 (50.15%) aligned exactly 1 time
##     1472198 (12.96%) aligned &gt;1 times
## 63.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1472198/7167705
## 2 925537/1472198
## 3 7888139/18404975
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0031-raw_1 and prad-0031-raw_2
## Single-end reads will be in prad-0031-raw_M
## 8989622 sequences in the BAM file
## 8989622 sequences exported
## 8989622 reads; of these:
##   8989622 (100.00%) were unpaired; of these:
##     2801546 (31.16%) aligned 0 times
##     4966792 (55.25%) aligned exactly 1 time
##     1221284 (13.59%) aligned &gt;1 times
## 68.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1221284/6188076
## 2 868459/1221284
## 3 6063489/15884116
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0032-raw_1 and prad-0032-raw_2
## Single-end reads will be in prad-0032-raw_M
## 12359185 sequences in the BAM file
## 12359185 sequences exported
## 12359185 reads; of these:
##   12359185 (100.00%) were unpaired; of these:
##     4251439 (34.40%) aligned 0 times
##     6461237 (52.28%) aligned exactly 1 time
##     1646509 (13.32%) aligned &gt;1 times
## 65.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1646509/8107746
## 2 1157379/1646509
## 3 8775076/21070682
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0033-raw_1 and prad-0033-raw_2
## Single-end reads will be in prad-0033-raw_M
## 10476454 sequences in the BAM file
## 10476454 sequences exported
## 10476454 reads; of these:
##   10476454 (100.00%) were unpaired; of these:
##     2081812 (19.87%) aligned 0 times
##     6065907 (57.90%) aligned exactly 1 time
##     2328735 (22.23%) aligned &gt;1 times
## 80.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2328735/8394642
## 2 1683322/2328735
## 3 10384304/24870275
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0034-raw_1 and prad-0034-raw_2
## Single-end reads will be in prad-0034-raw_M
## 10244642 sequences in the BAM file
## 10244642 sequences exported
## 10244642 reads; of these:
##   10244642 (100.00%) were unpaired; of these:
##     4015390 (39.20%) aligned 0 times
##     4545270 (44.37%) aligned exactly 1 time
##     1683982 (16.44%) aligned &gt;1 times
## 60.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1683982/6229252
## 2 1209811/1683982
## 3 7368341/16470616
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0035-raw_1 and prad-0035-raw_2
## Single-end reads will be in prad-0035-raw_M
## 9019060 sequences in the BAM file
## 9019060 sequences exported
## 9019060 reads; of these:
##   9019060 (100.00%) were unpaired; of these:
##     1882573 (20.87%) aligned 0 times
##     5322900 (59.02%) aligned exactly 1 time
##     1813587 (20.11%) aligned &gt;1 times
## 79.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1813587/7136487
## 2 1476377/1813587
## 3 5625745/15477437
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0036-raw_1 and prad-0036-raw_2
## Single-end reads will be in prad-0036-raw_M
## 20938160 sequences in the BAM file
## 20938160 sequences exported
## 20938160 reads; of these:
##   20938160 (100.00%) were unpaired; of these:
##     7426961 (35.47%) aligned 0 times
##     10953919 (52.32%) aligned exactly 1 time
##     2557280 (12.21%) aligned &gt;1 times
## 64.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2557280/13511199
## 2 1608146/2557280
## 3 10459020/28866357
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0037-raw_1 and prad-0037-raw_2
## Single-end reads will be in prad-0037-raw_M
## 9997473 sequences in the BAM file
## 9997473 sequences exported
## 9997473 reads; of these:
##   9997473 (100.00%) were unpaired; of these:
##     2048095 (20.49%) aligned 0 times
##     6036303 (60.38%) aligned exactly 1 time
##     1913075 (19.14%) aligned &gt;1 times
## 79.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1913075/7949378
## 2 1441130/1913075
## 3 11329259/26714130
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0038-raw_1 and prad-0038-raw_2
## Single-end reads will be in prad-0038-raw_M
## 13831213 sequences in the BAM file
## 13831213 sequences exported
## 13831213 reads; of these:
##   13831213 (100.00%) were unpaired; of these:
##     5218765 (37.73%) aligned 0 times
##     6767318 (48.93%) aligned exactly 1 time
##     1845130 (13.34%) aligned &gt;1 times
## 62.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1845130/8612448
## 2 1207870/1845130
## 3 8272377/20421308
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0039-raw_1 and prad-0039-raw_2
## Single-end reads will be in prad-0039-raw_M
## 8377054 sequences in the BAM file
## 8377054 sequences exported
## 8377054 reads; of these:
##   8377054 (100.00%) were unpaired; of these:
##     3089738 (36.88%) aligned 0 times
##     4086487 (48.78%) aligned exactly 1 time
##     1200829 (14.33%) aligned &gt;1 times
## 63.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1200829/5287316
## 2 804727/1200829
## 3 6837984/15651621
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0040-raw_1 and prad-0040-raw_2
## Single-end reads will be in prad-0040-raw_M
## 19790428 sequences in the BAM file
## 19790428 sequences exported
## 19790428 reads; of these:
##   19790428 (100.00%) were unpaired; of these:
##     7640592 (38.61%) aligned 0 times
##     9932248 (50.19%) aligned exactly 1 time
##     2217588 (11.21%) aligned &gt;1 times
## 61.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2217588/12149836
## 2 1499596/2217588
## 3 8018108/23796385
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0041-raw_1 and prad-0041-raw_2
## Single-end reads will be in prad-0041-raw_M
## 12128148 sequences in the BAM file
## 12128148 sequences exported
## 12128148 reads; of these:
##   12128148 (100.00%) were unpaired; of these:
##     3646417 (30.07%) aligned 0 times
##     6884575 (56.77%) aligned exactly 1 time
##     1597156 (13.17%) aligned &gt;1 times
## 69.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1597156/8481731
## 2 1104613/1597156
## 3 6687332/18459518
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0042-raw_1 and prad-0042-raw_2
## Single-end reads will be in prad-0042-raw_M
## 26429396 sequences in the BAM file
## 26429396 sequences exported
## 26429396 reads; of these:
##   26429396 (100.00%) were unpaired; of these:
##     7543383 (28.54%) aligned 0 times
##     15655837 (59.24%) aligned exactly 1 time
##     3230176 (12.22%) aligned &gt;1 times
## 71.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3230176/18886013
## 2 2129509/3230176
## 3 17275024/42205068
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0043-raw_1 and prad-0043-raw_2
## Single-end reads will be in prad-0043-raw_M
## 9613486 sequences in the BAM file
## 9613486 sequences exported
## 9613486 reads; of these:
##   9613486 (100.00%) were unpaired; of these:
##     3073780 (31.97%) aligned 0 times
##     5139942 (53.47%) aligned exactly 1 time
##     1399764 (14.56%) aligned &gt;1 times
## 68.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1399764/6539706
## 2 972799/1399764
## 3 7766156/17553955
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0044-raw_1 and prad-0044-raw_2
## Single-end reads will be in prad-0044-raw_M
## 10349808 sequences in the BAM file
## 10349808 sequences exported
## 10349808 reads; of these:
##   10349808 (100.00%) were unpaired; of these:
##     3735995 (36.10%) aligned 0 times
##     5186906 (50.12%) aligned exactly 1 time
##     1426907 (13.79%) aligned &gt;1 times
## 63.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1426907/6613813
## 2 901335/1426907
## 3 9461049/20690694
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0045-raw_1 and prad-0045-raw_2
## Single-end reads will be in prad-0045-raw_M
## 10578664 sequences in the BAM file
## 10578664 sequences exported
## 10578664 reads; of these:
##   10578664 (100.00%) were unpaired; of these:
##     2199891 (20.80%) aligned 0 times
##     5117075 (48.37%) aligned exactly 1 time
##     3261698 (30.83%) aligned &gt;1 times
## 79.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3261698/8378773
## 2 2156131/3261698
## 3 31419054/68740787
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 3.
## Output will be in prad-0046-raw_1 and prad-0046-raw_2
## Single-end reads will be in prad-0046-raw_M
## 6072144 sequences in the BAM file
## 6072144 sequences exported
## 6072144 reads; of these:
##   6072144 (100.00%) were unpaired; of these:
##     2086288 (34.36%) aligned 0 times
##     2859949 (47.10%) aligned exactly 1 time
##     1125907 (18.54%) aligned &gt;1 times
## 65.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1125907/3985856
## 2 870574/1125907
## 3 3827521/9515878
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0047-raw_1 and prad-0047-raw_2
## Single-end reads will be in prad-0047-raw_M
## 7548304 sequences in the BAM file
## 7548304 sequences exported
## 7548304 reads; of these:
##   7548304 (100.00%) were unpaired; of these:
##     1650970 (21.87%) aligned 0 times
##     4099340 (54.31%) aligned exactly 1 time
##     1797994 (23.82%) aligned &gt;1 times
## 78.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1797994/5897334
## 2 1274621/1797994
## 3 14089761/27366586
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0048-raw_1 and prad-0048-raw_2
## Single-end reads will be in prad-0048-raw_M
## 7643781 sequences in the BAM file
## 7643781 sequences exported
## 7643781 reads; of these:
##   7643781 (100.00%) were unpaired; of these:
##     2552529 (33.39%) aligned 0 times
##     3998072 (52.30%) aligned exactly 1 time
##     1093180 (14.30%) aligned &gt;1 times
## 66.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1093180/5091252
## 2 704435/1093180
## 3 5528930/13401748
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0049-raw_1 and prad-0049-raw_2
## Single-end reads will be in prad-0049-raw_M
## 19032097 sequences in the BAM file
## 19032097 sequences exported
## 19032097 reads; of these:
##   19032097 (100.00%) were unpaired; of these:
##     6046922 (31.77%) aligned 0 times
##     10610505 (55.75%) aligned exactly 1 time
##     2374670 (12.48%) aligned &gt;1 times
## 68.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2374670/12985175
## 2 1654852/2374670
## 3 9069419/26146226
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0050-raw_1 and prad-0050-raw_2
## Single-end reads will be in prad-0050-raw_M
## 13533983 sequences in the BAM file
## 13533983 sequences exported
## 13533983 reads; of these:
##   13533983 (100.00%) were unpaired; of these:
##     4352960 (32.16%) aligned 0 times
##     7625332 (56.34%) aligned exactly 1 time
##     1555691 (11.49%) aligned &gt;1 times
## 67.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1555691/9181023
## 2 1040504/1555691
## 3 6999027/19290646
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0051-raw_1 and prad-0051-raw_2
## Single-end reads will be in prad-0051-raw_M
## 12656853 sequences in the BAM file
## 12656853 sequences exported
## 12656853 reads; of these:
##   12656853 (100.00%) were unpaired; of these:
##     3584964 (28.32%) aligned 0 times
##     7446594 (58.83%) aligned exactly 1 time
##     1625295 (12.84%) aligned &gt;1 times
## 71.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1625295/9071889
## 2 1116253/1625295
## 3 8389900/22578489
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0052-raw_1 and prad-0052-raw_2
## Single-end reads will be in prad-0052-raw_M
## 8828738 sequences in the BAM file
## 8828738 sequences exported
## 8828738 reads; of these:
##   8828738 (100.00%) were unpaired; of these:
##     1668517 (18.90%) aligned 0 times
##     5778611 (65.45%) aligned exactly 1 time
##     1381610 (15.65%) aligned &gt;1 times
## 81.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1381610/7160221
## 2 963018/1381610
## 3 7127545/17506395
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0053-raw_1 and prad-0053-raw_2
## Single-end reads will be in prad-0053-raw_M
## 11793506 sequences in the BAM file
## 11793506 sequences exported
## 11793506 reads; of these:
##   11793506 (100.00%) were unpaired; of these:
##     3934868 (33.36%) aligned 0 times
##     6120555 (51.90%) aligned exactly 1 time
##     1738083 (14.74%) aligned &gt;1 times
## 66.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1738083/7858638
## 2 1215718/1738083
## 3 8066515/20139569
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0054-raw_1 and prad-0054-raw_2
## Single-end reads will be in prad-0054-raw_M
## 7533553 sequences in the BAM file
## 7533553 sequences exported
## 7533553 reads; of these:
##   7533553 (100.00%) were unpaired; of these:
##     2576929 (34.21%) aligned 0 times
##     3670423 (48.72%) aligned exactly 1 time
##     1286201 (17.07%) aligned &gt;1 times
## 65.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1286201/4956624
## 2 900680/1286201
## 3 5442066/13201225
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0055-raw_1 and prad-0055-raw_2
## Single-end reads will be in prad-0055-raw_M
## 15028046 sequences in the BAM file
## 15028046 sequences exported
## 15028046 reads; of these:
##   15028046 (100.00%) were unpaired; of these:
##     5236862 (34.85%) aligned 0 times
##     8026047 (53.41%) aligned exactly 1 time
##     1765137 (11.75%) aligned &gt;1 times
## 65.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1765137/9791184
## 2 1194642/1765137
## 3 11116375/24547511
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0056-raw_1 and prad-0056-raw_2
## Single-end reads will be in prad-0056-raw_M
## 36457450 sequences in the BAM file
## 36457450 sequences exported
## 36457450 reads; of these:
##   36457450 (100.00%) were unpaired; of these:
##     8768490 (24.05%) aligned 0 times
##     22824561 (62.61%) aligned exactly 1 time
##     4864399 (13.34%) aligned &gt;1 times
## 75.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4864399/27688960
## 2 3124916/4864399
## 3 38077844/76685244
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in prad-0057-raw_1 and prad-0057-raw_2
## Single-end reads will be in prad-0057-raw_M
## 6576014 sequences in the BAM file
## 6576014 sequences exported
## 6576014 reads; of these:
##   6576014 (100.00%) were unpaired; of these:
##     2032712 (30.91%) aligned 0 times
##     3037182 (46.19%) aligned exactly 1 time
##     1506120 (22.90%) aligned &gt;1 times
## 69.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1506120/4543302
## 2 1044201/1506120
## 3 10397863/21413719
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0058-raw_1 and prad-0058-raw_2
## Single-end reads will be in prad-0058-raw_M
## 7003582 sequences in the BAM file
## 7003582 sequences exported
## 7003582 reads; of these:
##   7003582 (100.00%) were unpaired; of these:
##     2378592 (33.96%) aligned 0 times
##     3614160 (51.60%) aligned exactly 1 time
##     1010830 (14.43%) aligned &gt;1 times
## 66.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1010830/4624990
## 2 668900/1010830
## 3 4502592/11424818
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0059-raw_1 and prad-0059-raw_2
## Single-end reads will be in prad-0059-raw_M
## 14033439 sequences in the BAM file
## 14033439 sequences exported
## 14033439 reads; of these:
##   14033439 (100.00%) were unpaired; of these:
##     2746410 (19.57%) aligned 0 times
##     8817624 (62.83%) aligned exactly 1 time
##     2469405 (17.60%) aligned &gt;1 times
## 80.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2469405/11287029
## 2 1710538/2469405
## 3 10586353/27969067
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0060-raw_1 and prad-0060-raw_2
## Single-end reads will be in prad-0060-raw_M
## 16931911 sequences in the BAM file
## 16931911 sequences exported
## 16931911 reads; of these:
##   16931911 (100.00%) were unpaired; of these:
##     6478546 (38.26%) aligned 0 times
##     8294402 (48.99%) aligned exactly 1 time
##     2158963 (12.75%) aligned &gt;1 times
## 61.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2158963/10453365
## 2 1408325/2158963
## 3 11181073/26448605
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0061-raw_1 and prad-0061-raw_2
## Single-end reads will be in prad-0061-raw_M
## 8530460 sequences in the BAM file
## 8530460 sequences exported
## 8530460 reads; of these:
##   8530460 (100.00%) were unpaired; of these:
##     1632021 (19.13%) aligned 0 times
##     5312801 (62.28%) aligned exactly 1 time
##     1585638 (18.59%) aligned &gt;1 times
## 80.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1585638/6898439
## 2 1175301/1585638
## 3 8202611/18621519
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0062-raw_1 and prad-0062-raw_2
## Single-end reads will be in prad-0062-raw_M
## 13347761 sequences in the BAM file
## 13347761 sequences exported
## 13347761 reads; of these:
##   13347761 (100.00%) were unpaired; of these:
##     2522595 (18.90%) aligned 0 times
##     8616485 (64.55%) aligned exactly 1 time
##     2208681 (16.55%) aligned &gt;1 times
## 81.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2208681/10825166
## 2 1644157/2208681
## 3 10300273/25624560
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0063-raw_1 and prad-0063-raw_2
## Single-end reads will be in prad-0063-raw_M
## 6489474 sequences in the BAM file
## 6489474 sequences exported
## 6489474 reads; of these:
##   6489474 (100.00%) were unpaired; of these:
##     2033657 (31.34%) aligned 0 times
##     3016131 (46.48%) aligned exactly 1 time
##     1439686 (22.18%) aligned &gt;1 times
## 68.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1439686/4455817
## 2 1109355/1439686
## 3 4728674/12092375
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0064-raw_1 and prad-0064-raw_2
## Single-end reads will be in prad-0064-raw_M
## 32143573 sequences in the BAM file
## 32143573 sequences exported
## 32143573 reads; of these:
##   32143573 (100.00%) were unpaired; of these:
##     9462699 (29.44%) aligned 0 times
##     19293995 (60.02%) aligned exactly 1 time
##     3386879 (10.54%) aligned &gt;1 times
## 70.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3386879/22680874
## 2 2205386/3386879
## 3 18345194/47144259
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0065-raw_1 and prad-0065-raw_2
## Single-end reads will be in prad-0065-raw_M
## 5597738 sequences in the BAM file
## 5597738 sequences exported
## 5597738 reads; of these:
##   5597738 (100.00%) were unpaired; of these:
##     1232940 (22.03%) aligned 0 times
##     3276117 (58.53%) aligned exactly 1 time
##     1088681 (19.45%) aligned &gt;1 times
## 77.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1088681/4364798
## 2 801790/1088681
## 3 4916690/12099834
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0066-raw_1 and prad-0066-raw_2
## Single-end reads will be in prad-0066-raw_M
## 14043345 sequences in the BAM file
## 14043345 sequences exported
## 14043345 reads; of these:
##   14043345 (100.00%) were unpaired; of these:
##     5432784 (38.69%) aligned 0 times
##     6537685 (46.55%) aligned exactly 1 time
##     2072876 (14.76%) aligned &gt;1 times
## 61.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2072876/8610561
## 2 1475709/2072876
## 3 9125156/21720139
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0067-raw_1 and prad-0067-raw_2
## Single-end reads will be in prad-0067-raw_M
## 7473729 sequences in the BAM file
## 7473729 sequences exported
## 7473729 reads; of these:
##   7473729 (100.00%) were unpaired; of these:
##     1457618 (19.50%) aligned 0 times
##     4467847 (59.78%) aligned exactly 1 time
##     1548264 (20.72%) aligned &gt;1 times
## 80.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1548264/6016111
## 2 1123227/1548264
## 3 8612505/18578058
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0068-raw_1 and prad-0068-raw_2
## Single-end reads will be in prad-0068-raw_M
## 9383823 sequences in the BAM file
## 9383823 sequences exported
## 9383823 reads; of these:
##   9383823 (100.00%) were unpaired; of these:
##     1567132 (16.70%) aligned 0 times
##     6266203 (66.78%) aligned exactly 1 time
##     1550488 (16.52%) aligned &gt;1 times
## 83.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1550488/7816691
## 2 1133583/1550488
## 3 7720974/18801263
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0069-raw_1 and prad-0069-raw_2
## Single-end reads will be in prad-0069-raw_M
## 11766609 sequences in the BAM file
## 11766609 sequences exported
## 11766609 reads; of these:
##   11766609 (100.00%) were unpaired; of these:
##     2398686 (20.39%) aligned 0 times
##     7182887 (61.04%) aligned exactly 1 time
##     2185036 (18.57%) aligned &gt;1 times
## 79.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2185036/9367923
## 2 1569952/2185036
## 3 10091512/25320378
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0070-raw_1 and prad-0070-raw_2
## Single-end reads will be in prad-0070-raw_M
## 11047187 sequences in the BAM file
## 11047187 sequences exported
## 11047187 reads; of these:
##   11047187 (100.00%) were unpaired; of these:
##     3464683 (31.36%) aligned 0 times
##     6177439 (55.92%) aligned exactly 1 time
##     1405065 (12.72%) aligned &gt;1 times
## 68.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1405065/7582504
## 2 902738/1405065
## 3 4961214/14990507
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0071-raw_1 and prad-0071-raw_2
## Single-end reads will be in prad-0071-raw_M
## 9924752 sequences in the BAM file
## 9924752 sequences exported
## 9924752 reads; of these:
##   9924752 (100.00%) were unpaired; of these:
##     3257988 (32.83%) aligned 0 times
##     5064844 (51.03%) aligned exactly 1 time
##     1601920 (16.14%) aligned &gt;1 times
## 67.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1601920/6666764
## 2 1102484/1601920
## 3 7344199/18037664
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0072-raw_1 and prad-0072-raw_2
## Single-end reads will be in prad-0072-raw_M
## 12524905 sequences in the BAM file
## 12524905 sequences exported
## 12524905 reads; of these:
##   12524905 (100.00%) were unpaired; of these:
##     4729279 (37.76%) aligned 0 times
##     6207190 (49.56%) aligned exactly 1 time
##     1588436 (12.68%) aligned &gt;1 times
## 62.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1588436/7795626
## 2 1091931/1588436
## 3 6167635/16850592
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0073-raw_1 and prad-0073-raw_2
## Single-end reads will be in prad-0073-raw_M
## 6917907 sequences in the BAM file
## 6917907 sequences exported
## 6917907 reads; of these:
##   6917907 (100.00%) were unpaired; of these:
##     2329083 (33.67%) aligned 0 times
##     3410242 (49.30%) aligned exactly 1 time
##     1178582 (17.04%) aligned &gt;1 times
## 66.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1178582/4588824
## 2 809538/1178582
## 3 5917021/13592180
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0074-raw_1 and prad-0074-raw_2
## Single-end reads will be in prad-0074-raw_M
## 18778580 sequences in the BAM file
## 18778580 sequences exported
## 18778580 reads; of these:
##   18778580 (100.00%) were unpaired; of these:
##     6772946 (36.07%) aligned 0 times
##     9711857 (51.72%) aligned exactly 1 time
##     2293777 (12.21%) aligned &gt;1 times
## 63.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2293777/12005634
## 2 1429030/2293777
## 3 10299613/26966646
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0075-raw_1 and prad-0075-raw_2
## Single-end reads will be in prad-0075-raw_M
## 9250438 sequences in the BAM file
## 9250438 sequences exported
## 9250438 reads; of these:
##   9250438 (100.00%) were unpaired; of these:
##     1845362 (19.95%) aligned 0 times
##     5820531 (62.92%) aligned exactly 1 time
##     1584545 (17.13%) aligned &gt;1 times
## 80.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1584545/7405076
## 2 1161361/1584545
## 3 6158322/16661520
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0076-raw_1 and prad-0076-raw_2
## Single-end reads will be in prad-0076-raw_M
## 9191492 sequences in the BAM file
## 9191492 sequences exported
## 9191492 reads; of these:
##   9191492 (100.00%) were unpaired; of these:
##     2917438 (31.74%) aligned 0 times
##     5047308 (54.91%) aligned exactly 1 time
##     1226746 (13.35%) aligned &gt;1 times
## 68.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1226746/6274054
## 2 801978/1226746
## 3 3740889/11758413
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0077-raw_1 and prad-0077-raw_2
## Single-end reads will be in prad-0077-raw_M
## 17419390 sequences in the BAM file
## 17419390 sequences exported
## 17419390 reads; of these:
##   17419390 (100.00%) were unpaired; of these:
##     5112001 (29.35%) aligned 0 times
##     10218790 (58.66%) aligned exactly 1 time
##     2088599 (11.99%) aligned &gt;1 times
## 70.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2088599/12307389
## 2 1412947/2088599
## 3 11859684/28413601
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0078-raw_1 and prad-0078-raw_2
## Single-end reads will be in prad-0078-raw_M
## 36922825 sequences in the BAM file
## 36922825 sequences exported
## 36922825 reads; of these:
##   36922825 (100.00%) were unpaired; of these:
##     9702768 (26.28%) aligned 0 times
##     21057474 (57.03%) aligned exactly 1 time
##     6162583 (16.69%) aligned &gt;1 times
## 73.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6162583/27220057
## 2 3994199/6162583
## 3 41700791/83827732
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 2.
## Output will be in prad-0079-raw_1 and prad-0079-raw_2
## Single-end reads will be in prad-0079-raw_M
## 14035495 sequences in the BAM file
## 14035495 sequences exported
## 14035495 reads; of these:
##   14035495 (100.00%) were unpaired; of these:
##     4286896 (30.54%) aligned 0 times
##     7505425 (53.47%) aligned exactly 1 time
##     2243174 (15.98%) aligned &gt;1 times
## 69.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2243174/9748599
## 2 1572844/2243174
## 3 9374880/23540309
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0080-raw_1 and prad-0080-raw_2
## Single-end reads will be in prad-0080-raw_M
## 9245318 sequences in the BAM file
## 9245318 sequences exported
## 9245318 reads; of these:
##   9245318 (100.00%) were unpaired; of these:
##     2727827 (29.50%) aligned 0 times
##     5330011 (57.65%) aligned exactly 1 time
##     1187480 (12.84%) aligned &gt;1 times
## 70.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1187480/6517491
## 2 782877/1187480
## 3 4884373/13794506
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0081-raw_1 and prad-0081-raw_2
## Single-end reads will be in prad-0081-raw_M
## 13316019 sequences in the BAM file
## 13316019 sequences exported
## 13316019 reads; of these:
##   13316019 (100.00%) were unpaired; of these:
##     3411811 (25.62%) aligned 0 times
##     7207726 (54.13%) aligned exactly 1 time
##     2696482 (20.25%) aligned &gt;1 times
## 74.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2696482/9904208
## 2 1803377/2696482
## 3 14132001/34036176
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0082-raw_1 and prad-0082-raw_2
## Single-end reads will be in prad-0082-raw_M
## 16392893 sequences in the BAM file
## 16392893 sequences exported
## 16392893 reads; of these:
##   16392893 (100.00%) were unpaired; of these:
##     2912996 (17.77%) aligned 0 times
##     10804223 (65.91%) aligned exactly 1 time
##     2675674 (16.32%) aligned &gt;1 times
## 82.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2675674/13479897
## 2 1906323/2675674
## 3 12964581/32465263
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0083-raw_1 and prad-0083-raw_2
## Single-end reads will be in prad-0083-raw_M
## 8777582 sequences in the BAM file
## 8777582 sequences exported
## 8777582 reads; of these:
##   8777582 (100.00%) were unpaired; of these:
##     1842367 (20.99%) aligned 0 times
##     3960590 (45.12%) aligned exactly 1 time
##     2974625 (33.89%) aligned &gt;1 times
## 79.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2974625/6935215
## 2 1945499/2974625
## 3 31748988/64750440
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 4.
## Output will be in prad-0084-raw_1 and prad-0084-raw_2
## Single-end reads will be in prad-0084-raw_M
## 18995708 sequences in the BAM file
## 18995708 sequences exported
## 18995708 reads; of these:
##   18995708 (100.00%) were unpaired; of these:
##     7461112 (39.28%) aligned 0 times
##     8705518 (45.83%) aligned exactly 1 time
##     2829078 (14.89%) aligned &gt;1 times
## 60.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2829078/11534596
## 2 2215103/2829078
## 3 9496856/25251901
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0085-raw_1 and prad-0085-raw_2
## Single-end reads will be in prad-0085-raw_M
## 10714136 sequences in the BAM file
## 10714136 sequences exported
## 10714136 reads; of these:
##   10714136 (100.00%) were unpaired; of these:
##     2043895 (19.08%) aligned 0 times
##     6788815 (63.36%) aligned exactly 1 time
##     1881426 (17.56%) aligned &gt;1 times
## 80.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1881426/8670241
## 2 1425585/1881426
## 3 6246753/17740202
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0086-raw_1 and prad-0086-raw_2
## Single-end reads will be in prad-0086-raw_M
## 26999561 sequences in the BAM file
## 26999561 sequences exported
## 26999561 reads; of these:
##   26999561 (100.00%) were unpaired; of these:
##     9854724 (36.50%) aligned 0 times
##     14540197 (53.85%) aligned exactly 1 time
##     2604640 (9.65%) aligned &gt;1 times
## 63.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2604640/17144837
## 2 1566489/2604640
## 3 13712874/35982554
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0087-raw_1 and prad-0087-raw_2
## Single-end reads will be in prad-0087-raw_M
## 9215268 sequences in the BAM file
## 9215268 sequences exported
## 9215268 reads; of these:
##   9215268 (100.00%) were unpaired; of these:
##     3172874 (34.43%) aligned 0 times
##     4604181 (49.96%) aligned exactly 1 time
##     1438213 (15.61%) aligned &gt;1 times
## 65.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1438213/6042394
## 2 973828/1438213
## 3 6264471/15956501
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0088-raw_1 and prad-0088-raw_2
## Single-end reads will be in prad-0088-raw_M
## 14534479 sequences in the BAM file
## 14534479 sequences exported
## 14534479 reads; of these:
##   14534479 (100.00%) were unpaired; of these:
##     5519253 (37.97%) aligned 0 times
##     7130104 (49.06%) aligned exactly 1 time
##     1885122 (12.97%) aligned &gt;1 times
## 62.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1885122/9015226
## 2 1282335/1885122
## 3 6629414/18607469
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0089-raw_1 and prad-0089-raw_2
## Single-end reads will be in prad-0089-raw_M
## 3313220 sequences in the BAM file
## 3313220 sequences exported
## 3313220 reads; of these:
##   3313220 (100.00%) were unpaired; of these:
##     1081571 (32.64%) aligned 0 times
##     1378210 (41.60%) aligned exactly 1 time
##     853439 (25.76%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 853439/2231649
## 2 596505/853439
## 3 6756775/13035541
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0090-raw_1 and prad-0090-raw_2
## Single-end reads will be in prad-0090-raw_M
## 10324452 sequences in the BAM file
## 10324452 sequences exported
## 10324452 reads; of these:
##   10324452 (100.00%) were unpaired; of these:
##     3096588 (29.99%) aligned 0 times
##     5935807 (57.49%) aligned exactly 1 time
##     1292057 (12.51%) aligned &gt;1 times
## 70.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1292057/7227864
## 2 836138/1292057
## 3 3963070/13130538
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0091-raw_1 and prad-0091-raw_2
## Single-end reads will be in prad-0091-raw_M
## 8020489 sequences in the BAM file
## 8020489 sequences exported
## 8020489 reads; of these:
##   8020489 (100.00%) were unpaired; of these:
##     2245846 (28.00%) aligned 0 times
##     4378460 (54.59%) aligned exactly 1 time
##     1396183 (17.41%) aligned &gt;1 times
## 72.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1396183/5774643
## 2 931912/1396183
## 3 5731674/14782178
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0092-raw_1 and prad-0092-raw_2
## Single-end reads will be in prad-0092-raw_M
## 12440313 sequences in the BAM file
## 12440313 sequences exported
## 12440313 reads; of these:
##   12440313 (100.00%) were unpaired; of these:
##     2223680 (17.87%) aligned 0 times
##     7941835 (63.84%) aligned exactly 1 time
##     2274798 (18.29%) aligned &gt;1 times
## 82.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2274798/10216633
## 2 1665877/2274798
## 3 10241467/25052260
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0093-raw_1 and prad-0093-raw_2
## Single-end reads will be in prad-0093-raw_M
## 5063729 sequences in the BAM file
## 5063729 sequences exported
## 5063729 reads; of these:
##   5063729 (100.00%) were unpaired; of these:
##     1017707 (20.10%) aligned 0 times
##     2954159 (58.34%) aligned exactly 1 time
##     1091863 (21.56%) aligned &gt;1 times
## 79.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1091863/4046022
## 2 790828/1091863
## 3 6231796/13213253
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0094-raw_1 and prad-0094-raw_2
## Single-end reads will be in prad-0094-raw_M
## 11686869 sequences in the BAM file
## 11686869 sequences exported
## 11686869 reads; of these:
##   11686869 (100.00%) were unpaired; of these:
##     3752120 (32.11%) aligned 0 times
##     6466594 (55.33%) aligned exactly 1 time
##     1468155 (12.56%) aligned &gt;1 times
## 67.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1468155/7934749
## 2 953479/1468155
## 3 5619962/16136660
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0095-raw_1 and prad-0095-raw_2
## Single-end reads will be in prad-0095-raw_M
## 17293523 sequences in the BAM file
## 17293523 sequences exported
## 17293523 reads; of these:
##   17293523 (100.00%) were unpaired; of these:
##     5369008 (31.05%) aligned 0 times
##     9567052 (55.32%) aligned exactly 1 time
##     2357463 (13.63%) aligned &gt;1 times
## 68.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2357463/11924515
## 2 1519868/2357463
## 3 13434025/31181298
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0096-raw_1 and prad-0096-raw_2
## Single-end reads will be in prad-0096-raw_M
## 14753074 sequences in the BAM file
## 14753074 sequences exported
## 14753074 reads; of these:
##   14753074 (100.00%) were unpaired; of these:
##     4411616 (29.90%) aligned 0 times
##     8559029 (58.02%) aligned exactly 1 time
##     1782429 (12.08%) aligned &gt;1 times
## 70.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1782429/10341458
## 2 1234751/1782429
## 3 12226887/26263784
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0097-raw_1 and prad-0097-raw_2
## Single-end reads will be in prad-0097-raw_M
## 8432471 sequences in the BAM file
## 8432471 sequences exported
## 8432471 reads; of these:
##   8432471 (100.00%) were unpaired; of these:
##     2775300 (32.91%) aligned 0 times
##     4442074 (52.68%) aligned exactly 1 time
##     1215097 (14.41%) aligned &gt;1 times
## 67.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1215097/5657171
## 2 844268/1215097
## 3 6325549/14752377
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0098-raw_1 and prad-0098-raw_2
## Single-end reads will be in prad-0098-raw_M
## 15879996 sequences in the BAM file
## 15879996 sequences exported
## 15879996 reads; of these:
##   15879996 (100.00%) were unpaired; of these:
##     4769543 (30.03%) aligned 0 times
##     9158628 (57.67%) aligned exactly 1 time
##     1951825 (12.29%) aligned &gt;1 times
## 69.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1951825/11110453
## 2 1323031/1951825
## 3 12037774/26762787
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0099-raw_1 and prad-0099-raw_2
## Single-end reads will be in prad-0099-raw_M
## 13206415 sequences in the BAM file
## 13206415 sequences exported
## 13206415 reads; of these:
##   13206415 (100.00%) were unpaired; of these:
##     4216374 (31.93%) aligned 0 times
##     7182103 (54.38%) aligned exactly 1 time
##     1807938 (13.69%) aligned &gt;1 times
## 68.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1807938/8990041
## 2 1303693/1807938
## 3 8504260/21867210
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0100-raw_1 and prad-0100-raw_2
## Single-end reads will be in prad-0100-raw_M
## 14490323 sequences in the BAM file
## 14490323 sequences exported
## 14490323 reads; of these:
##   14490323 (100.00%) were unpaired; of these:
##     4541699 (31.34%) aligned 0 times
##     8371021 (57.77%) aligned exactly 1 time
##     1577603 (10.89%) aligned &gt;1 times
## 68.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1577603/9948624
## 2 1022189/1577603
## 3 6524950/19697524
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0101-raw_1 and prad-0101-raw_2
## Single-end reads will be in prad-0101-raw_M
## 11078730 sequences in the BAM file
## 11078730 sequences exported
## 11078730 reads; of these:
##   11078730 (100.00%) were unpaired; of these:
##     2616693 (23.62%) aligned 0 times
##     6850083 (61.83%) aligned exactly 1 time
##     1611954 (14.55%) aligned &gt;1 times
## 76.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1611954/8462037
## 2 1179906/1611954
## 3 3716757/14315732
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0102-raw_1 and prad-0102-raw_2
## Single-end reads will be in prad-0102-raw_M
## 30065068 sequences in the BAM file
## 30065068 sequences exported
## 30065068 reads; of these:
##   30065068 (100.00%) were unpaired; of these:
##     4159872 (13.84%) aligned 0 times
##     22516928 (74.89%) aligned exactly 1 time
##     3388268 (11.27%) aligned &gt;1 times
## 86.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3388268/25905196
## 2 2383286/3388268
## 3 8395314/38870323
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 4.
## Output will be in prad-0103-raw_1 and prad-0103-raw_2
## Single-end reads will be in prad-0103-raw_M
## 10026649 sequences in the BAM file
## 10026649 sequences exported
## 10026649 reads; of these:
##   10026649 (100.00%) were unpaired; of these:
##     2337384 (23.31%) aligned 0 times
##     5657432 (56.42%) aligned exactly 1 time
##     2031833 (20.26%) aligned &gt;1 times
## 76.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2031833/7689265
## 2 1509690/2031833
## 3 7489588/20504746
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0104-raw_1 and prad-0104-raw_2
## Single-end reads will be in prad-0104-raw_M
## 18912977 sequences in the BAM file
## 18912977 sequences exported
## 18912977 reads; of these:
##   18912977 (100.00%) were unpaired; of these:
##     2574334 (13.61%) aligned 0 times
##     13253486 (70.08%) aligned exactly 1 time
##     3085157 (16.31%) aligned &gt;1 times
## 86.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3085157/16338643
## 2 2165874/3085157
## 3 7040746/27287288
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0105-raw_1 and prad-0105-raw_2
## Single-end reads will be in prad-0105-raw_M
## 4934902 sequences in the BAM file
## 4934902 sequences exported
## 4934902 reads; of these:
##   4934902 (100.00%) were unpaired; of these:
##     2299881 (46.60%) aligned 0 times
##     1861165 (37.71%) aligned exactly 1 time
##     773856 (15.68%) aligned &gt;1 times
## 53.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 773856/2635021
## 2 556792/773856
## 3 2708371/7303902
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0106-raw_1 and prad-0106-raw_2
## Single-end reads will be in prad-0106-raw_M
## 22376278 sequences in the BAM file
## 22376278 sequences exported
## 22376278 reads; of these:
##   22376278 (100.00%) were unpaired; of these:
##     7948629 (35.52%) aligned 0 times
##     11757174 (52.54%) aligned exactly 1 time
##     2670475 (11.93%) aligned &gt;1 times
## 64.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2670475/14427649
## 2 1922350/2670475
## 3 7581925/26850485
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0107-raw_1 and prad-0107-raw_2
## Single-end reads will be in prad-0107-raw_M
## 12221789 sequences in the BAM file
## 12221789 sequences exported
## 12221789 reads; of these:
##   12221789 (100.00%) were unpaired; of these:
##     4076489 (33.35%) aligned 0 times
##     6442720 (52.72%) aligned exactly 1 time
##     1702580 (13.93%) aligned &gt;1 times
## 66.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1702580/8145300
## 2 1202953/1702580
## 3 5909788/17160422
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0108-raw_1 and prad-0108-raw_2
## Single-end reads will be in prad-0108-raw_M
## 33311826 sequences in the BAM file
## 33311826 sequences exported
## 33311826 reads; of these:
##   33311826 (100.00%) were unpaired; of these:
##     13330088 (40.02%) aligned 0 times
##     15840639 (47.55%) aligned exactly 1 time
##     4141099 (12.43%) aligned &gt;1 times
## 59.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4141099/19981738
## 2 2770274/4141099
## 3 12138087/39912395
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0109-raw_1 and prad-0109-raw_2
## Single-end reads will be in prad-0109-raw_M
## 7448460 sequences in the BAM file
## 7448460 sequences exported
## 7448460 reads; of these:
##   7448460 (100.00%) were unpaired; of these:
##     1419240 (19.05%) aligned 0 times
##     4313922 (57.92%) aligned exactly 1 time
##     1715298 (23.03%) aligned &gt;1 times
## 80.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1715298/6029220
## 2 1197973/1715298
## 3 6378103/17211353
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0110-raw_1 and prad-0110-raw_2
## Single-end reads will be in prad-0110-raw_M
## 5883895 sequences in the BAM file
## 5883895 sequences exported
## 5883895 reads; of these:
##   5883895 (100.00%) were unpaired; of these:
##     1325011 (22.52%) aligned 0 times
##     3317031 (56.37%) aligned exactly 1 time
##     1241853 (21.11%) aligned &gt;1 times
## 77.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1241853/4558884
## 2 965219/1241853
## 3 5501682/13098000
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0111-raw_1 and prad-0111-raw_2
## Single-end reads will be in prad-0111-raw_M
## 11560640 sequences in the BAM file
## 11560640 sequences exported
## 11560640 reads; of these:
##   11560640 (100.00%) were unpaired; of these:
##     2358201 (20.40%) aligned 0 times
##     7038390 (60.88%) aligned exactly 1 time
##     2164049 (18.72%) aligned &gt;1 times
## 79.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2164049/9202439
## 2 1617051/2164049
## 3 9816745/25030010
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0112-raw_1 and prad-0112-raw_2
## Single-end reads will be in prad-0112-raw_M
## 12584924 sequences in the BAM file
## 12584924 sequences exported
## 12584924 reads; of these:
##   12584924 (100.00%) were unpaired; of these:
##     4012708 (31.89%) aligned 0 times
##     6606639 (52.50%) aligned exactly 1 time
##     1965577 (15.62%) aligned &gt;1 times
## 68.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1965577/8572216
## 2 1314079/1965577
## 3 7687162/21002904
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0113-raw_1 and prad-0113-raw_2
## Single-end reads will be in prad-0113-raw_M
## 7786789 sequences in the BAM file
## 7786789 sequences exported
## 7786789 reads; of these:
##   7786789 (100.00%) were unpaired; of these:
##     2651917 (34.06%) aligned 0 times
##     3468630 (44.55%) aligned exactly 1 time
##     1666242 (21.40%) aligned &gt;1 times
## 65.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1666242/5134872
## 2 1214773/1666242
## 3 3809334/11360325
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0114-raw_1 and prad-0114-raw_2
## Single-end reads will be in prad-0114-raw_M
## 12735716 sequences in the BAM file
## 12735716 sequences exported
## 12735716 reads; of these:
##   12735716 (100.00%) were unpaired; of these:
##     4088404 (32.10%) aligned 0 times
##     7207193 (56.59%) aligned exactly 1 time
##     1440119 (11.31%) aligned &gt;1 times
## 67.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1440119/8647312
## 2 960756/1440119
## 3 5453790/16515190
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0115-raw_1 and prad-0115-raw_2
## Single-end reads will be in prad-0115-raw_M
## 19132865 sequences in the BAM file
## 19132865 sequences exported
## 19132865 reads; of these:
##   19132865 (100.00%) were unpaired; of these:
##     4922371 (25.73%) aligned 0 times
##     11583854 (60.54%) aligned exactly 1 time
##     2626640 (13.73%) aligned &gt;1 times
## 74.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2626640/14210494
## 2 1832252/2626640
## 3 9684981/27809660
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0116-raw_1 and prad-0116-raw_2
## Single-end reads will be in prad-0116-raw_M
## 11917687 sequences in the BAM file
## 11917687 sequences exported
## 11917687 reads; of these:
##   11917687 (100.00%) were unpaired; of these:
##     3487019 (29.26%) aligned 0 times
##     6946171 (58.28%) aligned exactly 1 time
##     1484497 (12.46%) aligned &gt;1 times
## 70.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1484497/8430668
## 2 1014962/1484497
## 3 7402414/18318162
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0117-raw_1 and prad-0117-raw_2
## Single-end reads will be in prad-0117-raw_M
## 24425705 sequences in the BAM file
## 24425705 sequences exported
## 24425705 reads; of these:
##   24425705 (100.00%) were unpaired; of these:
##     10095181 (41.33%) aligned 0 times
##     11266935 (46.13%) aligned exactly 1 time
##     3063589 (12.54%) aligned &gt;1 times
## 58.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3063589/14330524
## 2 2067786/3063589
## 3 6769479/25231384
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0118-raw_1 and prad-0118-raw_2
## Single-end reads will be in prad-0118-raw_M
## 8405868 sequences in the BAM file
## 8405868 sequences exported
## 8405868 reads; of these:
##   8405868 (100.00%) were unpaired; of these:
##     2863398 (34.06%) aligned 0 times
##     3641622 (43.32%) aligned exactly 1 time
##     1900848 (22.61%) aligned &gt;1 times
## 65.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1900848/5542470
## 2 1328417/1900848
## 3 13420738/28769288
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0119-raw_1 and prad-0119-raw_2
## Single-end reads will be in prad-0119-raw_M
## 9588796 sequences in the BAM file
## 9588796 sequences exported
## 9588796 reads; of these:
##   9588796 (100.00%) were unpaired; of these:
##     3969309 (41.40%) aligned 0 times
##     4359115 (45.46%) aligned exactly 1 time
##     1260372 (13.14%) aligned &gt;1 times
## 58.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1260372/5619487
## 2 926950/1260372
## 3 6150427/15670030
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0120-raw_1 and prad-0120-raw_2
## Single-end reads will be in prad-0120-raw_M
## 6042020 sequences in the BAM file
## 6042020 sequences exported
## 6042020 reads; of these:
##   6042020 (100.00%) were unpaired; of these:
##     1362015 (22.54%) aligned 0 times
##     3508485 (58.07%) aligned exactly 1 time
##     1171520 (19.39%) aligned &gt;1 times
## 77.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1171520/4680005
## 2 808260/1171520
## 3 5452451/13340038
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0121-raw_1 and prad-0121-raw_2
## Single-end reads will be in prad-0121-raw_M
## 12411812 sequences in the BAM file
## 12411812 sequences exported
## 12411812 reads; of these:
##   12411812 (100.00%) were unpaired; of these:
##     4407801 (35.51%) aligned 0 times
##     6037669 (48.64%) aligned exactly 1 time
##     1966342 (15.84%) aligned &gt;1 times
## 64.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1966342/8004011
## 2 1343396/1966342
## 3 6114723/18603543
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0122-raw_1 and prad-0122-raw_2
## Single-end reads will be in prad-0122-raw_M
## 15608676 sequences in the BAM file
## 15608676 sequences exported
## 15608676 reads; of these:
##   15608676 (100.00%) were unpaired; of these:
##     5676248 (36.37%) aligned 0 times
##     7383175 (47.30%) aligned exactly 1 time
##     2549253 (16.33%) aligned &gt;1 times
## 63.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2549253/9932428
## 2 1783418/2549253
## 3 14027115/35599510
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0123-raw_1 and prad-0123-raw_2
## Single-end reads will be in prad-0123-raw_M
## 6463245 sequences in the BAM file
## 6463245 sequences exported
## 6463245 reads; of these:
##   6463245 (100.00%) were unpaired; of these:
##     2335737 (36.14%) aligned 0 times
##     3150615 (48.75%) aligned exactly 1 time
##     976893 (15.11%) aligned &gt;1 times
## 63.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 976893/4127508
## 2 682889/976893
## 3 3368299/10063142
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0124-raw_1 and prad-0124-raw_2
## Single-end reads will be in prad-0124-raw_M
## 7322967 sequences in the BAM file
## 7322967 sequences exported
## 7322967 reads; of these:
##   7322967 (100.00%) were unpaired; of these:
##     2562204 (34.99%) aligned 0 times
##     3757373 (51.31%) aligned exactly 1 time
##     1003390 (13.70%) aligned &gt;1 times
## 65.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1003390/4760763
## 2 714091/1003390
## 3 5829892/12564629
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0125-raw_1 and prad-0125-raw_2
## Single-end reads will be in prad-0125-raw_M
## 27427931 sequences in the BAM file
## 27427931 sequences exported
## 27427931 reads; of these:
##   27427931 (100.00%) were unpaired; of these:
##     7371708 (26.88%) aligned 0 times
##     17176355 (62.62%) aligned exactly 1 time
##     2879868 (10.50%) aligned &gt;1 times
## 73.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2879868/20056223
## 2 1893549/2879868
## 3 10240731/35717695
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0126-raw_1 and prad-0126-raw_2
## Single-end reads will be in prad-0126-raw_M
## 18293208 sequences in the BAM file
## 18293208 sequences exported
## 18293208 reads; of these:
##   18293208 (100.00%) were unpaired; of these:
##     6037227 (33.00%) aligned 0 times
##     10069623 (55.05%) aligned exactly 1 time
##     2186358 (11.95%) aligned &gt;1 times
## 67.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2186358/12255981
## 2 1484729/2186358
## 3 8060708/24122908
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0127-raw_1 and prad-0127-raw_2
## Single-end reads will be in prad-0127-raw_M
## 14565714 sequences in the BAM file
## 14565714 sequences exported
## 14565714 reads; of these:
##   14565714 (100.00%) were unpaired; of these:
##     4115522 (28.25%) aligned 0 times
##     8166211 (56.06%) aligned exactly 1 time
##     2283981 (15.68%) aligned &gt;1 times
## 71.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2283981/10450192
## 2 1624602/2283981
## 3 6940843/22128091
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0128-raw_1 and prad-0128-raw_2
## Single-end reads will be in prad-0128-raw_M
## 9300953 sequences in the BAM file
## 9300953 sequences exported
## 9300953 reads; of these:
##   9300953 (100.00%) were unpaired; of these:
##     4447790 (47.82%) aligned 0 times
##     3606743 (38.78%) aligned exactly 1 time
##     1246420 (13.40%) aligned &gt;1 times
## 52.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1246420/4853163
## 2 856316/1246420
## 3 2554214/9025906
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0129-raw_1 and prad-0129-raw_2
## Single-end reads will be in prad-0129-raw_M
## 8734656 sequences in the BAM file
## 8734656 sequences exported
## 8734656 reads; of these:
##   8734656 (100.00%) were unpaired; of these:
##     1800983 (20.62%) aligned 0 times
##     5286827 (60.53%) aligned exactly 1 time
##     1646846 (18.85%) aligned &gt;1 times
## 79.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1646846/6933673
## 2 1195079/1646846
## 3 8568875/19369653
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0130-raw_1 and prad-0130-raw_2
## Single-end reads will be in prad-0130-raw_M
## 7308280 sequences in the BAM file
## 7308280 sequences exported
## 7308280 reads; of these:
##   7308280 (100.00%) were unpaired; of these:
##     3072644 (42.04%) aligned 0 times
##     3149547 (43.10%) aligned exactly 1 time
##     1086089 (14.86%) aligned &gt;1 times
## 57.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1086089/4235636
## 2 825622/1086089
## 3 3907745/10088897
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0131-raw_1 and prad-0131-raw_2
## Single-end reads will be in prad-0131-raw_M
## 17519122 sequences in the BAM file
## 17519122 sequences exported
## 17519122 reads; of these:
##   17519122 (100.00%) were unpaired; of these:
##     7628424 (43.54%) aligned 0 times
##     7873379 (44.94%) aligned exactly 1 time
##     2017319 (11.51%) aligned &gt;1 times
## 56.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2017319/9890698
## 2 1463631/2017319
## 3 5084211/18039200
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0132-raw_1 and prad-0132-raw_2
## Single-end reads will be in prad-0132-raw_M
## 9730928 sequences in the BAM file
## 9730928 sequences exported
## 9730928 reads; of these:
##   9730928 (100.00%) were unpaired; of these:
##     2688154 (27.62%) aligned 0 times
##     5169733 (53.13%) aligned exactly 1 time
##     1873041 (19.25%) aligned &gt;1 times
## 72.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1873041/7042774
## 2 1196443/1873041
## 3 8322663/21645176
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0133-raw_1 and prad-0133-raw_2
## Single-end reads will be in prad-0133-raw_M
## 14766802 sequences in the BAM file
## 14766802 sequences exported
## 14766802 reads; of these:
##   14766802 (100.00%) were unpaired; of these:
##     5485140 (37.15%) aligned 0 times
##     6483219 (43.90%) aligned exactly 1 time
##     2798443 (18.95%) aligned &gt;1 times
## 62.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2798443/9281662
## 2 2002800/2798443
## 3 8504486/23251068
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0134-raw_1 and prad-0134-raw_2
## Single-end reads will be in prad-0134-raw_M
## 13722642 sequences in the BAM file
## 13722642 sequences exported
## 13722642 reads; of these:
##   13722642 (100.00%) were unpaired; of these:
##     2440736 (17.79%) aligned 0 times
##     7938084 (57.85%) aligned exactly 1 time
##     3343822 (24.37%) aligned &gt;1 times
## 82.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3343822/11281906
## 2 2351849/3343822
## 3 16269484/37020161
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0135-raw_1 and prad-0135-raw_2
## Single-end reads will be in prad-0135-raw_M
## 14490595 sequences in the BAM file
## 14490595 sequences exported
## 14490595 reads; of these:
##   14490595 (100.00%) were unpaired; of these:
##     4952876 (34.18%) aligned 0 times
##     7408971 (51.13%) aligned exactly 1 time
##     2128748 (14.69%) aligned &gt;1 times
## 65.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2128748/9537719
## 2 1434369/2128748
## 3 7634060/21254582
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0136-raw_1 and prad-0136-raw_2
## Single-end reads will be in prad-0136-raw_M
## 10341405 sequences in the BAM file
## 10341405 sequences exported
## 10341405 reads; of these:
##   10341405 (100.00%) were unpaired; of these:
##     3931529 (38.02%) aligned 0 times
##     4610012 (44.58%) aligned exactly 1 time
##     1799864 (17.40%) aligned &gt;1 times
## 61.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1799864/6409876
## 2 1264519/1799864
## 3 8451294/21534092
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0137-raw_1 and prad-0137-raw_2
## Single-end reads will be in prad-0137-raw_M
## 8854287 sequences in the BAM file
## 8854287 sequences exported
## 8854287 reads; of these:
##   8854287 (100.00%) were unpaired; of these:
##     3070988 (34.68%) aligned 0 times
##     4507702 (50.91%) aligned exactly 1 time
##     1275597 (14.41%) aligned &gt;1 times
## 65.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1275597/5783299
## 2 876220/1275597
## 3 4699841/13044967
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0138-raw_1 and prad-0138-raw_2
## Single-end reads will be in prad-0138-raw_M
## 6253130 sequences in the BAM file
## 6253130 sequences exported
## 6253130 reads; of these:
##   6253130 (100.00%) were unpaired; of these:
##     1144200 (18.30%) aligned 0 times
##     3879481 (62.04%) aligned exactly 1 time
##     1229449 (19.66%) aligned &gt;1 times
## 81.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1229449/5108930
## 2 892480/1229449
## 3 5508545/13044752
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0139-raw_1 and prad-0139-raw_2
## Single-end reads will be in prad-0139-raw_M
## 25283669 sequences in the BAM file
## 25283669 sequences exported
## 25283669 reads; of these:
##   25283669 (100.00%) were unpaired; of these:
##     8453305 (33.43%) aligned 0 times
##     13921891 (55.06%) aligned exactly 1 time
##     2908473 (11.50%) aligned &gt;1 times
## 66.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2908473/16830364
## 2 2062318/2908473
## 3 11200166/34721630
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0140-raw_1 and prad-0140-raw_2
## Single-end reads will be in prad-0140-raw_M
## 19309499 sequences in the BAM file
## 19309499 sequences exported
## 19309499 reads; of these:
##   19309499 (100.00%) were unpaired; of these:
##     5247348 (27.17%) aligned 0 times
##     10743569 (55.64%) aligned exactly 1 time
##     3318582 (17.19%) aligned &gt;1 times
## 72.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3318582/14062151
## 2 2370578/3318582
## 3 14241514/38749754
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0141-raw_1 and prad-0141-raw_2
## Single-end reads will be in prad-0141-raw_M
## 18524317 sequences in the BAM file
## 18524317 sequences exported
## 18524317 reads; of these:
##   18524317 (100.00%) were unpaired; of these:
##     5704384 (30.79%) aligned 0 times
##     10247425 (55.32%) aligned exactly 1 time
##     2572508 (13.89%) aligned &gt;1 times
## 69.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2572508/12819933
## 2 1900455/2572508
## 3 9951641/27215287
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0142-raw_1 and prad-0142-raw_2
## Single-end reads will be in prad-0142-raw_M
## 4002401 sequences in the BAM file
## 4002401 sequences exported
## 4002401 reads; of these:
##   4002401 (100.00%) were unpaired; of these:
##     1070966 (26.76%) aligned 0 times
##     2083984 (52.07%) aligned exactly 1 time
##     847451 (21.17%) aligned &gt;1 times
## 73.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 847451/2931435
## 2 610597/847451
## 3 4232002/9799352
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0143-raw_1 and prad-0143-raw_2
## Single-end reads will be in prad-0143-raw_M
## 7633994 sequences in the BAM file
## 7633994 sequences exported
## 7633994 reads; of these:
##   7633994 (100.00%) were unpaired; of these:
##     2644422 (34.64%) aligned 0 times
##     3627530 (47.52%) aligned exactly 1 time
##     1362042 (17.84%) aligned &gt;1 times
## 65.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1362042/4989572
## 2 952069/1362042
## 3 8910374/18206004
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0144-raw_1 and prad-0144-raw_2
## Single-end reads will be in prad-0144-raw_M
## 28485422 sequences in the BAM file
## 28485422 sequences exported
## 28485422 reads; of these:
##   28485422 (100.00%) were unpaired; of these:
##     10371345 (36.41%) aligned 0 times
##     14968863 (52.55%) aligned exactly 1 time
##     3145214 (11.04%) aligned &gt;1 times
## 63.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3145214/18114077
## 2 2130383/3145214
## 3 11741771/35118873
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0145-raw_1 and prad-0145-raw_2
## Single-end reads will be in prad-0145-raw_M
## 21055362 sequences in the BAM file
## 21055362 sequences exported
## 21055362 reads; of these:
##   21055362 (100.00%) were unpaired; of these:
##     6091193 (28.93%) aligned 0 times
##     11608961 (55.14%) aligned exactly 1 time
##     3355208 (15.94%) aligned &gt;1 times
## 71.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3355208/14964169
## 2 2395498/3355208
## 3 11484550/34479768
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0146-raw_1 and prad-0146-raw_2
## Single-end reads will be in prad-0146-raw_M
## 22392243 sequences in the BAM file
## 22392243 sequences exported
## 22392243 reads; of these:
##   22392243 (100.00%) were unpaired; of these:
##     6975945 (31.15%) aligned 0 times
##     11716834 (52.33%) aligned exactly 1 time
##     3699464 (16.52%) aligned &gt;1 times
## 68.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3699464/15416298
## 2 2701065/3699464
## 3 11974371/35717263
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0147-raw_1 and prad-0147-raw_2
## Single-end reads will be in prad-0147-raw_M
## 6005802 sequences in the BAM file
## 6005802 sequences exported
## 6005802 reads; of these:
##   6005802 (100.00%) were unpaired; of these:
##     2789772 (46.45%) aligned 0 times
##     2098321 (34.94%) aligned exactly 1 time
##     1117709 (18.61%) aligned &gt;1 times
## 53.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1117709/3216030
## 2 892548/1117709
## 3 2594366/7385728
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0148-raw_1 and prad-0148-raw_2
## Single-end reads will be in prad-0148-raw_M
## 5245021 sequences in the BAM file
## 5245021 sequences exported
## 5245021 reads; of these:
##   5245021 (100.00%) were unpaired; of these:
##     1094867 (20.87%) aligned 0 times
##     2743610 (52.31%) aligned exactly 1 time
##     1406544 (26.82%) aligned &gt;1 times
## 79.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1406544/4150154
## 2 1027448/1406544
## 3 8431795/17163742
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0149-raw_1 and prad-0149-raw_2
## Single-end reads will be in prad-0149-raw_M
## 8683620 sequences in the BAM file
## 8683620 sequences exported
## 8683620 reads; of these:
##   8683620 (100.00%) were unpaired; of these:
##     2796454 (32.20%) aligned 0 times
##     4352021 (50.12%) aligned exactly 1 time
##     1535145 (17.68%) aligned &gt;1 times
## 67.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1535145/5887166
## 2 1081566/1535145
## 3 8498745/19274498
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0150-raw_1 and prad-0150-raw_2
## Single-end reads will be in prad-0150-raw_M
## 20995598 sequences in the BAM file
## 20995598 sequences exported
## 20995598 reads; of these:
##   20995598 (100.00%) were unpaired; of these:
##     6522467 (31.07%) aligned 0 times
##     11893397 (56.65%) aligned exactly 1 time
##     2579734 (12.29%) aligned &gt;1 times
## 68.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2579734/14473131
## 2 1684895/2579734
## 3 14547516/34453891
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0151-raw_1 and prad-0151-raw_2
## Single-end reads will be in prad-0151-raw_M
## 17662476 sequences in the BAM file
## 17662476 sequences exported
## 17662476 reads; of these:
##   17662476 (100.00%) were unpaired; of these:
##     7956689 (45.05%) aligned 0 times
##     7601559 (43.04%) aligned exactly 1 time
##     2104228 (11.91%) aligned &gt;1 times
## 54.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2104228/9705787
## 2 1518912/2104228
## 3 5535931/18815379
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0152-raw_1 and prad-0152-raw_2
## Single-end reads will be in prad-0152-raw_M
## 19969366 sequences in the BAM file
## 19969366 sequences exported
## 19969366 reads; of these:
##   19969366 (100.00%) were unpaired; of these:
##     7657331 (38.35%) aligned 0 times
##     9780275 (48.98%) aligned exactly 1 time
##     2531760 (12.68%) aligned &gt;1 times
## 61.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2531760/12312035
## 2 1776080/2531760
## 3 7489096/23382759
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0153-raw_1 and prad-0153-raw_2
## Single-end reads will be in prad-0153-raw_M
## 9170167 sequences in the BAM file
## 9170167 sequences exported
## 9170167 reads; of these:
##   9170167 (100.00%) were unpaired; of these:
##     2958392 (32.26%) aligned 0 times
##     4779645 (52.12%) aligned exactly 1 time
##     1432130 (15.62%) aligned &gt;1 times
## 67.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1432130/6211775
## 2 1003274/1432130
## 3 6547937/16347076
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0154-raw_1 and prad-0154-raw_2
## Single-end reads will be in prad-0154-raw_M
## 8089668 sequences in the BAM file
## 8089668 sequences exported
## 8089668 reads; of these:
##   8089668 (100.00%) were unpaired; of these:
##     2912576 (36.00%) aligned 0 times
##     3702383 (45.77%) aligned exactly 1 time
##     1474709 (18.23%) aligned &gt;1 times
## 64.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1474709/5177092
## 2 985775/1474709
## 3 4086247/12214593
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0155-raw_1 and prad-0155-raw_2
## Single-end reads will be in prad-0155-raw_M
## 9183805 sequences in the BAM file
## 9183805 sequences exported
## 9183805 reads; of these:
##   9183805 (100.00%) were unpaired; of these:
##     2922798 (31.83%) aligned 0 times
##     4960648 (54.02%) aligned exactly 1 time
##     1300359 (14.16%) aligned &gt;1 times
## 68.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1300359/6261007
## 2 926309/1300359
## 3 4331140/12721592
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0156-raw_1 and prad-0156-raw_2
## Single-end reads will be in prad-0156-raw_M
## 7010809 sequences in the BAM file
## 7010809 sequences exported
## 7010809 reads; of these:
##   7010809 (100.00%) were unpaired; of these:
##     1505800 (21.48%) aligned 0 times
##     4187075 (59.72%) aligned exactly 1 time
##     1317934 (18.80%) aligned &gt;1 times
## 78.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1317934/5505009
## 2 958120/1317934
## 3 7236623/16759685
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0157-raw_1 and prad-0157-raw_2
## Single-end reads will be in prad-0157-raw_M
## 5642821 sequences in the BAM file
## 5642821 sequences exported
## 5642821 reads; of these:
##   5642821 (100.00%) were unpaired; of these:
##     1093507 (19.38%) aligned 0 times
##     2763670 (48.98%) aligned exactly 1 time
##     1785644 (31.64%) aligned &gt;1 times
## 80.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1785644/4549314
## 2 1153719/1785644
## 3 12522014/27368378
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0158-raw_1 and prad-0158-raw_2
## Single-end reads will be in prad-0158-raw_M
## 11771272 sequences in the BAM file
## 11771272 sequences exported
## 11771272 reads; of these:
##   11771272 (100.00%) were unpaired; of these:
##     3505915 (29.78%) aligned 0 times
##     5983293 (50.83%) aligned exactly 1 time
##     2282064 (19.39%) aligned &gt;1 times
## 70.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2282064/8265357
## 2 1550511/2282064
## 3 7798992/21552127
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0159-raw_1 and prad-0159-raw_2
## Single-end reads will be in prad-0159-raw_M
## 10388626 sequences in the BAM file
## 10388626 sequences exported
## 10388626 reads; of these:
##   10388626 (100.00%) were unpaired; of these:
##     3570152 (34.37%) aligned 0 times
##     4467097 (43.00%) aligned exactly 1 time
##     2351377 (22.63%) aligned &gt;1 times
## 65.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2351377/6818474
## 2 1821754/2351377
## 3 6597339/17674514
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0160-raw_1 and prad-0160-raw_2
## Single-end reads will be in prad-0160-raw_M
## 9506973 sequences in the BAM file
## 9506973 sequences exported
## 9506973 reads; of these:
##   9506973 (100.00%) were unpaired; of these:
##     1620615 (17.05%) aligned 0 times
##     5687297 (59.82%) aligned exactly 1 time
##     2199061 (23.13%) aligned &gt;1 times
## 82.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2199061/7886358
## 2 1666813/2199061
## 3 6604199/18771970
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0161-raw_1 and prad-0161-raw_2
## Single-end reads will be in prad-0161-raw_M
## 12302506 sequences in the BAM file
## 12302506 sequences exported
## 12302506 reads; of these:
##   12302506 (100.00%) were unpaired; of these:
##     3314928 (26.95%) aligned 0 times
##     6235245 (50.68%) aligned exactly 1 time
##     2752333 (22.37%) aligned &gt;1 times
## 73.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2752333/8987578
## 2 1977689/2752333
## 3 13757861/33496590
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0162-raw_1 and prad-0162-raw_2
## Single-end reads will be in prad-0162-raw_M
## 24017025 sequences in the BAM file
## 24017025 sequences exported
## 24017025 reads; of these:
##   24017025 (100.00%) were unpaired; of these:
##     4994719 (20.80%) aligned 0 times
##     15202479 (63.30%) aligned exactly 1 time
##     3819827 (15.90%) aligned &gt;1 times
## 79.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3819827/19022306
## 2 2688997/3819827
## 3 13897236/39017130
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0163-raw_1 and prad-0163-raw_2
## Single-end reads will be in prad-0163-raw_M
## 12710605 sequences in the BAM file
## 12710605 sequences exported
## 12710605 reads; of these:
##   12710605 (100.00%) were unpaired; of these:
##     7114399 (55.97%) aligned 0 times
##     4246478 (33.41%) aligned exactly 1 time
##     1349728 (10.62%) aligned &gt;1 times
## 44.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1349728/5596206
## 2 866492/1349728
## 3 4602008/13450523
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0164-raw_1 and prad-0164-raw_2
## Single-end reads will be in prad-0164-raw_M
## 9468889 sequences in the BAM file
## 9468889 sequences exported
## 9468889 reads; of these:
##   9468889 (100.00%) were unpaired; of these:
##     3045604 (32.16%) aligned 0 times
##     5037738 (53.20%) aligned exactly 1 time
##     1385547 (14.63%) aligned &gt;1 times
## 67.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1385547/6423285
## 2 966229/1385547
## 3 6830565/16075797
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0165-raw_1 and prad-0165-raw_2
## Single-end reads will be in prad-0165-raw_M
## 12733974 sequences in the BAM file
## 12733974 sequences exported
## 12733974 reads; of these:
##   12733974 (100.00%) were unpaired; of these:
##     4350201 (34.16%) aligned 0 times
##     6239105 (49.00%) aligned exactly 1 time
##     2144668 (16.84%) aligned &gt;1 times
## 65.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2144668/8383773
## 2 1495965/2144668
## 3 9705382/24210182
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0166-raw_1 and prad-0166-raw_2
## Single-end reads will be in prad-0166-raw_M
## 7544619 sequences in the BAM file
## 7544619 sequences exported
## 7544619 reads; of these:
##   7544619 (100.00%) were unpaired; of these:
##     2722551 (36.09%) aligned 0 times
##     3982197 (52.78%) aligned exactly 1 time
##     839871 (11.13%) aligned &gt;1 times
## 63.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 839871/4822068
## 2 554031/839871
## 3 3509639/10648822
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0167-raw_1 and prad-0167-raw_2
## Single-end reads will be in prad-0167-raw_M
## 15144575 sequences in the BAM file
## 15144575 sequences exported
## 15144575 reads; of these:
##   15144575 (100.00%) were unpaired; of these:
##     5945592 (39.26%) aligned 0 times
##     7112058 (46.96%) aligned exactly 1 time
##     2086925 (13.78%) aligned &gt;1 times
## 60.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2086925/9198983
## 2 1573669/2086925
## 3 6412681/18827148
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0168-raw_1 and prad-0168-raw_2
## Single-end reads will be in prad-0168-raw_M
## 10284702 sequences in the BAM file
## 10284702 sequences exported
## 10284702 reads; of these:
##   10284702 (100.00%) were unpaired; of these:
##     3169331 (30.82%) aligned 0 times
##     5557385 (54.04%) aligned exactly 1 time
##     1557986 (15.15%) aligned &gt;1 times
## 69.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1557986/7115371
## 2 1127516/1557986
## 3 6062288/16676477
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0169-raw_1 and prad-0169-raw_2
## Single-end reads will be in prad-0169-raw_M
## 27239266 sequences in the BAM file
## 27239266 sequences exported
## 27239266 reads; of these:
##   27239266 (100.00%) were unpaired; of these:
##     10178148 (37.37%) aligned 0 times
##     13859776 (50.88%) aligned exactly 1 time
##     3201342 (11.75%) aligned &gt;1 times
## 62.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3201342/17061118
## 2 2217158/3201342
## 3 10252085/31919974
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0170-raw_1 and prad-0170-raw_2
## Single-end reads will be in prad-0170-raw_M
## 7207218 sequences in the BAM file
## 7207218 sequences exported
## 7207218 reads; of these:
##   7207218 (100.00%) were unpaired; of these:
##     2962552 (41.11%) aligned 0 times
##     3319322 (46.06%) aligned exactly 1 time
##     925344 (12.84%) aligned &gt;1 times
## 58.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 925344/4244666
## 2 640359/925344
## 3 2986040/9132731
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0171-raw_1 and prad-0171-raw_2
## Single-end reads will be in prad-0171-raw_M
## 12401428 sequences in the BAM file
## 12401428 sequences exported
## 12401428 reads; of these:
##   12401428 (100.00%) were unpaired; of these:
##     3598951 (29.02%) aligned 0 times
##     6904468 (55.67%) aligned exactly 1 time
##     1898009 (15.30%) aligned &gt;1 times
## 70.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1898009/8802477
## 2 1416685/1898009
## 3 5860968/18543001
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0172-raw_1 and prad-0172-raw_2
## Single-end reads will be in prad-0172-raw_M
## 9587068 sequences in the BAM file
## 9587068 sequences exported
## 9587068 reads; of these:
##   9587068 (100.00%) were unpaired; of these:
##     2727950 (28.45%) aligned 0 times
##     4636759 (48.36%) aligned exactly 1 time
##     2222359 (23.18%) aligned &gt;1 times
## 71.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2222359/6859118
## 2 1581762/2222359
## 3 8361190/21386682
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0173-raw_1 and prad-0173-raw_2
## Single-end reads will be in prad-0173-raw_M
## 17231995 sequences in the BAM file
## 17231995 sequences exported
## 17231995 reads; of these:
##   17231995 (100.00%) were unpaired; of these:
##     3147111 (18.26%) aligned 0 times
##     10717956 (62.20%) aligned exactly 1 time
##     3366928 (19.54%) aligned &gt;1 times
## 81.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3366928/14084884
## 2 2325135/3366928
## 3 14703493/39515324
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0174-raw_1 and prad-0174-raw_2
## Single-end reads will be in prad-0174-raw_M
## 17836032 sequences in the BAM file
## 17836032 sequences exported
## 17836032 reads; of these:
##   17836032 (100.00%) were unpaired; of these:
##     3728115 (20.90%) aligned 0 times
##     8589855 (48.16%) aligned exactly 1 time
##     5518062 (30.94%) aligned &gt;1 times
## 79.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5518062/14107917
## 2 3469848/5518062
## 3 38257683/90606997
## [bam_sort_core] merging from 15 files...
## This looks like paired data from lane 7.
## Output will be in prad-0175-raw_1 and prad-0175-raw_2
## Single-end reads will be in prad-0175-raw_M
## 11247673 sequences in the BAM file
## 11247673 sequences exported
## 11247673 reads; of these:
##   11247673 (100.00%) were unpaired; of these:
##     3179408 (28.27%) aligned 0 times
##     6001793 (53.36%) aligned exactly 1 time
##     2066472 (18.37%) aligned &gt;1 times
## 71.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2066472/8068265
## 2 1379938/2066472
## 3 10146553/24716728
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0176-raw_1 and prad-0176-raw_2
## Single-end reads will be in prad-0176-raw_M
## 13623475 sequences in the BAM file
## 13623475 sequences exported
## 13623475 reads; of these:
##   13623475 (100.00%) were unpaired; of these:
##     5054504 (37.10%) aligned 0 times
##     6458040 (47.40%) aligned exactly 1 time
##     2110931 (15.49%) aligned &gt;1 times
## 62.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2110931/8568971
## 2 1578242/2110931
## 3 6862956/21780011
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0177-raw_1 and prad-0177-raw_2
## Single-end reads will be in prad-0177-raw_M
## 6600843 sequences in the BAM file
## 6600843 sequences exported
## 6600843 reads; of these:
##   6600843 (100.00%) were unpaired; of these:
##     1229441 (18.63%) aligned 0 times
##     3839346 (58.16%) aligned exactly 1 time
##     1532056 (23.21%) aligned &gt;1 times
## 81.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1532056/5371402
## 2 1119399/1532056
## 3 4620789/13109042
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0178-raw_1 and prad-0178-raw_2
## Single-end reads will be in prad-0178-raw_M
## 14060016 sequences in the BAM file
## 14060016 sequences exported
## 14060016 reads; of these:
##   14060016 (100.00%) were unpaired; of these:
##     3693703 (26.27%) aligned 0 times
##     8274398 (58.85%) aligned exactly 1 time
##     2091915 (14.88%) aligned &gt;1 times
## 73.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2091915/10366313
## 2 1412369/2091915
## 3 9954206/26970308
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0179-raw_1 and prad-0179-raw_2
## Single-end reads will be in prad-0179-raw_M
## 12656074 sequences in the BAM file
## 12656074 sequences exported
## 12656074 reads; of these:
##   12656074 (100.00%) were unpaired; of these:
##     4898884 (38.71%) aligned 0 times
##     6285535 (49.66%) aligned exactly 1 time
##     1471655 (11.63%) aligned &gt;1 times
## 61.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1471655/7757190
## 2 1019927/1471655
## 3 5123444/15578123
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0180-raw_1 and prad-0180-raw_2
## Single-end reads will be in prad-0180-raw_M
## 9110751 sequences in the BAM file
## 9110751 sequences exported
## 9110751 reads; of these:
##   9110751 (100.00%) were unpaired; of these:
##     1780544 (19.54%) aligned 0 times
##     5320946 (58.40%) aligned exactly 1 time
##     2009261 (22.05%) aligned &gt;1 times
## 80.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2009261/7330207
## 2 1395170/2009261
## 3 10628595/24426563
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0181-raw_1 and prad-0181-raw_2
## Single-end reads will be in prad-0181-raw_M
## 9965172 sequences in the BAM file
## 9965172 sequences exported
## 9965172 reads; of these:
##   9965172 (100.00%) were unpaired; of these:
##     3633784 (36.46%) aligned 0 times
##     4327773 (43.43%) aligned exactly 1 time
##     2003615 (20.11%) aligned &gt;1 times
## 63.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2003615/6331388
## 2 1347971/2003615
## 3 7592251/20802920
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0182-raw_1 and prad-0182-raw_2
## Single-end reads will be in prad-0182-raw_M
## 11895601 sequences in the BAM file
## 11895601 sequences exported
## 11895601 reads; of these:
##   11895601 (100.00%) were unpaired; of these:
##     2015194 (16.94%) aligned 0 times
##     7927299 (66.64%) aligned exactly 1 time
##     1953108 (16.42%) aligned &gt;1 times
## 83.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1953108/9880407
## 2 1382017/1953108
## 3 7178968/21524968
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0183-raw_1 and prad-0183-raw_2
## Single-end reads will be in prad-0183-raw_M
## 11689488 sequences in the BAM file
## 11689488 sequences exported
## 11689488 reads; of these:
##   11689488 (100.00%) were unpaired; of these:
##     5175986 (44.28%) aligned 0 times
##     4848234 (41.48%) aligned exactly 1 time
##     1665268 (14.25%) aligned &gt;1 times
## 55.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1665268/6513502
## 2 1207922/1665268
## 3 4767759/14380859
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0184-raw_1 and prad-0184-raw_2
## Single-end reads will be in prad-0184-raw_M
## 20374801 sequences in the BAM file
## 20374801 sequences exported
## 20374801 reads; of these:
##   20374801 (100.00%) were unpaired; of these:
##     7321749 (35.94%) aligned 0 times
##     10437945 (51.23%) aligned exactly 1 time
##     2615107 (12.84%) aligned &gt;1 times
## 64.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2615107/13053052
## 2 1835659/2615107
## 3 8648189/25993487
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0185-raw_1 and prad-0185-raw_2
## Single-end reads will be in prad-0185-raw_M
## 8715764 sequences in the BAM file
## 8715764 sequences exported
## 8715764 reads; of these:
##   8715764 (100.00%) were unpaired; of these:
##     3142666 (36.06%) aligned 0 times
##     3927309 (45.06%) aligned exactly 1 time
##     1645789 (18.88%) aligned &gt;1 times
## 63.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1645789/5573098
## 2 1167983/1645789
## 3 6078169/16332314
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0186-raw_1 and prad-0186-raw_2
## Single-end reads will be in prad-0186-raw_M
## 13624908 sequences in the BAM file
## 13624908 sequences exported
## 13624908 reads; of these:
##   13624908 (100.00%) were unpaired; of these:
##     2655431 (19.49%) aligned 0 times
##     7874791 (57.80%) aligned exactly 1 time
##     3094686 (22.71%) aligned &gt;1 times
## 80.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3094686/10969477
## 2 2175558/3094686
## 3 14466930/33484946
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0187-raw_1 and prad-0187-raw_2
## Single-end reads will be in prad-0187-raw_M
## 15588110 sequences in the BAM file
## 15588110 sequences exported
## 15588110 reads; of these:
##   15588110 (100.00%) were unpaired; of these:
##     4912090 (31.51%) aligned 0 times
##     8551895 (54.86%) aligned exactly 1 time
##     2124125 (13.63%) aligned &gt;1 times
## 68.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2124125/10676020
## 2 1506279/2124125
## 3 15260521/35175033
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0188-raw_1 and prad-0188-raw_2
## Single-end reads will be in prad-0188-raw_M
## 28515958 sequences in the BAM file
## 28515958 sequences exported
## 28515958 reads; of these:
##   28515958 (100.00%) were unpaired; of these:
##     14916437 (52.31%) aligned 0 times
##     11027336 (38.67%) aligned exactly 1 time
##     2572185 (9.02%) aligned &gt;1 times
## 47.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2572185/13599521
## 2 1616631/2572185
## 3 6649278/24812226
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0189-raw_1 and prad-0189-raw_2
## Single-end reads will be in prad-0189-raw_M
## 21511050 sequences in the BAM file
## 21511050 sequences exported
## 21511050 reads; of these:
##   21511050 (100.00%) were unpaired; of these:
##     8841895 (41.10%) aligned 0 times
##     9953472 (46.27%) aligned exactly 1 time
##     2715683 (12.62%) aligned &gt;1 times
## 58.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2715683/12669155
## 2 1756916/2715683
## 3 7398068/24787854
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0190-raw_1 and prad-0190-raw_2
## Single-end reads will be in prad-0190-raw_M
## 14780501 sequences in the BAM file
## 14780501 sequences exported
## 14780501 reads; of these:
##   14780501 (100.00%) were unpaired; of these:
##     4112042 (27.82%) aligned 0 times
##     8200719 (55.48%) aligned exactly 1 time
##     2467740 (16.70%) aligned &gt;1 times
## 72.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2467740/10668459
## 2 1601551/2467740
## 3 10827500/28376570
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0191-raw_1 and prad-0191-raw_2
## Single-end reads will be in prad-0191-raw_M
## 11049749 sequences in the BAM file
## 11049749 sequences exported
## 11049749 reads; of these:
##   11049749 (100.00%) were unpaired; of these:
##     3701697 (33.50%) aligned 0 times
##     5786365 (52.37%) aligned exactly 1 time
##     1561687 (14.13%) aligned &gt;1 times
## 66.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1561687/7348052
## 2 1068842/1561687
## 3 6130248/17011909
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0192-raw_1 and prad-0192-raw_2
## Single-end reads will be in prad-0192-raw_M
## 9437481 sequences in the BAM file
## 9437481 sequences exported
## 9437481 reads; of these:
##   9437481 (100.00%) were unpaired; of these:
##     3153698 (33.42%) aligned 0 times
##     4450259 (47.16%) aligned exactly 1 time
##     1833524 (19.43%) aligned &gt;1 times
## 66.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1833524/6283783
## 2 1323833/1833524
## 3 9154560/22482076
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0193-raw_1 and prad-0193-raw_2
## Single-end reads will be in prad-0193-raw_M
## 12896181 sequences in the BAM file
## 12896181 sequences exported
## 12896181 reads; of these:
##   12896181 (100.00%) were unpaired; of these:
##     4474747 (34.70%) aligned 0 times
##     6246099 (48.43%) aligned exactly 1 time
##     2175335 (16.87%) aligned &gt;1 times
## 65.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2175335/8421434
## 2 1548952/2175335
## 3 7111564/21257379
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0194-raw_1 and prad-0194-raw_2
## Single-end reads will be in prad-0194-raw_M
## 10018662 sequences in the BAM file
## 10018662 sequences exported
## 10018662 reads; of these:
##   10018662 (100.00%) were unpaired; of these:
##     4534409 (45.26%) aligned 0 times
##     4148405 (41.41%) aligned exactly 1 time
##     1335848 (13.33%) aligned &gt;1 times
## 54.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1335848/5484253
## 2 924015/1335848
## 3 3921580/12181413
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0195-raw_1 and prad-0195-raw_2
## Single-end reads will be in prad-0195-raw_M
## 11943715 sequences in the BAM file
## 11943715 sequences exported
## 11943715 reads; of these:
##   11943715 (100.00%) were unpaired; of these:
##     2308720 (19.33%) aligned 0 times
##     6893068 (57.71%) aligned exactly 1 time
##     2741927 (22.96%) aligned &gt;1 times
## 80.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2741927/9634995
## 2 1984951/2741927
## 3 9440179/24949100
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0196-raw_1 and prad-0196-raw_2
## Single-end reads will be in prad-0196-raw_M
## 18347491 sequences in the BAM file
## 18347491 sequences exported
## 18347491 reads; of these:
##   18347491 (100.00%) were unpaired; of these:
##     7535460 (41.07%) aligned 0 times
##     8399959 (45.78%) aligned exactly 1 time
##     2412072 (13.15%) aligned &gt;1 times
## 58.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2412072/10812031
## 2 1864941/2412072
## 3 8517888/24416233
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0197-raw_1 and prad-0197-raw_2
## Single-end reads will be in prad-0197-raw_M
## 18684621 sequences in the BAM file
## 18684621 sequences exported
## 18684621 reads; of these:
##   18684621 (100.00%) were unpaired; of these:
##     7215624 (38.62%) aligned 0 times
##     9611372 (51.44%) aligned exactly 1 time
##     1857625 (9.94%) aligned &gt;1 times
## 61.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1857625/11468997
## 2 1314817/1857625
## 3 7297017/21873085
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0198-raw_1 and prad-0198-raw_2
## Single-end reads will be in prad-0198-raw_M
## 10937475 sequences in the BAM file
## 10937475 sequences exported
## 10937475 reads; of these:
##   10937475 (100.00%) were unpaired; of these:
##     3373381 (30.84%) aligned 0 times
##     6126919 (56.02%) aligned exactly 1 time
##     1437175 (13.14%) aligned &gt;1 times
## 69.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1437175/7564094
## 2 897011/1437175
## 3 5334138/15921174
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0199-raw_1 and prad-0199-raw_2
## Single-end reads will be in prad-0199-raw_M
## 9055572 sequences in the BAM file
## 9055572 sequences exported
## 9055572 reads; of these:
##   9055572 (100.00%) were unpaired; of these:
##     3069408 (33.90%) aligned 0 times
##     4598624 (50.78%) aligned exactly 1 time
##     1387540 (15.32%) aligned &gt;1 times
## 66.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1387540/5986164
## 2 1004618/1387540
## 3 5350224/14361867
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0200-raw_1 and prad-0200-raw_2
## Single-end reads will be in prad-0200-raw_M
## 7958394 sequences in the BAM file
## 7958394 sequences exported
## 7958394 reads; of these:
##   7958394 (100.00%) were unpaired; of these:
##     3169196 (39.82%) aligned 0 times
##     3640087 (45.74%) aligned exactly 1 time
##     1149111 (14.44%) aligned &gt;1 times
## 60.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1149111/4789198
## 2 752601/1149111
## 3 4135768/11872128
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0201-raw_1 and prad-0201-raw_2
## Single-end reads will be in prad-0201-raw_M
## 20445809 sequences in the BAM file
## 20445809 sequences exported
## 20445809 reads; of these:
##   20445809 (100.00%) were unpaired; of these:
##     4225232 (20.67%) aligned 0 times
##     11434614 (55.93%) aligned exactly 1 time
##     4785963 (23.41%) aligned &gt;1 times
## 79.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4785963/16220577
## 2 3759446/4785963
## 3 17710616/44099467
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0202-raw_1 and prad-0202-raw_2
## Single-end reads will be in prad-0202-raw_M
## 9303775 sequences in the BAM file
## 9303775 sequences exported
## 9303775 reads; of these:
##   9303775 (100.00%) were unpaired; of these:
##     3236602 (34.79%) aligned 0 times
##     4277211 (45.97%) aligned exactly 1 time
##     1789962 (19.24%) aligned &gt;1 times
## 65.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1789962/6067173
## 2 1285404/1789962
## 3 8565660/21295569
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0203-raw_1 and prad-0203-raw_2
## Single-end reads will be in prad-0203-raw_M
## 6909261 sequences in the BAM file
## 6909261 sequences exported
## 6909261 reads; of these:
##   6909261 (100.00%) were unpaired; of these:
##     2361036 (34.17%) aligned 0 times
##     2854559 (41.31%) aligned exactly 1 time
##     1693666 (24.51%) aligned &gt;1 times
## 65.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1693666/4548225
## 2 1378610/1693666
## 3 5562827/13878877
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0204-raw_1 and prad-0204-raw_2
## Single-end reads will be in prad-0204-raw_M
## 5636184 sequences in the BAM file
## 5636184 sequences exported
## 5636184 reads; of these:
##   5636184 (100.00%) were unpaired; of these:
##     1290407 (22.90%) aligned 0 times
##     2610495 (46.32%) aligned exactly 1 time
##     1735282 (30.79%) aligned &gt;1 times
## 77.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1735282/4345777
## 2 1146135/1735282
## 3 17182218/33502622
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0205-raw_1 and prad-0205-raw_2
## Single-end reads will be in prad-0205-raw_M
## 13728038 sequences in the BAM file
## 13728038 sequences exported
## 13728038 reads; of these:
##   13728038 (100.00%) were unpaired; of these:
##     2421814 (17.64%) aligned 0 times
##     6969066 (50.77%) aligned exactly 1 time
##     4337158 (31.59%) aligned &gt;1 times
## 82.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4337158/11306224
## 2 3370200/4337158
## 3 35338870/73977533
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 5.
## Output will be in prad-0206-raw_1 and prad-0206-raw_2
## Single-end reads will be in prad-0206-raw_M
## 16143138 sequences in the BAM file
## 16143138 sequences exported
## 16143138 reads; of these:
##   16143138 (100.00%) were unpaired; of these:
##     5997367 (37.15%) aligned 0 times
##     7738255 (47.94%) aligned exactly 1 time
##     2407516 (14.91%) aligned &gt;1 times
## 62.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2407516/10145771
## 2 1662655/2407516
## 3 8373695/24755603
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0207-raw_1 and prad-0207-raw_2
## Single-end reads will be in prad-0207-raw_M
## 8700535 sequences in the BAM file
## 8700535 sequences exported
## 8700535 reads; of these:
##   8700535 (100.00%) were unpaired; of these:
##     4033642 (46.36%) aligned 0 times
##     3548277 (40.78%) aligned exactly 1 time
##     1118616 (12.86%) aligned &gt;1 times
## 53.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1118616/4666893
## 2 758650/1118616
## 3 2953235/9528956
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0208-raw_1 and prad-0208-raw_2
## Single-end reads will be in prad-0208-raw_M
## 11383056 sequences in the BAM file
## 11383056 sequences exported
## 11383056 reads; of these:
##   11383056 (100.00%) were unpaired; of these:
##     5440847 (47.80%) aligned 0 times
##     4447197 (39.07%) aligned exactly 1 time
##     1495012 (13.13%) aligned &gt;1 times
## 52.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1495012/5942209
## 2 1089314/1495012
## 3 4585851/13643729
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0209-raw_1 and prad-0209-raw_2
## Single-end reads will be in prad-0209-raw_M
## 13401856 sequences in the BAM file
## 13401856 sequences exported
## 13401856 reads; of these:
##   13401856 (100.00%) were unpaired; of these:
##     4508418 (33.64%) aligned 0 times
##     7252216 (54.11%) aligned exactly 1 time
##     1641222 (12.25%) aligned &gt;1 times
## 66.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1641222/8893438
## 2 1110422/1641222
## 3 5417591/16796727
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0210-raw_1 and prad-0210-raw_2
## Single-end reads will be in prad-0210-raw_M
## 16953621 sequences in the BAM file
## 16953621 sequences exported
## 16953621 reads; of these:
##   16953621 (100.00%) were unpaired; of these:
##     4982385 (29.39%) aligned 0 times
##     10117699 (59.68%) aligned exactly 1 time
##     1853537 (10.93%) aligned &gt;1 times
## 70.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1853537/11971236
## 2 1257354/1853537
## 3 10072400/25330228
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0211-raw_1 and prad-0211-raw_2
## Single-end reads will be in prad-0211-raw_M
## 14210827 sequences in the BAM file
## 14210827 sequences exported
## 14210827 reads; of these:
##   14210827 (100.00%) were unpaired; of these:
##     5552546 (39.07%) aligned 0 times
##     6363765 (44.78%) aligned exactly 1 time
##     2294516 (16.15%) aligned &gt;1 times
## 60.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2294516/8658281
## 2 1757221/2294516
## 3 11665495/27781236
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0212-raw_1 and prad-0212-raw_2
## Single-end reads will be in prad-0212-raw_M
## 15284693 sequences in the BAM file
## 15284693 sequences exported
## 15284693 reads; of these:
##   15284693 (100.00%) were unpaired; of these:
##     5240096 (34.28%) aligned 0 times
##     8044817 (52.63%) aligned exactly 1 time
##     1999780 (13.08%) aligned &gt;1 times
## 65.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1999780/10044597
## 2 1332046/1999780
## 3 12606115/28179182
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0213-raw_1 and prad-0213-raw_2
## Single-end reads will be in prad-0213-raw_M
## 7528326 sequences in the BAM file
## 7528326 sequences exported
## 7528326 reads; of these:
##   7528326 (100.00%) were unpaired; of these:
##     2145903 (28.50%) aligned 0 times
##     4208451 (55.90%) aligned exactly 1 time
##     1173972 (15.59%) aligned &gt;1 times
## 71.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1173972/5382423
## 2 849168/1173972
## 3 8300002/16864453
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0214-raw_1 and prad-0214-raw_2
## Single-end reads will be in prad-0214-raw_M
## 10038198 sequences in the BAM file
## 10038198 sequences exported
## 10038198 reads; of these:
##   10038198 (100.00%) were unpaired; of these:
##     2673755 (26.64%) aligned 0 times
##     5535820 (55.15%) aligned exactly 1 time
##     1828623 (18.22%) aligned &gt;1 times
## 73.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1828623/7364443
## 2 1295404/1828623
## 3 8477774/20588610
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0215-raw_1 and prad-0215-raw_2
## Single-end reads will be in prad-0215-raw_M
## 9103421 sequences in the BAM file
## 9103421 sequences exported
## 9103421 reads; of these:
##   9103421 (100.00%) were unpaired; of these:
##     1833328 (20.14%) aligned 0 times
##     5184583 (56.95%) aligned exactly 1 time
##     2085510 (22.91%) aligned &gt;1 times
## 79.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2085510/7270093
## 2 1441447/2085510
## 3 10612775/23916464
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0216-raw_1 and prad-0216-raw_2
## Single-end reads will be in prad-0216-raw_M
## 9380782 sequences in the BAM file
## 9380782 sequences exported
## 9380782 reads; of these:
##   9380782 (100.00%) were unpaired; of these:
##     3702229 (39.47%) aligned 0 times
##     4365620 (46.54%) aligned exactly 1 time
##     1312933 (14.00%) aligned &gt;1 times
## 60.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1312933/5678553
## 2 933253/1312933
## 3 5274551/13983242
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0217-raw_1 and prad-0217-raw_2
## Single-end reads will be in prad-0217-raw_M
## 13896712 sequences in the BAM file
## 13896712 sequences exported
## 13896712 reads; of these:
##   13896712 (100.00%) were unpaired; of these:
##     4002783 (28.80%) aligned 0 times
##     8226448 (59.20%) aligned exactly 1 time
##     1667481 (12.00%) aligned &gt;1 times
## 71.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1667481/9893929
## 2 1092999/1667481
## 3 9160930/22880319
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0218-raw_1 and prad-0218-raw_2
## Single-end reads will be in prad-0218-raw_M
## 13035296 sequences in the BAM file
## 13035296 sequences exported
## 13035296 reads; of these:
##   13035296 (100.00%) were unpaired; of these:
##     2814767 (21.59%) aligned 0 times
##     7511348 (57.62%) aligned exactly 1 time
##     2709181 (20.78%) aligned &gt;1 times
## 78.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2709181/10220529
## 2 1935159/2709181
## 3 11992674/29426769
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0219-raw_1 and prad-0219-raw_2
## Single-end reads will be in prad-0219-raw_M
## 9294562 sequences in the BAM file
## 9294562 sequences exported
## 9294562 reads; of these:
##   9294562 (100.00%) were unpaired; of these:
##     3301857 (35.52%) aligned 0 times
##     3764362 (40.50%) aligned exactly 1 time
##     2228343 (23.97%) aligned &gt;1 times
## 64.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2228343/5992705
## 2 1568563/2228343
## 3 14330998/32623637
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0220-raw_1 and prad-0220-raw_2
## Single-end reads will be in prad-0220-raw_M
## 6690841 sequences in the BAM file
## 6690841 sequences exported
## 6690841 reads; of these:
##   6690841 (100.00%) were unpaired; of these:
##     3097649 (46.30%) aligned 0 times
##     2488796 (37.20%) aligned exactly 1 time
##     1104396 (16.51%) aligned &gt;1 times
## 53.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1104396/3593192
## 2 726869/1104396
## 3 4957354/12893728
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0221-raw_1 and prad-0221-raw_2
## Single-end reads will be in prad-0221-raw_M
## 6016892 sequences in the BAM file
## 6016892 sequences exported
## 6016892 reads; of these:
##   6016892 (100.00%) were unpaired; of these:
##     1722051 (28.62%) aligned 0 times
##     2831138 (47.05%) aligned exactly 1 time
##     1463703 (24.33%) aligned &gt;1 times
## 71.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1463703/4294841
## 2 992977/1463703
## 3 10452966/22561343
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0222-raw_1 and prad-0222-raw_2
## Single-end reads will be in prad-0222-raw_M
## 24536181 sequences in the BAM file
## 24536181 sequences exported
## 24536181 reads; of these:
##   24536181 (100.00%) were unpaired; of these:
##     5252334 (21.41%) aligned 0 times
##     15882628 (64.73%) aligned exactly 1 time
##     3401219 (13.86%) aligned &gt;1 times
## 78.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3401219/19283847
## 2 2415753/3401219
## 3 13547691/38824196
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0223-raw_1 and prad-0223-raw_2
## Single-end reads will be in prad-0223-raw_M
## 21996514 sequences in the BAM file
## 21996514 sequences exported
## 21996514 reads; of these:
##   21996514 (100.00%) were unpaired; of these:
##     9399311 (42.73%) aligned 0 times
##     9771545 (44.42%) aligned exactly 1 time
##     2825658 (12.85%) aligned &gt;1 times
## 57.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2825658/12597203
## 2 1934169/2825658
## 3 7889508/25347200
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0224-raw_1 and prad-0224-raw_2
## Single-end reads will be in prad-0224-raw_M
## 17587683 sequences in the BAM file
## 17587683 sequences exported
## 17587683 reads; of these:
##   17587683 (100.00%) were unpaired; of these:
##     4711681 (26.79%) aligned 0 times
##     9913345 (56.37%) aligned exactly 1 time
##     2962657 (16.85%) aligned &gt;1 times
## 73.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2962657/12876002
## 2 2158455/2962657
## 3 9882488/26896916
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0225-raw_1 and prad-0225-raw_2
## Single-end reads will be in prad-0225-raw_M
## 13476150 sequences in the BAM file
## 13476150 sequences exported
## 13476150 reads; of these:
##   13476150 (100.00%) were unpaired; of these:
##     4858982 (36.06%) aligned 0 times
##     6410606 (47.57%) aligned exactly 1 time
##     2206562 (16.37%) aligned &gt;1 times
## 63.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2206562/8617168
## 2 1579962/2206562
## 3 6034392/18732565
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0226-raw_1 and prad-0226-raw_2
## Single-end reads will be in prad-0226-raw_M
## 23003618 sequences in the BAM file
## 23003618 sequences exported
## 23003618 reads; of these:
##   23003618 (100.00%) were unpaired; of these:
##     6328550 (27.51%) aligned 0 times
##     13210743 (57.43%) aligned exactly 1 time
##     3464325 (15.06%) aligned &gt;1 times
## 72.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3464325/16675068
## 2 2496251/3464325
## 3 8650921/30263465
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0227-raw_1 and prad-0227-raw_2
## Single-end reads will be in prad-0227-raw_M
## 9619349 sequences in the BAM file
## 9619349 sequences exported
## 9619349 reads; of these:
##   9619349 (100.00%) were unpaired; of these:
##     3088453 (32.11%) aligned 0 times
##     5095055 (52.97%) aligned exactly 1 time
##     1435841 (14.93%) aligned &gt;1 times
## 67.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1435841/6530896
## 2 1001212/1435841
## 3 6567863/16237433
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0228-raw_1 and prad-0228-raw_2
## Single-end reads will be in prad-0228-raw_M
## 13129207 sequences in the BAM file
## 13129207 sequences exported
## 13129207 reads; of these:
##   13129207 (100.00%) were unpaired; of these:
##     4051713 (30.86%) aligned 0 times
##     7014908 (53.43%) aligned exactly 1 time
##     2062586 (15.71%) aligned &gt;1 times
## 69.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2062586/9077494
## 2 1439231/2062586
## 3 9437289/23229733
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0229-raw_1 and prad-0229-raw_2
## Single-end reads will be in prad-0229-raw_M
## 12190935 sequences in the BAM file
## 12190935 sequences exported
## 12190935 reads; of these:
##   12190935 (100.00%) were unpaired; of these:
##     3750881 (30.77%) aligned 0 times
##     6311496 (51.77%) aligned exactly 1 time
##     2128558 (17.46%) aligned &gt;1 times
## 69.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2128558/8440054
## 2 1516977/2128558
## 3 11989735/27999925
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0230-raw_1 and prad-0230-raw_2
## Single-end reads will be in prad-0230-raw_M
## 14866342 sequences in the BAM file
## 14866342 sequences exported
## 14866342 reads; of these:
##   14866342 (100.00%) were unpaired; of these:
##     5036409 (33.88%) aligned 0 times
##     7891440 (53.08%) aligned exactly 1 time
##     1938493 (13.04%) aligned &gt;1 times
## 66.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1938493/9829933
## 2 1324220/1938493
## 3 6403645/19703601
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0231-raw_1 and prad-0231-raw_2
## Single-end reads will be in prad-0231-raw_M
## 9146575 sequences in the BAM file
## 9146575 sequences exported
## 9146575 reads; of these:
##   9146575 (100.00%) were unpaired; of these:
##     1879112 (20.54%) aligned 0 times
##     5513725 (60.28%) aligned exactly 1 time
##     1753738 (19.17%) aligned &gt;1 times
## 79.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1753738/7267463
## 2 1225451/1753738
## 3 4826204/15323877
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0232-raw_1 and prad-0232-raw_2
## Single-end reads will be in prad-0232-raw_M
## 12332106 sequences in the BAM file
## 12332106 sequences exported
## 12332106 reads; of these:
##   12332106 (100.00%) were unpaired; of these:
##     3899297 (31.62%) aligned 0 times
##     6585395 (53.40%) aligned exactly 1 time
##     1847414 (14.98%) aligned &gt;1 times
## 68.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1847414/8432809
## 2 1348210/1847414
## 3 7874707/20767966
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0233-raw_1 and prad-0233-raw_2
## Single-end reads will be in prad-0233-raw_M
## 17811266 sequences in the BAM file
## 17811266 sequences exported
## 17811266 reads; of these:
##   17811266 (100.00%) were unpaired; of these:
##     6195073 (34.78%) aligned 0 times
##     8010664 (44.98%) aligned exactly 1 time
##     3605529 (20.24%) aligned &gt;1 times
## 65.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3605529/11616193
## 2 2524497/3605529
## 3 11197212/30577368
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0234-raw_1 and prad-0234-raw_2
## Single-end reads will be in prad-0234-raw_M
## 12393900 sequences in the BAM file
## 12393900 sequences exported
## 12393900 reads; of these:
##   12393900 (100.00%) were unpaired; of these:
##     3754574 (30.29%) aligned 0 times
##     6791630 (54.80%) aligned exactly 1 time
##     1847696 (14.91%) aligned &gt;1 times
## 69.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1847696/8639326
## 2 1291519/1847696
## 3 9900745/22686565
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0235-raw_1 and prad-0235-raw_2
## Single-end reads will be in prad-0235-raw_M
## 10945807 sequences in the BAM file
## 10945807 sequences exported
## 10945807 reads; of these:
##   10945807 (100.00%) were unpaired; of these:
##     3409471 (31.15%) aligned 0 times
##     5934488 (54.22%) aligned exactly 1 time
##     1601848 (14.63%) aligned &gt;1 times
## 68.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1601848/7536336
## 2 1124228/1601848
## 3 6555935/17377858
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0236-raw_1 and prad-0236-raw_2
## Single-end reads will be in prad-0236-raw_M
## 11218948 sequences in the BAM file
## 11218948 sequences exported
## 11218948 reads; of these:
##   11218948 (100.00%) were unpaired; of these:
##     3504594 (31.24%) aligned 0 times
##     6104786 (54.41%) aligned exactly 1 time
##     1609568 (14.35%) aligned &gt;1 times
## 68.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1609568/7714354
## 2 1123514/1609568
## 3 5896865/16500051
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0237-raw_1 and prad-0237-raw_2
## Single-end reads will be in prad-0237-raw_M
## 16104257 sequences in the BAM file
## 16104257 sequences exported
## 16104257 reads; of these:
##   16104257 (100.00%) were unpaired; of these:
##     5063227 (31.44%) aligned 0 times
##     8395952 (52.13%) aligned exactly 1 time
##     2645078 (16.42%) aligned &gt;1 times
## 68.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2645078/11041030
## 2 1822789/2645078
## 3 15252919/34527526
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0238-raw_1 and prad-0238-raw_2
## Single-end reads will be in prad-0238-raw_M
## 12556483 sequences in the BAM file
## 12556483 sequences exported
## 12556483 reads; of these:
##   12556483 (100.00%) were unpaired; of these:
##     3878684 (30.89%) aligned 0 times
##     6923806 (55.14%) aligned exactly 1 time
##     1753993 (13.97%) aligned &gt;1 times
## 69.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1753993/8677799
## 2 1268097/1753993
## 3 7236171/19163889
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0239-raw_1 and prad-0239-raw_2
## Single-end reads will be in prad-0239-raw_M
## 14265011 sequences in the BAM file
## 14265011 sequences exported
## 14265011 reads; of these:
##   14265011 (100.00%) were unpaired; of these:
##     3901410 (27.35%) aligned 0 times
##     8422331 (59.04%) aligned exactly 1 time
##     1941270 (13.61%) aligned &gt;1 times
## 72.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1941270/10363601
## 2 1324507/1941270
## 3 6698339/20955976
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0240-raw_1 and prad-0240-raw_2
## Single-end reads will be in prad-0240-raw_M
## 9615895 sequences in the BAM file
## 9615895 sequences exported
## 9615895 reads; of these:
##   9615895 (100.00%) were unpaired; of these:
##     2830822 (29.44%) aligned 0 times
##     4894038 (50.90%) aligned exactly 1 time
##     1891035 (19.67%) aligned &gt;1 times
## 70.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1891035/6785073
## 2 1317821/1891035
## 3 11808165/25262562
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0241-raw_1 and prad-0241-raw_2
## Single-end reads will be in prad-0241-raw_M
## 17558789 sequences in the BAM file
## 17558789 sequences exported
## 17558789 reads; of these:
##   17558789 (100.00%) were unpaired; of these:
##     4726259 (26.92%) aligned 0 times
##     9279406 (52.85%) aligned exactly 1 time
##     3553124 (20.24%) aligned &gt;1 times
## 73.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3553124/12832530
## 2 2570254/3553124
## 3 9478994/28589664
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0242-raw_1 and prad-0242-raw_2
## Single-end reads will be in prad-0242-raw_M
## 13075033 sequences in the BAM file
## 13075033 sequences exported
## 13075033 reads; of these:
##   13075033 (100.00%) were unpaired; of these:
##     3337834 (25.53%) aligned 0 times
##     6775153 (51.82%) aligned exactly 1 time
##     2962046 (22.65%) aligned &gt;1 times
## 74.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2962046/9737199
## 2 2188374/2962046
## 3 16876492/38597880
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0243-raw_1 and prad-0243-raw_2
## Single-end reads will be in prad-0243-raw_M
## 22425468 sequences in the BAM file
## 22425468 sequences exported
## 22425468 reads; of these:
##   22425468 (100.00%) were unpaired; of these:
##     7391097 (32.96%) aligned 0 times
##     12683957 (56.56%) aligned exactly 1 time
##     2350414 (10.48%) aligned &gt;1 times
## 67.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2350414/15034371
## 2 1540151/2350414
## 3 8857708/28134168
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0244-raw_1 and prad-0244-raw_2
## Single-end reads will be in prad-0244-raw_M
## 8477746 sequences in the BAM file
## 8477746 sequences exported
## 8477746 reads; of these:
##   8477746 (100.00%) were unpaired; of these:
##     4002682 (47.21%) aligned 0 times
##     3380621 (39.88%) aligned exactly 1 time
##     1094443 (12.91%) aligned &gt;1 times
## 52.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1094443/4475064
## 2 728424/1094443
## 3 3440573/10443485
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0245-raw_1 and prad-0245-raw_2
## Single-end reads will be in prad-0245-raw_M
## 12557377 sequences in the BAM file
## 12557377 sequences exported
## 12557377 reads; of these:
##   12557377 (100.00%) were unpaired; of these:
##     4284765 (34.12%) aligned 0 times
##     6188828 (49.28%) aligned exactly 1 time
##     2083784 (16.59%) aligned &gt;1 times
## 65.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2083784/8272612
## 2 1373606/2083784
## 3 6080003/18489131
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0246-raw_1 and prad-0246-raw_2
## Single-end reads will be in prad-0246-raw_M
## 5987183 sequences in the BAM file
## 5987183 sequences exported
## 5987183 reads; of these:
##   5987183 (100.00%) were unpaired; of these:
##     1945013 (32.49%) aligned 0 times
##     3171736 (52.98%) aligned exactly 1 time
##     870434 (14.54%) aligned &gt;1 times
## 67.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 870434/4042170
## 2 605615/870434
## 3 3929305/10097609
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0247-raw_1 and prad-0247-raw_2
## Single-end reads will be in prad-0247-raw_M
## 5100393 sequences in the BAM file
## 5100393 sequences exported
## 5100393 reads; of these:
##   5100393 (100.00%) were unpaired; of these:
##     1839025 (36.06%) aligned 0 times
##     2294320 (44.98%) aligned exactly 1 time
##     967048 (18.96%) aligned &gt;1 times
## 63.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 967048/3261368
## 2 699497/967048
## 3 5253385/11798665
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0248-raw_1 and prad-0248-raw_2
## Single-end reads will be in prad-0248-raw_M
## 11914501 sequences in the BAM file
## 11914501 sequences exported
## 11914501 reads; of these:
##   11914501 (100.00%) were unpaired; of these:
##     3521226 (29.55%) aligned 0 times
##     6066259 (50.91%) aligned exactly 1 time
##     2327016 (19.53%) aligned &gt;1 times
## 70.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2327016/8393275
## 2 1672146/2327016
## 3 10214704/25913704
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0249-raw_1 and prad-0249-raw_2
## Single-end reads will be in prad-0249-raw_M
## 14322435 sequences in the BAM file
## 14322435 sequences exported
## 14322435 reads; of these:
##   14322435 (100.00%) were unpaired; of these:
##     4568403 (31.90%) aligned 0 times
##     8065218 (56.31%) aligned exactly 1 time
##     1688814 (11.79%) aligned &gt;1 times
## 68.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1688814/9754032
## 2 1202328/1688814
## 3 10791923/23978194
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0250-raw_1 and prad-0250-raw_2
## Single-end reads will be in prad-0250-raw_M
## 17760159 sequences in the BAM file
## 17760159 sequences exported
## 17760159 reads; of these:
##   17760159 (100.00%) were unpaired; of these:
##     9624770 (54.19%) aligned 0 times
##     6158771 (34.68%) aligned exactly 1 time
##     1976618 (11.13%) aligned &gt;1 times
## 45.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1976618/8135389
## 2 1340930/1976618
## 3 6546817/18976820
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0251-raw_1 and prad-0251-raw_2
## Single-end reads will be in prad-0251-raw_M
## 24032814 sequences in the BAM file
## 24032814 sequences exported
## 24032814 reads; of these:
##   24032814 (100.00%) were unpaired; of these:
##     9107007 (37.89%) aligned 0 times
##     12764100 (53.11%) aligned exactly 1 time
##     2161707 (8.99%) aligned &gt;1 times
## 62.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2161707/14925807
## 2 1568606/2161707
## 3 7808601/26159348
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0252-raw_1 and prad-0252-raw_2
## Single-end reads will be in prad-0252-raw_M
## 13632643 sequences in the BAM file
## 13632643 sequences exported
## 13632643 reads; of these:
##   13632643 (100.00%) were unpaired; of these:
##     4399628 (32.27%) aligned 0 times
##     7553386 (55.41%) aligned exactly 1 time
##     1679629 (12.32%) aligned &gt;1 times
## 67.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1679629/9233015
## 2 1192580/1679629
## 3 5557839/17488199
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0253-raw_1 and prad-0253-raw_2
## Single-end reads will be in prad-0253-raw_M
## 11613176 sequences in the BAM file
## 11613176 sequences exported
## 11613176 reads; of these:
##   11613176 (100.00%) were unpaired; of these:
##     5248762 (45.20%) aligned 0 times
##     4638187 (39.94%) aligned exactly 1 time
##     1726227 (14.86%) aligned &gt;1 times
## 54.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1726227/6364414
## 2 1252829/1726227
## 3 4954495/14640828
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0254-raw_1 and prad-0254-raw_2
## Single-end reads will be in prad-0254-raw_M
## 11277485 sequences in the BAM file
## 11277485 sequences exported
## 11277485 reads; of these:
##   11277485 (100.00%) were unpaired; of these:
##     4013677 (35.59%) aligned 0 times
##     5749469 (50.98%) aligned exactly 1 time
##     1514339 (13.43%) aligned &gt;1 times
## 64.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1514339/7263808
## 2 982124/1514339
## 3 8069464/18801004
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0255-raw_1 and prad-0255-raw_2
## Single-end reads will be in prad-0255-raw_M
## 5471915 sequences in the BAM file
## 5471915 sequences exported
## 5471915 reads; of these:
##   5471915 (100.00%) were unpaired; of these:
##     1836189 (33.56%) aligned 0 times
##     2637632 (48.20%) aligned exactly 1 time
##     998094 (18.24%) aligned &gt;1 times
## 66.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 998094/3635726
## 2 673778/998094
## 3 2503735/7897385
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0256-raw_1 and prad-0256-raw_2
## Single-end reads will be in prad-0256-raw_M
## 9571332 sequences in the BAM file
## 9571332 sequences exported
## 9571332 reads; of these:
##   9571332 (100.00%) were unpaired; of these:
##     2813573 (29.40%) aligned 0 times
##     5226984 (54.61%) aligned exactly 1 time
##     1530775 (15.99%) aligned &gt;1 times
## 70.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1530775/6757759
## 2 1084846/1530775
## 3 6676099/16763558
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0257-raw_1 and prad-0257-raw_2
## Single-end reads will be in prad-0257-raw_M
## 9884992 sequences in the BAM file
## 9884992 sequences exported
## 9884992 reads; of these:
##   9884992 (100.00%) were unpaired; of these:
##     3299802 (33.38%) aligned 0 times
##     5107967 (51.67%) aligned exactly 1 time
##     1477223 (14.94%) aligned &gt;1 times
## 66.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1477223/6585190
## 2 1057872/1477223
## 3 5297652/14722407
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0258-raw_1 and prad-0258-raw_2
## Single-end reads will be in prad-0258-raw_M
## 11240816 sequences in the BAM file
## 11240816 sequences exported
## 11240816 reads; of these:
##   11240816 (100.00%) were unpaired; of these:
##     3733664 (33.22%) aligned 0 times
##     4436703 (39.47%) aligned exactly 1 time
##     3070449 (27.32%) aligned &gt;1 times
## 66.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3070449/7507152
## 2 2283701/3070449
## 3 22185908/47939079
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in prad-0259-raw_1 and prad-0259-raw_2
## Single-end reads will be in prad-0259-raw_M
## 11546613 sequences in the BAM file
## 11546613 sequences exported
## 11546613 reads; of these:
##   11546613 (100.00%) were unpaired; of these:
##     3539154 (30.65%) aligned 0 times
##     6212243 (53.80%) aligned exactly 1 time
##     1795216 (15.55%) aligned &gt;1 times
## 69.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1795216/8007459
## 2 1288517/1795216
## 3 10155654/24898779
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0260-raw_1 and prad-0260-raw_2
## Single-end reads will be in prad-0260-raw_M
## 10773889 sequences in the BAM file
## 10773889 sequences exported
## 10773889 reads; of these:
##   10773889 (100.00%) were unpaired; of these:
##     3595221 (33.37%) aligned 0 times
##     5754338 (53.41%) aligned exactly 1 time
##     1424330 (13.22%) aligned &gt;1 times
## 66.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1424330/7178668
## 2 959180/1424330
## 3 5890809/15689153
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0261-raw_1 and prad-0261-raw_2
## Single-end reads will be in prad-0261-raw_M
## 15150379 sequences in the BAM file
## 15150379 sequences exported
## 15150379 reads; of these:
##   15150379 (100.00%) were unpaired; of these:
##     4884049 (32.24%) aligned 0 times
##     7841471 (51.76%) aligned exactly 1 time
##     2424859 (16.01%) aligned &gt;1 times
## 67.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2424859/10266330
## 2 1672991/2424859
## 3 6486494/20964333
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0262-raw_1 and prad-0262-raw_2
## Single-end reads will be in prad-0262-raw_M
## 11480506 sequences in the BAM file
## 11480506 sequences exported
## 11480506 reads; of these:
##   11480506 (100.00%) were unpaired; of these:
##     3668331 (31.95%) aligned 0 times
##     5577185 (48.58%) aligned exactly 1 time
##     2234990 (19.47%) aligned &gt;1 times
## 68.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2234990/7812175
## 2 1504861/2234990
## 3 15081784/33105028
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0263-raw_1 and prad-0263-raw_2
## Single-end reads will be in prad-0263-raw_M
## 11217132 sequences in the BAM file
## 11217132 sequences exported
## 11217132 reads; of these:
##   11217132 (100.00%) were unpaired; of these:
##     3409886 (30.40%) aligned 0 times
##     5856746 (52.21%) aligned exactly 1 time
##     1950500 (17.39%) aligned &gt;1 times
## 69.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1950500/7807246
## 2 1393150/1950500
## 3 11737548/26637637
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0264-raw_1 and prad-0264-raw_2
## Single-end reads will be in prad-0264-raw_M
## 5772126 sequences in the BAM file
## 5772126 sequences exported
## 5772126 reads; of these:
##   5772126 (100.00%) were unpaired; of these:
##     1868961 (32.38%) aligned 0 times
##     2925181 (50.68%) aligned exactly 1 time
##     977984 (16.94%) aligned &gt;1 times
## 67.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 977984/3903165
## 2 699198/977984
## 3 2850888/8153993
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0265-raw_1 and prad-0265-raw_2
## Single-end reads will be in prad-0265-raw_M
## 16445356 sequences in the BAM file
## 16445356 sequences exported
## 16445356 reads; of these:
##   16445356 (100.00%) were unpaired; of these:
##     7373944 (44.84%) aligned 0 times
##     6940850 (42.21%) aligned exactly 1 time
##     2130562 (12.96%) aligned &gt;1 times
## 55.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2130562/9071412
## 2 1440610/2130562
## 3 6693469/20493152
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0266-raw_1 and prad-0266-raw_2
## Single-end reads will be in prad-0266-raw_M
## 10533726 sequences in the BAM file
## 10533726 sequences exported
## 10533726 reads; of these:
##   10533726 (100.00%) were unpaired; of these:
##     3186367 (30.25%) aligned 0 times
##     5482710 (52.05%) aligned exactly 1 time
##     1864649 (17.70%) aligned &gt;1 times
## 69.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1864649/7347359
## 2 1292424/1864649
## 3 9136386/21315632
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0267-raw_1 and prad-0267-raw_2
## Single-end reads will be in prad-0267-raw_M
## 20344488 sequences in the BAM file
## 20344488 sequences exported
## 20344488 reads; of these:
##   20344488 (100.00%) were unpaired; of these:
##     8103150 (39.83%) aligned 0 times
##     9634346 (47.36%) aligned exactly 1 time
##     2606992 (12.81%) aligned &gt;1 times
## 60.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2606992/12241338
## 2 1886319/2606992
## 3 11630701/31026241
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0268-raw_1 and prad-0268-raw_2
## Single-end reads will be in prad-0268-raw_M
## 9087271 sequences in the BAM file
## 9087271 sequences exported
## 9087271 reads; of these:
##   9087271 (100.00%) were unpaired; of these:
##     2746261 (30.22%) aligned 0 times
##     4724687 (51.99%) aligned exactly 1 time
##     1616323 (17.79%) aligned &gt;1 times
## 69.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1616323/6341010
## 2 1188102/1616323
## 3 7310836/16702752
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0269-raw_1 and prad-0269-raw_2
## Single-end reads will be in prad-0269-raw_M
## 10744087 sequences in the BAM file
## 10744087 sequences exported
## 10744087 reads; of these:
##   10744087 (100.00%) were unpaired; of these:
##     3300055 (30.72%) aligned 0 times
##     5952245 (55.40%) aligned exactly 1 time
##     1491787 (13.88%) aligned &gt;1 times
## 69.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1491787/7444032
## 2 1095426/1491787
## 3 6561585/17280559
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0270-raw_1 and prad-0270-raw_2
## Single-end reads will be in prad-0270-raw_M
## 18259531 sequences in the BAM file
## 18259531 sequences exported
## 18259531 reads; of these:
##   18259531 (100.00%) were unpaired; of these:
##     5315147 (29.11%) aligned 0 times
##     10830124 (59.31%) aligned exactly 1 time
##     2114260 (11.58%) aligned &gt;1 times
## 70.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2114260/12944384
## 2 1412018/2114260
## 3 7340780/23256011
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0271-raw_1 and prad-0271-raw_2
## Single-end reads will be in prad-0271-raw_M
## 22470875 sequences in the BAM file
## 22470875 sequences exported
## 22470875 reads; of these:
##   22470875 (100.00%) were unpaired; of these:
##     9291708 (41.35%) aligned 0 times
##     10130072 (45.08%) aligned exactly 1 time
##     3049095 (13.57%) aligned &gt;1 times
## 58.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3049095/13179167
## 2 2136107/3049095
## 3 11215016/32474318
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in prad-0272-raw_1 and prad-0272-raw_2
## Single-end reads will be in prad-0272-raw_M
## 32982252 sequences in the BAM file
## 32982252 sequences exported
## 32982252 reads; of these:
##   32982252 (100.00%) were unpaired; of these:
##     11196729 (33.95%) aligned 0 times
##     18308497 (55.51%) aligned exactly 1 time
##     3477026 (10.54%) aligned &gt;1 times
## 66.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3477026/21785523
## 2 2333096/3477026
## 3 8635022/35623097
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in prad-0273-raw_1 and prad-0273-raw_2
## Single-end reads will be in prad-0273-raw_M
## 9168077 sequences in the BAM file
## 9168077 sequences exported
## 9168077 reads; of these:
##   9168077 (100.00%) were unpaired; of these:
##     3473574 (37.89%) aligned 0 times
##     4484545 (48.91%) aligned exactly 1 time
##     1209958 (13.20%) aligned &gt;1 times
## 62.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1209958/5694503
## 2 840525/1209958
## 3 4056863/11725265
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0274-raw_1 and prad-0274-raw_2
## Single-end reads will be in prad-0274-raw_M
## 15786359 sequences in the BAM file
## 15786359 sequences exported
## 15786359 reads; of these:
##   15786359 (100.00%) were unpaired; of these:
##     5365985 (33.99%) aligned 0 times
##     8276617 (52.43%) aligned exactly 1 time
##     2143757 (13.58%) aligned &gt;1 times
## 66.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2143757/10420374
## 2 1560136/2143757
## 3 7539811/21952445
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0275-raw_1 and prad-0275-raw_2
## Single-end reads will be in prad-0275-raw_M
## 5103476 sequences in the BAM file
## 5103476 sequences exported
## 5103476 reads; of these:
##   5103476 (100.00%) were unpaired; of these:
##     1666015 (32.64%) aligned 0 times
##     2683569 (52.58%) aligned exactly 1 time
##     753892 (14.77%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 753892/3437461
## 2 506926/753892
## 3 3446098/8319522
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0276-raw_1 and prad-0276-raw_2
## Single-end reads will be in prad-0276-raw_M
## 14671539 sequences in the BAM file
## 14671539 sequences exported
## 14671539 reads; of these:
##   14671539 (100.00%) were unpaired; of these:
##     6663115 (45.42%) aligned 0 times
##     6170530 (42.06%) aligned exactly 1 time
##     1837894 (12.53%) aligned &gt;1 times
## 54.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1837894/8008424
## 2 1171058/1837894
## 3 4295723/15112441
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0277-raw_1 and prad-0277-raw_2
## Single-end reads will be in prad-0277-raw_M
## 17298679 sequences in the BAM file
## 17298679 sequences exported
## 17298679 reads; of these:
##   17298679 (100.00%) were unpaired; of these:
##     2724154 (15.75%) aligned 0 times
##     10820676 (62.55%) aligned exactly 1 time
##     3753849 (21.70%) aligned &gt;1 times
## 84.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3753849/14574525
## 2 2713980/3753849
## 3 17173970/43665175
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in prad-0278-raw_1 and prad-0278-raw_2
## Single-end reads will be in prad-0278-raw_M
## 6819210 sequences in the BAM file
## 6819210 sequences exported
## 6819210 reads; of these:
##   6819210 (100.00%) were unpaired; of these:
##     2301610 (33.75%) aligned 0 times
##     2949817 (43.26%) aligned exactly 1 time
##     1567783 (22.99%) aligned &gt;1 times
## 66.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1567783/4517600
## 2 1118341/1567783
## 3 11379708/23553487
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0279-raw_1 and prad-0279-raw_2
## Single-end reads will be in prad-0279-raw_M
## 16368128 sequences in the BAM file
## 16368128 sequences exported
## 16368128 reads; of these:
##   16368128 (100.00%) were unpaired; of these:
##     3116308 (19.04%) aligned 0 times
##     10115633 (61.80%) aligned exactly 1 time
##     3136187 (19.16%) aligned &gt;1 times
## 80.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3136187/13251820
## 2 2244649/3136187
## 3 12683469/32326936
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0280-raw_1 and prad-0280-raw_2
## Single-end reads will be in prad-0280-raw_M
## 9305545 sequences in the BAM file
## 9305545 sequences exported
## 9305545 reads; of these:
##   9305545 (100.00%) were unpaired; of these:
##     3367133 (36.18%) aligned 0 times
##     4867336 (52.31%) aligned exactly 1 time
##     1071076 (11.51%) aligned &gt;1 times
## 63.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1071076/5938412
## 2 694269/1071076
## 3 3963388/11884516
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0281-raw_1 and prad-0281-raw_2
## Single-end reads will be in prad-0281-raw_M
## 12955704 sequences in the BAM file
## 12955704 sequences exported
## 12955704 reads; of these:
##   12955704 (100.00%) were unpaired; of these:
##     2622281 (20.24%) aligned 0 times
##     6932079 (53.51%) aligned exactly 1 time
##     3401344 (26.25%) aligned &gt;1 times
## 79.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3401344/10333423
## 2 2418191/3401344
## 3 16646738/37635611
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0282-raw_1 and prad-0282-raw_2
## Single-end reads will be in prad-0282-raw_M
## 21819185 sequences in the BAM file
## 21819185 sequences exported
## 21819185 reads; of these:
##   21819185 (100.00%) were unpaired; of these:
##     6041877 (27.69%) aligned 0 times
##     12933698 (59.28%) aligned exactly 1 time
##     2843610 (13.03%) aligned &gt;1 times
## 72.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2843610/15777308
## 2 1908512/2843610
## 3 9428581/30377092
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0283-raw_1 and prad-0283-raw_2
## Single-end reads will be in prad-0283-raw_M
## 30837212 sequences in the BAM file
## 30837212 sequences exported
## 30837212 reads; of these:
##   30837212 (100.00%) were unpaired; of these:
##     8258687 (26.78%) aligned 0 times
##     17535020 (56.86%) aligned exactly 1 time
##     5043505 (16.36%) aligned &gt;1 times
## 73.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5043505/22578525
## 2 3531943/5043505
## 3 23113142/62157278
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 4.
## Output will be in prad-0284-raw_1 and prad-0284-raw_2
## Single-end reads will be in prad-0284-raw_M
## 23109471 sequences in the BAM file
## 23109471 sequences exported
## 23109471 reads; of these:
##   23109471 (100.00%) were unpaired; of these:
##     6034754 (26.11%) aligned 0 times
##     13137117 (56.85%) aligned exactly 1 time
##     3937600 (17.04%) aligned &gt;1 times
## 73.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3937600/17074717
## 2 2909551/3937600
## 3 15940280/40935357
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0285-raw_1 and prad-0285-raw_2
## Single-end reads will be in prad-0285-raw_M
## 40570254 sequences in the BAM file
## 40570254 sequences exported
## 40570254 reads; of these:
##   40570254 (100.00%) were unpaired; of these:
##     11317668 (27.90%) aligned 0 times
##     16606787 (40.93%) aligned exactly 1 time
##     12645799 (31.17%) aligned &gt;1 times
## 72.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 12645799/29252586
## 2 8498172/12645799
## 3 50866297/115525866
## [bam_sort_core] merging from 19 files...
## This looks like paired data from lane 2.
## Output will be in prad-0286-raw_1 and prad-0286-raw_2
## Single-end reads will be in prad-0286-raw_M
## 8586497 sequences in the BAM file
## 8586497 sequences exported
## 8586497 reads; of these:
##   8586497 (100.00%) were unpaired; of these:
##     2540538 (29.59%) aligned 0 times
##     4274799 (49.79%) aligned exactly 1 time
##     1771160 (20.63%) aligned &gt;1 times
## 70.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1771160/6045959
## 2 1227564/1771160
## 3 11689755/25123431
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0287-raw_1 and prad-0287-raw_2
## Single-end reads will be in prad-0287-raw_M
## 12642954 sequences in the BAM file
## 12642954 sequences exported
## 12642954 reads; of these:
##   12642954 (100.00%) were unpaired; of these:
##     2396878 (18.96%) aligned 0 times
##     7871937 (62.26%) aligned exactly 1 time
##     2374139 (18.78%) aligned &gt;1 times
## 81.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2374139/10246076
## 2 1751553/2374139
## 3 10463071/27215254
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0288-raw_1 and prad-0288-raw_2
## Single-end reads will be in prad-0288-raw_M
## 8764650 sequences in the BAM file
## 8764650 sequences exported
## 8764650 reads; of these:
##   8764650 (100.00%) were unpaired; of these:
##     3109935 (35.48%) aligned 0 times
##     4105641 (46.84%) aligned exactly 1 time
##     1549074 (17.67%) aligned &gt;1 times
## 64.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1549074/5654715
## 2 1091925/1549074
## 3 8156661/18223946
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0289-raw_1 and prad-0289-raw_2
## Single-end reads will be in prad-0289-raw_M
## 14269813 sequences in the BAM file
## 14269813 sequences exported
## 14269813 reads; of these:
##   14269813 (100.00%) were unpaired; of these:
##     6458885 (45.26%) aligned 0 times
##     5982618 (41.92%) aligned exactly 1 time
##     1828310 (12.81%) aligned &gt;1 times
## 54.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1828310/7810928
## 2 1313344/1828310
## 3 4217814/14620380
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0290-raw_1 and prad-0290-raw_2
## Single-end reads will be in prad-0290-raw_M
## 8224573 sequences in the BAM file
## 8224573 sequences exported
## 8224573 reads; of these:
##   8224573 (100.00%) were unpaired; of these:
##     2442797 (29.70%) aligned 0 times
##     4164552 (50.64%) aligned exactly 1 time
##     1617224 (19.66%) aligned &gt;1 times
## 70.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1617224/5781776
## 2 1163016/1617224
## 3 9539149/21034529
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0291-raw_1 and prad-0291-raw_2
## Single-end reads will be in prad-0291-raw_M
## 7488986 sequences in the BAM file
## 7488986 sequences exported
## 7488986 reads; of these:
##   7488986 (100.00%) were unpaired; of these:
##     3004875 (40.12%) aligned 0 times
##     3382431 (45.17%) aligned exactly 1 time
##     1101680 (14.71%) aligned &gt;1 times
## 59.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1101680/4484111
## 2 823356/1101680
## 3 3780038/10810244
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0292-raw_1 and prad-0292-raw_2
## Single-end reads will be in prad-0292-raw_M
## 11554355 sequences in the BAM file
## 11554355 sequences exported
## 11554355 reads; of these:
##   11554355 (100.00%) were unpaired; of these:
##     3962244 (34.29%) aligned 0 times
##     6291764 (54.45%) aligned exactly 1 time
##     1300347 (11.25%) aligned &gt;1 times
## 65.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1300347/7592111
## 2 898112/1300347
## 3 3747892/13734153
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0293-raw_1 and prad-0293-raw_2
## Single-end reads will be in prad-0293-raw_M
## 5721718 sequences in the BAM file
## 5721718 sequences exported
## 5721718 reads; of these:
##   5721718 (100.00%) were unpaired; of these:
##     2485019 (43.43%) aligned 0 times
##     2173319 (37.98%) aligned exactly 1 time
##     1063380 (18.58%) aligned &gt;1 times
## 56.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1063380/3236699
## 2 739104/1063380
## 3 6462746/14297344
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0294-raw_1 and prad-0294-raw_2
## Single-end reads will be in prad-0294-raw_M
## 45766993 sequences in the BAM file
## 45766993 sequences exported
## 45766993 reads; of these:
##   45766993 (100.00%) were unpaired; of these:
##     13101069 (28.63%) aligned 0 times
##     26221535 (57.29%) aligned exactly 1 time
##     6444389 (14.08%) aligned &gt;1 times
## 71.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6444389/32665924
## 2 4619628/6444389
## 3 23667809/66246488
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 7.
## Output will be in prad-0295-raw_1 and prad-0295-raw_2
## Single-end reads will be in prad-0295-raw_M
## 14845718 sequences in the BAM file
## 14845718 sequences exported
## 14845718 reads; of these:
##   14845718 (100.00%) were unpaired; of these:
##     2953204 (19.89%) aligned 0 times
##     9691393 (65.28%) aligned exactly 1 time
##     2201121 (14.83%) aligned &gt;1 times
## 80.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2201121/11892514
## 2 1599465/2201121
## 3 10072787/27258111
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0296-raw_1 and prad-0296-raw_2
## Single-end reads will be in prad-0296-raw_M
## 18533482 sequences in the BAM file
## 18533482 sequences exported
## 18533482 reads; of these:
##   18533482 (100.00%) were unpaired; of these:
##     4607329 (24.86%) aligned 0 times
##     12175138 (65.69%) aligned exactly 1 time
##     1751015 (9.45%) aligned &gt;1 times
## 75.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1751015/13926153
## 2 1113462/1751015
## 3 10034457/27220314
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0297-raw_1 and prad-0297-raw_2
## Single-end reads will be in prad-0297-raw_M
## 10771561 sequences in the BAM file
## 10771561 sequences exported
## 10771561 reads; of these:
##   10771561 (100.00%) were unpaired; of these:
##     3589209 (33.32%) aligned 0 times
##     5440541 (50.51%) aligned exactly 1 time
##     1741811 (16.17%) aligned &gt;1 times
## 66.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1741811/7182352
## 2 1205422/1741811
## 3 4098659/13719388
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0298-raw_1 and prad-0298-raw_2
## Single-end reads will be in prad-0298-raw_M
## 8019440 sequences in the BAM file
## 8019440 sequences exported
## 8019440 reads; of these:
##   8019440 (100.00%) were unpaired; of these:
##     3197660 (39.87%) aligned 0 times
##     3637937 (45.36%) aligned exactly 1 time
##     1183843 (14.76%) aligned &gt;1 times
## 60.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1183843/4821780
## 2 860176/1183843
## 3 4891264/13289119
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0299-raw_1 and prad-0299-raw_2
## Single-end reads will be in prad-0299-raw_M
## 11175893 sequences in the BAM file
## 11175893 sequences exported
## 11175893 reads; of these:
##   11175893 (100.00%) were unpaired; of these:
##     4412673 (39.48%) aligned 0 times
##     5296248 (47.39%) aligned exactly 1 time
##     1466972 (13.13%) aligned &gt;1 times
## 60.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1466972/6763220
## 2 1057885/1466972
## 3 4128230/13593007
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0300-raw_1 and prad-0300-raw_2
## Single-end reads will be in prad-0300-raw_M
## 9715136 sequences in the BAM file
## 9715136 sequences exported
## 9715136 reads; of these:
##   9715136 (100.00%) were unpaired; of these:
##     3573031 (36.78%) aligned 0 times
##     4471136 (46.02%) aligned exactly 1 time
##     1670969 (17.20%) aligned &gt;1 times
## 63.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1670969/6142105
## 2 1183504/1670969
## 3 6728459/17187282
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0301-raw_1 and prad-0301-raw_2
## Single-end reads will be in prad-0301-raw_M
## 3785615 sequences in the BAM file
## 3785615 sequences exported
## 3785615 reads; of these:
##   3785615 (100.00%) were unpaired; of these:
##     589639 (15.58%) aligned 0 times
##     2205590 (58.26%) aligned exactly 1 time
##     990386 (26.16%) aligned &gt;1 times
## 84.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 990386/3195976
## 2 750693/990386
## 3 2936392/7889724
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0302-raw_1 and prad-0302-raw_2
## Single-end reads will be in prad-0302-raw_M
## 23311448 sequences in the BAM file
## 23311448 sequences exported
## 23311448 reads; of these:
##   23311448 (100.00%) were unpaired; of these:
##     7657745 (32.85%) aligned 0 times
##     12526794 (53.74%) aligned exactly 1 time
##     3126909 (13.41%) aligned &gt;1 times
## 67.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3126909/15653703
## 2 2247266/3126909
## 3 12289698/36512792
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in prad-0303-raw_1 and prad-0303-raw_2
## Single-end reads will be in prad-0303-raw_M
## 15621516 sequences in the BAM file
## 15621516 sequences exported
## 15621516 reads; of these:
##   15621516 (100.00%) were unpaired; of these:
##     5484546 (35.11%) aligned 0 times
##     8693728 (55.65%) aligned exactly 1 time
##     1443242 (9.24%) aligned &gt;1 times
## 64.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1443242/10136970
## 2 1010306/1443242
## 3 4325730/17203979
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0304-raw_1 and prad-0304-raw_2
## Single-end reads will be in prad-0304-raw_M
## 18125537 sequences in the BAM file
## 18125537 sequences exported
## 18125537 reads; of these:
##   18125537 (100.00%) were unpaired; of these:
##     6303204 (34.78%) aligned 0 times
##     9506846 (52.45%) aligned exactly 1 time
##     2315487 (12.77%) aligned &gt;1 times
## 65.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2315487/11822333
## 2 1638173/2315487
## 3 6402133/22201450
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0305-raw_1 and prad-0305-raw_2
## Single-end reads will be in prad-0305-raw_M
## 7271599 sequences in the BAM file
## 7271599 sequences exported
## 7271599 reads; of these:
##   7271599 (100.00%) were unpaired; of these:
##     2774568 (38.16%) aligned 0 times
##     3510400 (48.28%) aligned exactly 1 time
##     986631 (13.57%) aligned &gt;1 times
## 61.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 986631/4497031
## 2 649754/986631
## 3 3070225/9638945
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0306-raw_1 and prad-0306-raw_2
## Single-end reads will be in prad-0306-raw_M
## 6355216 sequences in the BAM file
## 6355216 sequences exported
## 6355216 reads; of these:
##   6355216 (100.00%) were unpaired; of these:
##     2920090 (45.95%) aligned 0 times
##     2683686 (42.23%) aligned exactly 1 time
##     751440 (11.82%) aligned &gt;1 times
## 54.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 751440/3435126
## 2 500894/751440
## 3 2137343/7047417
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0307-raw_1 and prad-0307-raw_2
## Single-end reads will be in prad-0307-raw_M
## 17073072 sequences in the BAM file
## 17073072 sequences exported
## 17073072 reads; of these:
##   17073072 (100.00%) were unpaired; of these:
##     5862474 (34.34%) aligned 0 times
##     9121172 (53.42%) aligned exactly 1 time
##     2089426 (12.24%) aligned &gt;1 times
## 65.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2089426/11210598
## 2 1407085/2089426
## 3 5885570/20701169
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0308-raw_1 and prad-0308-raw_2
## Single-end reads will be in prad-0308-raw_M
## 12428504 sequences in the BAM file
## 12428504 sequences exported
## 12428504 reads; of these:
##   12428504 (100.00%) were unpaired; of these:
##     1588690 (12.78%) aligned 0 times
##     7571141 (60.92%) aligned exactly 1 time
##     3268673 (26.30%) aligned &gt;1 times
## 87.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3268673/10839814
## 2 2676532/3268673
## 3 18667899/41316060
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0309-raw_1 and prad-0309-raw_2
## Single-end reads will be in prad-0309-raw_M
## 17776859 sequences in the BAM file
## 17776859 sequences exported
## 17776859 reads; of these:
##   17776859 (100.00%) were unpaired; of these:
##     5142543 (28.93%) aligned 0 times
##     9863759 (55.49%) aligned exactly 1 time
##     2770557 (15.59%) aligned &gt;1 times
## 71.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2770557/12634316
## 2 1839853/2770557
## 3 6759352/23784078
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0310-raw_1 and prad-0310-raw_2
## Single-end reads will be in prad-0310-raw_M
## 11626434 sequences in the BAM file
## 11626434 sequences exported
## 11626434 reads; of these:
##   11626434 (100.00%) were unpaired; of these:
##     4554388 (39.17%) aligned 0 times
##     5347321 (45.99%) aligned exactly 1 time
##     1724725 (14.83%) aligned &gt;1 times
## 60.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1724725/7072046
## 2 1168396/1724725
## 3 10912695/24060614
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0311-raw_1 and prad-0311-raw_2
## Single-end reads will be in prad-0311-raw_M
## 14809842 sequences in the BAM file
## 14809842 sequences exported
## 14809842 reads; of these:
##   14809842 (100.00%) were unpaired; of these:
##     4044787 (27.31%) aligned 0 times
##     8104582 (54.72%) aligned exactly 1 time
##     2660473 (17.96%) aligned &gt;1 times
## 72.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2660473/10765055
## 2 1996137/2660473
## 3 9455112/24531484
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0312-raw_1 and prad-0312-raw_2
## Single-end reads will be in prad-0312-raw_M
## 18849899 sequences in the BAM file
## 18849899 sequences exported
## 18849899 reads; of these:
##   18849899 (100.00%) were unpaired; of these:
##     5676832 (30.12%) aligned 0 times
##     10606577 (56.27%) aligned exactly 1 time
##     2566490 (13.62%) aligned &gt;1 times
## 69.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2566490/13173067
## 2 1864146/2566490
## 3 9125327/26406812
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0313-raw_1 and prad-0313-raw_2
## Single-end reads will be in prad-0313-raw_M
## 21105409 sequences in the BAM file
## 21105409 sequences exported
## 21105409 reads; of these:
##   21105409 (100.00%) were unpaired; of these:
##     5408791 (25.63%) aligned 0 times
##     11715696 (55.51%) aligned exactly 1 time
##     3980922 (18.86%) aligned &gt;1 times
## 74.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3980922/15696618
## 2 2947451/3980922
## 3 19783161/47318812
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0314-raw_1 and prad-0314-raw_2
## Single-end reads will be in prad-0314-raw_M
## 14242003 sequences in the BAM file
## 14242003 sequences exported
## 14242003 reads; of these:
##   14242003 (100.00%) were unpaired; of these:
##     2676518 (18.79%) aligned 0 times
##     8655987 (60.78%) aligned exactly 1 time
##     2909498 (20.43%) aligned &gt;1 times
## 81.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2909498/11565485
## 2 2095306/2909498
## 3 13992502/41879100
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0315-raw_1 and prad-0315-raw_2
## Single-end reads will be in prad-0315-raw_M
## 8974812 sequences in the BAM file
## 8974812 sequences exported
## 8974812 reads; of these:
##   8974812 (100.00%) were unpaired; of these:
##     1728575 (19.26%) aligned 0 times
##     5430478 (60.51%) aligned exactly 1 time
##     1815759 (20.23%) aligned &gt;1 times
## 80.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1815759/7246237
## 2 1314590/1815759
## 3 6042767/17336828
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0316-raw_1 and prad-0316-raw_2
## Single-end reads will be in prad-0316-raw_M
## 9975730 sequences in the BAM file
## 9975730 sequences exported
## 9975730 reads; of these:
##   9975730 (100.00%) were unpaired; of these:
##     3042313 (30.50%) aligned 0 times
##     5433014 (54.46%) aligned exactly 1 time
##     1500403 (15.04%) aligned &gt;1 times
## 69.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1500403/6933417
## 2 989889/1500403
## 3 8333370/19611048
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0317-raw_1 and prad-0317-raw_2
## Single-end reads will be in prad-0317-raw_M
## 7875109 sequences in the BAM file
## 7875109 sequences exported
## 7875109 reads; of these:
##   7875109 (100.00%) were unpaired; of these:
##     1413923 (17.95%) aligned 0 times
##     4848339 (61.57%) aligned exactly 1 time
##     1612847 (20.48%) aligned &gt;1 times
## 82.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1612847/6461186
## 2 1176798/1612847
## 3 6951664/18247274
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0318-raw_1 and prad-0318-raw_2
## Single-end reads will be in prad-0318-raw_M
## 8733703 sequences in the BAM file
## 8733703 sequences exported
## 8733703 reads; of these:
##   8733703 (100.00%) were unpaired; of these:
##     3344997 (38.30%) aligned 0 times
##     3874418 (44.36%) aligned exactly 1 time
##     1514288 (17.34%) aligned &gt;1 times
## 61.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1514288/5388706
## 2 1099859/1514288
## 3 8155144/20037987
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0319-raw_1 and prad-0319-raw_2
## Single-end reads will be in prad-0319-raw_M
## 11826637 sequences in the BAM file
## 11826637 sequences exported
## 11826637 reads; of these:
##   11826637 (100.00%) were unpaired; of these:
##     5041630 (42.63%) aligned 0 times
##     5322000 (45.00%) aligned exactly 1 time
##     1463007 (12.37%) aligned &gt;1 times
## 57.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1463007/6785007
## 2 984173/1463007
## 3 4326656/14125328
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0320-raw_1 and prad-0320-raw_2
## Single-end reads will be in prad-0320-raw_M
## 9492907 sequences in the BAM file
## 9492907 sequences exported
## 9492907 reads; of these:
##   9492907 (100.00%) were unpaired; of these:
##     2728596 (28.74%) aligned 0 times
##     5278038 (55.60%) aligned exactly 1 time
##     1486273 (15.66%) aligned &gt;1 times
## 71.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1486273/6764311
## 2 1099604/1486273
## 3 5135266/15542976
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0321-raw_1 and prad-0321-raw_2
## Single-end reads will be in prad-0321-raw_M
## 10767156 sequences in the BAM file
## 10767156 sequences exported
## 10767156 reads; of these:
##   10767156 (100.00%) were unpaired; of these:
##     3377233 (31.37%) aligned 0 times
##     6022469 (55.93%) aligned exactly 1 time
##     1367454 (12.70%) aligned &gt;1 times
## 68.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1367454/7389923
## 2 924039/1367454
## 3 3997705/13964001
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0322-raw_1 and prad-0322-raw_2
## Single-end reads will be in prad-0322-raw_M
## 8735325 sequences in the BAM file
## 8735325 sequences exported
## 8735325 reads; of these:
##   8735325 (100.00%) were unpaired; of these:
##     3824107 (43.78%) aligned 0 times
##     3791770 (43.41%) aligned exactly 1 time
##     1119448 (12.82%) aligned &gt;1 times
## 56.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1119448/4911218
## 2 812797/1119448
## 3 3178237/10084847
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0323-raw_1 and prad-0323-raw_2
## Single-end reads will be in prad-0323-raw_M
## 8681476 sequences in the BAM file
## 8681476 sequences exported
## 8681476 reads; of these:
##   8681476 (100.00%) were unpaired; of these:
##     3060914 (35.26%) aligned 0 times
##     4450269 (51.26%) aligned exactly 1 time
##     1170293 (13.48%) aligned &gt;1 times
## 64.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1170293/5620562
## 2 839195/1170293
## 3 5102387/13377992
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0324-raw_1 and prad-0324-raw_2
## Single-end reads will be in prad-0324-raw_M
## 27904562 sequences in the BAM file
## 27904562 sequences exported
## 27904562 reads; of these:
##   27904562 (100.00%) were unpaired; of these:
##     5884111 (21.09%) aligned 0 times
##     16850668 (60.39%) aligned exactly 1 time
##     5169783 (18.53%) aligned &gt;1 times
## 78.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5169783/22020451
## 2 3747048/5169783
## 3 25515688/63374934
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in prad-0325-raw_1 and prad-0325-raw_2
## Single-end reads will be in prad-0325-raw_M
## 22907322 sequences in the BAM file
## 22907322 sequences exported
## 22907322 reads; of these:
##   22907322 (100.00%) were unpaired; of these:
##     5595546 (24.43%) aligned 0 times
##     14683880 (64.10%) aligned exactly 1 time
##     2627896 (11.47%) aligned &gt;1 times
## 75.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2627896/17311776
## 2 1795821/2627896
## 3 14191620/36319300
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0326-raw_1 and prad-0326-raw_2
## Single-end reads will be in prad-0326-raw_M
## 11537927 sequences in the BAM file
## 11537927 sequences exported
## 11537927 reads; of these:
##   11537927 (100.00%) were unpaired; of these:
##     3533578 (30.63%) aligned 0 times
##     6365665 (55.17%) aligned exactly 1 time
##     1638684 (14.20%) aligned &gt;1 times
## 69.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1638684/8004349
## 2 1071370/1638684
## 3 5827893/17202097
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0327-raw_1 and prad-0327-raw_2
## Single-end reads will be in prad-0327-raw_M
## 9155411 sequences in the BAM file
## 9155411 sequences exported
## 9155411 reads; of these:
##   9155411 (100.00%) were unpaired; of these:
##     1785962 (19.51%) aligned 0 times
##     5807870 (63.44%) aligned exactly 1 time
##     1561579 (17.06%) aligned &gt;1 times
## 80.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1561579/7369449
## 2 1135336/1561579
## 3 8782521/21044598
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0328-raw_1 and prad-0328-raw_2
## Single-end reads will be in prad-0328-raw_M
## 7228094 sequences in the BAM file
## 7228094 sequences exported
## 7228094 reads; of these:
##   7228094 (100.00%) were unpaired; of these:
##     1456407 (20.15%) aligned 0 times
##     4027624 (55.72%) aligned exactly 1 time
##     1744063 (24.13%) aligned &gt;1 times
## 79.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1744063/5771687
## 2 1269103/1744063
## 3 17709510/32635731
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0329-raw_1 and prad-0329-raw_2
## Single-end reads will be in prad-0329-raw_M
## 20039266 sequences in the BAM file
## 20039266 sequences exported
## 20039266 reads; of these:
##   20039266 (100.00%) were unpaired; of these:
##     7417757 (37.02%) aligned 0 times
##     9982453 (49.81%) aligned exactly 1 time
##     2639056 (13.17%) aligned &gt;1 times
## 62.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2639056/12621509
## 2 1988845/2639056
## 3 9623702/27036717
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0330-raw_1 and prad-0330-raw_2
## Single-end reads will be in prad-0330-raw_M
## 11498432 sequences in the BAM file
## 11498432 sequences exported
## 11498432 reads; of these:
##   11498432 (100.00%) were unpaired; of these:
##     3913324 (34.03%) aligned 0 times
##     5910721 (51.40%) aligned exactly 1 time
##     1674387 (14.56%) aligned &gt;1 times
## 65.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1674387/7585108
## 2 1201412/1674387
## 3 4506257/14881776
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0331-raw_1 and prad-0331-raw_2
## Single-end reads will be in prad-0331-raw_M
## 11881616 sequences in the BAM file
## 11881616 sequences exported
## 11881616 reads; of these:
##   11881616 (100.00%) were unpaired; of these:
##     4556259 (38.35%) aligned 0 times
##     6001301 (50.51%) aligned exactly 1 time
##     1324056 (11.14%) aligned &gt;1 times
## 61.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1324056/7325357
## 2 920675/1324056
## 3 4065497/14024085
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0332-raw_1 and prad-0332-raw_2
## Single-end reads will be in prad-0332-raw_M
## 16045844 sequences in the BAM file
## 16045844 sequences exported
## 16045844 reads; of these:
##   16045844 (100.00%) were unpaired; of these:
##     6342848 (39.53%) aligned 0 times
##     7703653 (48.01%) aligned exactly 1 time
##     1999343 (12.46%) aligned &gt;1 times
## 60.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1999343/9702996
## 2 1440547/1999343
## 3 4896357/17504163
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0333-raw_1 and prad-0333-raw_2
## Single-end reads will be in prad-0333-raw_M
## 13461221 sequences in the BAM file
## 13461221 sequences exported
## 13461221 reads; of these:
##   13461221 (100.00%) were unpaired; of these:
##     3652212 (27.13%) aligned 0 times
##     7714496 (57.31%) aligned exactly 1 time
##     2094513 (15.56%) aligned &gt;1 times
## 72.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2094513/9809009
## 2 1523125/2094513
## 3 5792025/19262166
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0334-raw_1 and prad-0334-raw_2
## Single-end reads will be in prad-0334-raw_M
## 8842904 sequences in the BAM file
## 8842904 sequences exported
## 8842904 reads; of these:
##   8842904 (100.00%) were unpaired; of these:
##     3529680 (39.92%) aligned 0 times
##     3994189 (45.17%) aligned exactly 1 time
##     1319035 (14.92%) aligned &gt;1 times
## 60.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1319035/5313224
## 2 970084/1319035
## 3 3233587/10626731
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0335-raw_1 and prad-0335-raw_2
## Single-end reads will be in prad-0335-raw_M
## 10300563 sequences in the BAM file
## 10300563 sequences exported
## 10300563 reads; of these:
##   10300563 (100.00%) were unpaired; of these:
##     2806514 (27.25%) aligned 0 times
##     5810928 (56.41%) aligned exactly 1 time
##     1683121 (16.34%) aligned &gt;1 times
## 72.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1683121/7494049
## 2 1216459/1683121
## 3 4247615/14416589
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0336-raw_1 and prad-0336-raw_2
## Single-end reads will be in prad-0336-raw_M
## 8879528 sequences in the BAM file
## 8879528 sequences exported
## 8879528 reads; of these:
##   8879528 (100.00%) were unpaired; of these:
##     2985045 (33.62%) aligned 0 times
##     4460879 (50.24%) aligned exactly 1 time
##     1433604 (16.15%) aligned &gt;1 times
## 66.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1433604/5894483
## 2 1039744/1433604
## 3 4980291/13942077
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0337-raw_1 and prad-0337-raw_2
## Single-end reads will be in prad-0337-raw_M
## 33628070 sequences in the BAM file
## 33628070 sequences exported
## 33628070 reads; of these:
##   33628070 (100.00%) were unpaired; of these:
##     13856899 (41.21%) aligned 0 times
##     16471099 (48.98%) aligned exactly 1 time
##     3300072 (9.81%) aligned &gt;1 times
## 58.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3300072/19771171
## 2 2077770/3300072
## 3 7503591/32270796
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in prad-0338-raw_1 and prad-0338-raw_2
## Single-end reads will be in prad-0338-raw_M
## 13173108 sequences in the BAM file
## 13173108 sequences exported
## 13173108 reads; of these:
##   13173108 (100.00%) were unpaired; of these:
##     5110748 (38.80%) aligned 0 times
##     6433883 (48.84%) aligned exactly 1 time
##     1628477 (12.36%) aligned &gt;1 times
## 61.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1628477/8062360
## 2 1202145/1628477
## 3 4742480/15759371
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0339-raw_1 and prad-0339-raw_2
## Single-end reads will be in prad-0339-raw_M
## 9629655 sequences in the BAM file
## 9629655 sequences exported
## 9629655 reads; of these:
##   9629655 (100.00%) were unpaired; of these:
##     3174887 (32.97%) aligned 0 times
##     4668466 (48.48%) aligned exactly 1 time
##     1786302 (18.55%) aligned &gt;1 times
## 67.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1786302/6454768
## 2 1267887/1786302
## 3 9777000/22160080
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0340-raw_1 and prad-0340-raw_2
## Single-end reads will be in prad-0340-raw_M
## 6178833 sequences in the BAM file
## 6178833 sequences exported
## 6178833 reads; of these:
##   6178833 (100.00%) were unpaired; of these:
##     1722510 (27.88%) aligned 0 times
##     3514447 (56.88%) aligned exactly 1 time
##     941876 (15.24%) aligned &gt;1 times
## 72.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 941876/4456323
## 2 636128/941876
## 3 2681498/9036822
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0341-raw_1 and prad-0341-raw_2
## Single-end reads will be in prad-0341-raw_M
## 12182219 sequences in the BAM file
## 12182219 sequences exported
## 12182219 reads; of these:
##   12182219 (100.00%) were unpaired; of these:
##     4711072 (38.67%) aligned 0 times
##     5432483 (44.59%) aligned exactly 1 time
##     2038664 (16.73%) aligned &gt;1 times
## 61.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2038664/7471147
## 2 1360488/2038664
## 3 7319315/19499907
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0342-raw_1 and prad-0342-raw_2
## Single-end reads will be in prad-0342-raw_M
## 11501264 sequences in the BAM file
## 11501264 sequences exported
## 11501264 reads; of these:
##   11501264 (100.00%) were unpaired; of these:
##     4523557 (39.33%) aligned 0 times
##     5578298 (48.50%) aligned exactly 1 time
##     1399409 (12.17%) aligned &gt;1 times
## 60.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1399409/6977707
## 2 992966/1399409
## 3 3242596/12160777
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0343-raw_1 and prad-0343-raw_2
## Single-end reads will be in prad-0343-raw_M
## 6886665 sequences in the BAM file
## 6886665 sequences exported
## 6886665 reads; of these:
##   6886665 (100.00%) were unpaired; of these:
##     931415 (13.52%) aligned 0 times
##     3656093 (53.09%) aligned exactly 1 time
##     2299157 (33.39%) aligned &gt;1 times
## 86.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2299157/5955250
## 2 1519727/2299157
## 3 15739292/34888042
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0344-raw_1 and prad-0344-raw_2
## Single-end reads will be in prad-0344-raw_M
## 5479299 sequences in the BAM file
## 5479299 sequences exported
## 5479299 reads; of these:
##   5479299 (100.00%) were unpaired; of these:
##     1956806 (35.71%) aligned 0 times
##     2149315 (39.23%) aligned exactly 1 time
##     1373178 (25.06%) aligned &gt;1 times
## 64.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1373178/3522493
## 2 859756/1373178
## 3 7362193/17174186
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0345-raw_1 and prad-0345-raw_2
## Single-end reads will be in prad-0345-raw_M
## 9212070 sequences in the BAM file
## 9212070 sequences exported
## 9212070 reads; of these:
##   9212070 (100.00%) were unpaired; of these:
##     2973721 (32.28%) aligned 0 times
##     4991241 (54.18%) aligned exactly 1 time
##     1247108 (13.54%) aligned &gt;1 times
## 67.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1247108/6238349
## 2 851157/1247108
## 3 3945483/12797578
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0346-raw_1 and prad-0346-raw_2
## Single-end reads will be in prad-0346-raw_M
## 8129157 sequences in the BAM file
## 8129157 sequences exported
## 8129157 reads; of these:
##   8129157 (100.00%) were unpaired; of these:
##     3361699 (41.35%) aligned 0 times
##     3414233 (42.00%) aligned exactly 1 time
##     1353225 (16.65%) aligned &gt;1 times
## 58.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1353225/4767458
## 2 954861/1353225
## 3 3613466/10906841
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0347-raw_1 and prad-0347-raw_2
## Single-end reads will be in prad-0347-raw_M
## 18954905 sequences in the BAM file
## 18954905 sequences exported
## 18954905 reads; of these:
##   18954905 (100.00%) were unpaired; of these:
##     6999704 (36.93%) aligned 0 times
##     9411091 (49.65%) aligned exactly 1 time
##     2544110 (13.42%) aligned &gt;1 times
## 63.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2544110/11955201
## 2 1806410/2544110
## 3 5884832/21386370
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0348-raw_1 and prad-0348-raw_2
## Single-end reads will be in prad-0348-raw_M
## 11332336 sequences in the BAM file
## 11332336 sequences exported
## 11332336 reads; of these:
##   11332336 (100.00%) were unpaired; of these:
##     4416017 (38.97%) aligned 0 times
##     5331194 (47.04%) aligned exactly 1 time
##     1585125 (13.99%) aligned &gt;1 times
## 61.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1585125/6916319
## 2 1110271/1585125
## 3 6234225/17220234
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0349-raw_1 and prad-0349-raw_2
## Single-end reads will be in prad-0349-raw_M
## 10891178 sequences in the BAM file
## 10891178 sequences exported
## 10891178 reads; of these:
##   10891178 (100.00%) were unpaired; of these:
##     4011882 (36.84%) aligned 0 times
##     5396549 (49.55%) aligned exactly 1 time
##     1482747 (13.61%) aligned &gt;1 times
## 63.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1482747/6879296
## 2 971071/1482747
## 3 5047481/15750766
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0350-raw_1 and prad-0350-raw_2
## Single-end reads will be in prad-0350-raw_M
## 10099529 sequences in the BAM file
## 10099529 sequences exported
## 10099529 reads; of these:
##   10099529 (100.00%) were unpaired; of these:
##     3995563 (39.56%) aligned 0 times
##     4510248 (44.66%) aligned exactly 1 time
##     1593718 (15.78%) aligned &gt;1 times
## 60.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1593718/6103966
## 2 1215347/1593718
## 3 3650214/11793625
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0351-raw_1 and prad-0351-raw_2
## Single-end reads will be in prad-0351-raw_M
## 13145994 sequences in the BAM file
## 13145994 sequences exported
## 13145994 reads; of these:
##   13145994 (100.00%) were unpaired; of these:
##     4434204 (33.73%) aligned 0 times
##     6879814 (52.33%) aligned exactly 1 time
##     1831976 (13.94%) aligned &gt;1 times
## 66.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1831976/8711790
## 2 1267501/1831976
## 3 5790183/18470699
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0352-raw_1 and prad-0352-raw_2
## Single-end reads will be in prad-0352-raw_M
## 19167440 sequences in the BAM file
## 19167440 sequences exported
## 19167440 reads; of these:
##   19167440 (100.00%) were unpaired; of these:
##     6603507 (34.45%) aligned 0 times
##     10562330 (55.11%) aligned exactly 1 time
##     2001603 (10.44%) aligned &gt;1 times
## 65.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2001603/12563933
## 2 1407107/2001603
## 3 5379146/20822474
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0353-raw_1 and prad-0353-raw_2
## Single-end reads will be in prad-0353-raw_M
## 8779906 sequences in the BAM file
## 8779906 sequences exported
## 8779906 reads; of these:
##   8779906 (100.00%) were unpaired; of these:
##     3223508 (36.71%) aligned 0 times
##     4465638 (50.86%) aligned exactly 1 time
##     1090760 (12.42%) aligned &gt;1 times
## 63.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1090760/5556398
## 2 680405/1090760
## 3 3473998/11590987
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0354-raw_1 and prad-0354-raw_2
## Single-end reads will be in prad-0354-raw_M
## 6316482 sequences in the BAM file
## 6316482 sequences exported
## 6316482 reads; of these:
##   6316482 (100.00%) were unpaired; of these:
##     2496645 (39.53%) aligned 0 times
##     2881347 (45.62%) aligned exactly 1 time
##     938490 (14.86%) aligned &gt;1 times
## 60.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 938490/3819837
## 2 699663/938490
## 3 3162082/9084085
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0355-raw_1 and prad-0355-raw_2
## Single-end reads will be in prad-0355-raw_M
## 9777513 sequences in the BAM file
## 9777513 sequences exported
## 9777513 reads; of these:
##   9777513 (100.00%) were unpaired; of these:
##     1686080 (17.24%) aligned 0 times
##     6438667 (65.85%) aligned exactly 1 time
##     1652766 (16.90%) aligned &gt;1 times
## 82.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1652766/8091433
## 2 1232613/1652766
## 3 4798123/15779905
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0356-raw_1 and prad-0356-raw_2
## Single-end reads will be in prad-0356-raw_M
## 5689093 sequences in the BAM file
## 5689093 sequences exported
## 5689093 reads; of these:
##   5689093 (100.00%) were unpaired; of these:
##     2108111 (37.06%) aligned 0 times
##     2535316 (44.56%) aligned exactly 1 time
##     1045666 (18.38%) aligned &gt;1 times
## 62.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1045666/3580982
## 2 754108/1045666
## 3 4657105/11216078
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0357-raw_1 and prad-0357-raw_2
## Single-end reads will be in prad-0357-raw_M
## 17169008 sequences in the BAM file
## 17169008 sequences exported
## 17169008 reads; of these:
##   17169008 (100.00%) were unpaired; of these:
##     6902264 (40.20%) aligned 0 times
##     8324415 (48.49%) aligned exactly 1 time
##     1942329 (11.31%) aligned &gt;1 times
## 59.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1942329/10266744
## 2 1267603/1942329
## 3 5414827/19494429
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0358-raw_1 and prad-0358-raw_2
## Single-end reads will be in prad-0358-raw_M
## 6443059 sequences in the BAM file
## 6443059 sequences exported
## 6443059 reads; of these:
##   6443059 (100.00%) were unpaired; of these:
##     1788300 (27.76%) aligned 0 times
##     3692211 (57.31%) aligned exactly 1 time
##     962548 (14.94%) aligned &gt;1 times
## 72.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 962548/4654759
## 2 694899/962548
## 3 2912630/9467585
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0359-raw_1 and prad-0359-raw_2
## Single-end reads will be in prad-0359-raw_M
## 13085023 sequences in the BAM file
## 13085023 sequences exported
## 13085023 reads; of these:
##   13085023 (100.00%) were unpaired; of these:
##     5192590 (39.68%) aligned 0 times
##     6055374 (46.28%) aligned exactly 1 time
##     1837059 (14.04%) aligned &gt;1 times
## 60.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1837059/7892433
## 2 1307921/1837059
## 3 5958335/17833780
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0360-raw_1 and prad-0360-raw_2
## Single-end reads will be in prad-0360-raw_M
## 6957778 sequences in the BAM file
## 6957778 sequences exported
## 6957778 reads; of these:
##   6957778 (100.00%) were unpaired; of these:
##     1876192 (26.97%) aligned 0 times
##     3726251 (53.56%) aligned exactly 1 time
##     1355335 (19.48%) aligned &gt;1 times
## 73.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1355335/5081586
## 2 955669/1355335
## 3 3895415/11568471
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0361-raw_1 and prad-0361-raw_2
## Single-end reads will be in prad-0361-raw_M
## 8594657 sequences in the BAM file
## 8594657 sequences exported
## 8594657 reads; of these:
##   8594657 (100.00%) were unpaired; of these:
##     3754393 (43.68%) aligned 0 times
##     3756379 (43.71%) aligned exactly 1 time
##     1083885 (12.61%) aligned &gt;1 times
## 56.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1083885/4840264
## 2 730893/1083885
## 3 2700901/9274772
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0362-raw_1 and prad-0362-raw_2
## Single-end reads will be in prad-0362-raw_M
## 8584145 sequences in the BAM file
## 8584145 sequences exported
## 8584145 reads; of these:
##   8584145 (100.00%) were unpaired; of these:
##     3354671 (39.08%) aligned 0 times
##     4036732 (47.03%) aligned exactly 1 time
##     1192742 (13.89%) aligned &gt;1 times
## 60.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1192742/5229474
## 2 835427/1192742
## 3 2956247/10096923
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0363-raw_1 and prad-0363-raw_2
## Single-end reads will be in prad-0363-raw_M
## 10918976 sequences in the BAM file
## 10918976 sequences exported
## 10918976 reads; of these:
##   10918976 (100.00%) were unpaired; of these:
##     2890371 (26.47%) aligned 0 times
##     5940535 (54.41%) aligned exactly 1 time
##     2088070 (19.12%) aligned &gt;1 times
## 73.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2088070/8028605
## 2 1504124/2088070
## 3 5351336/16646253
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0364-raw_1 and prad-0364-raw_2
## Single-end reads will be in prad-0364-raw_M
## 8856446 sequences in the BAM file
## 8856446 sequences exported
## 8856446 reads; of these:
##   8856446 (100.00%) were unpaired; of these:
##     3473453 (39.22%) aligned 0 times
##     4043107 (45.65%) aligned exactly 1 time
##     1339886 (15.13%) aligned &gt;1 times
## 60.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1339886/5382993
## 2 903110/1339886
## 3 5481594/15010797
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0365-raw_1 and prad-0365-raw_2
## Single-end reads will be in prad-0365-raw_M
## 20392888 sequences in the BAM file
## 20392888 sequences exported
## 20392888 reads; of these:
##   20392888 (100.00%) were unpaired; of these:
##     7602227 (37.28%) aligned 0 times
##     10722869 (52.58%) aligned exactly 1 time
##     2067792 (10.14%) aligned &gt;1 times
## 62.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2067792/12790661
## 2 1501677/2067792
## 3 5143824/20899354
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0366-raw_1 and prad-0366-raw_2
## Single-end reads will be in prad-0366-raw_M
## 4710814 sequences in the BAM file
## 4710814 sequences exported
## 4710814 reads; of these:
##   4710814 (100.00%) were unpaired; of these:
##     1845645 (39.18%) aligned 0 times
##     1963208 (41.67%) aligned exactly 1 time
##     901961 (19.15%) aligned &gt;1 times
## 60.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 901961/2865169
## 2 670751/901961
## 3 2404914/6752871
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0367-raw_1 and prad-0367-raw_2
## Single-end reads will be in prad-0367-raw_M
## 9261621 sequences in the BAM file
## 9261621 sequences exported
## 9261621 reads; of these:
##   9261621 (100.00%) were unpaired; of these:
##     3169488 (34.22%) aligned 0 times
##     4740755 (51.19%) aligned exactly 1 time
##     1351378 (14.59%) aligned &gt;1 times
## 65.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1351378/6092133
## 2 963711/1351378
## 3 3604885/11993219
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0368-raw_1 and prad-0368-raw_2
## Single-end reads will be in prad-0368-raw_M
## 11424256 sequences in the BAM file
## 11424256 sequences exported
## 11424256 reads; of these:
##   11424256 (100.00%) were unpaired; of these:
##     3221938 (28.20%) aligned 0 times
##     6129337 (53.65%) aligned exactly 1 time
##     2072981 (18.15%) aligned &gt;1 times
## 71.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2072981/8202318
## 2 1496117/2072981
## 3 6973711/20516185
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0369-raw_1 and prad-0369-raw_2
## Single-end reads will be in prad-0369-raw_M
## 13315427 sequences in the BAM file
## 13315427 sequences exported
## 13315427 reads; of these:
##   13315427 (100.00%) were unpaired; of these:
##     4377554 (32.88%) aligned 0 times
##     7482770 (56.20%) aligned exactly 1 time
##     1455103 (10.93%) aligned &gt;1 times
## 67.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1455103/8937873
## 2 1006752/1455103
## 3 3860265/14886903
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0370-raw_1 and prad-0370-raw_2
## Single-end reads will be in prad-0370-raw_M
## 14536327 sequences in the BAM file
## 14536327 sequences exported
## 14536327 reads; of these:
##   14536327 (100.00%) were unpaired; of these:
##     5815075 (40.00%) aligned 0 times
##     6839902 (47.05%) aligned exactly 1 time
##     1881350 (12.94%) aligned &gt;1 times
## 60.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1881350/8721252
## 2 1307094/1881350
## 3 8647241/23856965
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0371-raw_1 and prad-0371-raw_2
## Single-end reads will be in prad-0371-raw_M
## 8110093 sequences in the BAM file
## 8110093 sequences exported
## 8110093 reads; of these:
##   8110093 (100.00%) were unpaired; of these:
##     2310105 (28.48%) aligned 0 times
##     4495657 (55.43%) aligned exactly 1 time
##     1304331 (16.08%) aligned &gt;1 times
## 71.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1304331/5799988
## 2 895352/1304331
## 3 3937271/12476776
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0372-raw_1 and prad-0372-raw_2
## Single-end reads will be in prad-0372-raw_M
## 13312008 sequences in the BAM file
## 13312008 sequences exported
## 13312008 reads; of these:
##   13312008 (100.00%) were unpaired; of these:
##     4536843 (34.08%) aligned 0 times
##     6566190 (49.33%) aligned exactly 1 time
##     2208975 (16.59%) aligned &gt;1 times
## 65.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2208975/8775165
## 2 1497531/2208975
## 3 9167161/24474228
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0373-raw_1 and prad-0373-raw_2
## Single-end reads will be in prad-0373-raw_M
## 13926904 sequences in the BAM file
## 13926904 sequences exported
## 13926904 reads; of these:
##   13926904 (100.00%) were unpaired; of these:
##     4773062 (34.27%) aligned 0 times
##     7393648 (53.09%) aligned exactly 1 time
##     1760194 (12.64%) aligned &gt;1 times
## 65.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1760194/9153842
## 2 1222438/1760194
## 3 4795623/16559123
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0374-raw_1 and prad-0374-raw_2
## Single-end reads will be in prad-0374-raw_M
## 5706316 sequences in the BAM file
## 5706316 sequences exported
## 5706316 reads; of these:
##   5706316 (100.00%) were unpaired; of these:
##     1872879 (32.82%) aligned 0 times
##     2584864 (45.30%) aligned exactly 1 time
##     1248573 (21.88%) aligned &gt;1 times
## 67.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1248573/3833437
## 2 924634/1248573
## 3 6134179/14708894
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0375-raw_1 and prad-0375-raw_2
## Single-end reads will be in prad-0375-raw_M
## 10501947 sequences in the BAM file
## 10501947 sequences exported
## 10501947 reads; of these:
##   10501947 (100.00%) were unpaired; of these:
##     4286367 (40.81%) aligned 0 times
##     5028969 (47.89%) aligned exactly 1 time
##     1186611 (11.30%) aligned &gt;1 times
## 59.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1186611/6215580
## 2 820077/1186611
## 3 3645577/12214863
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0376-raw_1 and prad-0376-raw_2
## Single-end reads will be in prad-0376-raw_M
## 14153750 sequences in the BAM file
## 14153750 sequences exported
## 14153750 reads; of these:
##   14153750 (100.00%) were unpaired; of these:
##     4554936 (32.18%) aligned 0 times
##     7711730 (54.49%) aligned exactly 1 time
##     1887084 (13.33%) aligned &gt;1 times
## 67.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1887084/9598814
## 2 1282622/1887084
## 3 4881938/17645833
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0377-raw_1 and prad-0377-raw_2
## Single-end reads will be in prad-0377-raw_M
## 6108467 sequences in the BAM file
## 6108467 sequences exported
## 6108467 reads; of these:
##   6108467 (100.00%) were unpaired; of these:
##     2101855 (34.41%) aligned 0 times
##     2472459 (40.48%) aligned exactly 1 time
##     1534153 (25.12%) aligned &gt;1 times
## 65.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1534153/4006612
## 2 1073775/1534153
## 3 9756045/21387086
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0378-raw_1 and prad-0378-raw_2
## Single-end reads will be in prad-0378-raw_M
## 17562650 sequences in the BAM file
## 17562650 sequences exported
## 17562650 reads; of these:
##   17562650 (100.00%) were unpaired; of these:
##     6365565 (36.24%) aligned 0 times
##     9064012 (51.61%) aligned exactly 1 time
##     2133073 (12.15%) aligned &gt;1 times
## 63.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2133073/11197085
## 2 1465189/2133073
## 3 5174443/19368898
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0379-raw_1 and prad-0379-raw_2
## Single-end reads will be in prad-0379-raw_M
## 15820200 sequences in the BAM file
## 15820200 sequences exported
## 15820200 reads; of these:
##   15820200 (100.00%) were unpaired; of these:
##     5982402 (37.81%) aligned 0 times
##     7749633 (48.99%) aligned exactly 1 time
##     2088165 (13.20%) aligned &gt;1 times
## 62.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2088165/9837798
## 2 1475538/2088165
## 3 5479261/18932444
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0380-raw_1 and prad-0380-raw_2
## Single-end reads will be in prad-0380-raw_M
## 9338772 sequences in the BAM file
## 9338772 sequences exported
## 9338772 reads; of these:
##   9338772 (100.00%) were unpaired; of these:
##     3432074 (36.75%) aligned 0 times
##     4334631 (46.42%) aligned exactly 1 time
##     1572067 (16.83%) aligned &gt;1 times
## 63.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1572067/5906698
## 2 1101615/1572067
## 3 7191731/18936705
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0381-raw_1 and prad-0381-raw_2
## Single-end reads will be in prad-0381-raw_M
## 14977487 sequences in the BAM file
## 14977487 sequences exported
## 14977487 reads; of these:
##   14977487 (100.00%) were unpaired; of these:
##     5583992 (37.28%) aligned 0 times
##     7462918 (49.83%) aligned exactly 1 time
##     1930577 (12.89%) aligned &gt;1 times
## 62.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1930577/9393495
## 2 1300245/1930577
## 3 4481522/16704350
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0382-raw_1 and prad-0382-raw_2
## Single-end reads will be in prad-0382-raw_M
## 14706035 sequences in the BAM file
## 14706035 sequences exported
## 14706035 reads; of these:
##   14706035 (100.00%) were unpaired; of these:
##     5950198 (40.46%) aligned 0 times
##     6247371 (42.48%) aligned exactly 1 time
##     2508466 (17.06%) aligned &gt;1 times
## 59.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2508466/8755837
## 2 1808486/2508466
## 3 6561808/19194951
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0383-raw_1 and prad-0383-raw_2
## Single-end reads will be in prad-0383-raw_M
## 7072269 sequences in the BAM file
## 7072269 sequences exported
## 7072269 reads; of these:
##   7072269 (100.00%) were unpaired; of these:
##     2366453 (33.46%) aligned 0 times
##     3399345 (48.07%) aligned exactly 1 time
##     1306471 (18.47%) aligned &gt;1 times
## 66.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1306471/4705816
## 2 898344/1306471
## 3 5292041/13589012
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0384-raw_1 and prad-0384-raw_2
## Single-end reads will be in prad-0384-raw_M
## 10256113 sequences in the BAM file
## 10256113 sequences exported
## 10256113 reads; of these:
##   10256113 (100.00%) were unpaired; of these:
##     3887771 (37.91%) aligned 0 times
##     4932792 (48.10%) aligned exactly 1 time
##     1435550 (14.00%) aligned &gt;1 times
## 62.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1435550/6368342
## 2 982224/1435550
## 3 3614750/12408195
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0385-raw_1 and prad-0385-raw_2
## Single-end reads will be in prad-0385-raw_M
## 12129013 sequences in the BAM file
## 12129013 sequences exported
## 12129013 reads; of these:
##   12129013 (100.00%) were unpaired; of these:
##     3434775 (28.32%) aligned 0 times
##     7057206 (58.18%) aligned exactly 1 time
##     1637032 (13.50%) aligned &gt;1 times
## 71.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1637032/8694238
## 2 1167888/1637032
## 3 3503156/14220236
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0386-raw_1 and prad-0386-raw_2
## Single-end reads will be in prad-0386-raw_M
## 8885963 sequences in the BAM file
## 8885963 sequences exported
## 8885963 reads; of these:
##   8885963 (100.00%) were unpaired; of these:
##     3172911 (35.71%) aligned 0 times
##     4430740 (49.86%) aligned exactly 1 time
##     1282312 (14.43%) aligned &gt;1 times
## 64.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1282312/5713052
## 2 902305/1282312
## 3 4114403/12499979
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0387-raw_1 and prad-0387-raw_2
## Single-end reads will be in prad-0387-raw_M
## 33593761 sequences in the BAM file
## 33593761 sequences exported
## 33593761 reads; of these:
##   33593761 (100.00%) were unpaired; of these:
##     11781353 (35.07%) aligned 0 times
##     17887293 (53.25%) aligned exactly 1 time
##     3925115 (11.68%) aligned &gt;1 times
## 64.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3925115/21812408
## 2 2706302/3925115
## 3 10303455/38707821
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0388-raw_1 and prad-0388-raw_2
## Single-end reads will be in prad-0388-raw_M
## 10670253 sequences in the BAM file
## 10670253 sequences exported
## 10670253 reads; of these:
##   10670253 (100.00%) were unpaired; of these:
##     3452385 (32.36%) aligned 0 times
##     5397734 (50.59%) aligned exactly 1 time
##     1820134 (17.06%) aligned &gt;1 times
## 67.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1820134/7217868
## 2 1301392/1820134
## 3 6848757/19123713
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0389-raw_1 and prad-0389-raw_2
## Single-end reads will be in prad-0389-raw_M
## 8993785 sequences in the BAM file
## 8993785 sequences exported
## 8993785 reads; of these:
##   8993785 (100.00%) were unpaired; of these:
##     3282929 (36.50%) aligned 0 times
##     4779602 (53.14%) aligned exactly 1 time
##     931254 (10.35%) aligned &gt;1 times
## 63.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 931254/5710856
## 2 659302/931254
## 3 3838278/12449355
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0390-raw_1 and prad-0390-raw_2
## Single-end reads will be in prad-0390-raw_M
## 11754864 sequences in the BAM file
## 11754864 sequences exported
## 11754864 reads; of these:
##   11754864 (100.00%) were unpaired; of these:
##     4917982 (41.84%) aligned 0 times
##     5284803 (44.96%) aligned exactly 1 time
##     1552079 (13.20%) aligned &gt;1 times
## 58.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1552079/6836882
## 2 1172806/1552079
## 3 3701651/12619674
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0391-raw_1 and prad-0391-raw_2
## Single-end reads will be in prad-0391-raw_M
## 5035328 sequences in the BAM file
## 5035328 sequences exported
## 5035328 reads; of these:
##   5035328 (100.00%) were unpaired; of these:
##     1777784 (35.31%) aligned 0 times
##     1996297 (39.65%) aligned exactly 1 time
##     1261247 (25.05%) aligned &gt;1 times
## 64.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1261247/3257544
## 2 930465/1261247
## 3 3947889/9823982
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0392-raw_1 and prad-0392-raw_2
## Single-end reads will be in prad-0392-raw_M
## 9839161 sequences in the BAM file
## 9839161 sequences exported
## 9839161 reads; of these:
##   9839161 (100.00%) were unpaired; of these:
##     3455958 (35.12%) aligned 0 times
##     4836714 (49.16%) aligned exactly 1 time
##     1546489 (15.72%) aligned &gt;1 times
## 64.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1546489/6383203
## 2 1120620/1546489
## 3 6393858/17121492
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0393-raw_1 and prad-0393-raw_2
## Single-end reads will be in prad-0393-raw_M
## 13740761 sequences in the BAM file
## 13740761 sequences exported
## 13740761 reads; of these:
##   13740761 (100.00%) were unpaired; of these:
##     4884735 (35.55%) aligned 0 times
##     7109995 (51.74%) aligned exactly 1 time
##     1746031 (12.71%) aligned &gt;1 times
## 64.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1746031/8856026
## 2 1218816/1746031
## 3 4852716/16581761
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0394-raw_1 and prad-0394-raw_2
## Single-end reads will be in prad-0394-raw_M
## 10742893 sequences in the BAM file
## 10742893 sequences exported
## 10742893 reads; of these:
##   10742893 (100.00%) were unpaired; of these:
##     4408638 (41.04%) aligned 0 times
##     5016369 (46.69%) aligned exactly 1 time
##     1317886 (12.27%) aligned &gt;1 times
## 58.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1317886/6334255
## 2 972752/1317886
## 3 3481334/11948809
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0395-raw_1 and prad-0395-raw_2
## Single-end reads will be in prad-0395-raw_M
## 7034754 sequences in the BAM file
## 7034754 sequences exported
## 7034754 reads; of these:
##   7034754 (100.00%) were unpaired; of these:
##     2832911 (40.27%) aligned 0 times
##     3163809 (44.97%) aligned exactly 1 time
##     1038034 (14.76%) aligned &gt;1 times
## 59.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1038034/4201843
## 2 708088/1038034
## 3 3185944/9582680
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0396-raw_1 and prad-0396-raw_2
## Single-end reads will be in prad-0396-raw_M
## 6894352 sequences in the BAM file
## 6894352 sequences exported
## 6894352 reads; of these:
##   6894352 (100.00%) were unpaired; of these:
##     2904608 (42.13%) aligned 0 times
##     3119765 (45.25%) aligned exactly 1 time
##     869979 (12.62%) aligned &gt;1 times
## 57.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 869979/3989744
## 2 594492/869979
## 3 2165421/7468897
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0397-raw_1 and prad-0397-raw_2
## Single-end reads will be in prad-0397-raw_M
## 11681756 sequences in the BAM file
## 11681756 sequences exported
## 11681756 reads; of these:
##   11681756 (100.00%) were unpaired; of these:
##     4371290 (37.42%) aligned 0 times
##     6048935 (51.78%) aligned exactly 1 time
##     1261531 (10.80%) aligned &gt;1 times
## 62.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1261531/7310466
## 2 899150/1261531
## 3 3512554/13143541
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0398-raw_1 and prad-0398-raw_2
## Single-end reads will be in prad-0398-raw_M
## 4464996 sequences in the BAM file
## 4464996 sequences exported
## 4464996 reads; of these:
##   4464996 (100.00%) were unpaired; of these:
##     1744938 (39.08%) aligned 0 times
##     2069942 (46.36%) aligned exactly 1 time
##     650116 (14.56%) aligned &gt;1 times
## 60.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 650116/2720058
## 2 491163/650116
## 3 1581108/5261399
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0399-raw_1 and prad-0399-raw_2
## Single-end reads will be in prad-0399-raw_M
## 11137252 sequences in the BAM file
## 11137252 sequences exported
## 11137252 reads; of these:
##   11137252 (100.00%) were unpaired; of these:
##     4020394 (36.10%) aligned 0 times
##     5927010 (53.22%) aligned exactly 1 time
##     1189848 (10.68%) aligned &gt;1 times
## 63.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1189848/7116858
## 2 800109/1189848
## 3 2737318/11625462
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0400-raw_1 and prad-0400-raw_2
## Single-end reads will be in prad-0400-raw_M
## 7733749 sequences in the BAM file
## 7733749 sequences exported
## 7733749 reads; of these:
##   7733749 (100.00%) were unpaired; of these:
##     2805278 (36.27%) aligned 0 times
##     3627476 (46.90%) aligned exactly 1 time
##     1300995 (16.82%) aligned &gt;1 times
## 63.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1300995/4928471
## 2 884007/1300995
## 3 3789772/11290845
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0401-raw_1 and prad-0401-raw_2
## Single-end reads will be in prad-0401-raw_M
## 9481599 sequences in the BAM file
## 9481599 sequences exported
## 9481599 reads; of these:
##   9481599 (100.00%) were unpaired; of these:
##     3698158 (39.00%) aligned 0 times
##     4535873 (47.84%) aligned exactly 1 time
##     1247568 (13.16%) aligned &gt;1 times
## 61.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1247568/5783441
## 2 850060/1247568
## 3 4049910/12701398
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0402-raw_1 and prad-0402-raw_2
## Single-end reads will be in prad-0402-raw_M
## 9679724 sequences in the BAM file
## 9679724 sequences exported
## 9679724 reads; of these:
##   9679724 (100.00%) were unpaired; of these:
##     2383192 (24.62%) aligned 0 times
##     5781441 (59.73%) aligned exactly 1 time
##     1515091 (15.65%) aligned &gt;1 times
## 75.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1515091/7296532
## 2 1094501/1515091
## 3 4869324/15393075
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0403-raw_1 and prad-0403-raw_2
## Single-end reads will be in prad-0403-raw_M
## 16118140 sequences in the BAM file
## 16118140 sequences exported
## 16118140 reads; of these:
##   16118140 (100.00%) were unpaired; of these:
##     5905597 (36.64%) aligned 0 times
##     8062854 (50.02%) aligned exactly 1 time
##     2149689 (13.34%) aligned &gt;1 times
## 63.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2149689/10212543
## 2 1528747/2149689
## 3 4836587/17780848
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0404-raw_1 and prad-0404-raw_2
## Single-end reads will be in prad-0404-raw_M
## 12672136 sequences in the BAM file
## 12672136 sequences exported
## 12672136 reads; of these:
##   12672136 (100.00%) were unpaired; of these:
##     4452084 (35.13%) aligned 0 times
##     6077096 (47.96%) aligned exactly 1 time
##     2142956 (16.91%) aligned &gt;1 times
## 64.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2142956/8220052
## 2 1457625/2142956
## 3 10743858/27185657
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0405-raw_1 and prad-0405-raw_2
## Single-end reads will be in prad-0405-raw_M
## 8080050 sequences in the BAM file
## 8080050 sequences exported
## 8080050 reads; of these:
##   8080050 (100.00%) were unpaired; of these:
##     2289989 (28.34%) aligned 0 times
##     4679586 (57.92%) aligned exactly 1 time
##     1110475 (13.74%) aligned &gt;1 times
## 71.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1110475/5790061
## 2 778821/1110475
## 3 3483680/11637372
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0406-raw_1 and prad-0406-raw_2
## Single-end reads will be in prad-0406-raw_M
## 8453742 sequences in the BAM file
## 8453742 sequences exported
## 8453742 reads; of these:
##   8453742 (100.00%) were unpaired; of these:
##     3093643 (36.59%) aligned 0 times
##     3998245 (47.30%) aligned exactly 1 time
##     1361854 (16.11%) aligned &gt;1 times
## 63.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1361854/5360099
## 2 1012681/1361854
## 3 3433926/10714817
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0407-raw_1 and prad-0407-raw_2
## Single-end reads will be in prad-0407-raw_M
## 16790965 sequences in the BAM file
## 16790965 sequences exported
## 16790965 reads; of these:
##   16790965 (100.00%) were unpaired; of these:
##     5557068 (33.10%) aligned 0 times
##     8740455 (52.05%) aligned exactly 1 time
##     2493442 (14.85%) aligned &gt;1 times
## 66.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2493442/11233897
## 2 1626819/2493442
## 3 9531256/26969556
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0408-raw_1 and prad-0408-raw_2
## Single-end reads will be in prad-0408-raw_M
## 17041754 sequences in the BAM file
## 17041754 sequences exported
## 17041754 reads; of these:
##   17041754 (100.00%) were unpaired; of these:
##     5445372 (31.95%) aligned 0 times
##     9121910 (53.53%) aligned exactly 1 time
##     2474472 (14.52%) aligned &gt;1 times
## 68.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2474472/11596382
## 2 1753241/2474472
## 3 5576198/20479054
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0409-raw_1 and prad-0409-raw_2
## Single-end reads will be in prad-0409-raw_M
## 8301850 sequences in the BAM file
## 8301850 sequences exported
## 8301850 reads; of these:
##   8301850 (100.00%) were unpaired; of these:
##     3161064 (38.08%) aligned 0 times
##     3856235 (46.45%) aligned exactly 1 time
##     1284551 (15.47%) aligned &gt;1 times
## 61.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1284551/5140786
## 2 950255/1284551
## 3 3406067/10560892
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0410-raw_1 and prad-0410-raw_2
## Single-end reads will be in prad-0410-raw_M
## 16250085 sequences in the BAM file
## 16250085 sequences exported
## 16250085 reads; of these:
##   16250085 (100.00%) were unpaired; of these:
##     4143766 (25.50%) aligned 0 times
##     9963335 (61.31%) aligned exactly 1 time
##     2142984 (13.19%) aligned &gt;1 times
## 74.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2142984/12106319
## 2 1420618/2142984
## 3 9598298/28993249
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0411-raw_1 and prad-0411-raw_2
## Single-end reads will be in prad-0411-raw_M
## 51075488 sequences in the BAM file
## 51075488 sequences exported
## 51075488 reads; of these:
##   51075488 (100.00%) were unpaired; of these:
##     14650983 (28.68%) aligned 0 times
##     28955122 (56.69%) aligned exactly 1 time
##     7469383 (14.62%) aligned &gt;1 times
## 71.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7469383/36424505
## 2 5268777/7469383
## 3 23015322/74741041
## [bam_sort_core] merging from 15 files...
## This looks like paired data from lane 1.
## Output will be in prad-0412-raw_1 and prad-0412-raw_2
## Single-end reads will be in prad-0412-raw_M
## 7625010 sequences in the BAM file
## 7625010 sequences exported
## 7625010 reads; of these:
##   7625010 (100.00%) were unpaired; of these:
##     1029827 (13.51%) aligned 0 times
##     4951514 (64.94%) aligned exactly 1 time
##     1643669 (21.56%) aligned &gt;1 times
## 86.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1643669/6595183
## 2 1196306/1643669
## 3 4827912/14622938
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0413-raw_1 and prad-0413-raw_2
## Single-end reads will be in prad-0413-raw_M
## 10090079 sequences in the BAM file
## 10090079 sequences exported
## 10090079 reads; of these:
##   10090079 (100.00%) were unpaired; of these:
##     2865353 (28.40%) aligned 0 times
##     5925461 (58.73%) aligned exactly 1 time
##     1299265 (12.88%) aligned &gt;1 times
## 71.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1299265/7224726
## 2 865317/1299265
## 3 3753667/13504576
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0414-raw_1 and prad-0414-raw_2
## Single-end reads will be in prad-0414-raw_M
## 9778859 sequences in the BAM file
## 9778859 sequences exported
## 9778859 reads; of these:
##   9778859 (100.00%) were unpaired; of these:
##     3538703 (36.19%) aligned 0 times
##     4877932 (49.88%) aligned exactly 1 time
##     1362224 (13.93%) aligned &gt;1 times
## 63.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1362224/6240156
## 2 885670/1362224
## 3 3677426/12522460
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0415-raw_1 and prad-0415-raw_2
## Single-end reads will be in prad-0415-raw_M
## 12039289 sequences in the BAM file
## 12039289 sequences exported
## 12039289 reads; of these:
##   12039289 (100.00%) were unpaired; of these:
##     4269567 (35.46%) aligned 0 times
##     6155100 (51.13%) aligned exactly 1 time
##     1614622 (13.41%) aligned &gt;1 times
## 64.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1614622/7769722
## 2 1155280/1614622
## 3 4577287/15452082
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0416-raw_1 and prad-0416-raw_2
## Single-end reads will be in prad-0416-raw_M
## 4227384 sequences in the BAM file
## 4227384 sequences exported
## 4227384 reads; of these:
##   4227384 (100.00%) were unpaired; of these:
##     1448841 (34.27%) aligned 0 times
##     2207436 (52.22%) aligned exactly 1 time
##     571107 (13.51%) aligned &gt;1 times
## 65.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 571107/2778543
## 2 410155/571107
## 3 2026529/6241314
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0417-raw_1 and prad-0417-raw_2
## Single-end reads will be in prad-0417-raw_M
## 8205648 sequences in the BAM file
## 8205648 sequences exported
## 8205648 reads; of these:
##   8205648 (100.00%) were unpaired; of these:
##     2053062 (25.02%) aligned 0 times
##     3189774 (38.87%) aligned exactly 1 time
##     2962812 (36.11%) aligned &gt;1 times
## 74.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2962812/6152586
## 2 1581235/2962812
## 3 20763238/45316119
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0418-raw_1 and prad-0418-raw_2
## Single-end reads will be in prad-0418-raw_M
## 16117022 sequences in the BAM file
## 16117022 sequences exported
## 16117022 reads; of these:
##   16117022 (100.00%) were unpaired; of these:
##     6383950 (39.61%) aligned 0 times
##     7825360 (48.55%) aligned exactly 1 time
##     1907712 (11.84%) aligned &gt;1 times
## 60.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1907712/9733072
## 2 1339236/1907712
## 3 5820646/19465004
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0419-raw_1 and prad-0419-raw_2
## Single-end reads will be in prad-0419-raw_M
## 15766453 sequences in the BAM file
## 15766453 sequences exported
## 15766453 reads; of these:
##   15766453 (100.00%) were unpaired; of these:
##     3865715 (24.52%) aligned 0 times
##     9729586 (61.71%) aligned exactly 1 time
##     2171152 (13.77%) aligned &gt;1 times
## 75.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2171152/11900738
## 2 1527500/2171152
## 3 6032288/21943375
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0420-raw_1 and prad-0420-raw_2
## Single-end reads will be in prad-0420-raw_M
## 11826745 sequences in the BAM file
## 11826745 sequences exported
## 11826745 reads; of these:
##   11826745 (100.00%) were unpaired; of these:
##     3528584 (29.84%) aligned 0 times
##     5581069 (47.19%) aligned exactly 1 time
##     2717092 (22.97%) aligned &gt;1 times
## 70.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2717092/8298161
## 2 1973425/2717092
## 3 24899374/54324654
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 4.
## Output will be in prad-0421-raw_1 and prad-0421-raw_2
## Single-end reads will be in prad-0421-raw_M
## 2673071 sequences in the BAM file
## 2673071 sequences exported
## 2673071 reads; of these:
##   2673071 (100.00%) were unpaired; of these:
##     733291 (27.43%) aligned 0 times
##     1082881 (40.51%) aligned exactly 1 time
##     856899 (32.06%) aligned &gt;1 times
## 72.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 856899/1939780
## 2 546269/856899
## 3 2312054/5968696
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0422-raw_1 and prad-0422-raw_2
## Single-end reads will be in prad-0422-raw_M
## 4515374 sequences in the BAM file
## 4515374 sequences exported
## 4515374 reads; of these:
##   4515374 (100.00%) were unpaired; of these:
##     1324880 (29.34%) aligned 0 times
##     2427346 (53.76%) aligned exactly 1 time
##     763148 (16.90%) aligned &gt;1 times
## 70.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 763148/3190494
## 2 592068/763148
## 3 2765789/7837031
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0423-raw_1 and prad-0423-raw_2
## Single-end reads will be in prad-0423-raw_M
## 10621974 sequences in the BAM file
## 10621974 sequences exported
## 10621974 reads; of these:
##   10621974 (100.00%) were unpaired; of these:
##     4041809 (38.05%) aligned 0 times
##     4612902 (43.43%) aligned exactly 1 time
##     1967263 (18.52%) aligned &gt;1 times
## 61.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1967263/6580165
## 2 1390770/1967263
## 3 7019272/18847215
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0424-raw_1 and prad-0424-raw_2
## Single-end reads will be in prad-0424-raw_M
## 5697401 sequences in the BAM file
## 5697401 sequences exported
## 5697401 reads; of these:
##   5697401 (100.00%) were unpaired; of these:
##     1832855 (32.17%) aligned 0 times
##     3099664 (54.40%) aligned exactly 1 time
##     764882 (13.43%) aligned &gt;1 times
## 67.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 764882/3864546
## 2 554620/764882
## 3 2136203/7371127
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0425-raw_1 and prad-0425-raw_2
## Single-end reads will be in prad-0425-raw_M
## 10480755 sequences in the BAM file
## 10480755 sequences exported
## 10480755 reads; of these:
##   10480755 (100.00%) were unpaired; of these:
##     3182813 (30.37%) aligned 0 times
##     5563074 (53.08%) aligned exactly 1 time
##     1734868 (16.55%) aligned &gt;1 times
## 69.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1734868/7297942
## 2 1134660/1734868
## 3 6315292/18494129
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0426-raw_1 and prad-0426-raw_2
## Single-end reads will be in prad-0426-raw_M
## 10615965 sequences in the BAM file
## 10615965 sequences exported
## 10615965 reads; of these:
##   10615965 (100.00%) were unpaired; of these:
##     3999293 (37.67%) aligned 0 times
##     5210975 (49.09%) aligned exactly 1 time
##     1405697 (13.24%) aligned &gt;1 times
## 62.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1405697/6616672
## 2 985080/1405697
## 3 4111658/13529935
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0427-raw_1 and prad-0427-raw_2
## Single-end reads will be in prad-0427-raw_M
## 44173311 sequences in the BAM file
## 44173311 sequences exported
## 44173311 reads; of these:
##   44173311 (100.00%) were unpaired; of these:
##     12096514 (27.38%) aligned 0 times
##     21742799 (49.22%) aligned exactly 1 time
##     10333998 (23.39%) aligned &gt;1 times
## 72.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 10333998/32076797
## 2 8444600/10333998
## 3 23340756/68909001
## [bam_sort_core] merging from 13 files...
## This looks like paired data from lane 1.
## Output will be in prad-0428-raw_1 and prad-0428-raw_2
## Single-end reads will be in prad-0428-raw_M
## 5326078 sequences in the BAM file
## 5326078 sequences exported
## 5326078 reads; of these:
##   5326078 (100.00%) were unpaired; of these:
##     691267 (12.98%) aligned 0 times
##     3218342 (60.43%) aligned exactly 1 time
##     1416469 (26.59%) aligned &gt;1 times
## 87.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1416469/4634811
## 2 1047630/1416469
## 3 5878407/14618569
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0429-raw_1 and prad-0429-raw_2
## Single-end reads will be in prad-0429-raw_M
## 18898481 sequences in the BAM file
## 18898481 sequences exported
## 18898481 reads; of these:
##   18898481 (100.00%) were unpaired; of these:
##     5806224 (30.72%) aligned 0 times
##     11274091 (59.66%) aligned exactly 1 time
##     1818166 (9.62%) aligned &gt;1 times
## 69.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1818166/13092257
## 2 1206021/1818166
## 3 4923404/21312379
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0430-raw_1 and prad-0430-raw_2
## Single-end reads will be in prad-0430-raw_M
## 4387179 sequences in the BAM file
## 4387179 sequences exported
## 4387179 reads; of these:
##   4387179 (100.00%) were unpaired; of these:
##     1123030 (25.60%) aligned 0 times
##     2327657 (53.06%) aligned exactly 1 time
##     936492 (21.35%) aligned &gt;1 times
## 74.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 936492/3264149
## 2 642824/936492
## 3 3600054/9494573
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0431-raw_1 and prad-0431-raw_2
## Single-end reads will be in prad-0431-raw_M
## 6837789 sequences in the BAM file
## 6837789 sequences exported
## 6837789 reads; of these:
##   6837789 (100.00%) were unpaired; of these:
##     1999769 (29.25%) aligned 0 times
##     3160162 (46.22%) aligned exactly 1 time
##     1677858 (24.54%) aligned &gt;1 times
## 70.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1677858/4838020
## 2 1192713/1677858
## 3 6068999/15114568
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0432-raw_1 and prad-0432-raw_2
## Single-end reads will be in prad-0432-raw_M
## 17641945 sequences in the BAM file
## 17641945 sequences exported
## 17641945 reads; of these:
##   17641945 (100.00%) were unpaired; of these:
##     4273929 (24.23%) aligned 0 times
##     10915066 (61.87%) aligned exactly 1 time
##     2452950 (13.90%) aligned &gt;1 times
## 75.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2452950/13368016
## 2 1704938/2452950
## 3 9260577/29380212
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in prad-0433-raw_1 and prad-0433-raw_2
## Single-end reads will be in prad-0433-raw_M
## 9739606 sequences in the BAM file
## 9739606 sequences exported
## 9739606 reads; of these:
##   9739606 (100.00%) were unpaired; of these:
##     2921518 (30.00%) aligned 0 times
##     5551731 (57.00%) aligned exactly 1 time
##     1266357 (13.00%) aligned &gt;1 times
## 70.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1266357/6818088
## 2 873066/1266357
## 3 3271731/12214977
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0434-raw_1 and prad-0434-raw_2
## Single-end reads will be in prad-0434-raw_M
## 8630333 sequences in the BAM file
## 8630333 sequences exported
## 8630333 reads; of these:
##   8630333 (100.00%) were unpaired; of these:
##     2698197 (31.26%) aligned 0 times
##     4423964 (51.26%) aligned exactly 1 time
##     1508172 (17.48%) aligned &gt;1 times
## 68.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1508172/5932136
## 2 1097651/1508172
## 3 4597086/13260437
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0435-raw_1 and prad-0435-raw_2
## Single-end reads will be in prad-0435-raw_M
## 6100773 sequences in the BAM file
## 6100773 sequences exported
## 6100773 reads; of these:
##   6100773 (100.00%) were unpaired; of these:
##     1870001 (30.65%) aligned 0 times
##     3053591 (50.05%) aligned exactly 1 time
##     1177181 (19.30%) aligned &gt;1 times
## 69.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1177181/4230772
## 2 854006/1177181
## 3 5675037/14114747
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0436-raw_1 and prad-0436-raw_2
## Single-end reads will be in prad-0436-raw_M
## 19156962 sequences in the BAM file
## 19156962 sequences exported
## 19156962 reads; of these:
##   19156962 (100.00%) were unpaired; of these:
##     5771791 (30.13%) aligned 0 times
##     11262227 (58.79%) aligned exactly 1 time
##     2122944 (11.08%) aligned &gt;1 times
## 69.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2122944/13385171
## 2 1369891/2122944
## 3 4992195/21599899
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0437-raw_1 and prad-0437-raw_2
## Single-end reads will be in prad-0437-raw_M
## 12803290 sequences in the BAM file
## 12803290 sequences exported
## 12803290 reads; of these:
##   12803290 (100.00%) were unpaired; of these:
##     3853947 (30.10%) aligned 0 times
##     7066439 (55.19%) aligned exactly 1 time
##     1882904 (14.71%) aligned &gt;1 times
## 69.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1882904/8949343
## 2 1231251/1882904
## 3 9011609/24953958
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0438-raw_1 and prad-0438-raw_2
## Single-end reads will be in prad-0438-raw_M
## 13400100 sequences in the BAM file
## 13400100 sequences exported
## 13400100 reads; of these:
##   13400100 (100.00%) were unpaired; of these:
##     3195067 (23.84%) aligned 0 times
##     8438866 (62.98%) aligned exactly 1 time
##     1766167 (13.18%) aligned &gt;1 times
## 76.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1766167/10205033
## 2 1201312/1766167
## 3 4337534/17347683
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0439-raw_1 and prad-0439-raw_2
## Single-end reads will be in prad-0439-raw_M
## 15612430 sequences in the BAM file
## 15612430 sequences exported
## 15612430 reads; of these:
##   15612430 (100.00%) were unpaired; of these:
##     6323905 (40.51%) aligned 0 times
##     7589334 (48.61%) aligned exactly 1 time
##     1699191 (10.88%) aligned &gt;1 times
## 59.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1699191/9288525
## 2 1129921/1699191
## 3 3784432/15428919
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0440-raw_1 and prad-0440-raw_2
## Single-end reads will be in prad-0440-raw_M
## 10922335 sequences in the BAM file
## 10922335 sequences exported
## 10922335 reads; of these:
##   10922335 (100.00%) were unpaired; of these:
##     2577370 (23.60%) aligned 0 times
##     6551424 (59.98%) aligned exactly 1 time
##     1793541 (16.42%) aligned &gt;1 times
## 76.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1793541/8344965
## 2 1298242/1793541
## 3 5444811/17361853
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0441-raw_1 and prad-0441-raw_2
## Single-end reads will be in prad-0441-raw_M
## 6587495 sequences in the BAM file
## 6587495 sequences exported
## 6587495 reads; of these:
##   6587495 (100.00%) were unpaired; of these:
##     2600632 (39.48%) aligned 0 times
##     3018807 (45.83%) aligned exactly 1 time
##     968056 (14.70%) aligned &gt;1 times
## 60.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 968056/3986863
## 2 643234/968056
## 3 3350704/9825586
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0442-raw_1 and prad-0442-raw_2
## Single-end reads will be in prad-0442-raw_M
## 6112218 sequences in the BAM file
## 6112218 sequences exported
## 6112218 reads; of these:
##   6112218 (100.00%) were unpaired; of these:
##     2462519 (40.29%) aligned 0 times
##     2649073 (43.34%) aligned exactly 1 time
##     1000626 (16.37%) aligned &gt;1 times
## 59.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1000626/3649699
## 2 706235/1000626
## 3 3001374/8558151
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0443-raw_1 and prad-0443-raw_2
## Single-end reads will be in prad-0443-raw_M
## 10463853 sequences in the BAM file
## 10463853 sequences exported
## 10463853 reads; of these:
##   10463853 (100.00%) were unpaired; of these:
##     4159804 (39.75%) aligned 0 times
##     4938866 (47.20%) aligned exactly 1 time
##     1365183 (13.05%) aligned &gt;1 times
## 60.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1365183/6304049
## 2 1007112/1365183
## 3 3367012/11786219
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0444-raw_1 and prad-0444-raw_2
## Single-end reads will be in prad-0444-raw_M
## 15484726 sequences in the BAM file
## 15484726 sequences exported
## 15484726 reads; of these:
##   15484726 (100.00%) were unpaired; of these:
##     5973864 (38.58%) aligned 0 times
##     7699083 (49.72%) aligned exactly 1 time
##     1811779 (11.70%) aligned &gt;1 times
## 61.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1811779/9510862
## 2 1257314/1811779
## 3 4665960/17058367
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0445-raw_1 and prad-0445-raw_2
## Single-end reads will be in prad-0445-raw_M
## 7688779 sequences in the BAM file
## 7688779 sequences exported
## 7688779 reads; of these:
##   7688779 (100.00%) were unpaired; of these:
##     2445468 (31.81%) aligned 0 times
##     4026224 (52.36%) aligned exactly 1 time
##     1217087 (15.83%) aligned &gt;1 times
## 68.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1217087/5243311
## 2 888953/1217087
## 3 4175675/12231372
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0446-raw_1 and prad-0446-raw_2
## Single-end reads will be in prad-0446-raw_M
## 10645210 sequences in the BAM file
## 10645210 sequences exported
## 10645210 reads; of these:
##   10645210 (100.00%) were unpaired; of these:
##     2432981 (22.86%) aligned 0 times
##     6504863 (61.11%) aligned exactly 1 time
##     1707366 (16.04%) aligned &gt;1 times
## 77.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1707366/8212229
## 2 1241926/1707366
## 3 4338762/15080504
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0447-raw_1 and prad-0447-raw_2
## Single-end reads will be in prad-0447-raw_M
## 4441054 sequences in the BAM file
## 4441054 sequences exported
## 4441054 reads; of these:
##   4441054 (100.00%) were unpaired; of these:
##     1450347 (32.66%) aligned 0 times
##     2359622 (53.13%) aligned exactly 1 time
##     631085 (14.21%) aligned &gt;1 times
## 67.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 631085/2990707
## 2 441084/631085
## 3 1990773/6309108
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0448-raw_1 and prad-0448-raw_2
## Single-end reads will be in prad-0448-raw_M
## 18706594 sequences in the BAM file
## 18706594 sequences exported
## 18706594 reads; of these:
##   18706594 (100.00%) were unpaired; of these:
##     6473403 (34.60%) aligned 0 times
##     10078619 (53.88%) aligned exactly 1 time
##     2154572 (11.52%) aligned &gt;1 times
## 65.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2154572/12233191
## 2 1495807/2154572
## 3 6745903/23028711
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0449-raw_1 and prad-0449-raw_2
## Single-end reads will be in prad-0449-raw_M
## 9736840 sequences in the BAM file
## 9736840 sequences exported
## 9736840 reads; of these:
##   9736840 (100.00%) were unpaired; of these:
##     3380588 (34.72%) aligned 0 times
##     5169977 (53.10%) aligned exactly 1 time
##     1186275 (12.18%) aligned &gt;1 times
## 65.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1186275/6356252
## 2 786654/1186275
## 3 3085759/11410759
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0450-raw_1 and prad-0450-raw_2
## Single-end reads will be in prad-0450-raw_M
## 8428262 sequences in the BAM file
## 8428262 sequences exported
## 8428262 reads; of these:
##   8428262 (100.00%) were unpaired; of these:
##     2114727 (25.09%) aligned 0 times
##     4961821 (58.87%) aligned exactly 1 time
##     1351714 (16.04%) aligned &gt;1 times
## 74.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1351714/6313535
## 2 1004863/1351714
## 3 4113558/13072830
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0451-raw_1 and prad-0451-raw_2
## Single-end reads will be in prad-0451-raw_M
## 9891076 sequences in the BAM file
## 9891076 sequences exported
## 9891076 reads; of these:
##   9891076 (100.00%) were unpaired; of these:
##     3126628 (31.61%) aligned 0 times
##     4768062 (48.21%) aligned exactly 1 time
##     1996386 (20.18%) aligned &gt;1 times
## 68.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1996386/6764448
## 2 1611226/1996386
## 3 4386472/13488024
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0452-raw_1 and prad-0452-raw_2
## Single-end reads will be in prad-0452-raw_M
## 16645140 sequences in the BAM file
## 16645140 sequences exported
## 16645140 reads; of these:
##   16645140 (100.00%) were unpaired; of these:
##     4917525 (29.54%) aligned 0 times
##     10034579 (60.29%) aligned exactly 1 time
##     1693036 (10.17%) aligned &gt;1 times
## 70.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1693036/11727615
## 2 1129255/1693036
## 3 4310159/18717792
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0453-raw_1 and prad-0453-raw_2
## Single-end reads will be in prad-0453-raw_M
## 23071145 sequences in the BAM file
## 23071145 sequences exported
## 23071145 reads; of these:
##   23071145 (100.00%) were unpaired; of these:
##     5298621 (22.97%) aligned 0 times
##     14981640 (64.94%) aligned exactly 1 time
##     2790884 (12.10%) aligned &gt;1 times
## 77.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2790884/17772524
## 2 1853537/2790884
## 3 6710381/28666052
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0454-raw_1 and prad-0454-raw_2
## Single-end reads will be in prad-0454-raw_M
## 11786020 sequences in the BAM file
## 11786020 sequences exported
## 11786020 reads; of these:
##   11786020 (100.00%) were unpaired; of these:
##     3882790 (32.94%) aligned 0 times
##     5658689 (48.01%) aligned exactly 1 time
##     2244541 (19.04%) aligned &gt;1 times
## 67.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2244541/7903230
## 2 1487952/2244541
## 3 9132242/24056499
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0455-raw_1 and prad-0455-raw_2
## Single-end reads will be in prad-0455-raw_M
## 11716585 sequences in the BAM file
## 11716585 sequences exported
## 11716585 reads; of these:
##   11716585 (100.00%) were unpaired; of these:
##     3967912 (33.87%) aligned 0 times
##     5738003 (48.97%) aligned exactly 1 time
##     2010670 (17.16%) aligned &gt;1 times
## 66.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2010670/7748673
## 2 1328302/2010670
## 3 13342157/32437668
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0456-raw_1 and prad-0456-raw_2
## Single-end reads will be in prad-0456-raw_M
## 12724492 sequences in the BAM file
## 12724492 sequences exported
## 12724492 reads; of these:
##   12724492 (100.00%) were unpaired; of these:
##     2978000 (23.40%) aligned 0 times
##     7992380 (62.81%) aligned exactly 1 time
##     1754112 (13.79%) aligned &gt;1 times
## 76.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1754112/9746492
## 2 1273614/1754112
## 3 4909498/17585860
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0457-raw_1 and prad-0457-raw_2
## Single-end reads will be in prad-0457-raw_M
## 8430637 sequences in the BAM file
## 8430637 sequences exported
## 8430637 reads; of these:
##   8430637 (100.00%) were unpaired; of these:
##     1809250 (21.46%) aligned 0 times
##     5390591 (63.94%) aligned exactly 1 time
##     1230796 (14.60%) aligned &gt;1 times
## 78.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1230796/6621387
## 2 854787/1230796
## 3 3822275/13033002
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0458-raw_1 and prad-0458-raw_2
## Single-end reads will be in prad-0458-raw_M
## 14297040 sequences in the BAM file
## 14297040 sequences exported
## 14297040 reads; of these:
##   14297040 (100.00%) were unpaired; of these:
##     4420137 (30.92%) aligned 0 times
##     8303249 (58.08%) aligned exactly 1 time
##     1573654 (11.01%) aligned &gt;1 times
## 69.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1573654/9876903
## 2 1066796/1573654
## 3 4190353/16757519
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0459-raw_1 and prad-0459-raw_2
## Single-end reads will be in prad-0459-raw_M
## 9423955 sequences in the BAM file
## 9423955 sequences exported
## 9423955 reads; of these:
##   9423955 (100.00%) were unpaired; of these:
##     2700205 (28.65%) aligned 0 times
##     5348880 (56.76%) aligned exactly 1 time
##     1374870 (14.59%) aligned &gt;1 times
## 71.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1374870/6723750
## 2 979282/1374870
## 3 4481646/14267235
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0460-raw_1 and prad-0460-raw_2
## Single-end reads will be in prad-0460-raw_M
## 7020070 sequences in the BAM file
## 7020070 sequences exported
## 7020070 reads; of these:
##   7020070 (100.00%) were unpaired; of these:
##     2582807 (36.79%) aligned 0 times
##     3223506 (45.92%) aligned exactly 1 time
##     1213757 (17.29%) aligned &gt;1 times
## 63.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1213757/4437263
## 2 864111/1213757
## 3 4527611/12176152
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0461-raw_1 and prad-0461-raw_2
## Single-end reads will be in prad-0461-raw_M
## 22877203 sequences in the BAM file
## 22877203 sequences exported
## 22877203 reads; of these:
##   22877203 (100.00%) were unpaired; of these:
##     6421295 (28.07%) aligned 0 times
##     14572176 (63.70%) aligned exactly 1 time
##     1883732 (8.23%) aligned &gt;1 times
## 71.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1883732/16455908
## 2 1269371/1883732
## 3 4714739/23944654
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0462-raw_1 and prad-0462-raw_2
## Single-end reads will be in prad-0462-raw_M
## 12001920 sequences in the BAM file
## 12001920 sequences exported
## 12001920 reads; of these:
##   12001920 (100.00%) were unpaired; of these:
##     2848405 (23.73%) aligned 0 times
##     7659402 (63.82%) aligned exactly 1 time
##     1494113 (12.45%) aligned &gt;1 times
## 76.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1494113/9153515
## 2 1046120/1494113
## 3 5570694/18770562
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0463-raw_1 and prad-0463-raw_2
## Single-end reads will be in prad-0463-raw_M
## 19202630 sequences in the BAM file
## 19202630 sequences exported
## 19202630 reads; of these:
##   19202630 (100.00%) were unpaired; of these:
##     6490964 (33.80%) aligned 0 times
##     10393045 (54.12%) aligned exactly 1 time
##     2318621 (12.07%) aligned &gt;1 times
## 66.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2318621/12711666
## 2 1694198/2318621
## 3 5566248/21423557
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0464-raw_1 and prad-0464-raw_2
## Single-end reads will be in prad-0464-raw_M
## 21633666 sequences in the BAM file
## 21633666 sequences exported
## 21633666 reads; of these:
##   21633666 (100.00%) were unpaired; of these:
##     2837005 (13.11%) aligned 0 times
##     15890692 (73.45%) aligned exactly 1 time
##     2905969 (13.43%) aligned &gt;1 times
## 86.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2905969/18796661
## 2 1987340/2905969
## 3 10754359/38110869
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in prad-0465-raw_1 and prad-0465-raw_2
## Single-end reads will be in prad-0465-raw_M
## 11554998 sequences in the BAM file
## 11554998 sequences exported
## 11554998 reads; of these:
##   11554998 (100.00%) were unpaired; of these:
##     3179536 (27.52%) aligned 0 times
##     6989106 (60.49%) aligned exactly 1 time
##     1386356 (12.00%) aligned &gt;1 times
## 72.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1386356/8375462
## 2 969821/1386356
## 3 3895346/14804751
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0466-raw_1 and prad-0466-raw_2
## Single-end reads will be in prad-0466-raw_M
## 13528697 sequences in the BAM file
## 13528697 sequences exported
## 13528697 reads; of these:
##   13528697 (100.00%) were unpaired; of these:
##     1921916 (14.21%) aligned 0 times
##     8889791 (65.71%) aligned exactly 1 time
##     2716990 (20.08%) aligned &gt;1 times
## 85.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2716990/11606781
## 2 1923139/2716990
## 3 10719612/29831712
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0467-raw_1 and prad-0467-raw_2
## Single-end reads will be in prad-0467-raw_M
## 6199546 sequences in the BAM file
## 6199546 sequences exported
## 6199546 reads; of these:
##   6199546 (100.00%) were unpaired; of these:
##     1471378 (23.73%) aligned 0 times
##     3600045 (58.07%) aligned exactly 1 time
##     1128123 (18.20%) aligned &gt;1 times
## 76.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1128123/4728168
## 2 867093/1128123
## 3 2979748/9558889
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0468-raw_1 and prad-0468-raw_2
## Single-end reads will be in prad-0468-raw_M
## 10363183 sequences in the BAM file
## 10363183 sequences exported
## 10363183 reads; of these:
##   10363183 (100.00%) were unpaired; of these:
##     3431467 (33.11%) aligned 0 times
##     5781012 (55.78%) aligned exactly 1 time
##     1150704 (11.10%) aligned &gt;1 times
## 66.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1150704/6931716
## 2 794708/1150704
## 3 3042281/11915515
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0469-raw_1 and prad-0469-raw_2
## Single-end reads will be in prad-0469-raw_M
## 10599813 sequences in the BAM file
## 10599813 sequences exported
## 10599813 reads; of these:
##   10599813 (100.00%) were unpaired; of these:
##     3253026 (30.69%) aligned 0 times
##     6092609 (57.48%) aligned exactly 1 time
##     1254178 (11.83%) aligned &gt;1 times
## 69.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1254178/7346787
## 2 882474/1254178
## 3 3363157/12870682
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0470-raw_1 and prad-0470-raw_2
## Single-end reads will be in prad-0470-raw_M
## 17902330 sequences in the BAM file
## 17902330 sequences exported
## 17902330 reads; of these:
##   17902330 (100.00%) were unpaired; of these:
##     4579168 (25.58%) aligned 0 times
##     10929891 (61.05%) aligned exactly 1 time
##     2393271 (13.37%) aligned &gt;1 times
## 74.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2393271/13323162
## 2 1773413/2393271
## 3 6736749/24235360
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0471-raw_1 and prad-0471-raw_2
## Single-end reads will be in prad-0471-raw_M
## 8432084 sequences in the BAM file
## 8432084 sequences exported
## 8432084 reads; of these:
##   8432084 (100.00%) were unpaired; of these:
##     1865023 (22.12%) aligned 0 times
##     5159731 (61.19%) aligned exactly 1 time
##     1407330 (16.69%) aligned &gt;1 times
## 77.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1407330/6567061
## 2 1016212/1407330
## 3 5187333/15328284
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0472-raw_1 and prad-0472-raw_2
## Single-end reads will be in prad-0472-raw_M
## 9335675 sequences in the BAM file
## 9335675 sequences exported
## 9335675 reads; of these:
##   9335675 (100.00%) were unpaired; of these:
##     3222131 (34.51%) aligned 0 times
##     4784988 (51.25%) aligned exactly 1 time
##     1328556 (14.23%) aligned &gt;1 times
## 65.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1328556/6113544
## 2 891095/1328556
## 3 4678351/14371761
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0473-raw_1 and prad-0473-raw_2
## Single-end reads will be in prad-0473-raw_M
## 7369639 sequences in the BAM file
## 7369639 sequences exported
## 7369639 reads; of these:
##   7369639 (100.00%) were unpaired; of these:
##     2392856 (32.47%) aligned 0 times
##     2945791 (39.97%) aligned exactly 1 time
##     2030992 (27.56%) aligned &gt;1 times
## 67.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2030992/4976783
## 2 1668271/2030992
## 3 6019058/14649640
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0474-raw_1 and prad-0474-raw_2
## Single-end reads will be in prad-0474-raw_M
## 13071959 sequences in the BAM file
## 13071959 sequences exported
## 13071959 reads; of these:
##   13071959 (100.00%) were unpaired; of these:
##     3533672 (27.03%) aligned 0 times
##     7534074 (57.64%) aligned exactly 1 time
##     2004213 (15.33%) aligned &gt;1 times
## 72.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2004213/9538287
## 2 1468147/2004213
## 3 6121554/19628677
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0475-raw_1 and prad-0475-raw_2
## Single-end reads will be in prad-0475-raw_M
## 5245868 sequences in the BAM file
## 5245868 sequences exported
## 5245868 reads; of these:
##   5245868 (100.00%) were unpaired; of these:
##     1632375 (31.12%) aligned 0 times
##     2756971 (52.56%) aligned exactly 1 time
##     856522 (16.33%) aligned &gt;1 times
## 68.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 856522/3613493
## 2 590835/856522
## 3 1907113/6693336
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0476-raw_1 and prad-0476-raw_2
## Single-end reads will be in prad-0476-raw_M
## 12507418 sequences in the BAM file
## 12507418 sequences exported
## 12507418 reads; of these:
##   12507418 (100.00%) were unpaired; of these:
##     3652306 (29.20%) aligned 0 times
##     7357755 (58.83%) aligned exactly 1 time
##     1497357 (11.97%) aligned &gt;1 times
## 70.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1497357/8855112
## 2 1026481/1497357
## 3 4657778/16739459
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0477-raw_1 and prad-0477-raw_2
## Single-end reads will be in prad-0477-raw_M
## 11681481 sequences in the BAM file
## 11681481 sequences exported
## 11681481 reads; of these:
##   11681481 (100.00%) were unpaired; of these:
##     2690482 (23.03%) aligned 0 times
##     7353503 (62.95%) aligned exactly 1 time
##     1637496 (14.02%) aligned &gt;1 times
## 76.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1637496/8990999
## 2 1186159/1637496
## 3 4065839/15435500
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0478-raw_1 and prad-0478-raw_2
## Single-end reads will be in prad-0478-raw_M
## 7787115 sequences in the BAM file
## 7787115 sequences exported
## 7787115 reads; of these:
##   7787115 (100.00%) were unpaired; of these:
##     1743620 (22.39%) aligned 0 times
##     4861220 (62.43%) aligned exactly 1 time
##     1182275 (15.18%) aligned &gt;1 times
## 77.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1182275/6043495
## 2 864270/1182275
## 3 3266637/11272501
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0479-raw_1 and prad-0479-raw_2
## Single-end reads will be in prad-0479-raw_M
## 16568443 sequences in the BAM file
## 16568443 sequences exported
## 16568443 reads; of these:
##   16568443 (100.00%) were unpaired; of these:
##     6076367 (36.67%) aligned 0 times
##     8335498 (50.31%) aligned exactly 1 time
##     2156578 (13.02%) aligned &gt;1 times
## 63.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2156578/10492076
## 2 1414251/2156578
## 3 5661492/19894956
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0480-raw_1 and prad-0480-raw_2
## Single-end reads will be in prad-0480-raw_M
## 12514109 sequences in the BAM file
## 12514109 sequences exported
## 12514109 reads; of these:
##   12514109 (100.00%) were unpaired; of these:
##     2907681 (23.24%) aligned 0 times
##     7955606 (63.57%) aligned exactly 1 time
##     1650822 (13.19%) aligned &gt;1 times
## 76.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1650822/9606428
## 2 1157623/1650822
## 3 5243432/18309599
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0481-raw_1 and prad-0481-raw_2
## Single-end reads will be in prad-0481-raw_M
## 26908231 sequences in the BAM file
## 26908231 sequences exported
## 26908231 reads; of these:
##   26908231 (100.00%) were unpaired; of these:
##     7544612 (28.04%) aligned 0 times
##     16580564 (61.62%) aligned exactly 1 time
##     2783055 (10.34%) aligned &gt;1 times
## 71.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2783055/19363619
## 2 1770091/2783055
## 3 9052848/34498518
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in prad-0482-raw_1 and prad-0482-raw_2
## Single-end reads will be in prad-0482-raw_M
## 17168741 sequences in the BAM file
## 17168741 sequences exported
## 17168741 reads; of these:
##   17168741 (100.00%) were unpaired; of these:
##     3618436 (21.08%) aligned 0 times
##     11413569 (66.48%) aligned exactly 1 time
##     2136736 (12.45%) aligned &gt;1 times
## 78.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2136736/13550305
## 2 1483235/2136736
## 3 5753449/22664763
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0483-raw_1 and prad-0483-raw_2
## Single-end reads will be in prad-0483-raw_M
## 8737749 sequences in the BAM file
## 8737749 sequences exported
## 8737749 reads; of these:
##   8737749 (100.00%) were unpaired; of these:
##     3297549 (37.74%) aligned 0 times
##     4343869 (49.71%) aligned exactly 1 time
##     1096331 (12.55%) aligned &gt;1 times
## 62.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1096331/5440200
## 2 665131/1096331
## 3 2411836/9495851
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0484-raw_1 and prad-0484-raw_2
## Single-end reads will be in prad-0484-raw_M
## 11552382 sequences in the BAM file
## 11552382 sequences exported
## 11552382 reads; of these:
##   11552382 (100.00%) were unpaired; of these:
##     4315604 (37.36%) aligned 0 times
##     5542145 (47.97%) aligned exactly 1 time
##     1694633 (14.67%) aligned &gt;1 times
## 62.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1694633/7236778
## 2 1206444/1694633
## 3 5423997/16230249
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0485-raw_1 and prad-0485-raw_2
## Single-end reads will be in prad-0485-raw_M
## 10297883 sequences in the BAM file
## 10297883 sequences exported
## 10297883 reads; of these:
##   10297883 (100.00%) were unpaired; of these:
##     3425677 (33.27%) aligned 0 times
##     5113721 (49.66%) aligned exactly 1 time
##     1758485 (17.08%) aligned &gt;1 times
## 66.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1758485/6872206
## 2 1246781/1758485
## 3 5506019/16098213
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0486-raw_1 and prad-0486-raw_2
## Single-end reads will be in prad-0486-raw_M
## 18594815 sequences in the BAM file
## 18594815 sequences exported
## 18594815 reads; of these:
##   18594815 (100.00%) were unpaired; of these:
##     5114261 (27.50%) aligned 0 times
##     10129560 (54.48%) aligned exactly 1 time
##     3350994 (18.02%) aligned &gt;1 times
## 72.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3350994/13480554
## 2 2466375/3350994
## 3 13446009/37472299
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in prad-0487-raw_1 and prad-0487-raw_2
## Single-end reads will be in prad-0487-raw_M
## 13817767 sequences in the BAM file
## 13817767 sequences exported
## 13817767 reads; of these:
##   13817767 (100.00%) were unpaired; of these:
##     3890743 (28.16%) aligned 0 times
##     8106558 (58.67%) aligned exactly 1 time
##     1820466 (13.17%) aligned &gt;1 times
## 71.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1820466/9927024
## 2 1273711/1820466
## 3 5788130/19320132
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0488-raw_1 and prad-0488-raw_2
## Single-end reads will be in prad-0488-raw_M
## 10512323 sequences in the BAM file
## 10512323 sequences exported
## 10512323 reads; of these:
##   10512323 (100.00%) were unpaired; of these:
##     3376723 (32.12%) aligned 0 times
##     4829585 (45.94%) aligned exactly 1 time
##     2306015 (21.94%) aligned &gt;1 times
## 67.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2306015/7135600
## 2 1654684/2306015
## 3 5679290/16494078
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0489-raw_1 and prad-0489-raw_2
## Single-end reads will be in prad-0489-raw_M
## 10873564 sequences in the BAM file
## 10873564 sequences exported
## 10873564 reads; of these:
##   10873564 (100.00%) were unpaired; of these:
##     3034055 (27.90%) aligned 0 times
##     6155022 (56.61%) aligned exactly 1 time
##     1684487 (15.49%) aligned &gt;1 times
## 72.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1684487/7839509
## 2 1240685/1684487
## 3 4825949/15485675
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0490-raw_1 and prad-0490-raw_2
## Single-end reads will be in prad-0490-raw_M
## 9837452 sequences in the BAM file
## 9837452 sequences exported
## 9837452 reads; of these:
##   9837452 (100.00%) were unpaired; of these:
##     2839406 (28.86%) aligned 0 times
##     4868535 (49.49%) aligned exactly 1 time
##     2129511 (21.65%) aligned &gt;1 times
## 71.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2129511/6998046
## 2 1487889/2129511
## 3 14923228/34829403
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0491-raw_1 and prad-0491-raw_2
## Single-end reads will be in prad-0491-raw_M
## 12457346 sequences in the BAM file
## 12457346 sequences exported
## 12457346 reads; of these:
##   12457346 (100.00%) were unpaired; of these:
##     3345437 (26.86%) aligned 0 times
##     7247798 (58.18%) aligned exactly 1 time
##     1864111 (14.96%) aligned &gt;1 times
## 73.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1864111/9111909
## 2 1263618/1864111
## 3 6913356/20894213
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0492-raw_1 and prad-0492-raw_2
## Single-end reads will be in prad-0492-raw_M
## 13971556 sequences in the BAM file
## 13971556 sequences exported
## 13971556 reads; of these:
##   13971556 (100.00%) were unpaired; of these:
##     3793842 (27.15%) aligned 0 times
##     8374782 (59.94%) aligned exactly 1 time
##     1802932 (12.90%) aligned &gt;1 times
## 72.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1802932/10177714
## 2 1231026/1802932
## 3 7253264/22707974
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0493-raw_1 and prad-0493-raw_2
## Single-end reads will be in prad-0493-raw_M
## 19959686 sequences in the BAM file
## 19959686 sequences exported
## 19959686 reads; of these:
##   19959686 (100.00%) were unpaired; of these:
##     5362288 (26.87%) aligned 0 times
##     12746414 (63.86%) aligned exactly 1 time
##     1850984 (9.27%) aligned &gt;1 times
## 73.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1850984/14597398
## 2 1164433/1850984
## 3 7709658/27816717
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0494-raw_1 and prad-0494-raw_2
## Single-end reads will be in prad-0494-raw_M
## 10557257 sequences in the BAM file
## 10557257 sequences exported
## 10557257 reads; of these:
##   10557257 (100.00%) were unpaired; of these:
##     3264350 (30.92%) aligned 0 times
##     5961764 (56.47%) aligned exactly 1 time
##     1331143 (12.61%) aligned &gt;1 times
## 69.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1331143/7292907
## 2 947902/1331143
## 3 4615362/15106801
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0495-raw_1 and prad-0495-raw_2
## Single-end reads will be in prad-0495-raw_M
## 7107780 sequences in the BAM file
## 7107780 sequences exported
## 7107780 reads; of these:
##   7107780 (100.00%) were unpaired; of these:
##     2577717 (36.27%) aligned 0 times
##     3352565 (47.17%) aligned exactly 1 time
##     1177498 (16.57%) aligned &gt;1 times
## 63.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1177498/4530063
## 2 885972/1177498
## 3 4480340/11985170
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0496-raw_1 and prad-0496-raw_2
## Single-end reads will be in prad-0496-raw_M
## 4146990 sequences in the BAM file
## 4146990 sequences exported
## 4146990 reads; of these:
##   4146990 (100.00%) were unpaired; of these:
##     1292943 (31.18%) aligned 0 times
##     2008121 (48.42%) aligned exactly 1 time
##     845926 (20.40%) aligned &gt;1 times
## 68.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 845926/2854047
## 2 634047/845926
## 3 3178763/8128693
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0497-raw_1 and prad-0497-raw_2
## Single-end reads will be in prad-0497-raw_M
## 6869668 sequences in the BAM file
## 6869668 sequences exported
## 6869668 reads; of these:
##   6869668 (100.00%) were unpaired; of these:
##     2138689 (31.13%) aligned 0 times
##     3454082 (50.28%) aligned exactly 1 time
##     1276897 (18.59%) aligned &gt;1 times
## 68.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1276897/4730979
## 2 909962/1276897
## 3 3694125/11070446
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0498-raw_1 and prad-0498-raw_2
## Single-end reads will be in prad-0498-raw_M
## 5713660 sequences in the BAM file
## 5713660 sequences exported
## 5713660 reads; of these:
##   5713660 (100.00%) were unpaired; of these:
##     1915648 (33.53%) aligned 0 times
##     2996448 (52.44%) aligned exactly 1 time
##     801564 (14.03%) aligned &gt;1 times
## 66.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 801564/3798012
## 2 553599/801564
## 3 2861918/8727637
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0499-raw_1 and prad-0499-raw_2
## Single-end reads will be in prad-0499-raw_M
## 9605216 sequences in the BAM file
## 9605216 sequences exported
## 9605216 reads; of these:
##   9605216 (100.00%) were unpaired; of these:
##     2874217 (29.92%) aligned 0 times
##     5292560 (55.10%) aligned exactly 1 time
##     1438439 (14.98%) aligned &gt;1 times
## 70.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1438439/6730999
## 2 1000954/1438439
## 3 4884016/15005225
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0500-raw_1 and prad-0500-raw_2
## Single-end reads will be in prad-0500-raw_M
## 12742434 sequences in the BAM file
## 12742434 sequences exported
## 12742434 reads; of these:
##   12742434 (100.00%) were unpaired; of these:
##     3585115 (28.14%) aligned 0 times
##     7399511 (58.07%) aligned exactly 1 time
##     1757808 (13.79%) aligned &gt;1 times
## 71.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1757808/9157319
## 2 1244038/1757808
## 3 4629544/16704122
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0501-raw_1 and prad-0501-raw_2
## Single-end reads will be in prad-0501-raw_M
## 8939192 sequences in the BAM file
## 8939192 sequences exported
## 8939192 reads; of these:
##   8939192 (100.00%) were unpaired; of these:
##     2532954 (28.34%) aligned 0 times
##     4771902 (53.38%) aligned exactly 1 time
##     1634336 (18.28%) aligned &gt;1 times
## 71.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1634336/6406238
## 2 1100459/1634336
## 3 10199709/24754301
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0502-raw_1 and prad-0502-raw_2
## Single-end reads will be in prad-0502-raw_M
## 25132195 sequences in the BAM file
## 25132195 sequences exported
## 25132195 reads; of these:
##   25132195 (100.00%) were unpaired; of these:
##     12236465 (48.69%) aligned 0 times
##     8872919 (35.30%) aligned exactly 1 time
##     4022811 (16.01%) aligned &gt;1 times
## 51.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4022811/12895730
## 2 2696865/4022811
## 3 23642539/56597844
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in prad-0503-raw_1 and prad-0503-raw_2
## Single-end reads will be in prad-0503-raw_M
## 5505949 sequences in the BAM file
## 5505949 sequences exported
## 5505949 reads; of these:
##   5505949 (100.00%) were unpaired; of these:
##     1757983 (31.93%) aligned 0 times
##     2522448 (45.81%) aligned exactly 1 time
##     1225518 (22.26%) aligned &gt;1 times
## 68.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1225518/3747966
## 2 917842/1225518
## 3 6306910/14998027
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0504-raw_1 and prad-0504-raw_2
## Single-end reads will be in prad-0504-raw_M
## 10977093 sequences in the BAM file
## 10977093 sequences exported
## 10977093 reads; of these:
##   10977093 (100.00%) were unpaired; of these:
##     3074000 (28.00%) aligned 0 times
##     6579677 (59.94%) aligned exactly 1 time
##     1323416 (12.06%) aligned &gt;1 times
## 72.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1323416/7903093
## 2 909677/1323416
## 3 5566134/17543573
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0505-raw_1 and prad-0505-raw_2
## Single-end reads will be in prad-0505-raw_M
## 6612027 sequences in the BAM file
## 6612027 sequences exported
## 6612027 reads; of these:
##   6612027 (100.00%) were unpaired; of these:
##     1829931 (27.68%) aligned 0 times
##     3174356 (48.01%) aligned exactly 1 time
##     1607740 (24.32%) aligned &gt;1 times
## 72.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1607740/4782096
## 2 1177429/1607740
## 3 10408245/23656873
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0506-raw_1 and prad-0506-raw_2
## Single-end reads will be in prad-0506-raw_M
## 7146416 sequences in the BAM file
## 7146416 sequences exported
## 7146416 reads; of these:
##   7146416 (100.00%) were unpaired; of these:
##     2445516 (34.22%) aligned 0 times
##     3568180 (49.93%) aligned exactly 1 time
##     1132720 (15.85%) aligned &gt;1 times
## 65.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1132720/4700900
## 2 798103/1132720
## 3 4731213/12812610
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0507-raw_1 and prad-0507-raw_2
## Single-end reads will be in prad-0507-raw_M
## 19268466 sequences in the BAM file
## 19268466 sequences exported
## 19268466 reads; of these:
##   19268466 (100.00%) were unpaired; of these:
##     5915479 (30.70%) aligned 0 times
##     11357594 (58.94%) aligned exactly 1 time
##     1995393 (10.36%) aligned &gt;1 times
## 69.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1995393/13352987
## 2 1363301/1995393
## 3 5095107/21435426
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0508-raw_1 and prad-0508-raw_2
## Single-end reads will be in prad-0508-raw_M
## 12061038 sequences in the BAM file
## 12061038 sequences exported
## 12061038 reads; of these:
##   12061038 (100.00%) were unpaired; of these:
##     3510262 (29.10%) aligned 0 times
##     6942992 (57.57%) aligned exactly 1 time
##     1607784 (13.33%) aligned &gt;1 times
## 70.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1607784/8550776
## 2 1109552/1607784
## 3 3769407/14642117
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0509-raw_1 and prad-0509-raw_2
## Single-end reads will be in prad-0509-raw_M
## 7101837 sequences in the BAM file
## 7101837 sequences exported
## 7101837 reads; of these:
##   7101837 (100.00%) were unpaired; of these:
##     2530857 (35.64%) aligned 0 times
##     3327796 (46.86%) aligned exactly 1 time
##     1243184 (17.51%) aligned &gt;1 times
## 64.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1243184/4570980
## 2 816982/1243184
## 3 4811129/12629814
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0510-raw_1 and prad-0510-raw_2
## Single-end reads will be in prad-0510-raw_M
## 9118555 sequences in the BAM file
## 9118555 sequences exported
## 9118555 reads; of these:
##   9118555 (100.00%) were unpaired; of these:
##     2414685 (26.48%) aligned 0 times
##     5150722 (56.49%) aligned exactly 1 time
##     1553148 (17.03%) aligned &gt;1 times
## 73.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1553148/6703870
## 2 1140995/1553148
## 3 4489496/13957633
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0511-raw_1 and prad-0511-raw_2
## Single-end reads will be in prad-0511-raw_M
## 14600839 sequences in the BAM file
## 14600839 sequences exported
## 14600839 reads; of these:
##   14600839 (100.00%) were unpaired; of these:
##     4435699 (30.38%) aligned 0 times
##     8033834 (55.02%) aligned exactly 1 time
##     2131306 (14.60%) aligned &gt;1 times
## 69.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2131306/10165140
## 2 1421920/2131306
## 3 8965920/25990775
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0512-raw_1 and prad-0512-raw_2
## Single-end reads will be in prad-0512-raw_M
## 10710643 sequences in the BAM file
## 10710643 sequences exported
## 10710643 reads; of these:
##   10710643 (100.00%) were unpaired; of these:
##     2958587 (27.62%) aligned 0 times
##     6432624 (60.06%) aligned exactly 1 time
##     1319432 (12.32%) aligned &gt;1 times
## 72.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1319432/7752056
## 2 892477/1319432
## 3 4057837/14375312
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0513-raw_1 and prad-0513-raw_2
## Single-end reads will be in prad-0513-raw_M
## 15968211 sequences in the BAM file
## 15968211 sequences exported
## 15968211 reads; of these:
##   15968211 (100.00%) were unpaired; of these:
##     4486989 (28.10%) aligned 0 times
##     9910199 (62.06%) aligned exactly 1 time
##     1571023 (9.84%) aligned &gt;1 times
## 71.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1571023/11481222
## 2 1021536/1571023
## 3 7002606/23378128
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0514-raw_1 and prad-0514-raw_2
## Single-end reads will be in prad-0514-raw_M
## 11097995 sequences in the BAM file
## 11097995 sequences exported
## 11097995 reads; of these:
##   11097995 (100.00%) were unpaired; of these:
##     2905168 (26.18%) aligned 0 times
##     5697807 (51.34%) aligned exactly 1 time
##     2495020 (22.48%) aligned &gt;1 times
## 73.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2495020/8192827
## 2 1773796/2495020
## 3 14658208/33329305
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0515-raw_1 and prad-0515-raw_2
## Single-end reads will be in prad-0515-raw_M
## 16787570 sequences in the BAM file
## 16787570 sequences exported
## 16787570 reads; of these:
##   16787570 (100.00%) were unpaired; of these:
##     4146495 (24.70%) aligned 0 times
##     10292204 (61.31%) aligned exactly 1 time
##     2348871 (13.99%) aligned &gt;1 times
## 75.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2348871/12641075
## 2 1579380/2348871
## 3 4657122/20066320
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0516-raw_1 and prad-0516-raw_2
## Single-end reads will be in prad-0516-raw_M
## 15854252 sequences in the BAM file
## 15854252 sequences exported
## 15854252 reads; of these:
##   15854252 (100.00%) were unpaired; of these:
##     4416058 (27.85%) aligned 0 times
##     9323163 (58.81%) aligned exactly 1 time
##     2115031 (13.34%) aligned &gt;1 times
## 72.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2115031/11438194
## 2 1491771/2115031
## 3 7264574/23313514
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0517-raw_1 and prad-0517-raw_2
## Single-end reads will be in prad-0517-raw_M
## 10519689 sequences in the BAM file
## 10519689 sequences exported
## 10519689 reads; of these:
##   10519689 (100.00%) were unpaired; of these:
##     3169738 (30.13%) aligned 0 times
##     5919267 (56.27%) aligned exactly 1 time
##     1430684 (13.60%) aligned &gt;1 times
## 69.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1430684/7349951
## 2 1005376/1430684
## 3 5061425/15680094
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0518-raw_1 and prad-0518-raw_2
## Single-end reads will be in prad-0518-raw_M
## 7151707 sequences in the BAM file
## 7151707 sequences exported
## 7151707 reads; of these:
##   7151707 (100.00%) were unpaired; of these:
##     1772696 (24.79%) aligned 0 times
##     4230659 (59.16%) aligned exactly 1 time
##     1148352 (16.06%) aligned &gt;1 times
## 75.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1148352/5379011
## 2 787311/1148352
## 3 4109983/12556683
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0519-raw_1 and prad-0519-raw_2
## Single-end reads will be in prad-0519-raw_M
## 10012766 sequences in the BAM file
## 10012766 sequences exported
## 10012766 reads; of these:
##   10012766 (100.00%) were unpaired; of these:
##     3530883 (35.26%) aligned 0 times
##     5048854 (50.42%) aligned exactly 1 time
##     1433029 (14.31%) aligned &gt;1 times
## 64.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1433029/6481883
## 2 988221/1433029
## 3 4959087/15189850
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0520-raw_1 and prad-0520-raw_2
## Single-end reads will be in prad-0520-raw_M
## 4830653 sequences in the BAM file
## 4830653 sequences exported
## 4830653 reads; of these:
##   4830653 (100.00%) were unpaired; of these:
##     1360657 (28.17%) aligned 0 times
##     2463749 (51.00%) aligned exactly 1 time
##     1006247 (20.83%) aligned &gt;1 times
## 71.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1006247/3469996
## 2 748125/1006247
## 3 3394982/9110705
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0521-raw_1 and prad-0521-raw_2
## Single-end reads will be in prad-0521-raw_M
## 4909935 sequences in the BAM file
## 4909935 sequences exported
## 4909935 reads; of these:
##   4909935 (100.00%) were unpaired; of these:
##     1456422 (29.66%) aligned 0 times
##     2402274 (48.93%) aligned exactly 1 time
##     1051239 (21.41%) aligned &gt;1 times
## 70.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1051239/3453513
## 2 788864/1051239
## 3 3639198/9482715
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0522-raw_1 and prad-0522-raw_2
## Single-end reads will be in prad-0522-raw_M
## 9773303 sequences in the BAM file
## 9773303 sequences exported
## 9773303 reads; of these:
##   9773303 (100.00%) were unpaired; of these:
##     2958799 (30.27%) aligned 0 times
##     5239820 (53.61%) aligned exactly 1 time
##     1574684 (16.11%) aligned &gt;1 times
## 69.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1574684/6814504
## 2 1061735/1574684
## 3 6054612/16996431
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0523-raw_1 and prad-0523-raw_2
## Single-end reads will be in prad-0523-raw_M
## 11773926 sequences in the BAM file
## 11773926 sequences exported
## 11773926 reads; of these:
##   11773926 (100.00%) were unpaired; of these:
##     3057518 (25.97%) aligned 0 times
##     6879115 (58.43%) aligned exactly 1 time
##     1837293 (15.60%) aligned &gt;1 times
## 74.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1837293/8716408
## 2 1300995/1837293
## 3 6920422/20612911
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0524-raw_1 and prad-0524-raw_2
## Single-end reads will be in prad-0524-raw_M
## 15113275 sequences in the BAM file
## 15113275 sequences exported
## 15113275 reads; of these:
##   15113275 (100.00%) were unpaired; of these:
##     4527634 (29.96%) aligned 0 times
##     8671974 (57.38%) aligned exactly 1 time
##     1913667 (12.66%) aligned &gt;1 times
## 70.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1913667/10585641
## 2 1304442/1913667
## 3 5207540/19303500
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0525-raw_1 and prad-0525-raw_2
## Single-end reads will be in prad-0525-raw_M
## 14402327 sequences in the BAM file
## 14402327 sequences exported
## 14402327 reads; of these:
##   14402327 (100.00%) were unpaired; of these:
##     4726679 (32.82%) aligned 0 times
##     6473433 (44.95%) aligned exactly 1 time
##     3202215 (22.23%) aligned &gt;1 times
## 67.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3202215/9675648
## 2 2604393/3202215
## 3 9742808/25819228
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0526-raw_1 and prad-0526-raw_2
## Single-end reads will be in prad-0526-raw_M
## 12273999 sequences in the BAM file
## 12273999 sequences exported
## 12273999 reads; of these:
##   12273999 (100.00%) were unpaired; of these:
##     3282420 (26.74%) aligned 0 times
##     7249567 (59.06%) aligned exactly 1 time
##     1742012 (14.19%) aligned &gt;1 times
## 73.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1742012/8991579
## 2 1218790/1742012
## 3 6456014/20102518
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0527-raw_1 and prad-0527-raw_2
## Single-end reads will be in prad-0527-raw_M
## 8353384 sequences in the BAM file
## 8353384 sequences exported
## 8353384 reads; of these:
##   8353384 (100.00%) were unpaired; of these:
##     2601001 (31.14%) aligned 0 times
##     3961528 (47.42%) aligned exactly 1 time
##     1790855 (21.44%) aligned &gt;1 times
## 68.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1790855/5752383
## 2 1226439/1790855
## 3 5769698/15076285
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0528-raw_1 and prad-0528-raw_2
## Single-end reads will be in prad-0528-raw_M
## 20265788 sequences in the BAM file
## 20265788 sequences exported
## 20265788 reads; of these:
##   20265788 (100.00%) were unpaired; of these:
##     5085229 (25.09%) aligned 0 times
##     12412790 (61.25%) aligned exactly 1 time
##     2767769 (13.66%) aligned &gt;1 times
## 74.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2767769/15180559
## 2 1892175/2767769
## 3 8293931/28582890
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0529-raw_1 and prad-0529-raw_2
## Single-end reads will be in prad-0529-raw_M
## 7357825 sequences in the BAM file
## 7357825 sequences exported
## 7357825 reads; of these:
##   7357825 (100.00%) were unpaired; of these:
##     1932010 (26.26%) aligned 0 times
##     3800969 (51.66%) aligned exactly 1 time
##     1624846 (22.08%) aligned &gt;1 times
## 73.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1624846/5425815
## 2 1178696/1624846
## 3 6420209/16463230
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0530-raw_1 and prad-0530-raw_2
## Single-end reads will be in prad-0530-raw_M
## 7417700 sequences in the BAM file
## 7417700 sequences exported
## 7417700 reads; of these:
##   7417700 (100.00%) were unpaired; of these:
##     2266634 (30.56%) aligned 0 times
##     3635742 (49.01%) aligned exactly 1 time
##     1515324 (20.43%) aligned &gt;1 times
## 69.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1515324/5151066
## 2 1091283/1515324
## 3 6708418/16058883
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0531-raw_1 and prad-0531-raw_2
## Single-end reads will be in prad-0531-raw_M
## 13207324 sequences in the BAM file
## 13207324 sequences exported
## 13207324 reads; of these:
##   13207324 (100.00%) were unpaired; of these:
##     4507870 (34.13%) aligned 0 times
##     6513490 (49.32%) aligned exactly 1 time
##     2185964 (16.55%) aligned &gt;1 times
## 65.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2185964/8699454
## 2 1502612/2185964
## 3 10056230/26542412
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0532-raw_1 and prad-0532-raw_2
## Single-end reads will be in prad-0532-raw_M
## 7683685 sequences in the BAM file
## 7683685 sequences exported
## 7683685 reads; of these:
##   7683685 (100.00%) were unpaired; of these:
##     2386279 (31.06%) aligned 0 times
##     3976318 (51.75%) aligned exactly 1 time
##     1321088 (17.19%) aligned &gt;1 times
## 68.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1321088/5297406
## 2 968469/1321088
## 3 3658792/11289039
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0533-raw_1 and prad-0533-raw_2
## Single-end reads will be in prad-0533-raw_M
## 9051480 sequences in the BAM file
## 9051480 sequences exported
## 9051480 reads; of these:
##   9051480 (100.00%) were unpaired; of these:
##     3077153 (34.00%) aligned 0 times
##     4654837 (51.43%) aligned exactly 1 time
##     1319490 (14.58%) aligned &gt;1 times
## 66.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1319490/5974327
## 2 940617/1319490
## 3 6437340/17150583
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0534-raw_1 and prad-0534-raw_2
## Single-end reads will be in prad-0534-raw_M
## 8665447 sequences in the BAM file
## 8665447 sequences exported
## 8665447 reads; of these:
##   8665447 (100.00%) were unpaired; of these:
##     2331746 (26.91%) aligned 0 times
##     4571943 (52.76%) aligned exactly 1 time
##     1761758 (20.33%) aligned &gt;1 times
## 73.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1761758/6333701
## 2 1384271/1761758
## 3 4550402/13407734
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0535-raw_1 and prad-0535-raw_2
## Single-end reads will be in prad-0535-raw_M
## 9264602 sequences in the BAM file
## 9264602 sequences exported
## 9264602 reads; of these:
##   9264602 (100.00%) were unpaired; of these:
##     2856637 (30.83%) aligned 0 times
##     4828066 (52.11%) aligned exactly 1 time
##     1579899 (17.05%) aligned &gt;1 times
## 69.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1579899/6407965
## 2 1192556/1579899
## 3 4999840/14512581
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0536-raw_1 and prad-0536-raw_2
## Single-end reads will be in prad-0536-raw_M
## 7015258 sequences in the BAM file
## 7015258 sequences exported
## 7015258 reads; of these:
##   7015258 (100.00%) were unpaired; of these:
##     2114939 (30.15%) aligned 0 times
##     3888809 (55.43%) aligned exactly 1 time
##     1011510 (14.42%) aligned &gt;1 times
## 69.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1011510/4900319
## 2 681605/1011510
## 3 3811243/11566876
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0537-raw_1 and prad-0537-raw_2
## Single-end reads will be in prad-0537-raw_M
## 5083569 sequences in the BAM file
## 5083569 sequences exported
## 5083569 reads; of these:
##   5083569 (100.00%) were unpaired; of these:
##     1481759 (29.15%) aligned 0 times
##     2243189 (44.13%) aligned exactly 1 time
##     1358621 (26.73%) aligned &gt;1 times
## 70.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1358621/3601810
## 2 1066203/1358621
## 3 4488706/11163409
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0538-raw_1 and prad-0538-raw_2
## Single-end reads will be in prad-0538-raw_M
## 10286688 sequences in the BAM file
## 10286688 sequences exported
## 10286688 reads; of these:
##   10286688 (100.00%) were unpaired; of these:
##     2700599 (26.25%) aligned 0 times
##     5916938 (57.52%) aligned exactly 1 time
##     1669151 (16.23%) aligned &gt;1 times
## 73.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1669151/7586089
## 2 1234192/1669151
## 3 4033573/13693281
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0539-raw_1 and prad-0539-raw_2
## Single-end reads will be in prad-0539-raw_M
## 7932312 sequences in the BAM file
## 7932312 sequences exported
## 7932312 reads; of these:
##   7932312 (100.00%) were unpaired; of these:
##     2435914 (30.71%) aligned 0 times
##     4154559 (52.38%) aligned exactly 1 time
##     1341839 (16.92%) aligned &gt;1 times
## 69.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1341839/5496398
## 2 944868/1341839
## 3 5898583/15888987
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0540-raw_1 and prad-0540-raw_2
## Single-end reads will be in prad-0540-raw_M
## 7337883 sequences in the BAM file
## 7337883 sequences exported
## 7337883 reads; of these:
##   7337883 (100.00%) were unpaired; of these:
##     2259872 (30.80%) aligned 0 times
##     3675927 (50.10%) aligned exactly 1 time
##     1402084 (19.11%) aligned &gt;1 times
## 69.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1402084/5078011
## 2 990769/1402084
## 3 7225705/18395210
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0541-raw_1 and prad-0541-raw_2
## Single-end reads will be in prad-0541-raw_M
## 14860225 sequences in the BAM file
## 14860225 sequences exported
## 14860225 reads; of these:
##   14860225 (100.00%) were unpaired; of these:
##     3530575 (23.76%) aligned 0 times
##     8923117 (60.05%) aligned exactly 1 time
##     2406533 (16.19%) aligned &gt;1 times
## 76.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2406533/11329650
## 2 1752821/2406533
## 3 7266605/23036167
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0542-raw_1 and prad-0542-raw_2
## Single-end reads will be in prad-0542-raw_M
## 7638564 sequences in the BAM file
## 7638564 sequences exported
## 7638564 reads; of these:
##   7638564 (100.00%) were unpaired; of these:
##     2017513 (26.41%) aligned 0 times
##     3512403 (45.98%) aligned exactly 1 time
##     2108648 (27.61%) aligned &gt;1 times
## 73.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2108648/5621051
## 2 1522570/2108648
## 3 16207525/35563564
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0543-raw_1 and prad-0543-raw_2
## Single-end reads will be in prad-0543-raw_M
## 11508563 sequences in the BAM file
## 11508563 sequences exported
## 11508563 reads; of these:
##   11508563 (100.00%) were unpaired; of these:
##     3817427 (33.17%) aligned 0 times
##     6336073 (55.06%) aligned exactly 1 time
##     1355063 (11.77%) aligned &gt;1 times
## 66.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1355063/7691136
## 2 950064/1355063
## 3 4091907/14495201
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0544-raw_1 and prad-0544-raw_2
## Single-end reads will be in prad-0544-raw_M
## 12203245 sequences in the BAM file
## 12203245 sequences exported
## 12203245 reads; of these:
##   12203245 (100.00%) were unpaired; of these:
##     3201100 (26.23%) aligned 0 times
##     7000865 (57.37%) aligned exactly 1 time
##     2001280 (16.40%) aligned &gt;1 times
## 73.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2001280/9002145
## 2 1452462/2001280
## 3 8629719/24005466
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0545-raw_1 and prad-0545-raw_2
## Single-end reads will be in prad-0545-raw_M
## 10966512 sequences in the BAM file
## 10966512 sequences exported
## 10966512 reads; of these:
##   10966512 (100.00%) were unpaired; of these:
##     2925321 (26.68%) aligned 0 times
##     4927049 (44.93%) aligned exactly 1 time
##     3114142 (28.40%) aligned &gt;1 times
## 73.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3114142/8041191
## 2 2494069/3114142
## 3 8182855/21168067
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0546-raw_1 and prad-0546-raw_2
## Single-end reads will be in prad-0546-raw_M
## 9647053 sequences in the BAM file
## 9647053 sequences exported
## 9647053 reads; of these:
##   9647053 (100.00%) were unpaired; of these:
##     2590159 (26.85%) aligned 0 times
##     5566063 (57.70%) aligned exactly 1 time
##     1490831 (15.45%) aligned &gt;1 times
## 73.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1490831/7056894
## 2 1064867/1490831
## 3 4918874/15301070
## [bam_sort_core] merging from 3 files...
</pre></div>

<p></div></div></p>

<h2>Count reads</h2>

<p>There&#39;s a few types of sRNAs. We can get most of them from the hg19 mappings, but for tRF-1, tRF-3, and mature tRNA halves, we need the mappings to mature tRNAs.</p>

<p>So here&#39;s each sRNA, by source and destination.</p>

<ul>
<li>tRF-1: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>immature tRNA 5&#39; half: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>mature tRNA 3&#39; half/tRF3: tRNAs/split from anticodon</li>
<li>mature tRNA 5&#39; half/tRF5: tRNAs/split from anticodon</li>
<li>miRNAs: hg19/miRbase</li>
<li>snoRNAs: hg19/DASHR</li>
</ul>

<div class="chunk" id="count-features-function"><div class="rcode"><div class="source"><pre class="knitr r">countFeaturesFromBam = function(metadata, features, dir = ".") {
  metadata = metadata[which(file.exists(paste0(dir, "/", metadata$shortnames, ".bam"))), ]
  trf1s_plus = features[features$class == "tRF-1" & strand(features) == "+"]
  trf1s_minus = features[features$class == "tRF-1" & strand(features) == "-"]
  trf3s = features[features$class == "tRF-3"]
  trf5s = features[features$class == "tRF-5"]
  features = features[!(features$class %in% c("tRF-1", "tRF-3", "tRF-5"))]
  # use a for loop because r will run out of memory
  counts = mclapply(1:nrow(metadata), function(i) {
    row = metadata[i, ]
    system(paste("echo Counting sRNAs from", row$shortnames))
    bamfile = paste0(dir, "/", row$shortnames, ".bam")
    sample = readGAlignments(bamfile)
    sample = convertChrNames(sample, "Ensembl")
    sample_counts = rbind(data.frame(counts = countOverlaps(features, sample, minoverlap = 10)),
                          data.frame(counts = countOverlaps(trf1s_plus, sample, minoverlap = 10, type = 'start')),
                          data.frame(counts = countOverlaps(trf1s_minus, sample, minoverlap = 10, type = 'end')),
                          data.frame(counts = countOverlaps(trf3s, sample, minoverlap = 10, type = 'end', maxgap = 3)), # in case of a CCA tail. We don't have to worry about aligning to 3 past the gap since these are the mature tRNAs.
                          data.frame(counts = countOverlaps(trf5s, sample, minoverlap = 10, type = 'start')))
    colnames(sample_counts) = row$barcode
    sample_counts
  }, mc.cores = 30)
  names = unlist(lapply(counts, colnames))
  counts = as.data.frame(matrix(unlist(counts), ncol = nrow(metadata)))
  rownames(counts) = c(features$tx_name, trf1s_plus$tx_name, trf1s_minus$tx_name, trf3s$tx_name, trf5s$tx_name)
  colnames(counts) = names
  counts
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-luad"><div class="rcode"><div class="source"><pre class="knitr r">luad_raw_counts <- countFeaturesFromBam(luad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/luad/mirna/')
save(file = 'luad-raw-counts.RData', luad_raw_counts)
save(file = 'luad-metadata.RData', luad_metadata) 
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-brca"><div class="rcode"><div class="source"><pre class="knitr r">brca_raw_counts <- countFeaturesFromBam(brca_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/brca/mirna/')
save(file = 'brca-raw-counts.RData', brca_raw_counts)
save(file = 'brca-metadata.RData', brca_metadata)
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-prad"><div class="rcode"><div class="source"><pre class="knitr r">prad_raw_counts <- countFeaturesFromBam(prad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/prad/mirna/')
save(file = 'prad-raw-counts.RData', prad_raw_counts)
save(file = 'prad-metadata.RData', prad_metadata)
</pre></div>

<p></div></div></p>

<div class="chunk" id="save-session"><div class="rcode"><div class="source"><pre class="knitr r">save.session("process-reads.RSession")
</pre></div>

<div class="output"><pre class="knitr r">## Saving search path..
## Saving list of loaded packages..
## Saving all data...
## Done.
</pre></div>

<p></div></div></p>

</body>

</html>
