<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>+/- </title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>





<style type="text/css">

</style>

<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>

<!-- highlight.js -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/styles/default.css" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/highlight.min.js"></script>
<script src="https://yandex.st/highlightjs/7.3/languages/r.min.js"></script>

<!-- bootstrap -->
<link href=https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css rel="stylesheet">
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>

<!-- Manific Popup -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/0.8.9/jquery.magnific-popup.min.js"></script>

<script defer="defer">
// Function to generate the dynamic table of contents
jQuery.fn.generate_TOC = function () {
  var base = $(this[0]);

  var selectors = ['h1', 'h2', 'h3', 'h4'];

  var last_ptr = [{}, {}, {}, {}];

  var anchors = {};

  generate_anchor = function (text) {
    var test = text.replace(/\W/g, '_');

    while(test in anchors){
      //if no suffix, add one
      if(test.match(/_\d+$/) === null){
        test = test + "_2";
      }
      //else generate unique id for duplicates by adding one to the suffix
      else {
        test = test.replace(/_(\d+)$/, function(match, number){ var num=+number+1; return("_" + num) });
      }
    }
    anchors[test]=1;
    return(test);
  }

  $(selectors.join(',')).each(function () {
    var heading = $(this);
    var idx = selectors.indexOf(heading.prop('tagName').toLowerCase());
    var itr = 0;

    while (itr <= idx) {
      if (jQuery.isEmptyObject(last_ptr[itr])) {
        last_ptr[itr] = $('<ul>').addClass('nav');
        if (itr === 0) {
          base.append(last_ptr[itr])
        } else {
          if(last_ptr[itr-1].children('li').length === 0){
            last_ptr[itr-1].append(last_ptr[itr]);
          }
          else {
            last_ptr[itr - 1].children('li').last().append(last_ptr[itr]);
          }
        }
      }
      itr++;
    }
    var anchor = generate_anchor(heading.text());
    heading.attr('id', anchor);
    var a = $('<a>')
    .text(heading.text())
    .attr('href', '#' + anchor);

  var li = $('<li>')
    .append(a);

  last_ptr[idx].append(li);
  for (i = idx + 1; i < last_ptr.length; i++) {
    last_ptr[i] = {};
  }
  });
}
/* run scripts when document is ready */
$(function() {
  "use strict";

  var $window = $(window);
  var $body = $(document.body);

  document.title = $('h1').first().text();

  /* size of thumbnails */
  var thumbsize = "col-md-3";

  var show_code =  false ;

  var show_output =  true ;

  var show_figure =  true ;

  /* included languages */
  var languages = [];

  /* Using render_html, so add in code block */
  $('pre.knitr').each(function(){
    $(this).removeClass('knitr');
    if($(this).find('code').length < 1){
      $(this).wrapInner('<code class=' + $(this).attr('class') + '></code>');
    }
  });

  /* style tables */
  $('table').addClass('table table-striped table-bordered table-hover table-condensed');

  /* add toggle panel to rcode blocks */
  $('div.source,div.output,div.message,div.warning,div.error').each(function() {
    var button = $('<h5 class="panel-title">+/- </h5>');

    if($(this).hasClass('source')){
      var code_block = $(this).find('code');
      var lang_type = code_block.attr('class');
      button.text(button.text() + lang_type + ' Code');
      button.addClass('source ' + lang_type);
      languages[lang_type]=0;
      code_block.each(function(i, e) {
        hljs.highlightBlock(e);
      });
      $(this).addClass('panel panel-primary ' + lang_type);
    }
    else if($(this).hasClass('output')){
      button.text(button.text() + 'Output');
      button.addClass('output');
      $(this).addClass('panel panel-success');
    }
    else if($(this).hasClass('message')){
      button.text(button.text() + 'Message');
      button.addClass('message');
      $(this).addClass('panel panel-info');
    }
    else if($(this).hasClass('warning')){
      button.text(button.text() + 'Warning');
      button.addClass('warning');
      $(this).addClass('panel panel-warning');
    }
    else if($(this).hasClass('error')){
      button.text(button.text() + 'Error');
      button.addClass('error');
      $(this).addClass('panel panel-danger');
    }
    else {
      //console.log($(this));
    }
    $(this).prepend($('<div class="panel-heading toggle" />').append(button));
  });

  /* give images a lightbox and thumbnail classes to allow lightbox and thumbnails TODO: make gallery if graphs are grouped */
  $('div.rimage img').each(function(){

    //remove rimage div
    $(this).unwrap();

    var a = $(this).
      wrap('<a href=# class="media-object pull-left mfp-image thumbnail ' + thumbsize + '"></a>').
      parent();

    var sibs = a.prevUntil('div.rimage,div.media');
    var div = $('<div class="media" />');
    if(sibs.length != 0){
      sibs.addClass('media-body');
      //need to reverse order as prevUntil puts objects in the order it found them
      $(sibs.get().reverse()).wrapAll(div).parent().prepend(a);
    }
    else {
      a.wrap(div);
    }
  });

  $('div.chunk').addClass('media');

  $('.rcode > .panel').addClass('media');
  /* Magnific Popup */
  $(".thumbnail").each(function(){
    $(this).magnificPopup({
      disableOn: 768,
      closeOnContentClick: true,

      type: 'image',
      items: {
        src: $(this).find('img').attr('src'),
      }
    });
  });

  /* add bootstrap classes */
  $('body').wrapInner('<div class="container"><div class="row"><div class="contents">');

  var create_language_links = function(){
    var text='';
    var language;
    for(language in languages){
      if(languages.hasOwnProperty(language)){
        text += '<li><a href=# class="toggle-global source ' + language + '" type="source.' + language + '">' + language + '</a></li>\n';
      }
    }
    return text;
  }

  var navbar =
  '<div class="navbar navbar-fixed-bottom navbar-inverse">\
    <div class="container">\
      <div class="navbar-header">\
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
        </button>\
      </div>\
      <div id="bottom-navbar" class="navbar-collapse collapse navbar-responsive-collapse">\
        <ul class="nav navbar-nav navbar-right">\
          <li class="nav"><p class="navbar-text">Toggle</p></li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Code <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Languages</li>'
              + create_language_links() +
              '<li><a href="#" type="all-source" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Output <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Type</li>\
                <li><a href="#" type="output" class="toggle-global">Output</a></li>\
                <li><a href="#" type="message" class="toggle-global">Message</a></li>\
                <li><a href="#" type="warning" class="toggle-global">Warning</a></li>\
                <li><a href="#" type="error" class="toggle-global">Error</a></li>\
                <li><a href="#" type="all-output" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li><a href="#" class="toggle-figure">Figures</a></li>\
        </ul>\
      </div><!-- /.nav-collapse -->\
    </div><!-- /.container -->\
  </div>';
  /* add navbar */
  $('.container').append(navbar);

  /* onclick toggle next code block */
  $('.toggle').click(function() {
    $(this).next('pre').slideToggle();
    return false;
  });

  // global toggles
  $('.toggle-global').click(function(){
    var type = $(this).attr('type');
    if(type == 'all-source'){
      for(var language in languages){
        $('li a[type="source.' + language + '"]').click();
      }
    }
    if(type == 'all-output'){
      $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error]').click();
    }
    else {
      if($(this).closest('li').hasClass('active')){
        $('div.' + type).children('pre').slideUp();
      }
      else {
        $('div.' + type).children('pre').slideDown();
      }
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });
  // global toggles figure, this is more complicated than just slideToggle because you have to hide the parent div as well
  $('.toggle-figure').click(function(){
    var imgs = $('.thumbnail img');
    if(imgs.is(":visible")){
      imgs.slideUp(400, function(){ $(this).parent().toggle(); });
    }
    else {
      imgs.parent().show();
      imgs.show();
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });

  /* add footer */
  $('body').wrapInner('<div id="wrap" />');
  $('#wrap').append('<div id="push" />');
  var p = $('p:contains("Author:")');
  var last_p = p.filter(':last');
  last_p.addClass('text-muted').attr('id','credit');
  last_p.append('<p>Styled with <a href="https://github.com/jimhester/knitrBootstrap">knitrBootstrap</a></p>');
  last_p = last_p.wrap('<div id="footer"><div class="container">').parent().parent();
  last_p.appendTo("body");

  $('.container > .row').prepend('<div class="col-md-3"><div id="toc" class="well sidebar sidenav affix hidden-print"/></div>');

  $('.contents').addClass('col-md-offset-3');

  /* table of contents */
  $('#toc').generate_TOC();

  if(show_code){
    /* toggle source buttons pressed */
    $('a.toggle-global.source').closest('li').addClass('active');
  }
  else {
    /* hide code blocks */
    $('div.source pre').hide();
  }

  if(show_output){
    /* toggle output buttons pressed */
    $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error], li a[type=all-output]').closest('li').addClass('active');
  }
  else {
    /* hide output blocks */
    $('div.output pre').hide();
  }

  if(show_figure){
    /* toggle figure button pressed */
    $('li a.toggle-figure').closest('li').addClass('active');
  }
  else {
    /* hide figures */
    $('.thumbnail').hide();
  }

  /* remove paragraphs with no content */
  $('p:empty').remove();

  $body.scrollspy({
    target: '.sidebar',
  });


  //TODO refresh on show/hide
  $window.on('load', function () {
    $body.scrollspy('refresh');
  })

});


</script>
<style>
/* Knitr_bootstrap styles */
#wrap .container-fluid {
  padding: 0;
  overflow: hidden;
}
.toggle{
  text-transform: capitalize;
}

.toggle-global{
  text-transform: capitalize;
}

/* Sticky footer styles */
* {
  margin:0;
}
html,
body {
    height: 100%;
    padding:0 !important;
    /* The html and body elements cannot have any padding or margin. */
    /*overflow-x: hidden;*/
}

/* Wrapper for page content to push down footer */
#wrap {
    min-height: 100%;
    height: auto !important;
    height: 100%;
    /* Negative indent footer by it's height */
    margin: 0 auto -120px;
}

/* Set the fixed height of the footer here */
#push,
#footer {
    height: 120px;
}

#footer {
  text-align: center;
}

/* Top level subheader elements.  These are the first nested items underneath a header element. */
.header li {
  font-size: 20px;
}

/* Makes the font smaller for all subheader elements. */
.sub-header li {
    font-size: 12px;
}

button.thumbnails {
  margin-left:0px;
}

/*
 * Side navigation
 *
 * Scrollspy and affixed enhanced navigation to highlight sections and secondary
 * sections of docs content.
 */

/* By default it's not affixed in mobile views, so undo that */
.sidebar.affix {
  position: static;
}

/* First level of nav */
.sidenav {
  margin-top: 30px;
  margin-bottom: 30px;
  padding-top:    10px;
  padding-bottom: 10px;
  border-radius: 5px;
}

/* All levels of nav */
.sidebar .nav > li > a {
  display: block;
  padding: 5px 20px;
}
.sidebar .nav > li > a:hover,
.sidebar .nav > li > a:focus {
  text-decoration: none;
  border-right: 1px solid;
}
.sidebar .nav > .active > a,
.sidebar .nav > .active:hover > a,
.sidebar .nav > .active:focus > a {
  font-weight: bold;
  background-color: transparent;
  border-right: 1px solid;
}

/* Nav: second level (shown on .active) */
.sidebar .nav .nav {
  display: none; /* Hide by default, but at >768px, show it */
  margin-bottom: 8px;
}
.sidebar .nav .nav > li > a {
  padding-top:    3px;
  padding-bottom: 3px;
  padding-left: 30px;
  font-size: 90%;
}

.sidebar .nav .nav .nav > li > a {
  padding-left: 40px;
}
.sidebar .nav .nav .nav .nav > li > a {
  padding-left: 50px;
}

/* Show and affix the side nav when space allows it */
@media screen and (min-width: 992px) {
  .sidebar .nav > .active > ul {
    display: block;
  }
  /* Widen the fixed sidebar */
  .sidebar.affix,
  .sidebar.affix-bottom {
    width: 213px;
  }
  .sidebar.affix-top,
  .sidebar.affix {
    position: fixed; /* Undo the static from mobile first approach */
    top: 30px;
  }
  .sidebar.affix-bottom {
    position: absolute; /* Undo the static from mobile first approach */
  }
  .sidebar.affix-bottom .sidenav,
  .sidebar.affix .sidenav {
    margin-top: 0;
    margin-bottom: 0;
  }
}
@media screen and (min-width: 1200px) {
  /* Widen the fixed sidebar again */
  .sidebar.affix-bottom,
  .sidebar.affix {
    width: 263px;
  }
}

#toc {
  padding: 10px 0px;
  margin:0;
  border:0;
}


.panel pre {
  border-radius: 0;
  border: 0;
  margin: 0;
  padding: 0;
}
.panel pre code {
  border-radius: 0;
}
/* Magnific Popup CSS */
.mfp-bg {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1042;
  overflow: hidden;
  position: fixed;
  background: #0b0b0b;
  opacity: 0.8;
  filter: alpha(opacity=80); }

.mfp-wrap {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1043;
  position: fixed;
  outline: none !important;
  -webkit-backface-visibility: hidden; }

.mfp-container {
  text-align: center;
  position: absolute;
  width: 100%;
  height: 100%;
  left: 0;
  top: 0;
  padding: 0 8px;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box; }

.mfp-container:before {
  content: '';
  display: inline-block;
  height: 100%;
  vertical-align: middle; }

.mfp-align-top .mfp-container:before {
  display: none; }

.mfp-content {
  position: relative;
  display: inline-block;
  vertical-align: middle;
  margin: 0 auto;
  text-align: left;
  z-index: 1045; }

.mfp-inline-holder .mfp-content,
.mfp-ajax-holder .mfp-content {
  width: 100%;
  cursor: auto; }

.mfp-ajax-cur {
  cursor: progress; }

.mfp-zoom-out-cur,
.mfp-zoom-out-cur .mfp-image-holder .mfp-close {
  cursor: -moz-zoom-out;
  cursor: -webkit-zoom-out;
  cursor: zoom-out; }

.mfp-zoom {
  cursor: pointer;
  cursor: -webkit-zoom-in;
  cursor: -moz-zoom-in;
  cursor: zoom-in; }

.mfp-auto-cursor .mfp-content {
  cursor: auto; }

.mfp-close,
.mfp-arrow,
.mfp-preloader,
.mfp-counter {
  -webkit-user-select: none;
  -moz-user-select: none;
  user-select: none; }

.mfp-loading.mfp-figure {
  display: none; }

.mfp-hide {
  display: none !important; }

.mfp-preloader {
  color: #cccccc;
  position: absolute;
  top: 50%;
  width: auto;
  text-align: center;
  margin-top: -0.8em;
  left: 8px;
  right: 8px;
  z-index: 1044; }

.mfp-preloader a {
  color: #cccccc; }

.mfp-preloader a:hover {
  color: white; }

.mfp-s-ready .mfp-preloader {
  display: none; }

.mfp-s-error .mfp-content {
  display: none; }

button.mfp-close,
button.mfp-arrow {
  overflow: visible;
  cursor: pointer;
  background: transparent;
  border: 0;
  -webkit-appearance: none;
  display: block;
  padding: 0;
  z-index: 1046;
  -webkit-box-shadow: none;
  box-shadow: none; }

button::-moz-focus-inner {
  padding: 0;
  border: 0; }

.mfp-close {
  width: 44px;
  height: 44px;
  line-height: 44px;
  position: absolute;
  right: 0;
  top: 0;
  text-decoration: none;
  text-align: center;
  opacity: 0.65;
  padding: 0 0 18px 10px;
  color: white;
  font-style: normal;
  font-size: 28px;
  font-family: Arial, Baskerville, monospace; }
  .mfp-close:hover, .mfp-close:focus {
    opacity: 1; }
  .mfp-close:active {
    top: 1px; }

.mfp-close-btn-in .mfp-close {
  color: #333333; }

.mfp-image-holder .mfp-close,
.mfp-iframe-holder .mfp-close {
  color: white;
  right: -6px;
  text-align: right;
  padding-right: 6px;
  width: 100%; }

.mfp-counter {
  position: absolute;
  top: 0;
  right: 0;
  color: #cccccc;
  font-size: 12px;
  line-height: 18px; }

.mfp-arrow {
  position: absolute;
  opacity: 0.65;
  margin: 0;
  top: 50%;
  margin-top: -55px;
  padding: 0;
  width: 90px;
  height: 110px;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0); }

.mfp-arrow:active {
  margin-top: -54px; }

.mfp-arrow:hover,
.mfp-arrow:focus {
  opacity: 1; }

.mfp-arrow:before, .mfp-arrow:after,
.mfp-arrow .mfp-b,
.mfp-arrow .mfp-a {
  content: '';
  display: block;
  width: 0;
  height: 0;
  position: absolute;
  left: 0;
  top: 0;
  margin-top: 35px;
  margin-left: 35px;
  border: medium inset transparent; }
.mfp-arrow:after,
.mfp-arrow .mfp-a {
  border-top-width: 13px;
  border-bottom-width: 13px;
  top: 8px; }
.mfp-arrow:before,
.mfp-arrow .mfp-b {
  border-top-width: 21px;
  border-bottom-width: 21px; }

.mfp-arrow-left {
  left: 0; }
  .mfp-arrow-left:after,
  .mfp-arrow-left .mfp-a {
    border-right: 17px solid white;
    margin-left: 31px; }
  .mfp-arrow-left:before,
  .mfp-arrow-left .mfp-b {
    margin-left: 25px;
    border-right: 27px solid #3f3f3f; }

.mfp-arrow-right {
  right: 0; }
  .mfp-arrow-right:after,
  .mfp-arrow-right .mfp-a {
    border-left: 17px solid white;
    margin-left: 39px; }
  .mfp-arrow-right:before,
  .mfp-arrow-right .mfp-b {
    border-left: 27px solid #3f3f3f; }

.mfp-iframe-holder {
  padding-top: 40px;
  padding-bottom: 40px; }

.mfp-iframe-holder .mfp-content {
  line-height: 0;
  width: 100%;
  max-width: 900px; }

.mfp-iframe-scaler {
  width: 100%;
  height: 0;
  overflow: hidden;
  padding-top: 56.25%; }

.mfp-iframe-scaler iframe {
  position: absolute;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: black; }

.mfp-iframe-holder .mfp-close {
  top: -40px; }

/* Main image in popup */
img.mfp-img {
  width: auto;
  max-width: 100%;
  height: auto;
  display: block;
  line-height: 0;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
  padding: 40px 0 40px;
  margin: 0 auto; }

/* The shadow behind the image */
.mfp-figure:after {
  content: '';
  position: absolute;
  left: 0;
  top: 40px;
  bottom: 40px;
  display: block;
  right: 0;
  width: auto;
  height: auto;
  z-index: -1;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: #444444; }

.mfp-figure {
  line-height: 0; }

.mfp-bottom-bar {
  margin-top: -36px;
  position: absolute;
  top: 100%;
  left: 0;
  width: 100%;
  cursor: auto; }

.mfp-title {
  text-align: left;
  line-height: 18px;
  color: #f3f3f3;
  word-wrap: break-word;
  padding-right: 36px; }

.mfp-figure small {
  color: #bdbdbd;
  display: block;
  font-size: 12px;
  line-height: 14px; }

.mfp-image-holder .mfp-content {
  max-width: 100%; }

.mfp-gallery .mfp-image-holder .mfp-figure {
  cursor: pointer; }

@media screen and (max-width: 800px) and (orientation: landscape), screen and (max-height: 300px) {
  /**
   * Remove all paddings around the image on small screen
   */
  .mfp-img-mobile .mfp-image-holder {
    padding-left: 0;
    padding-right: 0; }

  .mfp-img-mobile img.mfp-img {
    padding: 0; }

  /* The shadow behind the image */
  .mfp-img-mobile .mfp-figure:after {
    top: 0;
    bottom: 0; }

  .mfp-img-mobile .mfp-bottom-bar {
    background: rgba(0, 0, 0, 0.6);
    bottom: 0;
    margin: 0;
    top: auto;
    padding: 3px 5px;
    position: fixed;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box; }

  .mfp-img-mobile .mfp-bottom-bar:empty {
    padding: 0; }

  .mfp-img-mobile .mfp-counter {
    right: 5px;
    top: 3px; }

  .mfp-img-mobile .mfp-close {
    top: 0;
    right: 0;
    width: 35px;
    height: 35px;
    line-height: 35px;
    background: rgba(0, 0, 0, 0.6);
    position: fixed;
    text-align: center;
    padding: 0; }

  .mfp-img-mobile .mfp-figure small {
    display: inline;
    margin-left: 5px; } }
@media all and (max-width: 900px) {
  .mfp-arrow {
    -webkit-transform: scale(0.75);
    transform: scale(0.75); }

  .mfp-arrow-left {
    -webkit-transform-origin: 0;
    transform-origin: 0; }

  .mfp-arrow-right {
    -webkit-transform-origin: 100%;
    transform-origin: 100%; }

  .mfp-container {
    padding-left: 6px;
    padding-right: 6px; } }
.mfp-ie7 .mfp-img {
  padding: 0; }
.mfp-ie7 .mfp-bottom-bar {
  width: 600px;
  left: 50%;
  margin-left: -300px;
  margin-top: 5px;
  padding-bottom: 5px; }
.mfp-ie7 .mfp-container {
  padding: 0; }
.mfp-ie7 .mfp-content {
  padding-top: 44px; }
.mfp-ie7 .mfp-close {
  top: 0;
  right: 0;
  padding-top: 0; }

//Magnific overrides
.mfp-image img{
  background: white;
}
.mfp-figure:after {
  background: white;
}

</style>

 

</head>

<body>
<h1>Process data</h1>

<p><em>Last modified <code class="knitr inline">12:01 PM on Apr 18, 2016</code>. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/process-reads/</code>.</em></p>

<div class="chunk" id="libraries"><div class="rcode"><div class="source"><pre class="knitr r">library(stringr)
library(GenomicRanges)
library(GenomicAlignments)
library(BiocParallel)
library(reshape2)
library(Biobase)
library(DESeq2)
library(edgeR)
library(sva)
source('/inside/grotto/blin/programs/convertChrNames.R')
load('/inside/grotto/blin/data/hg19-srnas.RData')
</pre></div>

<p></div></div></p>

<p>This script downloads and remaps reads. </p>

<ul>
<li>v1: Prostate adenocarcinoma (PRAD)</li>
<li>v2: Lung adenocarcinoma (LUAD) and breast invasive carcinoma (BRCA)</li>
<li>v3: Hepatocellular carcinoma (LIHC), Lymphoid neoplasm diffuse large b-cell lymphoma (DLBC), colorectal (COAD and READ), pancreatic (PAAD), testicular germ cell (TGCT), lower grade glioma (LGG)

<ul>
<li>acute myeloid leukemia (AML and LAML), glioblastoma (GBM), osteosarcoma (OS) do not have enough samples for miRNA-Seq data.</li>
</ul></li>
</ul>

<h2>Download raw reads and metadata</h2>

<div class="chunk" id="parse-metadata-function"><div class="rcode"><div class="source"><pre class="knitr r"># metadata_file: sample metadata downloaded from CGHub in tab delimited form
# cancer_code: short abbreviation in lowercase describing the cancer type, e.g. "prad" or "luad"
parseMetadata <- function(metadata_file, cancer_code) {
  metadata <- read.table(metadata_file, header = TRUE, sep = '\t', as.is = TRUE)
  metadata <- metadata[, c('barcode', 'sample_type', 'filename', 'analysis_id', 'uploaded', 'participant_id')]
  # convert character "uploaded" column to date object for easier comparison
  metadata$uploaded <- strptime(gsub("/", "-", metadata$uploaded), "%Y-%m-%d")
  # start by getting matched pairs
  # first, check if there actually are normal tissue samples
  parsed_metadata <- data.frame()
  if (any(metadata$sample_type == "NT")) {
  # all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.
    paired <- subset(metadata, participant_id %in% subset(metadata, sample_type == "NT")$participant_id & participant_id %in% subset(metadata, sample_type == "TP")$participant_id)
    paired$paired <- TRUE # remember which samples are matched tumor/normal
    for (current_id in unique(paired$participant_id)) {
      nt <- subset(paired, participant_id == current_id & sample_type == "NT")
      tp <- subset(paired, participant_id == current_id & sample_type == "TP")
      parsed_metadata <- rbind(parsed_metadata, tp[which(tp$uploaded == max(tp$uploaded)), ], nt[which(nt$uploaded == max(nt$uploaded)), ])
    }
    singles <- metadata[-which(metadata$participant_id %in% parsed_metadata$participant_id), ]
  } else {
    singles <- metadata
  }
  # now for the unpaired samples
  singles$paired <- FALSE
  for (current_id in unique(singles$participant_id)) {
    ps <- subset(singles, participant_id == current_id) # get all samples for this participant
    parsed_metadata <- rbind(parsed_metadata, ps[which(ps$uploaded == max(ps$uploaded)), ])
  }
  # create sample ID numbers and new filenames
  numbering <- 1:nrow(parsed_metadata)
  parsed_metadata$shortids <- numbering
  numbering <- formatC(numbering, flag = "0", digits = 3)
  parsed_metadata$shortnames <- paste0(cancer_code, "-", numbering)
  # reencode upload date as a string - it's encoded as a time object, screwing up data frame operations
  parsed_metadata$uploaded <- as.character(parsed_metadata$uploaded)
  parsed_metadata
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="parse-metadata"><div class="rcode"><div class="source"><pre class="knitr r">luad_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/luad-summary.tab', 'luad')
brca_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/brca-summary.tab', 'brca')
prad_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/prad-summary.tab', 'prad')
dlbc_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/dlbc-summary.tab', 'dlbc')
lihc_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lihc-summary.tab', 'lihc')
coad_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/coad-summary.tab', 'coad')
paad_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/paad-summary.tab', 'paad')
tgct_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/tgct-summary.tab', 'tgct')
lgg_metadata <- parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lgg-summary.tab', 'lgg')
</pre></div>

<p></div></div></p>

<div class="chunk" id="download-files"><div class="rcode"><div class="source"><pre class="knitr r">metadatas <- list(luad = luad_metadata, brca = brca_metadata, prad = prad_metadata, dlbc = dlbc_metadata, lihc = lihc_metadata, coad = coad_metadata, paad = paad_metadata, tgct = tgct_metadata, lgg = lgg_metadata)
currentwd <- getwd()
setwd('/inside/grotto/blin/trna-markers/datasets/raw')
for (cancer_code in names(metadatas)) { 
  metadata <- metadatas[[cancer_code]]
  for (analysis_id in metadata$analysis_id) {
    if (file.exists(analysis_id)) next # if exists, assume file was downloaded correctly and is not corrupted
    system(paste0('echo gtdownload -c $CGKEY -d ', analysis_id, cancer_code))
    system(paste0('gtdownload -c $CGKEY -d ', analysis_id))
  }
  file.symlink(paste0(getwd(), '/', metadata$analysis_id, '/', metadata$barcode, '_mirna.bam'), paste0('/inside/grotto/blin/trna-markers/datasets/', cancer_code, '/mirna/', metadata$shortnames, '-raw.bam'))
}
setwd(currentwd)
</pre></div>

<p></div></div></p>

<p>We are missing <code>01810b1a-84e4-43d5-8a1e-42b132a1126f</code>, which is <code>brca-0939-TP.bam</code>. It&#39;s been removed for now.</p>

<h2>Remap reads</h2>

<p>To count both tRFs processed from mature and immature tRNAs, we need to map to both hg19 and mature tRNAs.</p>

<h3>hg19</h3>

<p>Andrew has a few settings that I&#39;ve incorporated too, with his reasoning:</p>

<ul>
<li>--very-sensitive: necessary due to mismatch caused by modified base misreads </li>
<li>--np 5 --ignore-quals: quality scores are irrelevant, and N should be scored as only slightly better than a mismatch. ignoring quals forces the mismatches to have a single score, which is necessary for precise N weighting.</li>
</ul>

<div class="chunk" id="remap"><div class="rcode"><div class="source"><pre class="knitr bash">remap() {
  currentwd=`pwd`
  cd $1
  for prefix in `ls *-raw.bam | cut -f 1,2 -d -`; do
    if [ ! -e $prefix.bam ]
    then
      if [ ! -e $prefix.fastq ]
      then
        bam2fastq --force -o $prefix-raw# `readlink $prefix-raw.bam`
        mv ${prefix}-raw_M $prefix.fastq
        rm ${prefix}-raw_1 ${prefix}-raw_2
      fi
      bowtie2 -x ~/grotto/data/hg19-tRNAgenome -k 100 --very-sensitive --ignore-quals --np 5 -p 32 -U ${prefix}.fastq -S $prefix-temp-1.sam
      samtools view -S -F 4 -h -u $prefix-temp-1.sam | samtools view -F 512 -h  - > $prefix-temp-2.sam # filter out untemp and low quality reads
      bowtie2-best-mapped.py $prefix-temp-2.sam | samtools view -S -u - | samtools sort - $prefix # auto appends .bam
    fi
    rm -f $prefix.fastq
    rm -f $prefix-temp-1.sam
    rm -f $prefix-temp-2.sam
  done
  cd $currentwd
}
remap /inside/grotto/blin/trna-markers/datasets/luad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/brca/mirna
#remap /inside/grotto/blin/trna-markers/datasets/prad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/dlbc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lihc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/coad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/paad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/tgct/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lgg/mirna
</pre></div>

<div class="output"><pre class="knitr bash">## 4367118 reads; of these:
##   4367118 (100.00%) were unpaired; of these:
##     649726 (14.88%) aligned 0 times
##     1625813 (37.23%) aligned exactly 1 time
##     2091579 (47.89%) aligned &gt;1 times
## 85.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2091579/3717392
## 2 1249610/2091579
## 3 15284343/30693751
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0003-raw_1 and luad-0003-raw_2
## Single-end reads will be in luad-0003-raw_M
## 3630276 sequences in the BAM file
## 3630276 sequences exported
## 3630276 reads; of these:
##   3630276 (100.00%) were unpaired; of these:
##     802162 (22.10%) aligned 0 times
##     1934400 (53.29%) aligned exactly 1 time
##     893714 (24.62%) aligned &gt;1 times
## 77.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 893714/2828114
## 2 674673/893714
## 3 2758819/7127763
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0004-raw_1 and luad-0004-raw_2
## Single-end reads will be in luad-0004-raw_M
## 5172654 sequences in the BAM file
## 5172654 sequences exported
## 5172654 reads; of these:
##   5172654 (100.00%) were unpaired; of these:
##     676525 (13.08%) aligned 0 times
##     1162583 (22.48%) aligned exactly 1 time
##     3333546 (64.45%) aligned &gt;1 times
## 86.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3333546/4496129
## 2 2035926/3333546
## 3 34535187/60566323
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in luad-0005-raw_1 and luad-0005-raw_2
## Single-end reads will be in luad-0005-raw_M
## 4147750 sequences in the BAM file
## 4147750 sequences exported
## 4147750 reads; of these:
##   4147750 (100.00%) were unpaired; of these:
##     783323 (18.89%) aligned 0 times
##     2170059 (52.32%) aligned exactly 1 time
##     1194368 (28.80%) aligned &gt;1 times
## 81.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1194368/3364427
## 2 920798/1194368
## 3 8599743/15565840
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0006-raw_1 and luad-0006-raw_2
## Single-end reads will be in luad-0006-raw_M
## 2987820 sequences in the BAM file
## 2987820 sequences exported
## 2987820 reads; of these:
##   2987820 (100.00%) were unpaired; of these:
##     752402 (25.18%) aligned 0 times
##     647084 (21.66%) aligned exactly 1 time
##     1588334 (53.16%) aligned &gt;1 times
## 74.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1588334/2235418
## 2 816940/1588334
## 3 9956685/19207380
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0007-raw_1 and luad-0007-raw_2
## Single-end reads will be in luad-0007-raw_M
## 1572389 sequences in the BAM file
## 1572389 sequences exported
## 1572389 reads; of these:
##   1572389 (100.00%) were unpaired; of these:
##     368625 (23.44%) aligned 0 times
##     812089 (51.65%) aligned exactly 1 time
##     391675 (24.91%) aligned &gt;1 times
## 76.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 391675/1203764
## 2 279950/391675
## 3 3618693/6688256
## This looks like paired data from lane 6.
## Output will be in luad-0008-raw_1 and luad-0008-raw_2
## Single-end reads will be in luad-0008-raw_M
## 2826392 sequences in the BAM file
## 2826392 sequences exported
## 2826392 reads; of these:
##   2826392 (100.00%) were unpaired; of these:
##     458438 (16.22%) aligned 0 times
##     759779 (26.88%) aligned exactly 1 time
##     1608175 (56.90%) aligned &gt;1 times
## 83.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1608175/2367954
## 2 972872/1608175
## 3 16343801/28774082
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0009-raw_1 and luad-0009-raw_2
## Single-end reads will be in luad-0009-raw_M
## 4579994 sequences in the BAM file
## 4579994 sequences exported
## 4579994 reads; of these:
##   4579994 (100.00%) were unpaired; of these:
##     837007 (18.28%) aligned 0 times
##     2747784 (60.00%) aligned exactly 1 time
##     995203 (21.73%) aligned &gt;1 times
## 81.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 995203/3742987
## 2 801355/995203
## 3 2269285/7178915
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0010-raw_1 and luad-0010-raw_2
## Single-end reads will be in luad-0010-raw_M
## 6275744 sequences in the BAM file
## 6275744 sequences exported
## 6275744 reads; of these:
##   6275744 (100.00%) were unpaired; of these:
##     758357 (12.08%) aligned 0 times
##     1030696 (16.42%) aligned exactly 1 time
##     4486691 (71.49%) aligned &gt;1 times
## 87.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4486691/5517387
## 2 2691393/4486691
## 3 48280922/74211967
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in luad-0011-raw_1 and luad-0011-raw_2
## Single-end reads will be in luad-0011-raw_M
## 4659716 sequences in the BAM file
## 4659716 sequences exported
## 4659716 reads; of these:
##   4659716 (100.00%) were unpaired; of these:
##     785559 (16.86%) aligned 0 times
##     2354964 (50.54%) aligned exactly 1 time
##     1519193 (32.60%) aligned &gt;1 times
## 83.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1519193/3874157
## 2 1185005/1519193
## 3 5958209/13078375
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0012-raw_1 and luad-0012-raw_2
## Single-end reads will be in luad-0012-raw_M
## 4305116 sequences in the BAM file
## 4305116 sequences exported
## 4305116 reads; of these:
##   4305116 (100.00%) were unpaired; of these:
##     1017720 (23.64%) aligned 0 times
##     577197 (13.41%) aligned exactly 1 time
##     2710199 (62.95%) aligned &gt;1 times
## 76.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2710199/3287396
## 2 1758639/2710199
## 3 26521440/49414340
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in luad-0013-raw_1 and luad-0013-raw_2
## Single-end reads will be in luad-0013-raw_M
## 12075425 sequences in the BAM file
## 12075425 sequences exported
## 12075425 reads; of these:
##   12075425 (100.00%) were unpaired; of these:
##     3028417 (25.08%) aligned 0 times
##     5808553 (48.10%) aligned exactly 1 time
##     3238455 (26.82%) aligned &gt;1 times
## 74.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3238455/9047008
## 2 2290019/3238455
## 3 29636009/58542037
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0014-raw_1 and luad-0014-raw_2
## Single-end reads will be in luad-0014-raw_M
## 9460121 sequences in the BAM file
## 9460121 sequences exported
## 9460121 reads; of these:
##   9460121 (100.00%) were unpaired; of these:
##     1679157 (17.75%) aligned 0 times
##     4646782 (49.12%) aligned exactly 1 time
##     3134182 (33.13%) aligned &gt;1 times
## 82.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3134182/7780964
## 2 2252076/3134182
## 3 14261934/29947298
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0015-raw_1 and luad-0015-raw_2
## Single-end reads will be in luad-0015-raw_M
## 8786613 sequences in the BAM file
## 8786613 sequences exported
## 8786613 reads; of these:
##   8786613 (100.00%) were unpaired; of these:
##     1530838 (17.42%) aligned 0 times
##     5334036 (60.71%) aligned exactly 1 time
##     1921739 (21.87%) aligned &gt;1 times
## 82.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1921739/7255775
## 2 1356548/1921739
## 3 8929770/22157558
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0016-raw_1 and luad-0016-raw_2
## Single-end reads will be in luad-0016-raw_M
## 13016514 sequences in the BAM file
## 13016514 sequences exported
## 13016514 reads; of these:
##   13016514 (100.00%) were unpaired; of these:
##     3759296 (28.88%) aligned 0 times
##     6151100 (47.26%) aligned exactly 1 time
##     3106118 (23.86%) aligned &gt;1 times
## 71.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3106118/9257218
## 2 2122474/3106118
## 3 8745060/22834721
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0017-raw_1 and luad-0017-raw_2
## Single-end reads will be in luad-0017-raw_M
## 10566238 sequences in the BAM file
## 10566238 sequences exported
## 10566238 reads; of these:
##   10566238 (100.00%) were unpaired; of these:
##     4390224 (41.55%) aligned 0 times
##     3780010 (35.77%) aligned exactly 1 time
##     2396004 (22.68%) aligned &gt;1 times
## 58.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2396004/6176014
## 2 1476888/2396004
## 3 19382490/38720534
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0018-raw_1 and luad-0018-raw_2
## Single-end reads will be in luad-0018-raw_M
## 14964335 sequences in the BAM file
## 14964335 sequences exported
## 14964335 reads; of these:
##   14964335 (100.00%) were unpaired; of these:
##     3254413 (21.75%) aligned 0 times
##     7116507 (47.56%) aligned exactly 1 time
##     4593415 (30.70%) aligned &gt;1 times
## 78.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4593415/11709922
## 2 3483223/4593415
## 3 19217376/38166117
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0019-raw_1 and luad-0019-raw_2
## Single-end reads will be in luad-0019-raw_M
## 2449877 sequences in the BAM file
## 2449877 sequences exported
## 2449877 reads; of these:
##   2449877 (100.00%) were unpaired; of these:
##     526026 (21.47%) aligned 0 times
##     1216271 (49.65%) aligned exactly 1 time
##     707580 (28.88%) aligned &gt;1 times
## 78.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 707580/1923851
## 2 525536/707580
## 3 6426220/11060824
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0020-raw_1 and luad-0020-raw_2
## Single-end reads will be in luad-0020-raw_M
## 3108521 sequences in the BAM file
## 3108521 sequences exported
## 3108521 reads; of these:
##   3108521 (100.00%) were unpaired; of these:
##     565837 (18.20%) aligned 0 times
##     725391 (23.34%) aligned exactly 1 time
##     1817293 (58.46%) aligned &gt;1 times
## 81.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1817293/2542684
## 2 1087645/1817293
## 3 17217158/31021705
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0021-raw_1 and luad-0021-raw_2
## Single-end reads will be in luad-0021-raw_M
## 6510804 sequences in the BAM file
## 6510804 sequences exported
## 6510804 reads; of these:
##   6510804 (100.00%) were unpaired; of these:
##     1631854 (25.06%) aligned 0 times
##     3680344 (56.53%) aligned exactly 1 time
##     1198606 (18.41%) aligned &gt;1 times
## 74.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1198606/4878950
## 2 840617/1198606
## 3 6064735/14507461
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0022-raw_1 and luad-0022-raw_2
## Single-end reads will be in luad-0022-raw_M
## 11360856 sequences in the BAM file
## 11360856 sequences exported
## 11360856 reads; of these:
##   11360856 (100.00%) were unpaired; of these:
##     3140225 (27.64%) aligned 0 times
##     5867363 (51.65%) aligned exactly 1 time
##     2353268 (20.71%) aligned &gt;1 times
## 72.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2353268/8220631
## 2 1588882/2353268
## 3 6107873/17687130
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0023-raw_1 and luad-0023-raw_2
## Single-end reads will be in luad-0023-raw_M
## 10672335 sequences in the BAM file
## 10672335 sequences exported
## 10672335 reads; of these:
##   10672335 (100.00%) were unpaired; of these:
##     3064379 (28.71%) aligned 0 times
##     4804234 (45.02%) aligned exactly 1 time
##     2803722 (26.27%) aligned &gt;1 times
## 71.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2803722/7607956
## 2 2119976/2803722
## 3 13824917/27605475
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0024-raw_1 and luad-0024-raw_2
## Single-end reads will be in luad-0024-raw_M
## 18797778 sequences in the BAM file
## 18797778 sequences exported
## 18797778 reads; of these:
##   18797778 (100.00%) were unpaired; of these:
##     5857067 (31.16%) aligned 0 times
##     9195252 (48.92%) aligned exactly 1 time
##     3745459 (19.93%) aligned &gt;1 times
## 68.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3745459/12940711
## 2 2477039/3745459
## 3 11395522/30619737
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in luad-0025-raw_1 and luad-0025-raw_2
## Single-end reads will be in luad-0025-raw_M
## 8825864 sequences in the BAM file
## 8825864 sequences exported
## 8825864 reads; of these:
##   8825864 (100.00%) were unpaired; of these:
##     2625795 (29.75%) aligned 0 times
##     4310698 (48.84%) aligned exactly 1 time
##     1889371 (21.41%) aligned &gt;1 times
## 70.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1889371/6200069
## 2 1076385/1889371
## 3 7460835/19136189
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0026-raw_1 and luad-0026-raw_2
## Single-end reads will be in luad-0026-raw_M
## 14870848 sequences in the BAM file
## 14870848 sequences exported
## 14870848 reads; of these:
##   14870848 (100.00%) were unpaired; of these:
##     4929427 (33.15%) aligned 0 times
##     6509458 (43.77%) aligned exactly 1 time
##     3431963 (23.08%) aligned &gt;1 times
## 66.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3431963/9941421
## 2 2459800/3431963
## 3 9842293/25522104
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0027-raw_1 and luad-0027-raw_2
## Single-end reads will be in luad-0027-raw_M
## 10831472 sequences in the BAM file
## 10831472 sequences exported
## 10831472 reads; of these:
##   10831472 (100.00%) were unpaired; of these:
##     2014732 (18.60%) aligned 0 times
##     6588367 (60.83%) aligned exactly 1 time
##     2228373 (20.57%) aligned &gt;1 times
## 81.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2228373/8816740
## 2 1595973/2228373
## 3 12661053/28175678
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0028-raw_1 and luad-0028-raw_2
## Single-end reads will be in luad-0028-raw_M
## 6834455 sequences in the BAM file
## 6834455 sequences exported
## 6834455 reads; of these:
##   6834455 (100.00%) were unpaired; of these:
##     1553149 (22.73%) aligned 0 times
##     3165691 (46.32%) aligned exactly 1 time
##     2115615 (30.96%) aligned &gt;1 times
## 77.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2115615/5281306
## 2 1652628/2115615
## 3 9099208/17666787
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0029-raw_1 and luad-0029-raw_2
## Single-end reads will be in luad-0029-raw_M
## 12622721 sequences in the BAM file
## 12622721 sequences exported
## 12622721 reads; of these:
##   12622721 (100.00%) were unpaired; of these:
##     4974488 (39.41%) aligned 0 times
##     5315738 (42.11%) aligned exactly 1 time
##     2332495 (18.48%) aligned &gt;1 times
## 60.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2332495/7648233
## 2 1608116/2332495
## 3 10208641/22949426
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0030-raw_1 and luad-0030-raw_2
## Single-end reads will be in luad-0030-raw_M
## 13372963 sequences in the BAM file
## 13372963 sequences exported
## 13372963 reads; of these:
##   13372963 (100.00%) were unpaired; of these:
##     4148266 (31.02%) aligned 0 times
##     6513458 (48.71%) aligned exactly 1 time
##     2711239 (20.27%) aligned &gt;1 times
## 68.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2711239/9224697
## 2 1848146/2711239
## 3 7864899/21237377
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0031-raw_1 and luad-0031-raw_2
## Single-end reads will be in luad-0031-raw_M
## 10237074 sequences in the BAM file
## 10237074 sequences exported
## 10237074 reads; of these:
##   10237074 (100.00%) were unpaired; of these:
##     1630504 (15.93%) aligned 0 times
##     6128975 (59.87%) aligned exactly 1 time
##     2477595 (24.20%) aligned &gt;1 times
## 84.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2477595/8606570
## 2 1758395/2477595
## 3 11169959/26638273
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0032-raw_1 and luad-0032-raw_2
## Single-end reads will be in luad-0032-raw_M
## 12495792 sequences in the BAM file
## 12495792 sequences exported
## 12495792 reads; of these:
##   12495792 (100.00%) were unpaired; of these:
##     3033006 (24.27%) aligned 0 times
##     5785566 (46.30%) aligned exactly 1 time
##     3677220 (29.43%) aligned &gt;1 times
## 75.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3677220/9462786
## 2 2852823/3677220
## 3 17442062/33887563
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0033-raw_1 and luad-0033-raw_2
## Single-end reads will be in luad-0033-raw_M
## 8184971 sequences in the BAM file
## 8184971 sequences exported
## 8184971 reads; of these:
##   8184971 (100.00%) were unpaired; of these:
##     2906951 (35.52%) aligned 0 times
##     3716513 (45.41%) aligned exactly 1 time
##     1561507 (19.08%) aligned &gt;1 times
## 64.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1561507/5278020
## 2 875012/1561507
## 3 10761077/24226458
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0034-raw_1 and luad-0034-raw_2
## Single-end reads will be in luad-0034-raw_M
## 14643300 sequences in the BAM file
## 14643300 sequences exported
## 14643300 reads; of these:
##   14643300 (100.00%) were unpaired; of these:
##     4108284 (28.06%) aligned 0 times
##     6932828 (47.34%) aligned exactly 1 time
##     3602188 (24.60%) aligned &gt;1 times
## 71.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3602188/10535016
## 2 2655513/3602188
## 3 9964133/25666507
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0035-raw_1 and luad-0035-raw_2
## Single-end reads will be in luad-0035-raw_M
## 5712529 sequences in the BAM file
## 5712529 sequences exported
## 5712529 reads; of these:
##   5712529 (100.00%) were unpaired; of these:
##     1960170 (34.31%) aligned 0 times
##     2777504 (48.62%) aligned exactly 1 time
##     974855 (17.07%) aligned &gt;1 times
## 65.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 974855/3752359
## 2 623586/974855
## 3 4872436/11801252
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0036-raw_1 and luad-0036-raw_2
## Single-end reads will be in luad-0036-raw_M
## 10498574 sequences in the BAM file
## 10498574 sequences exported
## 10498574 reads; of these:
##   10498574 (100.00%) were unpaired; of these:
##     3750797 (35.73%) aligned 0 times
##     5004862 (47.67%) aligned exactly 1 time
##     1742915 (16.60%) aligned &gt;1 times
## 64.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1742915/6747777
## 2 1194507/1742915
## 3 5963407/16017670
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0037-raw_1 and luad-0037-raw_2
## Single-end reads will be in luad-0037-raw_M
## 21391967 sequences in the BAM file
## 21391967 sequences exported
## 21391967 reads; of these:
##   21391967 (100.00%) were unpaired; of these:
##     3508864 (16.40%) aligned 0 times
##     12750799 (59.61%) aligned exactly 1 time
##     5132304 (23.99%) aligned &gt;1 times
## 83.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5132304/17883103
## 2 3930429/5132304
## 3 15741877/41518832
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in luad-0038-raw_1 and luad-0038-raw_2
## Single-end reads will be in luad-0038-raw_M
## 13356999 sequences in the BAM file
## 13356999 sequences exported
## 13356999 reads; of these:
##   13356999 (100.00%) were unpaired; of these:
##     3079983 (23.06%) aligned 0 times
##     6489248 (48.58%) aligned exactly 1 time
##     3787768 (28.36%) aligned &gt;1 times
## 76.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3787768/10277016
## 2 2849863/3787768
## 3 18234304/35031117
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0039-raw_1 and luad-0039-raw_2
## Single-end reads will be in luad-0039-raw_M
## 8563613 sequences in the BAM file
## 8563613 sequences exported
## 8563613 reads; of these:
##   8563613 (100.00%) were unpaired; of these:
##     2501271 (29.21%) aligned 0 times
##     4119343 (48.10%) aligned exactly 1 time
##     1942999 (22.69%) aligned &gt;1 times
## 70.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1942999/6062342
## 2 1412326/1942999
## 3 6115789/15347683
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0040-raw_1 and luad-0040-raw_2
## Single-end reads will be in luad-0040-raw_M
## 18504194 sequences in the BAM file
## 18504194 sequences exported
## 18504194 reads; of these:
##   18504194 (100.00%) were unpaired; of these:
##     5756079 (31.11%) aligned 0 times
##     8373418 (45.25%) aligned exactly 1 time
##     4374697 (23.64%) aligned &gt;1 times
## 68.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4374697/12748115
## 2 3150186/4374697
## 3 12007497/31026482
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0041-raw_1 and luad-0041-raw_2
## Single-end reads will be in luad-0041-raw_M
## 12197674 sequences in the BAM file
## 12197674 sequences exported
## 12197674 reads; of these:
##   12197674 (100.00%) were unpaired; of these:
##     3099567 (25.41%) aligned 0 times
##     6483863 (53.16%) aligned exactly 1 time
##     2614244 (21.43%) aligned &gt;1 times
## 74.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2614244/9098107
## 2 1963987/2614244
## 3 9977413/23637646
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0042-raw_1 and luad-0042-raw_2
## Single-end reads will be in luad-0042-raw_M
## 21332604 sequences in the BAM file
## 21332604 sequences exported
## 21332604 reads; of these:
##   21332604 (100.00%) were unpaired; of these:
##     4192195 (19.65%) aligned 0 times
##     12163177 (57.02%) aligned exactly 1 time
##     4977232 (23.33%) aligned &gt;1 times
## 80.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4977232/17140409
## 2 3513354/4977232
## 3 14488922/38592603
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in luad-0043-raw_1 and luad-0043-raw_2
## Single-end reads will be in luad-0043-raw_M
## 11894270 sequences in the BAM file
## 11894270 sequences exported
## 11894270 reads; of these:
##   11894270 (100.00%) were unpaired; of these:
##     3174721 (26.69%) aligned 0 times
##     5413262 (45.51%) aligned exactly 1 time
##     3306287 (27.80%) aligned &gt;1 times
## 73.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3306287/8719549
## 2 2322497/3306287
## 3 24180613/44772136
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0044-raw_1 and luad-0044-raw_2
## Single-end reads will be in luad-0044-raw_M
## 10599902 sequences in the BAM file
## 10599902 sequences exported
## 10599902 reads; of these:
##   10599902 (100.00%) were unpaired; of these:
##     2464492 (23.25%) aligned 0 times
##     5075356 (47.88%) aligned exactly 1 time
##     3060054 (28.87%) aligned &gt;1 times
## 76.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3060054/8135410
## 2 2351241/3060054
## 3 14647204/27729879
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0045-raw_1 and luad-0045-raw_2
## Single-end reads will be in luad-0045-raw_M
## 10554594 sequences in the BAM file
## 10554594 sequences exported
## 10554594 reads; of these:
##   10554594 (100.00%) were unpaired; of these:
##     3108362 (29.45%) aligned 0 times
##     4870588 (46.15%) aligned exactly 1 time
##     2575644 (24.40%) aligned &gt;1 times
## 70.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2575644/7446232
## 2 1665926/2575644
## 3 14214577/29166988
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0046-raw_1 and luad-0046-raw_2
## Single-end reads will be in luad-0046-raw_M
## 14899726 sequences in the BAM file
## 14899726 sequences exported
## 14899726 reads; of these:
##   14899726 (100.00%) were unpaired; of these:
##     3096150 (20.78%) aligned 0 times
##     7617884 (51.13%) aligned exactly 1 time
##     4185692 (28.09%) aligned &gt;1 times
## 79.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4185692/11803576
## 2 2982735/4185692
## 3 22079871/42304275
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0047-raw_1 and luad-0047-raw_2
## Single-end reads will be in luad-0047-raw_M
## 6823616 sequences in the BAM file
## 6823616 sequences exported
## 6823616 reads; of these:
##   6823616 (100.00%) were unpaired; of these:
##     1863838 (27.31%) aligned 0 times
##     3046980 (44.65%) aligned exactly 1 time
##     1912798 (28.03%) aligned &gt;1 times
## 72.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1912798/4959778
## 2 1389575/1912798
## 3 14965246/27818684
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0048-raw_1 and luad-0048-raw_2
## Single-end reads will be in luad-0048-raw_M
## 11455942 sequences in the BAM file
## 11455942 sequences exported
## 11455942 reads; of these:
##   11455942 (100.00%) were unpaired; of these:
##     3722659 (32.50%) aligned 0 times
##     5371961 (46.89%) aligned exactly 1 time
##     2361322 (20.61%) aligned &gt;1 times
## 67.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2361322/7733283
## 2 1665011/2361322
## 3 7043696/18566344
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0049-raw_1 and luad-0049-raw_2
## Single-end reads will be in luad-0049-raw_M
## 8573852 sequences in the BAM file
## 8573852 sequences exported
## 8573852 reads; of these:
##   8573852 (100.00%) were unpaired; of these:
##     2211421 (25.79%) aligned 0 times
##     4095798 (47.77%) aligned exactly 1 time
##     2266633 (26.44%) aligned &gt;1 times
## 74.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2266633/6362431
## 2 1556892/2266633
## 3 18439684/34097452
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0050-raw_1 and luad-0050-raw_2
## Single-end reads will be in luad-0050-raw_M
## 5646442 sequences in the BAM file
## 5646442 sequences exported
## 5646442 reads; of these:
##   5646442 (100.00%) were unpaired; of these:
##     1921184 (34.02%) aligned 0 times
##     2558102 (45.30%) aligned exactly 1 time
##     1167156 (20.67%) aligned &gt;1 times
## 65.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1167156/3725258
## 2 786084/1167156
## 3 3875546/10015314
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in luad-0051-raw_1 and luad-0051-raw_2
## Single-end reads will be in luad-0051-raw_M
## 11862796 sequences in the BAM file
## 11862796 sequences exported
## 11862796 reads; of these:
##   11862796 (100.00%) were unpaired; of these:
##     4016566 (33.86%) aligned 0 times
##     5382466 (45.37%) aligned exactly 1 time
##     2463764 (20.77%) aligned &gt;1 times
## 66.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2463764/7846230
## 2 1513348/2463764
## 3 9409993/23998564
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0052-raw_1 and luad-0052-raw_2
## Single-end reads will be in luad-0052-raw_M
## 12101059 sequences in the BAM file
## 12101059 sequences exported
## 12101059 reads; of these:
##   12101059 (100.00%) were unpaired; of these:
##     4118509 (34.03%) aligned 0 times
##     5513087 (45.56%) aligned exactly 1 time
##     2469463 (20.41%) aligned &gt;1 times
## 65.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2469463/7982550
## 2 1796605/2469463
## 3 7005095/18659699
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0053-raw_1 and luad-0053-raw_2
## Single-end reads will be in luad-0053-raw_M
## 6314057 sequences in the BAM file
## 6314057 sequences exported
## 6314057 reads; of these:
##   6314057 (100.00%) were unpaired; of these:
##     1688508 (26.74%) aligned 0 times
##     2485713 (39.37%) aligned exactly 1 time
##     2139836 (33.89%) aligned &gt;1 times
## 73.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2139836/4625549
## 2 1530305/2139836
## 3 12132771/21848450
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0054-raw_1 and luad-0054-raw_2
## Single-end reads will be in luad-0054-raw_M
## 10121377 sequences in the BAM file
## 10121377 sequences exported
## 10121377 reads; of these:
##   10121377 (100.00%) were unpaired; of these:
##     2067049 (20.42%) aligned 0 times
##     4752045 (46.95%) aligned exactly 1 time
##     3302283 (32.63%) aligned &gt;1 times
## 79.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3302283/8054328
## 2 2375081/3302283
## 3 15087218/28862722
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0055-raw_1 and luad-0055-raw_2
## Single-end reads will be in luad-0055-raw_M
## 7200575 sequences in the BAM file
## 7200575 sequences exported
## 7200575 reads; of these:
##   7200575 (100.00%) were unpaired; of these:
##     2174732 (30.20%) aligned 0 times
##     3588350 (49.83%) aligned exactly 1 time
##     1437493 (19.96%) aligned &gt;1 times
## 69.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1437493/5025843
## 2 997762/1437493
## 3 7347669/17490641
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0056-raw_1 and luad-0056-raw_2
## Single-end reads will be in luad-0056-raw_M
## 12856645 sequences in the BAM file
## 12856645 sequences exported
## 12856645 reads; of these:
##   12856645 (100.00%) were unpaired; of these:
##     3861598 (30.04%) aligned 0 times
##     6235583 (48.50%) aligned exactly 1 time
##     2759464 (21.46%) aligned &gt;1 times
## 69.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2759464/8995047
## 2 1937526/2759464
## 3 7923470/21167707
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0057-raw_1 and luad-0057-raw_2
## Single-end reads will be in luad-0057-raw_M
## 9076818 sequences in the BAM file
## 9076818 sequences exported
## 9076818 reads; of these:
##   9076818 (100.00%) were unpaired; of these:
##     2512844 (27.68%) aligned 0 times
##     3608339 (39.75%) aligned exactly 1 time
##     2955635 (32.56%) aligned &gt;1 times
## 72.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2955635/6563974
## 2 2064799/2955635
## 3 23861195/42990202
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0058-raw_1 and luad-0058-raw_2
## Single-end reads will be in luad-0058-raw_M
## 11096443 sequences in the BAM file
## 11096443 sequences exported
## 11096443 reads; of these:
##   11096443 (100.00%) were unpaired; of these:
##     2574951 (23.21%) aligned 0 times
##     5174724 (46.63%) aligned exactly 1 time
##     3346768 (30.16%) aligned &gt;1 times
## 76.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3346768/8521492
## 2 2479206/3346768
## 3 16398908/31537232
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0059-raw_1 and luad-0059-raw_2
## Single-end reads will be in luad-0059-raw_M
## 12949406 sequences in the BAM file
## 12949406 sequences exported
## 12949406 reads; of these:
##   12949406 (100.00%) were unpaired; of these:
##     4370257 (33.75%) aligned 0 times
##     5982654 (46.20%) aligned exactly 1 time
##     2596495 (20.05%) aligned &gt;1 times
## 66.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2596495/8579149
## 2 1583791/2596495
## 3 13170353/31691973
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0060-raw_1 and luad-0060-raw_2
## Single-end reads will be in luad-0060-raw_M
## 21951018 sequences in the BAM file
## 21951018 sequences exported
## 21951018 reads; of these:
##   21951018 (100.00%) were unpaired; of these:
##     6824780 (31.09%) aligned 0 times
##     11168967 (50.88%) aligned exactly 1 time
##     3957271 (18.03%) aligned &gt;1 times
## 68.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3957271/15126238
## 2 2686323/3957271
## 3 9770327/30379229
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0061-raw_1 and luad-0061-raw_2
## Single-end reads will be in luad-0061-raw_M
## 10933762 sequences in the BAM file
## 10933762 sequences exported
## 10933762 reads; of these:
##   10933762 (100.00%) were unpaired; of these:
##     4574839 (41.84%) aligned 0 times
##     3810127 (34.85%) aligned exactly 1 time
##     2548796 (23.31%) aligned &gt;1 times
## 58.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2548796/6358923
## 2 1650400/2548796
## 3 22233396/43193690
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0062-raw_1 and luad-0062-raw_2
## Single-end reads will be in luad-0062-raw_M
## 13624532 sequences in the BAM file
## 13624532 sequences exported
## 13624532 reads; of these:
##   13624532 (100.00%) were unpaired; of these:
##     4550047 (33.40%) aligned 0 times
##     6187018 (45.41%) aligned exactly 1 time
##     2887467 (21.19%) aligned &gt;1 times
## 66.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2887467/9074485
## 2 1927150/2887467
## 3 8357156/22215091
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0063-raw_1 and luad-0063-raw_2
## Single-end reads will be in luad-0063-raw_M
## 15957300 sequences in the BAM file
## 15957300 sequences exported
## 15957300 reads; of these:
##   15957300 (100.00%) were unpaired; of these:
##     5268046 (33.01%) aligned 0 times
##     8219448 (51.51%) aligned exactly 1 time
##     2469806 (15.48%) aligned &gt;1 times
## 66.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2469806/10689254
## 2 1459043/2469806
## 3 10902899/28326950
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0064-raw_1 and luad-0064-raw_2
## Single-end reads will be in luad-0064-raw_M
## 12954492 sequences in the BAM file
## 12954492 sequences exported
## 12954492 reads; of these:
##   12954492 (100.00%) were unpaired; of these:
##     4186386 (32.32%) aligned 0 times
##     5399769 (41.68%) aligned exactly 1 time
##     3368337 (26.00%) aligned &gt;1 times
## 67.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3368337/8768106
## 2 2366551/3368337
## 3 9321204/23266839
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0065-raw_1 and luad-0065-raw_2
## Single-end reads will be in luad-0065-raw_M
## 5719921 sequences in the BAM file
## 5719921 sequences exported
## 5719921 reads; of these:
##   5719921 (100.00%) were unpaired; of these:
##     1774961 (31.03%) aligned 0 times
##     2774086 (48.50%) aligned exactly 1 time
##     1170874 (20.47%) aligned &gt;1 times
## 68.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1170874/3944960
## 2 817841/1170874
## 3 4679746/11066433
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0066-raw_1 and luad-0066-raw_2
## Single-end reads will be in luad-0066-raw_M
## 11254396 sequences in the BAM file
## 11254396 sequences exported
## 11254396 reads; of these:
##   11254396 (100.00%) were unpaired; of these:
##     3743650 (33.26%) aligned 0 times
##     5087239 (45.20%) aligned exactly 1 time
##     2423507 (21.53%) aligned &gt;1 times
## 66.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2423507/7510746
## 2 1636118/2423507
## 3 7329286/18919442
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0067-raw_1 and luad-0067-raw_2
## Single-end reads will be in luad-0067-raw_M
## 11426070 sequences in the BAM file
## 11426070 sequences exported
## 11426070 reads; of these:
##   11426070 (100.00%) were unpaired; of these:
##     4957582 (43.39%) aligned 0 times
##     4166172 (36.46%) aligned exactly 1 time
##     2302316 (20.15%) aligned &gt;1 times
## 56.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2302316/6468488
## 2 1526556/2302316
## 3 17570769/36331907
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0068-raw_1 and luad-0068-raw_2
## Single-end reads will be in luad-0068-raw_M
## 18453534 sequences in the BAM file
## 18453534 sequences exported
## 18453534 reads; of these:
##   18453534 (100.00%) were unpaired; of these:
##     4123943 (22.35%) aligned 0 times
##     8530980 (46.23%) aligned exactly 1 time
##     5798611 (31.42%) aligned &gt;1 times
## 77.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5798611/14329591
## 2 4336397/5798611
## 3 27944115/54611099
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in luad-0069-raw_1 and luad-0069-raw_2
## Single-end reads will be in luad-0069-raw_M
## 12746656 sequences in the BAM file
## 12746656 sequences exported
## 12746656 reads; of these:
##   12746656 (100.00%) were unpaired; of these:
##     4023663 (31.57%) aligned 0 times
##     6664833 (52.29%) aligned exactly 1 time
##     2058160 (16.15%) aligned &gt;1 times
## 68.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2058160/8722993
## 2 1227481/2058160
## 3 10010551/25891884
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0070-raw_1 and luad-0070-raw_2
## Single-end reads will be in luad-0070-raw_M
## 15977046 sequences in the BAM file
## 15977046 sequences exported
## 15977046 reads; of these:
##   15977046 (100.00%) were unpaired; of these:
##     5287076 (33.09%) aligned 0 times
##     7020650 (43.94%) aligned exactly 1 time
##     3669320 (22.97%) aligned &gt;1 times
## 66.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3669320/10689970
## 2 2624594/3669320
## 3 9593462/25621980
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0071-raw_1 and luad-0071-raw_2
## Single-end reads will be in luad-0071-raw_M
## 9579320 sequences in the BAM file
## 9579320 sequences exported
## 9579320 reads; of these:
##   9579320 (100.00%) were unpaired; of these:
##     2630600 (27.46%) aligned 0 times
##     4774053 (49.84%) aligned exactly 1 time
##     2174667 (22.70%) aligned &gt;1 times
## 72.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2174667/6948720
## 2 1487597/2174667
## 3 9396402/20874338
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0072-raw_1 and luad-0072-raw_2
## Single-end reads will be in luad-0072-raw_M
## 13255633 sequences in the BAM file
## 13255633 sequences exported
## 13255633 reads; of these:
##   13255633 (100.00%) were unpaired; of these:
##     2697164 (20.35%) aligned 0 times
##     6676647 (50.37%) aligned exactly 1 time
##     3881822 (29.28%) aligned &gt;1 times
## 79.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3881822/10558469
## 2 2791233/3881822
## 3 11589431/27798272
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0073-raw_1 and luad-0073-raw_2
## Single-end reads will be in luad-0073-raw_M
## 12567241 sequences in the BAM file
## 12567241 sequences exported
## 12567241 reads; of these:
##   12567241 (100.00%) were unpaired; of these:
##     4986219 (39.68%) aligned 0 times
##     5386241 (42.86%) aligned exactly 1 time
##     2194781 (17.46%) aligned &gt;1 times
## 60.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2194781/7581022
## 2 1439328/2194781
## 3 14837940/32310110
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0074-raw_1 and luad-0074-raw_2
## Single-end reads will be in luad-0074-raw_M
## 5660175 sequences in the BAM file
## 5660175 sequences exported
## 5660175 reads; of these:
##   5660175 (100.00%) were unpaired; of these:
##     1884884 (33.30%) aligned 0 times
##     2663767 (47.06%) aligned exactly 1 time
##     1111524 (19.64%) aligned &gt;1 times
## 66.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1111524/3775291
## 2 759183/1111524
## 3 4215081/10609208
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0075-raw_1 and luad-0075-raw_2
## Single-end reads will be in luad-0075-raw_M
## 15443258 sequences in the BAM file
## 15443258 sequences exported
## 15443258 reads; of these:
##   15443258 (100.00%) were unpaired; of these:
##     4244130 (27.48%) aligned 0 times
##     7283876 (47.17%) aligned exactly 1 time
##     3915252 (25.35%) aligned &gt;1 times
## 72.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3915252/11199128
## 2 2740884/3915252
## 3 27890732/51647554
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in luad-0076-raw_1 and luad-0076-raw_2
## Single-end reads will be in luad-0076-raw_M
## 18905429 sequences in the BAM file
## 18905429 sequences exported
## 18905429 reads; of these:
##   18905429 (100.00%) were unpaired; of these:
##     5670781 (30.00%) aligned 0 times
##     10019022 (53.00%) aligned exactly 1 time
##     3215626 (17.01%) aligned &gt;1 times
## 70.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3215626/13234648
## 2 1975984/3215626
## 3 8827405/27142319
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0077-raw_1 and luad-0077-raw_2
## Single-end reads will be in luad-0077-raw_M
## 6293096 sequences in the BAM file
## 6293096 sequences exported
## 6293096 reads; of these:
##   6293096 (100.00%) were unpaired; of these:
##     1416335 (22.51%) aligned 0 times
##     3239026 (51.47%) aligned exactly 1 time
##     1637735 (26.02%) aligned &gt;1 times
## 77.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1637735/4876761
## 2 1161058/1637735
## 3 5053544/12624606
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0078-raw_1 and luad-0078-raw_2
## Single-end reads will be in luad-0078-raw_M
## 14470985 sequences in the BAM file
## 14470985 sequences exported
## 14470985 reads; of these:
##   14470985 (100.00%) were unpaired; of these:
##     4362177 (30.14%) aligned 0 times
##     7403758 (51.16%) aligned exactly 1 time
##     2705050 (18.69%) aligned &gt;1 times
## 69.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2705050/10108808
## 2 1900710/2705050
## 3 8168740/22760014
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0079-raw_1 and luad-0079-raw_2
## Single-end reads will be in luad-0079-raw_M
## 8961415 sequences in the BAM file
## 8961415 sequences exported
## 8961415 reads; of these:
##   8961415 (100.00%) were unpaired; of these:
##     2778025 (31.00%) aligned 0 times
##     4387848 (48.96%) aligned exactly 1 time
##     1795542 (20.04%) aligned &gt;1 times
## 69.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1795542/6183390
## 2 831704/1795542
## 3 7384517/18932218
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0080-raw_1 and luad-0080-raw_2
## Single-end reads will be in luad-0080-raw_M
## 15103315 sequences in the BAM file
## 15103315 sequences exported
## 15103315 reads; of these:
##   15103315 (100.00%) were unpaired; of these:
##     5063182 (33.52%) aligned 0 times
##     6809672 (45.09%) aligned exactly 1 time
##     3230461 (21.39%) aligned &gt;1 times
## 66.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3230461/10040133
## 2 2307982/3230461
## 3 9726599/25210631
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0081-raw_1 and luad-0081-raw_2
## Single-end reads will be in luad-0081-raw_M
## 9937787 sequences in the BAM file
## 9937787 sequences exported
## 9937787 reads; of these:
##   9937787 (100.00%) were unpaired; of these:
##     3417881 (34.39%) aligned 0 times
##     4410974 (44.39%) aligned exactly 1 time
##     2108932 (21.22%) aligned &gt;1 times
## 65.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2108932/6519906
## 2 1569872/2108932
## 3 11975620/23483562
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0082-raw_1 and luad-0082-raw_2
## Single-end reads will be in luad-0082-raw_M
## 12959563 sequences in the BAM file
## 12959563 sequences exported
## 12959563 reads; of these:
##   12959563 (100.00%) were unpaired; of these:
##     2833249 (21.86%) aligned 0 times
##     6181648 (47.70%) aligned exactly 1 time
##     3944666 (30.44%) aligned &gt;1 times
## 78.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3944666/10126314
## 2 3040386/3944666
## 3 11495420/27534797
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0083-raw_1 and luad-0083-raw_2
## Single-end reads will be in luad-0083-raw_M
## 6721654 sequences in the BAM file
## 6721654 sequences exported
## 6721654 reads; of these:
##   6721654 (100.00%) were unpaired; of these:
##     2036733 (30.30%) aligned 0 times
##     3172910 (47.20%) aligned exactly 1 time
##     1512011 (22.49%) aligned &gt;1 times
## 69.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1512011/4684921
## 2 942351/1512011
## 3 8499512/18885420
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0084-raw_1 and luad-0084-raw_2
## Single-end reads will be in luad-0084-raw_M
## 13085904 sequences in the BAM file
## 13085904 sequences exported
## 13085904 reads; of these:
##   13085904 (100.00%) were unpaired; of these:
##     3836437 (29.32%) aligned 0 times
##     6083483 (46.49%) aligned exactly 1 time
##     3165984 (24.19%) aligned &gt;1 times
## 70.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3165984/9249467
## 2 2309059/3165984
## 3 8999247/22894547
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0085-raw_1 and luad-0085-raw_2
## Single-end reads will be in luad-0085-raw_M
## 5147963 sequences in the BAM file
## 5147963 sequences exported
## 5147963 reads; of these:
##   5147963 (100.00%) were unpaired; of these:
##     1203641 (23.38%) aligned 0 times
##     2815638 (54.69%) aligned exactly 1 time
##     1128684 (21.92%) aligned &gt;1 times
## 76.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1128684/3944322
## 2 765480/1128684
## 3 7301622/16038467
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0086-raw_1 and luad-0086-raw_2
## Single-end reads will be in luad-0086-raw_M
## 10104615 sequences in the BAM file
## 10104615 sequences exported
## 10104615 reads; of these:
##   10104615 (100.00%) were unpaired; of these:
##     2223532 (22.01%) aligned 0 times
##     5534641 (54.77%) aligned exactly 1 time
##     2346442 (23.22%) aligned &gt;1 times
## 77.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2346442/7881083
## 2 1699904/2346442
## 3 8255738/20280934
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0087-raw_1 and luad-0087-raw_2
## Single-end reads will be in luad-0087-raw_M
## 7930995 sequences in the BAM file
## 7930995 sequences exported
## 7930995 reads; of these:
##   7930995 (100.00%) were unpaired; of these:
##     2330767 (29.39%) aligned 0 times
##     4191065 (52.84%) aligned exactly 1 time
##     1409163 (17.77%) aligned &gt;1 times
## 70.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1409163/5600228
## 2 976536/1409163
## 3 4529920/12466404
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0088-raw_1 and luad-0088-raw_2
## Single-end reads will be in luad-0088-raw_M
## 12635452 sequences in the BAM file
## 12635452 sequences exported
## 12635452 reads; of these:
##   12635452 (100.00%) were unpaired; of these:
##     3509148 (27.77%) aligned 0 times
##     5852935 (46.32%) aligned exactly 1 time
##     3273369 (25.91%) aligned &gt;1 times
## 72.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3273369/9126304
## 2 2260583/3273369
## 3 8864511/22683018
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0089-raw_1 and luad-0089-raw_2
## Single-end reads will be in luad-0089-raw_M
## 16729337 sequences in the BAM file
## 16729337 sequences exported
## 16729337 reads; of these:
##   16729337 (100.00%) were unpaired; of these:
##     6347326 (37.94%) aligned 0 times
##     7416562 (44.33%) aligned exactly 1 time
##     2965449 (17.73%) aligned &gt;1 times
## 62.06% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2965449/10382011
## 2 1993953/2965449
## 3 17472435/37900157
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0090-raw_1 and luad-0090-raw_2
## Single-end reads will be in luad-0090-raw_M
## 10776834 sequences in the BAM file
## 10776834 sequences exported
## 10776834 reads; of these:
##   10776834 (100.00%) were unpaired; of these:
##     2341516 (21.73%) aligned 0 times
##     5632082 (52.26%) aligned exactly 1 time
##     2803236 (26.01%) aligned &gt;1 times
## 78.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2803236/8435318
## 2 2043970/2803236
## 3 14121238/27304205
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0091-raw_1 and luad-0091-raw_2
## Single-end reads will be in luad-0091-raw_M
## 10256289 sequences in the BAM file
## 10256289 sequences exported
## 10256289 reads; of these:
##   10256289 (100.00%) were unpaired; of these:
##     4231026 (41.25%) aligned 0 times
##     4217774 (41.12%) aligned exactly 1 time
##     1807489 (17.62%) aligned &gt;1 times
## 58.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1807489/6025263
## 2 1191522/1807489
## 3 11259953/23942307
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0092-raw_1 and luad-0092-raw_2
## Single-end reads will be in luad-0092-raw_M
## 20480181 sequences in the BAM file
## 20480181 sequences exported
## 20480181 reads; of these:
##   20480181 (100.00%) were unpaired; of these:
##     4759344 (23.24%) aligned 0 times
##     9990281 (48.78%) aligned exactly 1 time
##     5730556 (27.98%) aligned &gt;1 times
## 76.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5730556/15720837
## 2 4260891/5730556
## 3 26840561/52334656
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0093-raw_1 and luad-0093-raw_2
## Single-end reads will be in luad-0093-raw_M
## 13503330 sequences in the BAM file
## 13503330 sequences exported
## 13503330 reads; of these:
##   13503330 (100.00%) were unpaired; of these:
##     836115 (6.19%) aligned 0 times
##     7978514 (59.09%) aligned exactly 1 time
##     4688701 (34.72%) aligned &gt;1 times
## 93.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4688701/12667215
## 2 3742674/4688701
## 3 24498168/48748127
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0094-raw_1 and luad-0094-raw_2
## Single-end reads will be in luad-0094-raw_M
## 7322109 sequences in the BAM file
## 7322109 sequences exported
## 7322109 reads; of these:
##   7322109 (100.00%) were unpaired; of these:
##     2000472 (27.32%) aligned 0 times
##     2968029 (40.54%) aligned exactly 1 time
##     2353608 (32.14%) aligned &gt;1 times
## 72.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2353608/5321637
## 2 1673313/2353608
## 3 12588761/25553997
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0095-raw_1 and luad-0095-raw_2
## Single-end reads will be in luad-0095-raw_M
## 11037826 sequences in the BAM file
## 11037826 sequences exported
## 11037826 reads; of these:
##   11037826 (100.00%) were unpaired; of these:
##     2871736 (26.02%) aligned 0 times
##     5067967 (45.91%) aligned exactly 1 time
##     3098123 (28.07%) aligned &gt;1 times
## 73.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3098123/8166090
## 2 2349043/3098123
## 3 15199760/31479273
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0096-raw_1 and luad-0096-raw_2
## Single-end reads will be in luad-0096-raw_M
## 10343936 sequences in the BAM file
## 10343936 sequences exported
## 10343936 reads; of these:
##   10343936 (100.00%) were unpaired; of these:
##     627129 (6.06%) aligned 0 times
##     6243635 (60.36%) aligned exactly 1 time
##     3473172 (33.58%) aligned &gt;1 times
## 93.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3473172/9716807
## 2 2695497/3473172
## 3 23245558/46156306
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0097-raw_1 and luad-0097-raw_2
## Single-end reads will be in luad-0097-raw_M
## 3541248 sequences in the BAM file
## 3541248 sequences exported
## 3541248 reads; of these:
##   3541248 (100.00%) were unpaired; of these:
##     819480 (23.14%) aligned 0 times
##     1871140 (52.84%) aligned exactly 1 time
##     850628 (24.02%) aligned &gt;1 times
## 76.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 850628/2721768
## 2 613472/850628
## 3 5337685/10460267
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0098-raw_1 and luad-0098-raw_2
## Single-end reads will be in luad-0098-raw_M
## 2930813 sequences in the BAM file
## 2930813 sequences exported
## 2930813 reads; of these:
##   2930813 (100.00%) were unpaired; of these:
##     605594 (20.66%) aligned 0 times
##     1301857 (44.42%) aligned exactly 1 time
##     1023362 (34.92%) aligned &gt;1 times
## 79.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1023362/2325219
## 2 781852/1023362
## 3 7480219/13121504
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0099-raw_1 and luad-0099-raw_2
## Single-end reads will be in luad-0099-raw_M
## 19155928 sequences in the BAM file
## 19155928 sequences exported
## 19155928 reads; of these:
##   19155928 (100.00%) were unpaired; of these:
##     4911703 (25.64%) aligned 0 times
##     9929848 (51.84%) aligned exactly 1 time
##     4314377 (22.52%) aligned &gt;1 times
## 74.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4314377/14244225
## 2 3308640/4314377
## 3 18528172/41878181
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0100-raw_1 and luad-0100-raw_2
## Single-end reads will be in luad-0100-raw_M
## 15924772 sequences in the BAM file
## 15924772 sequences exported
## 15924772 reads; of these:
##   15924772 (100.00%) were unpaired; of these:
##     4154367 (26.09%) aligned 0 times
##     7230841 (45.41%) aligned exactly 1 time
##     4539564 (28.51%) aligned &gt;1 times
## 73.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4539564/11770405
## 2 3473003/4539564
## 3 23853544/48595923
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in luad-0101-raw_1 and luad-0101-raw_2
## Single-end reads will be in luad-0101-raw_M
## 4017504 sequences in the BAM file
## 4017504 sequences exported
## 4017504 reads; of these:
##   4017504 (100.00%) were unpaired; of these:
##     847371 (21.09%) aligned 0 times
##     2030013 (50.53%) aligned exactly 1 time
##     1140120 (28.38%) aligned &gt;1 times
## 78.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1140120/3170133
## 2 858124/1140120
## 3 7863939/14674394
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0102-raw_1 and luad-0102-raw_2
## Single-end reads will be in luad-0102-raw_M
## 17848975 sequences in the BAM file
## 17848975 sequences exported
## 17848975 reads; of these:
##   17848975 (100.00%) were unpaired; of these:
##     2598871 (14.56%) aligned 0 times
##     10073637 (56.44%) aligned exactly 1 time
##     5176467 (29.00%) aligned &gt;1 times
## 85.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5176467/15250104
## 2 3811542/5176467
## 3 18412849/43412404
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in luad-0103-raw_1 and luad-0103-raw_2
## Single-end reads will be in luad-0103-raw_M
## 5941618 sequences in the BAM file
## 5941618 sequences exported
## 5941618 reads; of these:
##   5941618 (100.00%) were unpaired; of these:
##     1202867 (20.24%) aligned 0 times
##     3029086 (50.98%) aligned exactly 1 time
##     1709665 (28.77%) aligned &gt;1 times
## 79.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1709665/4738751
## 2 1333185/1709665
## 3 9009776/18295808
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0104-raw_1 and luad-0104-raw_2
## Single-end reads will be in luad-0104-raw_M
## 2006094 sequences in the BAM file
## 2006094 sequences exported
## 2006094 reads; of these:
##   2006094 (100.00%) were unpaired; of these:
##     436935 (21.78%) aligned 0 times
##     1070325 (53.35%) aligned exactly 1 time
##     498834 (24.87%) aligned &gt;1 times
## 78.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 498834/1569159
## 2 367556/498834
## 3 2728574/6158141
## This looks like paired data from lane 1.
## Output will be in luad-0105-raw_1 and luad-0105-raw_2
## Single-end reads will be in luad-0105-raw_M
## 11243115 sequences in the BAM file
## 11243115 sequences exported
## 11243115 reads; of these:
##   11243115 (100.00%) were unpaired; of these:
##     2776086 (24.69%) aligned 0 times
##     5714799 (50.83%) aligned exactly 1 time
##     2752230 (24.48%) aligned &gt;1 times
## 75.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2752230/8467029
## 2 2079636/2752230
## 3 14441515/30537229
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0106-raw_1 and luad-0106-raw_2
## Single-end reads will be in luad-0106-raw_M
## 9425052 sequences in the BAM file
## 9425052 sequences exported
## 9425052 reads; of these:
##   9425052 (100.00%) were unpaired; of these:
##     959744 (10.18%) aligned 0 times
##     5423925 (57.55%) aligned exactly 1 time
##     3041383 (32.27%) aligned &gt;1 times
## 89.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3041383/8465308
## 2 2334882/3041383
## 3 17692878/36098087
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0107-raw_1 and luad-0107-raw_2
## Single-end reads will be in luad-0107-raw_M
## 4649415 sequences in the BAM file
## 4649415 sequences exported
## 4649415 reads; of these:
##   4649415 (100.00%) were unpaired; of these:
##     966848 (20.80%) aligned 0 times
##     2730076 (58.72%) aligned exactly 1 time
##     952491 (20.49%) aligned &gt;1 times
## 79.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 952491/3682567
## 2 690832/952491
## 3 3275800/8987459
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in luad-0108-raw_1 and luad-0108-raw_2
## Single-end reads will be in luad-0108-raw_M
## 6223565 sequences in the BAM file
## 6223565 sequences exported
## 6223565 reads; of these:
##   6223565 (100.00%) were unpaired; of these:
##     900559 (14.47%) aligned 0 times
##     3605396 (57.93%) aligned exactly 1 time
##     1717610 (27.60%) aligned &gt;1 times
## 85.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1717610/5323006
## 2 1302890/1717610
## 3 9864750/21224423
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0109-raw_1 and luad-0109-raw_2
## Single-end reads will be in luad-0109-raw_M
## 13933361 sequences in the BAM file
## 13933361 sequences exported
## 13933361 reads; of these:
##   13933361 (100.00%) were unpaired; of these:
##     1674811 (12.02%) aligned 0 times
##     7592160 (54.49%) aligned exactly 1 time
##     4666390 (33.49%) aligned &gt;1 times
## 87.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4666390/12258550
## 2 3621993/4666390
## 3 22734659/48697254
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in luad-0110-raw_1 and luad-0110-raw_2
## Single-end reads will be in luad-0110-raw_M
## 17467887 sequences in the BAM file
## 17467887 sequences exported
## 17467887 reads; of these:
##   17467887 (100.00%) were unpaired; of these:
##     2409911 (13.80%) aligned 0 times
##     10125405 (57.97%) aligned exactly 1 time
##     4932571 (28.24%) aligned &gt;1 times
## 86.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4932571/15057976
## 2 3695938/4932571
## 3 22508957/50046225
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in luad-0111-raw_1 and luad-0111-raw_2
## Single-end reads will be in luad-0111-raw_M
## 3356367 sequences in the BAM file
## 3356367 sequences exported
## 3356367 reads; of these:
##   3356367 (100.00%) were unpaired; of these:
##     680262 (20.27%) aligned 0 times
##     1555287 (46.34%) aligned exactly 1 time
##     1120818 (33.39%) aligned &gt;1 times
## 79.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1120818/2676105
## 2 881706/1120818
## 3 7957021/13399621
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0112-raw_1 and luad-0112-raw_2
## Single-end reads will be in luad-0112-raw_M
## 21895427 sequences in the BAM file
## 21895427 sequences exported
## 21895427 reads; of these:
##   21895427 (100.00%) were unpaired; of these:
##     5589461 (25.53%) aligned 0 times
##     10877082 (49.68%) aligned exactly 1 time
##     5428884 (24.79%) aligned &gt;1 times
## 74.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5428884/16305966
## 2 4225131/5428884
## 3 20978982/47706353
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0113-raw_1 and luad-0113-raw_2
## Single-end reads will be in luad-0113-raw_M
## 3985097 sequences in the BAM file
## 3985097 sequences exported
## 3985097 reads; of these:
##   3985097 (100.00%) were unpaired; of these:
##     820532 (20.59%) aligned 0 times
##     2046062 (51.34%) aligned exactly 1 time
##     1118503 (28.07%) aligned &gt;1 times
## 79.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1118503/3164565
## 2 835154/1118503
## 3 10629718/20010731
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0114-raw_1 and luad-0114-raw_2
## Single-end reads will be in luad-0114-raw_M
## 10214486 sequences in the BAM file
## 10214486 sequences exported
## 10214486 reads; of these:
##   10214486 (100.00%) were unpaired; of these:
##     2624874 (25.70%) aligned 0 times
##     4769994 (46.70%) aligned exactly 1 time
##     2819618 (27.60%) aligned &gt;1 times
## 74.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2819618/7589612
## 2 2184055/2819618
## 3 10847659/24056080
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0115-raw_1 and luad-0115-raw_2
## Single-end reads will be in luad-0115-raw_M
## 2941175 sequences in the BAM file
## 2941175 sequences exported
## 2941175 reads; of these:
##   2941175 (100.00%) were unpaired; of these:
##     506516 (17.22%) aligned 0 times
##     1600501 (54.42%) aligned exactly 1 time
##     834158 (28.36%) aligned &gt;1 times
## 82.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 834158/2434659
## 2 573361/834158
## 3 3307629/7980183
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0116-raw_1 and luad-0116-raw_2
## Single-end reads will be in luad-0116-raw_M
## 2203258 sequences in the BAM file
## 2203258 sequences exported
## 2203258 reads; of these:
##   2203258 (100.00%) were unpaired; of these:
##     400283 (18.17%) aligned 0 times
##     1040154 (47.21%) aligned exactly 1 time
##     762821 (34.62%) aligned &gt;1 times
## 81.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 762821/1802975
## 2 585592/762821
## 3 5348658/9787470
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in luad-0117-raw_1 and luad-0117-raw_2
## Single-end reads will be in luad-0117-raw_M
## 5980409 sequences in the BAM file
## 5980409 sequences exported
## 5980409 reads; of these:
##   5980409 (100.00%) were unpaired; of these:
##     741275 (12.40%) aligned 0 times
##     3617817 (60.49%) aligned exactly 1 time
##     1621317 (27.11%) aligned &gt;1 times
## 87.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1621317/5239134
## 2 1204482/1621317
## 3 4439663/11674530
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0118-raw_1 and luad-0118-raw_2
## Single-end reads will be in luad-0118-raw_M
## 4278275 sequences in the BAM file
## 4278275 sequences exported
## 4278275 reads; of these:
##   4278275 (100.00%) were unpaired; of these:
##     722151 (16.88%) aligned 0 times
##     2832017 (66.20%) aligned exactly 1 time
##     724107 (16.93%) aligned &gt;1 times
## 83.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 724107/3556124
## 2 512555/724107
## 3 3673968/9531972
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0119-raw_1 and luad-0119-raw_2
## Single-end reads will be in luad-0119-raw_M
## 10322739 sequences in the BAM file
## 10322739 sequences exported
## 10322739 reads; of these:
##   10322739 (100.00%) were unpaired; of these:
##     2702550 (26.18%) aligned 0 times
##     5012768 (48.56%) aligned exactly 1 time
##     2607421 (25.26%) aligned &gt;1 times
## 73.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2607421/7620189
## 2 2010997/2607421
## 3 12680052/26847421
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0120-raw_1 and luad-0120-raw_2
## Single-end reads will be in luad-0120-raw_M
## 3715476 sequences in the BAM file
## 3715476 sequences exported
## 3715476 reads; of these:
##   3715476 (100.00%) were unpaired; of these:
##     765758 (20.61%) aligned 0 times
##     1827524 (49.19%) aligned exactly 1 time
##     1122194 (30.20%) aligned &gt;1 times
## 79.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1122194/2949718
## 2 893034/1122194
## 3 5064673/10683301
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0121-raw_1 and luad-0121-raw_2
## Single-end reads will be in luad-0121-raw_M
## 2615286 sequences in the BAM file
## 2615286 sequences exported
## 2615286 reads; of these:
##   2615286 (100.00%) were unpaired; of these:
##     541772 (20.72%) aligned 0 times
##     976450 (37.34%) aligned exactly 1 time
##     1097064 (41.95%) aligned &gt;1 times
## 79.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1097064/2073514
## 2 834392/1097064
## 3 12979318/21531093
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0122-raw_1 and luad-0122-raw_2
## Single-end reads will be in luad-0122-raw_M
## 8918477 sequences in the BAM file
## 8918477 sequences exported
## 8918477 reads; of these:
##   8918477 (100.00%) were unpaired; of these:
##     2340305 (26.24%) aligned 0 times
##     4093793 (45.90%) aligned exactly 1 time
##     2484379 (27.86%) aligned &gt;1 times
## 73.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2484379/6578172
## 2 1898236/2484379
## 3 13712547/28430208
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0123-raw_1 and luad-0123-raw_2
## Single-end reads will be in luad-0123-raw_M
## 7759720 sequences in the BAM file
## 7759720 sequences exported
## 7759720 reads; of these:
##   7759720 (100.00%) were unpaired; of these:
##     1507066 (19.42%) aligned 0 times
##     3941451 (50.79%) aligned exactly 1 time
##     2311203 (29.78%) aligned &gt;1 times
## 80.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2311203/6252654
## 2 1665649/2311203
## 3 16918741/34170801
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0124-raw_1 and luad-0124-raw_2
## Single-end reads will be in luad-0124-raw_M
## 16452140 sequences in the BAM file
## 16452140 sequences exported
## 16452140 reads; of these:
##   16452140 (100.00%) were unpaired; of these:
##     2921181 (17.76%) aligned 0 times
##     8855810 (53.83%) aligned exactly 1 time
##     4675149 (28.42%) aligned &gt;1 times
## 82.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4675149/13530959
## 2 3590790/4675149
## 3 25153806/55433073
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 1.
## Output will be in luad-0125-raw_1 and luad-0125-raw_2
## Single-end reads will be in luad-0125-raw_M
## 4018334 sequences in the BAM file
## 4018334 sequences exported
## 4018334 reads; of these:
##   4018334 (100.00%) were unpaired; of these:
##     890180 (22.15%) aligned 0 times
##     2519430 (62.70%) aligned exactly 1 time
##     608724 (15.15%) aligned &gt;1 times
## 77.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 608724/3128154
## 2 432238/608724
## 3 2523771/7191449
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0126-raw_1 and luad-0126-raw_2
## Single-end reads will be in luad-0126-raw_M
## 2434063 sequences in the BAM file
## 2434063 sequences exported
## 2434063 reads; of these:
##   2434063 (100.00%) were unpaired; of these:
##     451837 (18.56%) aligned 0 times
##     1358770 (55.82%) aligned exactly 1 time
##     623456 (25.61%) aligned &gt;1 times
## 81.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 623456/1982226
## 2 471857/623456
## 3 4826859/8773680
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0127-raw_1 and luad-0127-raw_2
## Single-end reads will be in luad-0127-raw_M
## 5634921 sequences in the BAM file
## 5634921 sequences exported
## 5634921 reads; of these:
##   5634921 (100.00%) were unpaired; of these:
##     1485505 (26.36%) aligned 0 times
##     2352236 (41.74%) aligned exactly 1 time
##     1797180 (31.89%) aligned &gt;1 times
## 73.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1797180/4149416
## 2 1416828/1797180
## 3 10647018/21705485
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0128-raw_1 and luad-0128-raw_2
## Single-end reads will be in luad-0128-raw_M
## 2224970 sequences in the BAM file
## 2224970 sequences exported
## 2224970 reads; of these:
##   2224970 (100.00%) were unpaired; of these:
##     515829 (23.18%) aligned 0 times
##     892821 (40.13%) aligned exactly 1 time
##     816320 (36.69%) aligned &gt;1 times
## 76.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 816320/1709141
## 2 602113/816320
## 3 9919768/16566531
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0129-raw_1 and luad-0129-raw_2
## Single-end reads will be in luad-0129-raw_M
## 2430085 sequences in the BAM file
## 2430085 sequences exported
## 2430085 reads; of these:
##   2430085 (100.00%) were unpaired; of these:
##     637999 (26.25%) aligned 0 times
##     1140933 (46.95%) aligned exactly 1 time
##     651153 (26.80%) aligned &gt;1 times
## 73.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 651153/1792086
## 2 490727/651153
## 3 5136403/9970412
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in luad-0130-raw_1 and luad-0130-raw_2
## Single-end reads will be in luad-0130-raw_M
## 5366363 sequences in the BAM file
## 5366363 sequences exported
## 5366363 reads; of these:
##   5366363 (100.00%) were unpaired; of these:
##     1157760 (21.57%) aligned 0 times
##     2701283 (50.34%) aligned exactly 1 time
##     1507320 (28.09%) aligned &gt;1 times
## 78.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1507320/4208603
## 2 1164152/1507320
## 3 8538341/16740908
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0131-raw_1 and luad-0131-raw_2
## Single-end reads will be in luad-0131-raw_M
## 2004862 sequences in the BAM file
## 2004862 sequences exported
## 2004862 reads; of these:
##   2004862 (100.00%) were unpaired; of these:
##     428475 (21.37%) aligned 0 times
##     895998 (44.69%) aligned exactly 1 time
##     680389 (33.94%) aligned &gt;1 times
## 78.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 680389/1576387
## 2 515050/680389
## 3 3830529/8141844
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0132-raw_1 and luad-0132-raw_2
## Single-end reads will be in luad-0132-raw_M
## 3909147 sequences in the BAM file
## 3909147 sequences exported
## 3909147 reads; of these:
##   3909147 (100.00%) were unpaired; of these:
##     673720 (17.23%) aligned 0 times
##     2350908 (60.14%) aligned exactly 1 time
##     884519 (22.63%) aligned &gt;1 times
## 82.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 884519/3235427
## 2 635069/884519
## 3 3293520/8695707
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in luad-0133-raw_1 and luad-0133-raw_2
## Single-end reads will be in luad-0133-raw_M
## 14370318 sequences in the BAM file
## 14370318 sequences exported
## 14370318 reads; of these:
##   14370318 (100.00%) were unpaired; of these:
##     1940667 (13.50%) aligned 0 times
##     8373245 (58.27%) aligned exactly 1 time
##     4056406 (28.23%) aligned &gt;1 times
## 86.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4056406/12429651
## 2 2894756/4056406
## 3 23969414/52842997
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in luad-0134-raw_1 and luad-0134-raw_2
## Single-end reads will be in luad-0134-raw_M
## 2953256 sequences in the BAM file
## 2953256 sequences exported
## 2953256 reads; of these:
##   2953256 (100.00%) were unpaired; of these:
##     557224 (18.87%) aligned 0 times
##     1748142 (59.19%) aligned exactly 1 time
##     647890 (21.94%) aligned &gt;1 times
## 81.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 647890/2396032
## 2 451771/647890
## 3 3015801/7229945
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0135-raw_1 and luad-0135-raw_2
## Single-end reads will be in luad-0135-raw_M
## 14266517 sequences in the BAM file
## 14266517 sequences exported
## 14266517 reads; of these:
##   14266517 (100.00%) were unpaired; of these:
##     1265073 (8.87%) aligned 0 times
##     8865941 (62.15%) aligned exactly 1 time
##     4135503 (28.99%) aligned &gt;1 times
## 91.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4135503/13001444
## 2 3126096/4135503
## 3 19178580/41783372
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0136-raw_1 and luad-0136-raw_2
## Single-end reads will be in luad-0136-raw_M
## 6974339 sequences in the BAM file
## 6974339 sequences exported
## 6974339 reads; of these:
##   6974339 (100.00%) were unpaired; of these:
##     1656986 (23.76%) aligned 0 times
##     2931222 (42.03%) aligned exactly 1 time
##     2386131 (34.21%) aligned &gt;1 times
## 76.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2386131/5317353
## 2 1710155/2386131
## 3 21855071/50235612
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in luad-0137-raw_1 and luad-0137-raw_2
## Single-end reads will be in luad-0137-raw_M
## 3488530 sequences in the BAM file
## 3488530 sequences exported
## 3488530 reads; of these:
##   3488530 (100.00%) were unpaired; of these:
##     809402 (23.20%) aligned 0 times
##     1949204 (55.87%) aligned exactly 1 time
##     729924 (20.92%) aligned &gt;1 times
## 76.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 729924/2679128
## 2 575670/729924
## 3 5247015/10351301
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0138-raw_1 and luad-0138-raw_2
## Single-end reads will be in luad-0138-raw_M
## 8659708 sequences in the BAM file
## 8659708 sequences exported
## 8659708 reads; of these:
##   8659708 (100.00%) were unpaired; of these:
##     876384 (10.12%) aligned 0 times
##     5770922 (66.64%) aligned exactly 1 time
##     2012402 (23.24%) aligned &gt;1 times
## 89.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2012402/7783324
## 2 1471169/2012402
## 3 13285374/29257782
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0139-raw_1 and luad-0139-raw_2
## Single-end reads will be in luad-0139-raw_M
## 4063157 sequences in the BAM file
## 4063157 sequences exported
## 4063157 reads; of these:
##   4063157 (100.00%) were unpaired; of these:
##     808657 (19.90%) aligned 0 times
##     2152010 (52.96%) aligned exactly 1 time
##     1102490 (27.13%) aligned &gt;1 times
## 80.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1102490/3254500
## 2 830629/1102490
## 3 9813400/17332798
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0140-raw_1 and luad-0140-raw_2
## Single-end reads will be in luad-0140-raw_M
## 3244008 sequences in the BAM file
## 3244008 sequences exported
## 3244008 reads; of these:
##   3244008 (100.00%) were unpaired; of these:
##     650964 (20.07%) aligned 0 times
##     1741047 (53.67%) aligned exactly 1 time
##     851997 (26.26%) aligned &gt;1 times
## 79.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 851997/2593044
## 2 637608/851997
## 3 6706824/12861702
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0141-raw_1 and luad-0141-raw_2
## Single-end reads will be in luad-0141-raw_M
## 1818557 sequences in the BAM file
## 1818557 sequences exported
## 1818557 reads; of these:
##   1818557 (100.00%) were unpaired; of these:
##     442166 (24.31%) aligned 0 times
##     850992 (46.79%) aligned exactly 1 time
##     525399 (28.89%) aligned &gt;1 times
## 75.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 525399/1376391
## 2 407885/525399
## 3 3078133/5624404
## This looks like paired data from lane 2.
## Output will be in luad-0142-raw_1 and luad-0142-raw_2
## Single-end reads will be in luad-0142-raw_M
## 14757431 sequences in the BAM file
## 14757431 sequences exported
## 14757431 reads; of these:
##   14757431 (100.00%) were unpaired; of these:
##     906871 (6.15%) aligned 0 times
##     8789956 (59.56%) aligned exactly 1 time
##     5060604 (34.29%) aligned &gt;1 times
## 93.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5060604/13850560
## 2 3856110/5060604
## 3 32585976/61425312
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0143-raw_1 and luad-0143-raw_2
## Single-end reads will be in luad-0143-raw_M
## 17032497 sequences in the BAM file
## 17032497 sequences exported
## 17032497 reads; of these:
##   17032497 (100.00%) were unpaired; of these:
##     1129536 (6.63%) aligned 0 times
##     10465559 (61.44%) aligned exactly 1 time
##     5437402 (31.92%) aligned &gt;1 times
## 93.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5437402/15902961
## 2 4178003/5437402
## 3 28029966/58353693
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 7.
## Output will be in luad-0144-raw_1 and luad-0144-raw_2
## Single-end reads will be in luad-0144-raw_M
## 3499129 sequences in the BAM file
## 3499129 sequences exported
## 3499129 reads; of these:
##   3499129 (100.00%) were unpaired; of these:
##     782848 (22.37%) aligned 0 times
##     1691858 (48.35%) aligned exactly 1 time
##     1024423 (29.28%) aligned &gt;1 times
## 77.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1024423/2716281
## 2 804378/1024423
## 3 7292524/13099431
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0145-raw_1 and luad-0145-raw_2
## Single-end reads will be in luad-0145-raw_M
## 11870217 sequences in the BAM file
## 11870217 sequences exported
## 11870217 reads; of these:
##   11870217 (100.00%) were unpaired; of these:
##     628857 (5.30%) aligned 0 times
##     5794887 (48.82%) aligned exactly 1 time
##     5446473 (45.88%) aligned &gt;1 times
## 94.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5446473/11241360
## 2 4628754/5446473
## 3 22266353/44548663
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in luad-0146-raw_1 and luad-0146-raw_2
## Single-end reads will be in luad-0146-raw_M
## 4328300 sequences in the BAM file
## 4328300 sequences exported
## 4328300 reads; of these:
##   4328300 (100.00%) were unpaired; of these:
##     840323 (19.41%) aligned 0 times
##     2387559 (55.16%) aligned exactly 1 time
##     1100418 (25.42%) aligned &gt;1 times
## 80.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1100418/3487977
## 2 819726/1100418
## 3 7733151/14772718
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0147-raw_1 and luad-0147-raw_2
## Single-end reads will be in luad-0147-raw_M
## 2161835 sequences in the BAM file
## 2161835 sequences exported
## 2161835 reads; of these:
##   2161835 (100.00%) were unpaired; of these:
##     421264 (19.49%) aligned 0 times
##     974491 (45.08%) aligned exactly 1 time
##     766080 (35.44%) aligned &gt;1 times
## 80.51% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 766080/1740571
## 2 583973/766080
## 3 4119254/7376587
## This looks like paired data from lane 3.
## Output will be in luad-0148-raw_1 and luad-0148-raw_2
## Single-end reads will be in luad-0148-raw_M
## 2365451 sequences in the BAM file
## 2365451 sequences exported
## 2365451 reads; of these:
##   2365451 (100.00%) were unpaired; of these:
##     575163 (24.32%) aligned 0 times
##     1085934 (45.91%) aligned exactly 1 time
##     704354 (29.78%) aligned &gt;1 times
## 75.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 704354/1790288
## 2 526457/704354
## 3 5466699/10480271
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0149-raw_1 and luad-0149-raw_2
## Single-end reads will be in luad-0149-raw_M
## 3765339 sequences in the BAM file
## 3765339 sequences exported
## 3765339 reads; of these:
##   3765339 (100.00%) were unpaired; of these:
##     596674 (15.85%) aligned 0 times
##     2237247 (59.42%) aligned exactly 1 time
##     931418 (24.74%) aligned &gt;1 times
## 84.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 931418/3168665
## 2 670788/931418
## 3 6961146/13451275
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0150-raw_1 and luad-0150-raw_2
## Single-end reads will be in luad-0150-raw_M
## 1877196 sequences in the BAM file
## 1877196 sequences exported
## 1877196 reads; of these:
##   1877196 (100.00%) were unpaired; of these:
##     477587 (25.44%) aligned 0 times
##     828981 (44.16%) aligned exactly 1 time
##     570628 (30.40%) aligned &gt;1 times
## 74.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 570628/1399609
## 2 396526/570628
## 3 5563745/10164279
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0151-raw_1 and luad-0151-raw_2
## Single-end reads will be in luad-0151-raw_M
## 1860772 sequences in the BAM file
## 1860772 sequences exported
## 1860772 reads; of these:
##   1860772 (100.00%) were unpaired; of these:
##     444617 (23.89%) aligned 0 times
##     531686 (28.57%) aligned exactly 1 time
##     884469 (47.53%) aligned &gt;1 times
## 76.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 884469/1416155
## 2 696837/884469
## 3 9993485/18542246
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0152-raw_1 and luad-0152-raw_2
## Single-end reads will be in luad-0152-raw_M
## 4210029 sequences in the BAM file
## 4210029 sequences exported
## 4210029 reads; of these:
##   4210029 (100.00%) were unpaired; of these:
##     818345 (19.44%) aligned 0 times
##     2609068 (61.97%) aligned exactly 1 time
##     782616 (18.59%) aligned &gt;1 times
## 80.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 782616/3391684
## 2 607426/782616
## 3 2130214/6624578
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0153-raw_1 and luad-0153-raw_2
## Single-end reads will be in luad-0153-raw_M
## 11581908 sequences in the BAM file
## 11581908 sequences exported
## 11581908 reads; of these:
##   11581908 (100.00%) were unpaired; of these:
##     1699457 (14.67%) aligned 0 times
##     7119663 (61.47%) aligned exactly 1 time
##     2762788 (23.85%) aligned &gt;1 times
## 85.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2762788/9882451
## 2 2007710/2762788
## 3 15727465/37528712
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in luad-0154-raw_1 and luad-0154-raw_2
## Single-end reads will be in luad-0154-raw_M
## 2738673 sequences in the BAM file
## 2738673 sequences exported
## 2738673 reads; of these:
##   2738673 (100.00%) were unpaired; of these:
##     540711 (19.74%) aligned 0 times
##     1486178 (54.27%) aligned exactly 1 time
##     711784 (25.99%) aligned &gt;1 times
## 80.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 711784/2197962
## 2 551145/711784
## 3 4941265/9813101
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0155-raw_1 and luad-0155-raw_2
## Single-end reads will be in luad-0155-raw_M
## 9205780 sequences in the BAM file
## 9205780 sequences exported
## 9205780 reads; of these:
##   9205780 (100.00%) were unpaired; of these:
##     568854 (6.18%) aligned 0 times
##     4793265 (52.07%) aligned exactly 1 time
##     3843661 (41.75%) aligned &gt;1 times
## 93.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3843661/8636926
## 2 2919879/3843661
## 3 27260647/52926177
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in luad-0156-raw_1 and luad-0156-raw_2
## Single-end reads will be in luad-0156-raw_M
## 23090635 sequences in the BAM file
## 23090635 sequences exported
## 23090635 reads; of these:
##   23090635 (100.00%) were unpaired; of these:
##     3872011 (16.77%) aligned 0 times
##     14647288 (63.43%) aligned exactly 1 time
##     4571336 (19.80%) aligned &gt;1 times
## 83.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4571336/19218624
## 2 3390677/4571336
## 3 19948755/49491709
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in luad-0157-raw_1 and luad-0157-raw_2
## Single-end reads will be in luad-0157-raw_M
## 2570573 sequences in the BAM file
## 2570573 sequences exported
## 2570573 reads; of these:
##   2570573 (100.00%) were unpaired; of these:
##     602528 (23.44%) aligned 0 times
##     1382466 (53.78%) aligned exactly 1 time
##     585579 (22.78%) aligned &gt;1 times
## 76.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 585579/1968045
## 2 441673/585579
## 3 5175680/9833097
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0158-raw_1 and luad-0158-raw_2
## Single-end reads will be in luad-0158-raw_M
## 3209183 sequences in the BAM file
## 3209183 sequences exported
## 3209183 reads; of these:
##   3209183 (100.00%) were unpaired; of these:
##     636524 (19.83%) aligned 0 times
##     1879904 (58.58%) aligned exactly 1 time
##     692755 (21.59%) aligned &gt;1 times
## 80.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 692755/2572659
## 2 530441/692755
## 3 6179576/12004023
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0159-raw_1 and luad-0159-raw_2
## Single-end reads will be in luad-0159-raw_M
## 7480384 sequences in the BAM file
## 7480384 sequences exported
## 7480384 reads; of these:
##   7480384 (100.00%) were unpaired; of these:
##     461075 (6.16%) aligned 0 times
##     4890259 (65.37%) aligned exactly 1 time
##     2129050 (28.46%) aligned &gt;1 times
## 93.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2129050/7019309
## 2 1587608/2129050
## 3 11719807/24672514
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0160-raw_1 and luad-0160-raw_2
## Single-end reads will be in luad-0160-raw_M
## 2161811 sequences in the BAM file
## 2161811 sequences exported
## 2161811 reads; of these:
##   2161811 (100.00%) were unpaired; of these:
##     493916 (22.85%) aligned 0 times
##     1095613 (50.68%) aligned exactly 1 time
##     572282 (26.47%) aligned &gt;1 times
## 77.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 572282/1667895
## 2 377420/572282
## 3 3308006/7582951
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0161-raw_1 and luad-0161-raw_2
## Single-end reads will be in luad-0161-raw_M
## 5442019 sequences in the BAM file
## 5442019 sequences exported
## 5442019 reads; of these:
##   5442019 (100.00%) were unpaired; of these:
##     764654 (14.05%) aligned 0 times
##     3019599 (55.49%) aligned exactly 1 time
##     1657766 (30.46%) aligned &gt;1 times
## 85.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1657766/4677365
## 2 1167353/1657766
## 3 8427979/18935361
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0162-raw_1 and luad-0162-raw_2
## Single-end reads will be in luad-0162-raw_M
## 3991971 sequences in the BAM file
## 3991971 sequences exported
## 3991971 reads; of these:
##   3991971 (100.00%) were unpaired; of these:
##     844010 (21.14%) aligned 0 times
##     2098520 (52.57%) aligned exactly 1 time
##     1049441 (26.29%) aligned &gt;1 times
## 78.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1049441/3147961
## 2 814807/1049441
## 3 6324617/12332273
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0163-raw_1 and luad-0163-raw_2
## Single-end reads will be in luad-0163-raw_M
## 11028031 sequences in the BAM file
## 11028031 sequences exported
## 11028031 reads; of these:
##   11028031 (100.00%) were unpaired; of these:
##     2820074 (25.57%) aligned 0 times
##     5465344 (49.56%) aligned exactly 1 time
##     2742613 (24.87%) aligned &gt;1 times
## 74.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2742613/8207957
## 2 2210258/2742613
## 3 12008843/26828416
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0164-raw_1 and luad-0164-raw_2
## Single-end reads will be in luad-0164-raw_M
## 15887113 sequences in the BAM file
## 15887113 sequences exported
## 15887113 reads; of these:
##   15887113 (100.00%) were unpaired; of these:
##     940406 (5.92%) aligned 0 times
##     9672714 (60.88%) aligned exactly 1 time
##     5273993 (33.20%) aligned &gt;1 times
## 94.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5273993/14946707
## 2 3997916/5273993
## 3 26036734/55088203
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in luad-0165-raw_1 and luad-0165-raw_2
## Single-end reads will be in luad-0165-raw_M
## 3338686 sequences in the BAM file
## 3338686 sequences exported
## 3338686 reads; of these:
##   3338686 (100.00%) were unpaired; of these:
##     625388 (18.73%) aligned 0 times
##     1959417 (58.69%) aligned exactly 1 time
##     753881 (22.58%) aligned &gt;1 times
## 81.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 753881/2713298
## 2 555322/753881
## 3 2695880/6940897
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0166-raw_1 and luad-0166-raw_2
## Single-end reads will be in luad-0166-raw_M
## 4502402 sequences in the BAM file
## 4502402 sequences exported
## 4502402 reads; of these:
##   4502402 (100.00%) were unpaired; of these:
##     684084 (15.19%) aligned 0 times
##     2668592 (59.27%) aligned exactly 1 time
##     1149726 (25.54%) aligned &gt;1 times
## 84.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1149726/3818318
## 2 922747/1149726
## 3 6626838/13329582
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0167-raw_1 and luad-0167-raw_2
## Single-end reads will be in luad-0167-raw_M
## 4612258 sequences in the BAM file
## 4612258 sequences exported
## 4612258 reads; of these:
##   4612258 (100.00%) were unpaired; of these:
##     924770 (20.05%) aligned 0 times
##     2431521 (52.72%) aligned exactly 1 time
##     1255967 (27.23%) aligned &gt;1 times
## 79.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1255967/3687488
## 2 941521/1255967
## 3 7461453/14201851
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0168-raw_1 and luad-0168-raw_2
## Single-end reads will be in luad-0168-raw_M
## 11264933 sequences in the BAM file
## 11264933 sequences exported
## 11264933 reads; of these:
##   11264933 (100.00%) were unpaired; of these:
##     2180562 (19.36%) aligned 0 times
##     6259637 (55.57%) aligned exactly 1 time
##     2824734 (25.08%) aligned &gt;1 times
## 80.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2824734/9084371
## 2 2003660/2824734
## 3 18781381/39184744
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0169-raw_1 and luad-0169-raw_2
## Single-end reads will be in luad-0169-raw_M
## 7969969 sequences in the BAM file
## 7969969 sequences exported
## 7969969 reads; of these:
##   7969969 (100.00%) were unpaired; of these:
##     1912846 (24.00%) aligned 0 times
##     4495170 (56.40%) aligned exactly 1 time
##     1561953 (19.60%) aligned &gt;1 times
## 76.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1561953/6057123
## 2 1040201/1561953
## 3 8947111/20435100
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0170-raw_1 and luad-0170-raw_2
## Single-end reads will be in luad-0170-raw_M
## 11807432 sequences in the BAM file
## 11807432 sequences exported
## 11807432 reads; of these:
##   11807432 (100.00%) were unpaired; of these:
##     2182100 (18.48%) aligned 0 times
##     6282105 (53.20%) aligned exactly 1 time
##     3343227 (28.31%) aligned &gt;1 times
## 81.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3343227/9625332
## 2 2422027/3343227
## 3 12339630/27883852
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0171-raw_1 and luad-0171-raw_2
## Single-end reads will be in luad-0171-raw_M
## 14569507 sequences in the BAM file
## 14569507 sequences exported
## 14569507 reads; of these:
##   14569507 (100.00%) were unpaired; of these:
##     1692201 (11.61%) aligned 0 times
##     7358242 (50.50%) aligned exactly 1 time
##     5519064 (37.88%) aligned &gt;1 times
## 88.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5519064/12877306
## 2 3324618/5519064
## 3 32223350/68239645
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in luad-0172-raw_1 and luad-0172-raw_2
## Single-end reads will be in luad-0172-raw_M
## 13057077 sequences in the BAM file
## 13057077 sequences exported
## 13057077 reads; of these:
##   13057077 (100.00%) were unpaired; of these:
##     3366922 (25.79%) aligned 0 times
##     7019157 (53.76%) aligned exactly 1 time
##     2670998 (20.46%) aligned &gt;1 times
## 74.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2670998/9690155
## 2 1895492/2670998
## 3 19903789/42644648
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in luad-0173-raw_1 and luad-0173-raw_2
## Single-end reads will be in luad-0173-raw_M
## 9704961 sequences in the BAM file
## 9704961 sequences exported
## 9704961 reads; of these:
##   9704961 (100.00%) were unpaired; of these:
##     2141456 (22.07%) aligned 0 times
##     5061633 (52.16%) aligned exactly 1 time
##     2501872 (25.78%) aligned &gt;1 times
## 77.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2501872/7563505
## 2 1595180/2501872
## 3 7154904/19883475
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0174-raw_1 and luad-0174-raw_2
## Single-end reads will be in luad-0174-raw_M
## 9770629 sequences in the BAM file
## 9770629 sequences exported
## 9770629 reads; of these:
##   9770629 (100.00%) were unpaired; of these:
##     3395067 (34.75%) aligned 0 times
##     3944849 (40.37%) aligned exactly 1 time
##     2430713 (24.88%) aligned &gt;1 times
## 65.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2430713/6375562
## 2 1859556/2430713
## 3 14453582/27042621
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0175-raw_1 and luad-0175-raw_2
## Single-end reads will be in luad-0175-raw_M
## 10023574 sequences in the BAM file
## 10023574 sequences exported
## 10023574 reads; of these:
##   10023574 (100.00%) were unpaired; of these:
##     2000056 (19.95%) aligned 0 times
##     4875370 (48.64%) aligned exactly 1 time
##     3148148 (31.41%) aligned &gt;1 times
## 80.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3148148/8023518
## 2 1902566/3148148
## 3 11516429/26382296
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0176-raw_1 and luad-0176-raw_2
## Single-end reads will be in luad-0176-raw_M
## 8571865 sequences in the BAM file
## 8571865 sequences exported
## 8571865 reads; of these:
##   8571865 (100.00%) were unpaired; of these:
##     828806 (9.67%) aligned 0 times
##     5042165 (58.82%) aligned exactly 1 time
##     2700894 (31.51%) aligned &gt;1 times
## 90.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2700894/7743059
## 2 2087178/2700894
## 3 13651853/27872917
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0177-raw_1 and luad-0177-raw_2
## Single-end reads will be in luad-0177-raw_M
## 13094713 sequences in the BAM file
## 13094713 sequences exported
## 13094713 reads; of these:
##   13094713 (100.00%) were unpaired; of these:
##     2435103 (18.60%) aligned 0 times
##     7353424 (56.16%) aligned exactly 1 time
##     3306186 (25.25%) aligned &gt;1 times
## 81.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3306186/10659610
## 2 2204774/3306186
## 3 18806288/43125349
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in luad-0178-raw_1 and luad-0178-raw_2
## Single-end reads will be in luad-0178-raw_M
## 8459213 sequences in the BAM file
## 8459213 sequences exported
## 8459213 reads; of these:
##   8459213 (100.00%) were unpaired; of these:
##     1632823 (19.30%) aligned 0 times
##     4359663 (51.54%) aligned exactly 1 time
##     2466727 (29.16%) aligned &gt;1 times
## 80.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2466727/6826390
## 2 1767602/2466727
## 3 9109998/20367614
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0179-raw_1 and luad-0179-raw_2
## Single-end reads will be in luad-0179-raw_M
## 12876401 sequences in the BAM file
## 12876401 sequences exported
## 12876401 reads; of these:
##   12876401 (100.00%) were unpaired; of these:
##     2431476 (18.88%) aligned 0 times
##     7492491 (58.19%) aligned exactly 1 time
##     2952434 (22.93%) aligned &gt;1 times
## 81.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2952434/10444925
## 2 2109134/2952434
## 3 18358365/40051630
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0180-raw_1 and luad-0180-raw_2
## Single-end reads will be in luad-0180-raw_M
## 11819275 sequences in the BAM file
## 11819275 sequences exported
## 11819275 reads; of these:
##   11819275 (100.00%) were unpaired; of these:
##     2038099 (17.24%) aligned 0 times
##     7572074 (64.07%) aligned exactly 1 time
##     2209102 (18.69%) aligned &gt;1 times
## 82.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2209102/9781176
## 2 1618495/2209102
## 3 7519624/20881169
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0181-raw_1 and luad-0181-raw_2
## Single-end reads will be in luad-0181-raw_M
## 5884582 sequences in the BAM file
## 5884582 sequences exported
## 5884582 reads; of these:
##   5884582 (100.00%) were unpaired; of these:
##     2403184 (40.84%) aligned 0 times
##     2492348 (42.35%) aligned exactly 1 time
##     989050 (16.81%) aligned &gt;1 times
## 59.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 989050/3481398
## 2 623264/989050
## 3 4251987/10627695
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0182-raw_1 and luad-0182-raw_2
## Single-end reads will be in luad-0182-raw_M
## 15791931 sequences in the BAM file
## 15791931 sequences exported
## 15791931 reads; of these:
##   15791931 (100.00%) were unpaired; of these:
##     2403366 (15.22%) aligned 0 times
##     9893979 (62.65%) aligned exactly 1 time
##     3494586 (22.13%) aligned &gt;1 times
## 84.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3494586/13388565
## 2 2475908/3494586
## 3 16451090/38721385
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in luad-0183-raw_1 and luad-0183-raw_2
## Single-end reads will be in luad-0183-raw_M
## 8570538 sequences in the BAM file
## 8570538 sequences exported
## 8570538 reads; of these:
##   8570538 (100.00%) were unpaired; of these:
##     2429567 (28.35%) aligned 0 times
##     3582837 (41.80%) aligned exactly 1 time
##     2558134 (29.85%) aligned &gt;1 times
## 71.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2558134/6140971
## 2 1806472/2558134
## 3 21029449/37526638
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0184-raw_1 and luad-0184-raw_2
## Single-end reads will be in luad-0184-raw_M
## 10811316 sequences in the BAM file
## 10811316 sequences exported
## 10811316 reads; of these:
##   10811316 (100.00%) were unpaired; of these:
##     4619349 (42.73%) aligned 0 times
##     4739539 (43.84%) aligned exactly 1 time
##     1452428 (13.43%) aligned &gt;1 times
## 57.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1452428/6191967
## 2 862255/1452428
## 3 3986523/12992346
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0185-raw_1 and luad-0185-raw_2
## Single-end reads will be in luad-0185-raw_M
## 12735160 sequences in the BAM file
## 12735160 sequences exported
## 12735160 reads; of these:
##   12735160 (100.00%) were unpaired; of these:
##     2844919 (22.34%) aligned 0 times
##     6819983 (53.55%) aligned exactly 1 time
##     3070258 (24.11%) aligned &gt;1 times
## 77.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3070258/9890241
## 2 2218293/3070258
## 3 8130270/22507419
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0186-raw_1 and luad-0186-raw_2
## Single-end reads will be in luad-0186-raw_M
## 11845981 sequences in the BAM file
## 11845981 sequences exported
## 11845981 reads; of these:
##   11845981 (100.00%) were unpaired; of these:
##     2274908 (19.20%) aligned 0 times
##     7096027 (59.90%) aligned exactly 1 time
##     2475046 (20.89%) aligned &gt;1 times
## 80.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2475046/9571073
## 2 1800495/2475046
## 3 11646410/27180001
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0187-raw_1 and luad-0187-raw_2
## Single-end reads will be in luad-0187-raw_M
## 13427704 sequences in the BAM file
## 13427704 sequences exported
## 13427704 reads; of these:
##   13427704 (100.00%) were unpaired; of these:
##     2724620 (20.29%) aligned 0 times
##     5540448 (41.26%) aligned exactly 1 time
##     5162636 (38.45%) aligned &gt;1 times
## 79.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5162636/10703084
## 2 3402066/5162636
## 3 69895419/125063362
## [bam_sort_core] merging from 16 files...
## This looks like paired data from lane 4.
## Output will be in luad-0188-raw_1 and luad-0188-raw_2
## Single-end reads will be in luad-0188-raw_M
## 8261799 sequences in the BAM file
## 8261799 sequences exported
## 8261799 reads; of these:
##   8261799 (100.00%) were unpaired; of these:
##     3197872 (38.71%) aligned 0 times
##     4023433 (48.70%) aligned exactly 1 time
##     1040494 (12.59%) aligned &gt;1 times
## 61.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1040494/5063927
## 2 612492/1040494
## 3 3901552/11833795
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0189-raw_1 and luad-0189-raw_2
## Single-end reads will be in luad-0189-raw_M
## 3071478 sequences in the BAM file
## 3071478 sequences exported
## 3071478 reads; of these:
##   3071478 (100.00%) were unpaired; of these:
##     547177 (17.81%) aligned 0 times
##     1762076 (57.37%) aligned exactly 1 time
##     762225 (24.82%) aligned &gt;1 times
## 82.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 762225/2524301
## 2 551728/762225
## 3 2974468/7170218
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0190-raw_1 and luad-0190-raw_2
## Single-end reads will be in luad-0190-raw_M
## 3102076 sequences in the BAM file
## 3102076 sequences exported
## 3102076 reads; of these:
##   3102076 (100.00%) were unpaired; of these:
##     607565 (19.59%) aligned 0 times
##     1610600 (51.92%) aligned exactly 1 time
##     883911 (28.49%) aligned &gt;1 times
## 80.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 883911/2494511
## 2 693379/883911
## 3 7140921/13708846
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0191-raw_1 and luad-0191-raw_2
## Single-end reads will be in luad-0191-raw_M
## 1214966 sequences in the BAM file
## 1214966 sequences exported
## 1214966 reads; of these:
##   1214966 (100.00%) were unpaired; of these:
##     313682 (25.82%) aligned 0 times
##     439082 (36.14%) aligned exactly 1 time
##     462202 (38.04%) aligned &gt;1 times
## 74.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 462202/901284
## 2 343022/462202
## 3 5411077/8895640
## This looks like paired data from lane 3.
## Output will be in luad-0192-raw_1 and luad-0192-raw_2
## Single-end reads will be in luad-0192-raw_M
## 16159290 sequences in the BAM file
## 16159290 sequences exported
## 16159290 reads; of these:
##   16159290 (100.00%) were unpaired; of these:
##     2082432 (12.89%) aligned 0 times
##     9589078 (59.34%) aligned exactly 1 time
##     4487780 (27.77%) aligned &gt;1 times
## 87.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4487780/14076858
## 2 3505377/4487780
## 3 16828295/39331287
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0193-raw_1 and luad-0193-raw_2
## Single-end reads will be in luad-0193-raw_M
## 6398872 sequences in the BAM file
## 6398872 sequences exported
## 6398872 reads; of these:
##   6398872 (100.00%) were unpaired; of these:
##     1393426 (21.78%) aligned 0 times
##     3549224 (55.47%) aligned exactly 1 time
##     1456222 (22.76%) aligned &gt;1 times
## 78.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1456222/5005446
## 2 1111949/1456222
## 3 8465462/17342485
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0194-raw_1 and luad-0194-raw_2
## Single-end reads will be in luad-0194-raw_M
## 1572074 sequences in the BAM file
## 1572074 sequences exported
## 1572074 reads; of these:
##   1572074 (100.00%) were unpaired; of these:
##     429922 (27.35%) aligned 0 times
##     632777 (40.25%) aligned exactly 1 time
##     509375 (32.40%) aligned &gt;1 times
## 72.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 509375/1142152
## 2 383991/509375
## 3 3988981/7348048
## This looks like paired data from lane 2.
## Output will be in luad-0195-raw_1 and luad-0195-raw_2
## Single-end reads will be in luad-0195-raw_M
## 4717559 sequences in the BAM file
## 4717559 sequences exported
## 4717559 reads; of these:
##   4717559 (100.00%) were unpaired; of these:
##     958213 (20.31%) aligned 0 times
##     2426495 (51.44%) aligned exactly 1 time
##     1332851 (28.25%) aligned &gt;1 times
## 79.69% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1332851/3759346
## 2 1019606/1332851
## 3 9396252/18713638
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0196-raw_1 and luad-0196-raw_2
## Single-end reads will be in luad-0196-raw_M
## 2430795 sequences in the BAM file
## 2430795 sequences exported
## 2430795 reads; of these:
##   2430795 (100.00%) were unpaired; of these:
##     511284 (21.03%) aligned 0 times
##     1408869 (57.96%) aligned exactly 1 time
##     510642 (21.01%) aligned &gt;1 times
## 78.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 510642/1919511
## 2 391731/510642
## 3 4089936/7882640
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0197-raw_1 and luad-0197-raw_2
## Single-end reads will be in luad-0197-raw_M
## 3356602 sequences in the BAM file
## 3356602 sequences exported
## 3356602 reads; of these:
##   3356602 (100.00%) were unpaired; of these:
##     655349 (19.52%) aligned 0 times
##     1707148 (50.86%) aligned exactly 1 time
##     994105 (29.62%) aligned &gt;1 times
## 80.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 994105/2701253
## 2 771901/994105
## 3 4087581/9121081
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0198-raw_1 and luad-0198-raw_2
## Single-end reads will be in luad-0198-raw_M
## 5745923 sequences in the BAM file
## 5745923 sequences exported
## 5745923 reads; of these:
##   5745923 (100.00%) were unpaired; of these:
##     1321645 (23.00%) aligned 0 times
##     3128263 (54.44%) aligned exactly 1 time
##     1296015 (22.56%) aligned &gt;1 times
## 77.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1296015/4424278
## 2 921020/1296015
## 3 7079475/16924749
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0199-raw_1 and luad-0199-raw_2
## Single-end reads will be in luad-0199-raw_M
## 2778936 sequences in the BAM file
## 2778936 sequences exported
## 2778936 reads; of these:
##   2778936 (100.00%) were unpaired; of these:
##     553261 (19.91%) aligned 0 times
##     1177290 (42.36%) aligned exactly 1 time
##     1048385 (37.73%) aligned &gt;1 times
## 80.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1048385/2225675
## 2 816811/1048385
## 3 8198578/14919902
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0200-raw_1 and luad-0200-raw_2
## Single-end reads will be in luad-0200-raw_M
## 2197154 sequences in the BAM file
## 2197154 sequences exported
## 2197154 reads; of these:
##   2197154 (100.00%) were unpaired; of these:
##     511423 (23.28%) aligned 0 times
##     631962 (28.76%) aligned exactly 1 time
##     1053769 (47.96%) aligned &gt;1 times
## 76.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1053769/1685731
## 2 800529/1053769
## 3 13500410/22778079
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0201-raw_1 and luad-0201-raw_2
## Single-end reads will be in luad-0201-raw_M
## 2362329 sequences in the BAM file
## 2362329 sequences exported
## 2362329 reads; of these:
##   2362329 (100.00%) were unpaired; of these:
##     558140 (23.63%) aligned 0 times
##     1243651 (52.65%) aligned exactly 1 time
##     560538 (23.73%) aligned &gt;1 times
## 76.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 560538/1804189
## 2 395924/560538
## 3 4077536/7832679
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0202-raw_1 and luad-0202-raw_2
## Single-end reads will be in luad-0202-raw_M
## 3874583 sequences in the BAM file
## 3874583 sequences exported
## 3874583 reads; of these:
##   3874583 (100.00%) were unpaired; of these:
##     765524 (19.76%) aligned 0 times
##     2033143 (52.47%) aligned exactly 1 time
##     1075916 (27.77%) aligned &gt;1 times
## 80.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1075916/3109059
## 2 829829/1075916
## 3 8727719/16019830
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0203-raw_1 and luad-0203-raw_2
## Single-end reads will be in luad-0203-raw_M
## 14579004 sequences in the BAM file
## 14579004 sequences exported
## 14579004 reads; of these:
##   14579004 (100.00%) were unpaired; of these:
##     918641 (6.30%) aligned 0 times
##     7535374 (51.69%) aligned exactly 1 time
##     6124989 (42.01%) aligned &gt;1 times
## 93.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6124989/13660363
## 2 4697866/6124989
## 3 44019396/83582105
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in luad-0204-raw_1 and luad-0204-raw_2
## Single-end reads will be in luad-0204-raw_M
## 2834592 sequences in the BAM file
## 2834592 sequences exported
## 2834592 reads; of these:
##   2834592 (100.00%) were unpaired; of these:
##     615384 (21.71%) aligned 0 times
##     1494766 (52.73%) aligned exactly 1 time
##     724442 (25.56%) aligned &gt;1 times
## 78.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 724442/2219208
## 2 529484/724442
## 3 6803490/12736909
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0205-raw_1 and luad-0205-raw_2
## Single-end reads will be in luad-0205-raw_M
## 8170060 sequences in the BAM file
## 8170060 sequences exported
## 8170060 reads; of these:
##   8170060 (100.00%) were unpaired; of these:
##     2381393 (29.15%) aligned 0 times
##     3784559 (46.32%) aligned exactly 1 time
##     2004108 (24.53%) aligned &gt;1 times
## 70.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2004108/5788667
## 2 1364418/2004108
## 3 16199458/30054051
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0206-raw_1 and luad-0206-raw_2
## Single-end reads will be in luad-0206-raw_M
## 8368961 sequences in the BAM file
## 8368961 sequences exported
## 8368961 reads; of these:
##   8368961 (100.00%) were unpaired; of these:
##     1714111 (20.48%) aligned 0 times
##     3097891 (37.02%) aligned exactly 1 time
##     3556959 (42.50%) aligned &gt;1 times
## 79.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3556959/6654850
## 2 2224436/3556959
## 3 42240454/79633652
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 1.
## Output will be in luad-0207-raw_1 and luad-0207-raw_2
## Single-end reads will be in luad-0207-raw_M
## 9362644 sequences in the BAM file
## 9362644 sequences exported
## 9362644 reads; of these:
##   9362644 (100.00%) were unpaired; of these:
##     2939240 (31.39%) aligned 0 times
##     4789150 (51.15%) aligned exactly 1 time
##     1634254 (17.46%) aligned &gt;1 times
## 68.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1634254/6423404
## 2 1049851/1634254
## 3 4807537/14892943
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0208-raw_1 and luad-0208-raw_2
## Single-end reads will be in luad-0208-raw_M
## 5501827 sequences in the BAM file
## 5501827 sequences exported
## 5501827 reads; of these:
##   5501827 (100.00%) were unpaired; of these:
##     2514939 (45.71%) aligned 0 times
##     2308260 (41.95%) aligned exactly 1 time
##     678628 (12.33%) aligned &gt;1 times
## 54.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 678628/2986888
## 2 363156/678628
## 3 1706156/6083367
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0209-raw_1 and luad-0209-raw_2
## Single-end reads will be in luad-0209-raw_M
## 7745195 sequences in the BAM file
## 7745195 sequences exported
## 7745195 reads; of these:
##   7745195 (100.00%) were unpaired; of these:
##     3228555 (41.68%) aligned 0 times
##     3730829 (48.17%) aligned exactly 1 time
##     785811 (10.15%) aligned &gt;1 times
## 58.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 785811/4516640
## 2 466460/785811
## 3 1837861/7876841
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0210-raw_1 and luad-0210-raw_2
## Single-end reads will be in luad-0210-raw_M
## 6336093 sequences in the BAM file
## 6336093 sequences exported
## 6336093 reads; of these:
##   6336093 (100.00%) were unpaired; of these:
##     2177344 (34.36%) aligned 0 times
##     2815516 (44.44%) aligned exactly 1 time
##     1343233 (21.20%) aligned &gt;1 times
## 65.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1343233/4158749
## 2 703112/1343233
## 3 7256879/17806611
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0211-raw_1 and luad-0211-raw_2
## Single-end reads will be in luad-0211-raw_M
## 10132254 sequences in the BAM file
## 10132254 sequences exported
## 10132254 reads; of these:
##   10132254 (100.00%) were unpaired; of these:
##     3631124 (35.84%) aligned 0 times
##     4770754 (47.08%) aligned exactly 1 time
##     1730376 (17.08%) aligned &gt;1 times
## 64.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1730376/6501130
## 2 1149801/1730376
## 3 5476843/15800104
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0212-raw_1 and luad-0212-raw_2
## Single-end reads will be in luad-0212-raw_M
## 10965347 sequences in the BAM file
## 10965347 sequences exported
## 10965347 reads; of these:
##   10965347 (100.00%) were unpaired; of these:
##     2659981 (24.26%) aligned 0 times
##     6603165 (60.22%) aligned exactly 1 time
##     1702201 (15.52%) aligned &gt;1 times
## 75.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1702201/8305366
## 2 1235581/1702201
## 3 5985643/18456855
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0213-raw_1 and luad-0213-raw_2
## Single-end reads will be in luad-0213-raw_M
## 10413270 sequences in the BAM file
## 10413270 sequences exported
## 10413270 reads; of these:
##   10413270 (100.00%) were unpaired; of these:
##     3644765 (35.00%) aligned 0 times
##     5151069 (49.47%) aligned exactly 1 time
##     1617436 (15.53%) aligned &gt;1 times
## 65.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1617436/6768505
## 2 971255/1617436
## 3 6626837/18100282
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0214-raw_1 and luad-0214-raw_2
## Single-end reads will be in luad-0214-raw_M
## 13056365 sequences in the BAM file
## 13056365 sequences exported
## 13056365 reads; of these:
##   13056365 (100.00%) were unpaired; of these:
##     3188456 (24.42%) aligned 0 times
##     5677933 (43.49%) aligned exactly 1 time
##     4189976 (32.09%) aligned &gt;1 times
## 75.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4189976/9867909
## 2 3026641/4189976
## 3 16836762/35651502
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0215-raw_1 and luad-0215-raw_2
## Single-end reads will be in luad-0215-raw_M
## 15275910 sequences in the BAM file
## 15275910 sequences exported
## 15275910 reads; of these:
##   15275910 (100.00%) were unpaired; of these:
##     2750437 (18.01%) aligned 0 times
##     6687847 (43.78%) aligned exactly 1 time
##     5837626 (38.21%) aligned &gt;1 times
## 81.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5837626/12525473
## 2 3867469/5837626
## 3 62574584/118779715
## [bam_sort_core] merging from 16 files...
## This looks like paired data from lane 7.
## Output will be in luad-0216-raw_1 and luad-0216-raw_2
## Single-end reads will be in luad-0216-raw_M
## 8576654 sequences in the BAM file
## 8576654 sequences exported
## 8576654 reads; of these:
##   8576654 (100.00%) were unpaired; of these:
##     1319426 (15.38%) aligned 0 times
##     4956059 (57.79%) aligned exactly 1 time
##     2301169 (26.83%) aligned &gt;1 times
## 84.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2301169/7257228
## 2 1570654/2301169
## 3 17782843/39244349
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0217-raw_1 and luad-0217-raw_2
## Single-end reads will be in luad-0217-raw_M
## 16535664 sequences in the BAM file
## 16535664 sequences exported
## 16535664 reads; of these:
##   16535664 (100.00%) were unpaired; of these:
##     6708244 (40.57%) aligned 0 times
##     8284723 (50.10%) aligned exactly 1 time
##     1542697 (9.33%) aligned &gt;1 times
## 59.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1542697/9827420
## 2 793407/1542697
## 3 3118377/15619276
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0218-raw_1 and luad-0218-raw_2
## Single-end reads will be in luad-0218-raw_M
## 16690884 sequences in the BAM file
## 16690884 sequences exported
## 16690884 reads; of these:
##   16690884 (100.00%) were unpaired; of these:
##     3166630 (18.97%) aligned 0 times
##     9280125 (55.60%) aligned exactly 1 time
##     4244129 (25.43%) aligned &gt;1 times
## 81.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4244129/13524254
## 2 2978603/4244129
## 3 22142562/48831132
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in luad-0219-raw_1 and luad-0219-raw_2
## Single-end reads will be in luad-0219-raw_M
## 10103377 sequences in the BAM file
## 10103377 sequences exported
## 10103377 reads; of these:
##   10103377 (100.00%) were unpaired; of these:
##     2000156 (19.80%) aligned 0 times
##     5309786 (52.55%) aligned exactly 1 time
##     2793435 (27.65%) aligned &gt;1 times
## 80.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2793435/8103221
## 2 2042041/2793435
## 3 22137398/49612552
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in luad-0220-raw_1 and luad-0220-raw_2
## Single-end reads will be in luad-0220-raw_M
## 10812443 sequences in the BAM file
## 10812443 sequences exported
## 10812443 reads; of these:
##   10812443 (100.00%) were unpaired; of these:
##     3067508 (28.37%) aligned 0 times
##     5263341 (48.68%) aligned exactly 1 time
##     2481594 (22.95%) aligned &gt;1 times
## 71.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2481594/7744935
## 2 1814225/2481594
## 3 12425276/26062099
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0221-raw_1 and luad-0221-raw_2
## Single-end reads will be in luad-0221-raw_M
## 13964579 sequences in the BAM file
## 13964579 sequences exported
## 13964579 reads; of these:
##   13964579 (100.00%) were unpaired; of these:
##     5778924 (41.38%) aligned 0 times
##     5259278 (37.66%) aligned exactly 1 time
##     2926377 (20.96%) aligned &gt;1 times
## 58.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2926377/8185655
## 2 1912015/2926377
## 3 7628162/21032426
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0222-raw_1 and luad-0222-raw_2
## Single-end reads will be in luad-0222-raw_M
## 10301201 sequences in the BAM file
## 10301201 sequences exported
## 10301201 reads; of these:
##   10301201 (100.00%) were unpaired; of these:
##     2624637 (25.48%) aligned 0 times
##     5787548 (56.18%) aligned exactly 1 time
##     1889016 (18.34%) aligned &gt;1 times
## 74.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1889016/7676564
## 2 1139736/1889016
## 3 9933825/24539684
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0223-raw_1 and luad-0223-raw_2
## Single-end reads will be in luad-0223-raw_M
## 13045620 sequences in the BAM file
## 13045620 sequences exported
## 13045620 reads; of these:
##   13045620 (100.00%) were unpaired; of these:
##     2836792 (21.75%) aligned 0 times
##     6853505 (52.53%) aligned exactly 1 time
##     3355323 (25.72%) aligned &gt;1 times
## 78.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3355323/10208828
## 2 2482315/3355323
## 3 17655741/37753038
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0224-raw_1 and luad-0224-raw_2
## Single-end reads will be in luad-0224-raw_M
## 14289739 sequences in the BAM file
## 14289739 sequences exported
## 14289739 reads; of these:
##   14289739 (100.00%) were unpaired; of these:
##     3286971 (23.00%) aligned 0 times
##     7963792 (55.73%) aligned exactly 1 time
##     3038976 (21.27%) aligned &gt;1 times
## 77.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3038976/11002768
## 2 2170610/3038976
## 3 11432447/27869753
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0225-raw_1 and luad-0225-raw_2
## Single-end reads will be in luad-0225-raw_M
## 13626162 sequences in the BAM file
## 13626162 sequences exported
## 13626162 reads; of these:
##   13626162 (100.00%) were unpaired; of these:
##     2479575 (18.20%) aligned 0 times
##     7393630 (54.26%) aligned exactly 1 time
##     3752957 (27.54%) aligned &gt;1 times
## 81.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3752957/11146587
## 2 2764183/3752957
## 3 13797189/31833533
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0226-raw_1 and luad-0226-raw_2
## Single-end reads will be in luad-0226-raw_M
## 6023133 sequences in the BAM file
## 6023133 sequences exported
## 6023133 reads; of these:
##   6023133 (100.00%) were unpaired; of these:
##     1884415 (31.29%) aligned 0 times
##     2988136 (49.61%) aligned exactly 1 time
##     1150582 (19.10%) aligned &gt;1 times
## 68.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1150582/4138718
## 2 636361/1150582
## 3 4040418/11250714
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0227-raw_1 and luad-0227-raw_2
## Single-end reads will be in luad-0227-raw_M
## 13804201 sequences in the BAM file
## 13804201 sequences exported
## 13804201 reads; of these:
##   13804201 (100.00%) were unpaired; of these:
##     3843072 (27.84%) aligned 0 times
##     6055120 (43.86%) aligned exactly 1 time
##     3906009 (28.30%) aligned &gt;1 times
## 72.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3906009/9961129
## 2 2810867/3906009
## 3 22196792/41035007
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0228-raw_1 and luad-0228-raw_2
## Single-end reads will be in luad-0228-raw_M
## 11068675 sequences in the BAM file
## 11068675 sequences exported
## 11068675 reads; of these:
##   11068675 (100.00%) were unpaired; of these:
##     4763748 (43.04%) aligned 0 times
##     4786725 (43.25%) aligned exactly 1 time
##     1518202 (13.72%) aligned &gt;1 times
## 56.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1518202/6304927
## 2 906493/1518202
## 3 3958857/13474671
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0229-raw_1 and luad-0229-raw_2
## Single-end reads will be in luad-0229-raw_M
## 9839979 sequences in the BAM file
## 9839979 sequences exported
## 9839979 reads; of these:
##   9839979 (100.00%) were unpaired; of these:
##     1845105 (18.75%) aligned 0 times
##     6063900 (61.63%) aligned exactly 1 time
##     1930974 (19.62%) aligned &gt;1 times
## 81.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1930974/7994874
## 2 1450886/1930974
## 3 5592859/17364322
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0230-raw_1 and luad-0230-raw_2
## Single-end reads will be in luad-0230-raw_M
## 15220945 sequences in the BAM file
## 15220945 sequences exported
## 15220945 reads; of these:
##   15220945 (100.00%) were unpaired; of these:
##     5325125 (34.99%) aligned 0 times
##     7057238 (46.37%) aligned exactly 1 time
##     2838582 (18.65%) aligned &gt;1 times
## 65.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2838582/9895820
## 2 1809833/2838582
## 3 9788604/25551673
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0231-raw_1 and luad-0231-raw_2
## Single-end reads will be in luad-0231-raw_M
## 15383696 sequences in the BAM file
## 15383696 sequences exported
## 15383696 reads; of these:
##   15383696 (100.00%) were unpaired; of these:
##     2549578 (16.57%) aligned 0 times
##     8547453 (55.56%) aligned exactly 1 time
##     4286665 (27.86%) aligned &gt;1 times
## 83.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4286665/12834118
## 2 2772370/4286665
## 3 24866755/56882095
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in luad-0232-raw_1 and luad-0232-raw_2
## Single-end reads will be in luad-0232-raw_M
## 9992683 sequences in the BAM file
## 9992683 sequences exported
## 9992683 reads; of these:
##   9992683 (100.00%) were unpaired; of these:
##     1782965 (17.84%) aligned 0 times
##     4842737 (48.46%) aligned exactly 1 time
##     3366981 (33.69%) aligned &gt;1 times
## 82.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3366981/8209718
## 2 2566299/3366981
## 3 26487625/54754339
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0233-raw_1 and luad-0233-raw_2
## Single-end reads will be in luad-0233-raw_M
## 11224018 sequences in the BAM file
## 11224018 sequences exported
## 11224018 reads; of these:
##   11224018 (100.00%) were unpaired; of these:
##     1826093 (16.27%) aligned 0 times
##     6036166 (53.78%) aligned exactly 1 time
##     3361759 (29.95%) aligned &gt;1 times
## 83.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3361759/9397925
## 2 2531276/3361759
## 3 22059734/40925888
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0234-raw_1 and luad-0234-raw_2
## Single-end reads will be in luad-0234-raw_M
## 13241725 sequences in the BAM file
## 13241725 sequences exported
## 13241725 reads; of these:
##   13241725 (100.00%) were unpaired; of these:
##     3448460 (26.04%) aligned 0 times
##     6730434 (50.83%) aligned exactly 1 time
##     3062831 (23.13%) aligned &gt;1 times
## 73.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3062831/9793265
## 2 2237271/3062831
## 3 12119781/29134862
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0235-raw_1 and luad-0235-raw_2
## Single-end reads will be in luad-0235-raw_M
## 14969397 sequences in the BAM file
## 14969397 sequences exported
## 14969397 reads; of these:
##   14969397 (100.00%) were unpaired; of these:
##     3042363 (20.32%) aligned 0 times
##     7666011 (51.21%) aligned exactly 1 time
##     4261023 (28.46%) aligned &gt;1 times
## 79.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4261023/11927034
## 2 2923960/4261023
## 3 14114005/33257317
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0236-raw_1 and luad-0236-raw_2
## Single-end reads will be in luad-0236-raw_M
## 6346721 sequences in the BAM file
## 6346721 sequences exported
## 6346721 reads; of these:
##   6346721 (100.00%) were unpaired; of these:
##     1464944 (23.08%) aligned 0 times
##     3667290 (57.78%) aligned exactly 1 time
##     1214487 (19.14%) aligned &gt;1 times
## 76.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1214487/4881777
## 2 892354/1214487
## 3 4416692/11439638
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0237-raw_1 and luad-0237-raw_2
## Single-end reads will be in luad-0237-raw_M
## 11239954 sequences in the BAM file
## 11239954 sequences exported
## 11239954 reads; of these:
##   11239954 (100.00%) were unpaired; of these:
##     4929949 (43.86%) aligned 0 times
##     4808331 (42.78%) aligned exactly 1 time
##     1501674 (13.36%) aligned &gt;1 times
## 56.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1501674/6310005
## 2 827300/1501674
## 3 4671134/14614948
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0238-raw_1 and luad-0238-raw_2
## Single-end reads will be in luad-0238-raw_M
## 17158452 sequences in the BAM file
## 17158452 sequences exported
## 17158452 reads; of these:
##   17158452 (100.00%) were unpaired; of these:
##     2254102 (13.14%) aligned 0 times
##     9099587 (53.03%) aligned exactly 1 time
##     5804763 (33.83%) aligned &gt;1 times
## 86.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5804763/14904350
## 2 4025981/5804763
## 3 38030961/81004632
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 7.
## Output will be in luad-0239-raw_1 and luad-0239-raw_2
## Single-end reads will be in luad-0239-raw_M
## 18537140 sequences in the BAM file
## 18537140 sequences exported
## 18537140 reads; of these:
##   18537140 (100.00%) were unpaired; of these:
##     3424680 (18.47%) aligned 0 times
##     10975331 (59.21%) aligned exactly 1 time
##     4137129 (22.32%) aligned &gt;1 times
## 81.53% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4137129/15112460
## 2 2972144/4137129
## 3 22726631/54417342
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 1.
## Output will be in luad-0240-raw_1 and luad-0240-raw_2
## Single-end reads will be in luad-0240-raw_M
## 5122061 sequences in the BAM file
## 5122061 sequences exported
## 5122061 reads; of these:
##   5122061 (100.00%) were unpaired; of these:
##     1770378 (34.56%) aligned 0 times
##     2428432 (47.41%) aligned exactly 1 time
##     923251 (18.02%) aligned &gt;1 times
## 65.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 923251/3351683
## 2 572194/923251
## 3 3552509/9433550
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0241-raw_1 and luad-0241-raw_2
## Single-end reads will be in luad-0241-raw_M
## 6029310 sequences in the BAM file
## 6029310 sequences exported
## 6029310 reads; of these:
##   6029310 (100.00%) were unpaired; of these:
##     1516592 (25.15%) aligned 0 times
##     2965966 (49.19%) aligned exactly 1 time
##     1546752 (25.65%) aligned &gt;1 times
## 74.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1546752/4512718
## 2 1119492/1546752
## 3 6857166/15456408
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0242-raw_1 and luad-0242-raw_2
## Single-end reads will be in luad-0242-raw_M
## 13941099 sequences in the BAM file
## 13941099 sequences exported
## 13941099 reads; of these:
##   13941099 (100.00%) were unpaired; of these:
##     5532008 (39.68%) aligned 0 times
##     6402777 (45.93%) aligned exactly 1 time
##     2006314 (14.39%) aligned &gt;1 times
## 60.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2006314/8409091
## 2 1176346/2006314
## 3 4766878/16586356
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0243-raw_1 and luad-0243-raw_2
## Single-end reads will be in luad-0243-raw_M
## 8896336 sequences in the BAM file
## 8896336 sequences exported
## 8896336 reads; of these:
##   8896336 (100.00%) were unpaired; of these:
##     1697329 (19.08%) aligned 0 times
##     4879285 (54.85%) aligned exactly 1 time
##     2319722 (26.08%) aligned &gt;1 times
## 80.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2319722/7199007
## 2 1655476/2319722
## 3 14486845/31237391
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0244-raw_1 and luad-0244-raw_2
## Single-end reads will be in luad-0244-raw_M
## 11730435 sequences in the BAM file
## 11730435 sequences exported
## 11730435 reads; of these:
##   11730435 (100.00%) were unpaired; of these:
##     3099297 (26.42%) aligned 0 times
##     5639178 (48.07%) aligned exactly 1 time
##     2991960 (25.51%) aligned &gt;1 times
## 73.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2991960/8631138
## 2 2181972/2991960
## 3 8505321/21880200
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0245-raw_1 and luad-0245-raw_2
## Single-end reads will be in luad-0245-raw_M
## 9299373 sequences in the BAM file
## 9299373 sequences exported
## 9299373 reads; of these:
##   9299373 (100.00%) were unpaired; of these:
##     2790631 (30.01%) aligned 0 times
##     3754310 (40.37%) aligned exactly 1 time
##     2754432 (29.62%) aligned &gt;1 times
## 69.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2754432/6508742
## 2 1593857/2754432
## 3 17207291/36893341
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0246-raw_1 and luad-0246-raw_2
## Single-end reads will be in luad-0246-raw_M
## 9108442 sequences in the BAM file
## 9108442 sequences exported
## 9108442 reads; of these:
##   9108442 (100.00%) were unpaired; of these:
##     1640637 (18.01%) aligned 0 times
##     5489586 (60.27%) aligned exactly 1 time
##     1978219 (21.72%) aligned &gt;1 times
## 81.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1978219/7467805
## 2 1436434/1978219
## 3 10816957/24099472
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0247-raw_1 and luad-0247-raw_2
## Single-end reads will be in luad-0247-raw_M
## 18091573 sequences in the BAM file
## 18091573 sequences exported
## 18091573 reads; of these:
##   18091573 (100.00%) were unpaired; of these:
##     4624218 (25.56%) aligned 0 times
##     9664876 (53.42%) aligned exactly 1 time
##     3802479 (21.02%) aligned &gt;1 times
## 74.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3802479/13467355
## 2 2433025/3802479
## 3 11510112/32639982
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0248-raw_1 and luad-0248-raw_2
## Single-end reads will be in luad-0248-raw_M
## 5151418 sequences in the BAM file
## 5151418 sequences exported
## 5151418 reads; of these:
##   5151418 (100.00%) were unpaired; of these:
##     1095680 (21.27%) aligned 0 times
##     2282150 (44.30%) aligned exactly 1 time
##     1773588 (34.43%) aligned &gt;1 times
## 78.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1773588/4055738
## 2 1173531/1773588
## 3 9537015/20315143
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0249-raw_1 and luad-0249-raw_2
## Single-end reads will be in luad-0249-raw_M
## 12287474 sequences in the BAM file
## 12287474 sequences exported
## 12287474 reads; of these:
##   12287474 (100.00%) were unpaired; of these:
##     2196804 (17.88%) aligned 0 times
##     6744921 (54.89%) aligned exactly 1 time
##     3345749 (27.23%) aligned &gt;1 times
## 82.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3345749/10090670
## 2 2441129/3345749
## 3 13707571/30255745
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0250-raw_1 and luad-0250-raw_2
## Single-end reads will be in luad-0250-raw_M
## 7996874 sequences in the BAM file
## 7996874 sequences exported
## 7996874 reads; of these:
##   7996874 (100.00%) were unpaired; of these:
##     1841993 (23.03%) aligned 0 times
##     3843167 (48.06%) aligned exactly 1 time
##     2311714 (28.91%) aligned &gt;1 times
## 76.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2311714/6154881
## 2 1600785/2311714
## 3 9701408/22890182
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0251-raw_1 and luad-0251-raw_2
## Single-end reads will be in luad-0251-raw_M
## 8610446 sequences in the BAM file
## 8610446 sequences exported
## 8610446 reads; of these:
##   8610446 (100.00%) were unpaired; of these:
##     1901270 (22.08%) aligned 0 times
##     4637675 (53.86%) aligned exactly 1 time
##     2071501 (24.06%) aligned &gt;1 times
## 77.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2071501/6709176
## 2 1508297/2071501
## 3 6279271/16194458
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0252-raw_1 and luad-0252-raw_2
## Single-end reads will be in luad-0252-raw_M
## 13193027 sequences in the BAM file
## 13193027 sequences exported
## 13193027 reads; of these:
##   13193027 (100.00%) were unpaired; of these:
##     2001766 (15.17%) aligned 0 times
##     7972727 (60.43%) aligned exactly 1 time
##     3218534 (24.40%) aligned &gt;1 times
## 84.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3218534/11191261
## 2 2526394/3218534
## 3 21834765/41818091
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0253-raw_1 and luad-0253-raw_2
## Single-end reads will be in luad-0253-raw_M
## 9636780 sequences in the BAM file
## 9636780 sequences exported
## 9636780 reads; of these:
##   9636780 (100.00%) were unpaired; of these:
##     3215890 (33.37%) aligned 0 times
##     4389192 (45.55%) aligned exactly 1 time
##     2031698 (21.08%) aligned &gt;1 times
## 66.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2031698/6420890
## 2 1238264/2031698
## 3 5343544/15404177
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0254-raw_1 and luad-0254-raw_2
## Single-end reads will be in luad-0254-raw_M
## 11998083 sequences in the BAM file
## 11998083 sequences exported
## 11998083 reads; of these:
##   11998083 (100.00%) were unpaired; of these:
##     2079123 (17.33%) aligned 0 times
##     6388787 (53.25%) aligned exactly 1 time
##     3530173 (29.42%) aligned &gt;1 times
## 82.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3530173/9918960
## 2 2840032/3530173
## 3 11037261/27158724
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0255-raw_1 and luad-0255-raw_2
## Single-end reads will be in luad-0255-raw_M
## 7974849 sequences in the BAM file
## 7974849 sequences exported
## 7974849 reads; of these:
##   7974849 (100.00%) were unpaired; of these:
##     2823967 (35.41%) aligned 0 times
##     3443356 (43.18%) aligned exactly 1 time
##     1707526 (21.41%) aligned &gt;1 times
## 64.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1707526/5150882
## 2 1028595/1707526
## 3 7876914/18807046
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0256-raw_1 and luad-0256-raw_2
## Single-end reads will be in luad-0256-raw_M
## 4491611 sequences in the BAM file
## 4491611 sequences exported
## 4491611 reads; of these:
##   4491611 (100.00%) were unpaired; of these:
##     1037869 (23.11%) aligned 0 times
##     2415639 (53.78%) aligned exactly 1 time
##     1038103 (23.11%) aligned &gt;1 times
## 76.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1038103/3453742
## 2 750462/1038103
## 3 3851718/9321381
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0257-raw_1 and luad-0257-raw_2
## Single-end reads will be in luad-0257-raw_M
## 10219660 sequences in the BAM file
## 10219660 sequences exported
## 10219660 reads; of these:
##   10219660 (100.00%) were unpaired; of these:
##     3994243 (39.08%) aligned 0 times
##     4729323 (46.28%) aligned exactly 1 time
##     1496094 (14.64%) aligned &gt;1 times
## 60.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1496094/6225417
## 2 985262/1496094
## 3 5413028/15495521
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0258-raw_1 and luad-0258-raw_2
## Single-end reads will be in luad-0258-raw_M
## 15349473 sequences in the BAM file
## 15349473 sequences exported
## 15349473 reads; of these:
##   15349473 (100.00%) were unpaired; of these:
##     6305895 (41.08%) aligned 0 times
##     6313893 (41.13%) aligned exactly 1 time
##     2729685 (17.78%) aligned &gt;1 times
## 58.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2729685/9043578
## 2 1560584/2729685
## 3 5132604/18125826
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0259-raw_1 and luad-0259-raw_2
## Single-end reads will be in luad-0259-raw_M
## 7242532 sequences in the BAM file
## 7242532 sequences exported
## 7242532 reads; of these:
##   7242532 (100.00%) were unpaired; of these:
##     3217722 (44.43%) aligned 0 times
##     2770989 (38.26%) aligned exactly 1 time
##     1253821 (17.31%) aligned &gt;1 times
## 55.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1253821/4024810
## 2 723477/1253821
## 3 3714545/10767067
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0260-raw_1 and luad-0260-raw_2
## Single-end reads will be in luad-0260-raw_M
## 13802873 sequences in the BAM file
## 13802873 sequences exported
## 13802873 reads; of these:
##   13802873 (100.00%) were unpaired; of these:
##     5817266 (42.15%) aligned 0 times
##     6507074 (47.14%) aligned exactly 1 time
##     1478533 (10.71%) aligned &gt;1 times
## 57.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1478533/7985607
## 2 941952/1478533
## 3 5509618/17736183
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0261-raw_1 and luad-0261-raw_2
## Single-end reads will be in luad-0261-raw_M
## 21246513 sequences in the BAM file
## 21246513 sequences exported
## 21246513 reads; of these:
##   21246513 (100.00%) were unpaired; of these:
##     3232442 (15.21%) aligned 0 times
##     12086410 (56.89%) aligned exactly 1 time
##     5927661 (27.90%) aligned &gt;1 times
## 84.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5927661/18014071
## 2 4204338/5927661
## 3 39903878/84314733
## [bam_sort_core] merging from 13 files...
## This looks like paired data from lane 6.
## Output will be in luad-0262-raw_1 and luad-0262-raw_2
## Single-end reads will be in luad-0262-raw_M
## 7010489 sequences in the BAM file
## 7010489 sequences exported
## 7010489 reads; of these:
##   7010489 (100.00%) were unpaired; of these:
##     2342527 (33.41%) aligned 0 times
##     3299855 (47.07%) aligned exactly 1 time
##     1368107 (19.52%) aligned &gt;1 times
## 66.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1368107/4667962
## 2 951148/1368107
## 3 4355261/11403788
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in luad-0263-raw_1 and luad-0263-raw_2
## Single-end reads will be in luad-0263-raw_M
## 9827519 sequences in the BAM file
## 9827519 sequences exported
## 9827519 reads; of these:
##   9827519 (100.00%) were unpaired; of these:
##     2287610 (23.28%) aligned 0 times
##     4332853 (44.09%) aligned exactly 1 time
##     3207056 (32.63%) aligned &gt;1 times
## 76.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3207056/7539909
## 2 1906217/3207056
## 3 7505555/20234931
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0264-raw_1 and luad-0264-raw_2
## Single-end reads will be in luad-0264-raw_M
## 8218665 sequences in the BAM file
## 8218665 sequences exported
## 8218665 reads; of these:
##   8218665 (100.00%) were unpaired; of these:
##     2824260 (34.36%) aligned 0 times
##     3732353 (45.41%) aligned exactly 1 time
##     1662052 (20.22%) aligned &gt;1 times
## 65.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1662052/5394405
## 2 1361252/1662052
## 3 3209614/10173421
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0265-raw_1 and luad-0265-raw_2
## Single-end reads will be in luad-0265-raw_M
## 21175840 sequences in the BAM file
## 21175840 sequences exported
## 21175840 reads; of these:
##   21175840 (100.00%) were unpaired; of these:
##     3365404 (15.89%) aligned 0 times
##     12189199 (57.56%) aligned exactly 1 time
##     5621237 (26.55%) aligned &gt;1 times
## 84.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5621237/17810436
## 2 4131266/5621237
## 3 34584729/71862736
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 4.
## Output will be in luad-0266-raw_1 and luad-0266-raw_2
## Single-end reads will be in luad-0266-raw_M
## 10371031 sequences in the BAM file
## 10371031 sequences exported
## 10371031 reads; of these:
##   10371031 (100.00%) were unpaired; of these:
##     2661172 (25.66%) aligned 0 times
##     4585610 (44.22%) aligned exactly 1 time
##     3124249 (30.12%) aligned &gt;1 times
## 74.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3124249/7709859
## 2 2177328/3124249
## 3 12669650/26879493
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0267-raw_1 and luad-0267-raw_2
## Single-end reads will be in luad-0267-raw_M
## 12509651 sequences in the BAM file
## 12509651 sequences exported
## 12509651 reads; of these:
##   12509651 (100.00%) were unpaired; of these:
##     4910129 (39.25%) aligned 0 times
##     6227805 (49.78%) aligned exactly 1 time
##     1371717 (10.97%) aligned &gt;1 times
## 60.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1371717/7599522
## 2 753675/1371717
## 3 2772565/12700422
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0268-raw_1 and luad-0268-raw_2
## Single-end reads will be in luad-0268-raw_M
## 4223989 sequences in the BAM file
## 4223989 sequences exported
## 4223989 reads; of these:
##   4223989 (100.00%) were unpaired; of these:
##     1250600 (29.61%) aligned 0 times
##     1878150 (44.46%) aligned exactly 1 time
##     1095239 (25.93%) aligned &gt;1 times
## 70.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1095239/2973389
## 2 799727/1095239
## 3 3483813/8452830
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in luad-0269-raw_1 and luad-0269-raw_2
## Single-end reads will be in luad-0269-raw_M
## 9998599 sequences in the BAM file
## 9998599 sequences exported
## 9998599 reads; of these:
##   9998599 (100.00%) were unpaired; of these:
##     1723906 (17.24%) aligned 0 times
##     5483544 (54.84%) aligned exactly 1 time
##     2791149 (27.92%) aligned &gt;1 times
## 82.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2791149/8274693
## 2 1950959/2791149
## 3 15587908/33397542
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0270-raw_1 and luad-0270-raw_2
## Single-end reads will be in luad-0270-raw_M
## 8843089 sequences in the BAM file
## 8843089 sequences exported
## 8843089 reads; of these:
##   8843089 (100.00%) were unpaired; of these:
##     1157269 (13.09%) aligned 0 times
##     4706912 (53.23%) aligned exactly 1 time
##     2978908 (33.69%) aligned &gt;1 times
## 86.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2978908/7685820
## 2 2139362/2978908
## 3 9933064/23126317
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0271-raw_1 and luad-0271-raw_2
## Single-end reads will be in luad-0271-raw_M
## 12285763 sequences in the BAM file
## 12285763 sequences exported
## 12285763 reads; of these:
##   12285763 (100.00%) were unpaired; of these:
##     4732054 (38.52%) aligned 0 times
##     6331408 (51.53%) aligned exactly 1 time
##     1222301 (9.95%) aligned &gt;1 times
## 61.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1222301/7553709
## 2 687261/1222301
## 3 2807818/12814095
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0272-raw_1 and luad-0272-raw_2
## Single-end reads will be in luad-0272-raw_M
## 8285683 sequences in the BAM file
## 8285683 sequences exported
## 8285683 reads; of these:
##   8285683 (100.00%) were unpaired; of these:
##     3025074 (36.51%) aligned 0 times
##     4033583 (48.68%) aligned exactly 1 time
##     1227026 (14.81%) aligned &gt;1 times
## 63.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1227026/5260609
## 2 739545/1227026
## 3 4061120/12167095
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0273-raw_1 and luad-0273-raw_2
## Single-end reads will be in luad-0273-raw_M
## 10872535 sequences in the BAM file
## 10872535 sequences exported
## 10872535 reads; of these:
##   10872535 (100.00%) were unpaired; of these:
##     2580905 (23.74%) aligned 0 times
##     4934853 (45.39%) aligned exactly 1 time
##     3356777 (30.87%) aligned &gt;1 times
## 76.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3356777/8291630
## 2 2352814/3356777
## 3 21159738/52209364
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 3.
## Output will be in luad-0274-raw_1 and luad-0274-raw_2
## Single-end reads will be in luad-0274-raw_M
## 16092885 sequences in the BAM file
## 16092885 sequences exported
## 16092885 reads; of these:
##   16092885 (100.00%) were unpaired; of these:
##     5339274 (33.18%) aligned 0 times
##     7450871 (46.30%) aligned exactly 1 time
##     3302740 (20.52%) aligned &gt;1 times
## 66.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3302740/10753611
## 2 2176620/3302740
## 3 10204465/27183894
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0275-raw_1 and luad-0275-raw_2
## Single-end reads will be in luad-0275-raw_M
## 23780311 sequences in the BAM file
## 23780311 sequences exported
## 23780311 reads; of these:
##   23780311 (100.00%) were unpaired; of these:
##     3895646 (16.38%) aligned 0 times
##     13594006 (57.16%) aligned exactly 1 time
##     6290659 (26.45%) aligned &gt;1 times
## 83.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6290659/19884665
## 2 4161801/6290659
## 3 27332756/66721869
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 3.
## Output will be in luad-0276-raw_1 and luad-0276-raw_2
## Single-end reads will be in luad-0276-raw_M
## 10407936 sequences in the BAM file
## 10407936 sequences exported
## 10407936 reads; of these:
##   10407936 (100.00%) were unpaired; of these:
##     3561630 (34.22%) aligned 0 times
##     5049382 (48.51%) aligned exactly 1 time
##     1796924 (17.26%) aligned &gt;1 times
## 65.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1796924/6846306
## 2 1140465/1796924
## 3 5182927/15553292
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0277-raw_1 and luad-0277-raw_2
## Single-end reads will be in luad-0277-raw_M
## 9943776 sequences in the BAM file
## 9943776 sequences exported
## 9943776 reads; of these:
##   9943776 (100.00%) were unpaired; of these:
##     2034978 (20.46%) aligned 0 times
##     5053227 (50.82%) aligned exactly 1 time
##     2855571 (28.72%) aligned &gt;1 times
## 79.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2855571/7908798
## 2 1831827/2855571
## 3 17900768/36654422
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0278-raw_1 and luad-0278-raw_2
## Single-end reads will be in luad-0278-raw_M
## 14922389 sequences in the BAM file
## 14922389 sequences exported
## 14922389 reads; of these:
##   14922389 (100.00%) were unpaired; of these:
##     6412989 (42.98%) aligned 0 times
##     6366122 (42.66%) aligned exactly 1 time
##     2143278 (14.36%) aligned &gt;1 times
## 57.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2143278/8509400
## 2 1338208/2143278
## 3 5587991/18869419
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0279-raw_1 and luad-0279-raw_2
## Single-end reads will be in luad-0279-raw_M
## 14242717 sequences in the BAM file
## 14242717 sequences exported
## 14242717 reads; of these:
##   14242717 (100.00%) were unpaired; of these:
##     4578182 (32.14%) aligned 0 times
##     7689208 (53.99%) aligned exactly 1 time
##     1975327 (13.87%) aligned &gt;1 times
## 67.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1975327/9664535
## 2 1188602/1975327
## 3 8401177/24426498
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0280-raw_1 and luad-0280-raw_2
## Single-end reads will be in luad-0280-raw_M
## 13517753 sequences in the BAM file
## 13517753 sequences exported
## 13517753 reads; of these:
##   13517753 (100.00%) were unpaired; of these:
##     4042536 (29.91%) aligned 0 times
##     6565634 (48.57%) aligned exactly 1 time
##     2909583 (21.52%) aligned &gt;1 times
## 70.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2909583/9475217
## 2 2084180/2909583
## 3 14778605/30648736
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0281-raw_1 and luad-0281-raw_2
## Single-end reads will be in luad-0281-raw_M
## 13694918 sequences in the BAM file
## 13694918 sequences exported
## 13694918 reads; of these:
##   13694918 (100.00%) were unpaired; of these:
##     5736238 (41.89%) aligned 0 times
##     6123883 (44.72%) aligned exactly 1 time
##     1834797 (13.40%) aligned &gt;1 times
## 58.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1834797/7958680
## 2 1097195/1834797
## 3 3609058/14276644
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0282-raw_1 and luad-0282-raw_2
## Single-end reads will be in luad-0282-raw_M
## 17617457 sequences in the BAM file
## 17617457 sequences exported
## 17617457 reads; of these:
##   17617457 (100.00%) were unpaired; of these:
##     3042736 (17.27%) aligned 0 times
##     10017589 (56.86%) aligned exactly 1 time
##     4557132 (25.87%) aligned &gt;1 times
## 82.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4557132/14574721
## 2 3312521/4557132
## 3 22531682/51522961
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 4.
## Output will be in luad-0283-raw_1 and luad-0283-raw_2
## Single-end reads will be in luad-0283-raw_M
## 8079043 sequences in the BAM file
## 8079043 sequences exported
## 8079043 reads; of these:
##   8079043 (100.00%) were unpaired; of these:
##     1803347 (22.32%) aligned 0 times
##     4868940 (60.27%) aligned exactly 1 time
##     1406756 (17.41%) aligned &gt;1 times
## 77.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1406756/6275696
## 2 996228/1406756
## 3 5701882/16191880
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0284-raw_1 and luad-0284-raw_2
## Single-end reads will be in luad-0284-raw_M
## 17957815 sequences in the BAM file
## 17957815 sequences exported
## 17957815 reads; of these:
##   17957815 (100.00%) were unpaired; of these:
##     3038199 (16.92%) aligned 0 times
##     10729658 (59.75%) aligned exactly 1 time
##     4189958 (23.33%) aligned &gt;1 times
## 83.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4189958/14919616
## 2 3216581/4189958
## 3 12508886/34468230
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in luad-0285-raw_1 and luad-0285-raw_2
## Single-end reads will be in luad-0285-raw_M
## 12175046 sequences in the BAM file
## 12175046 sequences exported
## 12175046 reads; of these:
##   12175046 (100.00%) were unpaired; of these:
##     2163061 (17.77%) aligned 0 times
##     7039533 (57.82%) aligned exactly 1 time
##     2972452 (24.41%) aligned &gt;1 times
## 82.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2972452/10011985
## 2 2108072/2972452
## 3 11085668/28057722
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0286-raw_1 and luad-0286-raw_2
## Single-end reads will be in luad-0286-raw_M
## 12979752 sequences in the BAM file
## 12979752 sequences exported
## 12979752 reads; of these:
##   12979752 (100.00%) were unpaired; of these:
##     2134434 (16.44%) aligned 0 times
##     6819007 (52.54%) aligned exactly 1 time
##     4026311 (31.02%) aligned &gt;1 times
## 83.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4026311/10845318
## 2 2676506/4026311
## 3 29934461/64056453
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 4.
## Output will be in luad-0287-raw_1 and luad-0287-raw_2
## Single-end reads will be in luad-0287-raw_M
## 12795274 sequences in the BAM file
## 12795274 sequences exported
## 12795274 reads; of these:
##   12795274 (100.00%) were unpaired; of these:
##     2264948 (17.70%) aligned 0 times
##     7457619 (58.28%) aligned exactly 1 time
##     3072707 (24.01%) aligned &gt;1 times
## 82.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3072707/10530326
## 2 2351243/3072707
## 3 13326648/29160916
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0288-raw_1 and luad-0288-raw_2
## Single-end reads will be in luad-0288-raw_M
## 7122370 sequences in the BAM file
## 7122370 sequences exported
## 7122370 reads; of these:
##   7122370 (100.00%) were unpaired; of these:
##     1693389 (23.78%) aligned 0 times
##     4146025 (58.21%) aligned exactly 1 time
##     1282956 (18.01%) aligned &gt;1 times
## 76.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1282956/5428981
## 2 894756/1282956
## 3 5688003/14350495
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0289-raw_1 and luad-0289-raw_2
## Single-end reads will be in luad-0289-raw_M
## 8674535 sequences in the BAM file
## 8674535 sequences exported
## 8674535 reads; of these:
##   8674535 (100.00%) were unpaired; of these:
##     2158927 (24.89%) aligned 0 times
##     4408304 (50.82%) aligned exactly 1 time
##     2107304 (24.29%) aligned &gt;1 times
## 75.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2107304/6515608
## 2 1465952/2107304
## 3 13559425/29501164
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0290-raw_1 and luad-0290-raw_2
## Single-end reads will be in luad-0290-raw_M
## 1374251 sequences in the BAM file
## 1374251 sequences exported
## 1374251 reads; of these:
##   1374251 (100.00%) were unpaired; of these:
##     413365 (30.08%) aligned 0 times
##     608824 (44.30%) aligned exactly 1 time
##     352062 (25.62%) aligned &gt;1 times
## 69.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 352062/960886
## 2 246431/352062
## 3 2390579/4826423
## This looks like paired data from lane 7.
## Output will be in luad-0291-raw_1 and luad-0291-raw_2
## Single-end reads will be in luad-0291-raw_M
## 14407082 sequences in the BAM file
## 14407082 sequences exported
## 14407082 reads; of these:
##   14407082 (100.00%) were unpaired; of these:
##     2171723 (15.07%) aligned 0 times
##     9655154 (67.02%) aligned exactly 1 time
##     2580205 (17.91%) aligned &gt;1 times
## 84.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2580205/12235359
## 2 1829467/2580205
## 3 10472220/30109810
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0292-raw_1 and luad-0292-raw_2
## Single-end reads will be in luad-0292-raw_M
## 7218929 sequences in the BAM file
## 7218929 sequences exported
## 7218929 reads; of these:
##   7218929 (100.00%) were unpaired; of these:
##     1317461 (18.25%) aligned 0 times
##     3882525 (53.78%) aligned exactly 1 time
##     2018943 (27.97%) aligned &gt;1 times
## 81.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2018943/5901468
## 2 1405509/2018943
## 3 7078835/17825891
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0293-raw_1 and luad-0293-raw_2
## Single-end reads will be in luad-0293-raw_M
## 16256065 sequences in the BAM file
## 16256065 sequences exported
## 16256065 reads; of these:
##   16256065 (100.00%) were unpaired; of these:
##     2705964 (16.65%) aligned 0 times
##     9897047 (60.88%) aligned exactly 1 time
##     3653054 (22.47%) aligned &gt;1 times
## 83.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3653054/13550101
## 2 2682952/3653054
## 3 11419954/32489530
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in luad-0294-raw_1 and luad-0294-raw_2
## Single-end reads will be in luad-0294-raw_M
## 16990571 sequences in the BAM file
## 16990571 sequences exported
## 16990571 reads; of these:
##   16990571 (100.00%) were unpaired; of these:
##     2060697 (12.13%) aligned 0 times
##     9509308 (55.97%) aligned exactly 1 time
##     5420566 (31.90%) aligned &gt;1 times
## 87.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5420566/14929874
## 2 4116283/5420566
## 3 29168921/64078598
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 6.
## Output will be in luad-0295-raw_1 and luad-0295-raw_2
## Single-end reads will be in luad-0295-raw_M
## 13219675 sequences in the BAM file
## 13219675 sequences exported
## 13219675 reads; of these:
##   13219675 (100.00%) were unpaired; of these:
##     5624111 (42.54%) aligned 0 times
##     5651591 (42.75%) aligned exactly 1 time
##     1943973 (14.71%) aligned &gt;1 times
## 57.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1943973/7595564
## 2 1241717/1943973
## 3 10637336/26812671
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0296-raw_1 and luad-0296-raw_2
## Single-end reads will be in luad-0296-raw_M
## 9657360 sequences in the BAM file
## 9657360 sequences exported
## 9657360 reads; of these:
##   9657360 (100.00%) were unpaired; of these:
##     4261577 (44.13%) aligned 0 times
##     3800435 (39.35%) aligned exactly 1 time
##     1595348 (16.52%) aligned &gt;1 times
## 55.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1595348/5395783
## 2 759105/1595348
## 3 5883323/16878660
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0297-raw_1 and luad-0297-raw_2
## Single-end reads will be in luad-0297-raw_M
## 16064406 sequences in the BAM file
## 16064406 sequences exported
## 16064406 reads; of these:
##   16064406 (100.00%) were unpaired; of these:
##     6216288 (38.70%) aligned 0 times
##     7689737 (47.87%) aligned exactly 1 time
##     2158381 (13.44%) aligned &gt;1 times
## 61.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2158381/9848118
## 2 1367764/2158381
## 3 6421575/20782385
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0298-raw_1 and luad-0298-raw_2
## Single-end reads will be in luad-0298-raw_M
## 13886535 sequences in the BAM file
## 13886535 sequences exported
## 13886535 reads; of these:
##   13886535 (100.00%) were unpaired; of these:
##     6105349 (43.97%) aligned 0 times
##     6462424 (46.54%) aligned exactly 1 time
##     1318762 (9.50%) aligned &gt;1 times
## 56.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1318762/7781186
## 2 687201/1318762
## 3 4321735/15326765
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0299-raw_1 and luad-0299-raw_2
## Single-end reads will be in luad-0299-raw_M
## 3092237 sequences in the BAM file
## 3092237 sequences exported
## 3092237 reads; of these:
##   3092237 (100.00%) were unpaired; of these:
##     562845 (18.20%) aligned 0 times
##     1780901 (57.59%) aligned exactly 1 time
##     748491 (24.21%) aligned &gt;1 times
## 81.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 748491/2529392
## 2 532376/748491
## 3 3792060/8127364
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0300-raw_1 and luad-0300-raw_2
## Single-end reads will be in luad-0300-raw_M
## 7970850 sequences in the BAM file
## 7970850 sequences exported
## 7970850 reads; of these:
##   7970850 (100.00%) were unpaired; of these:
##     3177886 (39.87%) aligned 0 times
##     3147619 (39.49%) aligned exactly 1 time
##     1645345 (20.64%) aligned &gt;1 times
## 60.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1645345/4792964
## 2 1129008/1645345
## 3 5117904/13575651
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0301-raw_1 and luad-0301-raw_2
## Single-end reads will be in luad-0301-raw_M
## 5887850 sequences in the BAM file
## 5887850 sequences exported
## 5887850 reads; of these:
##   5887850 (100.00%) were unpaired; of these:
##     1219756 (20.72%) aligned 0 times
##     3247099 (55.15%) aligned exactly 1 time
##     1420995 (24.13%) aligned &gt;1 times
## 79.28% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1420995/4668094
## 2 1029491/1420995
## 3 9149205/18706445
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0302-raw_1 and luad-0302-raw_2
## Single-end reads will be in luad-0302-raw_M
## 14973740 sequences in the BAM file
## 14973740 sequences exported
## 14973740 reads; of these:
##   14973740 (100.00%) were unpaired; of these:
##     2788159 (18.62%) aligned 0 times
##     7424697 (49.58%) aligned exactly 1 time
##     4760884 (31.79%) aligned &gt;1 times
## 81.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4760884/12185581
## 2 3867252/4760884
## 3 23715207/49885898
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 2.
## Output will be in luad-0303-raw_1 and luad-0303-raw_2
## Single-end reads will be in luad-0303-raw_M
## 6793112 sequences in the BAM file
## 6793112 sequences exported
## 6793112 reads; of these:
##   6793112 (100.00%) were unpaired; of these:
##     1544473 (22.74%) aligned 0 times
##     3935505 (57.93%) aligned exactly 1 time
##     1313134 (19.33%) aligned &gt;1 times
## 77.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1313134/5248639
## 2 896940/1313134
## 3 6851268/16594418
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0304-raw_1 and luad-0304-raw_2
## Single-end reads will be in luad-0304-raw_M
## 9589415 sequences in the BAM file
## 9589415 sequences exported
## 9589415 reads; of these:
##   9589415 (100.00%) were unpaired; of these:
##     2761584 (28.80%) aligned 0 times
##     4033027 (42.06%) aligned exactly 1 time
##     2794804 (29.14%) aligned &gt;1 times
## 71.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2794804/6827831
## 2 2091938/2794804
## 3 10050732/21577317
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0305-raw_1 and luad-0305-raw_2
## Single-end reads will be in luad-0305-raw_M
## 4160473 sequences in the BAM file
## 4160473 sequences exported
## 4160473 reads; of these:
##   4160473 (100.00%) were unpaired; of these:
##     837104 (20.12%) aligned 0 times
##     2317485 (55.70%) aligned exactly 1 time
##     1005884 (24.18%) aligned &gt;1 times
## 79.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1005884/3323369
## 2 701301/1005884
## 3 7237718/14455209
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0306-raw_1 and luad-0306-raw_2
## Single-end reads will be in luad-0306-raw_M
## 15273151 sequences in the BAM file
## 15273151 sequences exported
## 15273151 reads; of these:
##   15273151 (100.00%) were unpaired; of these:
##     2808717 (18.39%) aligned 0 times
##     8368803 (54.79%) aligned exactly 1 time
##     4095631 (26.82%) aligned &gt;1 times
## 81.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4095631/12464434
## 2 3260412/4095631
## 3 14829994/33959098
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0307-raw_1 and luad-0307-raw_2
## Single-end reads will be in luad-0307-raw_M
## 5207251 sequences in the BAM file
## 5207251 sequences exported
## 5207251 reads; of these:
##   5207251 (100.00%) were unpaired; of these:
##     1239682 (23.81%) aligned 0 times
##     2264218 (43.48%) aligned exactly 1 time
##     1703351 (32.71%) aligned &gt;1 times
## 76.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1703351/3967569
## 2 1086040/1703351
## 3 10335885/21907471
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0308-raw_1 and luad-0308-raw_2
## Single-end reads will be in luad-0308-raw_M
## 8078935 sequences in the BAM file
## 8078935 sequences exported
## 8078935 reads; of these:
##   8078935 (100.00%) were unpaired; of these:
##     2128115 (26.34%) aligned 0 times
##     3755788 (46.49%) aligned exactly 1 time
##     2195032 (27.17%) aligned &gt;1 times
## 73.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2195032/5950820
## 2 1262882/2195032
## 3 14006311/30503247
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0309-raw_1 and luad-0309-raw_2
## Single-end reads will be in luad-0309-raw_M
## 8113821 sequences in the BAM file
## 8113821 sequences exported
## 8113821 reads; of these:
##   8113821 (100.00%) were unpaired; of these:
##     1523007 (18.77%) aligned 0 times
##     4424806 (54.53%) aligned exactly 1 time
##     2166008 (26.70%) aligned &gt;1 times
## 81.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2166008/6590814
## 2 1601703/2166008
## 3 7878653/19379617
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0310-raw_1 and luad-0310-raw_2
## Single-end reads will be in luad-0310-raw_M
## 11966592 sequences in the BAM file
## 11966592 sequences exported
## 11966592 reads; of these:
##   11966592 (100.00%) were unpaired; of these:
##     4629123 (38.68%) aligned 0 times
##     5543465 (46.32%) aligned exactly 1 time
##     1794004 (14.99%) aligned &gt;1 times
## 61.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1794004/7337469
## 2 1057764/1794004
## 3 3653181/14114003
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0311-raw_1 and luad-0311-raw_2
## Single-end reads will be in luad-0311-raw_M
## 11152365 sequences in the BAM file
## 11152365 sequences exported
## 11152365 reads; of these:
##   11152365 (100.00%) were unpaired; of these:
##     2585122 (23.18%) aligned 0 times
##     6182324 (55.44%) aligned exactly 1 time
##     2384919 (21.38%) aligned &gt;1 times
## 76.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2384919/8567243
## 2 1804735/2384919
## 3 8852985/22508781
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0312-raw_1 and luad-0312-raw_2
## Single-end reads will be in luad-0312-raw_M
## 11412580 sequences in the BAM file
## 11412580 sequences exported
## 11412580 reads; of these:
##   11412580 (100.00%) were unpaired; of these:
##     4556374 (39.92%) aligned 0 times
##     4681900 (41.02%) aligned exactly 1 time
##     2174306 (19.05%) aligned &gt;1 times
## 60.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2174306/6856206
## 2 1377934/2174306
## 3 14967079/31546111
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0313-raw_1 and luad-0313-raw_2
## Single-end reads will be in luad-0313-raw_M
## 8268355 sequences in the BAM file
## 8268355 sequences exported
## 8268355 reads; of these:
##   8268355 (100.00%) were unpaired; of these:
##     2308051 (27.91%) aligned 0 times
##     4080042 (49.35%) aligned exactly 1 time
##     1880262 (22.74%) aligned &gt;1 times
## 72.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1880262/5960304
## 2 1356095/1880262
## 3 9274752/21026228
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0314-raw_1 and luad-0314-raw_2
## Single-end reads will be in luad-0314-raw_M
## 9722261 sequences in the BAM file
## 9722261 sequences exported
## 9722261 reads; of these:
##   9722261 (100.00%) were unpaired; of these:
##     2317130 (23.83%) aligned 0 times
##     4885357 (50.25%) aligned exactly 1 time
##     2519774 (25.92%) aligned &gt;1 times
## 76.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2519774/7405131
## 2 1720898/2519774
## 3 14071153/32248171
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0315-raw_1 and luad-0315-raw_2
## Single-end reads will be in luad-0315-raw_M
## 15893471 sequences in the BAM file
## 15893471 sequences exported
## 15893471 reads; of these:
##   15893471 (100.00%) were unpaired; of these:
##     2120438 (13.34%) aligned 0 times
##     6694175 (42.12%) aligned exactly 1 time
##     7078858 (44.54%) aligned &gt;1 times
## 86.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7078858/13773033
## 2 5798200/7078858
## 3 48163094/95683909
## [bam_sort_core] merging from 14 files...
## This looks like paired data from lane 1.
## Output will be in luad-0316-raw_1 and luad-0316-raw_2
## Single-end reads will be in luad-0316-raw_M
## 12848346 sequences in the BAM file
## 12848346 sequences exported
## 12848346 reads; of these:
##   12848346 (100.00%) were unpaired; of these:
##     4242722 (33.02%) aligned 0 times
##     6412296 (49.91%) aligned exactly 1 time
##     2193328 (17.07%) aligned &gt;1 times
## 66.98% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2193328/8605624
## 2 1370030/2193328
## 3 5934068/19318888
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0317-raw_1 and luad-0317-raw_2
## Single-end reads will be in luad-0317-raw_M
## 13941345 sequences in the BAM file
## 13941345 sequences exported
## 13941345 reads; of these:
##   13941345 (100.00%) were unpaired; of these:
##     3291025 (23.61%) aligned 0 times
##     6793624 (48.73%) aligned exactly 1 time
##     3856696 (27.66%) aligned &gt;1 times
## 76.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3856696/10650320
## 2 2677789/3856696
## 3 9637089/26561740
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0318-raw_1 and luad-0318-raw_2
## Single-end reads will be in luad-0318-raw_M
## 9070912 sequences in the BAM file
## 9070912 sequences exported
## 9070912 reads; of these:
##   9070912 (100.00%) were unpaired; of these:
##     1773452 (19.55%) aligned 0 times
##     5044144 (55.61%) aligned exactly 1 time
##     2253316 (24.84%) aligned &gt;1 times
## 80.45% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2253316/7297460
## 2 1540076/2253316
## 3 15023114/31041153
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0319-raw_1 and luad-0319-raw_2
## Single-end reads will be in luad-0319-raw_M
## 8995892 sequences in the BAM file
## 8995892 sequences exported
## 8995892 reads; of these:
##   8995892 (100.00%) were unpaired; of these:
##     2286201 (25.41%) aligned 0 times
##     3674413 (40.85%) aligned exactly 1 time
##     3035278 (33.74%) aligned &gt;1 times
## 74.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3035278/6709691
## 2 2165140/3035278
## 3 24169384/48429934
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in luad-0320-raw_1 and luad-0320-raw_2
## Single-end reads will be in luad-0320-raw_M
## 10566005 sequences in the BAM file
## 10566005 sequences exported
## 10566005 reads; of these:
##   10566005 (100.00%) were unpaired; of these:
##     4441017 (42.03%) aligned 0 times
##     4503870 (42.63%) aligned exactly 1 time
##     1621118 (15.34%) aligned &gt;1 times
## 57.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1621118/6124988
## 2 1032675/1621118
## 3 9332751/20502165
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0321-raw_1 and luad-0321-raw_2
## Single-end reads will be in luad-0321-raw_M
## 13164911 sequences in the BAM file
## 13164911 sequences exported
## 13164911 reads; of these:
##   13164911 (100.00%) were unpaired; of these:
##     5345681 (40.61%) aligned 0 times
##     6516119 (49.50%) aligned exactly 1 time
##     1303111 (9.90%) aligned &gt;1 times
## 59.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1303111/7819230
## 2 684267/1303111
## 3 3914392/15431903
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0322-raw_1 and luad-0322-raw_2
## Single-end reads will be in luad-0322-raw_M
## 18905922 sequences in the BAM file
## 18905922 sequences exported
## 18905922 reads; of these:
##   18905922 (100.00%) were unpaired; of these:
##     1627490 (8.61%) aligned 0 times
##     11444480 (60.53%) aligned exactly 1 time
##     5833952 (30.86%) aligned &gt;1 times
## 91.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5833952/17278432
## 2 4631744/5833952
## 3 21680229/48509856
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in luad-0323-raw_1 and luad-0323-raw_2
## Single-end reads will be in luad-0323-raw_M
## 7867314 sequences in the BAM file
## 7867314 sequences exported
## 7867314 reads; of these:
##   7867314 (100.00%) were unpaired; of these:
##     1832915 (23.30%) aligned 0 times
##     4148555 (52.73%) aligned exactly 1 time
##     1885844 (23.97%) aligned &gt;1 times
## 76.70% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1885844/6034399
## 2 1427061/1885844
## 3 5619525/14689410
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0324-raw_1 and luad-0324-raw_2
## Single-end reads will be in luad-0324-raw_M
## 13599096 sequences in the BAM file
## 13599096 sequences exported
## 13599096 reads; of these:
##   13599096 (100.00%) were unpaired; of these:
##     2309487 (16.98%) aligned 0 times
##     7652946 (56.28%) aligned exactly 1 time
##     3636663 (26.74%) aligned &gt;1 times
## 83.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3636663/11289609
## 2 2548294/3636663
## 3 22487197/57275767
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 2.
## Output will be in luad-0325-raw_1 and luad-0325-raw_2
## Single-end reads will be in luad-0325-raw_M
## 5214942 sequences in the BAM file
## 5214942 sequences exported
## 5214942 reads; of these:
##   5214942 (100.00%) were unpaired; of these:
##     1172333 (22.48%) aligned 0 times
##     2699074 (51.76%) aligned exactly 1 time
##     1343535 (25.76%) aligned &gt;1 times
## 77.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1343535/4042609
## 2 1019651/1343535
## 3 5046091/11638258
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0326-raw_1 and luad-0326-raw_2
## Single-end reads will be in luad-0326-raw_M
## 12061523 sequences in the BAM file
## 12061523 sequences exported
## 12061523 reads; of these:
##   12061523 (100.00%) were unpaired; of these:
##     2147160 (17.80%) aligned 0 times
##     7050142 (58.45%) aligned exactly 1 time
##     2864221 (23.75%) aligned &gt;1 times
## 82.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2864221/9914363
## 2 1957483/2864221
## 3 14432543/33866501
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0327-raw_1 and luad-0327-raw_2
## Single-end reads will be in luad-0327-raw_M
## 13025867 sequences in the BAM file
## 13025867 sequences exported
## 13025867 reads; of these:
##   13025867 (100.00%) were unpaired; of these:
##     2121120 (16.28%) aligned 0 times
##     6450868 (49.52%) aligned exactly 1 time
##     4453879 (34.19%) aligned &gt;1 times
## 83.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4453879/10904747
## 2 3188101/4453879
## 3 34282260/67238937
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 6.
## Output will be in luad-0328-raw_1 and luad-0328-raw_2
## Single-end reads will be in luad-0328-raw_M
## 9551592 sequences in the BAM file
## 9551592 sequences exported
## 9551592 reads; of these:
##   9551592 (100.00%) were unpaired; of these:
##     4118998 (43.12%) aligned 0 times
##     4020848 (42.10%) aligned exactly 1 time
##     1411746 (14.78%) aligned &gt;1 times
## 56.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1411746/5432594
## 2 841130/1411746
## 3 4985880/14251878
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0329-raw_1 and luad-0329-raw_2
## Single-end reads will be in luad-0329-raw_M
## 14119212 sequences in the BAM file
## 14119212 sequences exported
## 14119212 reads; of these:
##   14119212 (100.00%) were unpaired; of these:
##     2437168 (17.26%) aligned 0 times
##     8184398 (57.97%) aligned exactly 1 time
##     3497646 (24.77%) aligned &gt;1 times
## 82.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3497646/11682044
## 2 2733195/3497646
## 3 14100421/32255533
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0330-raw_1 and luad-0330-raw_2
## Single-end reads will be in luad-0330-raw_M
## 10356773 sequences in the BAM file
## 10356773 sequences exported
## 10356773 reads; of these:
##   10356773 (100.00%) were unpaired; of these:
##     2532807 (24.46%) aligned 0 times
##     5284618 (51.03%) aligned exactly 1 time
##     2539348 (24.52%) aligned &gt;1 times
## 75.54% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2539348/7823966
## 2 1656106/2539348
## 3 8798262/22678148
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0331-raw_1 and luad-0331-raw_2
## Single-end reads will be in luad-0331-raw_M
## 5702174 sequences in the BAM file
## 5702174 sequences exported
## 5702174 reads; of these:
##   5702174 (100.00%) were unpaired; of these:
##     1384156 (24.27%) aligned 0 times
##     2412101 (42.30%) aligned exactly 1 time
##     1905917 (33.42%) aligned &gt;1 times
## 75.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1905917/4318018
## 2 1347089/1905917
## 3 10320999/21716695
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0332-raw_1 and luad-0332-raw_2
## Single-end reads will be in luad-0332-raw_M
## 11033875 sequences in the BAM file
## 11033875 sequences exported
## 11033875 reads; of these:
##   11033875 (100.00%) were unpaired; of these:
##     1783004 (16.16%) aligned 0 times
##     6461414 (58.56%) aligned exactly 1 time
##     2789457 (25.28%) aligned &gt;1 times
## 83.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2789457/9250871
## 2 1930992/2789457
## 3 13451670/32528098
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0333-raw_1 and luad-0333-raw_2
## Single-end reads will be in luad-0333-raw_M
## 12113297 sequences in the BAM file
## 12113297 sequences exported
## 12113297 reads; of these:
##   12113297 (100.00%) were unpaired; of these:
##     3697776 (30.53%) aligned 0 times
##     5416739 (44.72%) aligned exactly 1 time
##     2998782 (24.76%) aligned &gt;1 times
## 69.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2998782/8415521
## 2 1794969/2998782
## 3 8057994/21625786
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0334-raw_1 and luad-0334-raw_2
## Single-end reads will be in luad-0334-raw_M
## 11359380 sequences in the BAM file
## 11359380 sequences exported
## 11359380 reads; of these:
##   11359380 (100.00%) were unpaired; of these:
##     3462390 (30.48%) aligned 0 times
##     5888783 (51.84%) aligned exactly 1 time
##     2008207 (17.68%) aligned &gt;1 times
## 69.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2008207/7896990
## 2 1231491/2008207
## 3 5002842/16363495
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0335-raw_1 and luad-0335-raw_2
## Single-end reads will be in luad-0335-raw_M
## 10165581 sequences in the BAM file
## 10165581 sequences exported
## 10165581 reads; of these:
##   10165581 (100.00%) were unpaired; of these:
##     3104491 (30.54%) aligned 0 times
##     5864549 (57.69%) aligned exactly 1 time
##     1196541 (11.77%) aligned &gt;1 times
## 69.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1196541/7061090
## 2 741478/1196541
## 3 5581852/16989815
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0336-raw_1 and luad-0336-raw_2
## Single-end reads will be in luad-0336-raw_M
## 10393110 sequences in the BAM file
## 10393110 sequences exported
## 10393110 reads; of these:
##   10393110 (100.00%) were unpaired; of these:
##     1898262 (18.26%) aligned 0 times
##     6235251 (59.99%) aligned exactly 1 time
##     2259597 (21.74%) aligned &gt;1 times
## 81.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2259597/8494848
## 2 1628379/2259597
## 3 10866643/25122193
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0337-raw_1 and luad-0337-raw_2
## Single-end reads will be in luad-0337-raw_M
## 11803328 sequences in the BAM file
## 11803328 sequences exported
## 11803328 reads; of these:
##   11803328 (100.00%) were unpaired; of these:
##     4312093 (36.53%) aligned 0 times
##     5993590 (50.78%) aligned exactly 1 time
##     1497645 (12.69%) aligned &gt;1 times
## 63.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1497645/7491235
## 2 933387/1497645
## 3 4355787/14817172
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0338-raw_1 and luad-0338-raw_2
## Single-end reads will be in luad-0338-raw_M
## 8391793 sequences in the BAM file
## 8391793 sequences exported
## 8391793 reads; of these:
##   8391793 (100.00%) were unpaired; of these:
##     2034301 (24.24%) aligned 0 times
##     3166926 (37.74%) aligned exactly 1 time
##     3190566 (38.02%) aligned &gt;1 times
## 75.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3190566/6357492
## 2 2294698/3190566
## 3 24776932/52426240
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in luad-0339-raw_1 and luad-0339-raw_2
## Single-end reads will be in luad-0339-raw_M
## 14348458 sequences in the BAM file
## 14348458 sequences exported
## 14348458 reads; of these:
##   14348458 (100.00%) were unpaired; of these:
##     3215055 (22.41%) aligned 0 times
##     7487765 (52.19%) aligned exactly 1 time
##     3645638 (25.41%) aligned &gt;1 times
## 77.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3645638/11133403
## 2 2614635/3645638
## 3 12179381/29363667
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0340-raw_1 and luad-0340-raw_2
## Single-end reads will be in luad-0340-raw_M
## 9925901 sequences in the BAM file
## 9925901 sequences exported
## 9925901 reads; of these:
##   9925901 (100.00%) were unpaired; of these:
##     2095940 (21.12%) aligned 0 times
##     5322055 (53.62%) aligned exactly 1 time
##     2507906 (25.27%) aligned &gt;1 times
## 78.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2507906/7829961
## 2 1750364/2507906
## 3 12158186/26501816
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0341-raw_1 and luad-0341-raw_2
## Single-end reads will be in luad-0341-raw_M
## 6576697 sequences in the BAM file
## 6576697 sequences exported
## 6576697 reads; of these:
##   6576697 (100.00%) were unpaired; of these:
##     1320380 (20.08%) aligned 0 times
##     3818551 (58.06%) aligned exactly 1 time
##     1437766 (21.86%) aligned &gt;1 times
## 79.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1437766/5256317
## 2 949644/1437766
## 3 6410133/16169604
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0342-raw_1 and luad-0342-raw_2
## Single-end reads will be in luad-0342-raw_M
## 9320574 sequences in the BAM file
## 9320574 sequences exported
## 9320574 reads; of these:
##   9320574 (100.00%) were unpaired; of these:
##     1904613 (20.43%) aligned 0 times
##     4333001 (46.49%) aligned exactly 1 time
##     3082960 (33.08%) aligned &gt;1 times
## 79.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3082960/7415961
## 2 2052625/3082960
## 3 17011255/35746875
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0343-raw_1 and luad-0343-raw_2
## Single-end reads will be in luad-0343-raw_M
## 14361770 sequences in the BAM file
## 14361770 sequences exported
## 14361770 reads; of these:
##   14361770 (100.00%) were unpaired; of these:
##     4117497 (28.67%) aligned 0 times
##     7224426 (50.30%) aligned exactly 1 time
##     3019847 (21.03%) aligned &gt;1 times
## 71.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3019847/10244273
## 2 2220838/3019847
## 3 13734950/29824981
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0344-raw_1 and luad-0344-raw_2
## Single-end reads will be in luad-0344-raw_M
## 9368336 sequences in the BAM file
## 9368336 sequences exported
## 9368336 reads; of these:
##   9368336 (100.00%) were unpaired; of these:
##     3197342 (34.13%) aligned 0 times
##     4835477 (51.62%) aligned exactly 1 time
##     1335517 (14.26%) aligned &gt;1 times
## 65.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1335517/6170994
## 2 895802/1335517
## 3 5061510/14128291
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0345-raw_1 and luad-0345-raw_2
## Single-end reads will be in luad-0345-raw_M
## 12243766 sequences in the BAM file
## 12243766 sequences exported
## 12243766 reads; of these:
##   12243766 (100.00%) were unpaired; of these:
##     4247752 (34.69%) aligned 0 times
##     5608932 (45.81%) aligned exactly 1 time
##     2387082 (19.50%) aligned &gt;1 times
## 65.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2387082/7996014
## 2 1843594/2387082
## 3 12740217/25695698
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0346-raw_1 and luad-0346-raw_2
## Single-end reads will be in luad-0346-raw_M
## 16216582 sequences in the BAM file
## 16216582 sequences exported
## 16216582 reads; of these:
##   16216582 (100.00%) were unpaired; of these:
##     2204450 (13.59%) aligned 0 times
##     7036876 (43.39%) aligned exactly 1 time
##     6975256 (43.01%) aligned &gt;1 times
## 86.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6975256/14012132
## 2 4699857/6975256
## 3 18775804/42996654
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in luad-0347-raw_1 and luad-0347-raw_2
## Single-end reads will be in luad-0347-raw_M
## 7801861 sequences in the BAM file
## 7801861 sequences exported
## 7801861 reads; of these:
##   7801861 (100.00%) were unpaired; of these:
##     3013145 (38.62%) aligned 0 times
##     3600151 (46.14%) aligned exactly 1 time
##     1188565 (15.23%) aligned &gt;1 times
## 61.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1188565/4788716
## 2 715930/1188565
## 3 5036134/13770536
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0348-raw_1 and luad-0348-raw_2
## Single-end reads will be in luad-0348-raw_M
## 4564039 sequences in the BAM file
## 4564039 sequences exported
## 4564039 reads; of these:
##   4564039 (100.00%) were unpaired; of these:
##     972927 (21.32%) aligned 0 times
##     2014200 (44.13%) aligned exactly 1 time
##     1576912 (34.55%) aligned &gt;1 times
## 78.68% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1576912/3591112
## 2 919758/1576912
## 3 7468571/15833952
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0349-raw_1 and luad-0349-raw_2
## Single-end reads will be in luad-0349-raw_M
## 9622442 sequences in the BAM file
## 9622442 sequences exported
## 9622442 reads; of these:
##   9622442 (100.00%) were unpaired; of these:
##     2233941 (23.22%) aligned 0 times
##     5609843 (58.30%) aligned exactly 1 time
##     1778658 (18.48%) aligned &gt;1 times
## 76.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1778658/7388501
## 2 1244229/1778658
## 3 6057805/17124677
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0350-raw_1 and luad-0350-raw_2
## Single-end reads will be in luad-0350-raw_M
## 7492919 sequences in the BAM file
## 7492919 sequences exported
## 7492919 reads; of these:
##   7492919 (100.00%) were unpaired; of these:
##     2601162 (34.71%) aligned 0 times
##     3199019 (42.69%) aligned exactly 1 time
##     1692738 (22.59%) aligned &gt;1 times
## 65.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1692738/4891757
## 2 1263946/1692738
## 3 9861607/18629995
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0351-raw_1 and luad-0351-raw_2
## Single-end reads will be in luad-0351-raw_M
## 9198838 sequences in the BAM file
## 9198838 sequences exported
## 9198838 reads; of these:
##   9198838 (100.00%) were unpaired; of these:
##     2944280 (32.01%) aligned 0 times
##     3486540 (37.90%) aligned exactly 1 time
##     2768018 (30.09%) aligned &gt;1 times
## 67.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2768018/6254558
## 2 1950634/2768018
## 3 20078516/36577950
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0352-raw_1 and luad-0352-raw_2
## Single-end reads will be in luad-0352-raw_M
## 14491161 sequences in the BAM file
## 14491161 sequences exported
## 14491161 reads; of these:
##   14491161 (100.00%) were unpaired; of these:
##     4507964 (31.11%) aligned 0 times
##     7149363 (49.34%) aligned exactly 1 time
##     2833834 (19.56%) aligned &gt;1 times
## 68.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2833834/9983197
## 2 1862312/2833834
## 3 6776971/21571470
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0353-raw_1 and luad-0353-raw_2
## Single-end reads will be in luad-0353-raw_M
## 5760685 sequences in the BAM file
## 5760685 sequences exported
## 5760685 reads; of these:
##   5760685 (100.00%) were unpaired; of these:
##     1663196 (28.87%) aligned 0 times
##     2376552 (41.25%) aligned exactly 1 time
##     1720937 (29.87%) aligned &gt;1 times
## 71.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1720937/4097489
## 2 1208668/1720937
## 3 16560854/29138976
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0354-raw_1 and luad-0354-raw_2
## Single-end reads will be in luad-0354-raw_M
## 12683861 sequences in the BAM file
## 12683861 sequences exported
## 12683861 reads; of these:
##   12683861 (100.00%) were unpaired; of these:
##     2037988 (16.07%) aligned 0 times
##     6940334 (54.72%) aligned exactly 1 time
##     3705539 (29.21%) aligned &gt;1 times
## 83.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3705539/10645873
## 2 2797358/3705539
## 3 19323195/38348298
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0355-raw_1 and luad-0355-raw_2
## Single-end reads will be in luad-0355-raw_M
## 9081038 sequences in the BAM file
## 9081038 sequences exported
## 9081038 reads; of these:
##   9081038 (100.00%) were unpaired; of these:
##     2804307 (30.88%) aligned 0 times
##     4789298 (52.74%) aligned exactly 1 time
##     1487433 (16.38%) aligned &gt;1 times
## 69.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1487433/6276731
## 2 897482/1487433
## 3 5290113/15931757
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0356-raw_1 and luad-0356-raw_2
## Single-end reads will be in luad-0356-raw_M
## 11845688 sequences in the BAM file
## 11845688 sequences exported
## 11845688 reads; of these:
##   11845688 (100.00%) were unpaired; of these:
##     2935518 (24.78%) aligned 0 times
##     3705848 (31.28%) aligned exactly 1 time
##     5204322 (43.93%) aligned &gt;1 times
## 75.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5204322/8910170
## 2 3886695/5204322
## 3 27177813/57793835
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 7.
## Output will be in luad-0357-raw_1 and luad-0357-raw_2
## Single-end reads will be in luad-0357-raw_M
## 10840761 sequences in the BAM file
## 10840761 sequences exported
## 10840761 reads; of these:
##   10840761 (100.00%) were unpaired; of these:
##     1791985 (16.53%) aligned 0 times
##     6827367 (62.98%) aligned exactly 1 time
##     2221409 (20.49%) aligned &gt;1 times
## 83.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2221409/9048776
## 2 1645576/2221409
## 3 5928007/18152327
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0358-raw_1 and luad-0358-raw_2
## Single-end reads will be in luad-0358-raw_M
## 15098938 sequences in the BAM file
## 15098938 sequences exported
## 15098938 reads; of these:
##   15098938 (100.00%) were unpaired; of these:
##     2921211 (19.35%) aligned 0 times
##     8647886 (57.27%) aligned exactly 1 time
##     3529841 (23.38%) aligned &gt;1 times
## 80.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3529841/12177727
## 2 2581553/3529841
## 3 9553352/27372829
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0359-raw_1 and luad-0359-raw_2
## Single-end reads will be in luad-0359-raw_M
## 9726162 sequences in the BAM file
## 9726162 sequences exported
## 9726162 reads; of these:
##   9726162 (100.00%) were unpaired; of these:
##     2680182 (27.56%) aligned 0 times
##     4162469 (42.80%) aligned exactly 1 time
##     2883511 (29.65%) aligned &gt;1 times
## 72.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2883511/7045980
## 2 2117329/2883511
## 3 22127655/39807895
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0360-raw_1 and luad-0360-raw_2
## Single-end reads will be in luad-0360-raw_M
## 11775437 sequences in the BAM file
## 11775437 sequences exported
## 11775437 reads; of these:
##   11775437 (100.00%) were unpaired; of these:
##     2885574 (24.51%) aligned 0 times
##     5634213 (47.85%) aligned exactly 1 time
##     3255650 (27.65%) aligned &gt;1 times
## 75.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3255650/8889863
## 2 2762615/3255650
## 3 9637032/23705891
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0361-raw_1 and luad-0361-raw_2
## Single-end reads will be in luad-0361-raw_M
## 7489600 sequences in the BAM file
## 7489600 sequences exported
## 7489600 reads; of these:
##   7489600 (100.00%) were unpaired; of these:
##     3301866 (44.09%) aligned 0 times
##     3124419 (41.72%) aligned exactly 1 time
##     1063315 (14.20%) aligned &gt;1 times
## 55.91% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1063315/4187734
## 2 721638/1063315
## 3 5583894/12809059
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0362-raw_1 and luad-0362-raw_2
## Single-end reads will be in luad-0362-raw_M
## 11308240 sequences in the BAM file
## 11308240 sequences exported
## 11308240 reads; of these:
##   11308240 (100.00%) were unpaired; of these:
##     4400259 (38.91%) aligned 0 times
##     5361334 (47.41%) aligned exactly 1 time
##     1546647 (13.68%) aligned &gt;1 times
## 61.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1546647/6907981
## 2 1010365/1546647
## 3 4442740/14481078
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0363-raw_1 and luad-0363-raw_2
## Single-end reads will be in luad-0363-raw_M
## 12829340 sequences in the BAM file
## 12829340 sequences exported
## 12829340 reads; of these:
##   12829340 (100.00%) were unpaired; of these:
##     1363976 (10.63%) aligned 0 times
##     4367770 (34.05%) aligned exactly 1 time
##     7097594 (55.32%) aligned &gt;1 times
## 89.37% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7097594/11465364
## 2 4656691/7097594
## 3 51336960/99977745
## [bam_sort_core] merging from 14 files...
## This looks like paired data from lane 7.
## Output will be in luad-0364-raw_1 and luad-0364-raw_2
## Single-end reads will be in luad-0364-raw_M
## 15224930 sequences in the BAM file
## 15224930 sequences exported
## 15224930 reads; of these:
##   15224930 (100.00%) were unpaired; of these:
##     3030721 (19.91%) aligned 0 times
##     8111618 (53.28%) aligned exactly 1 time
##     4082591 (26.82%) aligned &gt;1 times
## 80.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4082591/12194209
## 2 2931753/4082591
## 3 19582979/44593531
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in luad-0365-raw_1 and luad-0365-raw_2
## Single-end reads will be in luad-0365-raw_M
## 7400501 sequences in the BAM file
## 7400501 sequences exported
## 7400501 reads; of these:
##   7400501 (100.00%) were unpaired; of these:
##     1495499 (20.21%) aligned 0 times
##     3846251 (51.97%) aligned exactly 1 time
##     2058751 (27.82%) aligned &gt;1 times
## 79.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2058751/5905002
## 2 1508838/2058751
## 3 13187951/28290516
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0366-raw_1 and luad-0366-raw_2
## Single-end reads will be in luad-0366-raw_M
## 21099994 sequences in the BAM file
## 21099994 sequences exported
## 21099994 reads; of these:
##   21099994 (100.00%) were unpaired; of these:
##     3945980 (18.70%) aligned 0 times
##     11008865 (52.17%) aligned exactly 1 time
##     6145149 (29.12%) aligned &gt;1 times
## 81.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6145149/17154014
## 2 4477384/6145149
## 3 23055387/53902958
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 6.
## Output will be in luad-0367-raw_1 and luad-0367-raw_2
## Single-end reads will be in luad-0367-raw_M
## 7514286 sequences in the BAM file
## 7514286 sequences exported
## 7514286 reads; of these:
##   7514286 (100.00%) were unpaired; of these:
##     3080044 (40.99%) aligned 0 times
##     1994251 (26.54%) aligned exactly 1 time
##     2439991 (32.47%) aligned &gt;1 times
## 59.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2439991/4434242
## 2 1632622/2439991
## 3 23598608/45143317
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0368-raw_1 and luad-0368-raw_2
## Single-end reads will be in luad-0368-raw_M
## 8148185 sequences in the BAM file
## 8148185 sequences exported
## 8148185 reads; of these:
##   8148185 (100.00%) were unpaired; of these:
##     1641606 (20.15%) aligned 0 times
##     4821634 (59.17%) aligned exactly 1 time
##     1684945 (20.68%) aligned &gt;1 times
## 79.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1684945/6506579
## 2 1232879/1684945
## 3 10047040/22488337
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0369-raw_1 and luad-0369-raw_2
## Single-end reads will be in luad-0369-raw_M
## 7739353 sequences in the BAM file
## 7739353 sequences exported
## 7739353 reads; of these:
##   7739353 (100.00%) were unpaired; of these:
##     2160978 (27.92%) aligned 0 times
##     3514787 (45.41%) aligned exactly 1 time
##     2063588 (26.66%) aligned &gt;1 times
## 72.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2063588/5578375
## 2 1487655/2063588
## 3 14104276/25665447
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0370-raw_1 and luad-0370-raw_2
## Single-end reads will be in luad-0370-raw_M
## 5542186 sequences in the BAM file
## 5542186 sequences exported
## 5542186 reads; of these:
##   5542186 (100.00%) were unpaired; of these:
##     1104347 (19.93%) aligned 0 times
##     2468912 (44.55%) aligned exactly 1 time
##     1968927 (35.53%) aligned &gt;1 times
## 80.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1968927/4437839
## 2 1332400/1968927
## 3 11573094/25052758
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0371-raw_1 and luad-0371-raw_2
## Single-end reads will be in luad-0371-raw_M
## 14467007 sequences in the BAM file
## 14467007 sequences exported
## 14467007 reads; of these:
##   14467007 (100.00%) were unpaired; of these:
##     1876084 (12.97%) aligned 0 times
##     9245020 (63.90%) aligned exactly 1 time
##     3345903 (23.13%) aligned &gt;1 times
## 87.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3345903/12590923
## 2 2644050/3345903
## 3 10840942/28899469
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0372-raw_1 and luad-0372-raw_2
## Single-end reads will be in luad-0372-raw_M
## 13115544 sequences in the BAM file
## 13115544 sequences exported
## 13115544 reads; of these:
##   13115544 (100.00%) were unpaired; of these:
##     2304894 (17.57%) aligned 0 times
##     7393042 (56.37%) aligned exactly 1 time
##     3417608 (26.06%) aligned &gt;1 times
## 82.43% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3417608/10810650
## 2 2536390/3417608
## 3 14796449/32533832
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0373-raw_1 and luad-0373-raw_2
## Single-end reads will be in luad-0373-raw_M
## 19396038 sequences in the BAM file
## 19396038 sequences exported
## 19396038 reads; of these:
##   19396038 (100.00%) were unpaired; of these:
##     4670465 (24.08%) aligned 0 times
##     10599811 (54.65%) aligned exactly 1 time
##     4125762 (21.27%) aligned &gt;1 times
## 75.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4125762/14725573
## 2 3065304/4125762
## 3 16547065/40526030
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in luad-0374-raw_1 and luad-0374-raw_2
## Single-end reads will be in luad-0374-raw_M
## 12635262 sequences in the BAM file
## 12635262 sequences exported
## 12635262 reads; of these:
##   12635262 (100.00%) were unpaired; of these:
##     5013532 (39.68%) aligned 0 times
##     5912508 (46.79%) aligned exactly 1 time
##     1709222 (13.53%) aligned &gt;1 times
## 60.32% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1709222/7621730
## 2 1149410/1709222
## 3 4880140/15529309
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0375-raw_1 and luad-0375-raw_2
## Single-end reads will be in luad-0375-raw_M
## 2397364 sequences in the BAM file
## 2397364 sequences exported
## 2397364 reads; of these:
##   2397364 (100.00%) were unpaired; of these:
##     565838 (23.60%) aligned 0 times
##     1184319 (49.40%) aligned exactly 1 time
##     647207 (27.00%) aligned &gt;1 times
## 76.40% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 647207/1831526
## 2 425159/647207
## 3 4722789/9127888
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in luad-0376-raw_1 and luad-0376-raw_2
## Single-end reads will be in luad-0376-raw_M
## 7624858 sequences in the BAM file
## 7624858 sequences exported
## 7624858 reads; of these:
##   7624858 (100.00%) were unpaired; of these:
##     1602305 (21.01%) aligned 0 times
##     4108087 (53.88%) aligned exactly 1 time
##     1914466 (25.11%) aligned &gt;1 times
## 78.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1914466/6022553
## 2 1451812/1914466
## 3 9019494/19915784
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0377-raw_1 and luad-0377-raw_2
## Single-end reads will be in luad-0377-raw_M
## 20096408 sequences in the BAM file
## 20096408 sequences exported
## 20096408 reads; of these:
##   20096408 (100.00%) were unpaired; of these:
##     3448015 (17.16%) aligned 0 times
##     10639114 (52.94%) aligned exactly 1 time
##     6009279 (29.90%) aligned &gt;1 times
## 82.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6009279/16648393
## 2 3967112/6009279
## 3 29939498/70448469
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 1.
## Output will be in luad-0378-raw_1 and luad-0378-raw_2
## Single-end reads will be in luad-0378-raw_M
## 8380842 sequences in the BAM file
## 8380842 sequences exported
## 8380842 reads; of these:
##   8380842 (100.00%) were unpaired; of these:
##     1939412 (23.14%) aligned 0 times
##     4815315 (57.46%) aligned exactly 1 time
##     1626115 (19.40%) aligned &gt;1 times
## 76.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1626115/6441430
## 2 1050078/1626115
## 3 7863654/19895714
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0379-raw_1 and luad-0379-raw_2
## Single-end reads will be in luad-0379-raw_M
## 8360092 sequences in the BAM file
## 8360092 sequences exported
## 8360092 reads; of these:
##   8360092 (100.00%) were unpaired; of these:
##     1578188 (18.88%) aligned 0 times
##     4330140 (51.80%) aligned exactly 1 time
##     2451764 (29.33%) aligned &gt;1 times
## 81.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2451764/6781904
## 2 1759455/2451764
## 3 18873298/37917827
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0380-raw_1 and luad-0380-raw_2
## Single-end reads will be in luad-0380-raw_M
## 14273572 sequences in the BAM file
## 14273572 sequences exported
## 14273572 reads; of these:
##   14273572 (100.00%) were unpaired; of these:
##     5878293 (41.18%) aligned 0 times
##     6188952 (43.36%) aligned exactly 1 time
##     2206327 (15.46%) aligned &gt;1 times
## 58.82% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2206327/8395279
## 2 1276014/2206327
## 3 10792651/27892529
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0381-raw_1 and luad-0381-raw_2
## Single-end reads will be in luad-0381-raw_M
## 10391100 sequences in the BAM file
## 10391100 sequences exported
## 10391100 reads; of these:
##   10391100 (100.00%) were unpaired; of these:
##     2935041 (28.25%) aligned 0 times
##     4818001 (46.37%) aligned exactly 1 time
##     2638058 (25.39%) aligned &gt;1 times
## 71.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2638058/7456059
## 2 1820764/2638058
## 3 20223913/38214893
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0382-raw_1 and luad-0382-raw_2
## Single-end reads will be in luad-0382-raw_M
## 7830511 sequences in the BAM file
## 7830511 sequences exported
## 7830511 reads; of these:
##   7830511 (100.00%) were unpaired; of these:
##     2564469 (32.75%) aligned 0 times
##     3749455 (47.88%) aligned exactly 1 time
##     1516587 (19.37%) aligned &gt;1 times
## 67.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1516587/5266042
## 2 1076565/1516587
## 3 5463338/13763313
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0383-raw_1 and luad-0383-raw_2
## Single-end reads will be in luad-0383-raw_M
## 9148686 sequences in the BAM file
## 9148686 sequences exported
## 9148686 reads; of these:
##   9148686 (100.00%) were unpaired; of these:
##     2216267 (24.22%) aligned 0 times
##     5092358 (55.66%) aligned exactly 1 time
##     1840061 (20.11%) aligned &gt;1 times
## 75.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1840061/6932419
## 2 1281496/1840061
## 3 6803675/18101443
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0384-raw_1 and luad-0384-raw_2
## Single-end reads will be in luad-0384-raw_M
## 15144151 sequences in the BAM file
## 15144151 sequences exported
## 15144151 reads; of these:
##   15144151 (100.00%) were unpaired; of these:
##     2918521 (19.27%) aligned 0 times
##     5275352 (34.83%) aligned exactly 1 time
##     6950278 (45.89%) aligned &gt;1 times
## 80.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6950278/12225630
## 2 4637906/6950278
## 3 40663126/81705602
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 4.
## Output will be in luad-0385-raw_1 and luad-0385-raw_2
## Single-end reads will be in luad-0385-raw_M
## 8085421 sequences in the BAM file
## 8085421 sequences exported
## 8085421 reads; of these:
##   8085421 (100.00%) were unpaired; of these:
##     2108284 (26.08%) aligned 0 times
##     3833933 (47.42%) aligned exactly 1 time
##     2143204 (26.51%) aligned &gt;1 times
## 73.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2143204/5977137
## 2 1462216/2143204
## 3 13937569/28628555
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0386-raw_1 and luad-0386-raw_2
## Single-end reads will be in luad-0386-raw_M
## 11748365 sequences in the BAM file
## 11748365 sequences exported
## 11748365 reads; of these:
##   11748365 (100.00%) were unpaired; of these:
##     2862112 (24.36%) aligned 0 times
##     6504567 (55.37%) aligned exactly 1 time
##     2381686 (20.27%) aligned &gt;1 times
## 75.64% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2381686/8886253
## 2 1696452/2381686
## 3 9098628/24410687
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0387-raw_1 and luad-0387-raw_2
## Single-end reads will be in luad-0387-raw_M
## 10697276 sequences in the BAM file
## 10697276 sequences exported
## 10697276 reads; of these:
##   10697276 (100.00%) were unpaired; of these:
##     2811637 (26.28%) aligned 0 times
##     5343606 (49.95%) aligned exactly 1 time
##     2542033 (23.76%) aligned &gt;1 times
## 73.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2542033/7885639
## 2 1815532/2542033
## 3 12954235/27875424
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0388-raw_1 and luad-0388-raw_2
## Single-end reads will be in luad-0388-raw_M
## 6021540 sequences in the BAM file
## 6021540 sequences exported
## 6021540 reads; of these:
##   6021540 (100.00%) were unpaired; of these:
##     2143130 (35.59%) aligned 0 times
##     2300684 (38.21%) aligned exactly 1 time
##     1577726 (26.20%) aligned &gt;1 times
## 64.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1577726/3878410
## 2 1181756/1577726
## 3 9857969/17770585
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0389-raw_1 and luad-0389-raw_2
## Single-end reads will be in luad-0389-raw_M
## 13666051 sequences in the BAM file
## 13666051 sequences exported
## 13666051 reads; of these:
##   13666051 (100.00%) were unpaired; of these:
##     5691543 (41.65%) aligned 0 times
##     5068126 (37.09%) aligned exactly 1 time
##     2906382 (21.27%) aligned &gt;1 times
## 58.35% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2906382/7974508
## 2 1953431/2906382
## 3 17442665/35164069
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in luad-0390-raw_1 and luad-0390-raw_2
## Single-end reads will be in luad-0390-raw_M
## 15517378 sequences in the BAM file
## 15517378 sequences exported
## 15517378 reads; of these:
##   15517378 (100.00%) were unpaired; of these:
##     5288291 (34.08%) aligned 0 times
##     8093748 (52.16%) aligned exactly 1 time
##     2135339 (13.76%) aligned &gt;1 times
## 65.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2135339/10229087
## 2 1303159/2135339
## 3 6327198/21041353
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0391-raw_1 and luad-0391-raw_2
## Single-end reads will be in luad-0391-raw_M
## 18044338 sequences in the BAM file
## 18044338 sequences exported
## 18044338 reads; of these:
##   18044338 (100.00%) were unpaired; of these:
##     2748765 (15.23%) aligned 0 times
##     4963216 (27.51%) aligned exactly 1 time
##     10332357 (57.26%) aligned &gt;1 times
## 84.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 10332357/15295573
## 2 7475974/10332357
## 3 86697080/166467589
## [bam_sort_core] merging from 22 files...
## This looks like paired data from lane 4.
## Output will be in luad-0392-raw_1 and luad-0392-raw_2
## Single-end reads will be in luad-0392-raw_M
## 19520729 sequences in the BAM file
## 19520729 sequences exported
## 19520729 reads; of these:
##   19520729 (100.00%) were unpaired; of these:
##     3382323 (17.33%) aligned 0 times
##     11626157 (59.56%) aligned exactly 1 time
##     4512249 (23.12%) aligned &gt;1 times
## 82.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4512249/16138406
## 2 3348312/4512249
## 3 19132602/44231221
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in luad-0393-raw_1 and luad-0393-raw_2
## Single-end reads will be in luad-0393-raw_M
## 14672126 sequences in the BAM file
## 14672126 sequences exported
## 14672126 reads; of these:
##   14672126 (100.00%) were unpaired; of these:
##     1822940 (12.42%) aligned 0 times
##     9000177 (61.34%) aligned exactly 1 time
##     3849009 (26.23%) aligned &gt;1 times
## 87.58% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3849009/12849186
## 2 2937008/3849009
## 3 14967355/35700829
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0394-raw_1 and luad-0394-raw_2
## Single-end reads will be in luad-0394-raw_M
## 3207114 sequences in the BAM file
## 3207114 sequences exported
## 3207114 reads; of these:
##   3207114 (100.00%) were unpaired; of these:
##     734129 (22.89%) aligned 0 times
##     1525376 (47.56%) aligned exactly 1 time
##     947609 (29.55%) aligned &gt;1 times
## 77.11% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 947609/2472985
## 2 644649/947609
## 3 9324282/17423841
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0395-raw_1 and luad-0395-raw_2
## Single-end reads will be in luad-0395-raw_M
## 6431674 sequences in the BAM file
## 6431674 sequences exported
## 6431674 reads; of these:
##   6431674 (100.00%) were unpaired; of these:
##     1667567 (25.93%) aligned 0 times
##     3140632 (48.83%) aligned exactly 1 time
##     1623475 (25.24%) aligned &gt;1 times
## 74.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1623475/4764107
## 2 1166271/1623475
## 3 6765234/16609239
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0396-raw_1 and luad-0396-raw_2
## Single-end reads will be in luad-0396-raw_M
## 7277536 sequences in the BAM file
## 7277536 sequences exported
## 7277536 reads; of these:
##   7277536 (100.00%) were unpaired; of these:
##     1334491 (18.34%) aligned 0 times
##     3640586 (50.02%) aligned exactly 1 time
##     2302459 (31.64%) aligned &gt;1 times
## 81.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2302459/5943045
## 2 1633148/2302459
## 3 10708068/23579630
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0397-raw_1 and luad-0397-raw_2
## Single-end reads will be in luad-0397-raw_M
## 10559324 sequences in the BAM file
## 10559324 sequences exported
## 10559324 reads; of these:
##   10559324 (100.00%) were unpaired; of these:
##     3823241 (36.21%) aligned 0 times
##     4943906 (46.82%) aligned exactly 1 time
##     1792177 (16.97%) aligned &gt;1 times
## 63.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1792177/6736083
## 2 1121394/1792177
## 3 6985388/18945718
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0398-raw_1 and luad-0398-raw_2
## Single-end reads will be in luad-0398-raw_M
## 9034181 sequences in the BAM file
## 9034181 sequences exported
## 9034181 reads; of these:
##   9034181 (100.00%) were unpaired; of these:
##     3643577 (40.33%) aligned 0 times
##     3999968 (44.28%) aligned exactly 1 time
##     1390636 (15.39%) aligned &gt;1 times
## 59.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1390636/5390604
## 2 911215/1390636
## 3 4532023/13249262
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0399-raw_1 and luad-0399-raw_2
## Single-end reads will be in luad-0399-raw_M
## 11186120 sequences in the BAM file
## 11186120 sequences exported
## 11186120 reads; of these:
##   11186120 (100.00%) were unpaired; of these:
##     2611444 (23.35%) aligned 0 times
##     6076920 (54.33%) aligned exactly 1 time
##     2497756 (22.33%) aligned &gt;1 times
## 76.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2497756/8574676
## 2 1816312/2497756
## 3 16023481/33898052
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0400-raw_1 and luad-0400-raw_2
## Single-end reads will be in luad-0400-raw_M
## 12493275 sequences in the BAM file
## 12493275 sequences exported
## 12493275 reads; of these:
##   12493275 (100.00%) were unpaired; of these:
##     3637337 (29.11%) aligned 0 times
##     6406865 (51.28%) aligned exactly 1 time
##     2449073 (19.60%) aligned &gt;1 times
## 70.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2449073/8855938
## 2 1678687/2449073
## 3 7453114/20520304
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0401-raw_1 and luad-0401-raw_2
## Single-end reads will be in luad-0401-raw_M
## 11825414 sequences in the BAM file
## 11825414 sequences exported
## 11825414 reads; of these:
##   11825414 (100.00%) were unpaired; of these:
##     1550800 (13.11%) aligned 0 times
##     7175497 (60.68%) aligned exactly 1 time
##     3099117 (26.21%) aligned &gt;1 times
## 86.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3099117/10274614
## 2 2388308/3099117
## 3 12728342/29629733
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0402-raw_1 and luad-0402-raw_2
## Single-end reads will be in luad-0402-raw_M
## 6969944 sequences in the BAM file
## 6969944 sequences exported
## 6969944 reads; of these:
##   6969944 (100.00%) were unpaired; of these:
##     1221242 (17.52%) aligned 0 times
##     4535823 (65.08%) aligned exactly 1 time
##     1212879 (17.40%) aligned &gt;1 times
## 82.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1212879/5748702
## 2 834053/1212879
## 3 5028930/14735720
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0403-raw_1 and luad-0403-raw_2
## Single-end reads will be in luad-0403-raw_M
## 10648769 sequences in the BAM file
## 10648769 sequences exported
## 10648769 reads; of these:
##   10648769 (100.00%) were unpaired; of these:
##     4294646 (40.33%) aligned 0 times
##     5050289 (47.43%) aligned exactly 1 time
##     1303834 (12.24%) aligned &gt;1 times
## 59.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1303834/6354123
## 2 836693/1303834
## 3 4949007/14676225
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0404-raw_1 and luad-0404-raw_2
## Single-end reads will be in luad-0404-raw_M
## 11713538 sequences in the BAM file
## 11713538 sequences exported
## 11713538 reads; of these:
##   11713538 (100.00%) were unpaired; of these:
##     3255503 (27.79%) aligned 0 times
##     5299291 (45.24%) aligned exactly 1 time
##     3158744 (26.97%) aligned &gt;1 times
## 72.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3158744/8458035
## 2 2225383/3158744
## 3 22629998/42108131
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0405-raw_1 and luad-0405-raw_2
## Single-end reads will be in luad-0405-raw_M
## 10338510 sequences in the BAM file
## 10338510 sequences exported
## 10338510 reads; of these:
##   10338510 (100.00%) were unpaired; of these:
##     1826564 (17.67%) aligned 0 times
##     6195978 (59.93%) aligned exactly 1 time
##     2315968 (22.40%) aligned &gt;1 times
## 82.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2315968/8511946
## 2 1636293/2315968
## 3 7423168/21146075
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0406-raw_1 and luad-0406-raw_2
## Single-end reads will be in luad-0406-raw_M
## 14333337 sequences in the BAM file
## 14333337 sequences exported
## 14333337 reads; of these:
##   14333337 (100.00%) were unpaired; of these:
##     3844878 (26.82%) aligned 0 times
##     7525037 (52.50%) aligned exactly 1 time
##     2963422 (20.68%) aligned &gt;1 times
## 73.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2963422/10488459
## 2 2089465/2963422
## 3 13778244/30257865
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0407-raw_1 and luad-0407-raw_2
## Single-end reads will be in luad-0407-raw_M
## 10744114 sequences in the BAM file
## 10744114 sequences exported
## 10744114 reads; of these:
##   10744114 (100.00%) were unpaired; of these:
##     2299270 (21.40%) aligned 0 times
##     5206887 (48.46%) aligned exactly 1 time
##     3237957 (30.14%) aligned &gt;1 times
## 78.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3237957/8444844
## 2 2160583/3237957
## 3 22140977/44983841
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in luad-0408-raw_1 and luad-0408-raw_2
## Single-end reads will be in luad-0408-raw_M
## 13162812 sequences in the BAM file
## 13162812 sequences exported
## 13162812 reads; of these:
##   13162812 (100.00%) were unpaired; of these:
##     4574753 (34.76%) aligned 0 times
##     6246533 (47.46%) aligned exactly 1 time
##     2341526 (17.79%) aligned &gt;1 times
## 65.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2341526/8588059
## 2 1492627/2341526
## 3 13970359/33863944
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in luad-0409-raw_1 and luad-0409-raw_2
## Single-end reads will be in luad-0409-raw_M
## 10735185 sequences in the BAM file
## 10735185 sequences exported
## 10735185 reads; of these:
##   10735185 (100.00%) were unpaired; of these:
##     4342660 (40.45%) aligned 0 times
##     4995850 (46.54%) aligned exactly 1 time
##     1396675 (13.01%) aligned &gt;1 times
## 59.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1396675/6392525
## 2 934436/1396675
## 3 3744846/13051949
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0410-raw_1 and luad-0410-raw_2
## Single-end reads will be in luad-0410-raw_M
## 11641486 sequences in the BAM file
## 11641486 sequences exported
## 11641486 reads; of these:
##   11641486 (100.00%) were unpaired; of these:
##     5067360 (43.53%) aligned 0 times
##     4598538 (39.50%) aligned exactly 1 time
##     1975588 (16.97%) aligned &gt;1 times
## 56.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1975588/6574126
## 2 1302040/1975588
## 3 10964176/24077045
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0411-raw_1 and luad-0411-raw_2
## Single-end reads will be in luad-0411-raw_M
## 6385998 sequences in the BAM file
## 6385998 sequences exported
## 6385998 reads; of these:
##   6385998 (100.00%) were unpaired; of these:
##     1113991 (17.44%) aligned 0 times
##     3654573 (57.23%) aligned exactly 1 time
##     1617434 (25.33%) aligned &gt;1 times
## 82.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1617434/5272007
## 2 1094435/1617434
## 3 8387297/18352679
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0412-raw_1 and luad-0412-raw_2
## Single-end reads will be in luad-0412-raw_M
## 9996641 sequences in the BAM file
## 9996641 sequences exported
## 9996641 reads; of these:
##   9996641 (100.00%) were unpaired; of these:
##     3564727 (35.66%) aligned 0 times
##     4146357 (41.48%) aligned exactly 1 time
##     2285557 (22.86%) aligned &gt;1 times
## 64.34% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2285557/6431914
## 2 1428058/2285557
## 3 7060995/17845011
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0413-raw_1 and luad-0413-raw_2
## Single-end reads will be in luad-0413-raw_M
## 14346386 sequences in the BAM file
## 14346386 sequences exported
## 14346386 reads; of these:
##   14346386 (100.00%) were unpaired; of these:
##     4896959 (34.13%) aligned 0 times
##     7158559 (49.90%) aligned exactly 1 time
##     2290868 (15.97%) aligned &gt;1 times
## 65.87% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2290868/9449427
## 2 1395042/2290868
## 3 9977187/25967705
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0414-raw_1 and luad-0414-raw_2
## Single-end reads will be in luad-0414-raw_M
## 9337594 sequences in the BAM file
## 9337594 sequences exported
## 9337594 reads; of these:
##   9337594 (100.00%) were unpaired; of these:
##     2784799 (29.82%) aligned 0 times
##     4302392 (46.08%) aligned exactly 1 time
##     2250403 (24.10%) aligned &gt;1 times
## 70.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2250403/6552795
## 2 1467577/2250403
## 3 6413359/16805223
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0415-raw_1 and luad-0415-raw_2
## Single-end reads will be in luad-0415-raw_M
## 9172660 sequences in the BAM file
## 9172660 sequences exported
## 9172660 reads; of these:
##   9172660 (100.00%) were unpaired; of these:
##     1665423 (18.16%) aligned 0 times
##     5169234 (56.35%) aligned exactly 1 time
##     2338003 (25.49%) aligned &gt;1 times
## 81.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2338003/7507237
## 2 1690357/2338003
## 3 11590698/26108838
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0416-raw_1 and luad-0416-raw_2
## Single-end reads will be in luad-0416-raw_M
## 15470546 sequences in the BAM file
## 15470546 sequences exported
## 15470546 reads; of these:
##   15470546 (100.00%) were unpaired; of these:
##     4152344 (26.84%) aligned 0 times
##     7344150 (47.47%) aligned exactly 1 time
##     3974052 (25.69%) aligned &gt;1 times
## 73.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3974052/11318202
## 2 2994481/3974052
## 3 14040239/31792578
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0417-raw_1 and luad-0417-raw_2
## Single-end reads will be in luad-0417-raw_M
## 8362924 sequences in the BAM file
## 8362924 sequences exported
## 8362924 reads; of these:
##   8362924 (100.00%) were unpaired; of these:
##     3786293 (45.27%) aligned 0 times
##     2876476 (34.40%) aligned exactly 1 time
##     1700155 (20.33%) aligned &gt;1 times
## 54.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1700155/4576631
## 2 1169123/1700155
## 3 7731808/16494176
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0418-raw_1 and luad-0418-raw_2
## Single-end reads will be in luad-0418-raw_M
## 11257468 sequences in the BAM file
## 11257468 sequences exported
## 11257468 reads; of these:
##   11257468 (100.00%) were unpaired; of these:
##     4237287 (37.64%) aligned 0 times
##     5437409 (48.30%) aligned exactly 1 time
##     1582772 (14.06%) aligned &gt;1 times
## 62.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1582772/7020181
## 2 995280/1582772
## 3 4841644/15094419
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0419-raw_1 and luad-0419-raw_2
## Single-end reads will be in luad-0419-raw_M
## 11187338 sequences in the BAM file
## 11187338 sequences exported
## 11187338 reads; of these:
##   11187338 (100.00%) were unpaired; of these:
##     1901205 (16.99%) aligned 0 times
##     5475699 (48.95%) aligned exactly 1 time
##     3810434 (34.06%) aligned &gt;1 times
## 83.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3810434/9286133
## 2 2480022/3810434
## 3 24421039/53679482
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 4.
## Output will be in luad-0420-raw_1 and luad-0420-raw_2
## Single-end reads will be in luad-0420-raw_M
## 6834368 sequences in the BAM file
## 6834368 sequences exported
## 6834368 reads; of these:
##   6834368 (100.00%) were unpaired; of these:
##     1835588 (26.86%) aligned 0 times
##     3665748 (53.64%) aligned exactly 1 time
##     1333032 (19.50%) aligned &gt;1 times
## 73.14% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1333032/4998780
## 2 990175/1333032
## 3 5512449/13220362
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0421-raw_1 and luad-0421-raw_2
## Single-end reads will be in luad-0421-raw_M
## 15435567 sequences in the BAM file
## 15435567 sequences exported
## 15435567 reads; of these:
##   15435567 (100.00%) were unpaired; of these:
##     4837250 (31.34%) aligned 0 times
##     8526010 (55.24%) aligned exactly 1 time
##     2072307 (13.43%) aligned &gt;1 times
## 68.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2072307/10598317
## 2 1399481/2072307
## 3 4373688/17946427
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0422-raw_1 and luad-0422-raw_2
## Single-end reads will be in luad-0422-raw_M
## 12399242 sequences in the BAM file
## 12399242 sequences exported
## 12399242 reads; of these:
##   12399242 (100.00%) were unpaired; of these:
##     1926382 (15.54%) aligned 0 times
##     7483451 (60.35%) aligned exactly 1 time
##     2989409 (24.11%) aligned &gt;1 times
## 84.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2989409/10472860
## 2 1969084/2989409
## 3 11045109/29057763
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0423-raw_1 and luad-0423-raw_2
## Single-end reads will be in luad-0423-raw_M
## 12741578 sequences in the BAM file
## 12741578 sequences exported
## 12741578 reads; of these:
##   12741578 (100.00%) were unpaired; of these:
##     3797230 (29.80%) aligned 0 times
##     6146461 (48.24%) aligned exactly 1 time
##     2797887 (21.96%) aligned &gt;1 times
## 70.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2797887/8944348
## 2 1927133/2797887
## 3 18686280/37756521
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0424-raw_1 and luad-0424-raw_2
## Single-end reads will be in luad-0424-raw_M
## 12610451 sequences in the BAM file
## 12610451 sequences exported
## 12610451 reads; of these:
##   12610451 (100.00%) were unpaired; of these:
##     5050984 (40.05%) aligned 0 times
##     5794758 (45.95%) aligned exactly 1 time
##     1764709 (13.99%) aligned &gt;1 times
## 59.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1764709/7559467
## 2 1171859/1764709
## 3 5638018/16969478
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0425-raw_1 and luad-0425-raw_2
## Single-end reads will be in luad-0425-raw_M
## 10159495 sequences in the BAM file
## 10159495 sequences exported
## 10159495 reads; of these:
##   10159495 (100.00%) were unpaired; of these:
##     2425464 (23.87%) aligned 0 times
##     5486799 (54.01%) aligned exactly 1 time
##     2247232 (22.12%) aligned &gt;1 times
## 76.13% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2247232/7734031
## 2 1573257/2247232
## 3 7849549/20965504
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in luad-0426-raw_1 and luad-0426-raw_2
## Single-end reads will be in luad-0426-raw_M
## 15803043 sequences in the BAM file
## 15803043 sequences exported
## 15803043 reads; of these:
##   15803043 (100.00%) were unpaired; of these:
##     1757461 (11.12%) aligned 0 times
##     8396969 (53.14%) aligned exactly 1 time
##     5648613 (35.74%) aligned &gt;1 times
## 88.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5648613/14045582
## 2 3726663/5648613
## 3 27068579/58663259
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 3.
## Output will be in luad-0427-raw_1 and luad-0427-raw_2
## Single-end reads will be in luad-0427-raw_M
## 6322058 sequences in the BAM file
## 6322058 sequences exported
## 6322058 reads; of these:
##   6322058 (100.00%) were unpaired; of these:
##     1195009 (18.90%) aligned 0 times
##     3489390 (55.19%) aligned exactly 1 time
##     1637659 (25.90%) aligned &gt;1 times
## 81.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1637659/5127049
## 2 1202122/1637659
## 3 11045785/22374760
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0428-raw_1 and luad-0428-raw_2
## Single-end reads will be in luad-0428-raw_M
## 5920119 sequences in the BAM file
## 5920119 sequences exported
## 5920119 reads; of these:
##   5920119 (100.00%) were unpaired; of these:
##     1189908 (20.10%) aligned 0 times
##     2907160 (49.11%) aligned exactly 1 time
##     1823051 (30.79%) aligned &gt;1 times
## 79.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1823051/4730211
## 2 1215434/1823051
## 3 5233613/13247277
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0429-raw_1 and luad-0429-raw_2
## Single-end reads will be in luad-0429-raw_M
## 6718655 sequences in the BAM file
## 6718655 sequences exported
## 6718655 reads; of these:
##   6718655 (100.00%) were unpaired; of these:
##     1213358 (18.06%) aligned 0 times
##     3624798 (53.95%) aligned exactly 1 time
##     1880499 (27.99%) aligned &gt;1 times
## 81.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1880499/5505297
## 2 1375663/1880499
## 3 14415771/29627144
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0430-raw_1 and luad-0430-raw_2
## Single-end reads will be in luad-0430-raw_M
## 16810637 sequences in the BAM file
## 16810637 sequences exported
## 16810637 reads; of these:
##   16810637 (100.00%) were unpaired; of these:
##     5160967 (30.70%) aligned 0 times
##     9080386 (54.02%) aligned exactly 1 time
##     2569284 (15.28%) aligned &gt;1 times
## 69.30% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2569284/11649670
## 2 1584482/2569284
## 3 7596365/24826962
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0431-raw_1 and luad-0431-raw_2
## Single-end reads will be in luad-0431-raw_M
## 7461644 sequences in the BAM file
## 7461644 sequences exported
## 7461644 reads; of these:
##   7461644 (100.00%) were unpaired; of these:
##     1776753 (23.81%) aligned 0 times
##     3898154 (52.24%) aligned exactly 1 time
##     1786737 (23.95%) aligned &gt;1 times
## 76.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1786737/5684891
## 2 1211602/1786737
## 3 9238474/20096270
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0432-raw_1 and luad-0432-raw_2
## Single-end reads will be in luad-0432-raw_M
## 10815817 sequences in the BAM file
## 10815817 sequences exported
## 10815817 reads; of these:
##   10815817 (100.00%) were unpaired; of these:
##     3806301 (35.19%) aligned 0 times
##     5320795 (49.19%) aligned exactly 1 time
##     1688721 (15.61%) aligned &gt;1 times
## 64.81% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1688721/7009516
## 2 1084842/1688721
## 3 5834259/16457827
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0433-raw_1 and luad-0433-raw_2
## Single-end reads will be in luad-0433-raw_M
## 10441689 sequences in the BAM file
## 10441689 sequences exported
## 10441689 reads; of these:
##   10441689 (100.00%) were unpaired; of these:
##     2371459 (22.71%) aligned 0 times
##     5217522 (49.97%) aligned exactly 1 time
##     2852708 (27.32%) aligned &gt;1 times
## 77.29% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2852708/8070230
## 2 1995427/2852708
## 3 20317386/40421890
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0434-raw_1 and luad-0434-raw_2
## Single-end reads will be in luad-0434-raw_M
## 8814882 sequences in the BAM file
## 8814882 sequences exported
## 8814882 reads; of these:
##   8814882 (100.00%) were unpaired; of these:
##     3547100 (40.24%) aligned 0 times
##     4311725 (48.91%) aligned exactly 1 time
##     956057 (10.85%) aligned &gt;1 times
## 59.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 956057/5267782
## 2 598253/956057
## 3 2950643/10358932
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in luad-0435-raw_1 and luad-0435-raw_2
## Single-end reads will be in luad-0435-raw_M
## 14505756 sequences in the BAM file
## 14505756 sequences exported
## 14505756 reads; of these:
##   14505756 (100.00%) were unpaired; of these:
##     3421984 (23.59%) aligned 0 times
##     8276689 (57.06%) aligned exactly 1 time
##     2807083 (19.35%) aligned &gt;1 times
## 76.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2807083/11083772
## 2 1973295/2807083
## 3 14486159/35240825
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0436-raw_1 and luad-0436-raw_2
## Single-end reads will be in luad-0436-raw_M
## 13858317 sequences in the BAM file
## 13858317 sequences exported
## 13858317 reads; of these:
##   13858317 (100.00%) were unpaired; of these:
##     3538862 (25.54%) aligned 0 times
##     6352343 (45.84%) aligned exactly 1 time
##     3967112 (28.63%) aligned &gt;1 times
## 74.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3967112/10319455
## 2 2509728/3967112
## 3 9290866/25840015
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0437-raw_1 and luad-0437-raw_2
## Single-end reads will be in luad-0437-raw_M
## 15079534 sequences in the BAM file
## 15079534 sequences exported
## 15079534 reads; of these:
##   15079534 (100.00%) were unpaired; of these:
##     3906989 (25.91%) aligned 0 times
##     8215083 (54.48%) aligned exactly 1 time
##     2957462 (19.61%) aligned &gt;1 times
## 74.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2957462/11172545
## 2 1858339/2957462
## 3 10570449/30235694
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0438-raw_1 and luad-0438-raw_2
## Single-end reads will be in luad-0438-raw_M
## 15630957 sequences in the BAM file
## 15630957 sequences exported
## 15630957 reads; of these:
##   15630957 (100.00%) were unpaired; of these:
##     3878887 (24.82%) aligned 0 times
##     8412021 (53.82%) aligned exactly 1 time
##     3340049 (21.37%) aligned &gt;1 times
## 75.18% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3340049/11752070
## 2 2349869/3340049
## 3 12817946/32190536
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0439-raw_1 and luad-0439-raw_2
## Single-end reads will be in luad-0439-raw_M
## 7295134 sequences in the BAM file
## 7295134 sequences exported
## 7295134 reads; of these:
##   7295134 (100.00%) were unpaired; of these:
##     1536759 (21.07%) aligned 0 times
##     3437621 (47.12%) aligned exactly 1 time
##     2320754 (31.81%) aligned &gt;1 times
## 78.93% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2320754/5758375
## 2 1707038/2320754
## 3 14242878/28460694
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0440-raw_1 and luad-0440-raw_2
## Single-end reads will be in luad-0440-raw_M
## 9260616 sequences in the BAM file
## 9260616 sequences exported
## 9260616 reads; of these:
##   9260616 (100.00%) were unpaired; of these:
##     3241882 (35.01%) aligned 0 times
##     4100557 (44.28%) aligned exactly 1 time
##     1918177 (20.71%) aligned &gt;1 times
## 64.99% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1918177/6018734
## 2 1261756/1918177
## 3 6126745/15701931
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0441-raw_1 and luad-0441-raw_2
## Single-end reads will be in luad-0441-raw_M
## 13873352 sequences in the BAM file
## 13873352 sequences exported
## 13873352 reads; of these:
##   13873352 (100.00%) were unpaired; of these:
##     3435702 (24.76%) aligned 0 times
##     7848276 (56.57%) aligned exactly 1 time
##     2589374 (18.66%) aligned &gt;1 times
## 75.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2589374/10437650
## 2 1811273/2589374
## 3 15410461/34458541
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in luad-0442-raw_1 and luad-0442-raw_2
## Single-end reads will be in luad-0442-raw_M
## 15426178 sequences in the BAM file
## 15426178 sequences exported
## 15426178 reads; of these:
##   15426178 (100.00%) were unpaired; of these:
##     1409646 (9.14%) aligned 0 times
##     8712701 (56.48%) aligned exactly 1 time
##     5303831 (34.38%) aligned &gt;1 times
## 90.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5303831/14016532
## 2 3449040/5303831
## 3 31983143/68726708
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 4.
## Output will be in luad-0443-raw_1 and luad-0443-raw_2
## Single-end reads will be in luad-0443-raw_M
## 8556811 sequences in the BAM file
## 8556811 sequences exported
## 8556811 reads; of these:
##   8556811 (100.00%) were unpaired; of these:
##     2253253 (26.33%) aligned 0 times
##     4553529 (53.22%) aligned exactly 1 time
##     1750029 (20.45%) aligned &gt;1 times
## 73.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1750029/6303558
## 2 1157297/1750029
## 3 9298218/22444248
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0444-raw_1 and luad-0444-raw_2
## Single-end reads will be in luad-0444-raw_M
## 20492891 sequences in the BAM file
## 20492891 sequences exported
## 20492891 reads; of these:
##   20492891 (100.00%) were unpaired; of these:
##     3588970 (17.51%) aligned 0 times
##     9629772 (46.99%) aligned exactly 1 time
##     7274149 (35.50%) aligned &gt;1 times
## 82.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 7274149/16903921
## 2 4844935/7274149
## 3 55752445/120184907
## [bam_sort_core] merging from 18 files...
## This looks like paired data from lane 4.
## Output will be in luad-0445-raw_1 and luad-0445-raw_2
## Single-end reads will be in luad-0445-raw_M
## 8335922 sequences in the BAM file
## 8335922 sequences exported
## 8335922 reads; of these:
##   8335922 (100.00%) were unpaired; of these:
##     1933851 (23.20%) aligned 0 times
##     4764003 (57.15%) aligned exactly 1 time
##     1638068 (19.65%) aligned &gt;1 times
## 76.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1638068/6402071
## 2 1136574/1638068
## 3 5498014/16072761
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0446-raw_1 and luad-0446-raw_2
## Single-end reads will be in luad-0446-raw_M
## 13340056 sequences in the BAM file
## 13340056 sequences exported
## 13340056 reads; of these:
##   13340056 (100.00%) were unpaired; of these:
##     1886717 (14.14%) aligned 0 times
##     6877925 (51.56%) aligned exactly 1 time
##     4575414 (34.30%) aligned &gt;1 times
## 85.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4575414/11453339
## 2 3295790/4575414
## 3 21691327/45738460
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in luad-0447-raw_1 and luad-0447-raw_2
## Single-end reads will be in luad-0447-raw_M
## 8048070 sequences in the BAM file
## 8048070 sequences exported
## 8048070 reads; of these:
##   8048070 (100.00%) were unpaired; of these:
##     1655912 (20.58%) aligned 0 times
##     4375534 (54.37%) aligned exactly 1 time
##     2016624 (25.06%) aligned &gt;1 times
## 79.42% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2016624/6392158
## 2 1439991/2016624
## 3 8533504/20377864
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0448-raw_1 and luad-0448-raw_2
## Single-end reads will be in luad-0448-raw_M
## 3247963 sequences in the BAM file
## 3247963 sequences exported
## 3247963 reads; of these:
##   3247963 (100.00%) were unpaired; of these:
##     909033 (27.99%) aligned 0 times
##     1551476 (47.77%) aligned exactly 1 time
##     787454 (24.24%) aligned &gt;1 times
## 72.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 787454/2338930
## 2 530407/787454
## 3 3809433/8263177
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in luad-0449-raw_1 and luad-0449-raw_2
## Single-end reads will be in luad-0449-raw_M
## 18567892 sequences in the BAM file
## 18567892 sequences exported
## 18567892 reads; of these:
##   18567892 (100.00%) were unpaired; of these:
##     4717756 (25.41%) aligned 0 times
##     9707046 (52.28%) aligned exactly 1 time
##     4143090 (22.31%) aligned &gt;1 times
## 74.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4143090/13850136
## 2 3053692/4143090
## 3 19435145/43567764
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in luad-0450-raw_1 and luad-0450-raw_2
## Single-end reads will be in luad-0450-raw_M
## 6409465 sequences in the BAM file
## 6409465 sequences exported
## 6409465 reads; of these:
##   6409465 (100.00%) were unpaired; of these:
##     1536898 (23.98%) aligned 0 times
##     3458286 (53.96%) aligned exactly 1 time
##     1414281 (22.07%) aligned &gt;1 times
## 76.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1414281/4872567
## 2 1015426/1414281
## 3 5587221/13213643
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0451-raw_1 and luad-0451-raw_2
## Single-end reads will be in luad-0451-raw_M
## 11108789 sequences in the BAM file
## 11108789 sequences exported
## 11108789 reads; of these:
##   11108789 (100.00%) were unpaired; of these:
##     1476057 (13.29%) aligned 0 times
##     6950359 (62.57%) aligned exactly 1 time
##     2682373 (24.15%) aligned &gt;1 times
## 86.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2682373/9632732
## 2 1870148/2682373
## 3 9706403/25416522
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0452-raw_1 and luad-0452-raw_2
## Single-end reads will be in luad-0452-raw_M
## 16601613 sequences in the BAM file
## 16601613 sequences exported
## 16601613 reads; of these:
##   16601613 (100.00%) were unpaired; of these:
##     4283103 (25.80%) aligned 0 times
##     9641155 (58.07%) aligned exactly 1 time
##     2677355 (16.13%) aligned &gt;1 times
## 74.20% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2677355/12318510
## 2 2042396/2677355
## 3 12275500/33378221
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0453-raw_1 and luad-0453-raw_2
## Single-end reads will be in luad-0453-raw_M
## 15225679 sequences in the BAM file
## 15225679 sequences exported
## 15225679 reads; of these:
##   15225679 (100.00%) were unpaired; of these:
##     4035348 (26.50%) aligned 0 times
##     6842025 (44.94%) aligned exactly 1 time
##     4348306 (28.56%) aligned &gt;1 times
## 73.50% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4348306/11190331
## 2 3042385/4348306
## 3 23128416/43131712
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0454-raw_1 and luad-0454-raw_2
## Single-end reads will be in luad-0454-raw_M
## 7300676 sequences in the BAM file
## 7300676 sequences exported
## 7300676 reads; of these:
##   7300676 (100.00%) were unpaired; of these:
##     1816145 (24.88%) aligned 0 times
##     4235973 (58.02%) aligned exactly 1 time
##     1248558 (17.10%) aligned &gt;1 times
## 75.12% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1248558/5484531
## 2 835201/1248558
## 3 5164044/14218639
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0455-raw_1 and luad-0455-raw_2
## Single-end reads will be in luad-0455-raw_M
## 8680263 sequences in the BAM file
## 8680263 sequences exported
## 8680263 reads; of these:
##   8680263 (100.00%) were unpaired; of these:
##     2195670 (25.29%) aligned 0 times
##     4075535 (46.95%) aligned exactly 1 time
##     2409058 (27.75%) aligned &gt;1 times
## 74.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2409058/6484593
## 2 1627801/2409058
## 3 9071757/20232451
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0456-raw_1 and luad-0456-raw_2
## Single-end reads will be in luad-0456-raw_M
## 16353700 sequences in the BAM file
## 16353700 sequences exported
## 16353700 reads; of these:
##   16353700 (100.00%) were unpaired; of these:
##     2930500 (17.92%) aligned 0 times
##     9053292 (55.36%) aligned exactly 1 time
##     4369908 (26.72%) aligned &gt;1 times
## 82.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4369908/13423200
## 2 3181770/4369908
## 3 21516048/49536591
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 6.
## Output will be in luad-0457-raw_1 and luad-0457-raw_2
## Single-end reads will be in luad-0457-raw_M
## 9985365 sequences in the BAM file
## 9985365 sequences exported
## 9985365 reads; of these:
##   9985365 (100.00%) were unpaired; of these:
##     1875578 (18.78%) aligned 0 times
##     5448841 (54.57%) aligned exactly 1 time
##     2660946 (26.65%) aligned &gt;1 times
## 81.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2660946/8109787
## 2 1992030/2660946
## 3 14220617/31483534
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0458-raw_1 and luad-0458-raw_2
## Single-end reads will be in luad-0458-raw_M
## 6412983 sequences in the BAM file
## 6412983 sequences exported
## 6412983 reads; of these:
##   6412983 (100.00%) were unpaired; of these:
##     2027680 (31.62%) aligned 0 times
##     3017248 (47.05%) aligned exactly 1 time
##     1368055 (21.33%) aligned &gt;1 times
## 68.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1368055/4385303
## 2 954821/1368055
## 3 8074311/16768096
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0459-raw_1 and luad-0459-raw_2
## Single-end reads will be in luad-0459-raw_M
## 14563685 sequences in the BAM file
## 14563685 sequences exported
## 14563685 reads; of these:
##   14563685 (100.00%) were unpaired; of these:
##     1933637 (13.28%) aligned 0 times
##     8516839 (58.48%) aligned exactly 1 time
##     4113209 (28.24%) aligned &gt;1 times
## 86.72% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4113209/12630048
## 2 3202761/4113209
## 3 21666535/43519961
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in luad-0460-raw_1 and luad-0460-raw_2
## Single-end reads will be in luad-0460-raw_M
## 9805738 sequences in the BAM file
## 9805738 sequences exported
## 9805738 reads; of these:
##   9805738 (100.00%) were unpaired; of these:
##     2612465 (26.64%) aligned 0 times
##     4241269 (43.25%) aligned exactly 1 time
##     2952004 (30.10%) aligned &gt;1 times
## 73.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2952004/7193273
## 2 2061495/2952004
## 3 24159904/43197683
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in luad-0461-raw_1 and luad-0461-raw_2
## Single-end reads will be in luad-0461-raw_M
## 8525453 sequences in the BAM file
## 8525453 sequences exported
## 8525453 reads; of these:
##   8525453 (100.00%) were unpaired; of these:
##     1611973 (18.91%) aligned 0 times
##     4480575 (52.56%) aligned exactly 1 time
##     2432905 (28.54%) aligned &gt;1 times
## 81.09% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2432905/6913480
## 2 1702341/2432905
## 3 11621057/24965712
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0462-raw_1 and luad-0462-raw_2
## Single-end reads will be in luad-0462-raw_M
## 5936729 sequences in the BAM file
## 5936729 sequences exported
## 5936729 reads; of these:
##   5936729 (100.00%) were unpaired; of these:
##     2008571 (33.83%) aligned 0 times
##     2892221 (48.72%) aligned exactly 1 time
##     1035937 (17.45%) aligned &gt;1 times
## 66.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1035937/3928158
## 2 720006/1035937
## 3 3695737/9591975
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in luad-0463-raw_1 and luad-0463-raw_2
## Single-end reads will be in luad-0463-raw_M
## 5475567 sequences in the BAM file
## 5475567 sequences exported
## 5475567 reads; of these:
##   5475567 (100.00%) were unpaired; of these:
##     1223293 (22.34%) aligned 0 times
##     3163404 (57.77%) aligned exactly 1 time
##     1088870 (19.89%) aligned &gt;1 times
## 77.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1088870/4252274
## 2 785672/1088870
## 3 4946987/12031106
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in luad-0464-raw_1 and luad-0464-raw_2
## Single-end reads will be in luad-0464-raw_M
## 6658089 sequences in the BAM file
## 6658089 sequences exported
## 6658089 reads; of these:
##   6658089 (100.00%) were unpaired; of these:
##     1248798 (18.76%) aligned 0 times
##     3594299 (53.98%) aligned exactly 1 time
##     1814992 (27.26%) aligned &gt;1 times
## 81.24% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1814992/5409291
## 2 1235776/1814992
## 3 12365857/25901300
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0465-raw_1 and luad-0465-raw_2
## Single-end reads will be in luad-0465-raw_M
## 9579077 sequences in the BAM file
## 9579077 sequences exported
## 9579077 reads; of these:
##   9579077 (100.00%) were unpaired; of these:
##     1694405 (17.69%) aligned 0 times
##     5713988 (59.65%) aligned exactly 1 time
##     2170684 (22.66%) aligned &gt;1 times
## 82.31% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2170684/7884672
## 2 1546435/2170684
## 3 7494331/20639651
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0466-raw_1 and luad-0466-raw_2
## Single-end reads will be in luad-0466-raw_M
## 9793152 sequences in the BAM file
## 9793152 sequences exported
## 9793152 reads; of these:
##   9793152 (100.00%) were unpaired; of these:
##     3367816 (34.39%) aligned 0 times
##     5199152 (53.09%) aligned exactly 1 time
##     1226184 (12.52%) aligned &gt;1 times
## 65.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1226184/6425336
## 2 784107/1226184
## 3 5314025/15547644
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0467-raw_1 and luad-0467-raw_2
## Single-end reads will be in luad-0467-raw_M
## 14102692 sequences in the BAM file
## 14102692 sequences exported
## 14102692 reads; of these:
##   14102692 (100.00%) were unpaired; of these:
##     2827403 (20.05%) aligned 0 times
##     7763609 (55.05%) aligned exactly 1 time
##     3511680 (24.90%) aligned &gt;1 times
## 79.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3511680/11275289
## 2 2515655/3511680
## 3 21087979/45646645
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in luad-0468-raw_1 and luad-0468-raw_2
## Single-end reads will be in luad-0468-raw_M
## 16177871 sequences in the BAM file
## 16177871 sequences exported
## 16177871 reads; of these:
##   16177871 (100.00%) were unpaired; of these:
##     7011889 (43.34%) aligned 0 times
##     6497025 (40.16%) aligned exactly 1 time
##     2668957 (16.50%) aligned &gt;1 times
## 56.66% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2668957/9165982
## 2 1927858/2668957
## 3 5390257/17883735
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0469-raw_1 and luad-0469-raw_2
## Single-end reads will be in luad-0469-raw_M
## 9924893 sequences in the BAM file
## 9924893 sequences exported
## 9924893 reads; of these:
##   9924893 (100.00%) were unpaired; of these:
##     2917875 (29.40%) aligned 0 times
##     4795298 (48.32%) aligned exactly 1 time
##     2211720 (22.28%) aligned &gt;1 times
## 70.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2211720/7007018
## 2 1659431/2211720
## 3 8348902/19117076
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0470-raw_1 and luad-0470-raw_2
## Single-end reads will be in luad-0470-raw_M
## 13241107 sequences in the BAM file
## 13241107 sequences exported
## 13241107 reads; of these:
##   13241107 (100.00%) were unpaired; of these:
##     1992453 (15.05%) aligned 0 times
##     6477463 (48.92%) aligned exactly 1 time
##     4771191 (36.03%) aligned &gt;1 times
## 84.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4771191/11248654
## 2 3312856/4771191
## 3 42009537/82497693
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 6.
## Output will be in luad-0471-raw_1 and luad-0471-raw_2
## Single-end reads will be in luad-0471-raw_M
## 7614713 sequences in the BAM file
## 7614713 sequences exported
## 7614713 reads; of these:
##   7614713 (100.00%) were unpaired; of these:
##     2119714 (27.84%) aligned 0 times
##     3236820 (42.51%) aligned exactly 1 time
##     2258179 (29.66%) aligned &gt;1 times
## 72.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2258179/5494999
## 2 1422078/2258179
## 3 8522006/20135615
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0472-raw_1 and luad-0472-raw_2
## Single-end reads will be in luad-0472-raw_M
## 8651461 sequences in the BAM file
## 8651461 sequences exported
## 8651461 reads; of these:
##   8651461 (100.00%) were unpaired; of these:
##     2267577 (26.21%) aligned 0 times
##     4223979 (48.82%) aligned exactly 1 time
##     2159905 (24.97%) aligned &gt;1 times
## 73.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2159905/6383884
## 2 1554943/2159905
## 3 10100816/23009531
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0473-raw_1 and luad-0473-raw_2
## Single-end reads will be in luad-0473-raw_M
## 8248457 sequences in the BAM file
## 8248457 sequences exported
## 8248457 reads; of these:
##   8248457 (100.00%) were unpaired; of these:
##     869002 (10.54%) aligned 0 times
##     4562788 (55.32%) aligned exactly 1 time
##     2816667 (34.15%) aligned &gt;1 times
## 89.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2816667/7379455
## 2 2031606/2816667
## 3 22544770/45188238
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in luad-0474-raw_1 and luad-0474-raw_2
## Single-end reads will be in luad-0474-raw_M
## 16495888 sequences in the BAM file
## 16495888 sequences exported
## 16495888 reads; of these:
##   16495888 (100.00%) were unpaired; of these:
##     4081202 (24.74%) aligned 0 times
##     9906187 (60.05%) aligned exactly 1 time
##     2508499 (15.21%) aligned &gt;1 times
## 75.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2508499/12414686
## 2 1577037/2508499
## 3 7000918/24292136
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0475-raw_1 and luad-0475-raw_2
## Single-end reads will be in luad-0475-raw_M
## 13977640 sequences in the BAM file
## 13977640 sequences exported
## 13977640 reads; of these:
##   13977640 (100.00%) were unpaired; of these:
##     4296581 (30.74%) aligned 0 times
##     7571270 (54.17%) aligned exactly 1 time
##     2109789 (15.09%) aligned &gt;1 times
## 69.26% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2109789/9681059
## 2 1502874/2109789
## 3 4979768/17921737
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0476-raw_1 and luad-0476-raw_2
## Single-end reads will be in luad-0476-raw_M
## 9155088 sequences in the BAM file
## 9155088 sequences exported
## 9155088 reads; of these:
##   9155088 (100.00%) were unpaired; of these:
##     1931762 (21.10%) aligned 0 times
##     5011969 (54.75%) aligned exactly 1 time
##     2211357 (24.15%) aligned &gt;1 times
## 78.90% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2211357/7223326
## 2 1625307/2211357
## 3 11499886/25048831
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0477-raw_1 and luad-0477-raw_2
## Single-end reads will be in luad-0477-raw_M
## 15222195 sequences in the BAM file
## 15222195 sequences exported
## 15222195 reads; of these:
##   15222195 (100.00%) were unpaired; of these:
##     5103532 (33.53%) aligned 0 times
##     7757859 (50.96%) aligned exactly 1 time
##     2360804 (15.51%) aligned &gt;1 times
## 66.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2360804/10118663
## 2 1425491/2360804
## 3 5196986/18903878
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0478-raw_1 and luad-0478-raw_2
## Single-end reads will be in luad-0478-raw_M
## 5773997 sequences in the BAM file
## 5773997 sequences exported
## 5773997 reads; of these:
##   5773997 (100.00%) were unpaired; of these:
##     1697712 (29.40%) aligned 0 times
##     2365271 (40.96%) aligned exactly 1 time
##     1711014 (29.63%) aligned &gt;1 times
## 70.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1711014/4076285
## 2 1199355/1711014
## 3 6472618/14152036
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0479-raw_1 and luad-0479-raw_2
## Single-end reads will be in luad-0479-raw_M
## 15339375 sequences in the BAM file
## 15339375 sequences exported
## 15339375 reads; of these:
##   15339375 (100.00%) were unpaired; of these:
##     2108475 (13.75%) aligned 0 times
##     9025064 (58.84%) aligned exactly 1 time
##     4205836 (27.42%) aligned &gt;1 times
## 86.25% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4205836/13230900
## 2 2974526/4205836
## 3 23204453/51784003
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 4.
## Output will be in luad-0480-raw_1 and luad-0480-raw_2
## Single-end reads will be in luad-0480-raw_M
## 14098159 sequences in the BAM file
## 14098159 sequences exported
## 14098159 reads; of these:
##   14098159 (100.00%) were unpaired; of these:
##     5301906 (37.61%) aligned 0 times
##     6997805 (49.64%) aligned exactly 1 time
##     1798448 (12.76%) aligned &gt;1 times
## 62.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1798448/8796253
## 2 1141979/1798448
## 3 4601384/16717223
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0481-raw_1 and luad-0481-raw_2
## Single-end reads will be in luad-0481-raw_M
## 8187779 sequences in the BAM file
## 8187779 sequences exported
## 8187779 reads; of these:
##   8187779 (100.00%) were unpaired; of these:
##     2325112 (28.40%) aligned 0 times
##     4205078 (51.36%) aligned exactly 1 time
##     1657589 (20.24%) aligned &gt;1 times
## 71.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1657589/5862667
## 2 1161463/1657589
## 3 8884147/20675927
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0482-raw_1 and luad-0482-raw_2
## Single-end reads will be in luad-0482-raw_M
## 6310094 sequences in the BAM file
## 6310094 sequences exported
## 6310094 reads; of these:
##   6310094 (100.00%) were unpaired; of these:
##     1214552 (19.25%) aligned 0 times
##     3663449 (58.06%) aligned exactly 1 time
##     1432093 (22.70%) aligned &gt;1 times
## 80.75% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1432093/5095542
## 2 1006388/1432093
## 3 6562038/14941057
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0483-raw_1 and luad-0483-raw_2
## Single-end reads will be in luad-0483-raw_M
## 9579445 sequences in the BAM file
## 9579445 sequences exported
## 9579445 reads; of these:
##   9579445 (100.00%) were unpaired; of these:
##     1834058 (19.15%) aligned 0 times
##     5521519 (57.64%) aligned exactly 1 time
##     2223868 (23.21%) aligned &gt;1 times
## 80.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2223868/7745387
## 2 1315760/2223868
## 3 14350288/33667028
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0484-raw_1 and luad-0484-raw_2
## Single-end reads will be in luad-0484-raw_M
## 11767809 sequences in the BAM file
## 11767809 sequences exported
## 11767809 reads; of these:
##   11767809 (100.00%) were unpaired; of these:
##     2123103 (18.04%) aligned 0 times
##     6211614 (52.78%) aligned exactly 1 time
##     3433092 (29.17%) aligned &gt;1 times
## 81.96% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3433092/9644706
## 2 2554736/3433092
## 3 16097851/33427817
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0485-raw_1 and luad-0485-raw_2
## Single-end reads will be in luad-0485-raw_M
## 12198667 sequences in the BAM file
## 12198667 sequences exported
## 12198667 reads; of these:
##   12198667 (100.00%) were unpaired; of these:
##     5046750 (41.37%) aligned 0 times
##     5581500 (45.75%) aligned exactly 1 time
##     1570417 (12.87%) aligned &gt;1 times
## 58.63% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1570417/7151917
## 2 1021825/1570417
## 3 4511386/14693034
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in luad-0486-raw_1 and luad-0486-raw_2
## Single-end reads will be in luad-0486-raw_M
## 11285578 sequences in the BAM file
## 11285578 sequences exported
## 11285578 reads; of these:
##   11285578 (100.00%) were unpaired; of these:
##     4618665 (40.93%) aligned 0 times
##     5221759 (46.27%) aligned exactly 1 time
##     1445154 (12.81%) aligned &gt;1 times
## 59.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1445154/6666913
## 2 865861/1445154
## 3 3813506/13434171
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0487-raw_1 and luad-0487-raw_2
## Single-end reads will be in luad-0487-raw_M
## 12020495 sequences in the BAM file
## 12020495 sequences exported
## 12020495 reads; of these:
##   12020495 (100.00%) were unpaired; of these:
##     3939762 (32.78%) aligned 0 times
##     5950501 (49.50%) aligned exactly 1 time
##     2130232 (17.72%) aligned &gt;1 times
## 67.22% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2130232/8080733
## 2 1291344/2130232
## 3 7997581/21251652
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0488-raw_1 and luad-0488-raw_2
## Single-end reads will be in luad-0488-raw_M
## 11402103 sequences in the BAM file
## 11402103 sequences exported
## 11402103 reads; of these:
##   11402103 (100.00%) were unpaired; of these:
##     1429625 (12.54%) aligned 0 times
##     7059051 (61.91%) aligned exactly 1 time
##     2913427 (25.55%) aligned &gt;1 times
## 87.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2913427/9972478
## 2 1935642/2913427
## 3 15803747/37623984
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0489-raw_1 and luad-0489-raw_2
## Single-end reads will be in luad-0489-raw_M
## 17922098 sequences in the BAM file
## 17922098 sequences exported
## 17922098 reads; of these:
##   17922098 (100.00%) were unpaired; of these:
##     5293423 (29.54%) aligned 0 times
##     10310579 (57.53%) aligned exactly 1 time
##     2318096 (12.93%) aligned &gt;1 times
## 70.46% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2318096/12628675
## 2 1452656/2318096
## 3 5077365/21385041
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0490-raw_1 and luad-0490-raw_2
## Single-end reads will be in luad-0490-raw_M
## 5298972 sequences in the BAM file
## 5298972 sequences exported
## 5298972 reads; of these:
##   5298972 (100.00%) were unpaired; of these:
##     965008 (18.21%) aligned 0 times
##     3187839 (60.16%) aligned exactly 1 time
##     1146125 (21.63%) aligned &gt;1 times
## 81.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1146125/4333964
## 2 755646/1146125
## 3 5922602/14562417
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0491-raw_1 and luad-0491-raw_2
## Single-end reads will be in luad-0491-raw_M
## 10174913 sequences in the BAM file
## 10174913 sequences exported
## 10174913 reads; of these:
##   10174913 (100.00%) were unpaired; of these:
##     2648849 (26.03%) aligned 0 times
##     4825399 (47.42%) aligned exactly 1 time
##     2700665 (26.54%) aligned &gt;1 times
## 73.97% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2700665/7526064
## 2 1926330/2700665
## 3 20109123/37375320
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0492-raw_1 and luad-0492-raw_2
## Single-end reads will be in luad-0492-raw_M
## 17143775 sequences in the BAM file
## 17143775 sequences exported
## 17143775 reads; of these:
##   17143775 (100.00%) were unpaired; of these:
##     7249869 (42.29%) aligned 0 times
##     7275261 (42.44%) aligned exactly 1 time
##     2618645 (15.27%) aligned &gt;1 times
## 57.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2618645/9893906
## 2 1749848/2618645
## 3 14239491/32299340
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0493-raw_1 and luad-0493-raw_2
## Single-end reads will be in luad-0493-raw_M
## 17098676 sequences in the BAM file
## 17098676 sequences exported
## 17098676 reads; of these:
##   17098676 (100.00%) were unpaired; of these:
##     2831158 (16.56%) aligned 0 times
##     8777876 (51.34%) aligned exactly 1 time
##     5489642 (32.11%) aligned &gt;1 times
## 83.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 5489642/14267518
## 2 3744567/5489642
## 3 21285223/51875503
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 5.
## Output will be in luad-0494-raw_1 and luad-0494-raw_2
## Single-end reads will be in luad-0494-raw_M
## 9375889 sequences in the BAM file
## 9375889 sequences exported
## 9375889 reads; of these:
##   9375889 (100.00%) were unpaired; of these:
##     2060771 (21.98%) aligned 0 times
##     4614851 (49.22%) aligned exactly 1 time
##     2700267 (28.80%) aligned &gt;1 times
## 78.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2700267/7315118
## 2 1945767/2700267
## 3 8070450/20313086
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0495-raw_1 and luad-0495-raw_2
## Single-end reads will be in luad-0495-raw_M
## 9028191 sequences in the BAM file
## 9028191 sequences exported
## 9028191 reads; of these:
##   9028191 (100.00%) were unpaired; of these:
##     1622518 (17.97%) aligned 0 times
##     5080971 (56.28%) aligned exactly 1 time
##     2324702 (25.75%) aligned &gt;1 times
## 82.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2324702/7405673
## 2 1494296/2324702
## 3 11724761/27514516
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in luad-0496-raw_1 and luad-0496-raw_2
## Single-end reads will be in luad-0496-raw_M
## 12927017 sequences in the BAM file
## 12927017 sequences exported
## 12927017 reads; of these:
##   12927017 (100.00%) were unpaired; of these:
##     4013663 (31.05%) aligned 0 times
##     6942667 (53.71%) aligned exactly 1 time
##     1970687 (15.24%) aligned &gt;1 times
## 68.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1970687/8913354
## 2 1259097/1970687
## 3 6489176/20380049
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0497-raw_1 and luad-0497-raw_2
## Single-end reads will be in luad-0497-raw_M
## 7018589 sequences in the BAM file
## 7018589 sequences exported
## 7018589 reads; of these:
##   7018589 (100.00%) were unpaired; of these:
##     1779434 (25.35%) aligned 0 times
##     1943830 (27.70%) aligned exactly 1 time
##     3295325 (46.95%) aligned &gt;1 times
## 74.65% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3295325/5239155
## 2 2068068/3295325
## 3 26004443/50025128
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in luad-0498-raw_1 and luad-0498-raw_2
## Single-end reads will be in luad-0498-raw_M
## 9912451 sequences in the BAM file
## 9912451 sequences exported
## 9912451 reads; of these:
##   9912451 (100.00%) were unpaired; of these:
##     2177390 (21.97%) aligned 0 times
##     5458342 (55.07%) aligned exactly 1 time
##     2276719 (22.97%) aligned &gt;1 times
## 78.03% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2276719/7735061
## 2 1719346/2276719
## 3 8320470/20620096
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0499-raw_1 and luad-0499-raw_2
## Single-end reads will be in luad-0499-raw_M
## 21675454 sequences in the BAM file
## 21675454 sequences exported
## 21675454 reads; of these:
##   21675454 (100.00%) were unpaired; of these:
##     3033018 (13.99%) aligned 0 times
##     12419315 (57.30%) aligned exactly 1 time
##     6223121 (28.71%) aligned &gt;1 times
## 86.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6223121/18642436
## 2 4656624/6223121
## 3 22332333/52096869
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 1.
## Output will be in luad-0500-raw_1 and luad-0500-raw_2
## Single-end reads will be in luad-0500-raw_M
## 18714467 sequences in the BAM file
## 18714467 sequences exported
## 18714467 reads; of these:
##   18714467 (100.00%) were unpaired; of these:
##     2214220 (11.83%) aligned 0 times
##     10448750 (55.83%) aligned exactly 1 time
##     6051497 (32.34%) aligned &gt;1 times
## 88.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6051497/16500247
## 2 3956139/6051497
## 3 28728120/63990947
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in luad-0501-raw_1 and luad-0501-raw_2
## Single-end reads will be in luad-0501-raw_M
## 13277195 sequences in the BAM file
## 13277195 sequences exported
## 13277195 reads; of these:
##   13277195 (100.00%) were unpaired; of these:
##     4109338 (30.95%) aligned 0 times
##     6865944 (51.71%) aligned exactly 1 time
##     2301913 (17.34%) aligned &gt;1 times
## 69.05% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2301913/9167857
## 2 1431677/2301913
## 3 7748443/22853923
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in luad-0502-raw_1 and luad-0502-raw_2
## Single-end reads will be in luad-0502-raw_M
## 10825067 sequences in the BAM file
## 10825067 sequences exported
## 10825067 reads; of these:
##   10825067 (100.00%) were unpaired; of these:
##     4568319 (42.20%) aligned 0 times
##     4630590 (42.78%) aligned exactly 1 time
##     1626158 (15.02%) aligned &gt;1 times
## 57.80% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1626158/6256748
## 2 1022295/1626158
## 3 5296338/15548172
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in luad-0503-raw_1 and luad-0503-raw_2
## Single-end reads will be in luad-0503-raw_M
## 10456076 sequences in the BAM file
## 10456076 sequences exported
## 10456076 reads; of these:
##   10456076 (100.00%) were unpaired; of these:
##     2509100 (24.00%) aligned 0 times
##     6003304 (57.41%) aligned exactly 1 time
##     1943672 (18.59%) aligned &gt;1 times
## 76.00% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1943672/7946976
## 2 1516732/1943672
## 3 6475125/18189959
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0504-raw_1 and luad-0504-raw_2
## Single-end reads will be in luad-0504-raw_M
## 12162625 sequences in the BAM file
## 12162625 sequences exported
## 12162625 reads; of these:
##   12162625 (100.00%) were unpaired; of these:
##     2113313 (17.38%) aligned 0 times
##     6778555 (55.73%) aligned exactly 1 time
##     3270757 (26.89%) aligned &gt;1 times
## 82.62% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3270757/10049312
## 2 2247883/3270757
## 3 15293178/33145420
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0505-raw_1 and luad-0505-raw_2
## Single-end reads will be in luad-0505-raw_M
## 14078238 sequences in the BAM file
## 14078238 sequences exported
## 14078238 reads; of these:
##   14078238 (100.00%) were unpaired; of these:
##     3389880 (24.08%) aligned 0 times
##     8276264 (58.79%) aligned exactly 1 time
##     2412094 (17.13%) aligned &gt;1 times
## 75.92% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2412094/10688358
## 2 1545547/2412094
## 3 9360363/26163436
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0506-raw_1 and luad-0506-raw_2
## Single-end reads will be in luad-0506-raw_M
## 10211291 sequences in the BAM file
## 10211291 sequences exported
## 10211291 reads; of these:
##   10211291 (100.00%) were unpaired; of these:
##     1918326 (18.79%) aligned 0 times
##     6161706 (60.34%) aligned exactly 1 time
##     2131259 (20.87%) aligned &gt;1 times
## 81.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2131259/8292965
## 2 1552627/2131259
## 3 8736296/21438939
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0507-raw_1 and luad-0507-raw_2
## Single-end reads will be in luad-0507-raw_M
## 13344064 sequences in the BAM file
## 13344064 sequences exported
## 13344064 reads; of these:
##   13344064 (100.00%) were unpaired; of these:
##     5842905 (43.79%) aligned 0 times
##     4952201 (37.11%) aligned exactly 1 time
##     2548958 (19.10%) aligned &gt;1 times
## 56.21% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2548958/7501159
## 2 1631291/2548958
## 3 7910648/20954261
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0508-raw_1 and luad-0508-raw_2
## Single-end reads will be in luad-0508-raw_M
## 11323093 sequences in the BAM file
## 11323093 sequences exported
## 11323093 reads; of these:
##   11323093 (100.00%) were unpaired; of these:
##     4186937 (36.98%) aligned 0 times
##     4420580 (39.04%) aligned exactly 1 time
##     2715576 (23.98%) aligned &gt;1 times
## 63.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2715576/7136156
## 2 1767838/2715576
## 3 7248081/19874365
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0509-raw_1 and luad-0509-raw_2
## Single-end reads will be in luad-0509-raw_M
## 8846624 sequences in the BAM file
## 8846624 sequences exported
## 8846624 reads; of these:
##   8846624 (100.00%) were unpaired; of these:
##     3236775 (36.59%) aligned 0 times
##     3292419 (37.22%) aligned exactly 1 time
##     2317430 (26.20%) aligned &gt;1 times
## 63.41% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2317430/5609849
## 2 1413839/2317430
## 3 5662774/15481526
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0510-raw_1 and luad-0510-raw_2
## Single-end reads will be in luad-0510-raw_M
## 10050915 sequences in the BAM file
## 10050915 sequences exported
## 10050915 reads; of these:
##   10050915 (100.00%) were unpaired; of these:
##     3259651 (32.43%) aligned 0 times
##     4438708 (44.16%) aligned exactly 1 time
##     2352556 (23.41%) aligned &gt;1 times
## 67.57% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2352556/6791264
## 2 1489386/2352556
## 3 7932405/20047196
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0511-raw_1 and luad-0511-raw_2
## Single-end reads will be in luad-0511-raw_M
## 10317135 sequences in the BAM file
## 10317135 sequences exported
## 10317135 reads; of these:
##   10317135 (100.00%) were unpaired; of these:
##     2487968 (24.11%) aligned 0 times
##     5093432 (49.37%) aligned exactly 1 time
##     2735735 (26.52%) aligned &gt;1 times
## 75.89% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2735735/7829167
## 2 1845587/2735735
## 3 18063925/37563930
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in luad-0512-raw_1 and luad-0512-raw_2
## Single-end reads will be in luad-0512-raw_M
## 13762886 sequences in the BAM file
## 13762886 sequences exported
## 13762886 reads; of these:
##   13762886 (100.00%) were unpaired; of these:
##     3723876 (27.06%) aligned 0 times
##     6889284 (50.06%) aligned exactly 1 time
##     3149726 (22.89%) aligned &gt;1 times
## 72.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3149726/10039010
## 2 2256207/3149726
## 3 15151020/32768700
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0513-raw_1 and luad-0513-raw_2
## Single-end reads will be in luad-0513-raw_M
## 12074370 sequences in the BAM file
## 12074370 sequences exported
## 12074370 reads; of these:
##   12074370 (100.00%) were unpaired; of these:
##     1845748 (15.29%) aligned 0 times
##     6840988 (56.66%) aligned exactly 1 time
##     3387634 (28.06%) aligned &gt;1 times
## 84.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3387634/10228622
## 2 2436220/3387634
## 3 17771202/38793641
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0514-raw_1 and luad-0514-raw_2
## Single-end reads will be in luad-0514-raw_M
## 9959825 sequences in the BAM file
## 9959825 sequences exported
## 9959825 reads; of these:
##   9959825 (100.00%) were unpaired; of these:
##     1859676 (18.67%) aligned 0 times
##     6071367 (60.96%) aligned exactly 1 time
##     2028782 (20.37%) aligned &gt;1 times
## 81.33% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2028782/8100149
## 2 1318632/2028782
## 3 10020848/25928161
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0515-raw_1 and luad-0515-raw_2
## Single-end reads will be in luad-0515-raw_M
## 7641706 sequences in the BAM file
## 7641706 sequences exported
## 7641706 reads; of these:
##   7641706 (100.00%) were unpaired; of these:
##     1507376 (19.73%) aligned 0 times
##     3945657 (51.63%) aligned exactly 1 time
##     2188673 (28.64%) aligned &gt;1 times
## 80.27% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2188673/6134330
## 2 1532433/2188673
## 3 15548239/32527511
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0516-raw_1 and luad-0516-raw_2
## Single-end reads will be in luad-0516-raw_M
## 13096418 sequences in the BAM file
## 13096418 sequences exported
## 13096418 reads; of these:
##   13096418 (100.00%) were unpaired; of these:
##     2932460 (22.39%) aligned 0 times
##     5534513 (42.26%) aligned exactly 1 time
##     4629445 (35.35%) aligned &gt;1 times
## 77.61% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4629445/10163958
## 2 2265902/4629445
## 3 18830020/44907023
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 3.
## Output will be in luad-0517-raw_1 and luad-0517-raw_2
## Single-end reads will be in luad-0517-raw_M
## 13472180 sequences in the BAM file
## 13472180 sequences exported
## 13472180 reads; of these:
##   13472180 (100.00%) were unpaired; of these:
##     4640189 (34.44%) aligned 0 times
##     6569692 (48.76%) aligned exactly 1 time
##     2262299 (16.79%) aligned &gt;1 times
## 65.56% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2262299/8831991
## 2 1648185/2262299
## 3 15618713/31221139
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0518-raw_1 and luad-0518-raw_2
## Single-end reads will be in luad-0518-raw_M
## 10278833 sequences in the BAM file
## 10278833 sequences exported
## 10278833 reads; of these:
##   10278833 (100.00%) were unpaired; of these:
##     1979631 (19.26%) aligned 0 times
##     5693587 (55.39%) aligned exactly 1 time
##     2605615 (25.35%) aligned &gt;1 times
## 80.74% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2605615/8299202
## 2 1813946/2605615
## 3 9988024/24220723
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0519-raw_1 and luad-0519-raw_2
## Single-end reads will be in luad-0519-raw_M
## 10480035 sequences in the BAM file
## 10480035 sequences exported
## 10480035 reads; of these:
##   10480035 (100.00%) were unpaired; of these:
##     2520816 (24.05%) aligned 0 times
##     5173286 (49.36%) aligned exactly 1 time
##     2785933 (26.58%) aligned &gt;1 times
## 75.95% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2785933/7959219
## 2 2090609/2785933
## 3 8394966/21345269
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in luad-0520-raw_1 and luad-0520-raw_2
## Single-end reads will be in luad-0520-raw_M
## 20991244 sequences in the BAM file
## 20991244 sequences exported
## 20991244 reads; of these:
##   20991244 (100.00%) were unpaired; of these:
##     8866318 (42.24%) aligned 0 times
##     9055915 (43.14%) aligned exactly 1 time
##     3069011 (14.62%) aligned &gt;1 times
## 57.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3069011/12124926
## 2 1881628/3069011
## 3 8526594/26840547
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0521-raw_1 and luad-0521-raw_2
## Single-end reads will be in luad-0521-raw_M
## 6903122 sequences in the BAM file
## 6903122 sequences exported
## 6903122 reads; of these:
##   6903122 (100.00%) were unpaired; of these:
##     1891914 (27.41%) aligned 0 times
##     3476842 (50.37%) aligned exactly 1 time
##     1534366 (22.23%) aligned &gt;1 times
## 72.59% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1534366/5011208
## 2 1032747/1534366
## 3 10172084/21804168
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0522-raw_1 and luad-0522-raw_2
## Single-end reads will be in luad-0522-raw_M
## 13677857 sequences in the BAM file
## 13677857 sequences exported
## 13677857 reads; of these:
##   13677857 (100.00%) were unpaired; of these:
##     4995834 (36.52%) aligned 0 times
##     6388691 (46.71%) aligned exactly 1 time
##     2293332 (16.77%) aligned &gt;1 times
## 63.48% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2293332/8682023
## 2 1535218/2293332
## 3 6497077/19373720
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in luad-0523-raw_1 and luad-0523-raw_2
## Single-end reads will be in luad-0523-raw_M
## 12427587 sequences in the BAM file
## 12427587 sequences exported
## 12427587 reads; of these:
##   12427587 (100.00%) were unpaired; of these:
##     4730242 (38.06%) aligned 0 times
##     6246503 (50.26%) aligned exactly 1 time
##     1450842 (11.67%) aligned &gt;1 times
## 61.94% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1450842/7697345
## 2 856033/1450842
## 3 3183547/13338139
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in luad-0524-raw_1 and luad-0524-raw_2
## Single-end reads will be in luad-0524-raw_M
## 12504022 sequences in the BAM file
## 12504022 sequences exported
## 12504022 reads; of these:
##   12504022 (100.00%) were unpaired; of these:
##     5097781 (40.77%) aligned 0 times
##     5656527 (45.24%) aligned exactly 1 time
##     1749714 (13.99%) aligned &gt;1 times
## 59.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1749714/7406241
## 2 1134203/1749714
## 3 10259995/23604979
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0525-raw_1 and luad-0525-raw_2
## Single-end reads will be in luad-0525-raw_M
## 8793009 sequences in the BAM file
## 8793009 sequences exported
## 8793009 reads; of these:
##   8793009 (100.00%) were unpaired; of these:
##     1769481 (20.12%) aligned 0 times
##     4064341 (46.22%) aligned exactly 1 time
##     2959187 (33.65%) aligned &gt;1 times
## 79.88% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2959187/7023528
## 2 1961426/2959187
## 3 22724464/47487387
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in luad-0526-raw_1 and luad-0526-raw_2
## Single-end reads will be in luad-0526-raw_M
## 10021780 sequences in the BAM file
## 10021780 sequences exported
## 10021780 reads; of these:
##   10021780 (100.00%) were unpaired; of these:
##     2297495 (22.93%) aligned 0 times
##     5966710 (59.54%) aligned exactly 1 time
##     1757575 (17.54%) aligned &gt;1 times
## 77.07% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1757575/7724285
## 2 1228386/1757575
## 3 7558439/20390813
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in luad-0527-raw_1 and luad-0527-raw_2
## Single-end reads will be in luad-0527-raw_M
## 12339296 sequences in the BAM file
## 12339296 sequences exported
## 12339296 reads; of these:
##   12339296 (100.00%) were unpaired; of these:
##     4351506 (35.27%) aligned 0 times
##     5982583 (48.48%) aligned exactly 1 time
##     2005207 (16.25%) aligned &gt;1 times
## 64.73% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2005207/7987790
## 2 1297904/2005207
## 3 5647110/17494580
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0528-raw_1 and luad-0528-raw_2
## Single-end reads will be in luad-0528-raw_M
## 12459733 sequences in the BAM file
## 12459733 sequences exported
## 12459733 reads; of these:
##   12459733 (100.00%) were unpaired; of these:
##     2637402 (21.17%) aligned 0 times
##     6834253 (54.85%) aligned exactly 1 time
##     2988078 (23.98%) aligned &gt;1 times
## 78.83% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2988078/9822331
## 2 2189709/2988078
## 3 14722661/31943188
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in luad-0529-raw_1 and luad-0529-raw_2
## Single-end reads will be in luad-0529-raw_M
## 7899274 sequences in the BAM file
## 7899274 sequences exported
## 7899274 reads; of these:
##   7899274 (100.00%) were unpaired; of these:
##     3226405 (40.84%) aligned 0 times
##     3713234 (47.01%) aligned exactly 1 time
##     959635 (12.15%) aligned &gt;1 times
## 59.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 959635/4672869
## 2 542373/959635
## 3 3116307/10448507
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0530-raw_1 and luad-0530-raw_2
## Single-end reads will be in luad-0530-raw_M
## 11673528 sequences in the BAM file
## 11673528 sequences exported
## 11673528 reads; of these:
##   11673528 (100.00%) were unpaired; of these:
##     2084165 (17.85%) aligned 0 times
##     6696005 (57.36%) aligned exactly 1 time
##     2893358 (24.79%) aligned &gt;1 times
## 82.15% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2893358/9589363
## 2 2003420/2893358
## 3 20130184/43597611
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in luad-0531-raw_1 and luad-0531-raw_2
## Single-end reads will be in luad-0531-raw_M
## 7463747 sequences in the BAM file
## 7463747 sequences exported
## 7463747 reads; of these:
##   7463747 (100.00%) were unpaired; of these:
##     1734715 (23.24%) aligned 0 times
##     4146372 (55.55%) aligned exactly 1 time
##     1582660 (21.20%) aligned &gt;1 times
## 76.76% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1582660/5729032
## 2 1125452/1582660
## 3 8952422/20763604
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0532-raw_1 and luad-0532-raw_2
## Single-end reads will be in luad-0532-raw_M
## 10306162 sequences in the BAM file
## 10306162 sequences exported
## 10306162 reads; of these:
##   10306162 (100.00%) were unpaired; of these:
##     3038060 (29.48%) aligned 0 times
##     5437655 (52.76%) aligned exactly 1 time
##     1830447 (17.76%) aligned &gt;1 times
## 70.52% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1830447/7268102
## 2 1282274/1830447
## 3 5215363/15576630
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0533-raw_1 and luad-0533-raw_2
## Single-end reads will be in luad-0533-raw_M
## 31445467 sequences in the BAM file
## 31445467 sequences exported
## 31445467 reads; of these:
##   31445467 (100.00%) were unpaired; of these:
##     8249297 (26.23%) aligned 0 times
##     16870801 (53.65%) aligned exactly 1 time
##     6325369 (20.12%) aligned &gt;1 times
## 73.77% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 6325369/23196170
## 2 4762044/6325369
## 3 28076947/70422760
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 7.
## Output will be in luad-0534-raw_1 and luad-0534-raw_2
## Single-end reads will be in luad-0534-raw_M
## 16416271 sequences in the BAM file
## 16416271 sequences exported
## 16416271 reads; of these:
##   16416271 (100.00%) were unpaired; of these:
##     3054479 (18.61%) aligned 0 times
##     9248069 (56.33%) aligned exactly 1 time
##     4113723 (25.06%) aligned &gt;1 times
## 81.39% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 4113723/13361792
## 2 3001366/4113723
## 3 18030932/43036192
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in luad-0535-raw_1 and luad-0535-raw_2
## Single-end reads will be in luad-0535-raw_M
## 12102046 sequences in the BAM file
## 12102046 sequences exported
## 12102046 reads; of these:
##   12102046 (100.00%) were unpaired; of these:
##     2239497 (18.51%) aligned 0 times
##     6363672 (52.58%) aligned exactly 1 time
##     3498877 (28.91%) aligned &gt;1 times
## 81.49% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 3498877/9862549
## 2 2591010/3498877
## 3 14393167/31810203
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in luad-0536-raw_1 and luad-0536-raw_2
## Single-end reads will be in luad-0536-raw_M
## 6499075 sequences in the BAM file
## 6499075 sequences exported
## 6499075 reads; of these:
##   6499075 (100.00%) were unpaired; of these:
##     1910489 (29.40%) aligned 0 times
##     2670453 (41.09%) aligned exactly 1 time
##     1918133 (29.51%) aligned &gt;1 times
## 70.60% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1918133/4588586
## 2 1487675/1918133
## 3 8590367/19615820
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0537-raw_1 and luad-0537-raw_2
## Single-end reads will be in luad-0537-raw_M
## 7365435 sequences in the BAM file
## 7365435 sequences exported
## 7365435 reads; of these:
##   7365435 (100.00%) were unpaired; of these:
##     2231050 (30.29%) aligned 0 times
##     3824824 (51.93%) aligned exactly 1 time
##     1309561 (17.78%) aligned &gt;1 times
## 69.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1309561/5134385
## 2 851004/1309561
## 3 4400579/12743211
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0538-raw_1 and luad-0538-raw_2
## Single-end reads will be in luad-0538-raw_M
## 14918142 sequences in the BAM file
## 14918142 sequences exported
## 14918142 reads; of these:
##   14918142 (100.00%) were unpaired; of these:
##     4143210 (27.77%) aligned 0 times
##     7897130 (52.94%) aligned exactly 1 time
##     2877802 (19.29%) aligned &gt;1 times
## 72.23% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2877802/10774932
## 2 1980287/2877802
## 3 9596880/26072834
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0539-raw_1 and luad-0539-raw_2
## Single-end reads will be in luad-0539-raw_M
## 15819682 sequences in the BAM file
## 15819682 sequences exported
## 15819682 reads; of these:
##   15819682 (100.00%) were unpaired; of these:
##     4897957 (30.96%) aligned 0 times
##     8555731 (54.08%) aligned exactly 1 time
##     2365994 (14.96%) aligned &gt;1 times
## 69.04% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2365994/10921725
## 2 1600358/2365994
## 3 5726053/20392584
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0540-raw_1 and luad-0540-raw_2
## Single-end reads will be in luad-0540-raw_M
## 14866740 sequences in the BAM file
## 14866740 sequences exported
## 14866740 reads; of these:
##   14866740 (100.00%) were unpaired; of these:
##     4406885 (29.64%) aligned 0 times
##     7462310 (50.19%) aligned exactly 1 time
##     2997545 (20.16%) aligned &gt;1 times
## 70.36% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2997545/10459855
## 2 2058152/2997545
## 3 7060759/22178110
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0541-raw_1 and luad-0541-raw_2
## Single-end reads will be in luad-0541-raw_M
## 10154546 sequences in the BAM file
## 10154546 sequences exported
## 10154546 reads; of these:
##   10154546 (100.00%) were unpaired; of these:
##     3091897 (30.45%) aligned 0 times
##     5500324 (54.17%) aligned exactly 1 time
##     1562325 (15.39%) aligned &gt;1 times
## 69.55% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1562325/7062649
## 2 1093694/1562325
## 3 5626901/16809191
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in luad-0542-raw_1 and luad-0542-raw_2
## Single-end reads will be in luad-0542-raw_M
## 6710801 sequences in the BAM file
## 6710801 sequences exported
## 6710801 reads; of these:
##   6710801 (100.00%) were unpaired; of these:
##     2140583 (31.90%) aligned 0 times
##     2757719 (41.09%) aligned exactly 1 time
##     1812499 (27.01%) aligned &gt;1 times
## 68.10% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1812499/4570218
## 2 1175624/1812499
## 3 10818025/23881467
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0543-raw_1 and luad-0543-raw_2
## Single-end reads will be in luad-0543-raw_M
## 5874956 sequences in the BAM file
## 5874956 sequences exported
## 5874956 reads; of these:
##   5874956 (100.00%) were unpaired; of these:
##     1770807 (30.14%) aligned 0 times
##     2834977 (48.26%) aligned exactly 1 time
##     1269172 (21.60%) aligned &gt;1 times
## 69.86% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1269172/4104149
## 2 829561/1269172
## 3 8790910/18860702
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0544-raw_1 and luad-0544-raw_2
## Single-end reads will be in luad-0544-raw_M
## 11647427 sequences in the BAM file
## 11647427 sequences exported
## 11647427 reads; of these:
##   11647427 (100.00%) were unpaired; of these:
##     2793450 (23.98%) aligned 0 times
##     6484474 (55.67%) aligned exactly 1 time
##     2369503 (20.34%) aligned &gt;1 times
## 76.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2369503/8853977
## 2 1637651/2369503
## 3 8649611/23244500
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0545-raw_1 and luad-0545-raw_2
## Single-end reads will be in luad-0545-raw_M
## 16423115 sequences in the BAM file
## 16423115 sequences exported
## 16423115 reads; of these:
##   16423115 (100.00%) were unpaired; of these:
##     4953307 (30.16%) aligned 0 times
##     9271796 (56.46%) aligned exactly 1 time
##     2198012 (13.38%) aligned &gt;1 times
## 69.84% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2198012/11469808
## 2 1508034/2198012
## 3 4335671/18539215
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0546-raw_1 and luad-0546-raw_2
## Single-end reads will be in luad-0546-raw_M
## 6399385 sequences in the BAM file
## 6399385 sequences exported
## 6399385 reads; of these:
##   6399385 (100.00%) were unpaired; of these:
##     2066554 (32.29%) aligned 0 times
##     3046011 (47.60%) aligned exactly 1 time
##     1286820 (20.11%) aligned &gt;1 times
## 67.71% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1286820/4332831
## 2 857036/1286820
## 3 5008849/12532197
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in luad-0547-raw_1 and luad-0547-raw_2
## Single-end reads will be in luad-0547-raw_M
## 11846044 sequences in the BAM file
## 11846044 sequences exported
## 11846044 reads; of these:
##   11846044 (100.00%) were unpaired; of these:
##     3412836 (28.81%) aligned 0 times
##     6198498 (52.33%) aligned exactly 1 time
##     2234710 (18.86%) aligned &gt;1 times
## 71.19% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2234710/8433208
## 2 1499611/2234710
## 3 10785732/27027818
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0548-raw_1 and luad-0548-raw_2
## Single-end reads will be in luad-0548-raw_M
## 10457844 sequences in the BAM file
## 10457844 sequences exported
## 10457844 reads; of these:
##   10457844 (100.00%) were unpaired; of these:
##     3067028 (29.33%) aligned 0 times
##     5303468 (50.71%) aligned exactly 1 time
##     2087348 (19.96%) aligned &gt;1 times
## 70.67% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2087348/7390816
## 2 1438971/2087348
## 3 7668651/19999827
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0549-raw_1 and luad-0549-raw_2
## Single-end reads will be in luad-0549-raw_M
## 10367527 sequences in the BAM file
## 10367527 sequences exported
## 10367527 reads; of these:
##   10367527 (100.00%) were unpaired; of these:
##     3316997 (31.99%) aligned 0 times
##     5042597 (48.64%) aligned exactly 1 time
##     2007933 (19.37%) aligned &gt;1 times
## 68.01% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2007933/7050530
## 2 1372473/2007933
## 3 6392412/18132212
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in luad-0550-raw_1 and luad-0550-raw_2
## Single-end reads will be in luad-0550-raw_M
## 11142830 sequences in the BAM file
## 11142830 sequences exported
## 11142830 reads; of these:
##   11142830 (100.00%) were unpaired; of these:
##     3341073 (29.98%) aligned 0 times
##     5291354 (47.49%) aligned exactly 1 time
##     2510403 (22.53%) aligned &gt;1 times
## 70.02% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2510403/7801757
## 2 1822112/2510403
## 3 11297799/27014304
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0551-raw_1 and luad-0551-raw_2
## Single-end reads will be in luad-0551-raw_M
## 9958396 sequences in the BAM file
## 9958396 sequences exported
## 9958396 reads; of these:
##   9958396 (100.00%) were unpaired; of these:
##     3070151 (30.83%) aligned 0 times
##     5370806 (53.93%) aligned exactly 1 time
##     1517439 (15.24%) aligned &gt;1 times
## 69.17% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1517439/6888245
## 2 996558/1517439
## 3 5285964/15786350
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in luad-0552-raw_1 and luad-0552-raw_2
## Single-end reads will be in luad-0552-raw_M
## 12373262 sequences in the BAM file
## 12373262 sequences exported
## 12373262 reads; of these:
##   12373262 (100.00%) were unpaired; of these:
##     3777724 (30.53%) aligned 0 times
##     6271740 (50.69%) aligned exactly 1 time
##     2323798 (18.78%) aligned &gt;1 times
## 69.47% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2323798/8595538
## 2 1526359/2323798
## 3 8893129/23920796
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in luad-0553-raw_1 and luad-0553-raw_2
## Single-end reads will be in luad-0553-raw_M
## 11190324 sequences in the BAM file
## 11190324 sequences exported
## 11190324 reads; of these:
##   11190324 (100.00%) were unpaired; of these:
##     3382079 (30.22%) aligned 0 times
##     5789882 (51.74%) aligned exactly 1 time
##     2018363 (18.04%) aligned &gt;1 times
## 69.78% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2018363/7808245
## 2 1469785/2018363
## 3 5915054/17467685
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0554-raw_1 and luad-0554-raw_2
## Single-end reads will be in luad-0554-raw_M
## 11885680 sequences in the BAM file
## 11885680 sequences exported
## 11885680 reads; of these:
##   11885680 (100.00%) were unpaired; of these:
##     3665513 (30.84%) aligned 0 times
##     5286878 (44.48%) aligned exactly 1 time
##     2933289 (24.68%) aligned &gt;1 times
## 69.16% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2933289/8220167
## 2 2116641/2933289
## 3 7936893/21706651
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in luad-0555-raw_1 and luad-0555-raw_2
## Single-end reads will be in luad-0555-raw_M
## 10958770 sequences in the BAM file
## 10958770 sequences exported
## 10958770 reads; of these:
##   10958770 (100.00%) were unpaired; of these:
##     3420674 (31.21%) aligned 0 times
##     5346209 (48.78%) aligned exactly 1 time
##     2191887 (20.00%) aligned &gt;1 times
## 68.79% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2191887/7538096
## 2 1622253/2191887
## 3 4057070/13813066
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in luad-0556-raw_1 and luad-0556-raw_2
## Single-end reads will be in luad-0556-raw_M
## 12595076 sequences in the BAM file
## 12595076 sequences exported
## 12595076 reads; of these:
##   12595076 (100.00%) were unpaired; of these:
##     4353401 (34.56%) aligned 0 times
##     5475124 (43.47%) aligned exactly 1 time
##     2766551 (21.97%) aligned &gt;1 times
## 65.44% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2766551/8241675
## 2 2038432/2766551
## 3 15591457/30810157
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in luad-0557-raw_1 and luad-0557-raw_2
## Single-end reads will be in luad-0557-raw_M
## 8299060 sequences in the BAM file
## 8299060 sequences exported
## 8299060 reads; of these:
##   8299060 (100.00%) were unpaired; of these:
##     2400471 (28.92%) aligned 0 times
##     3467927 (41.79%) aligned exactly 1 time
##     2430662 (29.29%) aligned &gt;1 times
## 71.08% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 2430662/5898589
## 2 1609210/2430662
## 3 15920170/34971280
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in luad-0558-raw_1 and luad-0558-raw_2
## Single-end reads will be in luad-0558-raw_M
## 9711725 sequences in the BAM file
## 9711725 sequences exported
## 9711725 reads; of these:
##   9711725 (100.00%) were unpaired; of these:
##     1808506 (18.62%) aligned 0 times
##     5939060 (61.15%) aligned exactly 1 time
##     1964159 (20.22%) aligned &gt;1 times
## 81.38% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1964159/7903219
## 2 1277803/1964159
## 3 18850890/39905284
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in luad-0559-raw_1 and luad-0559-raw_2
## Single-end reads will be in luad-0559-raw_M
## 4081665 sequences in the BAM file
## 4081665 sequences exported
## 4081665 reads; of these:
##   4081665 (100.00%) were unpaired; of these:
##     1108320 (27.15%) aligned 0 times
##     1465298 (35.90%) aligned exactly 1 time
##     1508047 (36.95%) aligned &gt;1 times
## 72.85% overall alignment rate
## [samopen] SAM header is present: 530 sequences.
## [samopen] SAM header is present: 530 sequences.
## 1 1508047/2973345
## 2 958991/1508047
## 3 14641842/27517327
## [bam_sort_core] merging from 4 files...
</pre></div>

<p></div></div></p>

<h2>Count reads</h2>

<p>There&#39;s a few types of sRNAs. We can get most of them from the hg19 mappings, but for tRF-1, tRF-3, and mature tRNA halves, we need the mappings to mature tRNAs.</p>

<p>So here&#39;s each sRNA, by source and destination.</p>

<ul>
<li>tRF-1: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>immature tRNA 5&#39; half: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>mature tRNA 3&#39; half/tRF3: tRNAs/split from anticodon</li>
<li>mature tRNA 5&#39; half/tRF5: tRNAs/split from anticodon</li>
<li>miRNAs: hg19/miRbase</li>
<li>snoRNAs: hg19/DASHR</li>
</ul>

<div class="chunk" id="count-features-function"><div class="rcode"><div class="source"><pre class="knitr r">countFeaturesFromBam <- function(metadata, features, dir = ".") {
  metadata <- metadata[which(file.exists(paste0(dir, "/", metadata$shortnames, "-mapped.bam"))), ]

  # use a for loop because r will run out of memory
  counts <- mclapply(1:nrow(metadata), function(i) {
    row <- metadata[i, ]
    system(paste("echo Counting sRNAs from", row$shortnames))
    bamfile <- paste0(dir, "/", row$shortnames, "-mapped.bam")
    sample <- readGAlignments(bamfile)
    sample <- convertChrNames(sample, "Ensembl")
    sample_counts <- data.frame(countOverlaps(features, sample, maxgap = 0, type = "within", minoverlap = 10))
    colnames(sample_counts) <- row$barcode
    sample_counts
  }, mc.cores = 20)
  names <- unlist(lapply(counts, colnames))
  counts <- as.data.frame(matrix(unlist(counts), ncol = nrow(metadata)))
  rownames(counts) <- srnas$tx_name
  colnames(counts) <- names
  counts
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-luad"><div class="rcode"><div class="source"><pre class="knitr r">luad_raw_counts <- countFeaturesFromBam(luad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/luad/') 
</pre></div>

<div class="error"><pre class="knitr r">## Error in as.data.frame(matrix(unlist(counts), ncol = nrow(metadata))): error in evaluating the argument 'x' in selecting a method for function 'as.data.frame': Error in matrix(unlist(counts), ncol = nrow(metadata)) : data is too long
</pre></div>

<div class="source"><pre class="knitr r">save(file = 'luad-counts.RData', luad_raw_counts) 
</pre></div>

<div class="error"><pre class="knitr r">## Error in save(file = &quot;luad-counts.RData&quot;, luad_raw_counts): object 'luad_raw_counts' not found
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-brca"><div class="rcode"><div class="source"><pre class="knitr r">#brca_raw_counts <- countFeaturesFromBam(brca_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/brca/')
#save(file = 'brca-counts.RData', brca_raw_counts)
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-prad"><div class="rcode"><div class="source"><pre class="knitr r">#prad_raw_counts <- countFeaturesFromBam(prad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/prad/')
#save(file = 'prad-counts.RData', prad_raw_counts)
</pre></div>

<p></div></div></p>

<h3>Normalized counts</h3>

<ul>
<li>Remove features that don&#39;t exist in at least 5% of the samples</li>
<li>Remove features that have a combined count total of less than 20bp</li>
<li>Normalization involves calculating size factors, </li>
</ul>

<div class="chunk" id="normalize-counts"><div class="rcode"><div class="source"><pre class="knitr r">normalizeCounts <- function(counts, metadata) {
    counts <- counts[apply(counts, 1, function(row) length(which(row > 0)) > length(colnames(counts)) * 0.05), ]
    counts <- counts[apply(counts, 1, function(row) sum(as.numeric(row)) > 20), ]
    col_data <- data.frame(row.names = colnames(counts), condition = metadata[match(colnames(counts), metadata$barcode), ]$sample_type, type = 'single-read')
    dds <- DESeqDataSetFromMatrix(countData = counts, colData = col_data, design = ~ condition)
  # DESeq2 calculates geometric mean - not helpful if there's a count of 0. I don't want to use pseudocounts, so I'll use TMM instead
    sizeFactors(dds) <- calcNormFactors(counts(dds))
    counts(dds, normalized = TRUE)
}
luad_normalized_counts <- normalizeCounts(luad_raw_counts, luad_metadata)
</pre></div>

<div class="error"><pre class="knitr r">## Error in normalizeCounts(luad_raw_counts, luad_metadata): object 'luad_raw_counts' not found
</pre></div>

<div class="source"><pre class="knitr r">#brca_normalized_counts <- normalizeCounts(brca_raw_counts, brca_metadata)
#prad_normalized_counts <- normalizeCounts(prad_raw_counts, prad_metadata)
</pre></div>

<p></div></div></p>

<p>MA plots are log mean of counts (x axis) vs log fold change (y axis). But we don&#39;t need log mean of counts, mean of counts should do nicely.</p>

<div class="chunk" id="normalization-ma"><div class="rcode"><div class="source"><pre class="knitr r">plotMA <- function(raw_counts, normalized_counts, metadata, srnas) {
  srnas$class <- as.character(srnas$class)
  srnas[str_detect(srnas$class, "mi")]$class <- "miRNA"
  srnas[srnas$class %in% c("trf-1", "trf-3", 'trf-5', 'actRF-3', 'actRF-5')]$class <- "tsRNA"
  srnas[srnas$class %in% c('fivehalf', 'threehalf')]$class <- 'tRNA half'
  srnas[srnas$class %in% c("tRNA-5p", "tRNA-3p", "trailer")]$class <- 'tRNA flanking'

  tp_barcodes <- metadata[metadata$sample_type == "TP" & metadata$barcode %in% colnames(normalized_counts), ]$barcode
  nt_barcodes <- metadata[metadata$sample_type == "NT" & metadata$barcode %in% colnames(normalized_counts), ]$barcode
  # just use pseudocounts - this is just diagnostic and not used in actual data.
  raw_tp_counts <- rowMeans(raw_counts[, tp_barcodes]) + 1
  raw_nt_counts <- rowMeans(raw_counts[, nt_barcodes]) + 1
  tp_counts <- rowMeans(normalized_counts[, tp_barcodes]) + 1
  nt_counts <- rowMeans(normalized_counts[, nt_barcodes]) + 1
  df <- rbind(data.frame(log2FC = log2(raw_tp_counts) - log2(raw_nt_counts), log2counts = log2((raw_tp_counts + raw_nt_counts)/2), normalized = "Raw", sRNA = srnas$class[match(names(raw_tp_counts), srnas$tx_name)]),
              data.frame(log2FC = log2(tp_counts) - log2(nt_counts), log2counts = log2((tp_counts + nt_counts)/2), normalized = "Normalized", sRNA = srnas$class[match(names(tp_counts), srnas$tx_name)]))
  ggplot(df) + geom_point(aes(x = log2counts, y = log2FC, color = sRNA), size = 1, alpha = 0.3) + geom_hline(yintercept = 0, color = "red") + facet_wrap(~ normalized)
}
plotMA(luad_raw_counts, luad_normalized_counts, luad_metadata, srnas) + ggtitle("LUAD MA plot")
</pre></div>

<div class="error"><pre class="knitr r">## Error in `[[&lt;-`(`*tmp*`, name, value = &quot;tsRNA&quot;): 1 elements in value to replace 0 elements
</pre></div>

<div class="source"><pre class="knitr r">#plotMA(brca_raw_counts, brca_normalized_counts, brca_metadata, srnas) + ggtitle("BRCA MA plot")
#plotMA(prad_raw_counts, prad_normalized_counts, prad_metadata, srnas) + ggtitle("PRAD MA plot")
</pre></div>

<p></div></div></p>

<h2>Clinical data</h2>

<div class="chunk" id="get-clinical-data"><div class="rcode"><div class="source"><pre class="knitr r">processClinicalDataFile <- function(biotab_file, batch_file) {
  biotab <- read.delim(biotab_file, header = TRUE, sep = "\t", stringsAsFactors = FALSE)
  batches <- read.table(batch_file, header = TRUE, sep = "\t", as.is = c(FALSE, TRUE))
  biotab$bcr_patient_uuid <- tolower(biotab$bcr_patient_uuid) # all data file ids are in lowercase, not sure if matters
  biotab$days_survived <- replace(biotab$death_days_to, biotab$death_days_to == "[Not Applicable]", biotab$last_contact_days_to[which(biotab$death_days_to == "[Not Applicable]")]) # either days from initial prognosis to death or initial prognosis to censor time
  biotab$batch <- batches[match(biotab$bcr_patient_barcode, substr(batches$barcode, 1, 12)), "batch"]
  biotab
}
luad_clinical <- processClinicalDataFile('luad-clinical-biotab-04032015.tab', 'luad-clinical-batches-04032015.tab')
luad_clinical <- with(luad_clinical[match(luad_metadata$participant_id, luad_clinical$bcr_patient_uuid), ], 
                      data.frame(luad_metadata,
                                gender = gender, 
                                race = race,
                                smoker = ifelse(tobacco_smoking_history_indicator == "Lifelong Non-smoker", 0, 1), # NA doesn't return true or false, just NA. this is useful.
                                t_stage = ajcc_tumor_pathologic_pt, 
                                m_stage = ajcc_metastasis_pathologic_pm, 
                                n_stage = ajcc_nodes_pathologic_pn,
                                stage = ajcc_pathologic_tumor_stage,
                                days_survived = as.integer(days_survived), 
                                vital_status = ifelse(vital_status == "Alive", 0, 1),
                                age = age_at_initial_pathologic_diagnosis,
                                batch = batch))
# brca_clinical <- processClinicalDataFile('brca-clinical-biotab-04032015.tab', 'brca-clinical-batches-04032015.tab')
# brca_clinical <- with(brca_clinical[match(brca_metadata$participant_id, brca_clinical$bcr_patient_uuid), ], 
#                       data.frame(brca_metadata,
#                                 ethnicity = ethnicity,
#                                 race = race,
#                                 vital_status = ifelse(vital_status == "Alive", 0, 1),
#                                 days_survived = as.integer(days_survived),
#                                 age = age_at_diagnosis,
#                                 t_stage = ajcc_tumor_pathologic_pt, 
#                                 m_stage = ajcc_metastasis_pathologic_pm, 
#                                 n_stage = ajcc_nodes_pathologic_pn,
#                                 stage = ajcc_pathologic_tumor_stage,
#                                 er_status = er_status_by_ihc,
#                                 pr_status = pr_status_by_ihc,
#                                 her2_status = her2_status_by_ihc,
#                                 batch = batch))
# prad_clinical <- processClinicalDataFile('prad-clinical-biotab-04012015.tab', 'prad-clinical-batches-04012015.tab')
# prad_clinical <- with(prad_clinical[match(prad_metadata$participant_id, prad_clinical$bcr_patient_uuid), ],
#                       data.frame(prad_metadata,
#                                  days_survived = as.integer(days_survived),
#                                  vital_status = ifelse(vital_status == "Alive", 0, 1),
#                                  t_stage = pathologic_T,
#                                  m_stage = pathologic_M,
#                                  n_stage = pathologic_N,
#                                  gleason_score = as.integer(gleason_score),
#                                  age = abs(round(as.integer(birth_days_to)/365)),
#                                  batch = batch)) # maybe change to age_at _initial_pathologic_diagnosis? 
</pre></div>

<p></div></div></p>

<h2>Check for batch effects</h2>

<div class="chunk" id="batch-effects"><div class="rcode"><div class="source"><pre class="knitr r">removeBatchEffects <- function(metadata, counts) {
  metadata <- metadata[match(colnames(counts), metadata$barcode), ]
  mod <- model.matrix(~ sample_type, data = metadata)
  mod <- mod[!is.na(metadata$batch), ]
  counts <- counts[, -which(is.na(metadata$batch))]
  batch <- as.factor(as.character(metadata$batch[!is.na(metadata$batch)]))
  ComBat(dat = counts, batch = batch, mod = mod)
}
luad_adjusted_counts <- removeBatchEffects(luad_clinical, luad_normalized_counts)
</pre></div>

<div class="error"><pre class="knitr r">## Error in match(colnames(counts), metadata$barcode): error in evaluating the argument 'x' in selecting a method for function 'match': Error in colnames(counts) : 
##   error in evaluating the argument 'x' in selecting a method for function 'colnames': Error: object 'luad_normalized_counts' not found
</pre></div>

<div class="source"><pre class="knitr r">#brca_adjusted_counts <- removeBatchEffects(brca_clinical, brca_normalized_counts)
#prad_adjusted_counts <- removeBatchEffects(prad_clinical, prad_normalized_counts) 
</pre></div>

<p></div></div></p>

<div class="chunk" id="pca-function"><div class="rcode"><div class="source"><pre class="knitr r">plotPCA <- function(normalized_counts, adjusted_counts, metadata) {
  pca_before <- prcomp(t(normalized_counts), scale = TRUE)
  pca_after <- prcomp(t(adjusted_counts), scale = TRUE)
  df <- rbind(data.frame(PC1 = pca_before$x[, 1], PC2 = pca_before$x[, 2], adjusted = "Not adjusted"), data.frame(PC1 = pca_after$x[, 1], PC2 = pca_after$x[, 2], adjusted = "Adjusted"))
  df$sample_type <- metadata[match(c(rownames(pca_before$x), rownames(pca_after$x)), metadata$barcode), ]$sample_type
  df$batch <- metadata[match(c(rownames(pca_before$x), rownames(pca_after$x)), metadata$barcode), ]$batch
  ggplot(df) + geom_point(aes(x = PC1, y = PC2, color = batch, shape = batch, size = sample_type), alpha = 0.3) + scale_shape_manual(values = c(1:22, 34:60)[1:nlevels(df$batch)]) + scale_size_manual(values = c(2, 5)) + guides(color = guide_legend(ncol = 2), shape = guide_legend(ncol = 2), size = guide_legend(ncol = 2)) + facet_wrap(~ adjusted)
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="luad-pca"><div class="rcode"><div class="source"><pre class="knitr r">plotPCA(luad_normalized_counts, luad_adjusted_counts, luad_clinical) + ggtitle("PCA for batch effect adjusted LUAD data")
</pre></div>

<div class="error"><pre class="knitr r">## Error in t(normalized_counts): error in evaluating the argument 'x' in selecting a method for function 't': Error: object 'luad_normalized_counts' not found
</pre></div>

<p></div></div></p>

<div class="chunk" id="brca-pca"><div class="rcode"><div class="source"><pre class="knitr r">#plotPCA(brca_normalized_counts, brca_adjusted_counts, brca_clinical) + ggtitle("PCA for batch effect adjusted BRCA data")
</pre></div>

<p></div></div></p>

<div class="chunk" id="prad-pca"><div class="rcode"><div class="source"><pre class="knitr r">#plotPCA(prad_normalized_counts, prad_adjusted_counts, prad_clinical) + ggtitle("PCA for batch effect adjusted PRAD data")
</pre></div>

<p></div></div></p>

<div class="chunk" id="save-session"><div class="rcode"><div class="source"><pre class="knitr r">save(file = 'luad-counts.RData', luad_clinical, luad_adjusted_counts)
save(file = 'brca-counts.RData', brca_clinical, brca_adjusted_counts)
save(file = 'prad-counts.RData', prad_clinical, prad_adjusted_counts)
save(file = 'luad-metadata.RData', luad_clinical)
save(file = 'brca-metadata.RData', brca_clinical)
save(file = 'prad-metadata.RData', prad_clinical)
save.session("process-reads.RSession")
</pre></div>

<p></div></div></p>

</body>

</html>
