<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>+/- </title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>





<style type="text/css">

</style>

<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>

<!-- highlight.js -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/styles/default.css" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/7.3/highlight.min.js"></script>
<script src="https://yandex.st/highlightjs/7.3/languages/r.min.js"></script>

<!-- bootstrap -->
<link href=https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css rel="stylesheet">
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>

<!-- Manific Popup -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/magnific-popup.js/0.8.9/jquery.magnific-popup.min.js"></script>

<script defer="defer">
// Function to generate the dynamic table of contents
jQuery.fn.generate_TOC = function () {
  var base = $(this[0]);

  var selectors = ['h1', 'h2', 'h3', 'h4'];

  var last_ptr = [{}, {}, {}, {}];

  var anchors = {};

  generate_anchor = function (text) {
    var test = text.replace(/\W/g, '_');

    while(test in anchors){
      //if no suffix, add one
      if(test.match(/_\d+$/) === null){
        test = test + "_2";
      }
      //else generate unique id for duplicates by adding one to the suffix
      else {
        test = test.replace(/_(\d+)$/, function(match, number){ var num=+number+1; return("_" + num) });
      }
    }
    anchors[test]=1;
    return(test);
  }

  $(selectors.join(',')).each(function () {
    var heading = $(this);
    var idx = selectors.indexOf(heading.prop('tagName').toLowerCase());
    var itr = 0;

    while (itr <= idx) {
      if (jQuery.isEmptyObject(last_ptr[itr])) {
        last_ptr[itr] = $('<ul>').addClass('nav');
        if (itr === 0) {
          base.append(last_ptr[itr])
        } else {
          if(last_ptr[itr-1].children('li').length === 0){
            last_ptr[itr-1].append(last_ptr[itr]);
          }
          else {
            last_ptr[itr - 1].children('li').last().append(last_ptr[itr]);
          }
        }
      }
      itr++;
    }
    var anchor = generate_anchor(heading.text());
    heading.attr('id', anchor);
    var a = $('<a>')
    .text(heading.text())
    .attr('href', '#' + anchor);

  var li = $('<li>')
    .append(a);

  last_ptr[idx].append(li);
  for (i = idx + 1; i < last_ptr.length; i++) {
    last_ptr[i] = {};
  }
  });
}
/* run scripts when document is ready */
$(function() {
  "use strict";

  var $window = $(window);
  var $body = $(document.body);

  document.title = $('h1').first().text();

  /* size of thumbnails */
  var thumbsize = "col-md-3";

  var show_code =  false ;

  var show_output =  true ;

  var show_figure =  true ;

  /* included languages */
  var languages = [];

  /* Using render_html, so add in code block */
  $('pre.knitr').each(function(){
    $(this).removeClass('knitr');
    if($(this).find('code').length < 1){
      $(this).wrapInner('<code class=' + $(this).attr('class') + '></code>');
    }
  });

  /* style tables */
  $('table').addClass('table table-striped table-bordered table-hover table-condensed');

  /* add toggle panel to rcode blocks */
  $('div.source,div.output,div.message,div.warning,div.error').each(function() {
    var button = $('<h5 class="panel-title">+/- </h5>');

    if($(this).hasClass('source')){
      var code_block = $(this).find('code');
      var lang_type = code_block.attr('class');
      button.text(button.text() + lang_type + ' Code');
      button.addClass('source ' + lang_type);
      languages[lang_type]=0;
      code_block.each(function(i, e) {
        hljs.highlightBlock(e);
      });
      $(this).addClass('panel panel-primary ' + lang_type);
    }
    else if($(this).hasClass('output')){
      button.text(button.text() + 'Output');
      button.addClass('output');
      $(this).addClass('panel panel-success');
    }
    else if($(this).hasClass('message')){
      button.text(button.text() + 'Message');
      button.addClass('message');
      $(this).addClass('panel panel-info');
    }
    else if($(this).hasClass('warning')){
      button.text(button.text() + 'Warning');
      button.addClass('warning');
      $(this).addClass('panel panel-warning');
    }
    else if($(this).hasClass('error')){
      button.text(button.text() + 'Error');
      button.addClass('error');
      $(this).addClass('panel panel-danger');
    }
    else {
      //console.log($(this));
    }
    $(this).prepend($('<div class="panel-heading toggle" />').append(button));
  });

  /* give images a lightbox and thumbnail classes to allow lightbox and thumbnails TODO: make gallery if graphs are grouped */
  $('div.rimage img').each(function(){

    //remove rimage div
    $(this).unwrap();

    var a = $(this).
      wrap('<a href=# class="media-object pull-left mfp-image thumbnail ' + thumbsize + '"></a>').
      parent();

    var sibs = a.prevUntil('div.rimage,div.media');
    var div = $('<div class="media" />');
    if(sibs.length != 0){
      sibs.addClass('media-body');
      //need to reverse order as prevUntil puts objects in the order it found them
      $(sibs.get().reverse()).wrapAll(div).parent().prepend(a);
    }
    else {
      a.wrap(div);
    }
  });

  $('div.chunk').addClass('media');

  $('.rcode > .panel').addClass('media');
  /* Magnific Popup */
  $(".thumbnail").each(function(){
    $(this).magnificPopup({
      disableOn: 768,
      closeOnContentClick: true,

      type: 'image',
      items: {
        src: $(this).find('img').attr('src'),
      }
    });
  });

  /* add bootstrap classes */
  $('body').wrapInner('<div class="container"><div class="row"><div class="contents">');

  var create_language_links = function(){
    var text='';
    var language;
    for(language in languages){
      if(languages.hasOwnProperty(language)){
        text += '<li><a href=# class="toggle-global source ' + language + '" type="source.' + language + '">' + language + '</a></li>\n';
      }
    }
    return text;
  }

  var navbar =
  '<div class="navbar navbar-fixed-bottom navbar-inverse">\
    <div class="container">\
      <div class="navbar-header">\
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
          <span class="icon-bar"></span>\
        </button>\
      </div>\
      <div id="bottom-navbar" class="navbar-collapse collapse navbar-responsive-collapse">\
        <ul class="nav navbar-nav navbar-right">\
          <li class="nav"><p class="navbar-text">Toggle</p></li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Code <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Languages</li>'
              + create_language_links() +
              '<li><a href="#" type="all-source" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li class="dropup">\
            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Output <b class="caret"></b></a>\
            <ul class="dropdown-menu">\
              <li class="dropdown-header">Type</li>\
                <li><a href="#" type="output" class="toggle-global">Output</a></li>\
                <li><a href="#" type="message" class="toggle-global">Message</a></li>\
                <li><a href="#" type="warning" class="toggle-global">Warning</a></li>\
                <li><a href="#" type="error" class="toggle-global">Error</a></li>\
                <li><a href="#" type="all-output" class="toggle-global">All</a></li>\
            </ul>\
          </li>\
          <li><a href="#" class="toggle-figure">Figures</a></li>\
        </ul>\
      </div><!-- /.nav-collapse -->\
    </div><!-- /.container -->\
  </div>';
  /* add navbar */
  $('.container').append(navbar);

  /* onclick toggle next code block */
  $('.toggle').click(function() {
    $(this).next('pre').slideToggle();
    return false;
  });

  // global toggles
  $('.toggle-global').click(function(){
    var type = $(this).attr('type');
    if(type == 'all-source'){
      for(var language in languages){
        $('li a[type="source.' + language + '"]').click();
      }
    }
    if(type == 'all-output'){
      $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error]').click();
    }
    else {
      if($(this).closest('li').hasClass('active')){
        $('div.' + type).children('pre').slideUp();
      }
      else {
        $('div.' + type).children('pre').slideDown();
      }
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });
  // global toggles figure, this is more complicated than just slideToggle because you have to hide the parent div as well
  $('.toggle-figure').click(function(){
    var imgs = $('.thumbnail img');
    if(imgs.is(":visible")){
      imgs.slideUp(400, function(){ $(this).parent().toggle(); });
    }
    else {
      imgs.parent().show();
      imgs.show();
    }
    $(this).closest('li').toggleClass('active');
    return false;
  });

  /* add footer */
  $('body').wrapInner('<div id="wrap" />');
  $('#wrap').append('<div id="push" />');
  var p = $('p:contains("Author:")');
  var last_p = p.filter(':last');
  last_p.addClass('text-muted').attr('id','credit');
  last_p.append('<p>Styled with <a href="https://github.com/jimhester/knitrBootstrap">knitrBootstrap</a></p>');
  last_p = last_p.wrap('<div id="footer"><div class="container">').parent().parent();
  last_p.appendTo("body");

  $('.container > .row').prepend('<div class="col-md-3"><div id="toc" class="well sidebar sidenav affix hidden-print"/></div>');

  $('.contents').addClass('col-md-offset-3');

  /* table of contents */
  $('#toc').generate_TOC();

  if(show_code){
    /* toggle source buttons pressed */
    $('a.toggle-global.source').closest('li').addClass('active');
  }
  else {
    /* hide code blocks */
    $('div.source pre').hide();
  }

  if(show_output){
    /* toggle output buttons pressed */
    $('li a[type=output], li a[type=message], li a[type=warning], li a[type=error], li a[type=all-output]').closest('li').addClass('active');
  }
  else {
    /* hide output blocks */
    $('div.output pre').hide();
  }

  if(show_figure){
    /* toggle figure button pressed */
    $('li a.toggle-figure').closest('li').addClass('active');
  }
  else {
    /* hide figures */
    $('.thumbnail').hide();
  }

  /* remove paragraphs with no content */
  $('p:empty').remove();

  $body.scrollspy({
    target: '.sidebar',
  });


  //TODO refresh on show/hide
  $window.on('load', function () {
    $body.scrollspy('refresh');
  })

});


</script>
<style>
/* Knitr_bootstrap styles */
#wrap .container-fluid {
  padding: 0;
  overflow: hidden;
}
.toggle{
  text-transform: capitalize;
}

.toggle-global{
  text-transform: capitalize;
}

/* Sticky footer styles */
* {
  margin:0;
}
html,
body {
    height: 100%;
    padding:0 !important;
    /* The html and body elements cannot have any padding or margin. */
    /*overflow-x: hidden;*/
}

/* Wrapper for page content to push down footer */
#wrap {
    min-height: 100%;
    height: auto !important;
    height: 100%;
    /* Negative indent footer by it's height */
    margin: 0 auto -120px;
}

/* Set the fixed height of the footer here */
#push,
#footer {
    height: 120px;
}

#footer {
  text-align: center;
}

/* Top level subheader elements.  These are the first nested items underneath a header element. */
.header li {
  font-size: 20px;
}

/* Makes the font smaller for all subheader elements. */
.sub-header li {
    font-size: 12px;
}

button.thumbnails {
  margin-left:0px;
}

/*
 * Side navigation
 *
 * Scrollspy and affixed enhanced navigation to highlight sections and secondary
 * sections of docs content.
 */

/* By default it's not affixed in mobile views, so undo that */
.sidebar.affix {
  position: static;
}

/* First level of nav */
.sidenav {
  margin-top: 30px;
  margin-bottom: 30px;
  padding-top:    10px;
  padding-bottom: 10px;
  border-radius: 5px;
}

/* All levels of nav */
.sidebar .nav > li > a {
  display: block;
  padding: 5px 20px;
}
.sidebar .nav > li > a:hover,
.sidebar .nav > li > a:focus {
  text-decoration: none;
  border-right: 1px solid;
}
.sidebar .nav > .active > a,
.sidebar .nav > .active:hover > a,
.sidebar .nav > .active:focus > a {
  font-weight: bold;
  background-color: transparent;
  border-right: 1px solid;
}

/* Nav: second level (shown on .active) */
.sidebar .nav .nav {
  display: none; /* Hide by default, but at >768px, show it */
  margin-bottom: 8px;
}
.sidebar .nav .nav > li > a {
  padding-top:    3px;
  padding-bottom: 3px;
  padding-left: 30px;
  font-size: 90%;
}

.sidebar .nav .nav .nav > li > a {
  padding-left: 40px;
}
.sidebar .nav .nav .nav .nav > li > a {
  padding-left: 50px;
}

/* Show and affix the side nav when space allows it */
@media screen and (min-width: 992px) {
  .sidebar .nav > .active > ul {
    display: block;
  }
  /* Widen the fixed sidebar */
  .sidebar.affix,
  .sidebar.affix-bottom {
    width: 213px;
  }
  .sidebar.affix-top,
  .sidebar.affix {
    position: fixed; /* Undo the static from mobile first approach */
    top: 30px;
  }
  .sidebar.affix-bottom {
    position: absolute; /* Undo the static from mobile first approach */
  }
  .sidebar.affix-bottom .sidenav,
  .sidebar.affix .sidenav {
    margin-top: 0;
    margin-bottom: 0;
  }
}
@media screen and (min-width: 1200px) {
  /* Widen the fixed sidebar again */
  .sidebar.affix-bottom,
  .sidebar.affix {
    width: 263px;
  }
}

#toc {
  padding: 10px 0px;
  margin:0;
  border:0;
}


.panel pre {
  border-radius: 0;
  border: 0;
  margin: 0;
  padding: 0;
}
.panel pre code {
  border-radius: 0;
}
/* Magnific Popup CSS */
.mfp-bg {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1042;
  overflow: hidden;
  position: fixed;
  background: #0b0b0b;
  opacity: 0.8;
  filter: alpha(opacity=80); }

.mfp-wrap {
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: 1043;
  position: fixed;
  outline: none !important;
  -webkit-backface-visibility: hidden; }

.mfp-container {
  text-align: center;
  position: absolute;
  width: 100%;
  height: 100%;
  left: 0;
  top: 0;
  padding: 0 8px;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box; }

.mfp-container:before {
  content: '';
  display: inline-block;
  height: 100%;
  vertical-align: middle; }

.mfp-align-top .mfp-container:before {
  display: none; }

.mfp-content {
  position: relative;
  display: inline-block;
  vertical-align: middle;
  margin: 0 auto;
  text-align: left;
  z-index: 1045; }

.mfp-inline-holder .mfp-content,
.mfp-ajax-holder .mfp-content {
  width: 100%;
  cursor: auto; }

.mfp-ajax-cur {
  cursor: progress; }

.mfp-zoom-out-cur,
.mfp-zoom-out-cur .mfp-image-holder .mfp-close {
  cursor: -moz-zoom-out;
  cursor: -webkit-zoom-out;
  cursor: zoom-out; }

.mfp-zoom {
  cursor: pointer;
  cursor: -webkit-zoom-in;
  cursor: -moz-zoom-in;
  cursor: zoom-in; }

.mfp-auto-cursor .mfp-content {
  cursor: auto; }

.mfp-close,
.mfp-arrow,
.mfp-preloader,
.mfp-counter {
  -webkit-user-select: none;
  -moz-user-select: none;
  user-select: none; }

.mfp-loading.mfp-figure {
  display: none; }

.mfp-hide {
  display: none !important; }

.mfp-preloader {
  color: #cccccc;
  position: absolute;
  top: 50%;
  width: auto;
  text-align: center;
  margin-top: -0.8em;
  left: 8px;
  right: 8px;
  z-index: 1044; }

.mfp-preloader a {
  color: #cccccc; }

.mfp-preloader a:hover {
  color: white; }

.mfp-s-ready .mfp-preloader {
  display: none; }

.mfp-s-error .mfp-content {
  display: none; }

button.mfp-close,
button.mfp-arrow {
  overflow: visible;
  cursor: pointer;
  background: transparent;
  border: 0;
  -webkit-appearance: none;
  display: block;
  padding: 0;
  z-index: 1046;
  -webkit-box-shadow: none;
  box-shadow: none; }

button::-moz-focus-inner {
  padding: 0;
  border: 0; }

.mfp-close {
  width: 44px;
  height: 44px;
  line-height: 44px;
  position: absolute;
  right: 0;
  top: 0;
  text-decoration: none;
  text-align: center;
  opacity: 0.65;
  padding: 0 0 18px 10px;
  color: white;
  font-style: normal;
  font-size: 28px;
  font-family: Arial, Baskerville, monospace; }
  .mfp-close:hover, .mfp-close:focus {
    opacity: 1; }
  .mfp-close:active {
    top: 1px; }

.mfp-close-btn-in .mfp-close {
  color: #333333; }

.mfp-image-holder .mfp-close,
.mfp-iframe-holder .mfp-close {
  color: white;
  right: -6px;
  text-align: right;
  padding-right: 6px;
  width: 100%; }

.mfp-counter {
  position: absolute;
  top: 0;
  right: 0;
  color: #cccccc;
  font-size: 12px;
  line-height: 18px; }

.mfp-arrow {
  position: absolute;
  opacity: 0.65;
  margin: 0;
  top: 50%;
  margin-top: -55px;
  padding: 0;
  width: 90px;
  height: 110px;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0); }

.mfp-arrow:active {
  margin-top: -54px; }

.mfp-arrow:hover,
.mfp-arrow:focus {
  opacity: 1; }

.mfp-arrow:before, .mfp-arrow:after,
.mfp-arrow .mfp-b,
.mfp-arrow .mfp-a {
  content: '';
  display: block;
  width: 0;
  height: 0;
  position: absolute;
  left: 0;
  top: 0;
  margin-top: 35px;
  margin-left: 35px;
  border: medium inset transparent; }
.mfp-arrow:after,
.mfp-arrow .mfp-a {
  border-top-width: 13px;
  border-bottom-width: 13px;
  top: 8px; }
.mfp-arrow:before,
.mfp-arrow .mfp-b {
  border-top-width: 21px;
  border-bottom-width: 21px; }

.mfp-arrow-left {
  left: 0; }
  .mfp-arrow-left:after,
  .mfp-arrow-left .mfp-a {
    border-right: 17px solid white;
    margin-left: 31px; }
  .mfp-arrow-left:before,
  .mfp-arrow-left .mfp-b {
    margin-left: 25px;
    border-right: 27px solid #3f3f3f; }

.mfp-arrow-right {
  right: 0; }
  .mfp-arrow-right:after,
  .mfp-arrow-right .mfp-a {
    border-left: 17px solid white;
    margin-left: 39px; }
  .mfp-arrow-right:before,
  .mfp-arrow-right .mfp-b {
    border-left: 27px solid #3f3f3f; }

.mfp-iframe-holder {
  padding-top: 40px;
  padding-bottom: 40px; }

.mfp-iframe-holder .mfp-content {
  line-height: 0;
  width: 100%;
  max-width: 900px; }

.mfp-iframe-scaler {
  width: 100%;
  height: 0;
  overflow: hidden;
  padding-top: 56.25%; }

.mfp-iframe-scaler iframe {
  position: absolute;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: black; }

.mfp-iframe-holder .mfp-close {
  top: -40px; }

/* Main image in popup */
img.mfp-img {
  width: auto;
  max-width: 100%;
  height: auto;
  display: block;
  line-height: 0;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
  padding: 40px 0 40px;
  margin: 0 auto; }

/* The shadow behind the image */
.mfp-figure:after {
  content: '';
  position: absolute;
  left: 0;
  top: 40px;
  bottom: 40px;
  display: block;
  right: 0;
  width: auto;
  height: auto;
  z-index: -1;
  box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);
  background: #444444; }

.mfp-figure {
  line-height: 0; }

.mfp-bottom-bar {
  margin-top: -36px;
  position: absolute;
  top: 100%;
  left: 0;
  width: 100%;
  cursor: auto; }

.mfp-title {
  text-align: left;
  line-height: 18px;
  color: #f3f3f3;
  word-wrap: break-word;
  padding-right: 36px; }

.mfp-figure small {
  color: #bdbdbd;
  display: block;
  font-size: 12px;
  line-height: 14px; }

.mfp-image-holder .mfp-content {
  max-width: 100%; }

.mfp-gallery .mfp-image-holder .mfp-figure {
  cursor: pointer; }

@media screen and (max-width: 800px) and (orientation: landscape), screen and (max-height: 300px) {
  /**
   * Remove all paddings around the image on small screen
   */
  .mfp-img-mobile .mfp-image-holder {
    padding-left: 0;
    padding-right: 0; }

  .mfp-img-mobile img.mfp-img {
    padding: 0; }

  /* The shadow behind the image */
  .mfp-img-mobile .mfp-figure:after {
    top: 0;
    bottom: 0; }

  .mfp-img-mobile .mfp-bottom-bar {
    background: rgba(0, 0, 0, 0.6);
    bottom: 0;
    margin: 0;
    top: auto;
    padding: 3px 5px;
    position: fixed;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box; }

  .mfp-img-mobile .mfp-bottom-bar:empty {
    padding: 0; }

  .mfp-img-mobile .mfp-counter {
    right: 5px;
    top: 3px; }

  .mfp-img-mobile .mfp-close {
    top: 0;
    right: 0;
    width: 35px;
    height: 35px;
    line-height: 35px;
    background: rgba(0, 0, 0, 0.6);
    position: fixed;
    text-align: center;
    padding: 0; }

  .mfp-img-mobile .mfp-figure small {
    display: inline;
    margin-left: 5px; } }
@media all and (max-width: 900px) {
  .mfp-arrow {
    -webkit-transform: scale(0.75);
    transform: scale(0.75); }

  .mfp-arrow-left {
    -webkit-transform-origin: 0;
    transform-origin: 0; }

  .mfp-arrow-right {
    -webkit-transform-origin: 100%;
    transform-origin: 100%; }

  .mfp-container {
    padding-left: 6px;
    padding-right: 6px; } }
.mfp-ie7 .mfp-img {
  padding: 0; }
.mfp-ie7 .mfp-bottom-bar {
  width: 600px;
  left: 50%;
  margin-left: -300px;
  margin-top: 5px;
  padding-bottom: 5px; }
.mfp-ie7 .mfp-container {
  padding: 0; }
.mfp-ie7 .mfp-content {
  padding-top: 44px; }
.mfp-ie7 .mfp-close {
  top: 0;
  right: 0;
  padding-top: 0; }

//Magnific overrides
.mfp-image img{
  background: white;
}
.mfp-figure:after {
  background: white;
}

</style>

 

</head>

<body>
<h1>Process data</h1>

<p><em>Last modified <code class="knitr inline">02:24 PM on Apr 29, 2016</code>. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/process-reads/</code>.</em></p>

<div class="chunk" id="libraries"><div class="rcode"><div class="source"><pre class="knitr r">library(stringr)
library(GenomicRanges)
library(GenomicAlignments)
library(BiocParallel)
source('/inside/grotto/blin/programs/convertChrNames.R')
load('/inside/grotto/blin/data/hg19-srnas.RData')
</pre></div>

<p></div></div></p>

<p>This script downloads and remaps reads. </p>

<ul>
<li>v1: Prostate adenocarcinoma (PRAD)</li>
<li>v2: Lung adenocarcinoma (LUAD) and breast invasive carcinoma (BRCA)</li>
<li>v3: Hepatocellular carcinoma (LIHC), Lymphoid neoplasm diffuse large b-cell lymphoma (DLBC), colorectal (COAD and READ), pancreatic (PAAD), testicular germ cell (TGCT), lower grade glioma (LGG)

<ul>
<li>acute myeloid leukemia (AML and LAML), glioblastoma (GBM), osteosarcoma (OS) do not have enough samples for miRNA-Seq data.</li>
</ul></li>
</ul>

<h2>Parse file and clinical metadata</h2>

<div class="chunk" id="parse-metadata-function"><div class="rcode"><div class="source"><pre class="knitr r"># metadata_file: sample metadata downloaded from CGHub in tab delimited form
# cancer_code: short abbreviation in lowercase describing the cancer type, e.g. "prad" or "luad"
parseMetadata <- function(metadata_file, cancer_code) {
  metadata <- read.table(metadata_file, header = TRUE, sep = '\t', as.is = TRUE)
  metadata <- metadata[, c('barcode', 'sample_type', 'filename', 'analysis_id', 'analyte_type', 'uploaded', 'participant_id')]
  # convert character "uploaded" column to date object for easier comparison
  metadata$uploaded <- strptime(gsub("/", "-", metadata$uploaded), "%Y-%m-%d")
  # start by getting matched pairs
  # first, check if there actually are normal tissue samples
  parsed_metadata <- data.frame()
  if (any(metadata$sample_type == "NT")) {
  # all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.
    paired <- subset(metadata, participant_id %in% subset(metadata, sample_type == "NT")$participant_id & participant_id %in% subset(metadata, sample_type == "TP")$participant_id)
    paired$paired <- TRUE # remember which samples are matched tumor/normal
    for (current_id in unique(paired$participant_id)) {
      nt <- subset(paired, participant_id == current_id & sample_type == "NT")
      tp <- subset(paired, participant_id == current_id & sample_type == "TP")
      parsed_metadata <- rbind(parsed_metadata, tp[which(tp$uploaded == max(tp$uploaded)), ], nt[which(nt$uploaded == max(nt$uploaded)), ])
    }
    singles <- metadata[-which(metadata$participant_id %in% parsed_metadata$participant_id), ]
  } else {
    singles <- metadata
  }
  # now for the unpaired samples
  singles$paired <- FALSE
  for (current_id in unique(singles$participant_id)) {
    ps <- subset(singles, participant_id == current_id) # get all samples for this participant
    parsed_metadata <- rbind(parsed_metadata, ps[which(ps$uploaded == max(ps$uploaded)), ])
  }
  # create sample ID numbers and new filenames
  numbering <- 1:nrow(parsed_metadata)
  parsed_metadata$shortids <- numbering
  numbering <- formatC(numbering, flag = "0", digits = 3)
  parsed_metadata$shortnames <- paste0(cancer_code, "-", numbering)
  # reencode upload date as a string - it's encoded as a time object, screwing up data frame operations
  parsed_metadata$uploaded <- as.character(parsed_metadata$uploaded)
  parsed_metadata
}
processClinicalDataFile = function(biotab_file, batch_file) {
  biotab = read.delim(biotab_file, header = TRUE, sep = "\t", stringsAsFactors = FALSE)
  batches = read.table(batch_file, header = TRUE, sep = "\t", as.is = c(FALSE, TRUE))
  biotab$bcr_patient_uuid = tolower(biotab$bcr_patient_uuid) # all data file ids are in lowercase, not sure if matters
  biotab$days_survived = replace(biotab$death_days_to, biotab$death_days_to == "[Not Applicable]", biotab$last_contact_days_to[which(biotab$death_days_to == "[Not Applicable]")]) # either days from initial prognosis to death or initial prognosis to censor time
  biotab$batch = batches[match(biotab$bcr_patient_barcode, substr(batches$barcode, 1, 12)), "batch"]
  biotab
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="parse-metadata"><div class="rcode"><div class="source"><pre class="knitr r">luad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/luad-summary.tab', 'luad')
brca_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/brca-summary.tab', 'brca')
prad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/prad-summary.tab', 'prad')
dlbc_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/dlbc-summary.tab', 'dlbc')
lihc_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lihc-summary.tab', 'lihc')
coad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/coad-summary.tab', 'coad')
paad_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/paad-summary.tab', 'paad')
tgct_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/tgct-summary.tab', 'tgct')
lgg_metadata = parseMetadata('/inside/grotto/blin/trna-markers/process-reads/lgg-summary.tab', 'lgg')
</pre></div>

<p></div></div></p>

<h2>Clinical data</h2>

<div class="chunk" id="get-clinical-data"><div class="rcode"><div class="source"><pre class="knitr r">luad_clinical = processClinicalDataFile('luad-clinical-biotab-04032015.tab', 'luad-clinical-batches-04032015.tab')
luad_clinical = with(luad_clinical[match(luad_metadata$participant_id, luad_clinical$bcr_patient_uuid), ], 
                      data.frame(luad_metadata,
                                 gender = gender, 
                                 race = race,
                                 smoker = ifelse(tobacco_smoking_history_indicator == "Lifelong Non-smoker", 0, 1), # NA doesn't return true or false, just NA. this is useful.
                                 t_stage = ajcc_tumor_pathologic_pt, 
                                 m_stage = ajcc_metastasis_pathologic_pm, 
                                 n_stage = ajcc_nodes_pathologic_pn,
                                 stage = ajcc_pathologic_tumor_stage,
                                 days_survived = as.integer(days_survived), 
                                 vital_status = ifelse(vital_status == "Alive", 0, 1),
                                 age = age_at_initial_pathologic_diagnosis,
                                 batch = batch))
luad_metadata = luad_clinical
# brca_clinical <- processClinicalDataFile('brca-clinical-biotab-04032015.tab', 'brca-clinical-batches-04032015.tab')
# brca_clinical <- with(brca_clinical[match(brca_metadata$participant_id, brca_clinical$bcr_patient_uuid), ], 
#                       data.frame(brca_metadata,
#                                 ethnicity = ethnicity,
#                                 race = race,
#                                 vital_status = ifelse(vital_status == "Alive", 0, 1),
#                                 days_survived = as.integer(days_survived),
#                                 age = age_at_diagnosis,
#                                 t_stage = ajcc_tumor_pathologic_pt, 
#                                 m_stage = ajcc_metastasis_pathologic_pm, 
#                                 n_stage = ajcc_nodes_pathologic_pn,
#                                 stage = ajcc_pathologic_tumor_stage,
#                                 er_status = er_status_by_ihc,
#                                 pr_status = pr_status_by_ihc,
#                                 her2_status = her2_status_by_ihc,
#                                 batch = batch))
# prad_clinical <- processClinicalDataFile('prad-clinical-biotab-04012015.tab', 'prad-clinical-batches-04012015.tab')
# prad_clinical <- with(prad_clinical[match(prad_metadata$participant_id, prad_clinical$bcr_patient_uuid), ],
#                       data.frame(prad_metadata,
#                                  days_survived = as.integer(days_survived),
#                                  vital_status = ifelse(vital_status == "Alive", 0, 1),
#                                  t_stage = pathologic_T,
#                                  m_stage = pathologic_M,
#                                  n_stage = pathologic_N,
#                                  gleason_score = as.integer(gleason_score),
#                                  age = abs(round(as.integer(birth_days_to)/365)),
#                                  batch = batch)) # maybe change to age_at _initial_pathologic_diagnosis? 
</pre></div>

<p></div></div></p>

<div class="chunk" id="download-files"><div class="rcode"><div class="source"><pre class="knitr r">metadatas <- list(luad = luad_metadata, brca = brca_metadata, prad = prad_metadata, dlbc = dlbc_metadata, lihc = lihc_metadata, coad = coad_metadata, paad = paad_metadata, tgct = tgct_metadata, lgg = lgg_metadata)
currentwd <- getwd()
setwd('/inside/grotto/blin/trna-markers/datasets/raw')
for (cancer_code in names(metadatas)) { 
  metadata <- metadatas[[cancer_code]]
  for (analysis_id in metadata$analysis_id) {
    if (file.exists(analysis_id)) next # if exists, assume file was downloaded correctly and is not corrupted
    system(paste0('echo gtdownload -c $CGKEY -d ', analysis_id, cancer_code))
    system(paste0('gtdownload -c $CGKEY -d ', analysis_id))
  }
  file.symlink(paste0(getwd(), '/', metadata$analysis_id, '/', metadata$barcode, '_mirna.bam'), paste0('/inside/grotto/blin/trna-markers/datasets/', cancer_code, '/mirna/', metadata$shortnames, '-raw.bam'))
}
setwd(currentwd)
</pre></div>

<p></div></div></p>

<p>We are missing <code>01810b1a-84e4-43d5-8a1e-42b132a1126f</code>, which is <code>brca-0939-TP.bam</code>. It&#39;s been removed for now.</p>

<h2>Remap reads</h2>

<p>To count both tRFs processed from mature and immature tRNAs, we need to map to both hg19 and mature tRNAs.</p>

<h3>hg19</h3>

<p>Andrew has a few settings that I&#39;ve incorporated too, with his reasoning:</p>

<ul>
<li>--very-sensitive: necessary due to mismatch caused by modified base misreads </li>
<li>--np 5 --ignore-quals: quality scores are irrelevant, and N should be scored as only slightly better than a mismatch. ignoring quals forces the mismatches to have a single score, which is necessary for precise N weighting.</li>
</ul>

<div class="chunk" id="remap"><div class="rcode"><div class="source"><pre class="knitr bash">remap() {
  currentwd=`pwd`
  cd $1
  for prefix in `ls *-raw.bam | cut -f 1,2 -d -`; do
    if [ ! -e $prefix.bam ]
    then
      if [ ! -e $prefix.fastq ]
      then
        bam2fastq --force -o $prefix-raw# `readlink $prefix-raw.bam`
        mv ${prefix}-raw_M $prefix.fastq
        rm ${prefix}-raw_1 ${prefix}-raw_2
      fi
      bowtie2 -x ~/grotto/data/hg19-tRNAgenome -k 100 --very-sensitive --ignore-quals --np 5 -p 32 -U ${prefix}.fastq -S $prefix-temp-1.sam
      samtools view -S -F 4 -h -u $prefix-temp-1.sam | samtools view -F 512 -h  - > $prefix-temp-2.sam # filter out untemp and low quality reads
      bowtie2-best-mapped.py $prefix-temp-2.sam | samtools view -S -u - | samtools sort - $prefix # auto appends .bam
    fi
    rm -f $prefix.fastq
    rm -f $prefix-temp-1.sam
    rm -f $prefix-temp-2.sam
  done
  cd $currentwd
}
remap /inside/grotto/blin/trna-markers/datasets/luad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/brca/mirna
#remap /inside/grotto/blin/trna-markers/datasets/prad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/dlbc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lihc/mirna
#remap /inside/grotto/blin/trna-markers/datasets/coad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/paad/mirna
#remap /inside/grotto/blin/trna-markers/datasets/tgct/mirna
#remap /inside/grotto/blin/trna-markers/datasets/lgg/mirna
</pre></div>

<p></div></div></p>

<h2>Count reads</h2>

<p>There&#39;s a few types of sRNAs. We can get most of them from the hg19 mappings, but for tRF-1, tRF-3, and mature tRNA halves, we need the mappings to mature tRNAs.</p>

<p>So here&#39;s each sRNA, by source and destination.</p>

<ul>
<li>tRF-1: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>immature tRNA 5&#39; half: hg19/Trailers from Andrew&#39;s tRNA half db</li>
<li>mature tRNA 3&#39; half/tRF3: tRNAs/split from anticodon</li>
<li>mature tRNA 5&#39; half/tRF5: tRNAs/split from anticodon</li>
<li>miRNAs: hg19/miRbase</li>
<li>snoRNAs: hg19/DASHR</li>
</ul>

<div class="chunk" id="count-features-function"><div class="rcode"><div class="source"><pre class="knitr r">countFeaturesFromBam <- function(metadata, features, dir = ".") {
  metadata <- metadata[which(file.exists(paste0(dir, "/", metadata$shortnames, ".bam"))), ]

  # use a for loop because r will run out of memory
  counts <- mclapply(1:nrow(metadata), function(i) {
    row <- metadata[i, ]
    system(paste("echo Counting sRNAs from", row$shortnames))
    bamfile <- paste0(dir, "/", row$shortnames, ".bam")
    sample <- readGAlignments(bamfile)
    sample <- convertChrNames(sample, "Ensembl")
    sample_counts <- data.frame(countOverlaps(features, sample, minoverlap = 10)) 
    colnames(sample_counts) <- row$barcode
    sample_counts
  }, mc.cores = 20)
  names <- unlist(lapply(counts, colnames))
  counts <- as.data.frame(matrix(unlist(counts), ncol = nrow(metadata)))
  rownames(counts) <- srnas$tx_name
  colnames(counts) <- names
  counts
}
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-luad"><div class="rcode"><div class="source"><pre class="knitr r">luad_raw_counts <- countFeaturesFromBam(luad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/luad/mirna/') 
save(file = 'luad-raw-counts.RData', luad_raw_counts, luad_metadata) 
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-brca"><div class="rcode"><div class="source"><pre class="knitr r">#brca_raw_counts <- countFeaturesFromBam(brca_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/brca/')
#save(file = 'brca-counts.RData', brca_raw_counts)
</pre></div>

<p></div></div></p>

<div class="chunk" id="count-features-prad"><div class="rcode"><div class="source"><pre class="knitr r">#prad_raw_counts <- countFeaturesFromBam(prad_metadata, srnas, '/inside/grotto/blin/trna-markers/datasets/prad/')
#save(file = 'prad-counts.RData', prad_raw_counts)
</pre></div>

<p></div></div></p>

<div class="chunk" id="save-session"><div class="rcode"><div class="source"><pre class="knitr r">save.session("process-reads.RSession")
</pre></div>

<div class="output"><pre class="knitr r">## Saving search path..
## Saving list of loaded packages..
## Saving all data...
## Done.
</pre></div>

<p></div></div></p>

</body>

</html>
