<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

<title>process-reads</title>

<script type="text/javascript">
window.onload = function() {
  var imgs = document.getElementsByTagName('img'), i, img;
  for (i = 0; i < imgs.length; i++) {
    img = imgs[i];
    // center an image if it is the only element of its parent
    if (img.parentElement.childElementCount === 1)
      img.parentElement.style.textAlign = 'center';
  }
};
</script>

<!-- Styles for R syntax highlighter -->
<style type="text/css">
   pre .operator,
   pre .paren {
     color: rgb(104, 118, 135)
   }

   pre .literal {
     color: #990073
   }

   pre .number {
     color: #099;
   }

   pre .comment {
     color: #998;
     font-style: italic
   }

   pre .keyword {
     color: #900;
     font-weight: bold
   }

   pre .identifier {
     color: rgb(0, 0, 0);
   }

   pre .string {
     color: #d14;
   }
</style>

<!-- R syntax highlighter -->
<script type="text/javascript">
var hljs=new function(){function m(p){return p.replace(/&/gm,"&amp;").replace(/</gm,"&lt;")}function f(r,q,p){return RegExp(q,"m"+(r.cI?"i":"")+(p?"g":""))}function b(r){for(var p=0;p<r.childNodes.length;p++){var q=r.childNodes[p];if(q.nodeName=="CODE"){return q}if(!(q.nodeType==3&&q.nodeValue.match(/\s+/))){break}}}function h(t,s){var p="";for(var r=0;r<t.childNodes.length;r++){if(t.childNodes[r].nodeType==3){var q=t.childNodes[r].nodeValue;if(s){q=q.replace(/\n/g,"")}p+=q}else{if(t.childNodes[r].nodeName=="BR"){p+="\n"}else{p+=h(t.childNodes[r])}}}if(/MSIE [678]/.test(navigator.userAgent)){p=p.replace(/\r/g,"\n")}return p}function a(s){var r=s.className.split(/\s+/);r=r.concat(s.parentNode.className.split(/\s+/));for(var q=0;q<r.length;q++){var p=r[q].replace(/^language-/,"");if(e[p]){return p}}}function c(q){var p=[];(function(s,t){for(var r=0;r<s.childNodes.length;r++){if(s.childNodes[r].nodeType==3){t+=s.childNodes[r].nodeValue.length}else{if(s.childNodes[r].nodeName=="BR"){t+=1}else{if(s.childNodes[r].nodeType==1){p.push({event:"start",offset:t,node:s.childNodes[r]});t=arguments.callee(s.childNodes[r],t);p.push({event:"stop",offset:t,node:s.childNodes[r]})}}}}return t})(q,0);return p}function k(y,w,x){var q=0;var z="";var s=[];function u(){if(y.length&&w.length){if(y[0].offset!=w[0].offset){return(y[0].offset<w[0].offset)?y:w}else{return w[0].event=="start"?y:w}}else{return y.length?y:w}}function t(D){var A="<"+D.nodeName.toLowerCase();for(var B=0;B<D.attributes.length;B++){var C=D.attributes[B];A+=" "+C.nodeName.toLowerCase();if(C.value!==undefined&&C.value!==false&&C.value!==null){A+='="'+m(C.value)+'"'}}return A+">"}while(y.length||w.length){var v=u().splice(0,1)[0];z+=m(x.substr(q,v.offset-q));q=v.offset;if(v.event=="start"){z+=t(v.node);s.push(v.node)}else{if(v.event=="stop"){var p,r=s.length;do{r--;p=s[r];z+=("</"+p.nodeName.toLowerCase()+">")}while(p!=v.node);s.splice(r,1);while(r<s.length){z+=t(s[r]);r++}}}}return z+m(x.substr(q))}function j(){function q(x,y,v){if(x.compiled){return}var u;var s=[];if(x.k){x.lR=f(y,x.l||hljs.IR,true);for(var w in x.k){if(!x.k.hasOwnProperty(w)){continue}if(x.k[w] instanceof Object){u=x.k[w]}else{u=x.k;w="keyword"}for(var r in u){if(!u.hasOwnProperty(r)){continue}x.k[r]=[w,u[r]];s.push(r)}}}if(!v){if(x.bWK){x.b="\\b("+s.join("|")+")\\s"}x.bR=f(y,x.b?x.b:"\\B|\\b");if(!x.e&&!x.eW){x.e="\\B|\\b"}if(x.e){x.eR=f(y,x.e)}}if(x.i){x.iR=f(y,x.i)}if(x.r===undefined){x.r=1}if(!x.c){x.c=[]}x.compiled=true;for(var t=0;t<x.c.length;t++){if(x.c[t]=="self"){x.c[t]=x}q(x.c[t],y,false)}if(x.starts){q(x.starts,y,false)}}for(var p in e){if(!e.hasOwnProperty(p)){continue}q(e[p].dM,e[p],true)}}function d(B,C){if(!j.called){j();j.called=true}function q(r,M){for(var L=0;L<M.c.length;L++){if((M.c[L].bR.exec(r)||[null])[0]==r){return M.c[L]}}}function v(L,r){if(D[L].e&&D[L].eR.test(r)){return 1}if(D[L].eW){var M=v(L-1,r);return M?M+1:0}return 0}function w(r,L){return L.i&&L.iR.test(r)}function K(N,O){var M=[];for(var L=0;L<N.c.length;L++){M.push(N.c[L].b)}var r=D.length-1;do{if(D[r].e){M.push(D[r].e)}r--}while(D[r+1].eW);if(N.i){M.push(N.i)}return f(O,M.join("|"),true)}function p(M,L){var N=D[D.length-1];if(!N.t){N.t=K(N,E)}N.t.lastIndex=L;var r=N.t.exec(M);return r?[M.substr(L,r.index-L),r[0],false]:[M.substr(L),"",true]}function z(N,r){var L=E.cI?r[0].toLowerCase():r[0];var M=N.k[L];if(M&&M instanceof Array){return M}return false}function F(L,P){L=m(L);if(!P.k){return L}var r="";var O=0;P.lR.lastIndex=0;var M=P.lR.exec(L);while(M){r+=L.substr(O,M.index-O);var N=z(P,M);if(N){x+=N[1];r+='<span class="'+N[0]+'">'+M[0]+"</span>"}else{r+=M[0]}O=P.lR.lastIndex;M=P.lR.exec(L)}return r+L.substr(O,L.length-O)}function J(L,M){if(M.sL&&e[M.sL]){var r=d(M.sL,L);x+=r.keyword_count;return r.value}else{return F(L,M)}}function I(M,r){var L=M.cN?'<span class="'+M.cN+'">':"";if(M.rB){y+=L;M.buffer=""}else{if(M.eB){y+=m(r)+L;M.buffer=""}else{y+=L;M.buffer=r}}D.push(M);A+=M.r}function G(N,M,Q){var R=D[D.length-1];if(Q){y+=J(R.buffer+N,R);return false}var P=q(M,R);if(P){y+=J(R.buffer+N,R);I(P,M);return P.rB}var L=v(D.length-1,M);if(L){var O=R.cN?"</span>":"";if(R.rE){y+=J(R.buffer+N,R)+O}else{if(R.eE){y+=J(R.buffer+N,R)+O+m(M)}else{y+=J(R.buffer+N+M,R)+O}}while(L>1){O=D[D.length-2].cN?"</span>":"";y+=O;L--;D.length--}var r=D[D.length-1];D.length--;D[D.length-1].buffer="";if(r.starts){I(r.starts,"")}return R.rE}if(w(M,R)){throw"Illegal"}}var E=e[B];var D=[E.dM];var A=0;var x=0;var y="";try{var s,u=0;E.dM.buffer="";do{s=p(C,u);var t=G(s[0],s[1],s[2]);u+=s[0].length;if(!t){u+=s[1].length}}while(!s[2]);if(D.length>1){throw"Illegal"}return{r:A,keyword_count:x,value:y}}catch(H){if(H=="Illegal"){return{r:0,keyword_count:0,value:m(C)}}else{throw H}}}function g(t){var p={keyword_count:0,r:0,value:m(t)};var r=p;for(var q in e){if(!e.hasOwnProperty(q)){continue}var s=d(q,t);s.language=q;if(s.keyword_count+s.r>r.keyword_count+r.r){r=s}if(s.keyword_count+s.r>p.keyword_count+p.r){r=p;p=s}}if(r.language){p.second_best=r}return p}function i(r,q,p){if(q){r=r.replace(/^((<[^>]+>|\t)+)/gm,function(t,w,v,u){return w.replace(/\t/g,q)})}if(p){r=r.replace(/\n/g,"<br>")}return r}function n(t,w,r){var x=h(t,r);var v=a(t);var y,s;if(v){y=d(v,x)}else{return}var q=c(t);if(q.length){s=document.createElement("pre");s.innerHTML=y.value;y.value=k(q,c(s),x)}y.value=i(y.value,w,r);var u=t.className;if(!u.match("(\\s|^)(language-)?"+v+"(\\s|$)")){u=u?(u+" "+v):v}if(/MSIE [678]/.test(navigator.userAgent)&&t.tagName=="CODE"&&t.parentNode.tagName=="PRE"){s=t.parentNode;var p=document.createElement("div");p.innerHTML="<pre><code>"+y.value+"</code></pre>";t=p.firstChild.firstChild;p.firstChild.cN=s.cN;s.parentNode.replaceChild(p.firstChild,s)}else{t.innerHTML=y.value}t.className=u;t.result={language:v,kw:y.keyword_count,re:y.r};if(y.second_best){t.second_best={language:y.second_best.language,kw:y.second_best.keyword_count,re:y.second_best.r}}}function o(){if(o.called){return}o.called=true;var r=document.getElementsByTagName("pre");for(var p=0;p<r.length;p++){var q=b(r[p]);if(q){n(q,hljs.tabReplace)}}}function l(){if(window.addEventListener){window.addEventListener("DOMContentLoaded",o,false);window.addEventListener("load",o,false)}else{if(window.attachEvent){window.attachEvent("onload",o)}else{window.onload=o}}}var e={};this.LANGUAGES=e;this.highlight=d;this.highlightAuto=g;this.fixMarkup=i;this.highlightBlock=n;this.initHighlighting=o;this.initHighlightingOnLoad=l;this.IR="[a-zA-Z][a-zA-Z0-9_]*";this.UIR="[a-zA-Z_][a-zA-Z0-9_]*";this.NR="\\b\\d+(\\.\\d+)?";this.CNR="\\b(0[xX][a-fA-F0-9]+|(\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)";this.BNR="\\b(0b[01]+)";this.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|\\.|-|-=|/|/=|:|;|<|<<|<<=|<=|=|==|===|>|>=|>>|>>=|>>>|>>>=|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~";this.ER="(?![\\s\\S])";this.BE={b:"\\\\.",r:0};this.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[this.BE],r:0};this.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[this.BE],r:0};this.CLCM={cN:"comment",b:"//",e:"$"};this.CBLCLM={cN:"comment",b:"/\\*",e:"\\*/"};this.HCM={cN:"comment",b:"#",e:"$"};this.NM={cN:"number",b:this.NR,r:0};this.CNM={cN:"number",b:this.CNR,r:0};this.BNM={cN:"number",b:this.BNR,r:0};this.inherit=function(r,s){var p={};for(var q in r){p[q]=r[q]}if(s){for(var q in s){p[q]=s[q]}}return p}}();hljs.LANGUAGES.cpp=function(){var a={keyword:{"false":1,"int":1,"float":1,"while":1,"private":1,"char":1,"catch":1,"export":1,virtual:1,operator:2,sizeof:2,dynamic_cast:2,typedef:2,const_cast:2,"const":1,struct:1,"for":1,static_cast:2,union:1,namespace:1,unsigned:1,"long":1,"throw":1,"volatile":2,"static":1,"protected":1,bool:1,template:1,mutable:1,"if":1,"public":1,friend:2,"do":1,"return":1,"goto":1,auto:1,"void":2,"enum":1,"else":1,"break":1,"new":1,extern:1,using:1,"true":1,"class":1,asm:1,"case":1,typeid:1,"short":1,reinterpret_cast:2,"default":1,"double":1,register:1,explicit:1,signed:1,typename:1,"try":1,"this":1,"switch":1,"continue":1,wchar_t:1,inline:1,"delete":1,alignof:1,char16_t:1,char32_t:1,constexpr:1,decltype:1,noexcept:1,nullptr:1,static_assert:1,thread_local:1,restrict:1,_Bool:1,complex:1},built_in:{std:1,string:1,cin:1,cout:1,cerr:1,clog:1,stringstream:1,istringstream:1,ostringstream:1,auto_ptr:1,deque:1,list:1,queue:1,stack:1,vector:1,map:1,set:1,bitset:1,multiset:1,multimap:1,unordered_set:1,unordered_map:1,unordered_multiset:1,unordered_multimap:1,array:1,shared_ptr:1}};return{dM:{k:a,i:"</",c:[hljs.CLCM,hljs.CBLCLM,hljs.QSM,{cN:"string",b:"'\\\\?.",e:"'",i:"."},{cN:"number",b:"\\b(\\d+(\\.\\d*)?|\\.\\d+)(u|U|l|L|ul|UL|f|F)"},hljs.CNM,{cN:"preprocessor",b:"#",e:"$"},{cN:"stl_container",b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:a,r:10,c:["self"]}]}}}();hljs.LANGUAGES.r={dM:{c:[hljs.HCM,{cN:"number",b:"\\b0[xX][0-9a-fA-F]+[Li]?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+(?:[eE][+\\-]?\\d*)?L\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+\\.(?!\\d)(?:i\\b)?",e:hljs.IMMEDIATE_RE,r:1},{cN:"number",b:"\\b\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"keyword",b:"(?:tryCatch|library|setGeneric|setGroupGeneric)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\.",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\d+(?![\\w.])",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\b(?:function)",e:hljs.IMMEDIATE_RE,r:2},{cN:"keyword",b:"(?:if|in|break|next|repeat|else|for|return|switch|while|try|stop|warning|require|attach|detach|source|setMethod|setClass)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"literal",b:"(?:NA|NA_integer_|NA_real_|NA_character_|NA_complex_)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"literal",b:"(?:NULL|TRUE|FALSE|T|F|Inf|NaN)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"identifier",b:"[a-zA-Z.][a-zA-Z0-9._]*\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"<\\-(?!\\s*\\d)",e:hljs.IMMEDIATE_RE,r:2},{cN:"operator",b:"\\->|<\\-",e:hljs.IMMEDIATE_RE,r:1},{cN:"operator",b:"%%|~",e:hljs.IMMEDIATE_RE},{cN:"operator",b:">=|<=|==|!=|\\|\\||&&|=|\\+|\\-|\\*|/|\\^|>|<|!|&|\\||\\$|:",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"%",e:"%",i:"\\n",r:1},{cN:"identifier",b:"`",e:"`",r:0},{cN:"string",b:'"',e:'"',c:[hljs.BE],r:0},{cN:"string",b:"'",e:"'",c:[hljs.BE],r:0},{cN:"paren",b:"[[({\\])}]",e:hljs.IMMEDIATE_RE,r:0}]}};
hljs.initHighlightingOnLoad();
</script>



<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 13px;
}

body {
  max-width: 800px;
  margin: auto;
  padding: 1em;
  line-height: 20px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 {
   font-size:2.2em;
}

h2 {
   font-size:1.8em;
}

h3 {
   font-size:1.4em;
}

h4 {
   font-size:1.0em;
}

h5 {
   font-size:0.9em;
}

h6 {
   font-size:0.8em;
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre, img {
  max-width: 100%;
}
pre {
  overflow-x: auto;
}
pre code {
   display: block; padding: 0.5em;
}

code {
  font-size: 92%;
  border: 1px solid #ccc;
}

code[class] {
  background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * {
      background: transparent !important;
      color: black !important;
      filter:none !important;
      -ms-filter: none !important;
   }

   body {
      font-size:12pt;
      max-width:100%;
   }

   a, a:visited {
      text-decoration: underline;
   }

   hr {
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote {
      padding-right: 1em;
      page-break-inside: avoid;
   }

   tr, img {
      page-break-inside: avoid;
   }

   img {
      max-width: 100% !important;
   }

   @page :left {
      margin: 15mm 20mm 15mm 10mm;
   }

   @page :right {
      margin: 15mm 10mm 15mm 20mm;
   }

   p, h2, h3 {
      orphans: 3; widows: 3;
   }

   h2, h3 {
      page-break-after: avoid;
   }
}
</style>



</head>

<body>
<h2>process-reads</h2>

<p><em>Last modified 09:18 PM on Dec 16, 2014. This document, R session image, knitr cache, figures, and other associated datasets are located in <code>cruncher:/inside/grotto/blin/trna-markers/process-reads/</code>.</em></p>

<pre><code class="r">library(stringr)
source(&#39;/inside/grotto/blin/programs/convertChrNames.R&#39;)
</code></pre>

<p>tRNA fragments have been implicated in cancer. For instance, tRF-1001 was shown to be required for prostate cancer cell proliferation in <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933153">Lee et al. 2009</a>. We want to generate a list of other such tRNA fragments for further analysis. Rather than looking at the differential expression of tRFs, we will use miRNAs known to be involved in cancer as a proxy for correlating tRFs with cancer. To do that, we&#39;ll generate a list of TCGA PRAD datasets and download them, re-map the reads to a feature list that includes a few key miRNAs, then generate a matrix of counts. Correlations and corresponding statistical analyses can be found in <a href="correlate-features.html">correlate-features</a>.</p>

<h2>Tumor-normal matched data</h2>

<p>The TCGA BAM files are huge and can be upwards to 50 Gb per file. To the best of my knowledge, there is no easy way to generate a list of matched tumor-normal pairs (from the same patient, tumor sequencing data and normal tissue sequencing data) from <a href="https://browser.cghub.ucsc.edu/">CGHub</a>. I downloaded a <a href="file:///Users/blin/Desktop/grotto/trna-markers/process-reads/prad-summary.tab">summary tab-separated file</a>, filtering for PRAD miRNA-Seq assembled with hg19/GRCh37. </p>

<p>The TCGA data has a lot of extraneous samples. Each person either has one TP (primary tumor) dataset, one NT (normal tissue) dataset, or both. Newer data is preferred over older data. I first get matched TP-NT samples by participant ID, then for unmatched samples, I simply filter for the most recent sample (again by participant). Upon visual inspection, all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.</p>

<pre><code class="r">parseMetadata &lt;- function(metadata) {
  metadata &lt;- metadata[, c(&#39;barcode&#39;, &#39;sample_type&#39;, &#39;filename&#39;, &#39;analysis_id&#39;, &#39;uploaded&#39;, &#39;participant_id&#39;)]
  # convert character &quot;uploaded&quot; column to date object for easier comparison
  metadata$uploaded &lt;- strptime(gsub(&quot;/&quot;, &quot;-&quot;, metadata$uploaded), &quot;%Y-%m-%d&quot;)
  # start by getting matched pairs
  # all NT datasets have at least one TP dataset from the same patient (participant). We can filter out datasets by existence of the NT file, then grab the most recent TP and NT datasets for each patient.
  paired &lt;- subset(metadata, participant_id %in% unique(subset(metadata, sample_type==&quot;NT&quot;)$participant_id))
  paired$paired &lt;- TRUE # remember which samples are matched tumor/normal
  parsed_metadata &lt;- data.frame()
  for (current_id in unique(paired$participant_id)) {
    nt &lt;- subset(paired, participant_id == current_id &amp; sample_type == &quot;NT&quot;)
    tp &lt;- subset(paired, participant_id == current_id &amp; sample_type == &quot;TP&quot;)
    parsed_metadata &lt;- rbind(parsed_metadata, tp[which(tp$uploaded == max(tp$uploaded)), ], nt[which(nt$uploaded == max(nt$uploaded)), ])
  }
  # now for the NT and TP singles. 
  singles &lt;- metadata[which(!(metadata$participant_id %in% parsed_metadata$participant_id)), ]
  singles$paired &lt;- FALSE
  for (current_id in unique(singles$participant_id)) {
    ps &lt;- subset(singles, participant_id == current_id) # get all samples for this participant
    parsed_metadata &lt;- rbind(parsed_metadata, ps[which(ps$uploaded == max(ps$uploaded)), ])
  }
  parsed_metadata
}
</code></pre>

<pre><code class="r">prad_metadata &lt;- read.table(&#39;/inside/grotto/blin/trna-markers/process-reads/prad-summary.tab&#39;, header=TRUE, sep=&#39;\t&#39;, as.is=TRUE)
prad_metadata &lt;- parseMetadata(prad_metadata)
numbering &lt;- match(prad_metadata$participant_id, unique(prad_metadata$participant_id)) # assign each patient new ID, internal to this analysis
prad_metadata$shortids &lt;- numbering
numbering &lt;- formatC(numbering, flag = &quot;0&quot;, digits = 3) # convert to 4 digit number (may need to accomodate many more patients/samples in the future
prad_metadata$shortnames &lt;- paste0(&#39;prad-&#39;, numbering, &quot;-&quot;, prad_metadata$sample_type, &quot;.bam&quot;)
write(prad_metadata$analysis_id, file=&quot;prad-samples.txt&quot;)
</code></pre>

<p>A couple notes: first, it&#39;s difficult to work with the raw filenames (e.g. <code>TCGA-HC-8260-11A-01R-2262-13_mirna.bam</code>). We want to use something more along the lines of <code>prad-001-TP.bam</code>, with the number corresponding to a patient. </p>

<p>Second, the analysis IDs are written to a file so <code>bash</code> can run <code>gtdownload</code>. This may take a few hours or days to run, depending on how much TCGA data there is.</p>

<pre><code class="bash">cd /inside/grotto/blin/trna-markers/mirna/prad
for uuid in `cat /inside/grotto/blin/trna-markers/process-reads/prad-samples.txt`; do
  gtdownload -c $CGKEY -d $uuid;
  rm $uuid.gto;
  mv $uuid/*.bam .;
  rm -rf $uuid
done
cd /inside/grotto/blin/trna-markers/process-reads
</code></pre>

<pre><code class="r">currentwd &lt;- getwd() 
setwd(&#39;/inside/grotto/blin/trna-markers/mirna/prad&#39;)
file.rename(prad_metadata$filename, prad_metadata$shortnames)
</code></pre>

<pre><code>##   [1] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [12] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [23] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [34] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [45] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [56] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [67] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [78] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
##  [89] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [100] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [111] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [122] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [133] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [144] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [155] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [166] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [177] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [188] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [199] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [210] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [221] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [232] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [243] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [254] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [265] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [276] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [287] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [298] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [309] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [320] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [331] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [342] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [353] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [364] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [375] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [386] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [397] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [408] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [419] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [430] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [441] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [452] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [463] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [474] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [485] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [496] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [507] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [518] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [529] FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE
## [540] FALSE FALSE FALSE FALSE FALSE FALSE FALSE
</code></pre>

<pre><code class="r">setwd(currentwd)
</code></pre>

<h2>Remap reads</h2>

<p>Originally, the plan was to remap the reads to hg19 and to the extended feature list containing tRNA halves. The main advantage of this would be to enable multimapping reads, since the original mapping maps reads to single locations. We also don&#39;t know what assembler, options, or assumptions were made for the TCGA miRNA mapping. A side benefit of remapping would be to keep the cancer data comparable with our lab internal sequencing data.* </p>

<pre><code class="bash">cd /inside/grotto/blin/trna-markers/mirna/prad
for prefix in `ls *-TP.bam *-NT.bam | cut -f 1 -d .`; do
  if [ ! -e $prefix-mapped.bam ]
  then
    if [ ! -e $prefix.fastq ]
    then
      bam2fastq --force -o $prefix# $prefix.bam
      mv ${prefix}_M $prefix.fastq
      rm ${prefix}_1 ${prefix}_2
    fi
    bowtie2 -x ~/grotto/data/hg19 -k 100 -U ${prefix}.fastq -S $prefix-mapped-1.sam
    samtools view -S -F 4 -h -u $prefix-mapped-1.sam | samtools view -F 512 -h  - &gt; $prefix-mapped-2.sam
    bowtie2-best-mapped.py $prefix-mapped-2.sam | samtools view -S -u - | samtools sort - $prefix-mapped # auto appends .bam
  fi
  rm $prefix.fastq
  rm $prefix-mapped-1.sam
  rm $prefix-mapped-2.sam
done
cd /inside/grotto/blin/trna-markers/process-reads/
</code></pre>

<pre><code>## rm: cannot remove `prad-0001-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0001-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0001-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0001-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0001-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0001-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0002-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0002-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0002-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0002-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0002-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0002-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0003-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0003-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0003-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0003-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0003-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0003-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0004-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0004-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0004-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0004-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0004-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0004-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0005-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0005-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0005-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0005-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0005-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0005-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0006-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0006-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0006-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0006-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0006-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0006-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0007-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0007-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0007-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0007-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0007-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0007-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0008-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0008-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0008-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0008-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0008-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0008-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0009-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0009-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0009-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0009-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0009-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0009-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0010-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0010-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0010-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0010-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0010-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0010-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0011-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0011-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0011-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0011-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0011-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0011-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0012-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0012-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0012-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0012-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0012-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0012-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0013-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0013-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0013-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0013-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0013-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0013-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0014-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0014-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0014-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0014-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0014-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0014-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0015-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0015-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0015-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0015-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0015-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0015-TP-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0016-NT.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0016-NT-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0016-NT-mapped-2.sam&#39;: No such file or directory
## rm: cannot remove `prad-0016-TP.fastq&#39;: No such file or directory
## rm: cannot remove `prad-0016-TP-mapped-1.sam&#39;: No such file or directory
## rm: cannot remove `prad-0016-TP-mapped-2.sam&#39;: No such file or directory
## This looks like paired data from lane 4.
## Output will be in prad-0017-NT_1 and prad-0017-NT_2
## Single-end reads will be in prad-0017-NT_M
## 10244642 sequences in the BAM file
## 10244642 sequences exported
## 10244642 reads; of these:
##   10244642 (100.00%) were unpaired; of these:
##     4039799 (39.43%) aligned 0 times
##     4526647 (44.19%) aligned exactly 1 time
##     1678196 (16.38%) aligned &gt;1 times
## 60.57% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1678196/6204843
## 2 1421967/1678196
## 3 7985568/16245375
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0017-TP_1 and prad-0017-TP_2
## Single-end reads will be in prad-0017-TP_M
## 10476454 sequences in the BAM file
## 10476454 sequences exported
## 10476454 reads; of these:
##   10476454 (100.00%) were unpaired; of these:
##     2088572 (19.94%) aligned 0 times
##     6093407 (58.16%) aligned exactly 1 time
##     2294475 (21.90%) aligned &gt;1 times
## 80.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2294475/8387882
## 2 1774799/2294475
## 3 9763262/23172641
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0018-NT_1 and prad-0018-NT_2
## Single-end reads will be in prad-0018-NT_M
## 20938160 sequences in the BAM file
## 20938160 sequences exported
## 20938160 reads; of these:
##   20938160 (100.00%) were unpaired; of these:
##     7502087 (35.83%) aligned 0 times
##     10892380 (52.02%) aligned exactly 1 time
##     2543693 (12.15%) aligned &gt;1 times
## 64.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2543693/13436073
## 2 2038419/2543693
## 3 11544366/28439215
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0018-TP_1 and prad-0018-TP_2
## Single-end reads will be in prad-0018-TP_M
## 9019060 sequences in the BAM file
## 9019060 sequences exported
## 9019060 reads; of these:
##   9019060 (100.00%) were unpaired; of these:
##     1766000 (19.58%) aligned 0 times
##     5448469 (60.41%) aligned exactly 1 time
##     1804591 (20.01%) aligned &gt;1 times
## 80.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1804591/7253060
## 2 1528844/1804591
## 3 5646073/15292284
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0019-NT_1 and prad-0019-NT_2
## Single-end reads will be in prad-0019-NT_M
## 13831213 sequences in the BAM file
## 13831213 sequences exported
## 13831213 reads; of these:
##   13831213 (100.00%) were unpaired; of these:
##     5271111 (38.11%) aligned 0 times
##     6726508 (48.63%) aligned exactly 1 time
##     1833594 (13.26%) aligned &gt;1 times
## 61.89% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1833594/8560102
## 2 1482902/1833594
## 3 9006992/20093336
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0019-TP_1 and prad-0019-TP_2
## Single-end reads will be in prad-0019-TP_M
## 9997473 sequences in the BAM file
## 9997473 sequences exported
## 9997473 reads; of these:
##   9997473 (100.00%) were unpaired; of these:
##     2061948 (20.62%) aligned 0 times
##     6056474 (60.58%) aligned exactly 1 time
##     1879051 (18.80%) aligned &gt;1 times
## 79.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1879051/7935525
## 2 1515188/1879051
## 3 10109325/24100221
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0020-NT_1 and prad-0020-NT_2
## Single-end reads will be in prad-0020-NT_M
## 19790428 sequences in the BAM file
## 19790428 sequences exported
## 19790428 reads; of these:
##   19790428 (100.00%) were unpaired; of these:
##     7725451 (39.04%) aligned 0 times
##     9856272 (49.80%) aligned exactly 1 time
##     2208705 (11.16%) aligned &gt;1 times
## 60.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2208705/12064977
## 2 1702821/2208705
## 3 8524169/23519307
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0020-TP_1 and prad-0020-TP_2
## Single-end reads will be in prad-0020-TP_M
## 8377054 sequences in the BAM file
## 8377054 sequences exported
## 8377054 reads; of these:
##   8377054 (100.00%) were unpaired; of these:
##     3110812 (37.13%) aligned 0 times
##     4077116 (48.67%) aligned exactly 1 time
##     1189126 (14.20%) aligned &gt;1 times
## 62.87% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1189126/5266242
## 2 891598/1189126
## 3 7113357/15395823
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0021-NT_1 and prad-0021-NT_2
## Single-end reads will be in prad-0021-NT_M
## 26429396 sequences in the BAM file
## 26429396 sequences exported
## 26429396 reads; of these:
##   26429396 (100.00%) were unpaired; of these:
##     7691270 (29.10%) aligned 0 times
##     15521038 (58.73%) aligned exactly 1 time
##     3217088 (12.17%) aligned &gt;1 times
## 70.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3217088/18738126
## 2 2690094/3217088
## 3 18917894/41766081
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0021-TP_1 and prad-0021-TP_2
## Single-end reads will be in prad-0021-TP_M
## 12128148 sequences in the BAM file
## 12128148 sequences exported
## 12128148 reads; of these:
##   12128148 (100.00%) were unpaired; of these:
##     3702389 (30.53%) aligned 0 times
##     6844392 (56.43%) aligned exactly 1 time
##     1581367 (13.04%) aligned &gt;1 times
## 69.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1581367/8425759
## 2 1240115/1581367
## 3 6792458/17934049
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0022-NT_1 and prad-0022-NT_2
## Single-end reads will be in prad-0022-NT_M
## 10349808 sequences in the BAM file
## 10349808 sequences exported
## 10349808 reads; of these:
##   10349808 (100.00%) were unpaired; of these:
##     3783097 (36.55%) aligned 0 times
##     5151504 (49.77%) aligned exactly 1 time
##     1415207 (13.67%) aligned &gt;1 times
## 63.45% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1415207/6566711
## 2 1122696/1415207
## 3 9995884/20003763
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0022-TP_1 and prad-0022-TP_2
## Single-end reads will be in prad-0022-TP_M
## 9613486 sequences in the BAM file
## 9613486 sequences exported
## 9613486 reads; of these:
##   9613486 (100.00%) were unpaired; of these:
##     3106021 (32.31%) aligned 0 times
##     5114729 (53.20%) aligned exactly 1 time
##     1392736 (14.49%) aligned &gt;1 times
## 67.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1392736/6507465
## 2 1104243/1392736
## 3 8090240/17236180
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0023-NT_1 and prad-0023-NT_2
## Single-end reads will be in prad-0023-NT_M
## 6072144 sequences in the BAM file
## 6072144 sequences exported
## 6072144 reads; of these:
##   6072144 (100.00%) were unpaired; of these:
##     2109080 (34.73%) aligned 0 times
##     2843331 (46.83%) aligned exactly 1 time
##     1119733 (18.44%) aligned &gt;1 times
## 65.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1119733/3963064
## 2 961977/1119733
## 3 3953167/9234520
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0023-TP_1 and prad-0023-TP_2
## Single-end reads will be in prad-0023-TP_M
## 10578664 sequences in the BAM file
## 10578664 sequences exported
## 10578664 reads; of these:
##   10578664 (100.00%) were unpaired; of these:
##     2559952 (24.20%) aligned 0 times
##     5013667 (47.39%) aligned exactly 1 time
##     3005045 (28.41%) aligned &gt;1 times
## 75.80% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3005045/8018712
## 2 2216040/3005045
## 3 30630805/65897171
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 3.
## Output will be in prad-0024-NT_1 and prad-0024-NT_2
## Single-end reads will be in prad-0024-NT_M
## 7643781 sequences in the BAM file
## 7643781 sequences exported
## 7643781 reads; of these:
##   7643781 (100.00%) were unpaired; of these:
##     2588577 (33.87%) aligned 0 times
##     3970307 (51.94%) aligned exactly 1 time
##     1084897 (14.19%) aligned &gt;1 times
## 66.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1084897/5055204
## 2 838114/1084897
## 3 5689918/12885586
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0024-TP_1 and prad-0024-TP_2
## Single-end reads will be in prad-0024-TP_M
## 7548304 sequences in the BAM file
## 7548304 sequences exported
## 7548304 reads; of these:
##   7548304 (100.00%) were unpaired; of these:
##     1628855 (21.58%) aligned 0 times
##     4151736 (55.00%) aligned exactly 1 time
##     1767713 (23.42%) aligned &gt;1 times
## 78.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1767713/5919449
## 2 1376537/1767713
## 3 14087183/26392649
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0025-NT_1 and prad-0025-NT_2
## Single-end reads will be in prad-0025-NT_M
## 13533983 sequences in the BAM file
## 13533983 sequences exported
## 13533983 reads; of these:
##   13533983 (100.00%) were unpaired; of these:
##     4439073 (32.80%) aligned 0 times
##     7551737 (55.80%) aligned exactly 1 time
##     1543173 (11.40%) aligned &gt;1 times
## 67.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1543173/9094910
## 2 1243833/1543173
## 3 7378598/18753387
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0025-TP_1 and prad-0025-TP_2
## Single-end reads will be in prad-0025-TP_M
## 19032097 sequences in the BAM file
## 19032097 sequences exported
## 19032097 reads; of these:
##   19032097 (100.00%) were unpaired; of these:
##     6146574 (32.30%) aligned 0 times
##     10534177 (55.35%) aligned exactly 1 time
##     2351346 (12.35%) aligned &gt;1 times
## 67.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2351346/12885523
## 2 1867866/2351346
## 3 9383120/25594420
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0026-NT_1 and prad-0026-NT_2
## Single-end reads will be in prad-0026-NT_M
## 8828738 sequences in the BAM file
## 8828738 sequences exported
## 8828738 reads; of these:
##   8828738 (100.00%) were unpaired; of these:
##     1685586 (19.09%) aligned 0 times
##     5772073 (65.38%) aligned exactly 1 time
##     1371079 (15.53%) aligned &gt;1 times
## 80.91% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1371079/7143152
## 2 1061189/1371079
## 3 7243446/17024320
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0026-TP_1 and prad-0026-TP_2
## Single-end reads will be in prad-0026-TP_M
## 12656853 sequences in the BAM file
## 12656853 sequences exported
## 12656853 reads; of these:
##   12656853 (100.00%) were unpaired; of these:
##     3666779 (28.97%) aligned 0 times
##     7384012 (58.34%) aligned exactly 1 time
##     1606062 (12.69%) aligned &gt;1 times
## 71.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1606062/8990074
## 2 1205251/1606062
## 3 8326011/21614780
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0027-NT_1 and prad-0027-NT_2
## Single-end reads will be in prad-0027-NT_M
## 7533553 sequences in the BAM file
## 7533553 sequences exported
## 7533553 reads; of these:
##   7533553 (100.00%) were unpaired; of these:
##     2603199 (34.55%) aligned 0 times
##     3652167 (48.48%) aligned exactly 1 time
##     1278187 (16.97%) aligned &gt;1 times
## 65.45% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1278187/4930354
## 2 1013171/1278187
## 3 5643511/12802867
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0027-TP_1 and prad-0027-TP_2
## Single-end reads will be in prad-0027-TP_M
## 11793506 sequences in the BAM file
## 11793506 sequences exported
## 11793506 reads; of these:
##   11793506 (100.00%) were unpaired; of these:
##     3951792 (33.51%) aligned 0 times
##     6121278 (51.90%) aligned exactly 1 time
##     1720436 (14.59%) aligned &gt;1 times
## 66.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1720436/7841714
## 2 1337488/1720436
## 3 8070703/19421237
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0028-NT_1 and prad-0028-NT_2
## Single-end reads will be in prad-0028-NT_M
## 36457450 sequences in the BAM file
## 36457450 sequences exported
## 36457450 reads; of these:
##   36457450 (100.00%) were unpaired; of these:
##     8932142 (24.50%) aligned 0 times
##     22679035 (62.21%) aligned exactly 1 time
##     4846273 (13.29%) aligned &gt;1 times
## 75.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 4846273/27525308
## 2 3977250/4846273
## 3 40539883/75790032
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 6.
## Output will be in prad-0028-TP_1 and prad-0028-TP_2
## Single-end reads will be in prad-0028-TP_M
## 15028046 sequences in the BAM file
## 15028046 sequences exported
## 15028046 reads; of these:
##   15028046 (100.00%) were unpaired; of these:
##     5269003 (35.06%) aligned 0 times
##     8000104 (53.23%) aligned exactly 1 time
##     1758939 (11.70%) aligned &gt;1 times
## 64.94% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1758939/9759043
## 2 1426148/1758939
## 3 11781021/24248123
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0029-NT_1 and prad-0029-NT_2
## Single-end reads will be in prad-0029-NT_M
## 7003582 sequences in the BAM file
## 7003582 sequences exported
## 7003582 reads; of these:
##   7003582 (100.00%) were unpaired; of these:
##     2422693 (34.59%) aligned 0 times
##     3579662 (51.11%) aligned exactly 1 time
##     1001227 (14.30%) aligned &gt;1 times
## 65.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1001227/4580889
## 2 787177/1001227
## 3 4708230/11081261
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0029-TP_1 and prad-0029-TP_2
## Single-end reads will be in prad-0029-TP_M
## 6576014 sequences in the BAM file
## 6576014 sequences exported
## 6576014 reads; of these:
##   6576014 (100.00%) were unpaired; of these:
##     2059104 (31.31%) aligned 0 times
##     3040249 (46.23%) aligned exactly 1 time
##     1476661 (22.46%) aligned &gt;1 times
## 68.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1476661/4516910
## 2 1124489/1476661
## 3 10301793/20399754
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0030-NT_1 and prad-0030-NT_2
## Single-end reads will be in prad-0030-NT_M
## 16931911 sequences in the BAM file
## 16931911 sequences exported
## 16931911 reads; of these:
##   16931911 (100.00%) were unpaired; of these:
##     6544889 (38.65%) aligned 0 times
##     8242908 (48.68%) aligned exactly 1 time
##     2144114 (12.66%) aligned &gt;1 times
## 61.35% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2144114/10387022
## 2 1707965/2144114
## 3 11960591/25866404
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0030-TP_1 and prad-0030-TP_2
## Single-end reads will be in prad-0030-TP_M
## 14033439 sequences in the BAM file
## 14033439 sequences exported
## 14033439 reads; of these:
##   14033439 (100.00%) were unpaired; of these:
##     2802357 (19.97%) aligned 0 times
##     8793143 (62.66%) aligned exactly 1 time
##     2437939 (17.37%) aligned &gt;1 times
## 80.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2437939/11231082
## 2 1842741/2437939
## 3 10474396/26917904
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0031-NT_1 and prad-0031-NT_2
## Single-end reads will be in prad-0031-NT_M
## 13347761 sequences in the BAM file
## 13347761 sequences exported
## 13347761 reads; of these:
##   13347761 (100.00%) were unpaired; of these:
##     2575651 (19.30%) aligned 0 times
##     8580169 (64.28%) aligned exactly 1 time
##     2191941 (16.42%) aligned &gt;1 times
## 80.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2191941/10772110
## 2 1794800/2191941
## 3 10481697/25008831
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0031-TP_1 and prad-0031-TP_2
## Single-end reads will be in prad-0031-TP_M
## 8530460 sequences in the BAM file
## 8530460 sequences exported
## 8530460 reads; of these:
##   8530460 (100.00%) were unpaired; of these:
##     1621441 (19.01%) aligned 0 times
##     5332983 (62.52%) aligned exactly 1 time
##     1576036 (18.48%) aligned &gt;1 times
## 80.99% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1576036/6909019
## 2 1271232/1576036
## 3 8351401/18256183
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0032-NT_1 and prad-0032-NT_2
## Single-end reads will be in prad-0032-NT_M
## 32143573 sequences in the BAM file
## 32143573 sequences exported
## 32143573 reads; of these:
##   32143573 (100.00%) were unpaired; of these:
##     9630132 (29.96%) aligned 0 times
##     19146281 (59.56%) aligned exactly 1 time
##     3367160 (10.48%) aligned &gt;1 times
## 70.04% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3367160/22513441
## 2 2789763/3367160
## 3 20123297/46735365
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0032-TP_1 and prad-0032-TP_2
## Single-end reads will be in prad-0032-TP_M
## 6489474 sequences in the BAM file
## 6489474 sequences exported
## 6489474 reads; of these:
##   6489474 (100.00%) were unpaired; of these:
##     2049416 (31.58%) aligned 0 times
##     3008099 (46.35%) aligned exactly 1 time
##     1431959 (22.07%) aligned &gt;1 times
## 68.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1431959/4440058
## 2 1189724/1431959
## 3 4921741/11875079
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0033-NT_1 and prad-0033-NT_2
## Single-end reads will be in prad-0033-NT_M
## 14043345 sequences in the BAM file
## 14043345 sequences exported
## 14043345 reads; of these:
##   14043345 (100.00%) were unpaired; of these:
##     5472990 (38.97%) aligned 0 times
##     6505826 (46.33%) aligned exactly 1 time
##     2064529 (14.70%) aligned &gt;1 times
## 61.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2064529/8570355
## 2 1739782/2064529
## 3 9769459/21283805
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0033-TP_1 and prad-0033-TP_2
## Single-end reads will be in prad-0033-TP_M
## 5597738 sequences in the BAM file
## 5597738 sequences exported
## 5597738 reads; of these:
##   5597738 (100.00%) were unpaired; of these:
##     1233115 (22.03%) aligned 0 times
##     3289211 (58.76%) aligned exactly 1 time
##     1075412 (19.21%) aligned &gt;1 times
## 77.97% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1075412/4364623
## 2 850769/1075412
## 3 4932928/11817080
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0034-NT_1 and prad-0034-NT_2
## Single-end reads will be in prad-0034-NT_M
## 9383823 sequences in the BAM file
## 9383823 sequences exported
## 9383823 reads; of these:
##   9383823 (100.00%) were unpaired; of these:
##     1599070 (17.04%) aligned 0 times
##     6244671 (66.55%) aligned exactly 1 time
##     1540082 (16.41%) aligned &gt;1 times
## 82.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1540082/7784753
## 2 1260058/1540082
## 3 7929633/18469809
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0034-TP_1 and prad-0034-TP_2
## Single-end reads will be in prad-0034-TP_M
## 7473729 sequences in the BAM file
## 7473729 sequences exported
## 7473729 reads; of these:
##   7473729 (100.00%) were unpaired; of these:
##     1450148 (19.40%) aligned 0 times
##     4487890 (60.05%) aligned exactly 1 time
##     1535691 (20.55%) aligned &gt;1 times
## 80.60% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1535691/6023581
## 2 1219007/1535691
## 3 8621801/17955510
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0035-NT_1 and prad-0035-NT_2
## Single-end reads will be in prad-0035-NT_M
## 11047187 sequences in the BAM file
## 11047187 sequences exported
## 11047187 reads; of these:
##   11047187 (100.00%) were unpaired; of these:
##     3521046 (31.87%) aligned 0 times
##     6127291 (55.46%) aligned exactly 1 time
##     1398850 (12.66%) aligned &gt;1 times
## 68.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1398850/7526141
## 2 1108262/1398850
## 3 5409197/14728606
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0035-TP_1 and prad-0035-TP_2
## Single-end reads will be in prad-0035-TP_M
## 11766609 sequences in the BAM file
## 11766609 sequences exported
## 11766609 reads; of these:
##   11766609 (100.00%) were unpaired; of these:
##     2441336 (20.75%) aligned 0 times
##     7158377 (60.84%) aligned exactly 1 time
##     2166896 (18.42%) aligned &gt;1 times
## 79.25% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2166896/9325273
## 2 1690425/2166896
## 3 9988108/24262292
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0036-NT_1 and prad-0036-NT_2
## Single-end reads will be in prad-0036-NT_M
## 12524905 sequences in the BAM file
## 12524905 sequences exported
## 12524905 reads; of these:
##   12524905 (100.00%) were unpaired; of these:
##     4778028 (38.15%) aligned 0 times
##     6167889 (49.24%) aligned exactly 1 time
##     1578988 (12.61%) aligned &gt;1 times
## 61.85% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1578988/7746877
## 2 1239203/1578988
## 3 6465945/16477901
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0036-TP_1 and prad-0036-TP_2
## Single-end reads will be in prad-0036-TP_M
## 9924752 sequences in the BAM file
## 9924752 sequences exported
## 9924752 reads; of these:
##   9924752 (100.00%) were unpaired; of these:
##     3279756 (33.05%) aligned 0 times
##     5059066 (50.97%) aligned exactly 1 time
##     1585930 (15.98%) aligned &gt;1 times
## 66.95% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1585930/6644996
## 2 1181421/1585930
## 3 7232955/17258787
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0037-NT_1 and prad-0037-NT_2
## Single-end reads will be in prad-0037-NT_M
## 18778580 sequences in the BAM file
## 18778580 sequences exported
## 18778580 reads; of these:
##   18778580 (100.00%) were unpaired; of these:
##     6844416 (36.45%) aligned 0 times
##     9651814 (51.40%) aligned exactly 1 time
##     2282350 (12.15%) aligned &gt;1 times
## 63.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2282350/11934164
## 2 1782439/2282350
## 3 11287830/26583917
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0037-TP_1 and prad-0037-TP_2
## Single-end reads will be in prad-0037-TP_M
## 6917907 sequences in the BAM file
## 6917907 sequences exported
## 6917907 reads; of these:
##   6917907 (100.00%) were unpaired; of these:
##     2332009 (33.71%) aligned 0 times
##     3417051 (49.39%) aligned exactly 1 time
##     1168847 (16.90%) aligned &gt;1 times
## 66.29% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1168847/4585898
## 2 883492/1168847
## 3 6005285/13257963
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0038-NT_1 and prad-0038-NT_2
## Single-end reads will be in prad-0038-NT_M
## 9191492 sequences in the BAM file
## 9191492 sequences exported
## 9191492 reads; of these:
##   9191492 (100.00%) were unpaired; of these:
##     2973040 (32.35%) aligned 0 times
##     4999377 (54.39%) aligned exactly 1 time
##     1219075 (13.26%) aligned &gt;1 times
## 67.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1219075/6218452
## 2 980566/1219075
## 3 4107141/11583074
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0038-TP_1 and prad-0038-TP_2
## Single-end reads will be in prad-0038-TP_M
## 9250438 sequences in the BAM file
## 9250438 sequences exported
## 9250438 reads; of these:
##   9250438 (100.00%) were unpaired; of these:
##     1896081 (20.50%) aligned 0 times
##     5789392 (62.59%) aligned exactly 1 time
##     1564965 (16.92%) aligned &gt;1 times
## 79.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1564965/7354357
## 2 1240804/1564965
## 3 6139326/16120335
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0039-NT_1 and prad-0039-NT_2
## Single-end reads will be in prad-0039-NT_M
## 36922825 sequences in the BAM file
## 36922825 sequences exported
## 36922825 reads; of these:
##   36922825 (100.00%) were unpaired; of these:
##     9836075 (26.64%) aligned 0 times
##     20948012 (56.73%) aligned exactly 1 time
##     6138738 (16.63%) aligned &gt;1 times
## 73.36% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 6138738/27086750
## 2 5024498/6138738
## 3 44723686/82879812
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 7.
## Output will be in prad-0039-TP_1 and prad-0039-TP_2
## Single-end reads will be in prad-0039-TP_M
## 17419390 sequences in the BAM file
## 17419390 sequences exported
## 17419390 reads; of these:
##   17419390 (100.00%) were unpaired; of these:
##     5143631 (29.53%) aligned 0 times
##     10196956 (58.54%) aligned exactly 1 time
##     2078803 (11.93%) aligned &gt;1 times
## 70.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2078803/12275759
## 2 1691239/2078803
## 3 12557643/28025308
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0040-NT_1 and prad-0040-NT_2
## Single-end reads will be in prad-0040-NT_M
## 9245318 sequences in the BAM file
## 9245318 sequences exported
## 9245318 reads; of these:
##   9245318 (100.00%) were unpaired; of these:
##     2792425 (30.20%) aligned 0 times
##     5273967 (57.04%) aligned exactly 1 time
##     1178926 (12.75%) aligned &gt;1 times
## 69.80% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1178926/6452893
## 2 947666/1178926
## 3 5019041/13250752
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0040-TP_1 and prad-0040-TP_2
## Single-end reads will be in prad-0040-TP_M
## 14035495 sequences in the BAM file
## 14035495 sequences exported
## 14035495 reads; of these:
##   14035495 (100.00%) were unpaired; of these:
##     4368964 (31.13%) aligned 0 times
##     7444242 (53.04%) aligned exactly 1 time
##     2222289 (15.83%) aligned &gt;1 times
## 68.87% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2222289/9666531
## 2 1743371/2222289
## 3 9528783/22718882
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0041-NT_1 and prad-0041-NT_2
## Single-end reads will be in prad-0041-NT_M
## 16392893 sequences in the BAM file
## 16392893 sequences exported
## 16392893 reads; of these:
##   16392893 (100.00%) were unpaired; of these:
##     2991273 (18.25%) aligned 0 times
##     10744294 (65.54%) aligned exactly 1 time
##     2657326 (16.21%) aligned &gt;1 times
## 81.75% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2657326/13401620
## 2 2095620/2657326
## 3 13148231/31516708
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0041-TP_1 and prad-0041-TP_2
## Single-end reads will be in prad-0041-TP_M
## 13316019 sequences in the BAM file
## 13316019 sequences exported
## 13316019 reads; of these:
##   13316019 (100.00%) were unpaired; of these:
##     3503406 (26.31%) aligned 0 times
##     7157884 (53.75%) aligned exactly 1 time
##     2654729 (19.94%) aligned &gt;1 times
## 73.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2654729/9812613
## 2 1885713/2654729
## 3 13486311/31876814
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0042-NT_1 and prad-0042-NT_2
## Single-end reads will be in prad-0042-NT_M
## 18995708 sequences in the BAM file
## 18995708 sequences exported
## 18995708 reads; of these:
##   18995708 (100.00%) were unpaired; of these:
##     7503754 (39.50%) aligned 0 times
##     8673863 (45.66%) aligned exactly 1 time
##     2818091 (14.84%) aligned &gt;1 times
## 60.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2818091/11491954
## 2 2402858/2818091
## 3 9893994/24862969
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0042-TP_1 and prad-0042-TP_2
## Single-end reads will be in prad-0042-TP_M
## 8777582 sequences in the BAM file
## 8777582 sequences exported
## 8777582 reads; of these:
##   8777582 (100.00%) were unpaired; of these:
##     2082185 (23.72%) aligned 0 times
##     3902022 (44.45%) aligned exactly 1 time
##     2793375 (31.82%) aligned &gt;1 times
## 76.28% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2793375/6695397
## 2 1998752/2793375
## 3 30713323/61835372
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 4.
## Output will be in prad-0043-NT_1 and prad-0043-NT_2
## Single-end reads will be in prad-0043-NT_M
## 26999561 sequences in the BAM file
## 26999561 sequences exported
## 26999561 reads; of these:
##   26999561 (100.00%) were unpaired; of these:
##     9981325 (36.97%) aligned 0 times
##     14427634 (53.44%) aligned exactly 1 time
##     2590602 (9.59%) aligned &gt;1 times
## 63.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2590602/17018236
## 2 2128666/2590602
## 3 15273140/35573197
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0043-TP_1 and prad-0043-TP_2
## Single-end reads will be in prad-0043-TP_M
## 10714136 sequences in the BAM file
## 10714136 sequences exported
## 10714136 reads; of these:
##   10714136 (100.00%) were unpaired; of these:
##     2075556 (19.37%) aligned 0 times
##     6773940 (63.22%) aligned exactly 1 time
##     1864640 (17.40%) aligned &gt;1 times
## 80.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1864640/8638580
## 2 1508065/1864640
## 3 6293303/17348341
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0044-NT_1 and prad-0044-NT_2
## Single-end reads will be in prad-0044-NT_M
## 14534479 sequences in the BAM file
## 14534479 sequences exported
## 14534479 reads; of these:
##   14534479 (100.00%) were unpaired; of these:
##     5592539 (38.48%) aligned 0 times
##     7067293 (48.62%) aligned exactly 1 time
##     1874647 (12.90%) aligned &gt;1 times
## 61.52% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1874647/8941940
## 2 1452872/1874647
## 3 7022857/18277878
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0044-TP_1 and prad-0044-TP_2
## Single-end reads will be in prad-0044-TP_M
## 9215268 sequences in the BAM file
## 9215268 sequences exported
## 9215268 reads; of these:
##   9215268 (100.00%) were unpaired; of these:
##     3202845 (34.76%) aligned 0 times
##     4592672 (49.84%) aligned exactly 1 time
##     1419751 (15.41%) aligned &gt;1 times
## 65.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1419751/6012423
## 2 1054305/1419751
## 3 6346639/15522466
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0045-NT_1 and prad-0045-NT_2
## Single-end reads will be in prad-0045-NT_M
## 10324452 sequences in the BAM file
## 10324452 sequences exported
## 10324452 reads; of these:
##   10324452 (100.00%) were unpaired; of these:
##     3159307 (30.60%) aligned 0 times
##     5882254 (56.97%) aligned exactly 1 time
##     1282891 (12.43%) aligned &gt;1 times
## 69.40% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1282891/7165145
## 2 1028507/1282891
## 3 4346015/12914377
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0045-TP_1 and prad-0045-TP_2
## Single-end reads will be in prad-0045-TP_M
## 3313220 sequences in the BAM file
## 3313220 sequences exported
## 3313220 reads; of these:
##   3313220 (100.00%) were unpaired; of these:
##     1083505 (32.70%) aligned 0 times
##     1391992 (42.01%) aligned exactly 1 time
##     837723 (25.28%) aligned &gt;1 times
## 67.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 837723/2229715
## 2 640168/837723
## 3 6591562/12272694
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0046-NT_1 and prad-0046-NT_2
## Single-end reads will be in prad-0046-NT_M
## 12440313 sequences in the BAM file
## 12440313 sequences exported
## 12440313 reads; of these:
##   12440313 (100.00%) were unpaired; of these:
##     2268961 (18.24%) aligned 0 times
##     7914774 (63.62%) aligned exactly 1 time
##     2256578 (18.14%) aligned &gt;1 times
## 81.76% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2256578/10171352
## 2 1834279/2256578
## 3 10436970/24378943
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0046-TP_1 and prad-0046-TP_2
## Single-end reads will be in prad-0046-TP_M
## 8020489 sequences in the BAM file
## 8020489 sequences exported
## 8020489 reads; of these:
##   8020489 (100.00%) were unpaired; of these:
##     2279815 (28.42%) aligned 0 times
##     4357450 (54.33%) aligned exactly 1 time
##     1383224 (17.25%) aligned &gt;1 times
## 71.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1383224/5740674
## 2 979351/1383224
## 3 5707422/14341977
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0047-NT_1 and prad-0047-NT_2
## Single-end reads will be in prad-0047-NT_M
## 11686869 sequences in the BAM file
## 11686869 sequences exported
## 11686869 reads; of these:
##   11686869 (100.00%) were unpaired; of these:
##     3843053 (32.88%) aligned 0 times
##     6388939 (54.67%) aligned exactly 1 time
##     1454877 (12.45%) aligned &gt;1 times
## 67.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1454877/7843816
## 2 1151874/1454877
## 3 6044005/15807809
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0047-TP_1 and prad-0047-TP_2
## Single-end reads will be in prad-0047-TP_M
## 5063729 sequences in the BAM file
## 5063729 sequences exported
## 5063729 reads; of these:
##   5063729 (100.00%) were unpaired; of these:
##     1026672 (20.28%) aligned 0 times
##     2959876 (58.45%) aligned exactly 1 time
##     1077181 (21.27%) aligned &gt;1 times
## 79.72% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1077181/4037057
## 2 852134/1077181
## 3 6275308/12794872
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0048-NT_1 and prad-0048-NT_2
## Single-end reads will be in prad-0048-NT_M
## 14753074 sequences in the BAM file
## 14753074 sequences exported
## 14753074 reads; of these:
##   14753074 (100.00%) were unpaired; of these:
##     4470891 (30.30%) aligned 0 times
##     8506318 (57.66%) aligned exactly 1 time
##     1775865 (12.04%) aligned &gt;1 times
## 69.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1775865/10282183
## 2 1486153/1775865
## 3 13000086/25928604
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0048-TP_1 and prad-0048-TP_2
## Single-end reads will be in prad-0048-TP_M
## 17293523 sequences in the BAM file
## 17293523 sequences exported
## 17293523 reads; of these:
##   17293523 (100.00%) were unpaired; of these:
##     5388107 (31.16%) aligned 0 times
##     9562145 (55.29%) aligned exactly 1 time
##     2343271 (13.55%) aligned &gt;1 times
## 68.84% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2343271/11905416
## 2 1839215/2343271
## 3 14269721/30629784
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0049-NT_1 and prad-0049-NT_2
## Single-end reads will be in prad-0049-NT_M
## 15879996 sequences in the BAM file
## 15879996 sequences exported
## 15879996 reads; of these:
##   15879996 (100.00%) were unpaired; of these:
##     4858070 (30.59%) aligned 0 times
##     9082569 (57.20%) aligned exactly 1 time
##     1939357 (12.21%) aligned &gt;1 times
## 69.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1939357/11021926
## 2 1590661/1939357
## 3 12858931/26453143
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0049-TP_1 and prad-0049-TP_2
## Single-end reads will be in prad-0049-TP_M
## 8432471 sequences in the BAM file
## 8432471 sequences exported
## 8432471 reads; of these:
##   8432471 (100.00%) were unpaired; of these:
##     2803523 (33.25%) aligned 0 times
##     4435013 (52.59%) aligned exactly 1 time
##     1193935 (14.16%) aligned &gt;1 times
## 66.75% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1193935/5628948
## 2 958465/1193935
## 3 6575369/14347868
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0050-NT_1 and prad-0050-NT_2
## Single-end reads will be in prad-0050-NT_M
## 14490323 sequences in the BAM file
## 14490323 sequences exported
## 14490323 reads; of these:
##   14490323 (100.00%) were unpaired; of these:
##     4605327 (31.78%) aligned 0 times
##     8319756 (57.42%) aligned exactly 1 time
##     1565240 (10.80%) aligned &gt;1 times
## 68.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1565240/9884996
## 2 1259685/1565240
## 3 6829514/19121767
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0050-TP_1 and prad-0050-TP_2
## Single-end reads will be in prad-0050-TP_M
## 13206415 sequences in the BAM file
## 13206415 sequences exported
## 13206415 reads; of these:
##   13206415 (100.00%) were unpaired; of these:
##     4257302 (32.24%) aligned 0 times
##     7159668 (54.21%) aligned exactly 1 time
##     1789445 (13.55%) aligned &gt;1 times
## 67.76% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1789445/8949113
## 2 1445086/1789445
## 3 8530867/21082919
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0051-NT_1 and prad-0051-NT_2
## Single-end reads will be in prad-0051-NT_M
## 30065068 sequences in the BAM file
## 30065068 sequences exported
## 30065068 reads; of these:
##   30065068 (100.00%) were unpaired; of these:
##     4962922 (16.51%) aligned 0 times
##     21740569 (72.31%) aligned exactly 1 time
##     3361577 (11.18%) aligned &gt;1 times
## 83.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3361577/25102146
## 2 2640939/3361577
## 3 8706965/37679948
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 4.
## Output will be in prad-0051-TP_1 and prad-0051-TP_2
## Single-end reads will be in prad-0051-TP_M
## 11078730 sequences in the BAM file
## 11078730 sequences exported
## 11078730 reads; of these:
##   11078730 (100.00%) were unpaired; of these:
##     2722742 (24.58%) aligned 0 times
##     6755583 (60.98%) aligned exactly 1 time
##     1600405 (14.45%) aligned &gt;1 times
## 75.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1600405/8355988
## 2 1255033/1600405
## 3 3746659/13945702
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0052-NT_1 and prad-0052-NT_2
## Single-end reads will be in prad-0052-NT_M
## 18912977 sequences in the BAM file
## 18912977 sequences exported
## 18912977 reads; of these:
##   18912977 (100.00%) were unpaired; of these:
##     2934128 (15.51%) aligned 0 times
##     12925209 (68.34%) aligned exactly 1 time
##     3053640 (16.15%) aligned &gt;1 times
## 84.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3053640/15978849
## 2 2325143/3053640
## 3 7100547/26348505
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0052-TP_1 and prad-0052-TP_2
## Single-end reads will be in prad-0052-TP_M
## 10026649 sequences in the BAM file
## 10026649 sequences exported
## 10026649 reads; of these:
##   10026649 (100.00%) were unpaired; of these:
##     2418831 (24.12%) aligned 0 times
##     5616132 (56.01%) aligned exactly 1 time
##     1991686 (19.86%) aligned &gt;1 times
## 75.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1991686/7607818
## 2 1557328/1991686
## 3 6530431/18375172
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0053-TP_1 and prad-0053-TP_2
## Single-end reads will be in prad-0053-TP_M
## 4934902 sequences in the BAM file
## 4934902 sequences exported
## 4934902 reads; of these:
##   4934902 (100.00%) were unpaired; of these:
##     2264726 (45.89%) aligned 0 times
##     1913013 (38.76%) aligned exactly 1 time
##     757163 (15.34%) aligned &gt;1 times
## 54.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 757163/2670176
## 2 566228/757163
## 3 2308846/6528358
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0054-TP_1 and prad-0054-TP_2
## Single-end reads will be in prad-0054-TP_M
## 22376278 sequences in the BAM file
## 22376278 sequences exported
## 22376278 reads; of these:
##   22376278 (100.00%) were unpaired; of these:
##     8153100 (36.44%) aligned 0 times
##     11581846 (51.76%) aligned exactly 1 time
##     2641332 (11.80%) aligned &gt;1 times
## 63.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2641332/14223178
## 2 2063395/2641332
## 3 7380069/25784835
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0055-TP_1 and prad-0055-TP_2
## Single-end reads will be in prad-0055-TP_M
## 12221789 sequences in the BAM file
## 12221789 sequences exported
## 12221789 reads; of these:
##   12221789 (100.00%) were unpaired; of these:
##     4128156 (33.78%) aligned 0 times
##     6405764 (52.41%) aligned exactly 1 time
##     1687869 (13.81%) aligned &gt;1 times
## 66.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1687869/8093633
## 2 1311426/1687869
## 3 6054498/16817598
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0056-TP_1 and prad-0056-TP_2
## Single-end reads will be in prad-0056-TP_M
## 33311826 sequences in the BAM file
## 33311826 sequences exported
## 33311826 reads; of these:
##   33311826 (100.00%) were unpaired; of these:
##     13538618 (40.64%) aligned 0 times
##     15661535 (47.01%) aligned exactly 1 time
##     4111673 (12.34%) aligned &gt;1 times
## 59.36% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 4111673/19773208
## 2 2969349/4111673
## 3 11804084/38117017
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 7.
## Output will be in prad-0057-TP_1 and prad-0057-TP_2
## Single-end reads will be in prad-0057-TP_M
## 7448460 sequences in the BAM file
## 7448460 sequences exported
## 7448460 reads; of these:
##   7448460 (100.00%) were unpaired; of these:
##     1481940 (19.90%) aligned 0 times
##     4311905 (57.89%) aligned exactly 1 time
##     1654615 (22.21%) aligned &gt;1 times
## 80.10% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1654615/5966520
## 2 1188890/1654615
## 3 5248654/14958953
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0058-TP_1 and prad-0058-TP_2
## Single-end reads will be in prad-0058-TP_M
## 5883895 sequences in the BAM file
## 5883895 sequences exported
## 5883895 reads; of these:
##   5883895 (100.00%) were unpaired; of these:
##     1249144 (21.23%) aligned 0 times
##     3403993 (57.85%) aligned exactly 1 time
##     1230758 (20.92%) aligned &gt;1 times
## 78.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1230758/4634751
## 2 1004481/1230758
## 3 5264790/12507628
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0059-TP_1 and prad-0059-TP_2
## Single-end reads will be in prad-0059-TP_M
## 11560640 sequences in the BAM file
## 11560640 sequences exported
## 11560640 reads; of these:
##   11560640 (100.00%) were unpaired; of these:
##     2372066 (20.52%) aligned 0 times
##     7054251 (61.02%) aligned exactly 1 time
##     2134323 (18.46%) aligned &gt;1 times
## 79.48% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2134323/9188574
## 2 1700358/2134323
## 3 9169129/23331147
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0060-TP_1 and prad-0060-TP_2
## Single-end reads will be in prad-0060-TP_M
## 12584924 sequences in the BAM file
## 12584924 sequences exported
## 12584924 reads; of these:
##   12584924 (100.00%) were unpaired; of these:
##     4092071 (32.52%) aligned 0 times
##     6542217 (51.98%) aligned exactly 1 time
##     1950636 (15.50%) aligned &gt;1 times
## 67.48% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1950636/8492853
## 2 1492663/1950636
## 3 8025969/20423019
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0061-TP_1 and prad-0061-TP_2
## Single-end reads will be in prad-0061-TP_M
## 7786789 sequences in the BAM file
## 7786789 sequences exported
## 7786789 reads; of these:
##   7786789 (100.00%) were unpaired; of these:
##     2683317 (34.46%) aligned 0 times
##     3461232 (44.45%) aligned exactly 1 time
##     1642240 (21.09%) aligned &gt;1 times
## 65.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1642240/5103472
## 2 1246288/1642240
## 3 3653197/10862668
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0062-TP_1 and prad-0062-TP_2
## Single-end reads will be in prad-0062-TP_M
## 12735716 sequences in the BAM file
## 12735716 sequences exported
## 12735716 reads; of these:
##   12735716 (100.00%) were unpaired; of these:
##     4149079 (32.58%) aligned 0 times
##     7150863 (56.15%) aligned exactly 1 time
##     1435774 (11.27%) aligned &gt;1 times
## 67.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1435774/8586637
## 2 1117480/1435774
## 3 5731172/16222735
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0063-TP_1 and prad-0063-TP_2
## Single-end reads will be in prad-0063-TP_M
## 19132865 sequences in the BAM file
## 19132865 sequences exported
## 19132865 reads; of these:
##   19132865 (100.00%) were unpaired; of these:
##     5031022 (26.30%) aligned 0 times
##     11490176 (60.05%) aligned exactly 1 time
##     2611667 (13.65%) aligned &gt;1 times
## 73.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2611667/14101843
## 2 2084424/2611667
## 3 10213657/27392694
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0064-TP_1 and prad-0064-TP_2
## Single-end reads will be in prad-0064-TP_M
## 11917687 sequences in the BAM file
## 11917687 sequences exported
## 11917687 reads; of these:
##   11917687 (100.00%) were unpaired; of these:
##     3582907 (30.06%) aligned 0 times
##     6857930 (57.54%) aligned exactly 1 time
##     1476850 (12.39%) aligned &gt;1 times
## 69.94% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1476850/8334780
## 2 1224993/1476850
## 3 8042419/18084377
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0065-TP_1 and prad-0065-TP_2
## Single-end reads will be in prad-0065-TP_M
## 24425705 sequences in the BAM file
## 24425705 sequences exported
## 24425705 reads; of these:
##   24425705 (100.00%) were unpaired; of these:
##     10214270 (41.82%) aligned 0 times
##     11166457 (45.72%) aligned exactly 1 time
##     3044978 (12.47%) aligned &gt;1 times
## 58.18% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3044978/14211435
## 2 2218595/3044978
## 3 6918700/24717160
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0066-TP_1 and prad-0066-TP_2
## Single-end reads will be in prad-0066-TP_M
## 8405868 sequences in the BAM file
## 8405868 sequences exported
## 8405868 reads; of these:
##   8405868 (100.00%) were unpaired; of these:
##     2876718 (34.22%) aligned 0 times
##     3658844 (43.53%) aligned exactly 1 time
##     1870306 (22.25%) aligned &gt;1 times
## 65.78% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1870306/5529150
## 2 1402829/1870306
## 3 12043655/25683454
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0067-TP_1 and prad-0067-TP_2
## Single-end reads will be in prad-0067-TP_M
## 9588796 sequences in the BAM file
## 9588796 sequences exported
## 9588796 reads; of these:
##   9588796 (100.00%) were unpaired; of these:
##     4012025 (41.84%) aligned 0 times
##     4334934 (45.21%) aligned exactly 1 time
##     1241837 (12.95%) aligned &gt;1 times
## 58.16% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1241837/5576771
## 2 984383/1241837
## 3 5473037/14192367
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0068-TP_1 and prad-0068-TP_2
## Single-end reads will be in prad-0068-TP_M
## 6042020 sequences in the BAM file
## 6042020 sequences exported
## 6042020 reads; of these:
##   6042020 (100.00%) were unpaired; of these:
##     1366992 (22.62%) aligned 0 times
##     3520817 (58.27%) aligned exactly 1 time
##     1154211 (19.10%) aligned &gt;1 times
## 77.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1154211/4675028
## 2 848197/1154211
## 3 5207134/12575964
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0069-TP_1 and prad-0069-TP_2
## Single-end reads will be in prad-0069-TP_M
## 12411812 sequences in the BAM file
## 12411812 sequences exported
## 12411812 reads; of these:
##   12411812 (100.00%) were unpaired; of these:
##     4489940 (36.17%) aligned 0 times
##     5995992 (48.31%) aligned exactly 1 time
##     1925880 (15.52%) aligned &gt;1 times
## 63.83% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1925880/7921872
## 2 1393753/1925880
## 3 5842366/17692663
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0070-TP_1 and prad-0070-TP_2
## Single-end reads will be in prad-0070-TP_M
## 15608676 sequences in the BAM file
## 15608676 sequences exported
## 15608676 reads; of these:
##   15608676 (100.00%) were unpaired; of these:
##     5888487 (37.73%) aligned 0 times
##     7276364 (46.62%) aligned exactly 1 time
##     2443825 (15.66%) aligned &gt;1 times
## 62.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2443825/9720189
## 2 1821380/2443825
## 3 10771080/29220455
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0071-TP_1 and prad-0071-TP_2
## Single-end reads will be in prad-0071-TP_M
## 6463245 sequences in the BAM file
## 6463245 sequences exported
## 6463245 reads; of these:
##   6463245 (100.00%) were unpaired; of these:
##     2323424 (35.95%) aligned 0 times
##     3181798 (49.23%) aligned exactly 1 time
##     958023 (14.82%) aligned &gt;1 times
## 64.05% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 958023/4139821
## 2 701271/958023
## 3 3003446/9281958
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0072-TP_1 and prad-0072-TP_2
## Single-end reads will be in prad-0072-TP_M
## 7322967 sequences in the BAM file
## 7322967 sequences exported
## 7322967 reads; of these:
##   7322967 (100.00%) were unpaired; of these:
##     2584833 (35.30%) aligned 0 times
##     3739407 (51.06%) aligned exactly 1 time
##     998727 (13.64%) aligned &gt;1 times
## 64.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 998727/4738134
## 2 820834/998727
## 3 6118798/12366833
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0073-TP_1 and prad-0073-TP_2
## Single-end reads will be in prad-0073-TP_M
## 27427931 sequences in the BAM file
## 27427931 sequences exported
## 27427931 reads; of these:
##   27427931 (100.00%) were unpaired; of these:
##     7545007 (27.51%) aligned 0 times
##     17026544 (62.08%) aligned exactly 1 time
##     2856380 (10.41%) aligned &gt;1 times
## 72.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2856380/19882924
## 2 2079064/2856380
## 3 10531013/35136684
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0074-TP_1 and prad-0074-TP_2
## Single-end reads will be in prad-0074-TP_M
## 18293208 sequences in the BAM file
## 18293208 sequences exported
## 18293208 reads; of these:
##   18293208 (100.00%) were unpaired; of these:
##     6115297 (33.43%) aligned 0 times
##     10002649 (54.68%) aligned exactly 1 time
##     2175262 (11.89%) aligned &gt;1 times
## 66.57% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2175262/12177911
## 2 1758452/2175262
## 3 8925320/23841330
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0075-TP_1 and prad-0075-TP_2
## Single-end reads will be in prad-0075-TP_M
## 14565714 sequences in the BAM file
## 14565714 sequences exported
## 14565714 reads; of these:
##   14565714 (100.00%) were unpaired; of these:
##     4269220 (29.31%) aligned 0 times
##     8065954 (55.38%) aligned exactly 1 time
##     2230540 (15.31%) aligned &gt;1 times
## 70.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2230540/10296494
## 2 1655001/2230540
## 3 6267916/20499364
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0076-TP_1 and prad-0076-TP_2
## Single-end reads will be in prad-0076-TP_M
## 9300953 sequences in the BAM file
## 9300953 sequences exported
## 9300953 reads; of these:
##   9300953 (100.00%) were unpaired; of these:
##     4450223 (47.85%) aligned 0 times
##     3615857 (38.88%) aligned exactly 1 time
##     1234873 (13.28%) aligned &gt;1 times
## 52.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1234873/4850730
## 2 904458/1234873
## 3 2588710/8854859
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0077-TP_1 and prad-0077-TP_2
## Single-end reads will be in prad-0077-TP_M
## 8734656 sequences in the BAM file
## 8734656 sequences exported
## 8734656 reads; of these:
##   8734656 (100.00%) were unpaired; of these:
##     1806112 (20.68%) aligned 0 times
##     5294854 (60.62%) aligned exactly 1 time
##     1633690 (18.70%) aligned &gt;1 times
## 79.32% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1633690/6928544
## 2 1309462/1633690
## 3 8676678/18750741
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0078-TP_1 and prad-0078-TP_2
## Single-end reads will be in prad-0078-TP_M
## 7308280 sequences in the BAM file
## 7308280 sequences exported
## 7308280 reads; of these:
##   7308280 (100.00%) were unpaired; of these:
##     3099867 (42.42%) aligned 0 times
##     3131043 (42.84%) aligned exactly 1 time
##     1077370 (14.74%) aligned &gt;1 times
## 57.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1077370/4208413
## 2 872669/1077370
## 3 3770399/9589506
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0079-TP_1 and prad-0079-TP_2
## Single-end reads will be in prad-0079-TP_M
## 17519122 sequences in the BAM file
## 17519122 sequences exported
## 17519122 reads; of these:
##   17519122 (100.00%) were unpaired; of these:
##     7736116 (44.16%) aligned 0 times
##     7778243 (44.40%) aligned exactly 1 time
##     2004763 (11.44%) aligned &gt;1 times
## 55.84% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2004763/9783006
## 2 1556659/2004763
## 3 5157447/17631458
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0080-TP_1 and prad-0080-TP_2
## Single-end reads will be in prad-0080-TP_M
## 9730928 sequences in the BAM file
## 9730928 sequences exported
## 9730928 reads; of these:
##   9730928 (100.00%) were unpaired; of these:
##     2740991 (28.17%) aligned 0 times
##     5173332 (53.16%) aligned exactly 1 time
##     1816605 (18.67%) aligned &gt;1 times
## 71.83% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1816605/6989937
## 2 1188887/1816605
## 3 7244118/19584185
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0081-TP_1 and prad-0081-TP_2
## Single-end reads will be in prad-0081-TP_M
## 14766802 sequences in the BAM file
## 14766802 sequences exported
## 14766802 reads; of these:
##   14766802 (100.00%) were unpaired; of these:
##     5477328 (37.09%) aligned 0 times
##     6539171 (44.28%) aligned exactly 1 time
##     2750303 (18.62%) aligned &gt;1 times
## 62.91% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2750303/9289474
## 2 2048160/2750303
## 3 7724485/21583874
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0082-TP_1 and prad-0082-TP_2
## Single-end reads will be in prad-0082-TP_M
## 13722642 sequences in the BAM file
## 13722642 sequences exported
## 13722642 reads; of these:
##   13722642 (100.00%) were unpaired; of these:
##     2520808 (18.37%) aligned 0 times
##     7894255 (57.53%) aligned exactly 1 time
##     3307579 (24.10%) aligned &gt;1 times
## 81.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3307579/11201834
## 2 2597113/3307579
## 3 15921408/34746401
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0083-TP_1 and prad-0083-TP_2
## Single-end reads will be in prad-0083-TP_M
## 14490595 sequences in the BAM file
## 14490595 sequences exported
## 14490595 reads; of these:
##   14490595 (100.00%) were unpaired; of these:
##     5311461 (36.65%) aligned 0 times
##     7079681 (48.86%) aligned exactly 1 time
##     2099453 (14.49%) aligned &gt;1 times
## 63.35% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2099453/9179134
## 2 1582067/2099453
## 3 7748421/20292442
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0084-TP_1 and prad-0084-TP_2
## Single-end reads will be in prad-0084-TP_M
## 10341405 sequences in the BAM file
## 10341405 sequences exported
## 10341405 reads; of these:
##   10341405 (100.00%) were unpaired; of these:
##     3994891 (38.63%) aligned 0 times
##     4610590 (44.58%) aligned exactly 1 time
##     1735924 (16.79%) aligned &gt;1 times
## 61.37% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1735924/6346514
## 2 1270366/1735924
## 3 6838387/18416066
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0085-TP_1 and prad-0085-TP_2
## Single-end reads will be in prad-0085-TP_M
## 8854287 sequences in the BAM file
## 8854287 sequences exported
## 8854287 reads; of these:
##   8854287 (100.00%) were unpaired; of these:
##     3307207 (37.35%) aligned 0 times
##     4290870 (48.46%) aligned exactly 1 time
##     1256210 (14.19%) aligned &gt;1 times
## 62.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1256210/5547080
## 2 950885/1256210
## 3 4642292/12277406
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0086-TP_1 and prad-0086-TP_2
## Single-end reads will be in prad-0086-TP_M
## 6253130 sequences in the BAM file
## 6253130 sequences exported
## 6253130 reads; of these:
##   6253130 (100.00%) were unpaired; of these:
##     1151020 (18.41%) aligned 0 times
##     3883976 (62.11%) aligned exactly 1 time
##     1218134 (19.48%) aligned &gt;1 times
## 81.59% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1218134/5102110
## 2 967273/1218134
## 3 5617410/12784434
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0087-TP_1 and prad-0087-TP_2
## Single-end reads will be in prad-0087-TP_M
## 25283669 sequences in the BAM file
## 25283669 sequences exported
## 25283669 reads; of these:
##   25283669 (100.00%) were unpaired; of these:
##     8823815 (34.90%) aligned 0 times
##     13610570 (53.83%) aligned exactly 1 time
##     2849284 (11.27%) aligned &gt;1 times
## 65.10% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2849284/16459854
## 2 2223711/2849284
## 3 10816829/32765597
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0088-TP_1 and prad-0088-TP_2
## Single-end reads will be in prad-0088-TP_M
## 19309499 sequences in the BAM file
## 19309499 sequences exported
## 19309499 reads; of these:
##   19309499 (100.00%) were unpaired; of these:
##     5527438 (28.63%) aligned 0 times
##     10549320 (54.63%) aligned exactly 1 time
##     3232741 (16.74%) aligned &gt;1 times
## 71.37% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3232741/13782061
## 2 2403263/3232741
## 3 11550790/33602038
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 6.
## Output will be in prad-0089-TP_1 and prad-0089-TP_2
## Single-end reads will be in prad-0089-TP_M
## 18524317 sequences in the BAM file
## 18524317 sequences exported
## 18524317 reads; of these:
##   18524317 (100.00%) were unpaired; of these:
##     5782742 (31.22%) aligned 0 times
##     10176523 (54.94%) aligned exactly 1 time
##     2565052 (13.85%) aligned &gt;1 times
## 68.78% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2565052/12741575
## 2 2102924/2565052
## 3 10274429/26715149
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0090-TP_1 and prad-0090-TP_2
## Single-end reads will be in prad-0090-TP_M
## 4002401 sequences in the BAM file
## 4002401 sequences exported
## 4002401 reads; of these:
##   4002401 (100.00%) were unpaired; of these:
##     1065813 (26.63%) aligned 0 times
##     2095444 (52.35%) aligned exactly 1 time
##     841144 (21.02%) aligned &gt;1 times
## 73.37% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 841144/2936588
## 2 628901/841144
## 3 4055089/9284469
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0091-TP_1 and prad-0091-TP_2
## Single-end reads will be in prad-0091-TP_M
## 7633994 sequences in the BAM file
## 7633994 sequences exported
## 7633994 reads; of these:
##   7633994 (100.00%) were unpaired; of these:
##     2659077 (34.83%) aligned 0 times
##     3628395 (47.53%) aligned exactly 1 time
##     1346522 (17.64%) aligned &gt;1 times
## 65.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1346522/4974917
## 2 1042260/1346522
## 3 8726496/17211893
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0092-TP_1 and prad-0092-TP_2
## Single-end reads will be in prad-0092-TP_M
## 28485422 sequences in the BAM file
## 28485422 sequences exported
## 28485422 reads; of these:
##   28485422 (100.00%) were unpaired; of these:
##     11310004 (39.70%) aligned 0 times
##     14063420 (49.37%) aligned exactly 1 time
##     3111998 (10.92%) aligned &gt;1 times
## 60.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3111998/17175418
## 2 2440328/3111998
## 3 12268212/33566308
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0093-TP_1 and prad-0093-TP_2
## Single-end reads will be in prad-0093-TP_M
## 21055362 sequences in the BAM file
## 21055362 sequences exported
## 21055362 reads; of these:
##   21055362 (100.00%) were unpaired; of these:
##     6346491 (30.14%) aligned 0 times
##     11444509 (54.35%) aligned exactly 1 time
##     3264362 (15.50%) aligned &gt;1 times
## 69.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3264362/14708871
## 2 2456027/3264362
## 3 10194061/31574666
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0094-TP_1 and prad-0094-TP_2
## Single-end reads will be in prad-0094-TP_M
## 22392243 sequences in the BAM file
## 22392243 sequences exported
## 22392243 reads; of these:
##   22392243 (100.00%) were unpaired; of these:
##     7062152 (31.54%) aligned 0 times
##     11688936 (52.20%) aligned exactly 1 time
##     3641155 (16.26%) aligned &gt;1 times
## 68.46% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3641155/15330091
## 2 2747552/3641155
## 3 11025118/33593556
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0095-TP_1 and prad-0095-TP_2
## Single-end reads will be in prad-0095-TP_M
## 6005802 sequences in the BAM file
## 6005802 sequences exported
## 6005802 reads; of these:
##   6005802 (100.00%) were unpaired; of these:
##     2748808 (45.77%) aligned 0 times
##     2150894 (35.81%) aligned exactly 1 time
##     1106100 (18.42%) aligned &gt;1 times
## 54.23% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1106100/3256994
## 2 911262/1106100
## 3 2372053/6930786
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0096-TP_1 and prad-0096-TP_2
## Single-end reads will be in prad-0096-TP_M
## 5245021 sequences in the BAM file
## 5245021 sequences exported
## 5245021 reads; of these:
##   5245021 (100.00%) were unpaired; of these:
##     1060017 (20.21%) aligned 0 times
##     2792262 (53.24%) aligned exactly 1 time
##     1392742 (26.55%) aligned &gt;1 times
## 79.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1392742/4185004
## 2 1086199/1392742
## 3 8291256/16441412
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0097-TP_1 and prad-0097-TP_2
## Single-end reads will be in prad-0097-TP_M
## 8683620 sequences in the BAM file
## 8683620 sequences exported
## 8683620 reads; of these:
##   8683620 (100.00%) were unpaired; of these:
##     2805305 (32.31%) aligned 0 times
##     4360262 (50.21%) aligned exactly 1 time
##     1518053 (17.48%) aligned &gt;1 times
## 67.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1518053/5878315
## 2 1168792/1518053
## 3 8261491/18241793
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0098-TP_1 and prad-0098-TP_2
## Single-end reads will be in prad-0098-TP_M
## 20995598 sequences in the BAM file
## 20995598 sequences exported
## 20995598 reads; of these:
##   20995598 (100.00%) were unpaired; of these:
##     6578304 (31.33%) aligned 0 times
##     11852933 (56.45%) aligned exactly 1 time
##     2564361 (12.21%) aligned &gt;1 times
## 68.67% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2564361/14417294
## 2 2096498/2564361
## 3 15645879/33952163
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0099-TP_1 and prad-0099-TP_2
## Single-end reads will be in prad-0099-TP_M
## 17662476 sequences in the BAM file
## 17662476 sequences exported
## 17662476 reads; of these:
##   17662476 (100.00%) were unpaired; of these:
##     7938947 (44.95%) aligned 0 times
##     7641492 (43.26%) aligned exactly 1 time
##     2082037 (11.79%) aligned &gt;1 times
## 55.05% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2082037/9723529
## 2 1612474/2082037
## 3 5333402/18085120
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0100-TP_1 and prad-0100-TP_2
## Single-end reads will be in prad-0100-TP_M
## 19969366 sequences in the BAM file
## 19969366 sequences exported
## 19969366 reads; of these:
##   19969366 (100.00%) were unpaired; of these:
##     8054951 (40.34%) aligned 0 times
##     9403575 (47.09%) aligned exactly 1 time
##     2510840 (12.57%) aligned &gt;1 times
## 59.66% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2510840/11914415
## 2 1976241/2510840
## 3 7854003/22679598
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0101-TP_1 and prad-0101-TP_2
## Single-end reads will be in prad-0101-TP_M
## 9170167 sequences in the BAM file
## 9170167 sequences exported
## 9170167 reads; of these:
##   9170167 (100.00%) were unpaired; of these:
##     2955502 (32.23%) aligned 0 times
##     4793792 (52.28%) aligned exactly 1 time
##     1420873 (15.49%) aligned &gt;1 times
## 67.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1420873/6214665
## 2 1090499/1420873
## 3 6535530/15784194
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0102-TP_1 and prad-0102-TP_2
## Single-end reads will be in prad-0102-TP_M
## 8089668 sequences in the BAM file
## 8089668 sequences exported
## 8089668 reads; of these:
##   8089668 (100.00%) were unpaired; of these:
##     2896071 (35.80%) aligned 0 times
##     3757376 (46.45%) aligned exactly 1 time
##     1436221 (17.75%) aligned &gt;1 times
## 64.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1436221/5193597
## 2 1007086/1436221
## 3 3805957/11522507
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0103-TP_1 and prad-0103-TP_2
## Single-end reads will be in prad-0103-TP_M
## 9183805 sequences in the BAM file
## 9183805 sequences exported
## 9183805 reads; of these:
##   9183805 (100.00%) were unpaired; of these:
##     2948410 (32.10%) aligned 0 times
##     4941455 (53.81%) aligned exactly 1 time
##     1293940 (14.09%) aligned &gt;1 times
## 67.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1293940/6235395
## 2 990410/1293940
## 3 4422381/12514025
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0104-TP_1 and prad-0104-TP_2
## Single-end reads will be in prad-0104-TP_M
## 7010809 sequences in the BAM file
## 7010809 sequences exported
## 7010809 reads; of these:
##   7010809 (100.00%) were unpaired; of these:
##     1453166 (20.73%) aligned 0 times
##     4258326 (60.74%) aligned exactly 1 time
##     1299317 (18.53%) aligned &gt;1 times
## 79.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1299317/5557643
## 2 1024034/1299317
## 3 7156603/16157459
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0105-TP_1 and prad-0105-TP_2
## Single-end reads will be in prad-0105-TP_M
## 5642821 sequences in the BAM file
## 5642821 sequences exported
## 5642821 reads; of these:
##   5642821 (100.00%) were unpaired; of these:
##     1236228 (21.91%) aligned 0 times
##     2741004 (48.58%) aligned exactly 1 time
##     1665589 (29.52%) aligned &gt;1 times
## 78.09% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1665589/4406593
## 2 1179108/1665589
## 3 12113905/25861536
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0106-TP_1 and prad-0106-TP_2
## Single-end reads will be in prad-0106-TP_M
## 11771272 sequences in the BAM file
## 11771272 sequences exported
## 11771272 reads; of these:
##   11771272 (100.00%) were unpaired; of these:
##     3631544 (30.85%) aligned 0 times
##     5908109 (50.19%) aligned exactly 1 time
##     2231619 (18.96%) aligned &gt;1 times
## 69.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2231619/8139728
## 2 1614915/2231619
## 3 7125438/19917506
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0107-TP_1 and prad-0107-TP_2
## Single-end reads will be in prad-0107-TP_M
## 10388626 sequences in the BAM file
## 10388626 sequences exported
## 10388626 reads; of these:
##   10388626 (100.00%) were unpaired; of these:
##     3623936 (34.88%) aligned 0 times
##     4455070 (42.88%) aligned exactly 1 time
##     2309620 (22.23%) aligned &gt;1 times
## 65.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2309620/6764690
## 2 1818901/2309620
## 3 5854954/16179172
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0108-TP_1 and prad-0108-TP_2
## Single-end reads will be in prad-0108-TP_M
## 9506973 sequences in the BAM file
## 9506973 sequences exported
## 9506973 reads; of these:
##   9506973 (100.00%) were unpaired; of these:
##     1721105 (18.10%) aligned 0 times
##     5620617 (59.12%) aligned exactly 1 time
##     2165251 (22.78%) aligned &gt;1 times
## 81.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2165251/7785868
## 2 1681912/2165251
## 3 5782676/17119314
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0109-TP_1 and prad-0109-TP_2
## Single-end reads will be in prad-0109-TP_M
## 12302506 sequences in the BAM file
## 12302506 sequences exported
## 12302506 reads; of these:
##   12302506 (100.00%) were unpaired; of these:
##     3461794 (28.14%) aligned 0 times
##     6224357 (50.59%) aligned exactly 1 time
##     2616355 (21.27%) aligned &gt;1 times
## 71.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2616355/8840712
## 2 1940402/2616355
## 3 10467658/27145737
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0110-TP_1 and prad-0110-TP_2
## Single-end reads will be in prad-0110-TP_M
## 24017025 sequences in the BAM file
## 24017025 sequences exported
## 24017025 reads; of these:
##   24017025 (100.00%) were unpaired; of these:
##     5215961 (21.72%) aligned 0 times
##     15014255 (62.52%) aligned exactly 1 time
##     3786809 (15.77%) aligned &gt;1 times
## 78.28% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3786809/18801064
## 2 2998034/3786809
## 3 14357917/38046426
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0111-TP_1 and prad-0111-TP_2
## Single-end reads will be in prad-0111-TP_M
## 12710605 sequences in the BAM file
## 12710605 sequences exported
## 12710605 reads; of these:
##   12710605 (100.00%) were unpaired; of these:
##     6673932 (52.51%) aligned 0 times
##     4663858 (36.69%) aligned exactly 1 time
##     1372815 (10.80%) aligned &gt;1 times
## 47.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1372815/6036673
## 2 999827/1372815
## 3 4698400/13357517
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0112-TP_1 and prad-0112-TP_2
## Single-end reads will be in prad-0112-TP_M
## 9468889 sequences in the BAM file
## 9468889 sequences exported
## 9468889 reads; of these:
##   9468889 (100.00%) were unpaired; of these:
##     3066545 (32.39%) aligned 0 times
##     5024056 (53.06%) aligned exactly 1 time
##     1378288 (14.56%) aligned &gt;1 times
## 67.61% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1378288/6402344
## 2 1099519/1378288
## 3 7187066/15811190
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0113-TP_1 and prad-0113-TP_2
## Single-end reads will be in prad-0113-TP_M
## 12733974 sequences in the BAM file
## 12733974 sequences exported
## 12733974 reads; of these:
##   12733974 (100.00%) were unpaired; of these:
##     4395013 (34.51%) aligned 0 times
##     6202256 (48.71%) aligned exactly 1 time
##     2136705 (16.78%) aligned &gt;1 times
## 65.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2136705/8338961
## 2 1676867/2136705
## 3 10100304/23709266
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0114-TP_1 and prad-0114-TP_2
## Single-end reads will be in prad-0114-TP_M
## 7544619 sequences in the BAM file
## 7544619 sequences exported
## 7544619 reads; of these:
##   7544619 (100.00%) were unpaired; of these:
##     2807395 (37.21%) aligned 0 times
##     3909647 (51.82%) aligned exactly 1 time
##     827577 (10.97%) aligned &gt;1 times
## 62.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 827577/4737224
## 2 618123/827577
## 3 3359016/10026100
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0115-TP_1 and prad-0115-TP_2
## Single-end reads will be in prad-0115-TP_M
## 15144575 sequences in the BAM file
## 15144575 sequences exported
## 15144575 reads; of these:
##   15144575 (100.00%) were unpaired; of these:
##     6013231 (39.71%) aligned 0 times
##     7059552 (46.61%) aligned exactly 1 time
##     2071792 (13.68%) aligned &gt;1 times
## 60.29% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2071792/9131344
## 2 1679702/2071792
## 3 6485833/18378811
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0116-TP_1 and prad-0116-TP_2
## Single-end reads will be in prad-0116-TP_M
## 10284702 sequences in the BAM file
## 10284702 sequences exported
## 10284702 reads; of these:
##   10284702 (100.00%) were unpaired; of these:
##     3185946 (30.98%) aligned 0 times
##     5560660 (54.07%) aligned exactly 1 time
##     1538096 (14.96%) aligned &gt;1 times
## 69.02% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1538096/7098756
## 2 1218980/1538096
## 3 5892685/15849311
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0117-TP_1 and prad-0117-TP_2
## Single-end reads will be in prad-0117-TP_M
## 27239266 sequences in the BAM file
## 27239266 sequences exported
## 27239266 reads; of these:
##   27239266 (100.00%) were unpaired; of these:
##     11034724 (40.51%) aligned 0 times
##     13032601 (47.84%) aligned exactly 1 time
##     3171941 (11.64%) aligned &gt;1 times
## 59.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3171941/16204542
## 2 2543958/3171941
## 3 10897290/30621693
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0118-TP_1 and prad-0118-TP_2
## Single-end reads will be in prad-0118-TP_M
## 7207218 sequences in the BAM file
## 7207218 sequences exported
## 7207218 reads; of these:
##   7207218 (100.00%) were unpaired; of these:
##     2920489 (40.52%) aligned 0 times
##     3371636 (46.78%) aligned exactly 1 time
##     915093 (12.70%) aligned &gt;1 times
## 59.48% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 915093/4286729
## 2 675904/915093
## 3 2882099/8803388
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0119-TP_1 and prad-0119-TP_2
## Single-end reads will be in prad-0119-TP_M
## 12401428 sequences in the BAM file
## 12401428 sequences exported
## 12401428 reads; of these:
##   12401428 (100.00%) were unpaired; of these:
##     3603805 (29.06%) aligned 0 times
##     6931032 (55.89%) aligned exactly 1 time
##     1866591 (15.05%) aligned &gt;1 times
## 70.94% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1866591/8797623
## 2 1434767/1866591
## 3 5375779/17558803
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0120-TP_1 and prad-0120-TP_2
## Single-end reads will be in prad-0120-TP_M
## 9587068 sequences in the BAM file
## 9587068 sequences exported
## 9587068 reads; of these:
##   9587068 (100.00%) were unpaired; of these:
##     2857348 (29.80%) aligned 0 times
##     4584855 (47.82%) aligned exactly 1 time
##     2144865 (22.37%) aligned &gt;1 times
## 70.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2144865/6729720
## 2 1611448/2144865
## 3 6810147/18244172
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0121-TP_1 and prad-0121-TP_2
## Single-end reads will be in prad-0121-TP_M
## 17231995 sequences in the BAM file
## 17231995 sequences exported
## 17231995 reads; of these:
##   17231995 (100.00%) were unpaired; of these:
##     3405359 (19.76%) aligned 0 times
##     10553998 (61.25%) aligned exactly 1 time
##     3272638 (18.99%) aligned &gt;1 times
## 80.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3272638/13826636
## 2 2412935/3272638
## 3 12356709/34679221
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0122-TP_1 and prad-0122-TP_2
## Single-end reads will be in prad-0122-TP_M
## 17836032 sequences in the BAM file
## 17836032 sequences exported
## 17836032 reads; of these:
##   17836032 (100.00%) were unpaired; of these:
##     4110002 (23.04%) aligned 0 times
##     8553136 (47.95%) aligned exactly 1 time
##     5172894 (29.00%) aligned &gt;1 times
## 76.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 5172894/13726030
## 2 3645813/5172894
## 3 37762715/87284947
## [bam_sort_core] merging from 14 files...
## This looks like paired data from lane 7.
## Output will be in prad-0123-TP_1 and prad-0123-TP_2
## Single-end reads will be in prad-0123-TP_M
## 11247673 sequences in the BAM file
## 11247673 sequences exported
## 11247673 reads; of these:
##   11247673 (100.00%) were unpaired; of these:
##     3201306 (28.46%) aligned 0 times
##     6007615 (53.41%) aligned exactly 1 time
##     2038752 (18.13%) aligned &gt;1 times
## 71.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2038752/8046367
## 2 1443501/2038752
## 3 9929785/23578333
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0124-TP_1 and prad-0124-TP_2
## Single-end reads will be in prad-0124-TP_M
## 13623475 sequences in the BAM file
## 13623475 sequences exported
## 13623475 reads; of these:
##   13623475 (100.00%) were unpaired; of these:
##     5129297 (37.65%) aligned 0 times
##     6409424 (47.05%) aligned exactly 1 time
##     2084754 (15.30%) aligned &gt;1 times
## 62.35% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2084754/8494178
## 2 1646915/2084754
## 3 6337073/20507306
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0125-TP_1 and prad-0125-TP_2
## Single-end reads will be in prad-0125-TP_M
## 6600843 sequences in the BAM file
## 6600843 sequences exported
## 6600843 reads; of these:
##   6600843 (100.00%) were unpaired; of these:
##     1307149 (19.80%) aligned 0 times
##     3802204 (57.60%) aligned exactly 1 time
##     1491490 (22.60%) aligned &gt;1 times
## 80.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1491490/5293694
## 2 1144708/1491490
## 3 4195264/12141916
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0126-TP_1 and prad-0126-TP_2
## Single-end reads will be in prad-0126-TP_M
## 14060016 sequences in the BAM file
## 14060016 sequences exported
## 14060016 reads; of these:
##   14060016 (100.00%) were unpaired; of these:
##     3769042 (26.81%) aligned 0 times
##     8221758 (58.48%) aligned exactly 1 time
##     2069216 (14.72%) aligned &gt;1 times
## 73.19% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2069216/10290974
## 2 1509024/2069216
## 3 9647951/25477182
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0127-TP_1 and prad-0127-TP_2
## Single-end reads will be in prad-0127-TP_M
## 12656074 sequences in the BAM file
## 12656074 sequences exported
## 12656074 reads; of these:
##   12656074 (100.00%) were unpaired; of these:
##     5195770 (41.05%) aligned 0 times
##     6007238 (47.47%) aligned exactly 1 time
##     1453066 (11.48%) aligned &gt;1 times
## 58.95% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1453066/7460304
## 2 1140655/1453066
## 3 5242587/14899706
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0128-TP_1 and prad-0128-TP_2
## Single-end reads will be in prad-0128-TP_M
## 9110751 sequences in the BAM file
## 9110751 sequences exported
## 9110751 reads; of these:
##   9110751 (100.00%) were unpaired; of these:
##     1811505 (19.88%) aligned 0 times
##     5314343 (58.33%) aligned exactly 1 time
##     1984903 (21.79%) aligned &gt;1 times
## 80.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1984903/7299246
## 2 1534014/1984903
## 3 10568159/23374540
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0129-TP_1 and prad-0129-TP_2
## Single-end reads will be in prad-0129-TP_M
## 9965172 sequences in the BAM file
## 9965172 sequences exported
## 9965172 reads; of these:
##   9965172 (100.00%) were unpaired; of these:
##     3675874 (36.89%) aligned 0 times
##     4331304 (43.46%) aligned exactly 1 time
##     1957994 (19.65%) aligned &gt;1 times
## 63.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1957994/6289298
## 2 1386980/1957994
## 3 7182565/19653794
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0130-TP_1 and prad-0130-TP_2
## Single-end reads will be in prad-0130-TP_M
## 11895601 sequences in the BAM file
## 11895601 sequences exported
## 11895601 reads; of these:
##   11895601 (100.00%) were unpaired; of these:
##     2058500 (17.30%) aligned 0 times
##     7923853 (66.61%) aligned exactly 1 time
##     1913248 (16.08%) aligned &gt;1 times
## 82.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1913248/9837101
## 2 1426284/1913248
## 3 6431531/20059780
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0131-TP_1 and prad-0131-TP_2
## Single-end reads will be in prad-0131-TP_M
## 11689488 sequences in the BAM file
## 11689488 sequences exported
## 11689488 reads; of these:
##   11689488 (100.00%) were unpaired; of these:
##     5209966 (44.57%) aligned 0 times
##     4835290 (41.36%) aligned exactly 1 time
##     1644232 (14.07%) aligned &gt;1 times
## 55.43% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1644232/6479522
## 2 1254219/1644232
## 3 4342680/13380172
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0132-TP_1 and prad-0132-TP_2
## Single-end reads will be in prad-0132-TP_M
## 20374801 sequences in the BAM file
## 20374801 sequences exported
## 20374801 reads; of these:
##   20374801 (100.00%) were unpaired; of these:
##     7837697 (38.47%) aligned 0 times
##     9947214 (48.82%) aligned exactly 1 time
##     2589890 (12.71%) aligned &gt;1 times
## 61.53% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2589890/12537104
## 2 2040051/2589890
## 3 8930113/24970467
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0133-TP_1 and prad-0133-TP_2
## Single-end reads will be in prad-0133-TP_M
## 8715764 sequences in the BAM file
## 8715764 sequences exported
## 8715764 reads; of these:
##   8715764 (100.00%) were unpaired; of these:
##     3213581 (36.87%) aligned 0 times
##     3914797 (44.92%) aligned exactly 1 time
##     1587386 (18.21%) aligned &gt;1 times
## 63.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1587386/5502183
## 2 1189224/1587386
## 3 5227701/14553262
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0134-TP_1 and prad-0134-TP_2
## Single-end reads will be in prad-0134-TP_M
## 13624908 sequences in the BAM file
## 13624908 sequences exported
## 13624908 reads; of these:
##   13624908 (100.00%) were unpaired; of these:
##     2753319 (20.21%) aligned 0 times
##     7833099 (57.49%) aligned exactly 1 time
##     3038490 (22.30%) aligned &gt;1 times
## 79.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3038490/10871589
## 2 2476901/3038490
## 3 14877596/32265841
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0135-TP_1 and prad-0135-TP_2
## Single-end reads will be in prad-0135-TP_M
## 15588110 sequences in the BAM file
## 15588110 sequences exported
## 15588110 reads; of these:
##   15588110 (100.00%) were unpaired; of these:
##     4979295 (31.94%) aligned 0 times
##     8512127 (54.61%) aligned exactly 1 time
##     2096688 (13.45%) aligned &gt;1 times
## 68.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2096688/10608815
## 2 1652315/2096688
## 3 14682493/32827414
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0136-TP_1 and prad-0136-TP_2
## Single-end reads will be in prad-0136-TP_M
## 28515958 sequences in the BAM file
## 28515958 sequences exported
## 28515958 reads; of these:
##   28515958 (100.00%) were unpaired; of these:
##     14249440 (49.97%) aligned 0 times
##     11643614 (40.83%) aligned exactly 1 time
##     2622904 (9.20%) aligned &gt;1 times
## 50.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2622904/14266518
## 2 1931560/2622904
## 3 7067697/24904680
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0137-TP_1 and prad-0137-TP_2
## Single-end reads will be in prad-0137-TP_M
## 21511050 sequences in the BAM file
## 21511050 sequences exported
## 21511050 reads; of these:
##   21511050 (100.00%) were unpaired; of these:
##     8876931 (41.27%) aligned 0 times
##     9938062 (46.20%) aligned exactly 1 time
##     2696057 (12.53%) aligned &gt;1 times
## 58.73% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2696057/12634119
## 2 1908937/2696057
## 3 7345673/23959249
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0138-TP_1 and prad-0138-TP_2
## Single-end reads will be in prad-0138-TP_M
## 14780501 sequences in the BAM file
## 14780501 sequences exported
## 14780501 reads; of these:
##   14780501 (100.00%) were unpaired; of these:
##     4186491 (28.32%) aligned 0 times
##     8153028 (55.16%) aligned exactly 1 time
##     2440982 (16.51%) aligned &gt;1 times
## 71.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2440982/10594010
## 2 1690317/2440982
## 3 10734348/27438081
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0139-TP_1 and prad-0139-TP_2
## Single-end reads will be in prad-0139-TP_M
## 11049749 sequences in the BAM file
## 11049749 sequences exported
## 11049749 reads; of these:
##   11049749 (100.00%) were unpaired; of these:
##     3925598 (35.53%) aligned 0 times
##     5584070 (50.54%) aligned exactly 1 time
##     1540081 (13.94%) aligned &gt;1 times
## 64.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1540081/7124151
## 2 1164878/1540081
## 3 5915341/15950860
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0140-TP_1 and prad-0140-TP_2
## Single-end reads will be in prad-0140-TP_M
## 9437481 sequences in the BAM file
## 9437481 sequences exported
## 9437481 reads; of these:
##   9437481 (100.00%) were unpaired; of these:
##     3149782 (33.38%) aligned 0 times
##     4482985 (47.50%) aligned exactly 1 time
##     1804714 (19.12%) aligned &gt;1 times
## 66.62% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1804714/6287699
## 2 1345708/1804714
## 3 7613736/19668173
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0141-TP_1 and prad-0141-TP_2
## Single-end reads will be in prad-0141-TP_M
## 12896181 sequences in the BAM file
## 12896181 sequences exported
## 12896181 reads; of these:
##   12896181 (100.00%) were unpaired; of these:
##     4570336 (35.44%) aligned 0 times
##     6190315 (48.00%) aligned exactly 1 time
##     2135530 (16.56%) aligned &gt;1 times
## 64.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2135530/8325845
## 2 1613641/2135530
## 3 6692400/20066074
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0142-TP_1 and prad-0142-TP_2
## Single-end reads will be in prad-0142-TP_M
## 10018662 sequences in the BAM file
## 10018662 sequences exported
## 10018662 reads; of these:
##   10018662 (100.00%) were unpaired; of these:
##     4547091 (45.39%) aligned 0 times
##     4165789 (41.58%) aligned exactly 1 time
##     1305782 (13.03%) aligned &gt;1 times
## 54.61% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1305782/5471571
## 2 961760/1305782
## 3 3588654/11310985
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0143-TP_1 and prad-0143-TP_2
## Single-end reads will be in prad-0143-TP_M
## 11943715 sequences in the BAM file
## 11943715 sequences exported
## 11943715 reads; of these:
##   11943715 (100.00%) were unpaired; of these:
##     2599482 (21.76%) aligned 0 times
##     6656680 (55.73%) aligned exactly 1 time
##     2687553 (22.50%) aligned &gt;1 times
## 78.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2687553/9344233
## 2 2017847/2687553
## 3 8413878/22585650
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0144-TP_1 and prad-0144-TP_2
## Single-end reads will be in prad-0144-TP_M
## 18347491 sequences in the BAM file
## 18347491 sequences exported
## 18347491 reads; of these:
##   18347491 (100.00%) were unpaired; of these:
##     7500628 (40.88%) aligned 0 times
##     8448583 (46.05%) aligned exactly 1 time
##     2398280 (13.07%) aligned &gt;1 times
## 59.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2398280/10846863
## 2 1945632/2398280
## 3 8325649/23587916
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0145-TP_1 and prad-0145-TP_2
## Single-end reads will be in prad-0145-TP_M
## 18684621 sequences in the BAM file
## 18684621 sequences exported
## 18684621 reads; of these:
##   18684621 (100.00%) were unpaired; of these:
##     7425879 (39.74%) aligned 0 times
##     9414272 (50.39%) aligned exactly 1 time
##     1844470 (9.87%) aligned &gt;1 times
## 60.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1844470/11258742
## 2 1474081/1844470
## 3 7407961/21048368
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0146-TP_1 and prad-0146-TP_2
## Single-end reads will be in prad-0146-TP_M
## 10937475 sequences in the BAM file
## 10937475 sequences exported
## 10937475 reads; of these:
##   10937475 (100.00%) were unpaired; of these:
##     3475576 (31.78%) aligned 0 times
##     6041313 (55.23%) aligned exactly 1 time
##     1420586 (12.99%) aligned &gt;1 times
## 68.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1420586/7461899
## 2 1075083/1420586
## 3 5680253/15337426
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0147-TP_1 and prad-0147-TP_2
## Single-end reads will be in prad-0147-TP_M
## 9055572 sequences in the BAM file
## 9055572 sequences exported
## 9055572 reads; of these:
##   9055572 (100.00%) were unpaired; of these:
##     3118335 (34.44%) aligned 0 times
##     4561522 (50.37%) aligned exactly 1 time
##     1375715 (15.19%) aligned &gt;1 times
## 65.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1375715/5937237
## 2 1118793/1375715
## 3 5335079/13581503
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0148-TP_1 and prad-0148-TP_2
## Single-end reads will be in prad-0148-TP_M
## 7958394 sequences in the BAM file
## 7958394 sequences exported
## 7958394 reads; of these:
##   7958394 (100.00%) were unpaired; of these:
##     3219008 (40.45%) aligned 0 times
##     3615659 (45.43%) aligned exactly 1 time
##     1123727 (14.12%) aligned &gt;1 times
## 59.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1123727/4739386
## 2 765577/1123727
## 3 3790134/11048793
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0149-TP_1 and prad-0149-TP_2
## Single-end reads will be in prad-0149-TP_M
## 20445809 sequences in the BAM file
## 20445809 sequences exported
## 20445809 reads; of these:
##   20445809 (100.00%) were unpaired; of these:
##     4355969 (21.30%) aligned 0 times
##     11387310 (55.70%) aligned exactly 1 time
##     4702530 (23.00%) aligned &gt;1 times
## 78.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 4702530/16089840
## 2 3955912/4702530
## 3 16871620/41155914
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in prad-0150-TP_1 and prad-0150-TP_2
## Single-end reads will be in prad-0150-TP_M
## 9303775 sequences in the BAM file
## 9303775 sequences exported
## 9303775 reads; of these:
##   9303775 (100.00%) were unpaired; of these:
##     3329207 (35.78%) aligned 0 times
##     4239007 (45.56%) aligned exactly 1 time
##     1735561 (18.65%) aligned &gt;1 times
## 64.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1735561/5974568
## 2 1315488/1735561
## 3 7105870/18375797
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0151-TP_1 and prad-0151-TP_2
## Single-end reads will be in prad-0151-TP_M
## 6909261 sequences in the BAM file
## 6909261 sequences exported
## 6909261 reads; of these:
##   6909261 (100.00%) were unpaired; of these:
##     2346767 (33.97%) aligned 0 times
##     2894018 (41.89%) aligned exactly 1 time
##     1668476 (24.15%) aligned &gt;1 times
## 66.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1668476/4562494
## 2 1396536/1668476
## 3 4551257/12012050
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0152-TP_1 and prad-0152-TP_2
## Single-end reads will be in prad-0152-TP_M
## 5636184 sequences in the BAM file
## 5636184 sequences exported
## 5636184 reads; of these:
##   5636184 (100.00%) were unpaired; of these:
##     1281044 (22.73%) aligned 0 times
##     2645795 (46.94%) aligned exactly 1 time
##     1709345 (30.33%) aligned &gt;1 times
## 77.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1709345/4355140
## 2 1309465/1709345
## 3 17599420/32947576
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0153-TP_1 and prad-0153-TP_2
## Single-end reads will be in prad-0153-TP_M
## 13728038 sequences in the BAM file
## 13728038 sequences exported
## 13728038 reads; of these:
##   13728038 (100.00%) were unpaired; of these:
##     2681241 (19.53%) aligned 0 times
##     6948733 (50.62%) aligned exactly 1 time
##     4098064 (29.85%) aligned &gt;1 times
## 80.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 4098064/11046797
## 2 3375559/4098064
## 3 23659353/53706369
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 5.
## Output will be in prad-0154-TP_1 and prad-0154-TP_2
## Single-end reads will be in prad-0154-TP_M
## 16143138 sequences in the BAM file
## 16143138 sequences exported
## 16143138 reads; of these:
##   16143138 (100.00%) were unpaired; of these:
##     6097476 (37.77%) aligned 0 times
##     7702218 (47.71%) aligned exactly 1 time
##     2343444 (14.52%) aligned &gt;1 times
## 62.23% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2343444/10045662
## 2 1739192/2343444
## 3 7603272/22823397
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0155-TP_1 and prad-0155-TP_2
## Single-end reads will be in prad-0155-TP_M
## 8700535 sequences in the BAM file
## 8700535 sequences exported
## 8700535 reads; of these:
##   8700535 (100.00%) were unpaired; of these:
##     4027717 (46.29%) aligned 0 times
##     3568279 (41.01%) aligned exactly 1 time
##     1104539 (12.70%) aligned &gt;1 times
## 53.71% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1104539/4672818
## 2 805759/1104539
## 3 2929668/9262042
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0156-TP_1 and prad-0156-TP_2
## Single-end reads will be in prad-0156-TP_M
## 11383056 sequences in the BAM file
## 11383056 sequences exported
## 11383056 reads; of these:
##   11383056 (100.00%) were unpaired; of these:
##     5485445 (48.19%) aligned 0 times
##     4423253 (38.86%) aligned exactly 1 time
##     1474358 (12.95%) aligned &gt;1 times
## 51.81% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1474358/5897611
## 2 1168452/1474358
## 3 4282987/12719390
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0157-TP_1 and prad-0157-TP_2
## Single-end reads will be in prad-0157-TP_M
## 13401856 sequences in the BAM file
## 13401856 sequences exported
## 13401856 reads; of these:
##   13401856 (100.00%) were unpaired; of these:
##     4605188 (34.36%) aligned 0 times
##     7161522 (53.44%) aligned exactly 1 time
##     1635146 (12.20%) aligned &gt;1 times
## 65.64% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1635146/8796668
## 2 1335356/1635146
## 3 6116022/16563540
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0158-TP_1 and prad-0158-TP_2
## Single-end reads will be in prad-0158-TP_M
## 16953621 sequences in the BAM file
## 16953621 sequences exported
## 16953621 reads; of these:
##   16953621 (100.00%) were unpaired; of these:
##     5052948 (29.80%) aligned 0 times
##     10053375 (59.30%) aligned exactly 1 time
##     1847298 (10.90%) aligned &gt;1 times
## 70.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1847298/11900673
## 2 1538195/1847298
## 3 10922896/25059305
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0159-TP_1 and prad-0159-TP_2
## Single-end reads will be in prad-0159-TP_M
## 14210827 sequences in the BAM file
## 14210827 sequences exported
## 14210827 reads; of these:
##   14210827 (100.00%) were unpaired; of these:
##     5618448 (39.54%) aligned 0 times
##     6330766 (44.55%) aligned exactly 1 time
##     2261613 (15.91%) aligned &gt;1 times
## 60.46% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2261613/8592379
## 2 1833110/2261613
## 3 9956787/24534061
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0160-TP_1 and prad-0160-TP_2
## Single-end reads will be in prad-0160-TP_M
## 15284693 sequences in the BAM file
## 15284693 sequences exported
## 15284693 reads; of these:
##   15284693 (100.00%) were unpaired; of these:
##     5270709 (34.48%) aligned 0 times
##     8028070 (52.52%) aligned exactly 1 time
##     1985914 (12.99%) aligned &gt;1 times
## 65.52% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1985914/10013984
## 2 1566060/1985914
## 3 13195112/27655673
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0161-TP_1 and prad-0161-TP_2
## Single-end reads will be in prad-0161-TP_M
## 7528326 sequences in the BAM file
## 7528326 sequences exported
## 7528326 reads; of these:
##   7528326 (100.00%) were unpaired; of these:
##     2178685 (28.94%) aligned 0 times
##     4183349 (55.57%) aligned exactly 1 time
##     1166292 (15.49%) aligned &gt;1 times
## 71.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1166292/5349641
## 2 965592/1166292
## 3 8557250/16359249
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0162-TP_1 and prad-0162-TP_2
## Single-end reads will be in prad-0162-TP_M
## 10038198 sequences in the BAM file
## 10038198 sequences exported
## 10038198 reads; of these:
##   10038198 (100.00%) were unpaired; of these:
##     2717869 (27.08%) aligned 0 times
##     5525134 (55.04%) aligned exactly 1 time
##     1795195 (17.88%) aligned &gt;1 times
## 72.92% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1795195/7320329
## 2 1391139/1795195
## 3 8256187/19553042
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0163-TP_1 and prad-0163-TP_2
## Single-end reads will be in prad-0163-TP_M
## 9103421 sequences in the BAM file
## 9103421 sequences exported
## 9103421 reads; of these:
##   9103421 (100.00%) were unpaired; of these:
##     1877736 (20.63%) aligned 0 times
##     5167789 (56.77%) aligned exactly 1 time
##     2057896 (22.61%) aligned &gt;1 times
## 79.37% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2057896/7225685
## 2 1648583/2057896
## 3 10782819/22901019
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0164-TP_1 and prad-0164-TP_2
## Single-end reads will be in prad-0164-TP_M
## 9380782 sequences in the BAM file
## 9380782 sequences exported
## 9380782 reads; of these:
##   9380782 (100.00%) were unpaired; of these:
##     3757441 (40.05%) aligned 0 times
##     4326189 (46.12%) aligned exactly 1 time
##     1297152 (13.83%) aligned &gt;1 times
## 59.95% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1297152/5623341
## 2 992787/1297152
## 3 5059696/13242313
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0165-TP_1 and prad-0165-TP_2
## Single-end reads will be in prad-0165-TP_M
## 13896712 sequences in the BAM file
## 13896712 sequences exported
## 13896712 reads; of these:
##   13896712 (100.00%) were unpaired; of these:
##     4103629 (29.53%) aligned 0 times
##     8143027 (58.60%) aligned exactly 1 time
##     1650056 (11.87%) aligned &gt;1 times
## 70.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1650056/9793083
## 2 1337992/1650056
## 3 9686711/22206376
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0166-TP_1 and prad-0166-TP_2
## Single-end reads will be in prad-0166-TP_M
## 13035296 sequences in the BAM file
## 13035296 sequences exported
## 13035296 reads; of these:
##   13035296 (100.00%) were unpaired; of these:
##     2804877 (21.52%) aligned 0 times
##     7550171 (57.92%) aligned exactly 1 time
##     2680248 (20.56%) aligned &gt;1 times
## 78.48% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2680248/10230419
## 2 2033709/2680248
## 3 11882883/28512213
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0167-TP_1 and prad-0167-TP_2
## Single-end reads will be in prad-0167-TP_M
## 9294562 sequences in the BAM file
## 9294562 sequences exported
## 9294562 reads; of these:
##   9294562 (100.00%) were unpaired; of these:
##     3331934 (35.85%) aligned 0 times
##     3836282 (41.27%) aligned exactly 1 time
##     2126346 (22.88%) aligned &gt;1 times
## 64.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2126346/5962628
## 2 1589692/2126346
## 3 11280431/26821386
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0168-TP_1 and prad-0168-TP_2
## Single-end reads will be in prad-0168-TP_M
## 6690841 sequences in the BAM file
## 6690841 sequences exported
## 6690841 reads; of these:
##   6690841 (100.00%) were unpaired; of these:
##     3137072 (46.89%) aligned 0 times
##     2496297 (37.31%) aligned exactly 1 time
##     1057472 (15.80%) aligned &gt;1 times
## 53.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1057472/3553769
## 2 744088/1057472
## 3 4521339/11844818
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0169-TP_1 and prad-0169-TP_2
## Single-end reads will be in prad-0169-TP_M
## 6016892 sequences in the BAM file
## 6016892 sequences exported
## 6016892 reads; of these:
##   6016892 (100.00%) were unpaired; of these:
##     1757560 (29.21%) aligned 0 times
##     2835853 (47.13%) aligned exactly 1 time
##     1423479 (23.66%) aligned &gt;1 times
## 70.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1423479/4259332
## 2 1011421/1423479
## 3 9063181/19648701
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0170-TP_1 and prad-0170-TP_2
## Single-end reads will be in prad-0170-TP_M
## 24536181 sequences in the BAM file
## 24536181 sequences exported
## 24536181 reads; of these:
##   24536181 (100.00%) were unpaired; of these:
##     5526093 (22.52%) aligned 0 times
##     15643576 (63.76%) aligned exactly 1 time
##     3366512 (13.72%) aligned &gt;1 times
## 77.48% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3366512/19010088
## 2 2696039/3366512
## 3 13902867/37718763
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0171-TP_1 and prad-0171-TP_2
## Single-end reads will be in prad-0171-TP_M
## 21996514 sequences in the BAM file
## 21996514 sequences exported
## 21996514 reads; of these:
##   21996514 (100.00%) were unpaired; of these:
##     9499322 (43.19%) aligned 0 times
##     9706089 (44.13%) aligned exactly 1 time
##     2791103 (12.69%) aligned &gt;1 times
## 56.81% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2791103/12497192
## 2 2129966/2791103
## 3 7884736/24334683
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0172-TP_1 and prad-0172-TP_2
## Single-end reads will be in prad-0172-TP_M
## 17587683 sequences in the BAM file
## 17587683 sequences exported
## 17587683 reads; of these:
##   17587683 (100.00%) were unpaired; of these:
##     4737301 (26.94%) aligned 0 times
##     9894744 (56.26%) aligned exactly 1 time
##     2955638 (16.81%) aligned &gt;1 times
## 73.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2955638/12850382
## 2 2361160/2955638
## 3 10282091/26545283
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0173-TP_1 and prad-0173-TP_2
## Single-end reads will be in prad-0173-TP_M
## 13476150 sequences in the BAM file
## 13476150 sequences exported
## 13476150 reads; of these:
##   13476150 (100.00%) were unpaired; of these:
##     4960784 (36.81%) aligned 0 times
##     6336337 (47.02%) aligned exactly 1 time
##     2179029 (16.17%) aligned &gt;1 times
## 63.19% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2179029/8515366
## 2 1638184/2179029
## 3 5840605/17923273
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0174-TP_1 and prad-0174-TP_2
## Single-end reads will be in prad-0174-TP_M
## 23003618 sequences in the BAM file
## 23003618 sequences exported
## 23003618 reads; of these:
##   23003618 (100.00%) were unpaired; of these:
##     6629506 (28.82%) aligned 0 times
##     12951794 (56.30%) aligned exactly 1 time
##     3422318 (14.88%) aligned &gt;1 times
## 71.18% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3422318/16374112
## 2 2581739/3422318
## 3 8602515/29407353
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0175-TP_1 and prad-0175-TP_2
## Single-end reads will be in prad-0175-TP_M
## 9619349 sequences in the BAM file
## 9619349 sequences exported
## 9619349 reads; of these:
##   9619349 (100.00%) were unpaired; of these:
##     3127395 (32.51%) aligned 0 times
##     5068272 (52.69%) aligned exactly 1 time
##     1423682 (14.80%) aligned &gt;1 times
## 67.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1423682/6491954
## 2 1109613/1423682
## 3 6690256/15810264
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0176-TP_1 and prad-0176-TP_2
## Single-end reads will be in prad-0176-TP_M
## 13129207 sequences in the BAM file
## 13129207 sequences exported
## 13129207 reads; of these:
##   13129207 (100.00%) were unpaired; of these:
##     4068074 (30.98%) aligned 0 times
##     7015179 (53.43%) aligned exactly 1 time
##     2045954 (15.58%) aligned &gt;1 times
## 69.02% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2045954/9061133
## 2 1574226/2045954
## 3 9521109/22481807
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0177-TP_1 and prad-0177-TP_2
## Single-end reads will be in prad-0177-TP_M
## 12190935 sequences in the BAM file
## 12190935 sequences exported
## 12190935 reads; of these:
##   12190935 (100.00%) were unpaired; of these:
##     3782279 (31.03%) aligned 0 times
##     6301470 (51.69%) aligned exactly 1 time
##     2107186 (17.28%) aligned &gt;1 times
## 68.97% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2107186/8408656
## 2 1647157/2107186
## 3 11536130/26209009
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0178-TP_1 and prad-0178-TP_2
## Single-end reads will be in prad-0178-TP_M
## 14866342 sequences in the BAM file
## 14866342 sequences exported
## 14866342 reads; of these:
##   14866342 (100.00%) were unpaired; of these:
##     5093039 (34.26%) aligned 0 times
##     7846756 (52.78%) aligned exactly 1 time
##     1926547 (12.96%) aligned &gt;1 times
## 65.74% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1926547/9773303
## 2 1517864/1926547
## 3 6899205/19344662
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0179-TP_1 and prad-0179-TP_2
## Single-end reads will be in prad-0179-TP_M
## 9146575 sequences in the BAM file
## 9146575 sequences exported
## 9146575 reads; of these:
##   9146575 (100.00%) were unpaired; of these:
##     1997018 (21.83%) aligned 0 times
##     5405575 (59.10%) aligned exactly 1 time
##     1743982 (19.07%) aligned &gt;1 times
## 78.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1743982/7149557
## 2 1282190/1743982
## 3 4675576/14744841
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0180-TP_1 and prad-0180-TP_2
## Single-end reads will be in prad-0180-TP_M
## 12332106 sequences in the BAM file
## 12332106 sequences exported
## 12332106 reads; of these:
##   12332106 (100.00%) were unpaired; of these:
##     3944989 (31.99%) aligned 0 times
##     6557503 (53.17%) aligned exactly 1 time
##     1829614 (14.84%) aligned &gt;1 times
## 68.01% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1829614/8387117
## 2 1449750/1829614
## 3 7547458/19546003
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0181-TP_1 and prad-0181-TP_2
## Single-end reads will be in prad-0181-TP_M
## 17811266 sequences in the BAM file
## 17811266 sequences exported
## 17811266 reads; of these:
##   17811266 (100.00%) were unpaired; of these:
##     6261547 (35.15%) aligned 0 times
##     8019065 (45.02%) aligned exactly 1 time
##     3530654 (19.82%) aligned &gt;1 times
## 64.85% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3530654/11549719
## 2 2557554/3530654
## 3 10143361/28246795
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0182-TP_1 and prad-0182-TP_2
## Single-end reads will be in prad-0182-TP_M
## 12393900 sequences in the BAM file
## 12393900 sequences exported
## 12393900 reads; of these:
##   12393900 (100.00%) were unpaired; of these:
##     3781599 (30.51%) aligned 0 times
##     6776505 (54.68%) aligned exactly 1 time
##     1835796 (14.81%) aligned &gt;1 times
## 69.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1835796/8612301
## 2 1484286/1835796
## 3 10433383/22404182
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0183-TP_1 and prad-0183-TP_2
## Single-end reads will be in prad-0183-TP_M
## 10945807 sequences in the BAM file
## 10945807 sequences exported
## 10945807 reads; of these:
##   10945807 (100.00%) were unpaired; of these:
##     3476790 (31.76%) aligned 0 times
##     5882150 (53.74%) aligned exactly 1 time
##     1586867 (14.50%) aligned &gt;1 times
## 68.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1586867/7469017
## 2 1246270/1586867
## 3 6566457/16644659
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0184-TP_1 and prad-0184-TP_2
## Single-end reads will be in prad-0184-TP_M
## 11218948 sequences in the BAM file
## 11218948 sequences exported
## 11218948 reads; of these:
##   11218948 (100.00%) were unpaired; of these:
##     3523506 (31.41%) aligned 0 times
##     6091414 (54.30%) aligned exactly 1 time
##     1604028 (14.30%) aligned &gt;1 times
## 68.59% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1604028/7695442
## 2 1245182/1604028
## 3 6077351/16199928
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0185-TP_1 and prad-0185-TP_2
## Single-end reads will be in prad-0185-TP_M
## 16104257 sequences in the BAM file
## 16104257 sequences exported
## 16104257 reads; of these:
##   16104257 (100.00%) were unpaired; of these:
##     5141156 (31.92%) aligned 0 times
##     8360689 (51.92%) aligned exactly 1 time
##     2602412 (16.16%) aligned &gt;1 times
## 68.08% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2602412/10963101
## 2 1996881/2602412
## 3 15393006/33268680
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0186-TP_1 and prad-0186-TP_2
## Single-end reads will be in prad-0186-TP_M
## 12556483 sequences in the BAM file
## 12556483 sequences exported
## 12556483 reads; of these:
##   12556483 (100.00%) were unpaired; of these:
##     3951210 (31.47%) aligned 0 times
##     6865601 (54.68%) aligned exactly 1 time
##     1739672 (13.85%) aligned &gt;1 times
## 68.53% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1739672/8605273
## 2 1425971/1739672
## 3 7331918/18467181
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0187-TP_1 and prad-0187-TP_2
## Single-end reads will be in prad-0187-TP_M
## 14265011 sequences in the BAM file
## 14265011 sequences exported
## 14265011 reads; of these:
##   14265011 (100.00%) were unpaired; of these:
##     4029125 (28.24%) aligned 0 times
##     8316148 (58.30%) aligned exactly 1 time
##     1919738 (13.46%) aligned &gt;1 times
## 71.76% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1919738/10235886
## 2 1392444/1919738
## 3 6595928/20182555
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0188-TP_1 and prad-0188-TP_2
## Single-end reads will be in prad-0188-TP_M
## 9615895 sequences in the BAM file
## 9615895 sequences exported
## 9615895 reads; of these:
##   9615895 (100.00%) were unpaired; of these:
##     2891633 (30.07%) aligned 0 times
##     4869890 (50.64%) aligned exactly 1 time
##     1854372 (19.28%) aligned &gt;1 times
## 69.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1854372/6724262
## 2 1421393/1854372
## 3 11844965/24256841
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0189-TP_1 and prad-0189-TP_2
## Single-end reads will be in prad-0189-TP_M
## 17558789 sequences in the BAM file
## 17558789 sequences exported
## 17558789 reads; of these:
##   17558789 (100.00%) were unpaired; of these:
##     4875131 (27.76%) aligned 0 times
##     9171864 (52.24%) aligned exactly 1 time
##     3511794 (20.00%) aligned &gt;1 times
## 72.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3511794/12683658
## 2 2595989/3511794
## 3 8776983/26913592
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0190-TP_1 and prad-0190-TP_2
## Single-end reads will be in prad-0190-TP_M
## 13075033 sequences in the BAM file
## 13075033 sequences exported
## 13075033 reads; of these:
##   13075033 (100.00%) were unpaired; of these:
##     3491299 (26.70%) aligned 0 times
##     6729266 (51.47%) aligned exactly 1 time
##     2854468 (21.83%) aligned &gt;1 times
## 73.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2854468/9583734
## 2 2202203/2854468
## 3 13477843/32599711
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0191-TP_1 and prad-0191-TP_2
## Single-end reads will be in prad-0191-TP_M
## 22425468 sequences in the BAM file
## 22425468 sequences exported
## 22425468 reads; of these:
##   22425468 (100.00%) were unpaired; of these:
##     7572352 (33.77%) aligned 0 times
##     12510119 (55.79%) aligned exactly 1 time
##     2342997 (10.45%) aligned &gt;1 times
## 66.23% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2342997/14853116
## 2 1867966/2342997
## 3 9761617/27585582
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0192-TP_1 and prad-0192-TP_2
## Single-end reads will be in prad-0192-TP_M
## 8477746 sequences in the BAM file
## 8477746 sequences exported
## 8477746 reads; of these:
##   8477746 (100.00%) were unpaired; of these:
##     4000223 (47.18%) aligned 0 times
##     3412210 (40.25%) aligned exactly 1 time
##     1065313 (12.57%) aligned &gt;1 times
## 52.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1065313/4477523
## 2 764186/1065313
## 3 3077513/9531577
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0193-TP_1 and prad-0193-TP_2
## Single-end reads will be in prad-0193-TP_M
## 12557377 sequences in the BAM file
## 12557377 sequences exported
## 12557377 reads; of these:
##   12557377 (100.00%) were unpaired; of these:
##     4305543 (34.29%) aligned 0 times
##     6205713 (49.42%) aligned exactly 1 time
##     2046121 (16.29%) aligned &gt;1 times
## 65.71% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2046121/8251834
## 2 1404703/2046121
## 3 5821712/17791341
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0194-TP_1 and prad-0194-TP_2
## Single-end reads will be in prad-0194-TP_M
## 5987183 sequences in the BAM file
## 5987183 sequences exported
## 5987183 reads; of these:
##   5987183 (100.00%) were unpaired; of these:
##     1955568 (32.66%) aligned 0 times
##     3168314 (52.92%) aligned exactly 1 time
##     863301 (14.42%) aligned &gt;1 times
## 67.34% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 863301/4031615
## 2 653252/863301
## 3 3928689/9777070
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0195-TP_1 and prad-0195-TP_2
## Single-end reads will be in prad-0195-TP_M
## 5100393 sequences in the BAM file
## 5100393 sequences exported
## 5100393 reads; of these:
##   5100393 (100.00%) were unpaired; of these:
##     1851902 (36.31%) aligned 0 times
##     2297465 (45.04%) aligned exactly 1 time
##     951026 (18.65%) aligned &gt;1 times
## 63.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 951026/3248491
## 2 736933/951026
## 3 5151265/11257504
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0196-TP_1 and prad-0196-TP_2
## Single-end reads will be in prad-0196-TP_M
## 11914501 sequences in the BAM file
## 11914501 sequences exported
## 11914501 reads; of these:
##   11914501 (100.00%) were unpaired; of these:
##     3655317 (30.68%) aligned 0 times
##     6014477 (50.48%) aligned exactly 1 time
##     2244707 (18.84%) aligned &gt;1 times
## 69.32% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2244707/8259184
## 2 1716857/2244707
## 3 8304102/22078096
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0197-TP_1 and prad-0197-TP_2
## Single-end reads will be in prad-0197-TP_M
## 14322435 sequences in the BAM file
## 14322435 sequences exported
## 14322435 reads; of these:
##   14322435 (100.00%) were unpaired; of these:
##     4607982 (32.17%) aligned 0 times
##     8031704 (56.08%) aligned exactly 1 time
##     1682749 (11.75%) aligned &gt;1 times
## 67.83% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1682749/9714453
## 2 1417967/1682749
## 3 11413984/23690549
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0198-TP_1 and prad-0198-TP_2
## Single-end reads will be in prad-0198-TP_M
## 17760159 sequences in the BAM file
## 17760159 sequences exported
## 17760159 reads; of these:
##   17760159 (100.00%) were unpaired; of these:
##     8736232 (49.19%) aligned 0 times
##     6972032 (39.26%) aligned exactly 1 time
##     2051895 (11.55%) aligned &gt;1 times
## 50.81% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2051895/9023927
## 2 1510695/2051895
## 3 6709233/19401922
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0199-TP_1 and prad-0199-TP_2
## Single-end reads will be in prad-0199-TP_M
## 24032814 sequences in the BAM file
## 24032814 sequences exported
## 24032814 reads; of these:
##   24032814 (100.00%) were unpaired; of these:
##     9337234 (38.85%) aligned 0 times
##     12544647 (52.20%) aligned exactly 1 time
##     2150933 (8.95%) aligned &gt;1 times
## 61.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2150933/14695580
## 2 1743850/2150933
## 3 8084368/25559758
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0200-TP_1 and prad-0200-TP_2
## Single-end reads will be in prad-0200-TP_M
## 13632643 sequences in the BAM file
## 13632643 sequences exported
## 13632643 reads; of these:
##   13632643 (100.00%) were unpaired; of these:
##     4806977 (35.26%) aligned 0 times
##     7162385 (52.54%) aligned exactly 1 time
##     1663281 (12.20%) aligned &gt;1 times
## 64.74% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1663281/8825666
## 2 1336176/1663281
## 3 5660010/16575422
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0201-TP_1 and prad-0201-TP_2
## Single-end reads will be in prad-0201-TP_M
## 11613176 sequences in the BAM file
## 11613176 sequences exported
## 11613176 reads; of these:
##   11613176 (100.00%) were unpaired; of these:
##     5233168 (45.06%) aligned 0 times
##     4685226 (40.34%) aligned exactly 1 time
##     1694782 (14.59%) aligned &gt;1 times
## 54.94% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1694782/6380008
## 2 1281766/1694782
## 3 4528557/13663225
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0202-TP_1 and prad-0202-TP_2
## Single-end reads will be in prad-0202-TP_M
## 11277485 sequences in the BAM file
## 11277485 sequences exported
## 11277485 reads; of these:
##   11277485 (100.00%) were unpaired; of these:
##     4028776 (35.72%) aligned 0 times
##     5742885 (50.92%) aligned exactly 1 time
##     1505824 (13.35%) aligned &gt;1 times
## 64.28% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1505824/7248709
## 2 1151329/1505824
## 3 8550990/18538691
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0203-TP_1 and prad-0203-TP_2
## Single-end reads will be in prad-0203-TP_M
## 5471915 sequences in the BAM file
## 5471915 sequences exported
## 5471915 reads; of these:
##   5471915 (100.00%) were unpaired; of these:
##     1853474 (33.87%) aligned 0 times
##     2654347 (48.51%) aligned exactly 1 time
##     964094 (17.62%) aligned &gt;1 times
## 66.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 964094/3618441
## 2 701593/964094
## 3 2454440/7611742
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0204-TP_1 and prad-0204-TP_2
## Single-end reads will be in prad-0204-TP_M
## 9571332 sequences in the BAM file
## 9571332 sequences exported
## 9571332 reads; of these:
##   9571332 (100.00%) were unpaired; of these:
##     2830912 (29.58%) aligned 0 times
##     5219726 (54.54%) aligned exactly 1 time
##     1520694 (15.89%) aligned &gt;1 times
## 70.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1520694/6740420
## 2 1187035/1520694
## 3 6609676/16145959
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0205-TP_1 and prad-0205-TP_2
## Single-end reads will be in prad-0205-TP_M
## 9884992 sequences in the BAM file
## 9884992 sequences exported
## 9884992 reads; of these:
##   9884992 (100.00%) were unpaired; of these:
##     3492484 (35.33%) aligned 0 times
##     4937136 (49.95%) aligned exactly 1 time
##     1455372 (14.72%) aligned &gt;1 times
## 64.67% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1455372/6392508
## 2 1137251/1455372
## 3 5205942/13922780
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0206-TP_1 and prad-0206-TP_2
## Single-end reads will be in prad-0206-TP_M
## 11240816 sequences in the BAM file
## 11240816 sequences exported
## 11240816 reads; of these:
##   11240816 (100.00%) were unpaired; of these:
##     3908144 (34.77%) aligned 0 times
##     4443211 (39.53%) aligned exactly 1 time
##     2889461 (25.71%) aligned &gt;1 times
## 65.23% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2889461/7332672
## 2 2275405/2889461
## 3 15993677/36822122
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0207-TP_1 and prad-0207-TP_2
## Single-end reads will be in prad-0207-TP_M
## 11546613 sequences in the BAM file
## 11546613 sequences exported
## 11546613 reads; of these:
##   11546613 (100.00%) were unpaired; of these:
##     3599763 (31.18%) aligned 0 times
##     6184777 (53.56%) aligned exactly 1 time
##     1762073 (15.26%) aligned &gt;1 times
## 68.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1762073/7946850
## 2 1373811/1762073
## 3 9000749/22205125
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0208-TP_1 and prad-0208-TP_2
## Single-end reads will be in prad-0208-TP_M
## 10773889 sequences in the BAM file
## 10773889 sequences exported
## 10773889 reads; of these:
##   10773889 (100.00%) were unpaired; of these:
##     3651078 (33.89%) aligned 0 times
##     5706401 (52.97%) aligned exactly 1 time
##     1416410 (13.15%) aligned &gt;1 times
## 66.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1416410/7122811
## 2 1096261/1416410
## 3 6139886/15351815
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0209-TP_1 and prad-0209-TP_2
## Single-end reads will be in prad-0209-TP_M
## 15150379 sequences in the BAM file
## 15150379 sequences exported
## 15150379 reads; of these:
##   15150379 (100.00%) were unpaired; of these:
##     5067745 (33.45%) aligned 0 times
##     7703193 (50.84%) aligned exactly 1 time
##     2379441 (15.71%) aligned &gt;1 times
## 66.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2379441/10082634
## 2 1730872/2379441
## 3 6109309/19662682
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0210-TP_1 and prad-0210-TP_2
## Single-end reads will be in prad-0210-TP_M
## 11480506 sequences in the BAM file
## 11480506 sequences exported
## 11480506 reads; of these:
##   11480506 (100.00%) were unpaired; of these:
##     3742645 (32.60%) aligned 0 times
##     5555585 (48.39%) aligned exactly 1 time
##     2182276 (19.01%) aligned &gt;1 times
## 67.40% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2182276/7737861
## 2 1647667/2182276
## 3 14329690/30456005
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0211-TP_1 and prad-0211-TP_2
## Single-end reads will be in prad-0211-TP_M
## 11217132 sequences in the BAM file
## 11217132 sequences exported
## 11217132 reads; of these:
##   11217132 (100.00%) were unpaired; of these:
##     3415709 (30.45%) aligned 0 times
##     5884541 (52.46%) aligned exactly 1 time
##     1916882 (17.09%) aligned &gt;1 times
## 69.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1916882/7801423
## 2 1493384/1916882
## 3 11162084/24713642
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0212-TP_1 and prad-0212-TP_2
## Single-end reads will be in prad-0212-TP_M
## 5772126 sequences in the BAM file
## 5772126 sequences exported
## 5772126 reads; of these:
##   5772126 (100.00%) were unpaired; of these:
##     2018171 (34.96%) aligned 0 times
##     2784805 (48.25%) aligned exactly 1 time
##     969150 (16.79%) aligned &gt;1 times
## 65.04% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 969150/3753955
## 2 741101/969150
## 3 2895122/7828826
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0213-TP_1 and prad-0213-TP_2
## Single-end reads will be in prad-0213-TP_M
## 16445356 sequences in the BAM file
## 16445356 sequences exported
## 16445356 reads; of these:
##   16445356 (100.00%) were unpaired; of these:
##     7438054 (45.23%) aligned 0 times
##     6919654 (42.08%) aligned exactly 1 time
##     2087648 (12.69%) aligned &gt;1 times
## 54.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2087648/9007302
## 2 1518143/2087648
## 3 6017353/18722127
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0214-TP_1 and prad-0214-TP_2
## Single-end reads will be in prad-0214-TP_M
## 10533726 sequences in the BAM file
## 10533726 sequences exported
## 10533726 reads; of these:
##   10533726 (100.00%) were unpaired; of these:
##     3222139 (30.59%) aligned 0 times
##     5466763 (51.90%) aligned exactly 1 time
##     1844824 (17.51%) aligned &gt;1 times
## 69.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1844824/7311587
## 2 1418374/1844824
## 3 9107406/20432898
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0215-TP_1 and prad-0215-TP_2
## Single-end reads will be in prad-0215-TP_M
## 20344488 sequences in the BAM file
## 20344488 sequences exported
## 20344488 reads; of these:
##   20344488 (100.00%) were unpaired; of these:
##     8236957 (40.49%) aligned 0 times
##     9530202 (46.84%) aligned exactly 1 time
##     2577329 (12.67%) aligned &gt;1 times
## 59.51% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2577329/12107531
## 2 2023363/2577329
## 3 10591461/28448868
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0216-TP_1 and prad-0216-TP_2
## Single-end reads will be in prad-0216-TP_M
## 9087271 sequences in the BAM file
## 9087271 sequences exported
## 9087271 reads; of these:
##   9087271 (100.00%) were unpaired; of these:
##     2788923 (30.69%) aligned 0 times
##     4689659 (51.61%) aligned exactly 1 time
##     1608689 (17.70%) aligned &gt;1 times
## 69.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1608689/6298348
## 2 1339037/1608689
## 3 7694206/16421971
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0217-TP_1 and prad-0217-TP_2
## Single-end reads will be in prad-0217-TP_M
## 10744087 sequences in the BAM file
## 10744087 sequences exported
## 10744087 reads; of these:
##   10744087 (100.00%) were unpaired; of these:
##     3353086 (31.21%) aligned 0 times
##     5914778 (55.05%) aligned exactly 1 time
##     1476223 (13.74%) aligned &gt;1 times
## 68.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1476223/7391001
## 2 1214773/1476223
## 3 6412356/16364345
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0218-TP_1 and prad-0218-TP_2
## Single-end reads will be in prad-0218-TP_M
## 18259531 sequences in the BAM file
## 18259531 sequences exported
## 18259531 reads; of these:
##   18259531 (100.00%) were unpaired; of these:
##     5476698 (29.99%) aligned 0 times
##     10680780 (58.49%) aligned exactly 1 time
##     2102053 (11.51%) aligned &gt;1 times
## 70.01% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2102053/12782833
## 2 1654856/2102053
## 3 7883159/22940100
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0219-TP_1 and prad-0219-TP_2
## Single-end reads will be in prad-0219-TP_M
## 22470875 sequences in the BAM file
## 22470875 sequences exported
## 22470875 reads; of these:
##   22470875 (100.00%) were unpaired; of these:
##     9379887 (41.74%) aligned 0 times
##     10085706 (44.88%) aligned exactly 1 time
##     3005282 (13.37%) aligned &gt;1 times
## 58.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3005282/13090988
## 2 2297281/3005282
## 3 10528877/30195696
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in prad-0220-TP_1 and prad-0220-TP_2
## Single-end reads will be in prad-0220-TP_M
## 32982252 sequences in the BAM file
## 32982252 sequences exported
## 32982252 reads; of these:
##   32982252 (100.00%) were unpaired; of these:
##     11633398 (35.27%) aligned 0 times
##     17909874 (54.30%) aligned exactly 1 time
##     3438980 (10.43%) aligned &gt;1 times
## 64.73% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3438980/21348854
## 2 2466706/3438980
## 3 8816755/34602561
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in prad-0221-TP_1 and prad-0221-TP_2
## Single-end reads will be in prad-0221-TP_M
## 9168077 sequences in the BAM file
## 9168077 sequences exported
## 9168077 reads; of these:
##   9168077 (100.00%) were unpaired; of these:
##     3525449 (38.45%) aligned 0 times
##     4441173 (48.44%) aligned exactly 1 time
##     1201455 (13.10%) aligned &gt;1 times
## 61.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1201455/5642628
## 2 902829/1201455
## 3 4125548/11487575
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0222-TP_1 and prad-0222-TP_2
## Single-end reads will be in prad-0222-TP_M
## 15786359 sequences in the BAM file
## 15786359 sequences exported
## 15786359 reads; of these:
##   15786359 (100.00%) were unpaired; of these:
##     5389261 (34.14%) aligned 0 times
##     8265755 (52.36%) aligned exactly 1 time
##     2131343 (13.50%) aligned &gt;1 times
## 65.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2131343/10397098
## 2 1733352/2131343
## 3 7690001/21332174
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0223-TP_1 and prad-0223-TP_2
## Single-end reads will be in prad-0223-TP_M
## 5103476 sequences in the BAM file
## 5103476 sequences exported
## 5103476 reads; of these:
##   5103476 (100.00%) were unpaired; of these:
##     1690909 (33.13%) aligned 0 times
##     2663298 (52.19%) aligned exactly 1 time
##     749269 (14.68%) aligned &gt;1 times
## 66.87% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 749269/3412567
## 2 626348/749269
## 3 3689577/8152166
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0224-TP_1 and prad-0224-TP_2
## Single-end reads will be in prad-0224-TP_M
## 14671539 sequences in the BAM file
## 14671539 sequences exported
## 14671539 reads; of these:
##   14671539 (100.00%) were unpaired; of these:
##     6753516 (46.03%) aligned 0 times
##     6100317 (41.58%) aligned exactly 1 time
##     1817706 (12.39%) aligned &gt;1 times
## 53.97% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1817706/7918023
## 2 1250982/1817706
## 3 4331506/14654826
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0225-TP_1 and prad-0225-TP_2
## Single-end reads will be in prad-0225-TP_M
## 17298679 sequences in the BAM file
## 17298679 sequences exported
## 17298679 reads; of these:
##   17298679 (100.00%) were unpaired; of these:
##     2871367 (16.60%) aligned 0 times
##     10785014 (62.35%) aligned exactly 1 time
##     3642298 (21.06%) aligned &gt;1 times
## 83.40% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3642298/14427312
## 2 2717118/3642298
## 3 13807960/37634716
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 2.
## Output will be in prad-0226-TP_1 and prad-0226-TP_2
## Single-end reads will be in prad-0226-TP_M
## 6819210 sequences in the BAM file
## 6819210 sequences exported
## 6819210 reads; of these:
##   6819210 (100.00%) were unpaired; of these:
##     2326280 (34.11%) aligned 0 times
##     2948650 (43.24%) aligned exactly 1 time
##     1544280 (22.65%) aligned &gt;1 times
## 65.89% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1544280/4492930
## 2 1200342/1544280
## 3 10798235/21665306
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0227-TP_1 and prad-0227-TP_2
## Single-end reads will be in prad-0227-TP_M
## 16368128 sequences in the BAM file
## 16368128 sequences exported
## 16368128 reads; of these:
##   16368128 (100.00%) were unpaired; of these:
##     3201705 (19.56%) aligned 0 times
##     10050445 (61.40%) aligned exactly 1 time
##     3115978 (19.04%) aligned &gt;1 times
## 80.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3115978/13166423
## 2 2518766/3115978
## 3 13045720/31348727
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0228-TP_1 and prad-0228-TP_2
## Single-end reads will be in prad-0228-TP_M
## 9305545 sequences in the BAM file
## 9305545 sequences exported
## 9305545 reads; of these:
##   9305545 (100.00%) were unpaired; of these:
##     3431959 (36.88%) aligned 0 times
##     4811484 (51.71%) aligned exactly 1 time
##     1062102 (11.41%) aligned &gt;1 times
## 63.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1062102/5873586
## 2 826274/1062102
## 3 4300473/11619218
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0229-TP_1 and prad-0229-TP_2
## Single-end reads will be in prad-0229-TP_M
## 12955704 sequences in the BAM file
## 12955704 sequences exported
## 12955704 reads; of these:
##   12955704 (100.00%) were unpaired; of these:
##     2724064 (21.03%) aligned 0 times
##     6897061 (53.24%) aligned exactly 1 time
##     3334579 (25.74%) aligned &gt;1 times
## 78.97% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3334579/10231640
## 2 2630654/3334579
## 3 16434664/35576968
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0230-TP_1 and prad-0230-TP_2
## Single-end reads will be in prad-0230-TP_M
## 21819185 sequences in the BAM file
## 21819185 sequences exported
## 21819185 reads; of these:
##   21819185 (100.00%) were unpaired; of these:
##     6190497 (28.37%) aligned 0 times
##     12804520 (58.68%) aligned exactly 1 time
##     2824168 (12.94%) aligned &gt;1 times
## 71.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2824168/15628688
## 2 2034773/2824168
## 3 9589567/29795077
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0231-TP_1 and prad-0231-TP_2
## Single-end reads will be in prad-0231-TP_M
## 30837212 sequences in the BAM file
## 30837212 sequences exported
## 30837212 reads; of these:
##   30837212 (100.00%) were unpaired; of these:
##     8885848 (28.82%) aligned 0 times
##     17037843 (55.25%) aligned exactly 1 time
##     4913521 (15.93%) aligned &gt;1 times
## 71.18% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 4913521/21951364
## 2 3652740/4913521
## 3 19719422/55091907
## [bam_sort_core] merging from 11 files...
## This looks like paired data from lane 4.
## Output will be in prad-0232-TP_1 and prad-0232-TP_2
## Single-end reads will be in prad-0232-TP_M
## 23109471 sequences in the BAM file
## 23109471 sequences exported
## 23109471 reads; of these:
##   23109471 (100.00%) were unpaired; of these:
##     6138646 (26.56%) aligned 0 times
##     13063390 (56.53%) aligned exactly 1 time
##     3907435 (16.91%) aligned &gt;1 times
## 73.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3907435/16970825
## 2 3185090/3907435
## 3 15690440/39051524
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 3.
## Output will be in prad-0233-TP_1 and prad-0233-TP_2
## Single-end reads will be in prad-0233-TP_M
## 40570254 sequences in the BAM file
## 40570254 sequences exported
## 40570254 reads; of these:
##   40570254 (100.00%) were unpaired; of these:
##     11453422 (28.23%) aligned 0 times
##     16661713 (41.07%) aligned exactly 1 time
##     12455119 (30.70%) aligned &gt;1 times
## 71.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 12455119/29116832
## 2 8570471/12455119
## 3 43820238/102120510
## [bam_sort_core] merging from 17 files...
## This looks like paired data from lane 2.
## Output will be in prad-0234-TP_1 and prad-0234-TP_2
## Single-end reads will be in prad-0234-TP_M
## 8586497 sequences in the BAM file
## 8586497 sequences exported
## 8586497 reads; of these:
##   8586497 (100.00%) were unpaired; of these:
##     2564599 (29.87%) aligned 0 times
##     4284429 (49.90%) aligned exactly 1 time
##     1737469 (20.23%) aligned &gt;1 times
## 70.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1737469/6021898
## 2 1323487/1737469
## 3 11652883/24124987
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0235-TP_1 and prad-0235-TP_2
## Single-end reads will be in prad-0235-TP_M
## 12642954 sequences in the BAM file
## 12642954 sequences exported
## 12642954 reads; of these:
##   12642954 (100.00%) were unpaired; of these:
##     2424660 (19.18%) aligned 0 times
##     7871392 (62.26%) aligned exactly 1 time
##     2346902 (18.56%) aligned &gt;1 times
## 80.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2346902/10218294
## 2 1848378/2346902
## 3 9941318/25613608
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0236-TP_1 and prad-0236-TP_2
## Single-end reads will be in prad-0236-TP_M
## 8764650 sequences in the BAM file
## 8764650 sequences exported
## 8764650 reads; of these:
##   8764650 (100.00%) were unpaired; of these:
##     3131746 (35.73%) aligned 0 times
##     4101939 (46.80%) aligned exactly 1 time
##     1530965 (17.47%) aligned &gt;1 times
## 64.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1530965/5632904
## 2 1192930/1530965
## 3 8294040/17813708
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0237-TP_1 and prad-0237-TP_2
## Single-end reads will be in prad-0237-TP_M
## 14269813 sequences in the BAM file
## 14269813 sequences exported
## 14269813 reads; of these:
##   14269813 (100.00%) were unpaired; of these:
##     6483755 (45.44%) aligned 0 times
##     5975911 (41.88%) aligned exactly 1 time
##     1810147 (12.69%) aligned &gt;1 times
## 54.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1810147/7786058
## 2 1389800/1810147
## 3 4173746/14114766
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0238-TP_1 and prad-0238-TP_2
## Single-end reads will be in prad-0238-TP_M
## 8224573 sequences in the BAM file
## 8224573 sequences exported
## 8224573 reads; of these:
##   8224573 (100.00%) were unpaired; of these:
##     2470540 (30.04%) aligned 0 times
##     4164210 (50.63%) aligned exactly 1 time
##     1589823 (19.33%) aligned &gt;1 times
## 69.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1589823/5754033
## 2 1235102/1589823
## 3 8687677/18984923
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0239-TP_1 and prad-0239-TP_2
## Single-end reads will be in prad-0239-TP_M
## 7488986 sequences in the BAM file
## 7488986 sequences exported
## 7488986 reads; of these:
##   7488986 (100.00%) were unpaired; of these:
##     3000807 (40.07%) aligned 0 times
##     3401023 (45.41%) aligned exactly 1 time
##     1087156 (14.52%) aligned &gt;1 times
## 59.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1087156/4488179
## 2 859972/1087156
## 3 3383921/9962790
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0240-TP_1 and prad-0240-TP_2
## Single-end reads will be in prad-0240-TP_M
## 11554355 sequences in the BAM file
## 11554355 sequences exported
## 11554355 reads; of these:
##   11554355 (100.00%) were unpaired; of these:
##     4031795 (34.89%) aligned 0 times
##     6232863 (53.94%) aligned exactly 1 time
##     1289697 (11.16%) aligned &gt;1 times
## 65.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1289697/7522560
## 2 1010710/1289697
## 3 3810096/13297612
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0241-TP_1 and prad-0241-TP_2
## Single-end reads will be in prad-0241-TP_M
## 5721718 sequences in the BAM file
## 5721718 sequences exported
## 5721718 reads; of these:
##   5721718 (100.00%) were unpaired; of these:
##     2450668 (42.83%) aligned 0 times
##     2229327 (38.96%) aligned exactly 1 time
##     1041723 (18.21%) aligned &gt;1 times
## 57.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1041723/3271050
## 2 763178/1041723
## 3 5200956/12176151
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0242-TP_1 and prad-0242-TP_2
## Single-end reads will be in prad-0242-TP_M
## 45766993 sequences in the BAM file
## 45766993 sequences exported
## 45766993 reads; of these:
##   45766993 (100.00%) were unpaired; of these:
##     13256471 (28.97%) aligned 0 times
##     26084335 (56.99%) aligned exactly 1 time
##     6426187 (14.04%) aligned &gt;1 times
## 71.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 6426187/32510522
## 2 5191223/6426187
## 3 24784734/65227729
## [bam_sort_core] merging from 12 files...
## This looks like paired data from lane 7.
## Output will be in prad-0243-TP_1 and prad-0243-TP_2
## Single-end reads will be in prad-0243-TP_M
## 14845718 sequences in the BAM file
## 14845718 sequences exported
## 14845718 reads; of these:
##   14845718 (100.00%) were unpaired; of these:
##     3056488 (20.59%) aligned 0 times
##     9617334 (64.78%) aligned exactly 1 time
##     2171896 (14.63%) aligned &gt;1 times
## 79.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2171896/11789230
## 2 1713774/2171896
## 3 9970181/26319854
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0244-TP_1 and prad-0244-TP_2
## Single-end reads will be in prad-0244-TP_M
## 18533482 sequences in the BAM file
## 18533482 sequences exported
## 18533482 reads; of these:
##   18533482 (100.00%) were unpaired; of these:
##     4677265 (25.24%) aligned 0 times
##     12109931 (65.34%) aligned exactly 1 time
##     1746286 (9.42%) aligned &gt;1 times
## 74.76% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1746286/13856217
## 2 1421786/1746286
## 3 10936552/27001989
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0245-TP_1 and prad-0245-TP_2
## Single-end reads will be in prad-0245-TP_M
## 10771561 sequences in the BAM file
## 10771561 sequences exported
## 10771561 reads; of these:
##   10771561 (100.00%) were unpaired; of these:
##     3599086 (33.41%) aligned 0 times
##     5442481 (50.53%) aligned exactly 1 time
##     1729994 (16.06%) aligned &gt;1 times
## 66.59% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1729994/7172475
## 2 1280462/1729994
## 3 4147792/13446324
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0246-TP_1 and prad-0246-TP_2
## Single-end reads will be in prad-0246-TP_M
## 8019440 sequences in the BAM file
## 8019440 sequences exported
## 8019440 reads; of these:
##   8019440 (100.00%) were unpaired; of these:
##     3204058 (39.95%) aligned 0 times
##     3656143 (45.59%) aligned exactly 1 time
##     1159239 (14.46%) aligned &gt;1 times
## 60.05% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1159239/4815382
## 2 883720/1159239
## 3 4330716/12149646
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0247-TP_1 and prad-0247-TP_2
## Single-end reads will be in prad-0247-TP_M
## 11175893 sequences in the BAM file
## 11175893 sequences exported
## 11175893 reads; of these:
##   11175893 (100.00%) were unpaired; of these:
##     4467420 (39.97%) aligned 0 times
##     5263007 (47.09%) aligned exactly 1 time
##     1445466 (12.93%) aligned &gt;1 times
## 60.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1445466/6708473
## 2 1105261/1445466
## 3 3713651/12626381
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0248-TP_1 and prad-0248-TP_2
## Single-end reads will be in prad-0248-TP_M
## 9715136 sequences in the BAM file
## 9715136 sequences exported
## 9715136 reads; of these:
##   9715136 (100.00%) were unpaired; of these:
##     3598180 (37.04%) aligned 0 times
##     4470148 (46.01%) aligned exactly 1 time
##     1646808 (16.95%) aligned &gt;1 times
## 62.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1646808/6116956
## 2 1275800/1646808
## 3 6529520/16199539
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0249-TP_1 and prad-0249-TP_2
## Single-end reads will be in prad-0249-TP_M
## 3785615 sequences in the BAM file
## 3785615 sequences exported
## 3785615 reads; of these:
##   3785615 (100.00%) were unpaired; of these:
##     603201 (15.93%) aligned 0 times
##     2197739 (58.06%) aligned exactly 1 time
##     984675 (26.01%) aligned &gt;1 times
## 84.07% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 984675/3182414
## 2 819159/984675
## 3 2964752/7620876
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0250-TP_1 and prad-0250-TP_2
## Single-end reads will be in prad-0250-TP_M
## 23311448 sequences in the BAM file
## 23311448 sequences exported
## 23311448 reads; of these:
##   23311448 (100.00%) were unpaired; of these:
##     7953151 (34.12%) aligned 0 times
##     12316261 (52.83%) aligned exactly 1 time
##     3042036 (13.05%) aligned &gt;1 times
## 65.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3042036/15358297
## 2 2369756/3042036
## 3 10442486/32362765
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in prad-0251-TP_1 and prad-0251-TP_2
## Single-end reads will be in prad-0251-TP_M
## 15621516 sequences in the BAM file
## 15621516 sequences exported
## 15621516 reads; of these:
##   15621516 (100.00%) were unpaired; of these:
##     5629340 (36.04%) aligned 0 times
##     8566762 (54.84%) aligned exactly 1 time
##     1425414 (9.12%) aligned &gt;1 times
## 63.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1425414/9992176
## 2 1090716/1425414
## 3 4226991/16542036
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0252-TP_1 and prad-0252-TP_2
## Single-end reads will be in prad-0252-TP_M
## 18125537 sequences in the BAM file
## 18125537 sequences exported
## 18125537 reads; of these:
##   18125537 (100.00%) were unpaired; of these:
##     6365369 (35.12%) aligned 0 times
##     9465201 (52.22%) aligned exactly 1 time
##     2294967 (12.66%) aligned &gt;1 times
## 64.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2294967/11760168
## 2 1800232/2294967
## 3 6446830/21460390
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0253-TP_1 and prad-0253-TP_2
## Single-end reads will be in prad-0253-TP_M
## 7271599 sequences in the BAM file
## 7271599 sequences exported
## 7271599 reads; of these:
##   7271599 (100.00%) were unpaired; of these:
##     2764927 (38.02%) aligned 0 times
##     3533251 (48.59%) aligned exactly 1 time
##     973421 (13.39%) aligned &gt;1 times
## 61.98% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 973421/4506672
## 2 700376/973421
## 3 3039356/9301846
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0254-TP_1 and prad-0254-TP_2
## Single-end reads will be in prad-0254-TP_M
## 6355216 sequences in the BAM file
## 6355216 sequences exported
## 6355216 reads; of these:
##   6355216 (100.00%) were unpaired; of these:
##     2929523 (46.10%) aligned 0 times
##     2683793 (42.23%) aligned exactly 1 time
##     741900 (11.67%) aligned &gt;1 times
## 53.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 741900/3425693
## 2 537612/741900
## 3 2061816/6726388
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0255-TP_1 and prad-0255-TP_2
## Single-end reads will be in prad-0255-TP_M
## 17073072 sequences in the BAM file
## 17073072 sequences exported
## 17073072 reads; of these:
##   17073072 (100.00%) were unpaired; of these:
##     5905895 (34.59%) aligned 0 times
##     9092535 (53.26%) aligned exactly 1 time
##     2074642 (12.15%) aligned &gt;1 times
## 65.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2074642/11167177
## 2 1559183/2074642
## 3 5975504/20138187
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0256-TP_1 and prad-0256-TP_2
## Single-end reads will be in prad-0256-TP_M
## 12428504 sequences in the BAM file
## 12428504 sequences exported
## 12428504 reads; of these:
##   12428504 (100.00%) were unpaired; of these:
##     1667483 (13.42%) aligned 0 times
##     7545851 (60.71%) aligned exactly 1 time
##     3215170 (25.87%) aligned &gt;1 times
## 86.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3215170/10761021
## 2 2725832/3215170
## 3 14560417/34651458
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 3.
## Output will be in prad-0257-TP_1 and prad-0257-TP_2
## Single-end reads will be in prad-0257-TP_M
## 17776859 sequences in the BAM file
## 17776859 sequences exported
## 17776859 reads; of these:
##   17776859 (100.00%) were unpaired; of these:
##     5381635 (30.27%) aligned 0 times
##     9670427 (54.40%) aligned exactly 1 time
##     2724797 (15.33%) aligned &gt;1 times
## 69.73% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2724797/12395224
## 2 1957632/2724797
## 3 6664972/22772447
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0258-TP_1 and prad-0258-TP_2
## Single-end reads will be in prad-0258-TP_M
## 11626434 sequences in the BAM file
## 11626434 sequences exported
## 11626434 reads; of these:
##   11626434 (100.00%) were unpaired; of these:
##     4590053 (39.48%) aligned 0 times
##     5332647 (45.87%) aligned exactly 1 time
##     1703734 (14.65%) aligned &gt;1 times
## 60.52% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1703734/7036381
## 2 1294325/1703734
## 3 10623343/22466642
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0259-TP_1 and prad-0259-TP_2
## Single-end reads will be in prad-0259-TP_M
## 14809842 sequences in the BAM file
## 14809842 sequences exported
## 14809842 reads; of these:
##   14809842 (100.00%) were unpaired; of these:
##     4110198 (27.75%) aligned 0 times
##     8064244 (54.45%) aligned exactly 1 time
##     2635400 (17.79%) aligned &gt;1 times
## 72.25% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2635400/10699644
## 2 2184642/2635400
## 3 9563326/23735166
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0260-TP_1 and prad-0260-TP_2
## Single-end reads will be in prad-0260-TP_M
## 18849899 sequences in the BAM file
## 18849899 sequences exported
## 18849899 reads; of these:
##   18849899 (100.00%) were unpaired; of these:
##     5772657 (30.62%) aligned 0 times
##     10523223 (55.83%) aligned exactly 1 time
##     2554019 (13.55%) aligned &gt;1 times
## 69.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2554019/13077242
## 2 2066593/2554019
## 3 9352173/25777266
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0261-TP_1 and prad-0261-TP_2
## Single-end reads will be in prad-0261-TP_M
## 21105409 sequences in the BAM file
## 21105409 sequences exported
## 21105409 reads; of these:
##   21105409 (100.00%) were unpaired; of these:
##     5530602 (26.20%) aligned 0 times
##     11669892 (55.29%) aligned exactly 1 time
##     3904915 (18.50%) aligned &gt;1 times
## 73.80% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3904915/15574807
## 2 3178146/3904915
## 3 17647855/42188417
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in prad-0262-TP_1 and prad-0262-TP_2
## Single-end reads will be in prad-0262-TP_M
## 14242003 sequences in the BAM file
## 14242003 sequences exported
## 14242003 reads; of these:
##   14242003 (100.00%) were unpaired; of these:
##     3010854 (21.14%) aligned 0 times
##     8398706 (58.97%) aligned exactly 1 time
##     2832443 (19.89%) aligned &gt;1 times
## 78.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2832443/11231149
## 2 2145285/2832443
## 3 13099206/39729324
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0263-TP_1 and prad-0263-TP_2
## Single-end reads will be in prad-0263-TP_M
## 8974812 sequences in the BAM file
## 8974812 sequences exported
## 8974812 reads; of these:
##   8974812 (100.00%) were unpaired; of these:
##     1850025 (20.61%) aligned 0 times
##     5342318 (59.53%) aligned exactly 1 time
##     1782469 (19.86%) aligned &gt;1 times
## 79.39% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1782469/7124787
## 2 1345116/1782469
## 3 5353318/15890832
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0264-TP_1 and prad-0264-TP_2
## Single-end reads will be in prad-0264-TP_M
## 9975730 sequences in the BAM file
## 9975730 sequences exported
## 9975730 reads; of these:
##   9975730 (100.00%) were unpaired; of these:
##     3071554 (30.79%) aligned 0 times
##     5414392 (54.28%) aligned exactly 1 time
##     1489784 (14.93%) aligned &gt;1 times
## 69.21% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1489784/6904176
## 2 1119125/1489784
## 3 8489958/19061333
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0265-TP_1 and prad-0265-TP_2
## Single-end reads will be in prad-0265-TP_M
## 7875109 sequences in the BAM file
## 7875109 sequences exported
## 7875109 reads; of these:
##   7875109 (100.00%) were unpaired; of these:
##     1469284 (18.66%) aligned 0 times
##     4830119 (61.33%) aligned exactly 1 time
##     1575706 (20.01%) aligned &gt;1 times
## 81.34% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1575706/6405825
## 2 1185999/1575706
## 3 5637293/15919210
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0266-TP_1 and prad-0266-TP_2
## Single-end reads will be in prad-0266-TP_M
## 8733703 sequences in the BAM file
## 8733703 sequences exported
## 8733703 reads; of these:
##   8733703 (100.00%) were unpaired; of these:
##     3431758 (39.29%) aligned 0 times
##     3851386 (44.10%) aligned exactly 1 time
##     1450559 (16.61%) aligned &gt;1 times
## 60.71% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1450559/5301945
## 2 1113267/1450559
## 3 6373520/16701237
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0267-TP_1 and prad-0267-TP_2
## Single-end reads will be in prad-0267-TP_M
## 11826637 sequences in the BAM file
## 11826637 sequences exported
## 11826637 reads; of these:
##   11826637 (100.00%) were unpaired; of these:
##     5093813 (43.07%) aligned 0 times
##     5299295 (44.81%) aligned exactly 1 time
##     1433529 (12.12%) aligned &gt;1 times
## 56.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1433529/6732824
## 2 1047478/1433529
## 3 4021556/13155895
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0268-TP_1 and prad-0268-TP_2
## Single-end reads will be in prad-0268-TP_M
## 9492907 sequences in the BAM file
## 9492907 sequences exported
## 9492907 reads; of these:
##   9492907 (100.00%) were unpaired; of these:
##     2737662 (28.84%) aligned 0 times
##     5294962 (55.78%) aligned exactly 1 time
##     1460283 (15.38%) aligned &gt;1 times
## 71.16% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1460283/6755245
## 2 1116819/1460283
## 3 4672351/14635840
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0269-TP_1 and prad-0269-TP_2
## Single-end reads will be in prad-0269-TP_M
## 10767156 sequences in the BAM file
## 10767156 sequences exported
## 10767156 reads; of these:
##   10767156 (100.00%) were unpaired; of these:
##     3430218 (31.86%) aligned 0 times
##     5982575 (55.56%) aligned exactly 1 time
##     1354363 (12.58%) aligned &gt;1 times
## 68.14% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1354363/7336938
## 2 1028360/1354363
## 3 4026737/13515784
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0270-TP_1 and prad-0270-TP_2
## Single-end reads will be in prad-0270-TP_M
## 8735325 sequences in the BAM file
## 8735325 sequences exported
## 8735325 reads; of these:
##   8735325 (100.00%) were unpaired; of these:
##     3833310 (43.88%) aligned 0 times
##     3798502 (43.48%) aligned exactly 1 time
##     1103513 (12.63%) aligned &gt;1 times
## 56.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1103513/4902015
## 2 865356/1103513
## 3 3093799/9675137
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0271-TP_1 and prad-0271-TP_2
## Single-end reads will be in prad-0271-TP_M
## 8681476 sequences in the BAM file
## 8681476 sequences exported
## 8681476 reads; of these:
##   8681476 (100.00%) were unpaired; of these:
##     3061416 (35.26%) aligned 0 times
##     4458905 (51.36%) aligned exactly 1 time
##     1161155 (13.38%) aligned &gt;1 times
## 64.74% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1161155/5620060
## 2 904933/1161155
## 3 5039339/12862210
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0272-TP_1 and prad-0272-TP_2
## Single-end reads will be in prad-0272-TP_M
## 27904562 sequences in the BAM file
## 27904562 sequences exported
## 27904562 reads; of these:
##   27904562 (100.00%) were unpaired; of these:
##     6143731 (22.02%) aligned 0 times
##     16663264 (59.72%) aligned exactly 1 time
##     5097567 (18.27%) aligned &gt;1 times
## 77.98% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 5097567/21760831
## 2 4093914/5097567
## 3 23585932/57957868
## [bam_sort_core] merging from 10 files...
## This looks like paired data from lane 1.
## Output will be in prad-0273-TP_1 and prad-0273-TP_2
## Single-end reads will be in prad-0273-TP_M
## 22907322 sequences in the BAM file
## 22907322 sequences exported
## 22907322 reads; of these:
##   22907322 (100.00%) were unpaired; of these:
##     5730058 (25.01%) aligned 0 times
##     14555498 (63.54%) aligned exactly 1 time
##     2621766 (11.45%) aligned &gt;1 times
## 74.99% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2621766/17177264
## 2 2211469/2621766
## 3 15366822/35920378
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0274-TP_1 and prad-0274-TP_2
## Single-end reads will be in prad-0274-TP_M
## 11537927 sequences in the BAM file
## 11537927 sequences exported
## 11537927 reads; of these:
##   11537927 (100.00%) were unpaired; of these:
##     3609355 (31.28%) aligned 0 times
##     6298116 (54.59%) aligned exactly 1 time
##     1630456 (14.13%) aligned &gt;1 times
## 68.72% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1630456/7928572
## 2 1237828/1630456
## 3 6078073/16594268
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0275-TP_1 and prad-0275-TP_2
## Single-end reads will be in prad-0275-TP_M
## 9155411 sequences in the BAM file
## 9155411 sequences exported
## 9155411 reads; of these:
##   9155411 (100.00%) were unpaired; of these:
##     1827062 (19.96%) aligned 0 times
##     5786358 (63.20%) aligned exactly 1 time
##     1541991 (16.84%) aligned &gt;1 times
## 80.04% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1541991/7328349
## 2 1218534/1541991
## 3 8720242/20164811
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0276-TP_1 and prad-0276-TP_2
## Single-end reads will be in prad-0276-TP_M
## 7228094 sequences in the BAM file
## 7228094 sequences exported
## 7228094 reads; of these:
##   7228094 (100.00%) were unpaired; of these:
##     1452311 (20.09%) aligned 0 times
##     4049592 (56.03%) aligned exactly 1 time
##     1726191 (23.88%) aligned &gt;1 times
## 79.91% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1726191/5775783
## 2 1378844/1726191
## 3 17508187/30753772
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0277-TP_1 and prad-0277-TP_2
## Single-end reads will be in prad-0277-TP_M
## 20039266 sequences in the BAM file
## 20039266 sequences exported
## 20039266 reads; of these:
##   20039266 (100.00%) were unpaired; of these:
##     7553366 (37.69%) aligned 0 times
##     9862478 (49.22%) aligned exactly 1 time
##     2623422 (13.09%) aligned &gt;1 times
## 62.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2623422/12485900
## 2 2137289/2623422
## 3 9482382/25963955
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0278-TP_1 and prad-0278-TP_2
## Single-end reads will be in prad-0278-TP_M
## 11498432 sequences in the BAM file
## 11498432 sequences exported
## 11498432 reads; of these:
##   11498432 (100.00%) were unpaired; of these:
##     3988351 (34.69%) aligned 0 times
##     5854779 (50.92%) aligned exactly 1 time
##     1655302 (14.40%) aligned &gt;1 times
## 65.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1655302/7510081
## 2 1296743/1655302
## 3 4407590/14201437
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0279-TP_1 and prad-0279-TP_2
## Single-end reads will be in prad-0279-TP_M
## 11881616 sequences in the BAM file
## 11881616 sequences exported
## 11881616 reads; of these:
##   11881616 (100.00%) were unpaired; of these:
##     4574055 (38.50%) aligned 0 times
##     5999805 (50.50%) aligned exactly 1 time
##     1307756 (11.01%) aligned &gt;1 times
## 61.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1307756/7307561
## 2 968194/1307756
## 3 3887188/13477129
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0280-TP_1 and prad-0280-TP_2
## Single-end reads will be in prad-0280-TP_M
## 16045844 sequences in the BAM file
## 16045844 sequences exported
## 16045844 reads; of these:
##   16045844 (100.00%) were unpaired; of these:
##     6373649 (39.72%) aligned 0 times
##     7691434 (47.93%) aligned exactly 1 time
##     1980761 (12.34%) aligned &gt;1 times
## 60.28% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1980761/9672195
## 2 1536061/1980761
## 3 4832389/16913444
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0281-TP_1 and prad-0281-TP_2
## Single-end reads will be in prad-0281-TP_M
## 13461221 sequences in the BAM file
## 13461221 sequences exported
## 13461221 reads; of these:
##   13461221 (100.00%) were unpaired; of these:
##     3783572 (28.11%) aligned 0 times
##     7601789 (56.47%) aligned exactly 1 time
##     2075860 (15.42%) aligned &gt;1 times
## 71.89% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2075860/9677649
## 2 1604182/2075860
## 3 5374449/18138548
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0282-TP_1 and prad-0282-TP_2
## Single-end reads will be in prad-0282-TP_M
## 8842904 sequences in the BAM file
## 8842904 sequences exported
## 8842904 reads; of these:
##   8842904 (100.00%) were unpaired; of these:
##     3522886 (39.84%) aligned 0 times
##     4011559 (45.36%) aligned exactly 1 time
##     1308459 (14.80%) aligned &gt;1 times
## 60.16% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1308459/5320018
## 2 1014243/1308459
## 3 3112660/10236798
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0283-TP_1 and prad-0283-TP_2
## Single-end reads will be in prad-0283-TP_M
## 10300563 sequences in the BAM file
## 10300563 sequences exported
## 10300563 reads; of these:
##   10300563 (100.00%) were unpaired; of these:
##     2894457 (28.10%) aligned 0 times
##     5737862 (55.70%) aligned exactly 1 time
##     1668244 (16.20%) aligned &gt;1 times
## 71.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1668244/7406106
## 2 1280719/1668244
## 3 4141227/13863844
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0284-TP_1 and prad-0284-TP_2
## Single-end reads will be in prad-0284-TP_M
## 8879528 sequences in the BAM file
## 8879528 sequences exported
## 8879528 reads; of these:
##   8879528 (100.00%) were unpaired; of these:
##     3003517 (33.83%) aligned 0 times
##     4458144 (50.21%) aligned exactly 1 time
##     1417867 (15.97%) aligned &gt;1 times
## 66.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1417867/5876011
## 2 1116023/1417867
## 3 4852508/13284907
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0285-TP_1 and prad-0285-TP_2
## Single-end reads will be in prad-0285-TP_M
## 33628070 sequences in the BAM file
## 33628070 sequences exported
## 33628070 reads; of these:
##   33628070 (100.00%) were unpaired; of these:
##     13992937 (41.61%) aligned 0 times
##     16346441 (48.61%) aligned exactly 1 time
##     3288692 (9.78%) aligned &gt;1 times
## 58.39% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3288692/19635133
## 2 2303059/3288692
## 3 7644843/31533469
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 5.
## Output will be in prad-0286-TP_1 and prad-0286-TP_2
## Single-end reads will be in prad-0286-TP_M
## 13173108 sequences in the BAM file
## 13173108 sequences exported
## 13173108 reads; of these:
##   13173108 (100.00%) were unpaired; of these:
##     5112252 (38.81%) aligned 0 times
##     6441098 (48.90%) aligned exactly 1 time
##     1619758 (12.30%) aligned &gt;1 times
## 61.19% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1619758/8060856
## 2 1257524/1619758
## 3 4593978/15309102
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0287-TP_1 and prad-0287-TP_2
## Single-end reads will be in prad-0287-TP_M
## 9629655 sequences in the BAM file
## 9629655 sequences exported
## 9629655 reads; of these:
##   9629655 (100.00%) were unpaired; of these:
##     3208316 (33.32%) aligned 0 times
##     4655368 (48.34%) aligned exactly 1 time
##     1765971 (18.34%) aligned &gt;1 times
## 66.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1765971/6421339
## 2 1350554/1765971
## 3 9457196/20611084
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0288-TP_1 and prad-0288-TP_2
## Single-end reads will be in prad-0288-TP_M
## 6178833 sequences in the BAM file
## 6178833 sequences exported
## 6178833 reads; of these:
##   6178833 (100.00%) were unpaired; of these:
##     1741487 (28.18%) aligned 0 times
##     3506079 (56.74%) aligned exactly 1 time
##     931267 (15.07%) aligned &gt;1 times
## 71.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 931267/4437346
## 2 677542/931267
## 3 2442945/8488430
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0289-TP_1 and prad-0289-TP_2
## Single-end reads will be in prad-0289-TP_M
## 12182219 sequences in the BAM file
## 12182219 sequences exported
## 12182219 reads; of these:
##   12182219 (100.00%) were unpaired; of these:
##     4715597 (38.71%) aligned 0 times
##     5438714 (44.64%) aligned exactly 1 time
##     2027908 (16.65%) aligned &gt;1 times
## 61.29% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2027908/7466622
## 2 1434937/2027908
## 3 7024936/18631700
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0290-TP_1 and prad-0290-TP_2
## Single-end reads will be in prad-0290-TP_M
## 11501264 sequences in the BAM file
## 11501264 sequences exported
## 11501264 reads; of these:
##   11501264 (100.00%) were unpaired; of these:
##     4591559 (39.92%) aligned 0 times
##     5521103 (48.00%) aligned exactly 1 time
##     1388602 (12.07%) aligned &gt;1 times
## 60.08% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1388602/6909705
## 2 1082592/1388602
## 3 3274531/11794502
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0291-TP_1 and prad-0291-TP_2
## Single-end reads will be in prad-0291-TP_M
## 6886665 sequences in the BAM file
## 6886665 sequences exported
## 6886665 reads; of these:
##   6886665 (100.00%) were unpaired; of these:
##     920559 (13.37%) aligned 0 times
##     3683528 (53.49%) aligned exactly 1 time
##     2282578 (33.14%) aligned &gt;1 times
## 86.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2282578/5966106
## 2 1571565/2282578
## 3 14966156/33277172
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 7.
## Output will be in prad-0292-TP_1 and prad-0292-TP_2
## Single-end reads will be in prad-0292-TP_M
## 5479299 sequences in the BAM file
## 5479299 sequences exported
## 5479299 reads; of these:
##   5479299 (100.00%) were unpaired; of these:
##     1980000 (36.14%) aligned 0 times
##     2169060 (39.59%) aligned exactly 1 time
##     1330239 (24.28%) aligned &gt;1 times
## 63.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1330239/3499299
## 2 886581/1330239
## 3 6415094/15120394
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0293-TP_1 and prad-0293-TP_2
## Single-end reads will be in prad-0293-TP_M
## 9212070 sequences in the BAM file
## 9212070 sequences exported
## 9212070 reads; of these:
##   9212070 (100.00%) were unpaired; of these:
##     2994787 (32.51%) aligned 0 times
##     4983921 (54.10%) aligned exactly 1 time
##     1233362 (13.39%) aligned &gt;1 times
## 67.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1233362/6217283
## 2 937167/1233362
## 3 3862259/12240870
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0294-TP_1 and prad-0294-TP_2
## Single-end reads will be in prad-0294-TP_M
## 8129157 sequences in the BAM file
## 8129157 sequences exported
## 8129157 reads; of these:
##   8129157 (100.00%) were unpaired; of these:
##     3378254 (41.56%) aligned 0 times
##     3408566 (41.93%) aligned exactly 1 time
##     1342337 (16.51%) aligned &gt;1 times
## 58.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1342337/4750903
## 2 1013658/1342337
## 3 3398807/10303722
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0295-TP_1 and prad-0295-TP_2
## Single-end reads will be in prad-0295-TP_M
## 18954905 sequences in the BAM file
## 18954905 sequences exported
## 18954905 reads; of these:
##   18954905 (100.00%) were unpaired; of these:
##     7075243 (37.33%) aligned 0 times
##     9357621 (49.37%) aligned exactly 1 time
##     2522041 (13.31%) aligned &gt;1 times
## 62.67% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2522041/11879662
## 2 1970507/2522041
## 3 5958893/20786771
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0296-TP_1 and prad-0296-TP_2
## Single-end reads will be in prad-0296-TP_M
## 11332336 sequences in the BAM file
## 11332336 sequences exported
## 11332336 reads; of these:
##   11332336 (100.00%) were unpaired; of these:
##     4459031 (39.35%) aligned 0 times
##     5318766 (46.93%) aligned exactly 1 time
##     1554539 (13.72%) aligned &gt;1 times
## 60.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1554539/6873305
## 2 1169767/1554539
## 3 5958333/16187039
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0297-TP_1 and prad-0297-TP_2
## Single-end reads will be in prad-0297-TP_M
## 10891178 sequences in the BAM file
## 10891178 sequences exported
## 10891178 reads; of these:
##   10891178 (100.00%) were unpaired; of these:
##     4065712 (37.33%) aligned 0 times
##     5370336 (49.31%) aligned exactly 1 time
##     1455130 (13.36%) aligned &gt;1 times
## 62.67% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1455130/6825466
## 2 1051143/1455130
## 3 4519905/14322843
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0298-TP_1 and prad-0298-TP_2
## Single-end reads will be in prad-0298-TP_M
## 10099529 sequences in the BAM file
## 10099529 sequences exported
## 10099529 reads; of these:
##   10099529 (100.00%) were unpaired; of these:
##     3984736 (39.45%) aligned 0 times
##     4524458 (44.80%) aligned exactly 1 time
##     1590335 (15.75%) aligned &gt;1 times
## 60.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1590335/6114793
## 2 1280143/1590335
## 3 3656743/11559270
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0299-TP_1 and prad-0299-TP_2
## Single-end reads will be in prad-0299-TP_M
## 13145994 sequences in the BAM file
## 13145994 sequences exported
## 13145994 reads; of these:
##   13145994 (100.00%) were unpaired; of these:
##     4512563 (34.33%) aligned 0 times
##     6832162 (51.97%) aligned exactly 1 time
##     1801269 (13.70%) aligned &gt;1 times
## 65.67% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1801269/8633431
## 2 1401551/1801269
## 3 5621262/17404387
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0300-TP_1 and prad-0300-TP_2
## Single-end reads will be in prad-0300-TP_M
## 19167440 sequences in the BAM file
## 19167440 sequences exported
## 19167440 reads; of these:
##   19167440 (100.00%) were unpaired; of these:
##     6682805 (34.87%) aligned 0 times
##     10495479 (54.76%) aligned exactly 1 time
##     1989156 (10.38%) aligned &gt;1 times
## 65.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1989156/12484635
## 2 1573599/1989156
## 3 5636134/20492048
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0301-TP_1 and prad-0301-TP_2
## Single-end reads will be in prad-0301-TP_M
## 8779906 sequences in the BAM file
## 8779906 sequences exported
## 8779906 reads; of these:
##   8779906 (100.00%) were unpaired; of these:
##     3233799 (36.83%) aligned 0 times
##     4468140 (50.89%) aligned exactly 1 time
##     1077967 (12.28%) aligned &gt;1 times
## 63.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1077967/5546107
## 2 767333/1077967
## 3 3387875/11032345
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0302-TP_1 and prad-0302-TP_2
## Single-end reads will be in prad-0302-TP_M
## 6316482 sequences in the BAM file
## 6316482 sequences exported
## 6316482 reads; of these:
##   6316482 (100.00%) were unpaired; of these:
##     2506257 (39.68%) aligned 0 times
##     2895479 (45.84%) aligned exactly 1 time
##     914746 (14.48%) aligned &gt;1 times
## 60.32% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 914746/3810225
## 2 726662/914746
## 3 2945927/8537927
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0303-TP_1 and prad-0303-TP_2
## Single-end reads will be in prad-0303-TP_M
## 9777513 sequences in the BAM file
## 9777513 sequences exported
## 9777513 reads; of these:
##   9777513 (100.00%) were unpaired; of these:
##     1766335 (18.07%) aligned 0 times
##     6361661 (65.06%) aligned exactly 1 time
##     1649517 (16.87%) aligned &gt;1 times
## 81.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1649517/8011178
## 2 1282748/1649517
## 3 4631538/15218893
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0304-TP_1 and prad-0304-TP_2
## Single-end reads will be in prad-0304-TP_M
## 5689093 sequences in the BAM file
## 5689093 sequences exported
## 5689093 reads; of these:
##   5689093 (100.00%) were unpaired; of these:
##     2114176 (37.16%) aligned 0 times
##     2544878 (44.73%) aligned exactly 1 time
##     1030039 (18.11%) aligned &gt;1 times
## 62.84% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1030039/3574917
## 2 801177/1030039
## 3 4419399/10475879
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0305-TP_1 and prad-0305-TP_2
## Single-end reads will be in prad-0305-TP_M
## 17169008 sequences in the BAM file
## 17169008 sequences exported
## 17169008 reads; of these:
##   17169008 (100.00%) were unpaired; of these:
##     6965475 (40.57%) aligned 0 times
##     8288156 (48.27%) aligned exactly 1 time
##     1915377 (11.16%) aligned &gt;1 times
## 59.43% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1915377/10203533
## 2 1366800/1915377
## 3 5278824/18715690
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0306-TP_1 and prad-0306-TP_2
## Single-end reads will be in prad-0306-TP_M
## 6443059 sequences in the BAM file
## 6443059 sequences exported
## 6443059 reads; of these:
##   6443059 (100.00%) were unpaired; of these:
##     1835904 (28.49%) aligned 0 times
##     3652014 (56.68%) aligned exactly 1 time
##     955141 (14.82%) aligned &gt;1 times
## 71.51% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 955141/4607155
## 2 733760/955141
## 3 2666567/8891787
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0307-TP_1 and prad-0307-TP_2
## Single-end reads will be in prad-0307-TP_M
## 13085023 sequences in the BAM file
## 13085023 sequences exported
## 13085023 reads; of these:
##   13085023 (100.00%) were unpaired; of these:
##     5246799 (40.10%) aligned 0 times
##     6031577 (46.10%) aligned exactly 1 time
##     1806647 (13.81%) aligned &gt;1 times
## 59.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1806647/7838224
## 2 1391489/1806647
## 3 5893647/17152581
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0308-TP_1 and prad-0308-TP_2
## Single-end reads will be in prad-0308-TP_M
## 6957778 sequences in the BAM file
## 6957778 sequences exported
## 6957778 reads; of these:
##   6957778 (100.00%) were unpaired; of these:
##     1962812 (28.21%) aligned 0 times
##     3654995 (52.53%) aligned exactly 1 time
##     1339971 (19.26%) aligned &gt;1 times
## 71.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1339971/4994966
## 2 996867/1339971
## 3 3588470/10806026
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0309-TP_1 and prad-0309-TP_2
## Single-end reads will be in prad-0309-TP_M
## 8594657 sequences in the BAM file
## 8594657 sequences exported
## 8594657 reads; of these:
##   8594657 (100.00%) were unpaired; of these:
##     3753457 (43.67%) aligned 0 times
##     3766726 (43.83%) aligned exactly 1 time
##     1074474 (12.50%) aligned &gt;1 times
## 56.33% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1074474/4841200
## 2 776500/1074474
## 3 2623663/8945137
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0310-TP_1 and prad-0310-TP_2
## Single-end reads will be in prad-0310-TP_M
## 8584145 sequences in the BAM file
## 8584145 sequences exported
## 8584145 reads; of these:
##   8584145 (100.00%) were unpaired; of these:
##     3386481 (39.45%) aligned 0 times
##     4021225 (46.84%) aligned exactly 1 time
##     1176439 (13.70%) aligned &gt;1 times
## 60.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1176439/5197664
## 2 859242/1176439
## 3 2916462/9843956
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0311-TP_1 and prad-0311-TP_2
## Single-end reads will be in prad-0311-TP_M
## 10918976 sequences in the BAM file
## 10918976 sequences exported
## 10918976 reads; of these:
##   10918976 (100.00%) were unpaired; of these:
##     2990029 (27.38%) aligned 0 times
##     5863649 (53.70%) aligned exactly 1 time
##     2065298 (18.91%) aligned &gt;1 times
## 72.62% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2065298/7928947
## 2 1578531/2065298
## 3 5192967/15870331
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0312-TP_1 and prad-0312-TP_2
## Single-end reads will be in prad-0312-TP_M
## 8856446 sequences in the BAM file
## 8856446 sequences exported
## 8856446 reads; of these:
##   8856446 (100.00%) were unpaired; of these:
##     3476559 (39.25%) aligned 0 times
##     4051051 (45.74%) aligned exactly 1 time
##     1328836 (15.00%) aligned &gt;1 times
## 60.75% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1328836/5379887
## 2 952283/1328836
## 3 5070185/14105848
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0313-TP_1 and prad-0313-TP_2
## Single-end reads will be in prad-0313-TP_M
## 20392888 sequences in the BAM file
## 20392888 sequences exported
## 20392888 reads; of these:
##   20392888 (100.00%) were unpaired; of these:
##     7715459 (37.83%) aligned 0 times
##     10626336 (52.11%) aligned exactly 1 time
##     2051093 (10.06%) aligned &gt;1 times
## 62.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2051093/12677429
## 2 1591872/2051093
## 3 5093666/20307820
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0314-TP_1 and prad-0314-TP_2
## Single-end reads will be in prad-0314-TP_M
## 4710814 sequences in the BAM file
## 4710814 sequences exported
## 4710814 reads; of these:
##   4710814 (100.00%) were unpaired; of these:
##     1830881 (38.87%) aligned 0 times
##     1988595 (42.21%) aligned exactly 1 time
##     891338 (18.92%) aligned &gt;1 times
## 61.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 891338/2879933
## 2 696825/891338
## 3 2267108/6401795
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0315-TP_1 and prad-0315-TP_2
## Single-end reads will be in prad-0315-TP_M
## 9261621 sequences in the BAM file
## 9261621 sequences exported
## 9261621 reads; of these:
##   9261621 (100.00%) were unpaired; of these:
##     3208757 (34.65%) aligned 0 times
##     4718645 (50.95%) aligned exactly 1 time
##     1334219 (14.41%) aligned &gt;1 times
## 65.35% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1334219/6052864
## 2 1052899/1334219
## 3 3595904/11505051
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0316-TP_1 and prad-0316-TP_2
## Single-end reads will be in prad-0316-TP_M
## 11424256 sequences in the BAM file
## 11424256 sequences exported
## 11424256 reads; of these:
##   11424256 (100.00%) were unpaired; of these:
##     3333618 (29.18%) aligned 0 times
##     6100904 (53.40%) aligned exactly 1 time
##     1989734 (17.42%) aligned &gt;1 times
## 70.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1989734/8090638
## 2 1561996/1989734
## 3 6414749/18928863
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0317-TP_1 and prad-0317-TP_2
## Single-end reads will be in prad-0317-TP_M
## 13315427 sequences in the BAM file
## 13315427 sequences exported
## 13315427 reads; of these:
##   13315427 (100.00%) were unpaired; of these:
##     4462110 (33.51%) aligned 0 times
##     7408423 (55.64%) aligned exactly 1 time
##     1444894 (10.85%) aligned &gt;1 times
## 66.49% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1444894/8853317
## 2 1126091/1444894
## 3 4007401/14553763
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0318-TP_1 and prad-0318-TP_2
## Single-end reads will be in prad-0318-TP_M
## 14536327 sequences in the BAM file
## 14536327 sequences exported
## 14536327 reads; of these:
##   14536327 (100.00%) were unpaired; of these:
##     5880078 (40.45%) aligned 0 times
##     6814062 (46.88%) aligned exactly 1 time
##     1842187 (12.67%) aligned &gt;1 times
## 59.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1842187/8656249
## 2 1351659/1842187
## 3 6922249/20773508
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0319-TP_1 and prad-0319-TP_2
## Single-end reads will be in prad-0319-TP_M
## 8110093 sequences in the BAM file
## 8110093 sequences exported
## 8110093 reads; of these:
##   8110093 (100.00%) were unpaired; of these:
##     2355419 (29.04%) aligned 0 times
##     4465732 (55.06%) aligned exactly 1 time
##     1288942 (15.89%) aligned &gt;1 times
## 70.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1288942/5754674
## 2 947864/1288942
## 3 3634163/11742288
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0320-TP_1 and prad-0320-TP_2
## Single-end reads will be in prad-0320-TP_M
## 13312008 sequences in the BAM file
## 13312008 sequences exported
## 13312008 reads; of these:
##   13312008 (100.00%) were unpaired; of these:
##     4598294 (34.54%) aligned 0 times
##     6549041 (49.20%) aligned exactly 1 time
##     2164673 (16.26%) aligned &gt;1 times
## 65.46% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2164673/8713714
## 2 1618688/2164673
## 3 8393524/22283828
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0321-TP_1 and prad-0321-TP_2
## Single-end reads will be in prad-0321-TP_M
## 13926904 sequences in the BAM file
## 13926904 sequences exported
## 13926904 reads; of these:
##   13926904 (100.00%) were unpaired; of these:
##     4830317 (34.68%) aligned 0 times
##     7349170 (52.77%) aligned exactly 1 time
##     1747417 (12.55%) aligned &gt;1 times
## 65.32% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1747417/9096587
## 2 1351822/1747417
## 3 4977127/16234845
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0322-TP_1 and prad-0322-TP_2
## Single-end reads will be in prad-0322-TP_M
## 5706316 sequences in the BAM file
## 5706316 sequences exported
## 5706316 reads; of these:
##   5706316 (100.00%) were unpaired; of these:
##     1901065 (33.32%) aligned 0 times
##     2602858 (45.61%) aligned exactly 1 time
##     1202393 (21.07%) aligned &gt;1 times
## 66.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1202393/3805251
## 2 932858/1202393
## 3 4607935/11875150
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0323-TP_1 and prad-0323-TP_2
## Single-end reads will be in prad-0323-TP_M
## 10501947 sequences in the BAM file
## 10501947 sequences exported
## 10501947 reads; of these:
##   10501947 (100.00%) were unpaired; of these:
##     4283208 (40.78%) aligned 0 times
##     5047429 (48.06%) aligned exactly 1 time
##     1171310 (11.15%) aligned &gt;1 times
## 59.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1171310/6218739
## 2 895879/1171310
## 3 3639310/11837826
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0324-TP_1 and prad-0324-TP_2
## Single-end reads will be in prad-0324-TP_M
## 14153750 sequences in the BAM file
## 14153750 sequences exported
## 14153750 reads; of these:
##   14153750 (100.00%) were unpaired; of these:
##     4620086 (32.64%) aligned 0 times
##     7662299 (54.14%) aligned exactly 1 time
##     1871365 (13.22%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1871365/9533664
## 2 1437095/1871365
## 3 5078760/17266720
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0325-TP_1 and prad-0325-TP_2
## Single-end reads will be in prad-0325-TP_M
## 6108467 sequences in the BAM file
## 6108467 sequences exported
## 6108467 reads; of these:
##   6108467 (100.00%) were unpaired; of these:
##     2139072 (35.02%) aligned 0 times
##     2480335 (40.60%) aligned exactly 1 time
##     1489060 (24.38%) aligned &gt;1 times
## 64.98% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1489060/3969395
## 2 1139041/1489060
## 3 8923244/19215590
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0326-TP_1 and prad-0326-TP_2
## Single-end reads will be in prad-0326-TP_M
## 17562650 sequences in the BAM file
## 17562650 sequences exported
## 17562650 reads; of these:
##   17562650 (100.00%) were unpaired; of these:
##     6487877 (36.94%) aligned 0 times
##     8954854 (50.99%) aligned exactly 1 time
##     2119919 (12.07%) aligned &gt;1 times
## 63.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2119919/11074773
## 2 1591496/2119919
## 3 5242518/18866609
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0327-TP_1 and prad-0327-TP_2
## Single-end reads will be in prad-0327-TP_M
## 15820200 sequences in the BAM file
## 15820200 sequences exported
## 15820200 reads; of these:
##   15820200 (100.00%) were unpaired; of these:
##     6057553 (38.29%) aligned 0 times
##     7699839 (48.67%) aligned exactly 1 time
##     2062808 (13.04%) aligned &gt;1 times
## 61.71% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2062808/9762647
## 2 1550346/2062808
## 3 5056390/17815059
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0328-TP_1 and prad-0328-TP_2
## Single-end reads will be in prad-0328-TP_M
## 9338772 sequences in the BAM file
## 9338772 sequences exported
## 9338772 reads; of these:
##   9338772 (100.00%) were unpaired; of these:
##     3457017 (37.02%) aligned 0 times
##     4332402 (46.39%) aligned exactly 1 time
##     1549353 (16.59%) aligned &gt;1 times
## 62.98% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1549353/5881755
## 2 1196997/1549353
## 3 6829055/17592573
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0329-TP_1 and prad-0329-TP_2
## Single-end reads will be in prad-0329-TP_M
## 14977487 sequences in the BAM file
## 14977487 sequences exported
## 14977487 reads; of these:
##   14977487 (100.00%) were unpaired; of these:
##     5681715 (37.94%) aligned 0 times
##     7384336 (49.30%) aligned exactly 1 time
##     1911436 (12.76%) aligned &gt;1 times
## 62.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1911436/9295772
## 2 1412228/1911436
## 3 4523304/16188381
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0330-TP_1 and prad-0330-TP_2
## Single-end reads will be in prad-0330-TP_M
## 14706035 sequences in the BAM file
## 14706035 sequences exported
## 14706035 reads; of these:
##   14706035 (100.00%) were unpaired; of these:
##     5948738 (40.45%) aligned 0 times
##     6266958 (42.61%) aligned exactly 1 time
##     2490339 (16.93%) aligned &gt;1 times
## 59.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2490339/8757297
## 2 1874321/2490339
## 3 6202680/18235907
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0331-TP_1 and prad-0331-TP_2
## Single-end reads will be in prad-0331-TP_M
## 7072269 sequences in the BAM file
## 7072269 sequences exported
## 7072269 reads; of these:
##   7072269 (100.00%) were unpaired; of these:
##     2386663 (33.75%) aligned 0 times
##     3395968 (48.02%) aligned exactly 1 time
##     1289638 (18.24%) aligned &gt;1 times
## 66.25% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1289638/4685606
## 2 957087/1289638
## 3 4972665/12632254
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0332-TP_1 and prad-0332-TP_2
## Single-end reads will be in prad-0332-TP_M
## 10256113 sequences in the BAM file
## 10256113 sequences exported
## 10256113 reads; of these:
##   10256113 (100.00%) were unpaired; of these:
##     3918542 (38.21%) aligned 0 times
##     4919530 (47.97%) aligned exactly 1 time
##     1418041 (13.83%) aligned &gt;1 times
## 61.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1418041/6337571
## 2 1024412/1418041
## 3 3446559/11867428
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0333-TP_1 and prad-0333-TP_2
## Single-end reads will be in prad-0333-TP_M
## 12129013 sequences in the BAM file
## 12129013 sequences exported
## 12129013 reads; of these:
##   12129013 (100.00%) were unpaired; of these:
##     3588884 (29.59%) aligned 0 times
##     6916123 (57.02%) aligned exactly 1 time
##     1624006 (13.39%) aligned &gt;1 times
## 70.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1624006/8540129
## 2 1249381/1624006
## 3 3594252/13910319
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0334-TP_1 and prad-0334-TP_2
## Single-end reads will be in prad-0334-TP_M
## 8885963 sequences in the BAM file
## 8885963 sequences exported
## 8885963 reads; of these:
##   8885963 (100.00%) were unpaired; of these:
##     3211352 (36.14%) aligned 0 times
##     4408973 (49.62%) aligned exactly 1 time
##     1265638 (14.24%) aligned &gt;1 times
## 63.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1265638/5674611
## 2 972486/1265638
## 3 3994248/11893151
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0335-TP_1 and prad-0335-TP_2
## Single-end reads will be in prad-0335-TP_M
## 33593761 sequences in the BAM file
## 33593761 sequences exported
## 33593761 reads; of these:
##   33593761 (100.00%) were unpaired; of these:
##     12193511 (36.30%) aligned 0 times
##     17527264 (52.17%) aligned exactly 1 time
##     3872986 (11.53%) aligned &gt;1 times
## 63.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3872986/21400250
## 2 2905592/3872986
## 3 10119182/36981853
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 1.
## Output will be in prad-0336-TP_1 and prad-0336-TP_2
## Single-end reads will be in prad-0336-TP_M
## 10670253 sequences in the BAM file
## 10670253 sequences exported
## 10670253 reads; of these:
##   10670253 (100.00%) were unpaired; of these:
##     3507127 (32.87%) aligned 0 times
##     5371149 (50.34%) aligned exactly 1 time
##     1791977 (16.79%) aligned &gt;1 times
## 67.13% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1791977/7163126
## 2 1412935/1791977
## 3 6352236/17520477
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0337-TP_1 and prad-0337-TP_2
## Single-end reads will be in prad-0337-TP_M
## 8993785 sequences in the BAM file
## 8993785 sequences exported
## 8993785 reads; of these:
##   8993785 (100.00%) were unpaired; of these:
##     3336457 (37.10%) aligned 0 times
##     4737954 (52.68%) aligned exactly 1 time
##     919374 (10.22%) aligned &gt;1 times
## 62.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 919374/5657328
## 2 704747/919374
## 3 3409269/11442488
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0338-TP_1 and prad-0338-TP_2
## Single-end reads will be in prad-0338-TP_M
## 11754864 sequences in the BAM file
## 11754864 sequences exported
## 11754864 reads; of these:
##   11754864 (100.00%) were unpaired; of these:
##     4901219 (41.70%) aligned 0 times
##     5314640 (45.21%) aligned exactly 1 time
##     1539005 (13.09%) aligned &gt;1 times
## 58.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1539005/6853645
## 2 1236267/1539005
## 3 3697878/12338149
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0339-TP_1 and prad-0339-TP_2
## Single-end reads will be in prad-0339-TP_M
## 5035328 sequences in the BAM file
## 5035328 sequences exported
## 5035328 reads; of these:
##   5035328 (100.00%) were unpaired; of these:
##     1791734 (35.58%) aligned 0 times
##     2004491 (39.81%) aligned exactly 1 time
##     1239103 (24.61%) aligned &gt;1 times
## 64.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1239103/3243594
## 2 962220/1239103
## 3 3606923/8974460
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 4.
## Output will be in prad-0340-TP_1 and prad-0340-TP_2
## Single-end reads will be in prad-0340-TP_M
## 9839161 sequences in the BAM file
## 9839161 sequences exported
## 9839161 reads; of these:
##   9839161 (100.00%) were unpaired; of these:
##     3477722 (35.35%) aligned 0 times
##     4834841 (49.14%) aligned exactly 1 time
##     1526598 (15.52%) aligned &gt;1 times
## 64.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1526598/6361439
## 2 1193930/1526598
## 3 5921654/15857254
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0341-TP_1 and prad-0341-TP_2
## Single-end reads will be in prad-0341-TP_M
## 13740761 sequences in the BAM file
## 13740761 sequences exported
## 13740761 reads; of these:
##   13740761 (100.00%) were unpaired; of these:
##     4916557 (35.78%) aligned 0 times
##     7095972 (51.64%) aligned exactly 1 time
##     1728232 (12.58%) aligned &gt;1 times
## 64.22% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1728232/8824204
## 2 1341382/1728232
## 3 4810044/15939210
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0342-TP_1 and prad-0342-TP_2
## Single-end reads will be in prad-0342-TP_M
## 10742893 sequences in the BAM file
## 10742893 sequences exported
## 10742893 reads; of these:
##   10742893 (100.00%) were unpaired; of these:
##     4442862 (41.36%) aligned 0 times
##     4990055 (46.45%) aligned exactly 1 time
##     1309976 (12.19%) aligned &gt;1 times
## 58.64% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1309976/6300031
## 2 1045362/1309976
## 3 3426095/11522274
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0343-TP_1 and prad-0343-TP_2
## Single-end reads will be in prad-0343-TP_M
## 7034754 sequences in the BAM file
## 7034754 sequences exported
## 7034754 reads; of these:
##   7034754 (100.00%) were unpaired; of these:
##     2823961 (40.14%) aligned 0 times
##     3187364 (45.31%) aligned exactly 1 time
##     1023429 (14.55%) aligned &gt;1 times
## 59.86% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1023429/4210793
## 2 732784/1023429
## 3 2942244/9050723
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0344-TP_1 and prad-0344-TP_2
## Single-end reads will be in prad-0344-TP_M
## 6894352 sequences in the BAM file
## 6894352 sequences exported
## 6894352 reads; of these:
##   6894352 (100.00%) were unpaired; of these:
##     2888649 (41.90%) aligned 0 times
##     3144371 (45.61%) aligned exactly 1 time
##     861332 (12.49%) aligned &gt;1 times
## 58.10% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 861332/4005703
## 2 628380/861332
## 3 2154064/7298306
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0345-TP_1 and prad-0345-TP_2
## Single-end reads will be in prad-0345-TP_M
## 11681756 sequences in the BAM file
## 11681756 sequences exported
## 11681756 reads; of these:
##   11681756 (100.00%) were unpaired; of these:
##     4374749 (37.45%) aligned 0 times
##     6058977 (51.87%) aligned exactly 1 time
##     1248030 (10.68%) aligned &gt;1 times
## 62.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1248030/7307007
## 2 943131/1248030
## 3 3332876/12661591
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0346-TP_1 and prad-0346-TP_2
## Single-end reads will be in prad-0346-TP_M
## 4464996 sequences in the BAM file
## 4464996 sequences exported
## 4464996 reads; of these:
##   4464996 (100.00%) were unpaired; of these:
##     1708825 (38.27%) aligned 0 times
##     2111327 (47.29%) aligned exactly 1 time
##     644844 (14.44%) aligned &gt;1 times
## 61.73% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 644844/2756171
## 2 500869/644844
## 3 1523558/5140646
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0347-TP_1 and prad-0347-TP_2
## Single-end reads will be in prad-0347-TP_M
## 11137252 sequences in the BAM file
## 11137252 sequences exported
## 11137252 reads; of these:
##   11137252 (100.00%) were unpaired; of these:
##     4074874 (36.59%) aligned 0 times
##     5881180 (52.81%) aligned exactly 1 time
##     1181198 (10.61%) aligned &gt;1 times
## 63.41% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1181198/7062378
## 2 895804/1181198
## 3 2786505/11338835
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0348-TP_1 and prad-0348-TP_2
## Single-end reads will be in prad-0348-TP_M
## 7733749 sequences in the BAM file
## 7733749 sequences exported
## 7733749 reads; of these:
##   7733749 (100.00%) were unpaired; of these:
##     2827475 (36.56%) aligned 0 times
##     3618866 (46.79%) aligned exactly 1 time
##     1287408 (16.65%) aligned &gt;1 times
## 63.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1287408/4906274
## 2 954600/1287408
## 3 3736412/10800581
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0349-TP_1 and prad-0349-TP_2
## Single-end reads will be in prad-0349-TP_M
## 9481599 sequences in the BAM file
## 9481599 sequences exported
## 9481599 reads; of these:
##   9481599 (100.00%) were unpaired; of these:
##     3737165 (39.41%) aligned 0 times
##     4519405 (47.67%) aligned exactly 1 time
##     1225029 (12.92%) aligned &gt;1 times
## 60.59% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1225029/5744434
## 2 889959/1225029
## 3 3620599/11729354
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0350-TP_1 and prad-0350-TP_2
## Single-end reads will be in prad-0350-TP_M
## 9679724 sequences in the BAM file
## 9679724 sequences exported
## 9679724 reads; of these:
##   9679724 (100.00%) were unpaired; of these:
##     2493813 (25.76%) aligned 0 times
##     5688877 (58.77%) aligned exactly 1 time
##     1497034 (15.47%) aligned &gt;1 times
## 74.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1497034/7185911
## 2 1156889/1497034
## 3 4386317/14213336
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0351-TP_1 and prad-0351-TP_2
## Single-end reads will be in prad-0351-TP_M
## 16118140 sequences in the BAM file
## 16118140 sequences exported
## 16118140 reads; of these:
##   16118140 (100.00%) were unpaired; of these:
##     5954362 (36.94%) aligned 0 times
##     8030683 (49.82%) aligned exactly 1 time
##     2133095 (13.23%) aligned &gt;1 times
## 63.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2133095/10163778
## 2 1658624/2133095
## 3 4990133/17426256
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0352-TP_1 and prad-0352-TP_2
## Single-end reads will be in prad-0352-TP_M
## 12672136 sequences in the BAM file
## 12672136 sequences exported
## 12672136 reads; of these:
##   12672136 (100.00%) were unpaired; of these:
##     4487144 (35.41%) aligned 0 times
##     6084324 (48.01%) aligned exactly 1 time
##     2100668 (16.58%) aligned &gt;1 times
## 64.59% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2100668/8184992
## 2 1544277/2100668
## 3 9632931/24424882
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0353-TP_1 and prad-0353-TP_2
## Single-end reads will be in prad-0353-TP_M
## 8080050 sequences in the BAM file
## 8080050 sequences exported
## 8080050 reads; of these:
##   8080050 (100.00%) were unpaired; of these:
##     2332155 (28.86%) aligned 0 times
##     4645725 (57.50%) aligned exactly 1 time
##     1102170 (13.64%) aligned &gt;1 times
## 71.14% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1102170/5747895
## 2 824691/1102170
## 3 3254569/11062849
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0354-TP_1 and prad-0354-TP_2
## Single-end reads will be in prad-0354-TP_M
## 8453742 sequences in the BAM file
## 8453742 sequences exported
## 8453742 reads; of these:
##   8453742 (100.00%) were unpaired; of these:
##     3110853 (36.80%) aligned 0 times
##     3993344 (47.24%) aligned exactly 1 time
##     1349545 (15.96%) aligned &gt;1 times
## 63.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1349545/5342889
## 2 1079983/1349545
## 3 3393700/10340137
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0355-TP_1 and prad-0355-TP_2
## Single-end reads will be in prad-0355-TP_M
## 16790965 sequences in the BAM file
## 16790965 sequences exported
## 16790965 reads; of these:
##   16790965 (100.00%) were unpaired; of these:
##     5665754 (33.74%) aligned 0 times
##     8655450 (51.55%) aligned exactly 1 time
##     2469761 (14.71%) aligned &gt;1 times
## 66.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2469761/11125211
## 2 1779589/2469761
## 3 9347731/25675679
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0356-TP_1 and prad-0356-TP_2
## Single-end reads will be in prad-0356-TP_M
## 17041754 sequences in the BAM file
## 17041754 sequences exported
## 17041754 reads; of these:
##   17041754 (100.00%) were unpaired; of these:
##     5531116 (32.46%) aligned 0 times
##     9051578 (53.11%) aligned exactly 1 time
##     2459060 (14.43%) aligned &gt;1 times
## 67.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2459060/11510638
## 2 1903929/2459060
## 3 5804698/20081381
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0357-TP_1 and prad-0357-TP_2
## Single-end reads will be in prad-0357-TP_M
## 8301850 sequences in the BAM file
## 8301850 sequences exported
## 8301850 reads; of these:
##   8301850 (100.00%) were unpaired; of these:
##     3156719 (38.02%) aligned 0 times
##     3869756 (46.61%) aligned exactly 1 time
##     1275375 (15.36%) aligned &gt;1 times
## 61.98% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1275375/5145131
## 2 999570/1275375
## 3 3295053/10181600
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0358-TP_1 and prad-0358-TP_2
## Single-end reads will be in prad-0358-TP_M
## 16250085 sequences in the BAM file
## 16250085 sequences exported
## 16250085 reads; of these:
##   16250085 (100.00%) were unpaired; of these:
##     4429446 (27.26%) aligned 0 times
##     9729084 (59.87%) aligned exactly 1 time
##     2091555 (12.87%) aligned &gt;1 times
## 72.74% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2091555/11820639
## 2 1511845/2091555
## 3 8191121/25787490
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0359-TP_1 and prad-0359-TP_2
## Single-end reads will be in prad-0359-TP_M
## 51075488 sequences in the BAM file
## 51075488 sequences exported
## 51075488 reads; of these:
##   51075488 (100.00%) were unpaired; of these:
##     15182414 (29.73%) aligned 0 times
##     28515371 (55.83%) aligned exactly 1 time
##     7377703 (14.44%) aligned &gt;1 times
## 70.27% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 7377703/35893074
## 2 5545040/7377703
## 3 20832115/69532154
## [bam_sort_core] merging from 14 files...
## This looks like paired data from lane 1.
## Output will be in prad-0360-TP_1 and prad-0360-TP_2
## Single-end reads will be in prad-0360-TP_M
## 7625010 sequences in the BAM file
## 7625010 sequences exported
## 7625010 reads; of these:
##   7625010 (100.00%) were unpaired; of these:
##     1145682 (15.03%) aligned 0 times
##     4860403 (63.74%) aligned exactly 1 time
##     1618925 (21.23%) aligned &gt;1 times
## 84.97% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1618925/6479328
## 2 1253892/1618925
## 3 4501944/13665851
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0361-TP_1 and prad-0361-TP_2
## Single-end reads will be in prad-0361-TP_M
## 10090079 sequences in the BAM file
## 10090079 sequences exported
## 10090079 reads; of these:
##   10090079 (100.00%) were unpaired; of these:
##     2961450 (29.35%) aligned 0 times
##     5845566 (57.93%) aligned exactly 1 time
##     1283063 (12.72%) aligned &gt;1 times
## 70.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1283063/7128629
## 2 947711/1283063
## 3 3583374/12776625
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0362-TP_1 and prad-0362-TP_2
## Single-end reads will be in prad-0362-TP_M
## 9778859 sequences in the BAM file
## 9778859 sequences exported
## 9778859 reads; of these:
##   9778859 (100.00%) were unpaired; of these:
##     3552161 (36.32%) aligned 0 times
##     4889892 (50.00%) aligned exactly 1 time
##     1336806 (13.67%) aligned &gt;1 times
## 63.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1336806/6226698
## 2 954167/1336806
## 3 3523497/11870332
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0363-TP_1 and prad-0363-TP_2
## Single-end reads will be in prad-0363-TP_M
## 12039289 sequences in the BAM file
## 12039289 sequences exported
## 12039289 reads; of these:
##   12039289 (100.00%) were unpaired; of these:
##     4260112 (35.39%) aligned 0 times
##     6184879 (51.37%) aligned exactly 1 time
##     1594298 (13.24%) aligned &gt;1 times
## 64.61% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1594298/7779177
## 2 1227791/1594298
## 3 4387529/14763254
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0364-TP_1 and prad-0364-TP_2
## Single-end reads will be in prad-0364-TP_M
## 4227384 sequences in the BAM file
## 4227384 sequences exported
## 4227384 reads; of these:
##   4227384 (100.00%) were unpaired; of these:
##     1460919 (34.56%) aligned 0 times
##     2203174 (52.12%) aligned exactly 1 time
##     563291 (13.32%) aligned &gt;1 times
## 65.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 563291/2766465
## 2 435230/563291
## 3 1866125/5828257
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 3.
## Output will be in prad-0365-TP_1 and prad-0365-TP_2
## Single-end reads will be in prad-0365-TP_M
## 8205648 sequences in the BAM file
## 8205648 sequences exported
## 8205648 reads; of these:
##   8205648 (100.00%) were unpaired; of these:
##     2132645 (25.99%) aligned 0 times
##     3161380 (38.53%) aligned exactly 1 time
##     2911623 (35.48%) aligned &gt;1 times
## 74.01% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2911623/6073003
## 2 1621199/2911623
## 3 18729026/41402714
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 7.
## Output will be in prad-0366-TP_1 and prad-0366-TP_2
## Single-end reads will be in prad-0366-TP_M
## 16117022 sequences in the BAM file
## 16117022 sequences exported
## 16117022 reads; of these:
##   16117022 (100.00%) were unpaired; of these:
##     6467459 (40.13%) aligned 0 times
##     7760736 (48.15%) aligned exactly 1 time
##     1888827 (11.72%) aligned &gt;1 times
## 59.87% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1888827/9649563
## 2 1433686/1888827
## 3 5724492/18798595
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0367-TP_1 and prad-0367-TP_2
## Single-end reads will be in prad-0367-TP_M
## 15766453 sequences in the BAM file
## 15766453 sequences exported
## 15766453 reads; of these:
##   15766453 (100.00%) were unpaired; of these:
##     4088314 (25.93%) aligned 0 times
##     9531084 (60.45%) aligned exactly 1 time
##     2147055 (13.62%) aligned &gt;1 times
## 74.07% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2147055/11678139
## 2 1609424/2147055
## 3 5714278/20842065
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0368-TP_1 and prad-0368-TP_2
## Single-end reads will be in prad-0368-TP_M
## 11826745 sequences in the BAM file
## 11826745 sequences exported
## 11826745 reads; of these:
##   11826745 (100.00%) were unpaired; of these:
##     3599754 (30.44%) aligned 0 times
##     5567979 (47.08%) aligned exactly 1 time
##     2659012 (22.48%) aligned &gt;1 times
## 69.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2659012/8226991
## 2 2046689/2659012
## 3 20440550/45693642
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 4.
## Output will be in prad-0369-TP_1 and prad-0369-TP_2
## Single-end reads will be in prad-0369-TP_M
## 2673071 sequences in the BAM file
## 2673071 sequences exported
## 2673071 reads; of these:
##   2673071 (100.00%) were unpaired; of these:
##     737265 (27.58%) aligned 0 times
##     1089552 (40.76%) aligned exactly 1 time
##     846254 (31.66%) aligned &gt;1 times
## 72.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 846254/1935806
## 2 557180/846254
## 3 2244152/5743939
## This looks like paired data from lane 5.
## Output will be in prad-0370-TP_1 and prad-0370-TP_2
## Single-end reads will be in prad-0370-TP_M
## 4515374 sequences in the BAM file
## 4515374 sequences exported
## 4515374 reads; of these:
##   4515374 (100.00%) were unpaired; of these:
##     1345998 (29.81%) aligned 0 times
##     2417384 (53.54%) aligned exactly 1 time
##     751992 (16.65%) aligned &gt;1 times
## 70.19% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 751992/3169376
## 2 619163/751992
## 3 2443606/7116587
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0371-TP_1 and prad-0371-TP_2
## Single-end reads will be in prad-0371-TP_M
## 10621974 sequences in the BAM file
## 10621974 sequences exported
## 10621974 reads; of these:
##   10621974 (100.00%) were unpaired; of these:
##     4109728 (38.69%) aligned 0 times
##     4573157 (43.05%) aligned exactly 1 time
##     1939089 (18.26%) aligned &gt;1 times
## 61.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1939089/6512246
## 2 1453003/1939089
## 3 6030830/16765057
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0372-TP_1 and prad-0372-TP_2
## Single-end reads will be in prad-0372-TP_M
## 5697401 sequences in the BAM file
## 5697401 sequences exported
## 5697401 reads; of these:
##   5697401 (100.00%) were unpaired; of these:
##     1859677 (32.64%) aligned 0 times
##     3080478 (54.07%) aligned exactly 1 time
##     757246 (13.29%) aligned &gt;1 times
## 67.36% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 757246/3837724
## 2 594597/757246
## 3 2060400/7054665
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0373-TP_1 and prad-0373-TP_2
## Single-end reads will be in prad-0373-TP_M
## 10480755 sequences in the BAM file
## 10480755 sequences exported
## 10480755 reads; of these:
##   10480755 (100.00%) were unpaired; of these:
##     3199882 (30.53%) aligned 0 times
##     5563980 (53.09%) aligned exactly 1 time
##     1716893 (16.38%) aligned &gt;1 times
## 69.47% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1716893/7280873
## 2 1197689/1716893
## 3 5834435/17257408
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0374-TP_1 and prad-0374-TP_2
## Single-end reads will be in prad-0374-TP_M
## 10615965 sequences in the BAM file
## 10615965 sequences exported
## 10615965 reads; of these:
##   10615965 (100.00%) were unpaired; of these:
##     4042467 (38.08%) aligned 0 times
##     5189265 (48.88%) aligned exactly 1 time
##     1384233 (13.04%) aligned &gt;1 times
## 61.92% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1384233/6573498
## 2 1060704/1384233
## 3 3861872/12682902
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0375-TP_1 and prad-0375-TP_2
## Single-end reads will be in prad-0375-TP_M
## 44173311 sequences in the BAM file
## 44173311 sequences exported
## 44173311 reads; of these:
##   44173311 (100.00%) were unpaired; of these:
##     12390681 (28.05%) aligned 0 times
##     21513997 (48.70%) aligned exactly 1 time
##     10268633 (23.25%) aligned &gt;1 times
## 71.95% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 10268633/31782630
## 2 8611171/10268633
## 3 22134181/65748157
## [bam_sort_core] merging from 13 files...
## This looks like paired data from lane 1.
## Output will be in prad-0376-TP_1 and prad-0376-TP_2
## Single-end reads will be in prad-0376-TP_M
## 5326078 sequences in the BAM file
## 5326078 sequences exported
## 5326078 reads; of these:
##   5326078 (100.00%) were unpaired; of these:
##     748350 (14.05%) aligned 0 times
##     3189987 (59.89%) aligned exactly 1 time
##     1387741 (26.06%) aligned &gt;1 times
## 85.95% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1387741/4577728
## 2 1074859/1387741
## 3 4976018/12903024
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0377-TP_1 and prad-0377-TP_2
## Single-end reads will be in prad-0377-TP_M
## 18898481 sequences in the BAM file
## 18898481 sequences exported
## 18898481 reads; of these:
##   18898481 (100.00%) were unpaired; of these:
##     5953710 (31.50%) aligned 0 times
##     11141239 (58.95%) aligned exactly 1 time
##     1803532 (9.54%) aligned &gt;1 times
## 68.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1803532/12944771
## 2 1350074/1803532
## 3 4766152/20402689
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0378-TP_1 and prad-0378-TP_2
## Single-end reads will be in prad-0378-TP_M
## 4387179 sequences in the BAM file
## 4387179 sequences exported
## 4387179 reads; of these:
##   4387179 (100.00%) were unpaired; of these:
##     1125142 (25.65%) aligned 0 times
##     2344612 (53.44%) aligned exactly 1 time
##     917425 (20.91%) aligned &gt;1 times
## 74.35% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 917425/3262037
## 2 660298/917425
## 3 3222011/8704401
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0379-TP_1 and prad-0379-TP_2
## Single-end reads will be in prad-0379-TP_M
## 6837789 sequences in the BAM file
## 6837789 sequences exported
## 6837789 reads; of these:
##   6837789 (100.00%) were unpaired; of these:
##     1995446 (29.18%) aligned 0 times
##     3181396 (46.53%) aligned exactly 1 time
##     1660947 (24.29%) aligned &gt;1 times
## 70.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1660947/4842343
## 2 1226408/1660947
## 3 5390498/13761178
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0380-TP_1 and prad-0380-TP_2
## Single-end reads will be in prad-0380-TP_M
## 17641945 sequences in the BAM file
## 17641945 sequences exported
## 17641945 reads; of these:
##   17641945 (100.00%) were unpaired; of these:
##     4575342 (25.93%) aligned 0 times
##     10674646 (60.51%) aligned exactly 1 time
##     2391957 (13.56%) aligned &gt;1 times
## 74.07% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2391957/13066603
## 2 1798970/2391957
## 3 7660109/25784791
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 5.
## Output will be in prad-0381-TP_1 and prad-0381-TP_2
## Single-end reads will be in prad-0381-TP_M
## 9739606 sequences in the BAM file
## 9739606 sequences exported
## 9739606 reads; of these:
##   9739606 (100.00%) were unpaired; of these:
##     2978357 (30.58%) aligned 0 times
##     5497137 (56.44%) aligned exactly 1 time
##     1264112 (12.98%) aligned &gt;1 times
## 69.42% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1264112/6761249
## 2 928568/1264112
## 3 3229410/11914248
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0382-TP_1 and prad-0382-TP_2
## Single-end reads will be in prad-0382-TP_M
## 8630333 sequences in the BAM file
## 8630333 sequences exported
## 8630333 reads; of these:
##   8630333 (100.00%) were unpaired; of these:
##     2689332 (31.16%) aligned 0 times
##     4447577 (51.53%) aligned exactly 1 time
##     1493424 (17.30%) aligned &gt;1 times
## 68.84% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1493424/5941001
## 2 1152499/1493424
## 3 4479642/12681265
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0383-TP_1 and prad-0383-TP_2
## Single-end reads will be in prad-0383-TP_M
## 6100773 sequences in the BAM file
## 6100773 sequences exported
## 6100773 reads; of these:
##   6100773 (100.00%) were unpaired; of these:
##     1913961 (31.37%) aligned 0 times
##     3040594 (49.84%) aligned exactly 1 time
##     1146218 (18.79%) aligned &gt;1 times
## 68.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1146218/4186812
## 2 887604/1146218
## 3 4848622/12275511
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0384-TP_1 and prad-0384-TP_2
## Single-end reads will be in prad-0384-TP_M
## 19156962 sequences in the BAM file
## 19156962 sequences exported
## 19156962 reads; of these:
##   19156962 (100.00%) were unpaired; of these:
##     5909375 (30.85%) aligned 0 times
##     11135209 (58.13%) aligned exactly 1 time
##     2112378 (11.03%) aligned &gt;1 times
## 69.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2112378/13247587
## 2 1532428/2112378
## 3 5156483/21151503
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0385-TP_1 and prad-0385-TP_2
## Single-end reads will be in prad-0385-TP_M
## 12803290 sequences in the BAM file
## 12803290 sequences exported
## 12803290 reads; of these:
##   12803290 (100.00%) were unpaired; of these:
##     3991528 (31.18%) aligned 0 times
##     6971716 (54.45%) aligned exactly 1 time
##     1840046 (14.37%) aligned &gt;1 times
## 68.82% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1840046/8811762
## 2 1358167/1840046
## 3 8465571/23208504
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0386-TP_1 and prad-0386-TP_2
## Single-end reads will be in prad-0386-TP_M
## 13400100 sequences in the BAM file
## 13400100 sequences exported
## 13400100 reads; of these:
##   13400100 (100.00%) were unpaired; of these:
##     3359413 (25.07%) aligned 0 times
##     8291824 (61.88%) aligned exactly 1 time
##     1748863 (13.05%) aligned &gt;1 times
## 74.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1748863/10040687
## 2 1288632/1748863
## 3 4302234/16773985
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0387-TP_1 and prad-0387-TP_2
## Single-end reads will be in prad-0387-TP_M
## 15612430 sequences in the BAM file
## 15612430 sequences exported
## 15612430 reads; of these:
##   15612430 (100.00%) were unpaired; of these:
##     6416390 (41.10%) aligned 0 times
##     7512410 (48.12%) aligned exactly 1 time
##     1683630 (10.78%) aligned &gt;1 times
## 58.90% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1683630/9196040
## 2 1260875/1683630
## 3 3914404/15064532
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0388-TP_1 and prad-0388-TP_2
## Single-end reads will be in prad-0388-TP_M
## 10922335 sequences in the BAM file
## 10922335 sequences exported
## 10922335 reads; of these:
##   10922335 (100.00%) were unpaired; of these:
##     2670469 (24.45%) aligned 0 times
##     6483474 (59.36%) aligned exactly 1 time
##     1768392 (16.19%) aligned &gt;1 times
## 75.55% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1768392/8251866
## 2 1357233/1768392
## 3 4899451/16059992
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 7.
## Output will be in prad-0389-TP_1 and prad-0389-TP_2
## Single-end reads will be in prad-0389-TP_M
## 6587495 sequences in the BAM file
## 6587495 sequences exported
## 6587495 reads; of these:
##   6587495 (100.00%) were unpaired; of these:
##     2631775 (39.95%) aligned 0 times
##     3005227 (45.62%) aligned exactly 1 time
##     950493 (14.43%) aligned &gt;1 times
## 60.05% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 950493/3955720
## 2 681219/950493
## 3 2986260/8946640
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0390-TP_1 and prad-0390-TP_2
## Single-end reads will be in prad-0390-TP_M
## 6112218 sequences in the BAM file
## 6112218 sequences exported
## 6112218 reads; of these:
##   6112218 (100.00%) were unpaired; of these:
##     2467363 (40.37%) aligned 0 times
##     2659625 (43.51%) aligned exactly 1 time
##     985230 (16.12%) aligned &gt;1 times
## 59.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 985230/3644855
## 2 743920/985230
## 3 2883065/8136596
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0391-TP_1 and prad-0391-TP_2
## Single-end reads will be in prad-0391-TP_M
## 10463853 sequences in the BAM file
## 10463853 sequences exported
## 10463853 reads; of these:
##   10463853 (100.00%) were unpaired; of these:
##     4129309 (39.46%) aligned 0 times
##     4978071 (47.57%) aligned exactly 1 time
##     1356473 (12.96%) aligned &gt;1 times
## 60.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1356473/6334544
## 2 1066223/1356473
## 3 3257666/11417294
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0392-TP_1 and prad-0392-TP_2
## Single-end reads will be in prad-0392-TP_M
## 15484726 sequences in the BAM file
## 15484726 sequences exported
## 15484726 reads; of these:
##   15484726 (100.00%) were unpaired; of these:
##     6095818 (39.37%) aligned 0 times
##     7596716 (49.06%) aligned exactly 1 time
##     1792192 (11.57%) aligned &gt;1 times
## 60.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1792192/9388908
## 2 1355567/1792192
## 3 4594225/16368131
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0393-TP_1 and prad-0393-TP_2
## Single-end reads will be in prad-0393-TP_M
## 7688779 sequences in the BAM file
## 7688779 sequences exported
## 7688779 reads; of these:
##   7688779 (100.00%) were unpaired; of these:
##     2484084 (32.31%) aligned 0 times
##     4007338 (52.12%) aligned exactly 1 time
##     1197357 (15.57%) aligned &gt;1 times
## 67.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1197357/5204695
## 2 939039/1197357
## 3 3821787/11315408
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0394-TP_1 and prad-0394-TP_2
## Single-end reads will be in prad-0394-TP_M
## 10645210 sequences in the BAM file
## 10645210 sequences exported
## 10645210 reads; of these:
##   10645210 (100.00%) were unpaired; of these:
##     2546421 (23.92%) aligned 0 times
##     6409987 (60.21%) aligned exactly 1 time
##     1688802 (15.86%) aligned &gt;1 times
## 76.08% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1688802/8098789
## 2 1304760/1688802
## 3 4238472/14558085
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0395-TP_1 and prad-0395-TP_2
## Single-end reads will be in prad-0395-TP_M
## 4441054 sequences in the BAM file
## 4441054 sequences exported
## 4441054 reads; of these:
##   4441054 (100.00%) were unpaired; of these:
##     1472985 (33.17%) aligned 0 times
##     2346913 (52.85%) aligned exactly 1 time
##     621156 (13.99%) aligned &gt;1 times
## 66.83% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 621156/2968069
## 2 471431/621156
## 3 1876190/5951032
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0396-TP_1 and prad-0396-TP_2
## Single-end reads will be in prad-0396-TP_M
## 18706594 sequences in the BAM file
## 18706594 sequences exported
## 18706594 reads; of these:
##   18706594 (100.00%) were unpaired; of these:
##     6570086 (35.12%) aligned 0 times
##     10002905 (53.47%) aligned exactly 1 time
##     2133603 (11.41%) aligned &gt;1 times
## 64.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2133603/12136508
## 2 1617275/2133603
## 3 6650783/22165050
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0397-TP_1 and prad-0397-TP_2
## Single-end reads will be in prad-0397-TP_M
## 9736840 sequences in the BAM file
## 9736840 sequences exported
## 9736840 reads; of these:
##   9736840 (100.00%) were unpaired; of these:
##     3370489 (34.62%) aligned 0 times
##     5188843 (53.29%) aligned exactly 1 time
##     1177508 (12.09%) aligned &gt;1 times
## 65.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1177508/6366351
## 2 843449/1177508
## 3 3096256/11178355
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 4.
## Output will be in prad-0398-TP_1 and prad-0398-TP_2
## Single-end reads will be in prad-0398-TP_M
## 8428262 sequences in the BAM file
## 8428262 sequences exported
## 8428262 reads; of these:
##   8428262 (100.00%) were unpaired; of these:
##     2169537 (25.74%) aligned 0 times
##     4924945 (58.43%) aligned exactly 1 time
##     1333780 (15.83%) aligned &gt;1 times
## 74.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1333780/6258725
## 2 1053979/1333780
## 3 3772025/12223582
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0399-TP_1 and prad-0399-TP_2
## Single-end reads will be in prad-0399-TP_M
## 9891076 sequences in the BAM file
## 9891076 sequences exported
## 9891076 reads; of these:
##   9891076 (100.00%) were unpaired; of these:
##     3115573 (31.50%) aligned 0 times
##     4783973 (48.37%) aligned exactly 1 time
##     1991530 (20.13%) aligned &gt;1 times
## 68.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1991530/6775503
## 2 1668880/1991530
## 3 4386974/13274036
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0400-TP_1 and prad-0400-TP_2
## Single-end reads will be in prad-0400-TP_M
## 16645140 sequences in the BAM file
## 16645140 sequences exported
## 16645140 reads; of these:
##   16645140 (100.00%) were unpaired; of these:
##     5092706 (30.60%) aligned 0 times
##     9870700 (59.30%) aligned exactly 1 time
##     1681734 (10.10%) aligned &gt;1 times
## 69.40% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1681734/11552434
## 2 1309379/1681734
## 3 4524515/18195450
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0401-TP_1 and prad-0401-TP_2
## Single-end reads will be in prad-0401-TP_M
## 23071145 sequences in the BAM file
## 23071145 sequences exported
## 23071145 reads; of these:
##   23071145 (100.00%) were unpaired; of these:
##     5644349 (24.46%) aligned 0 times
##     14668716 (63.58%) aligned exactly 1 time
##     2758080 (11.95%) aligned &gt;1 times
## 75.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2758080/17426796
## 2 1992192/2758080
## 3 6714308/27754720
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 6.
## Output will be in prad-0402-TP_1 and prad-0402-TP_2
## Single-end reads will be in prad-0402-TP_M
## 11786020 sequences in the BAM file
## 11786020 sequences exported
## 11786020 reads; of these:
##   11786020 (100.00%) were unpaired; of these:
##     3917969 (33.24%) aligned 0 times
##     5650042 (47.94%) aligned exactly 1 time
##     2218009 (18.82%) aligned &gt;1 times
## 66.76% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2218009/7868051
## 2 1556518/2218009
## 3 8324543/22209645
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0403-TP_1 and prad-0403-TP_2
## Single-end reads will be in prad-0403-TP_M
## 11716585 sequences in the BAM file
## 11716585 sequences exported
## 11716585 reads; of these:
##   11716585 (100.00%) were unpaired; of these:
##     3992361 (34.07%) aligned 0 times
##     5764452 (49.20%) aligned exactly 1 time
##     1959772 (16.73%) aligned &gt;1 times
## 65.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1959772/7724224
## 2 1383665/1959772
## 3 11119442/28081815
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0404-TP_1 and prad-0404-TP_2
## Single-end reads will be in prad-0404-TP_M
## 12724492 sequences in the BAM file
## 12724492 sequences exported
## 12724492 reads; of these:
##   12724492 (100.00%) were unpaired; of these:
##     3073306 (24.15%) aligned 0 times
##     7908743 (62.15%) aligned exactly 1 time
##     1742443 (13.69%) aligned &gt;1 times
## 75.85% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1742443/9651186
## 2 1343516/1742443
## 3 4829426/17076037
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0405-TP_1 and prad-0405-TP_2
## Single-end reads will be in prad-0405-TP_M
## 8430637 sequences in the BAM file
## 8430637 sequences exported
## 8430637 reads; of these:
##   8430637 (100.00%) were unpaired; of these:
##     1887621 (22.39%) aligned 0 times
##     5329123 (63.21%) aligned exactly 1 time
##     1213893 (14.40%) aligned &gt;1 times
## 77.61% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1213893/6543016
## 2 921083/1213893
## 3 3492320/12159951
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0406-TP_1 and prad-0406-TP_2
## Single-end reads will be in prad-0406-TP_M
## 14297040 sequences in the BAM file
## 14297040 sequences exported
## 14297040 reads; of these:
##   14297040 (100.00%) were unpaired; of these:
##     4503063 (31.50%) aligned 0 times
##     8231525 (57.58%) aligned exactly 1 time
##     1562452 (10.93%) aligned &gt;1 times
## 68.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1562452/9793977
## 2 1196742/1562452
## 3 4203708/16253026
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0407-TP_1 and prad-0407-TP_2
## Single-end reads will be in prad-0407-TP_M
## 9423955 sequences in the BAM file
## 9423955 sequences exported
## 9423955 reads; of these:
##   9423955 (100.00%) were unpaired; of these:
##     2743719 (29.11%) aligned 0 times
##     5315913 (56.41%) aligned exactly 1 time
##     1364323 (14.48%) aligned &gt;1 times
## 70.89% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1364323/6680236
## 2 1058978/1364323
## 3 4323726/13612947
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0408-TP_1 and prad-0408-TP_2
## Single-end reads will be in prad-0408-TP_M
## 7020070 sequences in the BAM file
## 7020070 sequences exported
## 7020070 reads; of these:
##   7020070 (100.00%) were unpaired; of these:
##     2622369 (37.36%) aligned 0 times
##     3202416 (45.62%) aligned exactly 1 time
##     1195285 (17.03%) aligned &gt;1 times
## 62.64% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1195285/4397701
## 2 920927/1195285
## 3 4175883/11221748
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0409-TP_1 and prad-0409-TP_2
## Single-end reads will be in prad-0409-TP_M
## 22877203 sequences in the BAM file
## 22877203 sequences exported
## 22877203 reads; of these:
##   22877203 (100.00%) were unpaired; of these:
##     6698347 (29.28%) aligned 0 times
##     14309664 (62.55%) aligned exactly 1 time
##     1869192 (8.17%) aligned &gt;1 times
## 70.72% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1869192/16178856
## 2 1443000/1869192
## 3 4756307/23194674
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 4.
## Output will be in prad-0410-TP_1 and prad-0410-TP_2
## Single-end reads will be in prad-0410-TP_M
## 12001920 sequences in the BAM file
## 12001920 sequences exported
## 12001920 reads; of these:
##   12001920 (100.00%) were unpaired; of these:
##     2976356 (24.80%) aligned 0 times
##     7567684 (63.05%) aligned exactly 1 time
##     1457880 (12.15%) aligned &gt;1 times
## 75.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1457880/9025564
## 2 1125165/1457880
## 3 4739325/16813441
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0411-TP_1 and prad-0411-TP_2
## Single-end reads will be in prad-0411-TP_M
## 19202630 sequences in the BAM file
## 19202630 sequences exported
## 19202630 reads; of these:
##   19202630 (100.00%) were unpaired; of these:
##     6546816 (34.09%) aligned 0 times
##     10353236 (53.92%) aligned exactly 1 time
##     2302578 (11.99%) aligned &gt;1 times
## 65.91% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2302578/12655814
## 2 1797203/2302578
## 3 5593156/20988903
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0412-TP_1 and prad-0412-TP_2
## Single-end reads will be in prad-0412-TP_M
## 21633666 sequences in the BAM file
## 21633666 sequences exported
## 21633666 reads; of these:
##   21633666 (100.00%) were unpaired; of these:
##     3288783 (15.20%) aligned 0 times
##     15491119 (71.61%) aligned exactly 1 time
##     2853764 (13.19%) aligned &gt;1 times
## 84.80% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2853764/18344883
## 2 2177423/2853764
## 3 9223142/34337442
## [bam_sort_core] merging from 8 files...
## This looks like paired data from lane 5.
## Output will be in prad-0413-TP_1 and prad-0413-TP_2
## Single-end reads will be in prad-0413-TP_M
## 11554998 sequences in the BAM file
## 11554998 sequences exported
## 11554998 reads; of these:
##   11554998 (100.00%) were unpaired; of these:
##     3254935 (28.17%) aligned 0 times
##     6923867 (59.92%) aligned exactly 1 time
##     1376196 (11.91%) aligned &gt;1 times
## 71.83% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1376196/8300063
## 2 1066899/1376196
## 3 3765299/14168674
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0414-TP_1 and prad-0414-TP_2
## Single-end reads will be in prad-0414-TP_M
## 13528697 sequences in the BAM file
## 13528697 sequences exported
## 13528697 reads; of these:
##   13528697 (100.00%) were unpaired; of these:
##     2149699 (15.89%) aligned 0 times
##     8728637 (64.52%) aligned exactly 1 time
##     2650361 (19.59%) aligned &gt;1 times
## 84.11% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2650361/11378998
## 2 2030555/2650361
## 3 9787958/27381382
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0415-TP_1 and prad-0415-TP_2
## Single-end reads will be in prad-0415-TP_M
## 6199546 sequences in the BAM file
## 6199546 sequences exported
## 6199546 reads; of these:
##   6199546 (100.00%) were unpaired; of these:
##     1488701 (24.01%) aligned 0 times
##     3593602 (57.97%) aligned exactly 1 time
##     1117243 (18.02%) aligned &gt;1 times
## 75.99% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1117243/4710845
## 2 896003/1117243
## 3 2816961/9135175
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0416-TP_1 and prad-0416-TP_2
## Single-end reads will be in prad-0416-TP_M
## 10363183 sequences in the BAM file
## 10363183 sequences exported
## 10363183 reads; of these:
##   10363183 (100.00%) were unpaired; of these:
##     3496963 (33.74%) aligned 0 times
##     5723527 (55.23%) aligned exactly 1 time
##     1142693 (11.03%) aligned &gt;1 times
## 66.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1142693/6866220
## 2 879288/1142693
## 3 2980492/11470906
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0417-TP_1 and prad-0417-TP_2
## Single-end reads will be in prad-0417-TP_M
## 10599813 sequences in the BAM file
## 10599813 sequences exported
## 10599813 reads; of these:
##   10599813 (100.00%) were unpaired; of these:
##     3333103 (31.44%) aligned 0 times
##     6025154 (56.84%) aligned exactly 1 time
##     1241556 (11.71%) aligned &gt;1 times
## 68.56% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1241556/7266710
## 2 972367/1241556
## 3 3218675/12208940
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0418-TP_1 and prad-0418-TP_2
## Single-end reads will be in prad-0418-TP_M
## 17902330 sequences in the BAM file
## 17902330 sequences exported
## 17902330 reads; of these:
##   17902330 (100.00%) were unpaired; of these:
##     4819310 (26.92%) aligned 0 times
##     10712199 (59.84%) aligned exactly 1 time
##     2370821 (13.24%) aligned &gt;1 times
## 73.08% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2370821/13083020
## 2 1873039/2370821
## 3 6336089/22942992
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 4.
## Output will be in prad-0419-TP_1 and prad-0419-TP_2
## Single-end reads will be in prad-0419-TP_M
## 8432084 sequences in the BAM file
## 8432084 sequences exported
## 8432084 reads; of these:
##   8432084 (100.00%) were unpaired; of these:
##     1922390 (22.80%) aligned 0 times
##     5118354 (60.70%) aligned exactly 1 time
##     1391340 (16.50%) aligned &gt;1 times
## 77.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1391340/6509694
## 2 1066992/1391340
## 3 4690299/14234840
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0420-TP_1 and prad-0420-TP_2
## Single-end reads will be in prad-0420-TP_M
## 9335675 sequences in the BAM file
## 9335675 sequences exported
## 9335675 reads; of these:
##   9335675 (100.00%) were unpaired; of these:
##     3258734 (34.91%) aligned 0 times
##     4770869 (51.10%) aligned exactly 1 time
##     1306072 (13.99%) aligned &gt;1 times
## 65.09% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1306072/6076941
## 2 964818/1306072
## 3 4223470/13195583
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0421-TP_1 and prad-0421-TP_2
## Single-end reads will be in prad-0421-TP_M
## 7369639 sequences in the BAM file
## 7369639 sequences exported
## 7369639 reads; of these:
##   7369639 (100.00%) were unpaired; of these:
##     2358450 (32.00%) aligned 0 times
##     2989428 (40.56%) aligned exactly 1 time
##     2021761 (27.43%) aligned &gt;1 times
## 68.00% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2021761/5011189
## 2 1701216/2021761
## 3 5536147/13703000
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0422-TP_1 and prad-0422-TP_2
## Single-end reads will be in prad-0422-TP_M
## 13071959 sequences in the BAM file
## 13071959 sequences exported
## 13071959 reads; of these:
##   13071959 (100.00%) were unpaired; of these:
##     3577941 (27.37%) aligned 0 times
##     7506561 (57.42%) aligned exactly 1 time
##     1987457 (15.20%) aligned &gt;1 times
## 72.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1987457/9494018
## 2 1537387/1987457
## 3 5776870/18664215
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0423-TP_1 and prad-0423-TP_2
## Single-end reads will be in prad-0423-TP_M
## 5245868 sequences in the BAM file
## 5245868 sequences exported
## 5245868 reads; of these:
##   5245868 (100.00%) were unpaired; of these:
##     1658664 (31.62%) aligned 0 times
##     2737004 (52.17%) aligned exactly 1 time
##     850200 (16.21%) aligned &gt;1 times
## 68.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 850200/3587204
## 2 627997/850200
## 3 1941475/6577117
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0424-TP_1 and prad-0424-TP_2
## Single-end reads will be in prad-0424-TP_M
## 12507418 sequences in the BAM file
## 12507418 sequences exported
## 12507418 reads; of these:
##   12507418 (100.00%) were unpaired; of these:
##     3787729 (30.28%) aligned 0 times
##     7238972 (57.88%) aligned exactly 1 time
##     1480717 (11.84%) aligned &gt;1 times
## 69.72% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1480717/8719689
## 2 1125091/1480717
## 3 4382646/15747808
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0425-TP_1 and prad-0425-TP_2
## Single-end reads will be in prad-0425-TP_M
## 11681481 sequences in the BAM file
## 11681481 sequences exported
## 11681481 reads; of these:
##   11681481 (100.00%) were unpaired; of these:
##     2830416 (24.23%) aligned 0 times
##     7224605 (61.85%) aligned exactly 1 time
##     1626460 (13.92%) aligned &gt;1 times
## 75.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1626460/8851065
## 2 1256704/1626460
## 3 4062247/14992208
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0426-TP_1 and prad-0426-TP_2
## Single-end reads will be in prad-0426-TP_M
## 7787115 sequences in the BAM file
## 7787115 sequences exported
## 7787115 reads; of these:
##   7787115 (100.00%) were unpaired; of these:
##     1815475 (23.31%) aligned 0 times
##     4800289 (61.64%) aligned exactly 1 time
##     1171351 (15.04%) aligned &gt;1 times
## 76.69% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1171351/5971640
## 2 918278/1171351
## 3 3146356/10798239
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 7.
## Output will be in prad-0427-TP_1 and prad-0427-TP_2
## Single-end reads will be in prad-0427-TP_M
## 16568443 sequences in the BAM file
## 16568443 sequences exported
## 16568443 reads; of these:
##   16568443 (100.00%) were unpaired; of these:
##     6211860 (37.49%) aligned 0 times
##     8218827 (49.61%) aligned exactly 1 time
##     2137756 (12.90%) aligned &gt;1 times
## 62.51% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2137756/10356583
## 2 1527152/2137756
## 3 5632096/19197284
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0428-TP_1 and prad-0428-TP_2
## Single-end reads will be in prad-0428-TP_M
## 12514109 sequences in the BAM file
## 12514109 sequences exported
## 12514109 reads; of these:
##   12514109 (100.00%) were unpaired; of these:
##     3032031 (24.23%) aligned 0 times
##     7855152 (62.77%) aligned exactly 1 time
##     1626926 (13.00%) aligned &gt;1 times
## 75.77% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1626926/9482078
## 2 1249457/1626926
## 3 5100271/17595226
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0429-TP_1 and prad-0429-TP_2
## Single-end reads will be in prad-0429-TP_M
## 26908231 sequences in the BAM file
## 26908231 sequences exported
## 26908231 reads; of these:
##   26908231 (100.00%) were unpaired; of these:
##     7762014 (28.85%) aligned 0 times
##     16382859 (60.88%) aligned exactly 1 time
##     2763358 (10.27%) aligned &gt;1 times
## 71.15% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2763358/19146217
## 2 1980532/2763358
## 3 9096921/33362384
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 4.
## Output will be in prad-0430-TP_1 and prad-0430-TP_2
## Single-end reads will be in prad-0430-TP_M
## 17168741 sequences in the BAM file
## 17168741 sequences exported
## 17168741 reads; of these:
##   17168741 (100.00%) were unpaired; of these:
##     3798281 (22.12%) aligned 0 times
##     11257255 (65.57%) aligned exactly 1 time
##     2113205 (12.31%) aligned &gt;1 times
## 77.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2113205/13370460
## 2 1604583/2113205
## 3 5752138/21954827
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 7.
## Output will be in prad-0431-TP_1 and prad-0431-TP_2
## Single-end reads will be in prad-0431-TP_M
## 8737749 sequences in the BAM file
## 8737749 sequences exported
## 8737749 reads; of these:
##   8737749 (100.00%) were unpaired; of these:
##     3334334 (38.16%) aligned 0 times
##     4320948 (49.45%) aligned exactly 1 time
##     1082467 (12.39%) aligned &gt;1 times
## 61.84% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1082467/5403415
## 2 718093/1082467
## 3 2447956/9279186
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 7.
## Output will be in prad-0432-TP_1 and prad-0432-TP_2
## Single-end reads will be in prad-0432-TP_M
## 11552382 sequences in the BAM file
## 11552382 sequences exported
## 11552382 reads; of these:
##   11552382 (100.00%) were unpaired; of these:
##     4375938 (37.88%) aligned 0 times
##     5502860 (47.63%) aligned exactly 1 time
##     1673584 (14.49%) aligned &gt;1 times
## 62.12% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1673584/7176444
## 2 1279572/1673584
## 3 4997415/15086441
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0433-TP_1 and prad-0433-TP_2
## Single-end reads will be in prad-0433-TP_M
## 10297883 sequences in the BAM file
## 10297883 sequences exported
## 10297883 reads; of these:
##   10297883 (100.00%) were unpaired; of these:
##     3419150 (33.20%) aligned 0 times
##     5139988 (49.91%) aligned exactly 1 time
##     1738745 (16.88%) aligned &gt;1 times
## 66.80% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1738745/6878733
## 2 1310314/1738745
## 3 5343238/15447152
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0434-TP_1 and prad-0434-TP_2
## Single-end reads will be in prad-0434-TP_M
## 18594815 sequences in the BAM file
## 18594815 sequences exported
## 18594815 reads; of these:
##   18594815 (100.00%) were unpaired; of these:
##     5307970 (28.55%) aligned 0 times
##     9996263 (53.76%) aligned exactly 1 time
##     3290582 (17.70%) aligned &gt;1 times
## 71.45% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3290582/13286845
## 2 2578230/3290582
## 3 11800809/33499002
## [bam_sort_core] merging from 7 files...
## This looks like paired data from lane 1.
## Output will be in prad-0435-TP_1 and prad-0435-TP_2
## Single-end reads will be in prad-0435-TP_M
## 13817767 sequences in the BAM file
## 13817767 sequences exported
## 13817767 reads; of these:
##   13817767 (100.00%) were unpaired; of these:
##     4033481 (29.19%) aligned 0 times
##     7988320 (57.81%) aligned exactly 1 time
##     1795966 (13.00%) aligned &gt;1 times
## 70.81% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1795966/9784286
## 2 1381618/1795966
## 3 5682296/18446886
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0436-TP_1 and prad-0436-TP_2
## Single-end reads will be in prad-0436-TP_M
## 10512323 sequences in the BAM file
## 10512323 sequences exported
## 10512323 reads; of these:
##   10512323 (100.00%) were unpaired; of these:
##     3342130 (31.79%) aligned 0 times
##     4878590 (46.41%) aligned exactly 1 time
##     2291603 (21.80%) aligned &gt;1 times
## 68.21% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2291603/7170193
## 2 1704217/2291603
## 3 5421606/15783738
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0437-TP_1 and prad-0437-TP_2
## Single-end reads will be in prad-0437-TP_M
## 10873564 sequences in the BAM file
## 10873564 sequences exported
## 10873564 reads; of these:
##   10873564 (100.00%) were unpaired; of these:
##     3075978 (28.29%) aligned 0 times
##     6125564 (56.33%) aligned exactly 1 time
##     1672022 (15.38%) aligned &gt;1 times
## 71.71% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1672022/7797586
## 2 1316209/1672022
## 3 4790927/15033406
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0438-TP_1 and prad-0438-TP_2
## Single-end reads will be in prad-0438-TP_M
## 9837452 sequences in the BAM file
## 9837452 sequences exported
## 9837452 reads; of these:
##   9837452 (100.00%) were unpaired; of these:
##     2893963 (29.42%) aligned 0 times
##     4875975 (49.57%) aligned exactly 1 time
##     2067514 (21.02%) aligned &gt;1 times
## 70.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2067514/6943489
## 2 1547831/2067514
## 3 12684993/30022750
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0439-TP_1 and prad-0439-TP_2
## Single-end reads will be in prad-0439-TP_M
## 12457346 sequences in the BAM file
## 12457346 sequences exported
## 12457346 reads; of these:
##   12457346 (100.00%) were unpaired; of these:
##     3416368 (27.42%) aligned 0 times
##     7198542 (57.79%) aligned exactly 1 time
##     1842436 (14.79%) aligned &gt;1 times
## 72.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1842436/9040978
## 2 1360573/1842436
## 3 6575268/19744550
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0440-TP_1 and prad-0440-TP_2
## Single-end reads will be in prad-0440-TP_M
## 13971556 sequences in the BAM file
## 13971556 sequences exported
## 13971556 reads; of these:
##   13971556 (100.00%) were unpaired; of these:
##     3960566 (28.35%) aligned 0 times
##     8239580 (58.97%) aligned exactly 1 time
##     1771410 (12.68%) aligned &gt;1 times
## 71.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1771410/10010990
## 2 1347902/1771410
## 3 6573849/20782648
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0441-TP_1 and prad-0441-TP_2
## Single-end reads will be in prad-0441-TP_M
## 19959686 sequences in the BAM file
## 19959686 sequences exported
## 19959686 reads; of these:
##   19959686 (100.00%) were unpaired; of these:
##     5465971 (27.39%) aligned 0 times
##     12666060 (63.46%) aligned exactly 1 time
##     1827655 (9.16%) aligned &gt;1 times
## 72.61% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1827655/14493715
## 2 1307601/1827655
## 3 7048751/25967423
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 1.
## Output will be in prad-0442-TP_1 and prad-0442-TP_2
## Single-end reads will be in prad-0442-TP_M
## 10557257 sequences in the BAM file
## 10557257 sequences exported
## 10557257 reads; of these:
##   10557257 (100.00%) were unpaired; of these:
##     3331690 (31.56%) aligned 0 times
##     5910271 (55.98%) aligned exactly 1 time
##     1315296 (12.46%) aligned &gt;1 times
## 68.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1315296/7225567
## 2 1016238/1315296
## 3 4201713/13988497
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0443-TP_1 and prad-0443-TP_2
## Single-end reads will be in prad-0443-TP_M
## 7107780 sequences in the BAM file
## 7107780 sequences exported
## 7107780 reads; of these:
##   7107780 (100.00%) were unpaired; of these:
##     2563769 (36.07%) aligned 0 times
##     3381762 (47.58%) aligned exactly 1 time
##     1162249 (16.35%) aligned &gt;1 times
## 63.93% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1162249/4544011
## 2 924596/1162249
## 3 4063699/11034174
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0444-TP_1 and prad-0444-TP_2
## Single-end reads will be in prad-0444-TP_M
## 4146990 sequences in the BAM file
## 4146990 sequences exported
## 4146990 reads; of these:
##   4146990 (100.00%) were unpaired; of these:
##     1299983 (31.35%) aligned 0 times
##     2011977 (48.52%) aligned exactly 1 time
##     835030 (20.14%) aligned &gt;1 times
## 68.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 835030/2847007
## 2 659997/835030
## 3 2957882/7542142
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0445-TP_1 and prad-0445-TP_2
## Single-end reads will be in prad-0445-TP_M
## 6869668 sequences in the BAM file
## 6869668 sequences exported
## 6869668 reads; of these:
##   6869668 (100.00%) were unpaired; of these:
##     2127607 (30.97%) aligned 0 times
##     3478513 (50.64%) aligned exactly 1 time
##     1263548 (18.39%) aligned &gt;1 times
## 69.03% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1263548/4742061
## 2 949734/1263548
## 3 3606832/10725604
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 6.
## Output will be in prad-0446-TP_1 and prad-0446-TP_2
## Single-end reads will be in prad-0446-TP_M
## 5713660 sequences in the BAM file
## 5713660 sequences exported
## 5713660 reads; of these:
##   5713660 (100.00%) were unpaired; of these:
##     1914358 (33.50%) aligned 0 times
##     3007695 (52.64%) aligned exactly 1 time
##     791607 (13.85%) aligned &gt;1 times
## 66.50% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 791607/3799302
## 2 585739/791607
## 3 2616928/8117945
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0447-TP_1 and prad-0447-TP_2
## Single-end reads will be in prad-0447-TP_M
## 9605216 sequences in the BAM file
## 9605216 sequences exported
## 9605216 reads; of these:
##   9605216 (100.00%) were unpaired; of these:
##     2940116 (30.61%) aligned 0 times
##     5246051 (54.62%) aligned exactly 1 time
##     1419049 (14.77%) aligned &gt;1 times
## 69.39% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1419049/6665100
## 2 1068850/1419049
## 3 4491323/13907821
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0448-TP_1 and prad-0448-TP_2
## Single-end reads will be in prad-0448-TP_M
## 12742434 sequences in the BAM file
## 12742434 sequences exported
## 12742434 reads; of these:
##   12742434 (100.00%) were unpaired; of these:
##     3657738 (28.71%) aligned 0 times
##     7343747 (57.63%) aligned exactly 1 time
##     1740949 (13.66%) aligned &gt;1 times
## 71.29% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1740949/9084696
## 2 1344620/1740949
## 3 4585760/16063867
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0449-TP_1 and prad-0449-TP_2
## Single-end reads will be in prad-0449-TP_M
## 8939192 sequences in the BAM file
## 8939192 sequences exported
## 8939192 reads; of these:
##   8939192 (100.00%) were unpaired; of these:
##     2590430 (28.98%) aligned 0 times
##     4761170 (53.26%) aligned exactly 1 time
##     1587592 (17.76%) aligned &gt;1 times
## 71.02% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1587592/6348762
## 2 1177629/1587592
## 3 8934787/21769008
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 4.
## Output will be in prad-0450-TP_1 and prad-0450-TP_2
## Single-end reads will be in prad-0450-TP_M
## 25132195 sequences in the BAM file
## 25132195 sequences exported
## 25132195 reads; of these:
##   25132195 (100.00%) were unpaired; of these:
##     12271802 (48.83%) aligned 0 times
##     8970420 (35.69%) aligned exactly 1 time
##     3889973 (15.48%) aligned &gt;1 times
## 51.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3889973/12860393
## 2 2823162/3889973
## 3 20674126/49267244
## [bam_sort_core] merging from 9 files...
## This looks like paired data from lane 1.
## Output will be in prad-0451-TP_1 and prad-0451-TP_2
## Single-end reads will be in prad-0451-TP_M
## 5505949 sequences in the BAM file
## 5505949 sequences exported
## 5505949 reads; of these:
##   5505949 (100.00%) were unpaired; of these:
##     1807387 (32.83%) aligned 0 times
##     2527396 (45.90%) aligned exactly 1 time
##     1171166 (21.27%) aligned &gt;1 times
## 67.17% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1171166/3698562
## 2 935268/1171166
## 3 4944742/12195595
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0452-TP_1 and prad-0452-TP_2
## Single-end reads will be in prad-0452-TP_M
## 10977093 sequences in the BAM file
## 10977093 sequences exported
## 10977093 reads; of these:
##   10977093 (100.00%) were unpaired; of these:
##     3135081 (28.56%) aligned 0 times
##     6533390 (59.52%) aligned exactly 1 time
##     1308622 (11.92%) aligned &gt;1 times
## 71.44% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1308622/7842012
## 2 998837/1308622
## 3 5153131/16381485
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0453-TP_1 and prad-0453-TP_2
## Single-end reads will be in prad-0453-TP_M
## 6612027 sequences in the BAM file
## 6612027 sequences exported
## 6612027 reads; of these:
##   6612027 (100.00%) were unpaired; of these:
##     1863868 (28.19%) aligned 0 times
##     3182065 (48.13%) aligned exactly 1 time
##     1566094 (23.69%) aligned &gt;1 times
## 71.81% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1566094/4748159
## 2 1196711/1566094
## 3 8224833/19637779
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 6.
## Output will be in prad-0454-TP_1 and prad-0454-TP_2
## Single-end reads will be in prad-0454-TP_M
## 7146416 sequences in the BAM file
## 7146416 sequences exported
## 7146416 reads; of these:
##   7146416 (100.00%) were unpaired; of these:
##     2401562 (33.61%) aligned 0 times
##     3627677 (50.76%) aligned exactly 1 time
##     1117177 (15.63%) aligned &gt;1 times
## 66.39% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1117177/4744854
## 2 826271/1117177
## 3 4106609/11612046
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0455-TP_1 and prad-0455-TP_2
## Single-end reads will be in prad-0455-TP_M
## 19268466 sequences in the BAM file
## 19268466 sequences exported
## 19268466 reads; of these:
##   19268466 (100.00%) were unpaired; of these:
##     5996740 (31.12%) aligned 0 times
##     11288782 (58.59%) aligned exactly 1 time
##     1982944 (10.29%) aligned &gt;1 times
## 68.88% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1982944/13271726
## 2 1489723/1982944
## 3 5133643/20937872
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0456-TP_1 and prad-0456-TP_2
## Single-end reads will be in prad-0456-TP_M
## 12061038 sequences in the BAM file
## 12061038 sequences exported
## 12061038 reads; of these:
##   12061038 (100.00%) were unpaired; of these:
##     3608870 (29.92%) aligned 0 times
##     6856565 (56.85%) aligned exactly 1 time
##     1595603 (13.23%) aligned &gt;1 times
## 70.08% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1595603/8452168
## 2 1202491/1595603
## 3 3786455/14186232
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0457-TP_1 and prad-0457-TP_2
## Single-end reads will be in prad-0457-TP_M
## 7101837 sequences in the BAM file
## 7101837 sequences exported
## 7101837 reads; of these:
##   7101837 (100.00%) were unpaired; of these:
##     2529909 (35.62%) aligned 0 times
##     3346828 (47.13%) aligned exactly 1 time
##     1225100 (17.25%) aligned &gt;1 times
## 64.38% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1225100/4571928
## 2 857638/1225100
## 3 4447333/11715868
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0458-TP_1 and prad-0458-TP_2
## Single-end reads will be in prad-0458-TP_M
## 9118555 sequences in the BAM file
## 9118555 sequences exported
## 9118555 reads; of these:
##   9118555 (100.00%) were unpaired; of these:
##     2484355 (27.25%) aligned 0 times
##     5091724 (55.84%) aligned exactly 1 time
##     1542476 (16.92%) aligned &gt;1 times
## 72.75% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1542476/6634200
## 2 1187568/1542476
## 3 4354830/13413154
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0459-TP_1 and prad-0459-TP_2
## Single-end reads will be in prad-0459-TP_M
## 14600839 sequences in the BAM file
## 14600839 sequences exported
## 14600839 reads; of these:
##   14600839 (100.00%) were unpaired; of these:
##     4482828 (30.70%) aligned 0 times
##     8022897 (54.95%) aligned exactly 1 time
##     2095114 (14.35%) aligned &gt;1 times
## 69.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2095114/10118011
## 2 1497715/2095114
## 3 7935470/23764327
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0460-TP_1 and prad-0460-TP_2
## Single-end reads will be in prad-0460-TP_M
## 10710643 sequences in the BAM file
## 10710643 sequences exported
## 10710643 reads; of these:
##   10710643 (100.00%) were unpaired; of these:
##     3010530 (28.11%) aligned 0 times
##     6393708 (59.69%) aligned exactly 1 time
##     1306405 (12.20%) aligned &gt;1 times
## 71.89% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1306405/7700113
## 2 965499/1306405
## 3 3961363/13800531
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0461-TP_1 and prad-0461-TP_2
## Single-end reads will be in prad-0461-TP_M
## 15968211 sequences in the BAM file
## 15968211 sequences exported
## 15968211 reads; of these:
##   15968211 (100.00%) were unpaired; of these:
##     4631697 (29.01%) aligned 0 times
##     9786299 (61.29%) aligned exactly 1 time
##     1550215 (9.71%) aligned &gt;1 times
## 70.99% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1550215/11336514
## 2 1155182/1550215
## 3 6584991/21914448
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0462-TP_1 and prad-0462-TP_2
## Single-end reads will be in prad-0462-TP_M
## 11097995 sequences in the BAM file
## 11097995 sequences exported
## 11097995 reads; of these:
##   11097995 (100.00%) were unpaired; of these:
##     2962649 (26.70%) aligned 0 times
##     5674202 (51.13%) aligned exactly 1 time
##     2461144 (22.18%) aligned &gt;1 times
## 73.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2461144/8135346
## 2 1857352/2461144
## 3 14095742/31562711
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0463-TP_1 and prad-0463-TP_2
## Single-end reads will be in prad-0463-TP_M
## 16787570 sequences in the BAM file
## 16787570 sequences exported
## 16787570 reads; of these:
##   16787570 (100.00%) were unpaired; of these:
##     4328717 (25.79%) aligned 0 times
##     10124114 (60.31%) aligned exactly 1 time
##     2334739 (13.91%) aligned &gt;1 times
## 74.21% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2334739/12458853
## 2 1681181/2334739
## 3 4831215/19735501
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 3.
## Output will be in prad-0464-TP_1 and prad-0464-TP_2
## Single-end reads will be in prad-0464-TP_M
## 15854252 sequences in the BAM file
## 15854252 sequences exported
## 15854252 reads; of these:
##   15854252 (100.00%) were unpaired; of these:
##     4538748 (28.63%) aligned 0 times
##     9225572 (58.19%) aligned exactly 1 time
##     2089932 (13.18%) aligned &gt;1 times
## 71.37% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2089932/11315504
## 2 1613227/2089932
## 3 7029144/22192358
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 1.
## Output will be in prad-0465-TP_1 and prad-0465-TP_2
## Single-end reads will be in prad-0465-TP_M
## 10519689 sequences in the BAM file
## 10519689 sequences exported
## 10519689 reads; of these:
##   10519689 (100.00%) were unpaired; of these:
##     3203835 (30.46%) aligned 0 times
##     5895951 (56.05%) aligned exactly 1 time
##     1419903 (13.50%) aligned &gt;1 times
## 69.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1419903/7315854
## 2 1084989/1419903
## 3 4989108/15156853
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0466-TP_1 and prad-0466-TP_2
## Single-end reads will be in prad-0466-TP_M
## 7151707 sequences in the BAM file
## 7151707 sequences exported
## 7151707 reads; of these:
##   7151707 (100.00%) were unpaired; of these:
##     1837310 (25.69%) aligned 0 times
##     4191680 (58.61%) aligned exactly 1 time
##     1122717 (15.70%) aligned &gt;1 times
## 74.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1122717/5314397
## 2 823257/1122717
## 3 3682449/11472462
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0467-TP_1 and prad-0467-TP_2
## Single-end reads will be in prad-0467-TP_M
## 10012766 sequences in the BAM file
## 10012766 sequences exported
## 10012766 reads; of these:
##   10012766 (100.00%) were unpaired; of these:
##     3512791 (35.08%) aligned 0 times
##     5087040 (50.81%) aligned exactly 1 time
##     1412935 (14.11%) aligned &gt;1 times
## 64.92% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1412935/6499975
## 2 1035493/1412935
## 3 4512571/14187358
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0468-TP_1 and prad-0468-TP_2
## Single-end reads will be in prad-0468-TP_M
## 4830653 sequences in the BAM file
## 4830653 sequences exported
## 4830653 reads; of these:
##   4830653 (100.00%) were unpaired; of these:
##     1378457 (28.54%) aligned 0 times
##     2460500 (50.94%) aligned exactly 1 time
##     991696 (20.53%) aligned &gt;1 times
## 71.46% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 991696/3452196
## 2 768144/991696
## 3 3238163/8669984
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 5.
## Output will be in prad-0469-TP_1 and prad-0469-TP_2
## Single-end reads will be in prad-0469-TP_M
## 4909935 sequences in the BAM file
## 4909935 sequences exported
## 4909935 reads; of these:
##   4909935 (100.00%) were unpaired; of these:
##     1425818 (29.04%) aligned 0 times
##     2441338 (49.72%) aligned exactly 1 time
##     1042779 (21.24%) aligned &gt;1 times
## 70.96% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1042779/3484117
## 2 811043/1042779
## 3 3412893/8969792
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 1.
## Output will be in prad-0470-TP_1 and prad-0470-TP_2
## Single-end reads will be in prad-0470-TP_M
## 9773303 sequences in the BAM file
## 9773303 sequences exported
## 9773303 reads; of these:
##   9773303 (100.00%) were unpaired; of these:
##     3010608 (30.80%) aligned 0 times
##     5203913 (53.25%) aligned exactly 1 time
##     1558782 (15.95%) aligned &gt;1 times
## 69.20% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1558782/6762695
## 2 1125153/1558782
## 3 5653974/15926452
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0471-TP_1 and prad-0471-TP_2
## Single-end reads will be in prad-0471-TP_M
## 11773926 sequences in the BAM file
## 11773926 sequences exported
## 11773926 reads; of these:
##   11773926 (100.00%) were unpaired; of these:
##     3148830 (26.74%) aligned 0 times
##     6821830 (57.94%) aligned exactly 1 time
##     1803266 (15.32%) aligned &gt;1 times
## 73.26% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1803266/8625096
## 2 1374470/1803266
## 3 6221590/18904742
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 1.
## Output will be in prad-0472-TP_1 and prad-0472-TP_2
## Single-end reads will be in prad-0472-TP_M
## 15113275 sequences in the BAM file
## 15113275 sequences exported
## 15113275 reads; of these:
##   15113275 (100.00%) were unpaired; of these:
##     4647461 (30.75%) aligned 0 times
##     8575919 (56.74%) aligned exactly 1 time
##     1889895 (12.50%) aligned &gt;1 times
## 69.25% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1889895/10465814
## 2 1433886/1889895
## 3 5109609/18454268
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 5.
## Output will be in prad-0473-TP_1 and prad-0473-TP_2
## Single-end reads will be in prad-0473-TP_M
## 14402327 sequences in the BAM file
## 14402327 sequences exported
## 14402327 reads; of these:
##   14402327 (100.00%) were unpaired; of these:
##     4655021 (32.32%) aligned 0 times
##     6571116 (45.63%) aligned exactly 1 time
##     3176190 (22.05%) aligned &gt;1 times
## 67.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3176190/9747306
## 2 2661776/3176190
## 3 8678413/23690900
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0474-TP_1 and prad-0474-TP_2
## Single-end reads will be in prad-0474-TP_M
## 12273999 sequences in the BAM file
## 12273999 sequences exported
## 12273999 reads; of these:
##   12273999 (100.00%) were unpaired; of these:
##     3429545 (27.94%) aligned 0 times
##     7133216 (58.12%) aligned exactly 1 time
##     1711238 (13.94%) aligned &gt;1 times
## 72.06% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1711238/8844454
## 2 1296589/1711238
## 3 5804309/18411008
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 3.
## Output will be in prad-0475-TP_1 and prad-0475-TP_2
## Single-end reads will be in prad-0475-TP_M
## 8353384 sequences in the BAM file
## 8353384 sequences exported
## 8353384 reads; of these:
##   8353384 (100.00%) were unpaired; of these:
##     2607286 (31.21%) aligned 0 times
##     3977858 (47.62%) aligned exactly 1 time
##     1768240 (21.17%) aligned &gt;1 times
## 68.79% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1768240/5746098
## 2 1273071/1768240
## 3 5501312/14288247
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 3.
## Output will be in prad-0476-TP_1 and prad-0476-TP_2
## Single-end reads will be in prad-0476-TP_M
## 20265788 sequences in the BAM file
## 20265788 sequences exported
## 20265788 reads; of these:
##   20265788 (100.00%) were unpaired; of these:
##     5226477 (25.79%) aligned 0 times
##     12302340 (60.70%) aligned exactly 1 time
##     2736971 (13.51%) aligned &gt;1 times
## 74.21% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2736971/15039311
## 2 2040801/2736971
## 3 8150435/27426381
## [bam_sort_core] merging from 6 files...
## This looks like paired data from lane 2.
## Output will be in prad-0477-TP_1 and prad-0477-TP_2
## Single-end reads will be in prad-0477-TP_M
## 7357825 sequences in the BAM file
## 7357825 sequences exported
## 7357825 reads; of these:
##   7357825 (100.00%) were unpaired; of these:
##     1971110 (26.79%) aligned 0 times
##     3788221 (51.49%) aligned exactly 1 time
##     1598494 (21.73%) aligned &gt;1 times
## 73.21% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1598494/5386715
## 2 1218286/1598494
## 3 5705120/14893987
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0478-TP_1 and prad-0478-TP_2
## Single-end reads will be in prad-0478-TP_M
## 7417700 sequences in the BAM file
## 7417700 sequences exported
## 7417700 reads; of these:
##   7417700 (100.00%) were unpaired; of these:
##     2356358 (31.77%) aligned 0 times
##     3572447 (48.16%) aligned exactly 1 time
##     1488895 (20.07%) aligned &gt;1 times
## 68.23% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1488895/5061342
## 2 1146024/1488895
## 3 6481205/15025117
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0479-TP_1 and prad-0479-TP_2
## Single-end reads will be in prad-0479-TP_M
## 13207324 sequences in the BAM file
## 13207324 sequences exported
## 13207324 reads; of these:
##   13207324 (100.00%) were unpaired; of these:
##     4551902 (34.46%) aligned 0 times
##     6507649 (49.27%) aligned exactly 1 time
##     2147773 (16.26%) aligned &gt;1 times
## 65.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2147773/8655422
## 2 1594142/2147773
## 3 9017111/24007358
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 5.
## Output will be in prad-0480-TP_1 and prad-0480-TP_2
## Single-end reads will be in prad-0480-TP_M
## 7683685 sequences in the BAM file
## 7683685 sequences exported
## 7683685 reads; of these:
##   7683685 (100.00%) were unpaired; of these:
##     2363414 (30.76%) aligned 0 times
##     4008031 (52.16%) aligned exactly 1 time
##     1312240 (17.08%) aligned &gt;1 times
## 69.24% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1312240/5320271
## 2 1012300/1312240
## 3 3493772/10808383
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 6.
## Output will be in prad-0481-TP_1 and prad-0481-TP_2
## Single-end reads will be in prad-0481-TP_M
## 9051480 sequences in the BAM file
## 9051480 sequences exported
## 9051480 reads; of these:
##   9051480 (100.00%) were unpaired; of these:
##     3106138 (34.32%) aligned 0 times
##     4650252 (51.38%) aligned exactly 1 time
##     1295090 (14.31%) aligned &gt;1 times
## 65.68% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1295090/5945342
## 2 990422/1295090
## 3 5451277/15137450
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0482-TP_1 and prad-0482-TP_2
## Single-end reads will be in prad-0482-TP_M
## 8665447 sequences in the BAM file
## 8665447 sequences exported
## 8665447 reads; of these:
##   8665447 (100.00%) were unpaired; of these:
##     2381846 (27.49%) aligned 0 times
##     4537077 (52.36%) aligned exactly 1 time
##     1746524 (20.16%) aligned &gt;1 times
## 72.51% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1746524/6283601
## 2 1434543/1746524
## 3 4421443/12897209
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0483-TP_1 and prad-0483-TP_2
## Single-end reads will be in prad-0483-TP_M
## 9264602 sequences in the BAM file
## 9264602 sequences exported
## 9264602 reads; of these:
##   9264602 (100.00%) were unpaired; of these:
##     2822975 (30.47%) aligned 0 times
##     4872577 (52.59%) aligned exactly 1 time
##     1569050 (16.94%) aligned &gt;1 times
## 69.53% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1569050/6441627
## 2 1243997/1569050
## 3 4742554/13854757
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0484-TP_1 and prad-0484-TP_2
## Single-end reads will be in prad-0484-TP_M
## 7015258 sequences in the BAM file
## 7015258 sequences exported
## 7015258 reads; of these:
##   7015258 (100.00%) were unpaired; of these:
##     2153841 (30.70%) aligned 0 times
##     3867328 (55.13%) aligned exactly 1 time
##     994089 (14.17%) aligned &gt;1 times
## 69.30% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 994089/4861417
## 2 732025/994089
## 3 3416310/10573636
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 5.
## Output will be in prad-0485-TP_1 and prad-0485-TP_2
## Single-end reads will be in prad-0485-TP_M
## 5083569 sequences in the BAM file
## 5083569 sequences exported
## 5083569 reads; of these:
##   5083569 (100.00%) were unpaired; of these:
##     1451180 (28.55%) aligned 0 times
##     2287519 (45.00%) aligned exactly 1 time
##     1344870 (26.46%) aligned &gt;1 times
## 71.45% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1344870/3632389
## 2 1086560/1344870
## 3 3949127/10114291
## [bam_sort_core] merging from 2 files...
## This looks like paired data from lane 2.
## Output will be in prad-0486-TP_1 and prad-0486-TP_2
## Single-end reads will be in prad-0486-TP_M
## 10286688 sequences in the BAM file
## 10286688 sequences exported
## 10286688 reads; of these:
##   10286688 (100.00%) were unpaired; of these:
##     2815326 (27.37%) aligned 0 times
##     5827281 (56.65%) aligned exactly 1 time
##     1644081 (15.98%) aligned &gt;1 times
## 72.63% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1644081/7471362
## 2 1303163/1644081
## 3 4061976/13289894
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0487-TP_1 and prad-0487-TP_2
## Single-end reads will be in prad-0487-TP_M
## 7932312 sequences in the BAM file
## 7932312 sequences exported
## 7932312 reads; of these:
##   7932312 (100.00%) were unpaired; of these:
##     2486861 (31.35%) aligned 0 times
##     4129421 (52.06%) aligned exactly 1 time
##     1316030 (16.59%) aligned &gt;1 times
## 68.65% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1316030/5445451
## 2 991769/1316030
## 3 5125660/14181037
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 1.
## Output will be in prad-0488-TP_1 and prad-0488-TP_2
## Single-end reads will be in prad-0488-TP_M
## 7337883 sequences in the BAM file
## 7337883 sequences exported
## 7337883 reads; of these:
##   7337883 (100.00%) were unpaired; of these:
##     2316878 (31.57%) aligned 0 times
##     3639731 (49.60%) aligned exactly 1 time
##     1381274 (18.82%) aligned &gt;1 times
## 68.43% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1381274/5021005
## 2 1050816/1381274
## 3 6289159/16244654
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0489-TP_1 and prad-0489-TP_2
## Single-end reads will be in prad-0489-TP_M
## 14860225 sequences in the BAM file
## 14860225 sequences exported
## 14860225 reads; of these:
##   14860225 (100.00%) were unpaired; of these:
##     3634300 (24.46%) aligned 0 times
##     8848333 (59.54%) aligned exactly 1 time
##     2377592 (16.00%) aligned &gt;1 times
## 75.54% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2377592/11225925
## 2 1873730/2377592
## 3 7079403/22013306
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 2.
## Output will be in prad-0490-TP_1 and prad-0490-TP_2
## Single-end reads will be in prad-0490-TP_M
## 7638564 sequences in the BAM file
## 7638564 sequences exported
## 7638564 reads; of these:
##   7638564 (100.00%) were unpaired; of these:
##     2099985 (27.49%) aligned 0 times
##     3477585 (45.53%) aligned exactly 1 time
##     2060994 (26.98%) aligned &gt;1 times
## 72.51% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 2060994/5538579
## 2 1557362/2060994
## 3 13165800/29752334
## [bam_sort_core] merging from 5 files...
## This looks like paired data from lane 6.
## Output will be in prad-0491-TP_1 and prad-0491-TP_2
## Single-end reads will be in prad-0491-TP_M
## 11508563 sequences in the BAM file
## 11508563 sequences exported
## 11508563 reads; of these:
##   11508563 (100.00%) were unpaired; of these:
##     3846476 (33.42%) aligned 0 times
##     6315891 (54.88%) aligned exactly 1 time
##     1346196 (11.70%) aligned &gt;1 times
## 66.58% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1346196/7662087
## 2 1019756/1346196
## 3 3970193/13929877
## [bam_sort_core] merging from 3 files...
## This looks like paired data from lane 2.
## Output will be in prad-0492-TP_1 and prad-0492-TP_2
## Single-end reads will be in prad-0492-TP_M
## 12203245 sequences in the BAM file
## 12203245 sequences exported
## 12203245 reads; of these:
##   12203245 (100.00%) were unpaired; of these:
##     3293126 (26.99%) aligned 0 times
##     6933527 (56.82%) aligned exactly 1 time
##     1976592 (16.20%) aligned &gt;1 times
## 73.01% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1976592/8910119
## 2 1530906/1976592
## 3 7746636/21922725
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0493-TP_1 and prad-0493-TP_2
## Single-end reads will be in prad-0493-TP_M
## 10966512 sequences in the BAM file
## 10966512 sequences exported
## 10966512 reads; of these:
##   10966512 (100.00%) were unpaired; of these:
##     2993332 (27.30%) aligned 0 times
##     4894816 (44.63%) aligned exactly 1 time
##     3078364 (28.07%) aligned &gt;1 times
## 72.70% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 3078364/7973180
## 2 2554154/3078364
## 3 7526150/19617973
## [bam_sort_core] merging from 4 files...
## This looks like paired data from lane 2.
## Output will be in prad-0494-TP_1 and prad-0494-TP_2
## Single-end reads will be in prad-0494-TP_M
## 9647053 sequences in the BAM file
## 9647053 sequences exported
## 9647053 reads; of these:
##   9647053 (100.00%) were unpaired; of these:
##     2671506 (27.69%) aligned 0 times
##     5503240 (57.05%) aligned exactly 1 time
##     1472307 (15.26%) aligned &gt;1 times
## 72.31% overall alignment rate
## [samopen] SAM header is present: 93 sequences.
## [samopen] SAM header is present: 93 sequences.
## 1 1472307/6975547
## 2 1110403/1472307
## 3 4608759/14372714
## [bam_sort_core] merging from 3 files...
</code></pre>

<p>This code is adapted from external scripts. We iterate through all downloaded BAM files, convert them to fastq using <code>bam2fastq</code>, map them to hg19 and mature tRNAs with <code>bowtie2</code> with a maximum of 100 mappings, and filter out unmapped and low quality reads using <code>samtools</code>**. <code>bowtie2-best-mapped.py</code>, adapted from Andrew&#39;s script, finds the best scoring multimappings from <code>bowtie2</code> output and discards multimappings with lower scores.</p>

<pre><code class="r">save(prad_metadata, file = &#39;prad-metadata.RData&#39;)
save.image(&quot;process-reads-image.RData&quot;)
</code></pre>

</body>

</html>
